---
autotoc: true
---
<div class="title">February 2015 Galaxy Update</div>

<div class='right'>
<table>
  <tr>
  </tr>
</table>

</div>

<div class='left'><a href='/GalaxyUpdates/'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the February *[Galaxy Update](/GalaxyUpdates)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/GalaxyUpdates)* complement the *[Galaxy Development News Briefs](/DevNewsBriefs)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# New Papers

55 new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in January. Some highlights:

* [Open pipelines for integrated tumor genome profiles reveal differences between pancreatic cancer tumors and cell lines](http://dx.doi.org/10.1002/cam4.360)
    [Jeremy Goecks](/JeremyGoecks), Bassel F. El-Rayes, Shishir K. Maithel, H. Jean Khoury, [James Taylor](/JamesTaylor), Michael R. Rossi, *Cancer Medicine* (December 2014), doi:10.1002/cam4.360
* [Merging OpenLifeData with SADI services using Galaxy and Docker](http://www.biorxiv.org/content/biorxiv/early/2015/01/10/013615)
    Mikel E. Aranguren, *bioRxiv*
* [NCBI BLAST+ integrated into Galaxy](http://www.biorxiv.org/content/early/2015/01/21/014043)
    Peter J. A. Cock, [John M. Chilton](/JohnChilton), [Björn Grüning](/BjoernGruening), James E. Johnson, [Nicola Soranzo](/NicolaSoranzo), *bioRxiv*
* [SmileFinder: a resampling-based approach to evaluate signatures of selection from genome-wide sets of matching allele frequency data in two or more diploid populations](http://www.gigasciencejournal.com/content/4/1/1/abstract)
    Wilfried M Guiblet, Kai Zhao, Stephen J O'Brien, Steven E Massey, Alfred L Roca and Taras K Oleksyk, *[GigaScience](http://www.gigasciencejournal.com/)* 2015, 4:1<br />
    [Supplementary materials](http://galaxy.cbiit.cuhk.edu.hk/u/gigascience/g/wguiblet2014) (as a Galaxy Page)

The new papers were related to:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 36 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>



# Events

## GCC2015: 6-8 July, Norwich UK

<div class='left'><a href='http://gcc2015.tsl.ac.uk/'><img src='/Images/Logos/GCC2015LogoWide600.png' alt='Sponsor GCC2015' width="170" /></a></div>

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is the Galaxy community's annual gathering of users, developers, and administrators.  [Previous GCC's](/Events/GCC2014) have drawn over 200 participants, and we expect that to happen again in 2015.  GCC2015 is being hosted by [The Sainsbury Lab](http://tsl.ac.uk/) in Norwich, UK, immediately before [BOSC](http://www.open-bio.org/wiki/BOSC_2015) and [ISMB/ECCB](http://www.iscb.org/ismbeccb2015) in Dublin.

### Early Registration opens in February

*Early* registration (save *heaps)* and [abstract submission](http://gcc2015.tsl.ac.uk/abstracts/) will open later in February.  If you work in data-intensive life science research, then it is hard find a meeting more relevant than GCC2015.  We look forward to seeing you there.
 

### Training Day Topic Voting Closes TODAY

<div class='right'><a href='http://bit.ly/gcc2015vote'><img src='/Images/Logos/GCC2015_TrainingDay128.png' alt='Vote now!' /></a>
<div class='center'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://bit.ly/gcc2015vote'>Vote now!</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div></div>

**[Voting](http://bit.ly/gcc2015vote) on what topics should be offered at the [GCC2015 Training Day](http://gcc2015.tsl.ac.uk/training-day/) closes today, 30 January.**

[Topics](http://gcc2015.tsl.ac.uk/training-day/) for the [GCC2015 Training Day](http://gcc2015.tsl.ac.uk/training-day/) are selected by you, the Galaxy Community.  There are [nominated topics](http://gcc2015.tsl.ac.uk/training-day/) spanning from basic usage to advanced deployment.  No matter what you do with Galaxy, there are topics for you to choose from.  Your vote will determine the topics that are offered, which topics should be offered more than once,  and which ones should not be scheduled at the same time.  Your vote matters. 

The Training Day schedule, including instructors, will be published before early registration opens in February.

**About the [GCC2015 Training Day](http://gcc2015.tsl.ac.uk/training-day/):**

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) will start on 6 July with a [Training Day](http://gcc2015.tsl.ac.uk/training-day/) featuring parallel tracks, each with several multi-hour workshops. There will be at least one complete track  about using Galaxy for biological research, and at least one full track on deploying and managing Galaxy instances.

### Sponsorships

#### GenomeWeb
 
We are delighted to have [GenomeWeb](http://www.genomeweb.com/) as a returning [Silver Sponsor](http://gcc2015.tsl.ac.uk/sponsors) for GCC2015.

<div class='right'><a href='http://www.genomoeweb.com/'><img src='/Images/Logos/GenomeWebLogo300.jpg' alt='GenomeWeb' width="200" /></a></div>

 [GenomeWeb](http://www.genomeweb.com/) is an independent online news organization based in New York. Since 1997, [GenomeWeb](http://www.genomeweb.com/) has served the global community of scientists, technology professionals, and executives who use and develop the latest advanced tools in molecular biology research and molecular diagnostics. Our editorial mission is to cover the scientific and economic ecosystem spurred by the advent of high-throughput genome sequencing. We operate the [largest online newsroom focused on advanced molecular research tools](https://www.genomeweb.com/breaking-news) in order to provide our readers with exclusive news and in-depth analysis of this rapidly evolving market.

#### Globus Genomics

And we are delighted to announce that [Globus Genomics](http://www.globus.org/genomics) is also a returning [Silver Sponsor](http://gcc2015.tsl.ac.uk/sponsors) for GCC2015.

<div class='right'><a href='http://www.globus.org/genomics'><img src='/Images/Logos/GlobusGenomics.png' alt='Globus Genomics' width="175" /></a></div>

 [Globus Genomics](http://www.globus.org/genomics) is an integrated solution for Next Gen Sequencing analysis utilizing technologies in big data management and big data analysis. Globus Genomics combines big data management capabilities of [Globus Online](http://www.globusonline.org) with the Galaxy framework and high throughput computing capabilities on Amazon Web Services.  With [Globus Genomics](http://www.globus.org/genomics) researchers can easily transfer large data sets and analyze the data in Galaxy. Globus Genomics is a non-profit service offering delivered and maintained by the Globus team at the [Computation Institute](https://www.ci.uchicago.edu/), [University of Chicago](https://www.uchicago.edu/). 

#### The Genome Analysis Centre (TGAC)

Finally, please welcome [The Genome Analysis Centre (TGAC)](http://www.tgac.ac.uk) as first-time [Silver Sponsor](http://gcc2015.tsl.ac.uk/sponsors) for GCC!

<div class='right'><a href='http://www.tgac.ac.uk/'><img src='/Images/Logos/TGACLogoWSupport300.jpg' alt='The Genome Analysis Centre' width="185" /></a></div>

 [The Genome Analysis Centre (TGAC)](http://www.tgac.ac.uk) is a world-class research institute focusing on the development of genomics and computational biology. TGAC is based on the [Norwich Research Park](http://www.norwichresearchpark.com/home.aspx) and receives strategic funding from the [Biotechnology and Biological Science Research Council (BBSRC)](http://www.bbsrc.com/home/home.aspx) as well as from other research funders. TGAC offers a state of the art DNA sequencing facility, unique by its operation of multiple complementary technologies for data generation. The Institute hosts one of the largest computing hardware facilities dedicated to life science research in Europe. It is also actively involved in developing novel platforms to provide access to computational tools and processing capacity for academic and industrial users and promoting applications of computational Bioscience. Additionally, the Institute is a member of the [Galaxy Training Network](/Teach/Trainers/#tgac-the-genome-analysis-centre) and offers a [Training programme](http://www.tgac.ac.uk/361_Division/) through courses and workshops, and has an [Education and Public Engagement programme](http://www.tgac.ac.uk/education-public-engagement/) targeting schools, teachers and the general public. 

#### Call for Sponsors

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is now accepting [Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/).  Your organisation can play a prominent part in the Galaxy community by sponsoring GCC2015.  Sponsorship is an excellent way to raise your organization’s visibility.

Several sponsorship levels are available, including two levels of premier sponsorships that include presentations.  Premium sponsorships are limited, however, so you are encouraged to act soon.

Please let the [organisers|GCC2015 Organising Committee](http://gcc2015.tsl.ac.uk/) ([gcc2015-org AT lists DOT galaxyproject DOT org](gcc2015-org AT lists DOT galaxyproject DOT org)) know if you are interested in helping make this event a success.

## 19 February GalaxyAdmins Meetup

<div class='left'><a href='/Community/GalaxyAdmins/Meetups/2015_02_19/'><img src='/Images/Logos/GalaxyAdmins.png' alt='GalaxyAdmins meetup February 19' width="150" /></a></div>

The first [GalaxyAdmins](/Community/GalaxyAdmins) [meetup of 2015](/Community/GalaxyAdmins/Meetups/2015_02_19) will happen online on Thursday, 19 February.  [GalaxyAdmins](/Community/GalaxyAdmins) is a special interest group for Galaxy community members who are responsible for Galaxy installations.

A summary of both the user and admin/developer [2014 Galaxy Community Questionnaires](/News/2014Questionnaire) will be presented by [Dave Clements](/DaveClements), a *Galaxy Project Update* will be offered by a [Galaxy Team member](/GalaxyTeam), and   [Hans-Rudolf Hotz](/HansrudolfHotz) will lead a short discussion on *GalaxyAdmins direction*.

Thanks to everyone for letting us know what dates and times worked best for you.  This time and day of the week worked for a remarkable 91% of respondents.

## January Baltimore Area Galaxy Meetup Report

<div class='right'><a href='/Events/Meetups/Baltimore/2015-01-22/'><img src='/Events/Meetups/Baltimore/GalaxyBaltimoreMeetupLogo400.png' alt='January 2015 Galaxy Baltimore Meetup' width="150" /></a></div>

The [first Baltimore Area Galaxy meetup](/Events/Meetups/Baltimore/2015-01-22) *was sold out.* It was the right mix of current Galaxy users, people who want to learn how to use Galaxy, and students in Biology/Computer science programs. It was good to see academics unwind with a few beers and talk about their ideas to use Galaxy. The idea behind the meetup was to build a local community of Galaxy users, and to get to know one another personally. This would in turn foster collaborations and also bring forward better feedback to the [Galaxy Team](/GalaxyTeam) as well. 

The second meetup is going to be hosted in February, for which we hope to have a slightly more focussed agenda. To talk about one or a few of the topics raised during the course of the month. There is now a [Baltimore newsletter](https://lists.galaxyproject.org/listinfo/baltimore) ([subscribe](https://lists.galaxyproject.org/listinfo/baltimore) if you are interested) which goes out to the members who signed up for the first meetup. We hope to have more people in the coming months, and to grow a stronger Galaxy community in Baltimore. 

[Enis Afgan](/EnisAfgan) & [Nitesh Turaga](/NiteshTuraga)


## Other Events

<div class='center'> 
<a href='http://www.supagro.fr/web/pages/?page=1964&id_page=2666'><img src='/Images/Logos/SupAgroMontpelierLogo.jpg' alt='Analyse bioinformatique de séquences sous Galaxy' height="60" /></a> &nbsp;
<a href='/Community/GalaxyAdmins/Meetups/2015_02_19/'><img src='/Images/Logos/GalaxyAdmins.png' alt='19 February GalaxyAdmins Web Meetup' height="56" /></a> &nbsp;
<a href='http://www.qfab.org/events/upcoming/'><img src='/Images/Logos/QFABLogo.png' alt='QFAB Workshops' height="80" /></a>
<a href='http://training.bioinformatics.ucdavis.edu/2015/01/12/rna-seq-and-chip-seq-analysis-with-galaxy/'><img src='/Images/Logos/UCDavisGenomeCenter.png' alt='RNA-Seq and ChIP-Seq Analysis with Galaxy' height="50" /></a> &nbsp;
</div>


<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif/' /></a></div>

There are [upcoming events](/Events) in 5 countries on 4 continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> February 9-13 </th>
    <td> <em><a href='http://www.supagro.fr/web/pages/?page=1964&id_page=2666'>Analyse bioinformatique de séquences sous Galaxy</a></em> </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnSouth_Green'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> Montpellier, France </td>
    <td> <a href="mailto:jean-francois.dufayard@cirad.fr">J.F. Dufayard</a> </td>
  </tr>
  <tr>
    <th rowspan=3> February 16-18 </th>
    <td> <em>Accessible and Reproducible Large-Scale Analysis with Galaxy</em> </td>
    <td rowspan=1> <a href='http://www.triconference.com/Sequencing-Analysis/'>Genome and Transcriptome Analysis</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='/JamesTaylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <td> <em>Large-Scale NGS data Analysis on Amazon Web Services Using Globus Genomic</em> </td>
    <td rowspan=2> <a href='http://www.triconference.com/Integration-Analysis-Visualization/'>Genomics & Sequencing Data Integration, Analysis and Visualization</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='http://www.ci.anl.gov/profile/250'>Ravi Madduri</a> </td>
  </tr>
  <tr>
    <td> <em>iReport: An Integrative “omics” Reporting and Visualisation Platform</em> </td>
    <td> Andrew Stubbs </td>
  </tr>
  <tr>
    <th> February 19 </th>
    <td> <em><a href='/Community/GalaxyAdmins/Meetups/2015_02_19/'>February 2015 GalaxyAdmins Web Meetup</a></em> </td>
    <td> <em>Online</em> </td>
    <td> <a href='/HansrudolfHotz/'>Hans-Rudolf Hotz</a>, <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> March 2 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-rna-seq-analysis-using-galaxy/'>RNA-Seq analysis using Galaxy</a></em> </td>
    <td> <div class='right'><a href='/Teach/Trainers/#qfab-bioinformatics'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> March 3-5 </th>
    <td> <a href='ATTACHMENT_URLDocuments/Flyers/DéveloppementEtIntégrationDApplicationsSousGalaxy.pdf'>Développement et intégration d’applications sous Galaxy</a> </td>
    <td> <div class='right'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></div> La Chapelle Sur Erdre, France </td>
    <td> <a href="mailto:frederique.malipier@nantes.inra.fr">Frederique Malipier</a> </td>
  </tr>
  <tr>
    <th> March 23-26 </th>
    <td> <a href='http://training.bioinformatics.ucdavis.edu/2015/01/12/rna-seq-and-chip-seq-analysis-with-galaxy/'>RNA-Seq and ChIP-Seq Analysis with Galaxy</a> </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnUCDavis'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> <a href='http://bioinformatics.ucdavis.edu/'>UC Davis Bioinformatics Core</a>, Davis, California, United States </td>
    <td> <a href="mailto:bioinformatics.core@ucdavis.edu">UC Davis Bioinformatics</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> April 14-15 </th>
    <td> <em><a href='http://www.globusworld.org/'>GlobusWorld 2015</a></em> </td>
    <td> Chicago, Illinois, United States </td>
    <td> <a href="mailto:outreach AT globus.org">Globus Outreach</a> </td>
  </tr>
  <tr>
    <th> April 20-21 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-2/'>Workshop: Extended RNA-Seq analysis</a></em> </td>
    <td> <div class='right'><a href='/Teach/Trainers/#qfab-bioinformatics'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> April 28 </th>
    <td> </strong><a href='http://wiki.pitagora-galaxy.org/wiki/index.php/Galaxy_Workshop_Tokyo_2015'>Galaxy Workshop Tokyo 2015</a><strong> </td>
    <td> <a href='http://www.rcast.u-tokyo.ac.jp/index_en.html'>RCAST, The University of Tokyo</a>, Japan </td>
    <td> <a href='http://wiki.pitagora-galaxy.org/wiki/index.php/Galaxy_Workshop_Tokyo_2015#Contact'>Ryota Yamanaka</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> </strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a><strong> </td>
    <td> <div class='right'><a href='/Teach/Trainers/'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div>   <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none"> <a href='/Teach/Trainers/'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none"> <span style="font-size: smaller;"> <a href='/Teach/Trainers/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring/'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Software Developer for the Refinery Team](https://sjobs.brassring.com/tgwebhost/jobdetails.aspx?jobId=1118216), Boston, Massachusetts, United States
* [Genome analysis PostDoc](/News/GenomeAnalysisPostDocOslo) in Oslo
* [Bioinformatics scientist/engineer](/News/MDCBerlinBuchOpening) for RNA bioinformatics and next-generation sequencing, [MDC Berlin-Buch](https://www/c-berlin.de/)  
* [Bioinformatics Research Officer / Postdoc](http://bit.ly/1uUGUyh), Gene and Stem Cell Therapy, [The Centenary Institute](http://www.centenary.org.au/), Sydney, Australia  
* [Postdoc Position](http://bit.ly/1JSbytl), Cancer Bioinformatics, [VIB](http://www.vib.be/), Leuven, Belgium 

* [Senior Development Engineer - Bioinformatics](http://bit.ly/1yGvefn), and [Bioinformatician II](http://bit.ly/1vBhgxB), University of Massachusetts Medical School
* Searching for [bioinformaticians, post-docs, PhD students and software engineers](/News/BioinformaticianFreiburg2014) in Freiburg, Germany at [Max Planck Institute of Immunobiology and Epigenetics](http://www.ie-freiburg.mpg.de/jobs), and the [Bioinformatics Group at the University of Freiburg](http://www.bioinf.uni-freiburg.de//?en)
* **The [Galaxy Project is hiring software engineers and post-docs](/GalaxyIsHiring)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />

# New Public Servers

Several new [public Galaxy servers](/PublicGalaxyServers) were added in January.

## RiboGalaxy

<div class='right'><div class='solid'><a href='http://ribogalaxy.ucc.ie/'><img src='/PublicGalaxyServers/RiboGalaxyLogo.png' alt='RiboGalaxy' width="200" /></a></div>
 
[RiboGalaxy](http://ribogalaxy.ucc.ie/) provides on-line tools for the analysis and visualization of ribo-seq data obtained with the ribosome profiling technique. It is a freely available web server for processing and analysing ribosome profiling (ribo-seq) data with the visualization functionality provided by [GWIPS-viz](http://gwips.ucc.ie/). !RiboGalaxy provides a compact suite of tools specifically tailored for the alignment and visualization of ribo-seq and corresponding mRNA-seq data. Users can take advantage of the published workflows on !RiboGalaxy which reduce the multi-step alignment process to a minimum of inputs.

!RiboGalaxy has its own dedicated [Help page](http://ribogalaxy.ucc.ie/static/help.html). Please post any questions you may have on our [RiboGalaxy Forum](http://gwips.ucc.ie/Forum/). Anonymous users can use !RiboGalaxy. However, the upload and processing of datasets larger than 2GB and the use of advanced features such as published workflows, will require the user to be registered on !RiboGalaxy. 

!RiboGalaxy is supported by [Science Foundation Ireland](http://www.sfi.ie/).


## CardioVascular Research Grid (CVRG)

<div class='left'><div class='solid'><a href='http://cvrg.galaxycloud.org/'><img src='/PublicGalaxyServers/CVRGHome.png' alt='CardioVascular Research Grid (CVRG)' width="200" /></a></div>

The [CardioVascular Research Grid (CVRG)](http://www.cvrgrid.org/) project provides [CVRG Galaxy](http://cvrg.galaxycloud.org/) for secure seamless access to study data and analysis tools.  Users can transfer data to CVRG Galaxy at high speed using [Globus Connect](https://www.globus.org/globus-connect-personal). CVRG Galaxy provides a wide range of analysis algorithms, including [Physionet](http://www.physionet.org/) algorithms for ECG analysis, and stored workflows  that simplify the process of data analysis. CVRG Galaxy also has tools that help users run their analyses faster by using multiple processors on the  Amazon Elastic Compute Cloud. CVRG Galaxy can annotate data by  retrieving ontology terms from the [Bioportal](http://bioportal.bioontology.org/) ontology server.

There is [CVRG Galaxy Wiki Page](http://wiki.cvrgrid.org/index.php/CVRG_Galaxy) for support.  To use CVRG Galaxy, an account is required, and anyone can create an account

"The CVRG project is supported by the [National Heart Lung & Blood Institute](http://www.nhlbi.nih.gov/). The project is based at the [Institute for Computational Medicine](http://www.icm.jhu.edu/) at the Johns Hopkins University, [Department of Biomedical Informatics](https://medschool.vanderbilt.edu/dbmi/) at Vanderbilt University Medical Center, [The College of Computing and Informatics](http://cci.uncc.edu/) at UNC Charlotte, [The Center for Comprehensive Informatics](http://cci.emory.edu/cms/index.html) at Emory University, [The College of Engineering and Applied Sciences](http://www.stonybrook.edu/sb/departments/ceas.shtml) at Stony Brook University, and the [Computation Institute](http://www.ci.anl.gov/) at The University of Chicago."

## Center for Phage Technology (CPT)

<div class='right'><div class='solid'><a href='https://cpt.tamu.edu/galaxy-public/'><img src='/PublicGalaxyServers/CPTLogo.png' alt='Center for Phage Technology (CPT)' width="200" /></a></div>

The [Center for Phage Technology (CPT) Galaxy Server](https://cpt.tamu.edu/galaxy-public/) includes 50 additional tools, and PAUSE (Pile-Up Analysis Using Starts & Ends) V1 and V2 tool sets. There are also several [published workflows](https://cpt.tamu.edu/galaxy-public/workflow/list_published) demonstrating PAUSE for both paired and single-end reads.

An account is required, and anyone can create an account. See [this FAQ](https://cpt.tamu.edu/galaxy-faq-ever-needed-a-question-answered/) or contact [esr+gxfaq@tamu.edu](Eric Rasche) for support.

CPT Galaxy is sponsored by the [Center for Phage Technology (CPT)](https://cpt.tamu.edu/), Texas A&M University

# Galaxy Community Hubs

<div class='solid'>
<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Teach/'><img src='/Images/Logos/GTNLogo600.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Logs/'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Deployments/'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong>Share your <a href='/Teach/Resources/#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers/#add-a-trainer'>experience</a> <em>now</em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Logs/#add-a-log-page'>Share your experience</a> <em><a href='/Community/Logs/#add-a-log-page'>now</a></em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Deployments/#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/Community/Deployments/#add-your-galaxy-deployment'>now</a></em> <strong> </td>
  </tr>
</table>

</div>

<div class='right'><br />
<a href='http://www.irb.hr/eng'><img src='/Images/Logos/IRBLogo300.jpg' alt='Ruđer Bošković Institute (RBI)' width="100" /></a></div>


## New GTN Member: Ruđer Bošković Institute

We are pleased to welcome the [Ruđer Bošković Institute](http://www.irb.hr/) in Zagreb, Croatia as the [newest member](/Teach/Trainers/#ruđer-bošković-institute-rbi) of [Galaxy Training Network (GTN)](/Teach/GTN). Dobrodošli!


# New Releases

## Galaxy 2015/01/13 Distribution

<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GetGalaxyOrg.png' alt='width=175' /></a></div>

**[Complete News Brief](http://wiki.galaxyproject.org/DevNewsBriefs/2015_01_13)**

**Security**

Several *critical* security vulnerabilities were recently discovered by Bartlomiej Balcerek and Mateusz Stahl at the [Wroclaw Centre for Networking and Supercomputing](http://www.wcss.pl/en/). This [stable Galaxy release contains fixes for those vulnerabilities](/DevNewsBriefs/2015_01_13/#security). **The Galaxy Team strongly encourages Galaxy server administrators to update their Galaxy servers immediately.**

**IPython Integration**

Thanks to the awesome work of community members [Björn Grüning](/BjoernGruening) and [Eric Rasche](/EricRasche), [Galaxy now features integration with the popular IPython project](/DevNewsBriefs/2015_01_13/#ipython_integration). The [Galaxy-IPython](https://github.com/bgruening/galaxy-ipython) project has been merged into Galaxy core and made into a generic plugin framework of interactive environments based on Docker. 

** Tool Form Upgrade (for Beta Testing) **

In Galaxy's development branch, the basic tool from has been [redesigned and modernized to address certain limitations in Galaxy's responsiveness when working with longer forms containing multiple parameter choices](/DevNewsBriefs/2015_01_13/#tool_form_upgrade_28for_beta_testing29). This new tool form will become the default with the next release - but we are hoping tool author's and power users enable it and provide feedback during this release cycle in order to ensure it is working ideally when it becomes the default.

** Get The Distribution **

<table>
  <tr>
    <td rowspan=3 style=" border: none"> <a href='http://getgalaxy.org/'><img src='http://galaxy.psu.edu/static/getgalaxy.png' alt='getgalaxy' width=70 /></a> &nbsp;&nbsp; </td>
    <td colspan=2 style=" border: none"> </strong><a href='http://wiki.galaxyproject.org/Admin/Get%20Galaxy'>getgalaxy.org</a><strong> </td>
  </tr>
  <tr>
    <td style=" border: none"> </strong><a href='http://galaxy-dist.readthedocs.org'>galaxy-dist.readthedocs.org</a><strong> </td>
    <td style=" border: none"> </td>
  </tr>
  <tr>
    <td style=" border: none"> </strong><a href='http://bitbucket.org/galaxy/galaxy-dist'>bitbucket.org/galaxy/galaxy-dist</a><strong> </td>
    <td style=" border: none"> </td>
  </tr>
  <tr>
    <td style=" border: none"> </td>
  </tr>
  <tr>
    <td style=" border: none"> </strong>new<strong>: </td>
    <td style=" border: none"> <code>$ hg clone https://bitbucket.org/galaxy/galaxy-dist#stable </code> </td>
  </tr>
  <tr>
    <td style=" border: none"> </strong>upgrade<strong>: </td>
    <td style=" border: none"> <code>$ hg pull </code> <br /> <code>$ hg update latest_2015.01.13</code> </td>
  </tr>
</table>



## Planemo 0.2.0

Version 0.2.0 of [Planemo](https://github.com/galaxyproject/planemo/blob/master/README.rst) was released in January.  Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.  Updates included:
* Improvements to way Planemo loads its own copy of Galaxy modules to prevent various conflicts when launching Galaxy from Planemo. ([Pull Request 56](https://github.com/galaxyproject/planemo/pull/56))
* Allow setting various test output options in ``~/.planemo.yml`` and disabling JSON output. 
* More experimental Brew and Tool Shed options that should not be considered part of Planemo's stable API. See [this presentation](http://bit.ly/gxbrew1) for more details.
* Fix ``project_init`` for BSD tar (thanks to [Nitesh Turaga](/NiteshTuraga) for the bug report.)
* Documentation fixes for tool linting command (thanks to [Nicola Soranzo](/NicolaSoranzo)).

## BioBlend
[BioBlend v0.5.2 was released in October](/News/BioBlendV0.5.2).  [BioBlend](https://github.com/galaxyproject/bioblend) is a python library for interacting with CloudMan and the [Galaxy API](/Learn/API).  In January the [BioBlend source](https://github.com/galaxyproject/bioblend) was moved to the [galaxyproject Github account](https://github.com/galaxyproject).

## CloudMan and blend4j

New versions [CloudMan](/GalaxyUpdates/2014_09/#august-2014-cloudman-release), and [blend4j](/GalaxyUpdates/2014_09/#blend4j-011-release) were released in August.

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

## Galaxy Project ToolShed Repos

Here are new contributions for the past month.

In no particular order: 

### New Tools

* *From [rnateam](https://toolshed.g2.bx.psu.edu/view/rnateam):*
  * [mirdeep2](https://toolshed.g2.bx.psu.edu/view/rnateam/mirdeep2): identification of novel and known miRNAs MiRDeep2 is a software package for identification of novel and known miRNAs in deep sequencing data. Furthermore, it can be used for miRNA expression profiling across samples.
  * [mirdeep2_mapper](https://toolshed.g2.bx.psu.edu/view/rnateam/mirdeep2_mapper): MiRDeep2 Mapper - process and map reads to a reference genome The mapper module is designed as a tool to process deep sequencing reads and/or map them to the reference genome. The module works in sequence space, and can process or map data that is in sequence fasta format. A number of the functions of the mapper module are implemented specifically with Solexa/Illumina data in mind.
  * [mirdeep2_quantifier](https://toolshed.g2.bx.psu.edu/view/rnateam/mirdeep2_quantifier): Imported from capsule None fast quantitation of reads mapping to known miRBase precursors The module maps the deep sequencing reads to predefined miRNA precursors and determines by that the expression of the corresponding miRNAs. First, the predefined mature miRNA sequences are mapped to the predefined precursors. Optionally, predefined star sequences can be mapped to the precursors too. By that the mature and star sequence in the precursors are determined. Second, the deep sequencing reads are mapped to the precursors. The number of reads falling into an interval 2nt upstream and 5nt downstream of the mature/star sequence is determined.

* *From [hammock](https://toolshed.g2.bx.psu.edu/view/hammock):*
  * [hammock](https://toolshed.g2.bx.psu.edu/view/hammock/hammock): Cluster large amounts of short peptide sequences and generate MSAs of resulting clusters.

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [snpsift_dbnsfp_datatypes](https://toolshed.g2.bx.psu.edu/view/iuc/snpsift_dbnsfp_datatypes): Galaxy applicable data formats for !SnpSift. snpsift_dbnsfp tool Allows for the creation of a snpsift dbnsfp Galaxy dataset from a tabular file that can be used by !SnpSift dbnsfp to annotate a vcf formatted file.
  * [snpeff_datatypes](https://toolshed.g2.bx.psu.edu/view/iuc/snpeff_datatypes): Galaxy applicable data formats for !SnpEff tools Galaxy applicable data formats for !SnpEff tools.  The snpeffdb datatype provides the means to represent a !SnpEff genome database in a Galaxy dataset.
  * [snpsift_dbnsfp](https://toolshed.g2.bx.psu.edu/view/iuc/snpsift_dbnsfp): snpEff !SnpSift dbnsfp tool from Pablo Cingolani Annotates variants on genes using the dbNSFP, an integrated database of human functional predictions from multiple algorithms (SIFT, Polyphen2, LRT and !MutationTaster, PhyloP and GERP++, etc.) http://snpeff.sourceforge.net/SnpSift.html#dbNSFP !SnpEff and !SnpSift are developed by Pablo Cingolani at http://snpeff.sourceforge.net/ <br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening), Jim Johnson, [Nicola Soranzo](/NicolaSoranzo)<br />
      Repository-Development: https://github.com/galaxy-iuc/tool_shed/
  * [snpsift](https://toolshed.g2.bx.psu.edu/view/iuc/snpsift): snpEff !SnpSift tools from Pablo Cingolani Variant annotation and effect prediction tools. It annotates and predicts the effects of variants on genes. This repository should manage tool dependencies. There are data managers for the Galaxy admin to download genome databases. !SnpEff is developed by Pablo Cingolani at http://snpeff.sourceforge.net/ <br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening), Jim Johnson, [Nicola Soranzo](/NicolaSoranzo)<br />
      Repository-Development: https://github.com/galaxy-iuc/tool_shed /<br />
  * [snpsift_genesets](https://toolshed.g2.bx.psu.edu/view/iuc/snpsift_genesets): snpEff !SnpSift genesets tool from Pablo Cingolani Annotates variants with !GeneSets, such as Gene Ontology (GO), KEGG, Reactome, etc.; 

     http://snpeff.sourceforge.net/SnpSift.html#geneSets !SnpEff and !SnpSift are developed by Pablo Cingolani at http://snpeff.sourceforge.net/ <br />
     Repository-Maintainer: [Björn Grüning](/BjoernGruening), Jim Johnson, [Nicola Soranzo](/NicolaSoranzo)<br />
     Repository-Development: https://github.com/galaxy-iuc/tool_shed/
* [snpsift_dbnsfp_generic](https://toolshed.g2.bx.psu.edu/view/iuc/snpsift_dbnsfp_generic): snpEff !SnpSift dbnsfp tool that can use any dbnsfp-like annotation data Annotates variants on genes using a tabular set of annotation values such as those from the dbNSFP, an integrated database of human functional predictions from multiple algorithms (SIFT, Polyphen2, LRT and !MutationTaster, PhyloP and GERP++, etc.)<br />
    This tool determines the available annotations from the input, so it can be used for other organisms other than human or other annotation values than available from dbNSFP. http://snpeff.sourceforge.net/SnpSift.html#dbNSFP <br />
    !SnpEff and !SnpSift are developed by Pablo Cingolani at http://snpeff.sourceforge.net/ <br />
    Repository-Maintainer: [Björn Grüning](/BjoernGruening), Jim Johnson, [Nicola Soranzo](/NicolaSoranzo)<br />
    Repository-Development: https://github.com/galaxy-iuc/tool_shed/

* *From [amawla](https://toolshed.g2.bx.psu.edu/view/amawla):*
  * [edger](https://toolshed.g2.bx.psu.edu/view/amawla/edger): Empirical analysis of digital gene expression data. GVL. !PeterMac Estimates differential gene expression for short read sequence count using methods appropriate for count data. <br />
      Performs digital differential gene expression analysis between groups (eg a treatment and control). Biological replicates provide information about experimental variability required for reliable inference. <br />
      Designed for easier use in Galaxy.<br />
      R package requirements: 
      * edgeR 
      * limma 

     Contributors: Monica Britton, Blythe Durbin-Johnson, Joseph Fass, Nikhil Joshi, Alex Mawla

* *From [jjohnson](https://toolshed.g2.bx.psu.edu/view/jjohnson):*
  * [gffread](https://toolshed.g2.bx.psu.edu/view/jjohnson/gffread): cufflinks gffread filters and/or converts GFF3/GTF2 records cufflinks gffread filters and/or converts GFF3/GTF2 records and can produce cdna, CDS, peptide fasta sequences 

* *From [mvdbeek](https://toolshed.g2.bx.psu.edu/view/mvdbeek):*
  * [add_input_name_as_column](https://toolshed.g2.bx.psu.edu/view/mvdbeek/add_input_name_as_column): Add input name as column on an existing tabular file Retrieves the histroy name of the input dataset and adds it as the last column. Very useful when working with dataset collections.

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/peterjc):*
  * [seq_filter_by_mapping](https://toolshed.g2.bx.psu.edu/view/peterjc/seq_filter_by_mapping): Uploaded v0.0.2, fixed some error messages Filter sequencing reads using SAM/BAM mapping files This tool is a short Python script (using Biopython library functions) which divides a FASTA, FASTQ, or SFF file in two, those sequences which do or do not map according to given SAM/BAM file(s).<br />
      Example uses include mapping of FASTQ reads against a known contaminant in order to remove reads prior to a de novo assembly.<br />
      See https://github.com/peterjc/pico_galaxy/tree/master/tools/seq_filter_by_mapping

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [bamtools](https://toolshed.g2.bx.psu.edu/view/devteam/bamtools): A collection of tools for manipulation of bamfiles BAMTools is a collection of utilities for splitting BAM files. It is based on BAMtools suite of tools by Derek Barnett (https://github.com/pezmaster31/bamtools). <br />
      !GitHub repo for this collection of tools can be [found here](https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools).
  * [bamtools_split](https://toolshed.g2.bx.psu.edu/view/devteam/bamtools_split): Utility for splitting BAM files. It is based on BAMtools suite of tools by Derek Barnett (https://github.com/pezmaster31/bamtools).<br /> 
      !GitHub repo for this collection of tools can be [found here](https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools/bamtools_split).

* [bamtools_filter](https://toolshed.g2.bx.psu.edu/view/devteam/bamtools_filter): Utility for filtering BAM files. It is based on BAMtools suite of tools by Derek Barnett (https://github.com/pezmaster31/bamtools). <br />
    !GitHub repo for this collection of tools can be [found here](https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools/bamtools_filter).

* *From [bigrna](https://toolshed.g2.bx.psu.edu/view/bigrna):*
  * [gpsrna](https://toolshed.g2.bx.psu.edu/view/bigrna/gpsrna): Plant small RNA analysis toolkit, microRNA identification and quantification, multiple distinct small RNAs identification, small interacting RNAs identification and quantification.

* *From [fubar](https://toolshed.g2.bx.psu.edu/view/fubar):*
  * [tool_factory_2](https://toolshed.g2.bx.psu.edu/view/fubar/tool_factory_2): Initial commit of code in iuc github repository. updated version of the tool factory  Includes arbitrary parameters and multiple inputs. Cost is more complex parameter parsing but examples are provided. Now allows selecting any installed interpreter tool shed package or using the system default!

* *From [nikhil-joshi](https://toolshed.g2.bx.psu.edu/view/nikhil-joshi):*
  * [sam2counts_edger](https://toolshed.g2.bx.psu.edu/view/nikhil-joshi/sam2counts_edger): Takes SAM files and outputs a table of counts that can be used with edgeR. 

### Tool Suites

* *From [rnateam](https://toolshed.g2.bx.psu.edu/view/rnateam):*
  * [suite_mirdeep_2_0](https://toolshed.g2.bx.psu.edu/view/rnateam/suite_mirdeep_2_0): The suite of tool wrappers for mirDeep2. MiRDeep2 is a software package for identification of novel and known miRNAs in deep sequencing data. Furthermore, it can be used for miRNA expression profiling across samples.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [suite_bamtools_2_3_0](https://toolshed.g2.bx.psu.edu/view/devteam/suite_bamtools_2_3_0):  BAMTools is a collection of utilities for manipulating BAM files. It is based on BAMtools suite of tools by Derek Barnett (https://github.com/pezmaster31/bamtools). 
      !GitHub repo for this collection of tools can be [found here](https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools).

### Data Managers
* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc/): *
  * [data_manager_snpeff](https://toolshed.g2.bx.psu.edu/view/iuc/data_manager_snpeff): Manage !SnpEff genome reference data Data managers for the Galaxy admin to download !SnpEff genome databases. 
      !SnpEff is developed by Pablo Cingolani at http://snpeff.sourceforge.net/ <br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening), Jim Johnson, [Nicola Soranzo](/NicolaSoranzo)<br />
      Repository-Development: https://github.com/galaxy-iuc/tool_shed/

### Packages / Tool Dependency Definitions

* *From [biopython](https://toolshed.g2.bx.psu.edu/view/biopython):*
  * [package_biopython_1_65](https://toolshed.g2.bx.psu.edu/view/biopython/package_biopython_1_65): Uploaded with !NumPy 1.9 dependency. Downloads and compiles version 1.65 of the Biopython package.  The Biopython Project is an international association of developers of freely available Python tools for computational molecular biology.<br />
      http://www.biopython.org

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [package_vienna_rna_1_8](https://toolshed.g2.bx.psu.edu/view/iuc/package_vienna_rna_1_8): Imported from capsule None  Contains a tool dependency definition that downloads and compiles version 1.8 of the Vienna RNA package.  The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.<br />
      http://www.tbi.univie.ac.at/RNA /<br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening)<br />
      Repository-Development: https://github.com/bgruening/galaxytools
  * [package_perl_pdf_api2_2_023](https://toolshed.g2.bx.psu.edu/view/iuc/package_perl_pdf_api2_2_023): Contains a tool dependency definition that downloads and compiles version 2.023 of the perl PDF API 
  * [package_randfold_2_0](https://toolshed.g2.bx.psu.edu/view/iuc/package_randfold_2_0): Contains a tool dependency definition that downloads and compiles version 2.0 of randfold. This is RANDFOLD version 2. The software compute the probability that, for a given sequence, the Minimum Free Energy (MFE) of the secondary structure is different from MFE computed with random sequences. <br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening) & Cameron Smith<br />
      Repository-Development: https://github.com/bgruening/galaxytools
  * [package_boost_1_55](https://toolshed.g2.bx.psu.edu/view/iuc/package_boost_1_55): Tool dependency definition that downloads and compiles version 1.55 of the boost C++ libraries Boost provides free peer-reviewed portable C++ source libraries.<br />
      http://www.boost.org /<br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening)<br />
      Repository-Development: https://github.com/bgruening/galaxytools
  * [package_rhmm_1_5_0](https://toolshed.g2.bx.psu.edu/view/iuc/package_rhmm_1_5_0): Installs the RHmm CRAN library. Discrete, univariate or multivariate gaussian, mixture of univariate or multivariate gaussian HMM functions for simulation and estimation. 
  * [package_snpeff_4_0](https://toolshed.g2.bx.psu.edu/view/iuc/package_snpeff_4_0): Contains a tool dependency definition that installs version 4.0 !SnpEff, a variant annotation and effect prediction tool. !SnpEff is a variant annotation and effect prediction tool by Pablo Cingolani. <br />
      It annotates and predicts the effects of variants on genes (such as amino acid changes).<br /> 
      !SnpEff and !SnpSift ( http://snpeff.sourceforge.net/ )
  * [package_bedtools_2_22](https://toolshed.g2.bx.psu.edu/view/iuc/package_bedtools_2_22): Contains a tool dependency definition that downloads and compiles version 2.22 of bedtools. bedtools - a swiss army knife for genome arithmetic<br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening)<br />
      Repository-Development: https://github.com/galaxyproject/tools-iuc/tree/master/packages/package_bedtools_2_22
  * [package_squid_1_9g](https://toolshed.g2.bx.psu.edu/view/iuc/package_squid_1_9g): Tool dependency definition that downloads and compiles version 1.9g of the squid library. SQUID is a freely redistributable library of C code functions for sequence analysis. SQUID also includes a number of small utility programs.<br />
      Repository-Maintainer: [Björn Grüning](/BjoernGruening) & Cameron Smith<br />
      Repository-Development: https://github.com/bgruening/galaxytools

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [package_bamtools_2_3_0_2d7685d2ae](https://toolshed.g2.bx.psu.edu/view/devteam/package_bamtools_2_3_0_2d7685d2ae): bamtools - a collection of utilities for processing of bam files Binary files in this package are compiled from source code version 2d7685d2ae.  This is package dependency for tools relying on bamtools toolkit developed by by Derek Barnett (https://github.com/pezmaster31/bamtools). This package is distributed as x86_64 binaries only. These binaries should work on any of our stated supported linux platforms other than RHEL/CentOS 5.

* *From [malbuquerque](https://toolshed.g2.bx.psu.edu/view/malbuquerque):*
  * [package_seqtk_1_0_0](https://toolshed.g2.bx.psu.edu/view/malbuquerque/package_seqtk_1_0_0): Builds Seqtk version 1.0.0 "Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip" -Heng Li, Broad Institute
  * [package_bamtools_2_3_0](https://toolshed.g2.bx.psu.edu/view/malbuquerque/package_bamtools_2_3_0): !BamTools - a collection of utilities for processing of bam files - for all architectures 
  * [package_cmake_3_1_0](https://toolshed.g2.bx.psu.edu/view/malbuquerque/package_cmake_3_1_0): Builds CMake version 3.1.0 - Cross Platform Makefile Generator 
  * [package_ucsc_user_apps_310](https://toolshed.g2.bx.psu.edu/view/malbuquerque/package_ucsc_user_apps_310): Builds most UCSC user applications Only those tools which successfully build without using mysql_config. Specifically only those in the kent/src/utils directory (those publicly available).


# Other News
* An [IPython Notebook that builds Galaxy tool_dependencies.xml file for Perl packages](http://nbviewer.ipython.org/github/bgruening/notebooks/blob/master/Perl/perl_package_to_galaxy_tool_dependency.ipynb), from [Björn Grüning](/BjoernGruening)
* [NSF Award CIF21 DIBBS: Tripal Gateway, a Platform for Next-Generation Data Analysis and Sharing](http://nsf.gov/awardsearch/showAward?AWD_ID=1443040).  Will integrate [Tripal](http://tripal.info/) and Galaxy, and allow Tripal community databases to create and share workflows.  (See also this story in [HPCwire](http://www.hpcwire.com/2015/01/21/nsf-signs-1-4-million-check-advance-community-database/).)
* [Dan Blankenberg](/Dan) on a [useful way to handle tools with outputs tied to tool version](https://github.com/galaxyproject/tools-iuc/issues/22#issuecomment-69930103) in Galaxy
* In [galaxy-central](https://bitbucket.org/galaxy/galaxy-central/) now: [tools can explicitly produce collections of datasets](https://bitbucket.org/galaxy/galaxy-central/pull-request/634/allow-tools-to-explicitly-produce-dataset/diff) 
* [Getting your 23 and Me data into Galaxy](http://genomeintelligence.org/?p=745), at [Genome Intelligence](http://genomeintelligence.org/) 
* [What are the differences](https://www.biostars.org/p/126034/) between [Biostars - bioinformatics explained](https://www.biostars.org/) and [Biostars - Galaxy explained](https://biostar.usegalaxy.org/)?

## And a Huge Thank You!

The [Galaxy Team](/GalaxyTeam) would like to thank everyone who took the time to respond to the project's [call for support](/News/GalaxyProjectGrantSupport).  These will be extremely helpful with our renewal.  
