---
title: March 2014 Galaxy Update
---
<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the March 2014 *[Galaxy Update](/galaxy-updates/)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# New Papers

*54 papers* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in Febrary. Some papers that may be particularly interesting to the Galaxy community:

* [Dissemination of scientific software with Galaxy ToolShed](https://doi.org/10.1186/gb4161), by [Blankenberg](/people/dan/), [Von Kuster](/people/greg-vonkuster/), [Bouvier](/people/dave-bouvier/), [Baker](/people/dannon-baker/), [Afgan](/people/enis-afgan/), Stoler, the [Galaxy Team](/galaxy-team/), [Taylor](/people/james-taylor/) and [Nekrutenko](/people/anton/), *Genome Biology,* Vol. 15, No. 2. (20 February 2014), 403, doi:10.1186/gb4161

* [Binding Sites Analyser (BiSA): Software for Genomic Binding Sites Archiving and Overlap Analysis](https://doi.org/10.1371/journal.pone.0087301), by Khushi, *et al.,* *PLoS ONE*, Vol. 9, No. 2. (12 February 2014), e87301, doi:10.1371/journal.pone.0087301

* [Oqtans: a multifunctional workbench for RNA-seq data analysis](https://doi.org/10.1186/1471-2105-15-s3-a7), by Sreedharan, *et al.*, *BMC Bioinformatics*, Vol. 15, No. Suppl 3. (2014), A7, doi:10.1186/1471-2105-15-s3-a7

How those papers were tagged:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> = </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 26 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>


# GigaScience Galaxy Series

<div class='left'><br /><a href='http://www.gigasciencejournal.com/'><img src="/images/logos/GigaScienceLogo250.png" alt="GigaScience Journal" width="160" /></a></div>

The *[GigaScience](http://www.gigasciencejournal.com/)* "Galaxy: Data Intensive and Reproducible Research" series [announced](http://blogs.biomedcentral.com/gigablog/2013/04/05/call-for-papers-for-a-special-gcc2013-galaxy-series/) for the [2013 Galaxy Community Conference](/events/gcc2013/) has just published its [first papers](http://www.gigasciencejournal.com/series/Galaxy), and is continuing to take submissions. Whilst the series considers best practice papers, discussion, as well as novel uses of Galaxy, these first papers are examples of Galaxy toolkits, with a [genome diversity tool collection](https://doi.org/10.1186/2047-217X-2-17) presented from the Miller lab at Pennsylvania State, and a [set of analytical and visualisation tools for Complete Genomics sequencing data](https://doi.org/10.1186/2047-217X-3-1) from the Stubbs lab at Erasmus Medical College. The genome diversity toolkit is their first example that integrates with their [GigaGalaxy server](http://gigagalaxy.net/), with SVG graphs to help visualize how input datasets, workflows and histories are related to each example analysis in the paper. BGI is continuing to cover the article processing charges until the end of this year, so for more information see their [blog posting](http://blogs.biomedcentral.com/gigablog/2014/02/06/rewarding-reproducibility-first-papers-in-our-galaxy-series-utilizing-our-gigagalaxy-platform/).

# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Stage M2 Recherche : Provenance dans un système de workflows pour la bioinformatique](https://lists.galaxyproject.org/archives/list/galaxy-france@lists.galaxyproject.org/thread/3PXIAVFR2YHMZAZXCDHDXNMGBSZJVFGR/#3PXIAVFR2YHMZAZXCDHDXNMGBSZJVFGR)
* [Stage M1/M2, Development and integration of computational methods into the Galaxy analysis framework](https://lists.galaxyproject.org/archives/list/galaxy-france@lists.galaxyproject.org/thread/BUHZTVWUF4P7X3WXMR4BVZTK45QVCNPD/#BUHZTVWUF4P7X3WXMR4BVZTK45QVCNPD), Institut Curie, Paris
* [Statistical Genomics Postdoc](/news/statistical-genomics-post-doc-penn-state/) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**, *including positions at George Washington University*.

* [Postdoctoral position in genomics/bio-informatics (NGS)](http://bit.ly/1eq5TvM) at University of Rennes 1 (France)  

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

# Events

## Galaxy Australasia Workshop 2014: 24-25 March

**The [1st Galaxy Australasia Workshop 2014 (GAW 2014)](http://australianbioinformatics.net/gaw-2014) will be held in Melbourne, Australia on 24 and 25th March 2014.**

<div class='right'><a href='http://australianbioinformatics.net/gaw-2014'><img src="/images/logos/GAW2014-200.png" alt="Galaxy Australasia Workshop 2014 (GAW2014)"  /></a></div>

The [Galaxy Australasia Workshop](http://australianbioinformatics.net/gaw-2014) is a great opportunity for you to participate in two full days of presentations, discussions, poster sessions, keynotes and lightning talks, all about ways of using Galaxy for high-throughput biology, imaging and other scientific applications. The workshop will also include Training Sessions taught by Galaxy developers and master users.  GAW 2014 will run 24 and 25th March, immediately preceding [Computational and Simulation Sciences and eResearch](http://wp.csiro.au/css/) in Melbourne.  GAW 2014 will also include poster session, keynote speakers.

<div class='left'><a href='http://mcec.com.au/'><img src="/events/gaw2014/MCC.png" alt="Melbourne Convention Centre" width=150 /></a></div>

[Registration is open](http://australianbioinformatics.net/gaw-2014).  Registration is also *free, but space is limited.*

The first day (Monday) will be in Clayton, next to [the Australian Synchrotron](http://www.synchrotron.org.au/).  For the 2nd day (Tuesday) we will move downtown the [Melbourne Convention Centre](http://mcec.com.au/), the venue for the [Computational and Simulation Sciences and eResearch](http://wp.csiro.au/css/).
<br />

## Globus World 2014

<div class='right'><a href='http://www.globusworld.org/'><img src="/images/logos/GlobusGenomics.png" alt="Globus World 2014" width="180" /></a></div>

**[GlobusWorld](http://www.globusworld.org/)** is this year’s biggest gathering of all things [Globus](https://www.globus.org/).  GlobusWorld features a *Biosciences/Genomics Program* devoted to research data in genomics and biosciences including [Globus Genomics](/cloud/#globus-genomics), a Globus and Galaxy based platform for genomic analysis.  [GlobusWorld](http://www.globusworld.org/) is being held April 15-17, in Chicago.

Learn how to get more out of your data, and connect with other HPC administrators, research scientists, and developers who are building and using cutting-edge solutions to wrangle data.

Galaxy will have a presence at this meeting, and [GCC2014](/events/gcc2014/) is a Silver Sponsor of [GlobusWorld](http://www.globusworld.org/).

## GCC2014: June 30 - July 2, Baltimore

<div class='right'><a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoTall200.png" alt="GCC2014: June 30 - July 2" width="150" /></a></div>

The [2014 Galaxy Community Conference (GCC2014)](/events/gcc2014/) will be held June 30 through July 2, at the [Homewood Campus](http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/) of [Johns Hopkins University](http://jhu.edu), in [Baltimore, Maryland](http://visitors.baltimorecity.gov/), United States.

### Registration is Open

[Early registration is now open](/events/gcc2014/register/). *Early registration saves more than 70% on registration costs, and [Training Day](/events/gcc2014/training-day/) registration is an additional 55% off if you register for both at the same time.*  This is by far the most affordable option, with early registration fees starting at less than $50 per day.  When you register you can also reserve [lodging at Charles Commons](/events/gcc2014/logistics/#lodging), a very affordable housing option in the same building as the conference.

*[Training Day](/events/gcc2014/training-day/)* is an opportunity to learn about all things Galaxy including using Galaxy, deploying and managing Galaxy, extending Galaxy, and Galaxy internals.  There are 5 parallel tracks, each with 3 sessions, with each of those sessions two and half hours long.  *That's 15 sessions and over 37 hours of workshop material.*

### Abstract Submission is Open

[Abstract submission for both oral presentations and posters](/events/gcc2014/abstracts/) is also open.  Abstract submission for oral presentations closes April 4, while poster submission closes April 25. Poster authors will be notified of acceptance status within two weeks of submission, while presentation authors will be notified no later than May 2.  Please consider presenting your work. If you are dealing with big biological data, then this meeting wants to hear about it.

Accepted talks and selected posters from GCC2014 are also eligible for consideration to appear in the *[GigaScience](http://www.gigasciencejournal.com/)* ["Galaxy: Data Intensive and Reproducible Research"](/galaxy-updates/2014-03/#gigascience-galaxy-series) series described above.


### Sponsorships

<div class='left'>
<a href='http://aws.amazon.com/'><img src="/images/logos/AWSLogo1280.jpg" alt="Amazon Web Services" width="200" /></a>
</div>
<div class='right'>
<a href='http://bio.maryland.gov/'><img src="/images/logos/BioMaryland400.png" alt="BioMaryland" width="200" /></a>
</div>

We are pleased to have [Amazon Web Services (AWS)](http://aws.amazon.com/) as the sponsor for the GCC [Training Day](/events/gcc2014/training-day/) for the third year in a row.  Also please welcome [BioMaryland](http://bio.maryland.gov/) as a [Bronze Sponsor](/events/gcc2014/sponsor-exhibit/) for [GCC2014](/events/gcc2014/).  BioMaryland is new to the GCC.

There are still [Silver and Bronze sponsorships](/events/gcc2014/sponsor-exhibit/) available. Please contact the [Organizers](/events/gcc2014/organizers/) if your organization would like to help sponsor this event.

### Exhibitors

In 2014 we are also adding [non-sponsor exhibit spaces](/events/gcc2014/sponsor-exhibit/#exhibit) in addition to the sponsor exhibits.  This will significantly increase the size of the exhibit floor. Please contact the [Organizers](/events/gcc2014/organizers/) if your organization would like to have an exhibit space at GCC2014.

## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div> There are several other [events of interest](/events/) in the next few months. Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<br />

<div class='center'>
<a href='/events/gaw2014/'><img src="/images/logos/GAW2014-200.png" alt="Register now for Galaxy Australasia Workshop (GAW 2014)" height="60" /></a> &nbsp;
<a href='http://globusworld.org/'><img src="/images/logos/GlobusWorld2014Logo.png" alt="GlobusWorld 2014" height="60" /></a> &nbsp;
<a href='http://www.bio-itworldexpo.com/'><img src="/images/logos/BioITWorld2014Logo.png" alt="Bio-IT World 2014" /></a> &nbsp;
<a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoWide200.png" alt="GCC2014 early registration and abstract submission are now open" width="150" /></a>
</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> March 24-25 </th>
    <td> <strong><a href='/events/gaw2014/'>Galaxy Australasia Workshop 2014 (GAW2014)</a></strong> </td>
    <td> Melbourne, Australia </td>
    <td> <a href='/events/gaw2014/#organisers'>Organisers</a> </td>
  </tr>
  <tr>
    <th> March 26-30 </th>
    <td> <em>Galaxy toolset for Drosophila genomics</em><div class='indent'>and one-on-one help in the Flybase Demonstrations Room</div> </td>
    <td> <a href='http://www.genetics-gsa.org/drosophila/2014/'>Drosophila Research Conference</a>, San Diego, California, United States </td>
    <td> <a href='/people/anton/'>Anton Nekrutenko</a>, <a href='/people/jennifer-jackson/'>Jen Jackson</a> </td>
  </tr>
  <tr>
    <th> April 15-17 </th>
    <td> <em>Biosciences/Genomics Program</em> </td>
    <td> <a href='http://globusworld.org/'>GlobusWorld</a>, Chicago, Illinois, United States </td>
    <td> <a href="mailto:outreach AT globus DOT org">Globus Outreach</a> </td>
  </tr>
  <tr>
    <th rowspan=6> April 29 - May 1 </th>
    <td> <em><a href='http://www.bio-itworldexpo.com/Bio-It_Expo_Content.aspx?id=121423'>W1: Integrated Research Data Management for Next Gen Sequencing Analysis Using Galaxy and Globus Online Software-as-a-Service</a></em> </td>
    <td rowspan=6> <a href='http://www.bio-itworldexpo.com/'>BioIT World</a> </td>
    <td> <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a>, <a href='https://www.urmc.rochester.edu/people/21779848-alex-r-paciorkowski'>Alex Paciorkowski</a>, <a href='http://www.ci.uchicago.edu/profile/328'>Vas Vasiliadis</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.bio-itworldexpo.com/Bio-It_Expo_Content.aspx?id=135146'>W4: Analyzing NGS Data in Galaxy</a></em> </td>
    <td> <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a>, <a href='/people/nate/'>Nate Coraor</a>, <a href='http://bioteam.net/company-leadership/'>Adam Kraut</a>, Tristan Lubinski </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.bio-itworldexpo.com/Bio-It_Expo_Content.aspx?id=135152'>W14: Running a Local Galaxy Instance</a></em> </td>
    <td> <a href='http://bioteam.net/company-leadership/'>Adam Kraut</a>, <a href='/people/nate/'>Nate Coraor</a>, <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a>, Tristan Lubinski, James Reaney </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.bio-itworldexpo.com/Next-Gen-Sequencing-Informatics/'>Globus Genomics: An End-to-End NGS Analysis Service on the Cloud for Researchers and Core Labs</a></em> </td>
    <td> <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.bio-itworldexpo.com/Data-Visualization/'>Integrated Analysis and Visualization of Large-Scale Biological Data with the Refinery Platform</a></em> </td>
    <td> <a href='http://gehlenborg.com/'>Nils Gehlenborg</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.bio-itworldexpo.com/Data-Visualization/'>Web-Based Visualization and Visual Analysis for High-Throughput Genomics with Galaxy</a></em> </td>
    <td> <a href='/people/jeremy-goecks/'>Jeremy Goecks</a> </td>
  </tr>
  <tr>
    <th> May 6-7 </th>
    <td> <em>Scaling Galaxy for Big Data</em> </td>
    <td> <a href='http://www.tgac.ac.uk/SeqAhead_Scientific_Meeting_and_Management_Committee_Meeting/'>NGS Data after the Gold Rush</a>, <a href='http://www.tgac.ac.uk/'>TGAC</a>, Norwich, United Kingdom </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> May 9 </th>
    <td> <em>Galaxy Workshop</em> </td>
    <td> <a href='http://www.tgac.ac.uk/'>The Genome Analysis Centre (TGAC)</a>, Norwich, United Kingdom </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> May 27 </th>
    <td> <em><a href='http://www.mipro.hr/LinkClick.aspx?fileticket=y1%2bUY%2b54jB4%3d&tabid=110&mid=461&language=en-US&forcedownload=true'>Enabling Data Analysis with Galaxy CloudMan workshop</a></em> </td>
    <td> <a href='http://www.mipro.hr/CallForPapers/tabid/176/language/en-US/Default.aspx'>MIPRO 2014</a>, Opatija, Croatia </td>
    <td> <a href='/people/enis-afgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> <em><a href='http://bioinformatics.ca/workshops/2014/informatics-high-throughput-sequencing-data-2014'>Informatics on High Throughput Sequencing Data Workshop</a></em> </td>
    <td> Toronto, Canada </td>
    <td> <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 10 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module17-10juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td rowspan=2> <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2014</a>, <a href='http://migale.jouy.inra.fr/'>INRA Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin AT jouy DOT inra DOT fr">Véronique Martin</a>, <a href='http://genome.jouy.inra.fr/~schbath/'>Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 11 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module8bis-11juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> June 16 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module17-16juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td rowspan=2> <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2014</a>, <a href='http://migale.jouy.inra.fr/'>INRA Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin AT jouy DOT inra DOT fr">Véronique Martin</a>, <a href='http://genome.jouy.inra.fr/~schbath/'>Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 17 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module8bis-17juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> June 16-20 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2014/02/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-june-16-20-2014/'>Using Galaxy for Analysis of High Throughput Sequence Data</a></em> </td>
    <td> <a href='http://training.bioinformatics.ucdavis.edu/'>UC Davis</a>, California, United States </td>
    <td> <a href="mailto:training DOT ucdbio AT gmail DOT com">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th> June 28-30 </th>
    <td> <a href='/events/gcc2014/hackathon/'>Galaxy Hackathon</a> </td>
    <td rowspan=2> <a href='http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/'>Homewood Campus</a> of <a href='http://jhu.edu'>Johns Hopkins University</a>, <a href='http://visitors.baltimorecity.gov/'>Baltimore, Maryland</a>, United States </td>
    <td> <a href='/events/gcc2014/organizers/'>Organizers</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 2 </th>
    <td> <strong><a href='/events/gcc2014/'>2014 Galaxy Community Conference (GCC2014)</a></strong> </td>
    <td> <a href='/events/gcc2014/organizers/'>Organizers</a> </td>
  </tr>
  <tr>
    <th rowspan=3> September 30 - October 2 </th>
    <td> Galaxy Training and Demo Day </td>
    <td> Bern (Switzerland) </td>
    <td rowspan=3> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <td> <em><a href='/events/switzerland2014/'>(second Swiss) Galaxy Workshop</a></em> </td>
    <td> SyBIT Tech Day, Bern (Switzerland) </td>
  </tr>
  <tr>
    <td> <em><a href='/events/germany2014/'>German Galaxy Developers Day</a></em> </td>
    <td> Freiburg (Germany) </td>
  </tr>
</table>

<br />

# New Public Servers

One new [public Galaxy server](/use/), PGTB was added, and the NELLY server is back.

## Galaxy PGTB (Virtual Biodiversity Lab)

<div class='right solid'><a href='http://www.pgtb.u-bordeaux2.fr/'><img src="/use/archive/galaxy-pgtb-virtual-biodiversity-lab/pgtb.png" alt="PGTB Galaxy"  /></a></div>

* *Links:*
    * **[PGTB Galaxy - Virtual Biodiversity Lab](https://galaxy-pgtp.pierroton.inra.fr/)**
    * **[Plateforme Genome Transcriptome de Bordeaux](http://www.pgtb.u-bordeaux2.fr/)**
* *Domain/Purpose:*
    This is a standard Galaxy instance implemented with specific tools for Biodiversity (Biodiversity Virtual Lab) and NGS (Ion Torrent from the PGTB facility) analysis.
* *Comments:*
    Due to current disk space limitations, registration is required. Account request on demand at [Email](mailto:contact DOT pgtp AT pierroton DOT inra DOT fr).
* *User Support:*
    [Email](mailto:contact DOT pgtp AT pierroton DOT inra DOT fr)
* *Quotas:*
    Registered users: 50GB;
* *Sponsor(s):*
    [INRA](http://www.inra.fr/) and [Equipex Xyloforest](http://www.xyloforest.org/) and [Labex CEBA](http://www.labex-ceba.fr/) and [Plateforme Genome Transcriptome de Bordeaux](http://www.pgtb.u-bordeaux2.fr/)

# Galaxy Distributions

The most recent version of CloudMan was [released in January 2014](/news/cloudman-release201401/).

## Feb 10, 2014 Galaxy Distribution

<div class='right'>
<a href='http://usegalaxy.org'><img src="/images/news-graphics/2014_02_10_trackster-css.png" alt="Trackster" width="400" /></a><br />*Trackster*</div>
**Complete [News Brief](http://wiki.galaxyproject.org/DevNewsBriefs/2014-02-10)**
<br />
<br />
**Highlights:**

* Visualization upgrades, including Trackster CSS styling
* Multiple Tools migrated to the Tool Shed for a leaner distribution
* Redesign of UI rendering: new icons, new font, history pane updates
* API functionality upgrades featuring a new master admin API key and
* Tool Shed updates a focus on repository metadata, displays, installs, and tests
* Over 35 new community contributions added

[http://getgalaxy.org](http://getgalaxy.org)

[http://bitbucket.org/galaxy/galaxy-dist](http://bitbucket.org/galaxy/galaxy-dist)

[http://galaxy-dist.readthedocs.org](http://galaxy-dist.readthedocs.org)

```

new:       $ hg clone https://bitbucket.org/galaxy/galaxy-dist#stable

upgrade:   $ hg pull 
           $ hg update release_2014.02.10
```


# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a></div>
<div class='right'><img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style="text-align: center;"> &nbsp;&nbsp; <a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a> </em> &nbsp;&nbsp;<br /> </td>
  </tr>
</table>

<br /><br />

The *[Community Log Board](/news/)* and  *Deployment Catalog* *[Galaxy community hubs](/community/)* were launched in December.  If you have a deployment, or experience you want to share then please publish them.

# ToolShed Contributions

<div class='right'><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

## New Repositories in the Galaxy Project ToolShed

* [phylorelatives](http://bit.ly/1dIJIDG): Relatedness of minor alelle sequences in NJ tree
* [advanced_grep_from_file](http://bit.ly/1m6Gfkr): Extended Grep Using Perl, fetch lines after match and get patterns from file
* [getalleleseq](http://bit.ly/MwAlN3): Generate FASTA sequence from alleles table
* [hetbox](http://bit.ly/1jOLTtg): minor variant boxplot
* [find_in_reference](http://bit.ly/MwAPmq): filter tab file where a field is a substring of a field in any line of a reference
* [filter_bed_on_splice_junctions](http://bit.ly/1jDm1wY): Filter out lines w/ junctions in reference bed file of known junctions
* [translate_bed_sequences](http://bit.ly/1eNHFKg): 3 frame translation of BED file augmented with a sequence column
* [rsem](http://bit.ly/1biTdtX): RNA SEQ by Expectation Maximization
* [snpeff_to_peptides](http://bit.ly/1h2zMUr): Creates a peptide fasta from a SnpEff VCF output and an Ensembl all_peps.fa
* [synteny_mapper](http://bit.ly/1nLamwQ): find microrearrangements w/in syntenic regions, viz & compare UCSC features  
* [bamtools_wrappers](http://bit.ly/1jOONxS): Wrappers for the bamtools package
* [fastq_qc_trimmer](http://bit.ly/1kSTVRV): Parallel FASTQ QC Trimmer
* [fastq_all_to_sanger](http://bit.ly/N74geH): Very Fast groomer of quality formats to Sanger
* [prada](http://bit.ly/1miEAs8): pipeline for RNAseq data analysis for Illumina paired end data
* [shrnaseq](http://bit.ly/1myEYpV): shRNAseq differential representation analysis using edgeR  
* [segemehl](http://bit.ly/1o4LyQx): short read mapping with gaps
* [blockbuster](http://bit.ly/1euOkZK): Detection of blocks of overlapping reads  
* [twistdna](http://bit.ly/1mGzXf3): prediction of superhelically destabilized regions  
* [rnaz](http://bit.ly/1myhRIE): predicting structural noncoding RNAs  

# Other News

* GOBLET is collecting [Bioinformatics Training needs](https://www.surveymonkey.com/s/GOBLETsurvey) across the globe
