---
autotoc: true
title: October 2016 Galaxy News
---


<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 24em;">  </td>
  </tr>
</table>

</div>

<div class='right'><a href='/GalaxyUpdates'><img src='/Images/GalaxyLogos/GalaxyNews.png' alt='Galaxy News' width=150 /></a></div>

Welcome to the October 2016 *[Galactic News](/src/GalaxyUpdates/index.md)*, a summary of what is going on in the Galaxy community. 

The big news this month is a series of upcoming events:

* **[Galaxy training contribution fest](#galaxy-training-contribution-fest-6-7-october-online)**: 
  * October 6-7, Online 
* **[Swiss-German Galaxy Days](#swiss-german-galaxy-days)**:
  * October 20-21, Freiburg, Germany
* **[Galaxy & Hacktoberfest](#galaxy--hacktoberfest)**:
  * Month of October, online
* **[Bioconda Contribution Fest](#conda-conda-conda)**:
  * November 2-3, [Online](https://github.com/bioconda/bioconda-recipes/issues/2277)
* **[Galaxy Admin Training](#galaxy-admin-training-november-7-11-salt-lake-city-utah)**:
  * November 7-11, Salt Lake City, Utah
  * *On time registration ends October 16*
* **[Galaxy Australasia Meeting (GAMe 2017)](#galaxy-australasia-meeting-game-2017-registration--abstract-submission-are-open)**: 
  * February 3-9, Melbourne, Victoria, Australia
  * *Registration and abstract submission are now open.* 

If you have anything to include in the next News, please send it to [outreach AT galaxyproject DOT org](Galaxy Outreach). 


# Events

There are a wealth of events coming up.  Here are some highlights, and a report on one from last month too.

## Galaxy Australasia Meeting (GAMe 2017): Registration & Abstract Submission are Open

<div class='right'><a href='https://www.embl-abr.org.au/game2017/'><img src='/Images/Logos/GAMeLogo200.png' alt='Galaxy Australasia Meeting 2017' width="120" /></a></div>

We are pleased to announce that [early registration](https://www.embl-abr.org.au/game2017/registration/) and [talk and poster abstract submission](https://www.embl-abr.org.au/game2017/submit-your-talkposter/) are now open for GAMe 2017. This meeting will bring together biomedical researchers, bioinformaticians, infrastructure providers, and data producers from across Australia and Asia to share expertise across many levels.

The [conference](https://www.embl-abr.org.au/game2017/conference/) starts 4 February and features two full days of keynotes, accepted and sponsor talks, poster and sponsor sessions, birds­-of-­a­-feather gatherings, a conference dinner and lots of opportunities for networking.

Prior to the conference we are offering a [Researcher Training Day](https://www.embl-abr.org.au/game2017/training-day/) on 3 February, aimed at biomedical researchers who need to analyse their biological data. The conference is followed by a [four day workshop on Galaxy server administration](https://www.embl-abr.org.au/game2017/training-for-admin-workshop/) for those working at that level.

[Talk abstracts are due 30 November](https://www.embl-abr.org.au/game2017/submit-your-talkposter/) and [early registration ends 31 December](https://www.embl-abr.org.au/game2017/registration/). Register now and save up to 43% off regular registration rates.  There are also [scholarships available for eligible students and postdocs](https://www.embl-abr.org.au/game2017/registration/#scholarships). The scholarship application deadline is 18 November.

We look forward to welcoming you to Melbourne. Remember to pack for our fabulous February weather!

*[GAMe 2017 Organising Committee](https://www.embl-abr.org.au/game2017/organisers/)* (#game_2017)

*And if you are based in Australia you might also be interested in the newly launched [Galaxy Australia Community](#galaxy-australia-community-launched).* 

## Galaxy training contribution fest, 6-7 October, Online

<div class='right'><a href='https://gist.github.com/bgruening/a54a8600c5cddca82424ad13569708c3'><img src='/Images/GalaxyLogos/GTNLogo300.png' alt='Galaxy training contribution fest, 6-7 October, Online' width="170" /></a></div>

The [Galaxy Training Network (GTN)](/src/Teach/GTN/index.md) is planning **[a contribution fest](https://gist.github.com/bgruening/a54a8600c5cddca82424ad13569708c3)** to consolidate our training material and improve the overall training  experience in Galaxy on the 6.-7th October 2016!

More information about the project can be found [here](https://gist.github.com/bgruening/a54a8600c5cddca82424ad13569708c3) and a [small presentation here](http://bgruening.github.io/training-material/shared/slides/project_presentation#/).  Registration (which is entirely optional - you can just show up) can be done via [this GitHub issue](https://github.com/bgruening/training-material/issues/58).  There is also a [list of potential issues to work on as well](https://github.com/bgruening/training-material/issues) (and more are welcome).

This will be an online contribution fest and the fun to contribute should be priority one! There is enough to work on in all areas, from programming to fix spelling mistakes. So everyone is welcome to improve the Galaxy training material.

Ciao,

[Bérénice](http://bebatut.fr/) and [Bjoern](/src/BjoernGruening/index.md)


## Galaxy Admin Training, November 7-11, Salt Lake City, Utah

<div class='left solid center'>
<a href='/Events/AdminTraining2016'><img src='/Images/Logos/AdminTraining2016-500.png' alt='Galaxy Project Admin Training' width=200" /></a><br />
November 7-11, 2016<br />University of Utah<br />Salt Lake City, Utah<br />United States<br /><br /> [Register by October 16](/src/Events/AdminTraining2016/Registration/index.md) 
</div>

**Register for [Galaxy Admin Training 2016](/src/Events/AdminTraining2016/index.md) by October 16.**

[Galaxy Admin Training 2016](/src/Events/AdminTraining2016/index.md) is a weeklong event offering basic and advanced sessions.  It is being held in Salt Lake City, Utah, November 7-11, 2016, the week before [Supercomputing 16 (SC16)](http://sc16.supercomputing.org/) meets in Salt Lake.  

**[Basics Session: November 7-8](/src/Events/AdminTraining2016/BasicsSession/index.md)**

This two day [basics session](/src/Events/AdminTraining2016/BasicsSession/index.md) will introduce participants to what you need to know to get a Galaxy server up and running on a standalone server. You'll also learn how to extend your Galaxy with your own tools and tools from the community, and how to define reference data in your server as well.

**[Advanced Session: November 9-11](/src/Events/AdminTraining2016/AdvancedSession/index.md)**

The three day [advanced session](/src/Events/AdminTraining2016/AdvancedSession/index.md) will build on topics covered in the [basics section](/src/Events/AdminTraining2016/BasicsSession/index.md).  This session both extends those topics, such as tool definition, and adds new ones, such as working with compute clusters and heterogeneous resources. The goal of the advanced workshop is to enable Galaxy administrators to create robust, high-performance Galaxy instances that take full advantage of available tools and compute and storage resources.

[Register by October 16](/src/Events/AdminTraining2016/Registration/index.md) and save up to $230 off late registration rates (and up to $500 if you work in industry).  *Space is also limited.*

<br />

<div class='center'>
<a href='http://www.chpc.utah.edu'><img src='/Images/Logos/UtahCHPCLogo.png' alt='University of Utah Center for High Performance Computing (CHPC)' height="90" /></a> &nbsp; 
<a href='http://healthsciences.utah.edu/'><img src='/Images/Logos/UUtahHealthSciences.png' alt='University of Utah' height="90" /></a> <br />
<a href='http://medicine.utah.edu/dbmi/'><img src='/Images/Logos/UUtahDBMI.png' alt='University of Utah Department of Biomedical Informatics' width="220" /></a>
<a href='http://medicine.utah.edu/ccts/bmic/'><img src='/Images/Logos/UUtahCCTS_BMIC.png' alt='University of Utah Clinical & Translational Science Biomedical Informatics Core (CCTS BMIC)' width="220" /></a><br />
<a href='http://ucgd.genetics.utah.edu/'><img src='/Images/Logos/USTAR-UCGD-Logo.jpg' alt='USTAR Center for Genetic Discovery (UCGD)' height="90" /></a>
</div>

Galaxy Admin Training 2016 is hosted by the [University of Utah Center for High Performance Computing (CHPC)](http://www.chpc.utah.edu/), the [USTAR Center for Genetic Discovery (UCGD)](http://ucgd.genetics.utah.edu/), the  [Department of Biomedical Informatics](http://medicine.utah.edu/dbmi/) and the [Clinical & Translational Science Biomedical Informatics Core (CCTS BMIC)](http://medicine.utah.edu/ccts/bmic/) at the [University of Utah](http://utah.edu).

We hope to see you in Salt Lake!


## Swiss-German Galaxy Days

<div class='right'>
<a href='/Events/SG2016'><img src='/Images/Logos/SG2016T.V2_logo.png' alt='2016 Swiss German Galaxy Tour' height="120" /></a>
<div class='center'><br />**[Register Now](https://docs.google.com/forms/d/1mRO4uNXQd8jL1I9JNiQ4xd0fnrZKlQPQSORgJCpDeqg)**</div></div>

The 2016 editions of [Swiss-German Galaxy Days](/src/Events/SG2016/index.md) will be held 20-21 October in Freiburg (Germany).

Registration is free, but space is limited on both days. We recommend you register soon to secure your spot for one or both days of the SG2016Tour: 

The two days will be held in the same location, but will have a different focus.  The [first day](/src/Events/Switzerland2016/index.md) will start with talks, discussions and small demonstrations. This day will be problem / technology centric with [small demonstrations and hands-on sessions](/src/Events/Switzerland2016/index.md). Everyone is expected to have their own computer.

[The second day](/src/Events/Germany2016/index.md) is a *developers* day and we are planning to have 3 to 4 hands-on sessions (1.5h). Every session will start with a 10 minute talk and theoretical introduction, followed by 1h of hacking and 20 minutes discussion and a small break until the next session begins.  We have a few [proposed and prepared topics](/src/Events/Germany2016/index.md#ideas) but we are flexible so we can discover new areas of the Galaxy.

## Galaxy & Hacktoberfest

<div class='left'><a href='https://hacktoberfest.digitalocean.com/'><img src='/Images/Logos/Hacktoberfest.png' alt='Hacktoberfest' width="200" /></a></div>

Galaxy and [CloudBridge](http://cloudbridge.readthedocs.io/en/latest/) have joined the [Hacktoberfest](https://hacktoberfest.digitalocean.com/) event this year. If you [register](https://hacktoberfest.digitalocean.com/sign_up/register) with your !GitHub account and create 4 or more pull requests during October you will get a free T-shirt and world-wide appreciation! You can browse the entry level issues in the Galaxy code by checking the hacktoberfest label ([Galaxy](https://github.com/galaxyproject/galaxy/labels/hacktoberfest), [CloudBridge](https://github.com/gvlproject/cloudbridge/labels/hacktoberfest)). !CloudBridge already has a [new contributor](https://github.com/gvlproject/cloudbridge/issues/22) so hurry and hack away!

<div class='right'><br /><a href='http://blogs.biomedcentral.com/gigablog/2016/09/20/guest-posting-introducing-conda-for-galaxy/'><img src='/Images/Logos/BioCondaGalaxy.jpg' alt='BioConda and Galaxy' width="200" /></a></div>

## Conda, Conda, Conda!

The Galaxy v16.07 release ([see below](#galaxy-v1607)) included a "Shift of Galaxy tool dependencies to Conda" and that shift is reflected in a whole bunch of activity: 

* **[Conda as a new standard for Galaxy tool dependencies](http://blogs.biomedcentral.com/gigablog/2016/09/20/guest-posting-introducing-conda-for-galaxy/)**, a guest blog posting by [Björn Grüning](/src/BjoernGruening/index.md) in *[GigaScience's](http://www.gigasciencejournal.com/)* [GigaBlog](http://blogs.biomedcentral.com/gigablog/).
* a Galaxy-P Conda contribution fest in September ([see report below](#galaxy-p-conda-contribution-fest-report)), 
* enhancements to [Planemo](#planemo-0332), and 
* an upcoming **[Bioconda Contribution Fest](https://github.com/bioconda/bioconda-recipes/issues/2277)** on November 2-3.




<div class='right'><br /><a href='https://gist.github.com/bgruening/97b1fa16b92f5ab30b1e7c193b286432'><img src='/Images/Logos/GalaxyPCondaHack.png' alt='Galaxy-P Conda Contribution Fest report' height="100" /></a></div>

### Galaxy-P Conda Contribution Fest Report

The [Galaxy-P Conda Contribution Fest](https://github.com/galaxyproteomics/tools-galaxyp/issues/57#issuecomment-245934625) was held online, 27-28 September, with the aim to port as many tool dependencies to Conda packages as possible. Conda packages are the future of Galaxy’s tool dependency management and will yield more robust tools and automatic CI testing of all Galaxy-P tools.  More than 20 pull requests were create and merged to [BioConda](https://bioconda.github.io/).  See the [event report](https://gist.github.com/bgruening/97b1fa16b92f5ab30b1e7c193b286432) for details.

Thanks to everyone who participated!


## All upcoming events

There are [a plentitude of Galaxy related events coming up](/src/Events/index.md) in the next few months:

<div class='center'> 
<a href='https://gist.github.com/bgruening/a54a8600c5cddca82424ad13569708c3'><img src='/Images/GalaxyLogos/GTNLogo300.png' alt='Galaxy Training Contribution Fest' height="75" /></a> &nbsp;
<a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=83'><img src='/Images/Logos/GenoToulLogo.png' alt='Metagenomic: amplicon analysis' height="60" /></a> &nbsp;
<a href='https://www.facebook.com/events/1593583060947735/'><img src='/Images/Logos/CBioVikingsTrans.png' alt='Galaxy Workshop' height="80" /></a>
<a href='http://www.france-bioinformatique.fr/en/evenements/galaxy4bioinformatics'><img src='/Images/Logos/IFBGalaxyWG.png' alt=' Galaxy4Bioinformatics'  /></a>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<a href='/Events/AdminTraining2016'><img src='/Images/Logos/AdminTraining2016-500.png' alt='Galaxy Project Admin Training' width=120" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> October 6-7 </th>
    <td> </strong><a href='https://gist.github.com/bgruening/a54a8600c5cddca82424ad13569708c3'>Galaxy Training Contribution Fest</a><strong> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/World) </em>Online<em> </td>
    <td> <a href='/BjoernGruening'>Björn Grüning</a> </td>
  </tr>
  <tr>
    <th rowspan=2> October 10 </th>
    <td> </em><a href='https://drive.google.com/file/d/0BwK61gB7NIcic1BnVmhxMFJEM3M/view'>Omics Data Services Platform: Supporting bacterial pathogen multi omics analysis</a><em> </td>
    <td rowspan=2> PLACEHOLDER_INCLUDE(/Events/Badges/Australia)  <a href='https://nectar.org.au/workshop-science-gateways/'>International Workshop on Science Gateways</a>, Melbourne, Australia </td>
    <td> Andrew Lonie </td>
  </tr>
  <tr>
    <td> </em><a href='https://drive.google.com/file/d/0BwK61gB7NIcia1EtUTBkaUgxU28/view'>Microbial Genomics Virtual Laboratory: Supporting bacterial pathogen multi omics analysis</a><em> </td>
    <td> Simon Gladman </td>
  </tr>
  <tr>
    <th> October 10-13 </th>
    <td> </em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=83'>Metagenomic: amplicon analysis</a><em> <div class='indent red'> Registration closes 4 October</div> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) INRA Auzeville, Auzeville, France </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) Sarah Maman </td>
  </tr>
  <tr>
    <th> October 13 </th>
    <td> </em><a href='https://www.facebook.com/events/1593583060947735/'>Galaxy Workshop</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) Copenhagen, Denmark </td>
    <td> <a href='http://jmgi.tk/'>Jose Maria (Txema) Gonzalez Izarzugaza</a> </td>
  </tr>
  <tr>
    <th> October 18-20 </th>
    <td> </em><a href='http://www.france-bioinformatique.fr/en/evenements/galaxy4bioinformatics'>Galaxy4Bioinformatics</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) Lyon, France </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='http://www.france-bioinformatique.fr/en/node/17'>Contacts</a> </td>
  </tr>
  <tr>
    <th> October 20 </th>
    <td> </em>Genome Program Joint Workshop<em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Asia) <a href='http://kunsan.ac.kr'>Kunsan National University</a>, Gunsan, South Korea </td>
    <td> <a href='/anton'>Anton Nekrutenko</a> </td>
  </tr>
  <tr>
    <th> October 20-21 </th>
    <td> </em><a href='http://www.ebi.ac.uk/training/events/2016/bioexcel-workflow-training-computational-biomolecular-research?platform=hootsuite'>BioExcel: Workflow Training for Computational Biomolecular Research</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) Barcelona Supercomputing Center, Barcelona, Spain </td>
    <td> <a href='https://siastd.uab.es/pcde/fitxa_persona.jsf;jsessionid=2cdbb231e18e06bc59f766ae8951:4crB?id=161b3185c321992a'>Antonio Miguel Espinosa Morales</a> </td>
  </tr>
  <tr>
    <th> October 20 </th>
    <td> </em><a href='/Events/Switzerland2016'>(third Swiss) Galaxy Workshop</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) </strong><a href='/Events/SG2016'>2016 Swiss-German Galaxy Days</a><strong> in Freiburg, Germany </td>
    <td> <a href='/HansrudolfHotz'>Hans-Rudolf Hotz</a> and <a href='/BjoernGruening'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <th> October 21 </th>
    <td> </em><a href='/Events/Germany2016'>(second German) Galaxy Developers Day</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) </strong><a href='/Events/SG2016'>2016 Swiss-German Galaxy Days</a><strong> in Freiburg, Germany </td>
    <td> <a href='/HansrudolfHotz'>Hans-Rudolf Hotz</a> and <a href='/BjoernGruening'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <th> October 24-27 </th>
    <td> </em>The Galaxy tool lifecycle supporting reproducibility in scientific research<em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) <a href='http://www.igst.it/nettab/2016/'>NETTAB 2016 / EMBNet Joint Meeting on Reproducibility in Bioinformatics</a>, Rome, Italy </td>
    <td> <a href='/DannonBaker'>Dannon Baker</a>, <a href='/BjoernGruening'>Björn Grüning</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> November 2-3 </th>
    <td> </em><a href='https://github.com/bioconda/bioconda-recipes/issues/2277'>Bioconda Contribution Fest</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/World) </em>Online<em> </td>
    <td> <a href='/BjoernGruening'>Björn Grüning</a> </td>
  </tr>
  <tr>
    <th> November 7-8 </th>
    <td> </strong><a href='/Events/AdminTraining2016/BasicsSession'>Galaxy Admin Training: Introduction</a><strong> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) Salt Lake City, Utah, United States </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='/GalaxyTeam'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <th> November 7-9 </th>
    <td> </em><a href='http://www.canceropole-idf.fr/formation-ngs-chipseq'>NGS & Cancer : Analyses ChIP-Seq</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe)  Paris, France </td>
    <td> <a href="mailto:communication@canceropole-idf.fr">Cancéropôle Île-de-France</a> </td>
  </tr>
  <tr>
    <th> November 8-20 </th>
    <td> </em><a href='https://meetings.cshl.edu/courses.aspx?course=c-seqtec&year=16'>Advanced Sequencing Technologies & Applications</a><em> <div class='indent'>Application deadline is July 15</div> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) <a href='https://cshl.edu/'>Cold Spring Harbor Laboratory</a>, New York, United States </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='/JeremyGoecks'>Jeremy Goecks</a> </td>
  </tr>
  <tr>
    <th> November 9-11 </th>
    <td> </strong><a href='/Events/AdminTraining2016/AdvancedSession'>Galaxy Admin Training: Advanced</a><strong> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) Salt Lake City, Utah, United States </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='/GalaxyTeam'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <th> November 15-16 </th>
    <td> </em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de données RNA-seq sous l’environnement Galaxy</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) PRABI, Lyon, France </td>
    <td> <a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Contact</a> </td>
  </tr>
  <tr>
    <th> November 20-25 </th>
    <td> </em><a href='http://www.france-bioinformatique.fr/evenements/EBA2016'>Ecole Bioinformatique AVIESAN</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) <a href='http://www.sb-roscoff.fr/'>Station Biologique, Roscoff</a>, France </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href="mailto:ecole-bioinfo@aviesan.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 21-23 </th>
    <td> </em><a href='https://urgi.versailles.inra.fr/content/view/full/3794'>Annotation of transposable elements</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) URGI, INRA Versailles, France </td>
    <td> <a href='https://urgi.versailles.inra.fr/content/view/full/3794'>Contact</a> </td>
  </tr>
  <tr>
    <th> November 28 - December 2 </th>
    <td> </em><a href='http://workflow4metabolomics.org/training/w4mcourse2016'>Traitement des donnéesmétabolomiques sous Galaxy</a><em> <div class='indent red'>Les pré-inscriptions sont ouvertes jusqu'au 15 juillet 2016</div> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe)  <a href='http://www.sb-roscoff.fr/'>Station Biologique, Roscoff</a>, France </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16)  <a href="mailto:contact@workflow4metabolomics.org">Contact</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" text-align: center; background-color: #eef;"> <span style="font-size: larger;"> </strong>2017<strong> </span> </td>
  </tr>
  <tr>
    <th> January 14-18 </th>
    <td> </em>Galaxy Workshop<em> </td>
    <td> <a href='http://www.intlpag.org/'>Plant and Animal Genome XXV</a>, San Diego, California, United States </td>
    <td> <a href='/DaveClements'>Dave Clements</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> February 3-9 </th>
    <td> </strong><a href='http://www.embl-abr.org.au/GAMe2017'>Galaxy Australasia Meeting (GAMe 2017)</a><strong> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Australia) University of Melbourne, Victoria, Australia </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='https://www.embl-abr.org.au/organisers/'>Organisers</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 26-30 </th>
    <td> </strong><a href='https://gcc2017.sciencesconf.org/'>2017 Galaxy Community Conference (GCC2017)</a><strong> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) Montpellier, France </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href="mailto:gcc2017_organization@groupes.france-bioinformatique.fr">Organisers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/Teach/Trainers'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/Teach/Trainers'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

---

# Galaxy Australia Community Launched

<div class='right'> <a href='https://www.embl-abr.org.au/galaxyaustralia/'><img src='/Images/Logos/GalaxyAustralia.png' alt='Galaxy Australia Community' width="180" /></a>
</div>

Today we officially launch the [Galaxy Australia](https://www.embl-abr.org.au/galaxyaustralia/) community with the opening of registrations for the Galaxy Australasia Meeting 2017 (GAMe 2017) being held in Melbourne over 3-9 February. Australian Galaxy users and administrators will now have more opportunities to interact and collaborate to identify and address the needs of the local community. The community will also be a hub for Australian Galaxy training courses and offer a catalog of Australian Galaxy servers.

The establishment of Galaxy Australia is supported by the [EMBL Australia Bioinformatics Resource (EMBL-ABR)](https://www.embl-abr.org.au/) in its appointed role of supporting the development of skills and training in the Australian life science research community. At the EMBL-ABR Hub, hosted at the Victorian Life Sciences Computation Initiative (VLSCI), Galaxy is used extensively for teaching and data analysis directly as well as through the Australian-made [Genomics Virtual Laboratory](https://www.genome.edu.au/). VLSCI’s expert bioinformaticians and professional staff will help promote community activities and training.

<div class='right'>
<a href='https://twitter.com/galaxyaustralia'><img src='/Images/Logos/TwitterBird300.png' alt='@GalaxyAustralia' height="50" /></a> 
</div>

Galaxy Australia has a has a Twitter feed ([@GalaxyAustralia](https://twitter.com/galaxyaustralia)) for announcements and discussion relevant to the community.  

**[Galaxy Australasia Meeting 2017 (GAMe 2017)](https://www.embl-abr.org.au/game2017)**

Our first activity is to organise the [Galaxy Australasia Meeting 2017 (GAMe 2017)](https://www.embl-abr.org.au/game2017), 3-9 February in Melbourne.  Registration and abstract submission are open.  See [the announcement](/src/News/GalaxyAustralia/index.md) for full details.



# New Publications

**117 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in September.  The total number of publication is now over 3600.

Some highlights from last month:

 **[The Essential Components of a Successful Galaxy Service](http://link.springer.com/article/10.1007/s10723-016-9379-6)**
   Annette !McGrath, Steve !McMahon, Sean Li, Joel Ludbey, Tim Ho. *Journal of Grid Computing* (2016), pp. 1-11, doi:10.1007/s10723-016-9379-6

 **[Enhancing pre-defined workflows with ad hoc analytics using Galaxy, Docker and Jupyter](http://biorxiv.org/content/early/2016/09/22/075457)**
   [Björn Grüning](/src/BjoernGruening/index.md), [Eric Rasche](/src/EricRasche/index.md), Boris Rebolledo-Jaramillo, [Carl Eberhard](/src/CarlEberhard/index.md), Torsten Houwaart, [John Chilton](/JohnChilton), [Nathan Coraor](/nate), Rolf Backofen, [James Taylor](/JamesTaylor), [Anton Nekrutenko](/src/anton/index.md). *bioRxiv* (16 September 2016), 075457, doi:10.1101/075457

 **[Genotyping of evolving prokaryotic populations](http://dx.doi.org/10.7287/peerj.preprints.2449v1)**
   Markus Zojer, Lisa N. Schuster, Frederik Schulz, Alexander Pfundner, Matthias Horn, Thomas Rattei. *PeerJ Preprints* (14 September 2016), doi:10.7287/peerj.preprints.2449v1

 **[Identifying Regions Enriched in a ChIP-seq Data Set (Peak Finding)](https://www.ncbi.nlm.nih.gov/pubmed/27574193)**
   Jui-Hung Hung, Zhiping Weng. *Cold Spring Harbor Protocols* (29 August 2016), doi:10.1101/pdb.prot093187

 **[Metavisitor, a suite of Galaxy tools for simple and rapid detection and discovery of viruses in deep sequence data](http://dx.doi.org/10.1101/048983)**
   Guillaume Carissimo, Marius van den Beek, Juliana Pegoraro, Kenneth D. Vernick, Christophe Antoniewski. *bioRxiv* (2016), doi:10.1101/048983

 **[Cloudflow - A framework for MapReduce pipeline development in Biomedical Research](http://dx.doi.org/10.1109/mipro.2015.7160259)**
   Lukas Forer, [Enis Afgan](/src/EnisAfgan/index.md), Hansi Weißensteiner, Davor Davidović, Günther Specht, Florian Kronenberg, Sebastian Schönherr. 2015 38th International Convention on Information and Communication Technology, Electronics and Microelectronics (MIPRO) (May 2015), pp. 172-177, doi:10.1109/mipro.2015.7160259

## Tagged Publications

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 27 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 58 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 31 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


---

# Who's Hiring

<div class='right'><br />![Please Help! Yes you!](/src/Images/GalaxyIsExpandingCloud.png)</div>

The Galaxy is expanding! Please help it grow.

* [6 month position](https://twitter.com/bioinformAnt/status/773465361127710720) at [Public Health England](https://www.gov.uk/government/organisations/public-health-england) working [National Microbiology Laboratory](https://www.nml-lnm.gc.ca/index-eng.htm) in Canada to compare antimicrobial resistance isolates and software using [IRIDA](http://www.irida.ca/)/Galaxy
* [Intégration d'outils pour la métagénomique phytovirale dans Galaxy](http://www.sfbi.fr/content/int%C3%A9gration-doutils-pour-la-m%C3%A9tag%C3%A9nomique-phytovirale-dans-galaxy-0), INRA, Bordeaux, France
* [Post­doctoral Research Position](https://drive.google.com/file/d/0B5_H88SQ1C_7Z3dJY3FzU0hQMUU/view?pref=2&pli=1), Multi­omic Bioinformatics, University of Minnesota
* [Postdoctoral Researcher: Forest Genomics Database and Software Developer](http://compgenomics.lab.uconn.edu/join/), University of Connecticut.  "... data sharing among partner databases through Galaxy modules." <div class='red'>Act fast.</span>  

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

---


# Public Galaxy Server News

There are [over 80 publicly accessible Galaxy servers](/src/PublicGalaxyServers/index.md) and [five semi-public Galaxy services](/src/SemiPublicGalaxyServers/index.md).  Here's what happened with them in September.

## New Public Galaxy Servers

### Galaxy-CEFAP

<div class='right solid'><a href='http://cefap.icb.usp.br/galaxy'><img src='/PublicGalaxyServers/CEFAP-Logo.png' alt='Galaxy-CEFAP' width="200" /></a></div>

[Galaxy-CEFAP](http://cefap.icb.usp.br/galaxy) offers a set of tools to perform RNA-Seq and miRNA. User support is available via [geninfo AT icb DOT usp DOT br](Email), and there is an [FTP tutorial](http://cefap.icb.usp.br/galaxy/etc/galaxy/web/welcome_upload_ftp.pdf) as well.  Anyone can use the server, but it is necessary to create a login first.  Email [geninfo AT icb DOT usp DOT br](geninfo AT icb DOT usp DOT br) to request an account. There is a storage quota for all users.

[Galaxy-CEFAP](http://cefap.icb.usp.br/galaxy) is supported by [Instituto de Ciências Biomédicas - ICB](http://www3.icb.usp.br/), [Universidade de São Paulo - USP](http://www5.usp.br/), and [Fundação de Amparo à Pesquisa de SP - FAPESP](http://www.fapesp.br/) hosted by [Centro de Facilidades de Apoio à Pesquisa - CEFAP-USP](http://cefap.icb.usp.br/).

### VarCap

<div class='right solid'><a href='http://galaxy.csb.univie.ac.at:8080/'><img src='/PublicGalaxyServers/VarCapWorkflow.png' alt='VarCap' width="200" /></a></div>

[VarCap Galaxy](http://galaxy.csb.univie.ac.at:8080/) performs reliable prediction of different types of variants even at low frequencies. See
 Zojer M, Schuster LN, Schulz F, Pfundner A, Horn M, Rattei T. (2016) [Genotyping of evolving prokaryotic populations](https://peerj.com/preprints/2449/). *PeerJ Preprints* 4:e2449v1 doi: 10.7287/peerj.preprints.2449v1

for a description.  A [tutorial](http://galaxy.csb.univie.ac.at:8080/static/welcome.html) is also available.  Users need to create a login to use the server.  [VarCap Galaxy](http://galaxy.csb.univie.ac.at:8080/) is supported by [Department of Microbiology and Ecosystems Science](http://dmes.univie.ac.at/) at the [University of Vienna](http://univie.ac.at/).

## Semi-Public Galaxy Services

Jetstream was officially launched in September and was added to the [Semi-Public Galaxy Services list](/src/SemiPublicGalaxyServers/index.md):

### United States: Jetstream

<div class='right solid'><a href='https://galaxy.plgrid.pl'><img src='/Images/Logos/JetstreamRectClipped.jpg' alt='Jetstream' width="280" /></a></div>

* *Links:*
  * **[Galaxy on Jetstream](/src/Cloud/Jetstream/index.md)**
* *Eligibility:*
  * [Jetstream](http://jetstream-cloud.org/) is part of [XSEDE](https://www.xsede.org/), a "collection of integrated advanced digital resources and services" and is funded by NSF. You need to be eligible for an [XSEDE allocation](https://portal.xsede.org/allocation-policies) to use Jetstream, which means must be based at a U.S. institution. And although XSEDE is NSF-funded, "projects need not be supported by NSF grants" to receive an allocation.
* *Comments:*
  * Jetstream enables researchers to launch, use, and shutdown their own Galaxy servers that have been pre-configured similar to the [Main Galaxy server](https://usegalaxy.org/). If you want to further customize your launched server you can [become an administrator](/src/Cloud/Jetstream/ssh/index.md#adding_galaxy_admin_user) and even [access the server from the shell](/src/Cloud/Jetstream/ssh/index.md).
* *User Support:*
  * See the [Galaxy on Jetstream documentation](/src/Cloud/Jetstream/index.md)
  * [FAQ](/src/Cloud/Jetstream/FAQ/index.md)
* *Quotas:* 
  * Limits are determined by the size of your allocation and the size of the instances you launch.
* *Sponsor(s):*
  * [The National Science Foundation (NSF)](http://www.nsf.gov/)
  * [Jetstream](http://jetstream-cloud.org/) is based at [Indiana University](https://www.iu.edu/) and Jetstream resources are hosted by Indiana University and the [Texas Advanced Computing Center (TACC)](https://www.tacc.utexas.edu/).

# Trinity comes to usegalaxy.org

<div class='right'><a href='https://trinityrnaseq.github.io'><img src='/Images/Logos/TrinityLogo.png' alt='Trinity assembler' width="150" /></a></div>

For those looking to perform de novo transcriptome assembly of RNA-seq data, we are pleased to announce that the [Trinity assembler](https://trinityrnaseq.github.io) is now available in beta on the Galaxy Project's free, public Galaxy server, [usegalaxy.org](https://usegalaxy.org/). Due to the high memory requirements of de novo assembly, Trinity runs on the [Bridges](http://www.psc.edu/bridges) high memory HPC system at the [Pittsburgh Supercomputing Center](https://www.psc.edu), via a science gateway grant from [XSEDE](https://www.xsede.org). Please try the tool out and be sure to report any problems via Galaxy's reporting mechanism.

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Teach'><img src='/Images/GalaxyLogos/GTNLogo300.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Community/Logs'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Community/Deployments'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong>Share your <a href='/Teach/Resources#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers#add-a-trainer'>experience</a> </em>now<em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/Community/Logs#add-a-log-page'>Share your experience</a> </em><a href='/Community/Logs#add-a-log-page'>now</a><em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/Community/Deployments#add-your-galaxy-deployment'>Describe your instance</a> </em><a href='/Community/Deployments#add-your-galaxy-deployment'>now</a><em> <strong> </td>
  </tr>
</table>

<br />

One new trainng resource was added in September:

* [Microbial data analysis with Orione](/src/Teach/Resource/MicrobialWithOrione/index.md), by 鈴木 治夫 (Haruo Suzuki).


---

# Releases

## galaxy-lib 16.10.2

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

The September releases work with recents changes to Galaxy and initial mulled scripts and container resolver.

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src='/Images/GalaxyLogos/planemo-logo.png' alt='' width="200" /></a></div>

## Planemo 0.33.2

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  **Planemo saw seven releases in September.**  The biggest news is that Planemo now uses the brand new [Galaxy Tool XML Schema Definition](https://github.com/galaxyproject/planemo/blob/master/planemo/xml/xsd/tool/galaxy.xsd) to validate tool definitions.  A lot happened with Conda support too. 

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/0.33.1/HISTORY.rst). 



## Earlier Releases

Other packages that have been released in the prior 4 months.

### Galaxy v16.07

<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GalaxyNewLogo_GalaxyProject_Trans.png' alt='GalaxyProject' width="200" /></a></div>
 
The [Galaxy Committers team](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) is pleased to announce the **[July 2016 (v16.07) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.07_announce.html)**.

<span style="font-size: larger;"> Highlights </span>

**Shift of Galaxy tool dependencies to Conda**

Galaxy admins obtained the ability to install Galaxy tool’s dependencies using the Conda package manager. This is a Beta feature and we encourage interested deployers to opt-in by modifying configuration. Documentation that explains this switch and answers FAQ is [available](https://docs.galaxyproject.org/en/master/admin/conda_faq.html).

**Dynamic tool destinations**

Our friends from Canada National Microbiology Laboratory enhanced Galaxy with feature that allows dynamic mapping of tools to destinations based on finer grained admin-specified rules. Please see the [wiki](/src/Admin/Config/Jobs/index.md#dynamic-destination-mapping). Implemented in [PR #2579](https://github.com/galaxyproject/galaxy/pull/2579).

**Galaxy chat**

Admins can now plug in the included communication server to enable users of their instance to use real-time chat within the Galaxy interface. Please see the [documentation](https://docs.galaxyproject.org/en/master/admin/chat.html) to learn how to activate and use this feature. Implemented in [PR #2515](https://github.com/galaxyproject/galaxy/pull/2515).

<span style="font-size: larger;"> [Github](https://github.com/galaxyproject/galaxy) </span>

**New Galaxy repository**
```
$ git clone -b release_16.07 https://github.com/galaxyproject/galaxy.git
```


**Update of existing Galaxy repository**
```
$ git checkout release_16.07 && git pull --ff-only origin release_16.07
```


See [our wiki](/src/Develop/SourceCode/index.md) for additional details regarding the source code locations.

<br />

See the [full release notes](https://docs.galaxyproject.org/en/master/releases/16.07_announce.html) for more.

#### Galaxy Docker Image 16.07

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 16.07](https://github.com/bgruening/docker-galaxy-stable/releases/tag/16.07) as well.  

### CloudBridge 0.1.1

[CloudBridge](https://github.com/gvlproject/cloudbridge) provides a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud. The latest release adds support for advanced networking management and !OpenStack Keystone v3 support (required to use NSF Jetstream cloud).

### BioBlend 0.8.0

[BioBlend](http://bioblend.readthedocs.org/) is a Python library for interacting with CloudMan and Galaxy‘s API.  !BioBlend makes it possible to script and automate the process of cloud infrastructure provisioning and scaling via CloudMan, and running of analyses via Galaxy.

See the [release notes](https://github.com/galaxyproject/bioblend/releases) for what's new in release 0.8.0.

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src='/Images/GalaxyLogos/pulsar_transparent.png' alt='Pulsar' width="170" /></a></div>

### Pulsar 0.7.1 - 0.7.2

[Pulsar](https://pypi.python.org/pypi/pulsar-app/) updates were released in August.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.


### blend4php 0.1 alpha

<div class='right'><a href='https://github.com/galaxyproject/blend4php'><img src='/Images/GalaxyLogos/blend4phpLogo.png' alt='' width="220" /></a></div>

The alpha release of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html) was announced in June. It follows the lead of [BioBlend](https://bioblend.readthedocs.io/en/latest/) which provides a Python package for interacting with Galaxy and CloudMan--hence the use of 'blend' in the name of this package. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases. Integration of Tripal and Galaxy will allow the community research databases to provide next-generation analytical tools to their users using Galaxy. However, this library was created independently of Tripal to support integration of any PHP application with Galaxy.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/docs-v0.1a/html/index.html) page for full information.


### CloudMan 16.04

<div class='right'><a href='/CloudMan'><img src='/Images/GalaxyLogos/cloudman-logo.jpg' alt='CloudMan' width="300" /></a></div>

An update to Galaxy !CloudMan on AWS was released in May. CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

This is a minor update release with the following changes:
* Galaxy 16.04 update
* Availability on Amazon's Ireland region 
* A couple of bug fixes

See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a more complete set of changes.

### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

---

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

[Tool Shed contributions for September](/src/ToolShed/Contributions/2016_09/index.md).

# Other News

From [Ravi Madduri](http://twitter.com/madduri)<br />
 [Globus Genomics](https://www.globus.org/genomics) RO1 was funded by [NHGRI](http://genome.gov). 

From [Björn Grüning](http://twitter.com/bjoerngruening)
 [HTTPS support was added to the Galaxy Docker image](https://github.com/bgruening/docker-galaxy-stable/pull/240). Joint work by [Marius van den Beek](http://twitter.com/mvdbeek) and [Alexander Lenail](http://twitter.com/zfrenchee) 
