<div class="title">Galactic November News!</div>

<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em"> TABLE_OF_CONTENTS </td>
  </tr>
</table>

</div>

<div class='left'><a href='/GalaxyUpdates.md'><img src='/Images/GalaxyLogos/GalaxyNews.png' alt='Galaxy News' width=150 /></a></div>

Welcome to the November 2015 *[Galactic News](/GalaxyUpdates)*, a summary of what is going on in the Galaxy community. 

If you have anything to include in the next News, please send it to [outreach AT galaxyproject DOT org](Galaxy Outreach). 

# New Papers

**81** new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in October. Highlights include:

* **[Genomics Virtual Laboratory: A Practical Bioinformatics Workbench for the Cloud](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0140829)**
    [Enis Afgan](/EnisAfgan), Clare Sloggett, Nuwan Goonasekera, Igor Makunin, Derek Benson, Mark Crowe, Simon Gladman, Yousef Kowsar, Michael Pheasant, Ron Horst, Andrew Lonie, *PLoS ONE* October 26, 2015 DOI: 10.1371/journal.pone.0140829.<br />
    See also [Australian-made Genomics Virtual Laboratory: A Practical Bioinformatics Workbench for the Cloud, ...](https://www.vlsci.org.au/news/GVLnews2015).

* **[Analyzing HT-SELEX data with the Galaxy Project tools - a web based bioinformatics platform for biomedical research](http://dx.doi.org/10.1016/j.ymeth.2015.10.008)**
    William H. Thiel, Paloma H. Giangrande, *Methods* (October 2015), doi:10.1016/j.ymeth.2015.10.008

* **[ReproPhylo: An Environment for Reproducible Phylogenomics](http://dx.doi.org/10.1371/journal.pcbi.1004447)**  
    Amir Szitenberg, Max John, Mark L. Blaxter, David H. Lunt, *PLoS Comput Biol*, Vol. 11, No. 9. (3 September 2015), e1004447, doi:10.1371/journal.pcbi.1004447

* **[RNA-Seq UD: A bioinformatics platform for RNA-Seq analysis](http://dx.doi.org/10.1109/cisti.2015.7170565)**
    Miguel Ramirez, Cristian A. Rojas-Quintero, Nelson E. Vera-Parra, in "Information Systems and Technologies (CISTI), 2015 10th Iberian Conference on" (June 2015), pp. 1-5, doi:10.1109/cisti.2015.7170565


* **[BioImg.org](http://bioimg.org): [A Catalog of Virtual Machine Images for the Life Sciences](http://ow.ly/S3VJU)**
    Martin Dahl√∂, Fr√©d√©ric Haziza, Aleksi Kallio, Eija Korpelainen, Erik Bongcam-Rudloff and Ola Spjuth, *Bioinformatics and Biology Insights* 2015:9 125‚Äì128 doi: 10.4137/BBI.S28636

* **[SBMLsqueezer 2: context-sensitive creation of kinetic equations in biochemical networks](http://dx.doi.org/10.1186/s12918-015-0212-9)**
    Andreas Dr√§ger, Daniel C. Zielinski, Roland Keller, *et al*., *BMC Systems Biology*, Vol. 9, No. 1. (09 October 2015), 68, doi:10.1186/s12918-015-0212-9

* **[RDF2Graph a tool to recover, understand and validate the ontology of an RDF resource](http://dx.doi.org/10.1186/s13326-015-0038-9)**
    Jesse C. J. van Dam, Jasper J. Koehorst, Peter J. Schaap, Vitor A. P. Martins dos Santos, Maria Suarez-Diez, *Journal of Biomedical Semantics*, Vol. 6, No. 1. (23 October 2015), 39, doi:10.1186/s13326-015-0038-9

* **[MicRhoDE: a curated database for the analysis of microbial rhodopsin diversity and evolution](http://dx.doi.org/10.1093/database/bav080)**
    Dominique Boeuf, St√©phane Audic, Loraine Brillet-Gu√©guen, Christophe Caron, Christian Jeanthon, *Database*, Vol. 2015 (01 January 2015), bav080, doi:10.1093/database/bav080

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 18 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 16 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 43 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


----

# Events

<div class='left'><br />
<a href='/Events/GCC2016/Training.md'><img src='/Events/GCC2016/GCC2016TrainingLogo400.png' alt='GCC2016 Training' width="150" /></a><br />
<table>
  <tr>
    <th> &nbsp; <a href='http://bit.ly/gcc2016trnvote'>Vote now!</a> &nbsp; </th>
  </tr>
</table>

</div>

## GCC2016 Training Topic Vote Ends This Friday

Thanks to everyone who nominated topics for [GCC2016 Training](/Events/GCC2016/Training).  Please take a few minutes to [vote on the topics](http://bit.ly/gcc2016trnvote) you want to see offered at GCC2016 Training.

The [2016 Galaxy Community Conference (GCC2016)](/Events/GCC2016) features two full days of training on June 26-27.  Each 
day will have multiple tracks, with each track featuring several sessions throughout the day.  GCC2016 will be held at [Indiana University](http://indiana.edu) in Bloomington, Indiana, United States.

**[Topic voting](http://bit.ly/gcc2016trnvote) closes November 6.**

Topics will then be selected and scheduled based on topic interest, and the organizers' ability to confirm instructors for each session.  Some very popular sessions may be scheduled more than once. The final schedule will be posted before registration opens (which we are hoping will be the end of November).


## IUC Contribution Fest - Metagenomics Tools and Workflows

[We](/IUC) are planning a [remote hackathon on 30th of November and 1st of December for developers to work mainly on Galaxy metagenomic tools](https://github.com/galaxyproject/tools-iuc/issues/299).

The remote nature of this gives people who don't have the opportunity to come to GCC hackathons (which have always been productive and a lot of fun) a chance to participate in a Galaxy hackathon. Having a well defined topic will allow us to accomplish a lot and let people who don't have particular tasks in mind find something to work on very quickly.

We are [collecting ideas to work on](https://github.com/galaxyproject/tools-iuc/issues/299), but we expect to attract the most participation by simply getting tool developers interested in getting help adding collection support to their existing tools and workflows to show up and participate.

If you are interested in participating in the hackathon but not interested in actual tool development - we will assemble a list of smaller, manageable Python and !JavaScript tasks to work on and certainly documentation is a chronically lacking for collections so we could use help there and no actual coding would be required.

We encourage ideas or advice about how to organize this so please let us know. A core group will be available on IRC all day and we will have 4 google hangouts across those days to organize, answer questions, and report progress.

We will do our best to coordinate and make this hackathon a nice and productive experience and we would like to especially focus on working reasonable hours and discourage overnighters.

All [forms of contribution](https://github.com/galaxyproject/tools-iuc/issues/299) are welcome!

See [the event home page for more](https://github.com/galaxyproject/tools-iuc/issues/299).

<div class='center'> 
<a href='http://www.festivalofgenomicscalifornia.com/'><img src='/Images/Logos/FestivalOfGenomicsCaliforniaLogo.png' alt='3 Workshop: Galaxy for the Analysis of Very Large Biomedical Data; Common Workflow Language on Galaxy, Part of Portable Bioinformatics; Single-Cell Sequencing: Analysis & Visualization of Copy Number Variations' height="60" /></a>
<a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'><img src='/Images/Logos/PRABILogo.png' alt='Analyse de donn√©es RNA-seq sous l‚Äôenvironnement Galaxy' height="70" /></a>
<a href='http://www.france-bioinformatique.fr/?q=fr/galaxy-2'><img src='/Images/Logos/IFB_galaxywg.png' alt='Galaxy Day 2015' height="70" /></a>
<a href='http://www.bioplatforms.com/current-training-courses/'><img src='/Images/Logos/BioplatformsAustralia.png' alt='Galaxy Training from Bioplatforms Australia and Intersect' height="40" /></a>
</div>


## Call for GCC2017 Hosts

<table>
  <tr>
    <td style=" text-align: center; border: none"> <a href='/GCC2011.md'><img src='/Events/GCC2011Logo400.png' alt='' width="120" /></a>&nbsp;&nbsp; <a href='/Events/GCC2012.md'><img src='/Events/GCC2012/GCC2012Logo200.png' alt='2012 Galaxy Community Conference' width="85" /></a>&nbsp;&nbsp;&nbsp; <a href='/Events/GCC2013.md'><img src='/Images/Logos/GCC2013Logo200.png' alt='2013 Galaxy Community Conference (GCC2013)' width="100px" /></a>&nbsp;&nbsp;&nbsp; <a href='/Events/GCC2014.md'><img src='/Images/Logos/GCC2014LogoWide200.png' alt='GCC2014, June 30-July2' width="130" /></a>&nbsp;&nbsp;&nbsp; <a href='http://gcc2015.tsl.ac.uk/'><img src='/Images/Logos/GCC2015LogoWide600.png' alt='2015 Galaxy Community Conference (GCC2015)' width="100" /></a> &nbsp;&nbsp; <a href='http://galaxyproject.org/GCC2016'><img src='/Events/GCC2016/GCC2016LogoFull_big.png' alt='2016 Galaxy Community Conference (GCC2016)' width="110px" /></a> </td>
  </tr>
  <tr>
    <td colspan=6 style=" text-align: center; border: none"> <div class='solid'><div class='red'> <span style="font-size: larger;"> </strong>GCC2017 needs a host!<strong><br /></span> <span style="font-size: smaller;">(and a logo)</span> </td>
  </tr>
</table>


<br />

**The [call for hosts for GCC2017](ATTACHMENT_URLDocuments/GCC2016CallForHosts.pdf) closes 30 November.**  If you are interested, please see [this announcement](/News/GCC2017CallForHost). 

## October GalaxyAdmins Meetup Video and Slides

<div class='right'><a href='/Community/GalaxyAdmins/Meetups/2015_10_15.md'><img src='/Images/GalaxyLogos/GalaxyAdmins.png' alt='GalaxyAdmins meetup October 15' width="175" /></a><br /><br />
<a href='http://www.analome.com/'><img src='/Images/Logos/Analome.png' alt='Analome' width="175" /></a>
</div>

Video and slides for the [GalaxyAdmins meetup on October 15 are now available](/Community/GalaxyAdmins/Meetups/2015_10_15). [David Kovalic](https://www.linkedin.com/in/davidkovalic) presented on *Galaxy at [Analome](http://analome.com/) ‚Äì Outreach, Opportunity and Challenges*.  This was followed by an active discussion about plans to migrate this wiki to another platform.

[GalaxyAdmins](/Community/GalaxyAdmins) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  We're skipping the December meetup (wouldn't you rather be on holiday?) so the next meetup will be in February.  Hope to see you there.

----


## Upcoming Events

There are [upcoming events](/Events) on 3 continents in 4 countries, *and a veritable globular cluster of events in France this month*.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='center'> 
<a href='http://www.festivalofgenomicscalifornia.com/'><img src='/Images/Logos/FestivalOfGenomicsCaliforniaLogo.png' alt='3 Workshop: Galaxy for the Analysis of Very Large Biomedical Data; Common Workflow Language on Galaxy, Part of Portable Bioinformatics; Single-Cell Sequencing: Analysis & Visualization of Copy Number Variations' height="60" /></a>
<a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'><img src='/Images/Logos/PRABILogo.png' alt='Analyse de donn√©es RNA-seq sous l‚Äôenvironnement Galaxy' height="70" /></a>
<a href='http://www.france-bioinformatique.fr/?q=fr/galaxy-2'><img src='/Images/Logos/IFB_galaxywg.png' alt='Galaxy Day 2015' height="70" /></a>
<a href='http://www.bioplatforms.com/current-training-courses/'><img src='/Images/Logos/BioplatformsAustralia.png' alt='Galaxy Training from Bioplatforms Australia and Intersect' height="40" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th rowspan=3> November 3 </th>
    <td> <em><a href='http://www.festivalofgenomicscalifornia.com/workshops/galaxy-for-the-analysis-of-very-large-biomedical-data/'>Galaxy for the Analysis of Very Large Biomedical Data Workshop</a></em> </td>
    <td rowspan=3> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.festivalofgenomicscalifornia.com/'>Festival of Genomics California</a>, San Mateo, California, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='/Dan.md'>Dan Blankenberg</a>, <a href='/JohnChilton.md'>John Chilton</a> </td>
  </tr>
  <tr>
    <td> <em>Common Workflow Language on Galaxy</em><div class='indent'>Part of <em><a href='http://bit.ly/1POcrKL'>Portable Bioinformatics Workshop</a></em> </td>
    <td> <a href='/JohnChilton.md'>John Chilton</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.festivalofgenomicscalifornia.com/workshops/single-cell-sequencing-analysis-visualisation-of-copy-number-variations/'>Single-Cell Sequencing: Analysis & Visualization of Copy Number Variations Workshop</a></em> </td>
    <td> <a href='https://www.linkedin.com/in/robertaboukhalil'>Robert Aboukhalil</a>, <a href='https://www.linkedin.com/in/tylergarvin'>Tyler Garvin</a> </td>
  </tr>
  <tr>
    <th> November 12 </th>
    <td> <em><a href='http://bioteam.net/bioteam-appliance/galaxy-edition/galaxy-training/'>BioTeam & UC Davis Webinar: Introduction to Genomics Data Analysis with Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/World) <a href='http://bioteam.net/bioteam-appliance/galaxy-edition/galaxy-training/'>Webinar</a> - Open to all </td>
    <td> <a href="mailto:tom@bioteam.net">Email</a></td>
  </tr>
  <tr>
    <th> November 15-16 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de donn√©es RNA-seq sous l‚Äôenvironnement Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <div class='right'><a href='/Teach/Trainers.md#galaxy--prabi-project'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a> </div> <a href="mailto:contact@fc3bio.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 17-18 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de donn√©es RNA-seq sous l‚Äôenvironnement Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <div class='right'><a href='/Teach/Trainers.md#galaxy--prabi-project'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a> </div> <a href="mailto:contact@fc3bio.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 18 </th>
    <td> <a href='http://france.list.galaxyproject.org/Re-bioinfo-Hackathon-Galaxy-2015-ouverture-des-inscriptions-td152.html'>Hackathon Galaxy 2015</a> <div class='indent'><a href='http://www.france-bioinformatique.fr/?q=fr/galaxy-2/Hackathon_18nov'>Register here</a></div> </td>
    <td rowspan=2> INCLUDE(/Events/Badges/Europe) <a href='http://www.pasteur.fr/'>Institut Pasteur</a>, Paris, France </td>
    <td rowspan=2> <a href="mailto:gtgalaxy@groupes.france-bioinformatique.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 19 </th>
    <td> <a href='http://www.france-bioinformatique.fr/?q=fr/galaxy-2'>Galaxy Day 2015</a> </td>
  </tr>
  <tr>
    <th> November 30 - December 1 </th>
    <td> <a href='https://github.com/galaxyproject/tools-iuc/issues/299'>IUC Contribution Fest - Metagenomics Tools and Workflows</a> </td>
    <td> INCLUDE(/Events/Badges/World) <em>Online</em> </td>
    <td> <a href='/IUC.md'>/IUC.md</a>, <a href='/JohnChilton.md'>John Chilton</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> December 3 </th>
    <td> <em><a href='http://www.bioplatforms.com/rna-seq-using-galaxy-dec15/'>RNA-Seq Analysis Using Galaxy</a></em> </td>
    <td rowspan=2> INCLUDE(/Events/Badges/Australia) University of Newcastle, Newcastle, NSW, Australia </td>
    <td rowspan=2> INCLUDE(/Teach/GTN/Badge16) Mark Crowe, <a href="mailto:kchamp@bioplatforms.com">Katherine Champ</a> </td>
  </tr>
  <tr>
    <th> December 4 </th>
    <td> <em><a href='http://www.bioplatforms.com/variant-detection-using-galaxy-dec15/'>Variant Detection Using Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> December<br />7-11 </th>
    <td> <em><a href='http://bis15.amsi.org.au/galaxy/'>Galaxy and the Genomics Virtual Lab</a></em> </td>
    <td> INCLUDE(/Events/Badges/Australia) <a href='http://bis15.amsi.org.au/'>BioInfoSummer 2015</a>, University of Sydney, NSW, Australia </td>
    <td> </td>
  </tr>
  <tr>
    <td> </td>
    <td colspan=4 style=" background-color: #eef text-align: center;"> </strong>2016<strong> </td>
  </tr>
  <tr>
    <th> January 8 </th>
    <td> <em><a href='/Events/ASU2016.md'>Introduction to Galaxy Workshop</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://asu.edu/'>Arizona State University</a>, Tempe, Arizona, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='https://sols.asu.edu/people/nico-franz'>Nico Franz</a>, <a href='https://sols.asu.edu/people/melissa-wilson-sayres'>Melissa A. Wilson Sayres</a>, <a href='/anton.md'>Anton Nekrutenko</a> </td>
  </tr>
  <tr>
    <th rowspan=2> January 9-13 </th>
    <td> <em><a href='https://pag.confex.com/pag/xxiv/webprogram/Paper19738.html'>TreeGenes: Enabling visualization and analysis in forest tree genomics</a></em> </td>
    <td rowspan=2> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.intlpag.org/'>Plant and Animal Genome (PAG XXIV)</a>, San Diego, California, United States </td>
    <td> Emily Grau </td>
  </tr>
  <tr>
    <td> <em><a href='https://pag.confex.com/pag/xxiv/webprogram/Session3098.html'>Galaxy for SNP and Variant Data Analysis</a></em> </td>
    <td> <a href='/DaveClements.md'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> January 19 </th>
    <td> <em><a href='http://www.festivalofgenomicslondon.com/an-introduction-to-clinical-bioinformatics/'>An introduction to clinical bioinformatics</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) <a href='http://www.festivalofgenomicslondon.com/'>Festival of Genomics London</a>, London, United Kingdom </td>
    <td> <a href='http://www.festivalofgenomicslondon.com/speaker/ang-davies/'>Ang Davies</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> February 20-23 </th>
    <td> <em><a href='https://conf.abrf.org/the-galaxy-platform'>(SW4) The Galaxy Platform for Multi-Omic Data Analysis and Informatics</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica)  <a href='https://conf.abrf.org/abrf-2016'>ABRF 2016</a>, Fort Lauderdale, Florida, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='https://cbs.umn.edu/contacts/pratik-d-jagtap'>Pratik Jagtap</a>, <a href='https://cbs.umn.edu/contacts/timothy-j-griffinhttps://cbs.umn.edu/contacts/timothy-j-griffin'>Tim Griffin</a>, <a href='/DaveClements.md'>Dave Clements</a>, <a href='http://experts.umn.edu/en/persons/candace-r-guerrero(c3134cd2-fe98-4d90-b5de-77fce1921d29).html'>Candace Guerrero</a>, <a href='http://www-users.cs.umn.edu/~onsongo/'>Getiria Onsongo</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> June 25-29 </th>
    <td> </strong><em><a href='/Events/GCC2016.md'>2016 Galaxy Community Conference (GCC2016)</a><strong></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16)  <a href='/Events/GCC2016/Organizers.md'>Organizers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none"> <a href='/Teach/Trainers.md'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none"> <span style="font-size: smaller;"> <a href='/Teach/Trainers.md'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring.md'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Postdoc developing advanced MS-based metabolomics bioinformatics methods](http://jobs.sciencecareers.org/job/383809/bioinformatics-of-mass-spectrometry-based-metabolomics/), University of Minnesota. *Closes November 30.*
* [Software Developer / Bioinformatician](https://ig14.i-grasp.com/fe/tpl_embl01.asp?s=4A515F4E5A565B1A&jobid=54309,8734341283&key=64564906&c=342372129814&pagestamp=seqbdpnrnmbvdpxtgb), European Molecular Biology Laboratory (EMBL), Heidelberg, Germany
* **The [Galaxy Project is hiring software engineers and post-docs](/GalaxyIsHiring)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />

----

# New Public Galaxy Servers

[1 new publicly accessible Galaxy servers](/PublicGalaxyServers) was added in October, bringing the total dangerously close to 80.

## Galaxy@PRABI

<div class='right'><div class='solid'><a href='http://galaxy.prabi.fr/'><img src='/Images/Logos/PRABILogo.png' alt='Galaxy@PRABI' width="200" /></a></div>

* *Links:*
  * [Galaxy@PRABI](http://galaxy.prabi.fr/)
  * [PRABI Galaxy Tool Shed](http://toolshed.prabi.fr/)
* *Domain/Purpose:*
  * A general purpose Galaxy instance that includes bioinformatics tools developed by the research teams working in the perimeter of the PRABI core facility, including *kissplice/kissDE, TETools, !SexDetector,* and *priam* available [through our local toolshed instance](http://toolshed.prabi.fr). 
* *Comments:*
  * Dedicated server: !PowerEdge R920 Rack Server - 64 CPU, 512 GB RAM, 15To GB of disk space. 
* *User Support:*
  * [galaxy@prabi user list](http://listes.univ-lyon1.fr/sympa/info/galaxy-user) (in French) 
  * Email support: [galaxy DASH support AT listes DOT univ-lyon1 DOT fr](galaxy DASH support AT listes DOT univ-lyon1 DOT fr) 
  * We also provide galaxy [training courses](/Teach/Trainers.md#galaxy_40_prabi_project) for RNA-seq and ChIP-seq data analysis.
* *Quotas:* 
  * Disk space is limited to 100 MB for unregistered users, registered users are free to use up to 50 GB.
  * Users from the FRBioenvis and LECA teams are free to use up to 250 GB disk space.
  * Can be increased up to 1-2 TB in collaborative projects (please contact [galaxy DASH support AT listes DOT univ-lyon1 DOT fr](galaxy@PRABI Admins).)
* *Sponsor(s):*
  * [PRABI](http://www.prabi.fr)
  * [FRBioenvis](http://bioenvis.universite-lyon.fr), [Universit√© Lyon 1](http://www.univ-lyon1.fr), [LECA](http://www-leca.ujf-grenoble.fr)
<br />

## RDF2Graph

<div class='right'><div class='solid'><a href='http://semantics.systemsbiology.nl/RDF2Graph/'><img src='/PublicGalaxyServers/RDF2Graph_reactome.png' alt='RDF2Graph using Reactome' width="200" /></a></div>

* *Link:*
  * [RDF2Graph Server](http://semantics.systemsbiology.nl/RDF2Graph/)
* *Domain/Purpose:*
  * RDF2Graph recovers the structure of an RDF resource.
* *Comments:*
  * Graphs can be exported to [Cytoscape](http://www.cytoscape.org/), OWL, and [ShEx](http://www.w3.org/2001/sw/wiki/ShEx).
  * See "[RDF2Graph a tool to recover, understand and validate the ontology of an RDF resource](http://www.jbiomedsem.com/content/6/1/39/)" by van Dam *et al.*, *Journal of Biomedical Semantics* 2015, 6:39 (2013)
* *User Support:*
  * [RDF2Graph Manual](http://semantics.systemsbiology.nl/RDF2Graph/static/Manual.html)
* *Quotas:* 
  * Server can be used by anyone, but you'll need to create an account.
  * The RDF resource is limited to 20,000,000 lines.
* *Sponsor(s):*
  * [Laboratory of Systems and Synthetic Biology](http://www.wageningenur.nl/en/Expertise-Services/Chair-groups/Agrotechnology-and-Food-Sciences/Laboratory-of-Systems-and-Synthetic-Biology.htm), [Wageningen University](http://www.wageningenur.nl/)

----


# Releases

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src='/Images/Logos/PlanemoLogo.png' alt='' width="150" /></a></div>

## Planemo 0.15.0 - 0.18.1

[Planemo](https://pypi.python.org/pypi/planemo) saw a meteor storm (sorry!) of activity in October with 4+ releases.

Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.  [Planemo 0.18.1](https://pypi.python.org/pypi/planemo/0.18.1#history) is the most recent release. See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).

## BioBlend 0.7.0

[BioBlend version 0.7.0](https://github.com/galaxyproject/bioblend/) was released few days ago. !BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/Learn/API).  CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG).


## Other Releases

<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GalaxyNewLogo_GalaxyProject_Trans.png' alt='GalaxyProject' width="200" /></a></div>

 **[July 2015 Galaxy Release (v 15.07)](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html)**
   [Galaxy Release v 15.07](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html) includes enhancements to Interactive Environments and the Workflow Editor, an update of the Policies for Committers and Pull Requests and [much more](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html).<br /><br />


 **Mid 2015 Galaxy CloudMan Release**

<div class='right'><a href='/CloudMan.md'><img src='/Images/Logos/CloudManWideBlackLogo.png' alt='CloudMan' width="150" /></a></div>

<div class='indent'>
   This is the most complete update of the Galaxy on the Cloud platform to date with all the cloud components being built from the ground up. See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG) for a complete set of changes or just give it a try: see the all new [Getting Started](/CloudMan/GettingStarted) documentation. 
</div>

 **Security Advisories**

   Three security vulnerabilities were announced and patched in August.  The Galaxy Committers have released a number of fixes which have been rolled in to affected versions of Galaxy dating back to the 14.10 release.

<div class='indent'><div class='indent'>
* **[Galaxy XSS Reflection Vulnerability](https://lists.galaxyproject.org/pipermail/galaxy-announce/2015-August/000148.html)**
* **[Galaxy LDAP Authentication Vulnerability](https://lists.galaxyproject.org/pipermail/galaxy-announce/2015-August/000147.html)**
* **[Tool Shed Unauthorized Upload Vulnerability](https://lists.galaxyproject.org/pipermail/galaxy-announce/2015-August/000149.html)**
</div></div>


 **Pulsar**
   [Pulsar 0.5.0](https://pypi.python.org/pypi/pulsar-app/0.5.0) was released in May.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.
 
 **CloudMan **
   The [most recent edition of CloudMan](/GalaxyUpdates/2014_09.md#august-2014-cloudman-release) was released in August.

 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and !BioCloudCentral.


----


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Teach.md'><img src='/Images/GalaxyLogos/GTNLogo300.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Logs.md'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Deployments.md'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong>Share your <a href='/Teach/Resources.md#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers.md#add-a-trainer'>experience</a> <em>now</em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Logs.md#add-a-log-page'>Share your experience</a> <em><a href='/Community/Logs.md#add-a-log-page'>now</a></em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Deployments.md#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/Community/Deployments.md#add-your-galaxy-deployment'>now</a></em> <strong> </td>
  </tr>
</table>

<br />

2 new **[Galaxy Training Network members](/Teach/Trainers)** were added last month:

* **[Noor Biotechnologies Ltd.](/Teach/Trainers.md#noor-biotechnologies-ltd)**
    [Noor Biotechnologies](http://noorbiotech.com/) has a presence in Malaysia and Saudi Arabia, and is available for training worldwide.

* **[University of Minnesota Minnesota Supercomputing Institute Research Informatics Solutions](/Teach/Trainers.md#university-of-minnesota-minnesota-supercomputing-institute-research-informatics-solutions)**
    [RIS](https://www.msi.umn.edu/about/staff/riss) is a group of experienced bioinformatics experts within the Supercomputing Institute at the University of Minnesota.

And 2 new **[Community Log Board](/Community/Logs)** entries were added as well.  Both are about wrapping Tools for Galaxy:

* **[Galaxy Tool Generating Dataset Collections](/Community/Log/2015/ToolGeneratingDatasets)**
    [Steve Cassidy](http://web.science.mq.edu.au/~cassidy/wordpress/about-steve-cassidy/) documented how create dataset collections in Galaxy tool definitions. (And [Alveo](http://alveo.edu.au/) is also a cool, non-life science application of Galaxy.)

* **[How Galaxy resolves dependencies (or not)](/Community/Log/2015/HowGalaxyResolveDependenciesOrNot)**
    [Peter van Heusden](https://github.com/pvanheus/) documents tool dependencies that specify how to source the actual packages that implement a tool‚Äôs commands.


----
<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

Ran out of time to compile these.  Look for a double batch in December.

----

# Other News

* Have a [look at a fresh chart of Main's usage](http://pic.twitter.com/G9RxTeXc0x) showing the improvements in Galaxy reports by [Daniel Bouchard](https://twitter.com/D_J_Bouchard).

* From [Bj√∂rn Gr√ºning](/BjoernGruening):
  * üíï this community! DESeq2 update requested by @mikelove: the [Galaxy wrapper is available](https://github.com/galaxyproject/tools-iuc/pull/294) before BioC version

* From [John Chilton](/JohnChilton):
  * [Versioned Tap Hackathon Proposal](https://github.com/Homebrew/homebrew-science/issues/2876)
  * [Galaxy visualizations](https://github.com/galaxyproject/galaxy/pull/378#event-427974576) from [Eteri Sokhoyan](https://github.com/eteriSokhoyan) and powered by [Cytoscape.js](http://js.cytoscape.org/) have been merged.

* [Martin ƒåech](/Marten):
  * [A list of tools presented at Genome Informatics 2015](https://github.com/galaxyproject/tools-iuc/issues/361) that might be of interest to the Galaxy community.

* From [Oleg Shpynov](https://github.com/olegs)
  * Published [our first application for the Galaxy Project](https://github.com/JetBrains-Research/galaxy) 

* From [Dannon Baker](/DannonBaker):
  * [A proposal for bug and issue handling, and roadmap management](https://github.com/galaxyproject/galaxy/pull/902)

* [Det'Rprok workflow](http://bit.ly/1aWNeVE) from [this paper](http://www.sciencedirect.com/science/article/pii/S1046202313002041) updated at !MyExperiment.

* We decided not to have a [Data Carpentry](http://www.datacarpentry.org/) / [Software Carpentry](http://software-carpentry.org/) *Train the Trainer* event before [GCC2016](/Events/GCC2016).  However, we have [nominated a topic on Best Practices and Resources for Galaxy Training](/Events/GCC2016/Training.md#galactically-speaking-best-practices-and-resources-for-galaxy-training).  If that interests you then *[please vote](http://bit.ly/gcc2016vote) for it*.

## Galaxy Codebase Reaches 20 000 Commits

<a href='https://github.com/galaxyproject/galaxy'><img src='/Images/20000Commits.png' alt='Galaxy codebase' width="100%" /></a>

In September [Galaxy turned 10 years old](/GalaxyUpdates/2015_10.md#galaxy-turns-10) by one important measure.  Another milestone was reached in October when the [Galaxy codebase](https://github.com/galaxyproject/galaxy) surpassed of 20000 commits. And appropriately (and if I can count), the 20,000th was a pull request commit from community member [Eric Rasche](/EricRasche) originally submitted by team member [Nate Coraor](/nate).

Our thanks go to the community and the team alike.
