---
title: October 2014 Galaxy Update
---
<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the October 2014 *[Galaxy Update](/galaxy-updates/)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Galaxy Needs Your Input!

The Galaxy Project is preparing for our next grant cycle and we are seeking your feedback and comments on on all things Galaxy.  There are two questionnaires, each with a different focus, based on how you interact with Galaxy:

1. [Galaxy User Questionnaire](https://docs.google.com/forms/d/1NT3ma2g7rR-mWA_vgOg2Ia_6VTFiZ86um2JhX2SecCk/formResponse)
1. [Galaxy Admin, Tool Developer, and Galaxy Developer Questionnaire](https://docs.google.com/forms/d/1lszXJ2lDJ2dYCWKXBk1qT1jO7XdDDfxXmGRCUMZFX2U/viewform)

Please take a few minutes and fill out whichever surveys apply to you.  The questionnaires are structured so you can skip topics that don't apply to you, and every question is optional.

*And, to thank you for your time and effort, the Galaxy Project will increase your storage quota on [usegalaxy.org](https://usegalaxy.org) by 50GB, a 20% increase.*

Let your voice be heard!

# IRC Channel Policy Change

<div class='left'><img src="/images/icons/ChatBalloons.png" alt="Galaxy IRC Channel" width="60" /></div>

A proposed change for the #galaxyproject IRC channel was [proposed](/news/irc-public-log-proposal/), and then [discussed, and approved on Galaxy Biostar](http://bit.ly/gxybs8804).  Starting sometime in October, posts to this channel will be made available in a searchable archive on the web.

Thanks to everyone who participated in the decision, and [those at GCC2014 who suggested this](/events/gcc2014/bofs/galaxy-admins/).


# Events

<div class='left'><br /><a href='/events/eccb2014/'><img src="/images/logos/ECCB2014Logo.png" alt="Galaxy at ECCB'14 / JOBIM" height="75" /></a></div>

## Galaxy at ECCB'14

Galaxy had a strong presence at [ECCB'14](/events/eccb2014/).  Galaxy was featured in workshops, demos and posters.  Couldn't make it?  No worries, as [most slides and posters are now available online](/events/eccb2014/).


## Other Events

<div class='center'>
<a href='/events/sg2014/'><img src="/images/logos/SG2014Logo400.png" alt="2014 Swiss German Galaxy Tour" height="90" /></a>
<a href='http://www.qfab.org/events/upcoming/'><img src="/images/logos/QFABLogo.png" alt="QFAB Workshops" height="80" /></a>
<a href='http://wiki.data.dtls.nl/index.php/DTL_Programmers_Meeting'><img src="/images/logos/DTLLogo150.png" alt="iReport - A tool to create flexible, scalable Galaxy tool interface, presented at the DTL Programmers Meeting" height="90" /></a>
<a href='http://galaxy4bioinformatics.sb-roscoff.fr/'><img src="/images/logos/SB-RoscoffBig.JPG" alt="Des bonnes pratiques d'intégration d'outils sous Galaxy" height="60" /></a> &nbsp;
<a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=62'><img src="/images/logos/GenoToulLogo.png" alt="Galaxy Training Days" height="60" /></a> &nbsp;
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div>

There are [upcoming events](/events/) in Switzerland, Germany, Australia, Norway, France, Italy, and the United States.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table class="table">
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th rowspan=3> September 30 - October 2 </th>
    <td> <a href='/events/switzerland2014/trainingday/'>Galaxy Training and Demo Day</a> </td>
    <td rowspan=3> <strong><a href='/events/sg2014/'>2014 Swiss-German Galaxy Tour</a></strong> with events in <strong>Bern, Switzerland</strong> and <strong>Freiburg, Germany</strong> </td>
    <td rowspan=3> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <td> <em><a href='/events/switzerland2014/'>(second Swiss) Galaxy Workshop</a></em> </td>
  </tr>
  <tr>
    <td> <em><a href='/events/germany2014/'>German Galaxy Developers Day</a></em> </td>
  </tr>
  <tr>
    <th> October 1 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-rna-seq-analysis-using-galaxy-brisbane-01-october-brisbane/'>Workshop: RNA-Seq analysis using Galaxy</a></em> </td>
    <td> The University of Queensland, Brisbane, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> October 6 - 24 </th>
    <td> <em><a href='http://wiki.uio.no/projects/clsi/index.php/INF-BIOX121_H14'>Intensive course in High Throughput Sequencing technologies and bioinformatics analysis</a></em> </td>
    <td> University of Oslo, Oslo, Norway </td>
    <td> <a href="mailto:lex.nederbragt AT ibv.uio.no">Lex Nederbragt</a>, <a href="mailto:karin.lagesen AT medisin.uio.no">Karin Lagesen</a> </td>
  </tr>
  <tr>
    <th> October 10 </th>
    <td> <em>iReport - A tool to create flexible, scalable Galaxy tool interface</em> </td>
    <td> <a href='http://wiki.data.dtls.nl/index.php/DTL_Programmers_Meeting'>DTL Programmers Meeting</a>, <a href='http://www.surffoundation.nl/'>SURF Foundation</a>, <a href='http://bit.ly/1r1L5VF'>Utrecht</a>, The Netherlands </td>
    <td> Saskia Hiltemann </td>
  </tr>
  <tr>
    <th> October 29 - November 4 </th>
    <td> <a href='http://meetings.cshl.edu/courses/2014/c-ecg14.shtml'>Computational & Comparative Genomics Course</a> <div class='indent'>Application Deadline: July 15</div> </td>
    <td> <a href='http://www.cshl.edu/'>Cold Spring Harbor Laboratories (CSHL)</a>, New York, United States </td>
    <td> <a href='http://faculty.virginia.edu/wrpearson/'>William Pearson</a>, <a href='http://mcb.illinois.edu/faculty/profile/ljstubbs'>Lisa Stubbs</a> </td>
  </tr>
  <tr>
    <th> November 3-5 </th>
    <td> <em><a href='http://galaxy4bioinformatics.sb-roscoff.fr/'>Des bonnes pratiques d'intégration d'outils sous Galaxy</a></em> </td>
    <td> <a href='http://www.sb-roscoff.fr/'>Station Biologique de Roscoff</a>, France </td>
    <td> <a href="mailto:christophe.caron@sb-roscoff.fr">Christophe Caron</a> </td>
  </tr>
  <tr>
    <th> November 3-6 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=62'>Galaxy training days</a></em> </td>
    <td> <a href='http://bioinfo.genotoul.fr/index.php?id=4#c93'>INRA de Toulouse Midi-Pyrénées</a>, France </td>
    <td> <a href='http://bioinfo.genotoul.fr/index.php?id=74'>GenoToul Bioinformatics Team</a> </td>
  </tr>
  <tr>
    <th> November 18-20 </th>
    <td> <em><a href='http://www.bioinformatica.crs4.it/training'>Analisi dati Next Generation Sequencing con Galaxy</a></em> </td>
    <td> Cagliari, Italy </td>
    <td> <a href="mailto:ngs-courses@crs4.it">CRS4</a> </td>
  </tr>
  <tr>
    <th> November 19-20 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-1920-november-brisbane/'>Workshop: Extended RNA-Seq analysis</a></em> </td>
    <td> The University of Queensland, Brisbane, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 26-28 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-et-ChIP-seq-sequencage-haut-debit--a-l-aide-d-outils-orientes-vers-un-public-de-biologistes_a39.html'>RNA-Seq & ChIP-Seq analysis course using Galaxy</a></em> </td>
    <td> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <a href="mailto:navratil AT prabi DOT fr">Navratil V</a>., <a href="mailto:oger AT prabi DOT fr">Oger C.</a>, <a href="mailto:veber AT prabi DOT fr">Veber P.</a>, <a href="mailto:deschamps AT prabi DOT fr">Deschamps C.</a>, <a href="mailto:perriere AT prabi DOT fr">Perriere G.</a> </td>
  </tr>
  <tr>
    <th colspan=4> 2015 </th>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> <strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a></strong> </td>
    <td> <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>


# New Papers

The [Galaxy CiteULike library](http://bit.ly/gxyCiteULike) of publications reached 1800 papers this month.  *71 papers (a new record)* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in September, including:

* [Executing SADI services in Galaxy](http://bit.ly/1CB7snv), by Aranguren, *et al.* *Journal of Biomedical Semantics*, Vol. 5, No. 1. (2014), 42, doi:10.1186/2041-1480-5-42

* [A Survey of Cloud-Based Service Computing Solutions for Mammalian Genomics](http://bit.ly/1vpSz3i), by Church & Goscinski, *IEEE Transactions on Services Computing,* DOI: 10.1109/TSC.2014.2353645

* [An automated infrastructure to support high-throughput bioinformatics](http://bit.ly/1ryfGJB), by Cuccuru, *et al.* High Performance Computing &amp; Simulation (HPCS), 2014 International Conference on (July 2014), pp. 600-607, doi:10.1109/hpcsim.2014.6903742

* [Experiences building Globus Genomics: a next-generation sequencing analysis service using Galaxy, Globus, and Amazon Web Services](http://bit.ly/1qNSS6N), by Madduri, *et al.* *Concurrency and Computation: Practice and Experience,* Special issue on XSEDE13, Volume 26, Issue 13, pages 2266–2279, 10 September 2014

* [MIRPIPE – quantification of microRNAs in niche model organisms](http://bit.ly/1vmnSvP), by Kuenne, *et al.* *Bioinformatics* (2014) doi: 10.1093/bioinformatics/btu573

* [ballaxy: web services for structural bioinformatics](http://bit.ly/1tu5CTI), by Hildebrandt, *et al.* *Bioinformatics* (2014) doi: 10.1093/bioinformatics/btu574

The new papers were tagged in many different areas:

<table class="table">
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a> </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 22 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 31 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a> </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [CDD Ingénieur NGS](http://bit.ly/YGIDHy) - Institut Curie, Paris, France  
* [Emploi CDD Ingénieur Bioinformatique - ChIP-seq](http://bit.ly/1paQDYu), Marseille, France
* [Research Specialist](http://bit.ly/1vbagWb), Michigan State University, United States
* [Bioinformatics and Computational Biology](http://bit.ly/1uysJJH), [US Army Engineer Research and Development Center’s Environmental Laboratory](http://el.erdc.usace.army.mil/index.cfm), Vicksburg, MS, United States
* [Computational Science Developer I](http://bit.ly/1lLkAMR), Cold Spring Harbor Laboratory (CSHL), New York, United States
* [Statistical Genomics Postdoc](/news/statistical-genomics-post-doc-penn-state/) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

# New Public Servers

Two new [public Galaxy server](/use/) was added to the published list in September:

## GalaxEast

<div class='right solid'><a href='http://www.galaxeast.fr'><img src="/use/galaxeast/galaxeast.png" alt="GalaxEast"  /></a></div>

[GalaxEast](http://www.galaxeast.fr) aims at providing a large range of bioinformatics tools for the analysis of various types of Omics data. It supports reproducible computational research by providing an environment for performing and recording bioinformatics analyses.

The [GalaxEast](http://www.galaxeast.fr) project has the following main objectives:

* Provide the academic scientific community with an open and powerful Galaxy instance with a guaranteed availability. The platform offers access to cutting-edge and up-to-date tools for Omics data analysis with help and support.
* Propose innovative developments and new helpful tools packaged for Galaxy (available in the GalaxEast toolshed)
* Promote the packaging of new developments for Galaxy (through wrappers and/or toolshed packages).

[GalaxEast](http://www.galaxeast.fr) is supported by [IGBMC](http://www.igbmc.fr/), [CNRS](http://www.cnrs.fr/), [Inserm](http://www.inserm.fr/), and [Université Strasbourg](http://www.unistra.fr/).

See *[GalaxEast: an open and powerful Galaxy instance for integrative Omics data analysis](https://depot.galaxyproject.org/hub/attachments/documents/posters/2014ECCB_GalaxEast.pdf),* poster presented at [ECCB'14](/events/eccb2014/) by Stephanie Le Gras, *et al.* for more.

## MIRPIPE

<div class='right solid'><a href='https://bioinformatics.mpi-bn.mpg.de/'><img src="/use/archive/mpi-hlr-bioinformatics-server/mirpipe-home-page.png" alt="MIRPIPE - the miRNA identification pipeline" width="300" /></a></div>

[MIRPIPE](https://bioinformatics.mpi-bn.mpg.de/) focuses on quantification of microRNA based on smallRNA sequencing reads.  From the home page:
 In opposition to present algorithms that generally rely on genomic data to identify miRNAs, MIRPIPE focuses on niche model organisms that lack such information. Among the MIRPIPE features are automatic trimming and adapter removal of raw RNA-Seq reads originating from various sequencing instruments, clustering of isomiRs, and quantification of detected miRNAs by homology search versus public or user uploaded reference databases.

See "[MIRPIPE – quantification of microRNAs in niche model organisms](http://bit.ly/1t4oQvo)," C. Kuenne, *et al.* for more.  [Email support](https://bioinformatics.mpi-bn.mpg.de/help.html) and a [MIRPIPE Manual](https://bioinformatics.mpi-bn.mpg.de/static/mirpipe_manual.pdf) are provided.  MIRPIPE is sponsored by the [Max Planck Institute for Heart and Lung Research](http://www.mpi-hlr.de/).

# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a></div>
<div class='right'><img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style="text-align: center;"> &nbsp;&nbsp; <a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a> </em> &nbsp;&nbsp;<br /> </td>
  </tr>
</table>

<br /><br />

<div class='left'><br />
<a href='http://environmentalomics.org/bio-linux-galaxy/'><img src="/images/logos/EOS.gif" alt="Overview of Galaxy on Bio-Linux 8" width="100" /></a>
</div>

The deployment details for the [GalaxEast public server](/use/galaxeast/) were posted in September.  Tracey Timms-Wilson's (of the [NERC Environmental 'Omics Synthesis Centre](http://environmentalomics.org/)) [Overview of Galaxy on Bio-Linux 8](http://environmentalomics.org/bio-linux-galaxy/) page was also added to the [Community Log Board](/news/).

The *Deployment Catalog* and *[Community Log Board](/news/)* *[Galaxy community hubs](/community/)* were launched in 2013.  If you have a Galaxy deployment, or [experience](/news/) you want to share then please publish them **this month.**

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# New Releases

New versions of [Galaxy](/galaxy-updates/2014-09/#august-11-2014-galaxy-distribution), [CloudMan](/galaxy-updates/2014-09/#august-2014-cloudman-release), [BioBlend](/galaxy-updates/2014-09/#bioblend-051-release), and [blend4j](/galaxy-updates/2014-09/#blend4j-011-release) were all released in August.  And so was [Galaxy IPython](/galaxy-updates/2014-09/#galaxy-ipython) too.

Look for a new Galaxy distribution in October.

# ToolShed Contributions

## Galaxy Project ToolShed Repos

Here are new contributions for the past *two* months.

In no particular order:

### Tools

* *From [saket-choudhary](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/)*
    * [sift_web](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/sift_web): PROVEAN and SIFT predictions for a list of human genome variants.

* *From [gbcs-embl-heidelberg](https://toolshed.g2.bx.psu.edu/view/gbcs-embl-heidelberg/)*
    * [jemultiplexer](https://toolshed.g2.bx.psu.edu/view/gbcs-embl-heidelberg/jemultiplexer): debarcoding/demultiplexing tool for FASTQ files accommodating all complex multiplexing protocols (iCLIP, molecule barcoding, ...).

* *From [mcharles](https://toolshed.g2.bx.psu.edu/view/mcharles/)*
    * [rapsodyn](https://toolshed.g2.bx.psu.edu/view/mcharles/rapsodyn): tools and workflow used for the [Rapsodyn Project](http://www.rapsodyn.fr/)
    * [rapsosnp](https://toolshed.g2.bx.psu.edu/view/mcharles/rapsosnp): workflow to detect and select SNP for the rapsodyn project

* *From [ayllon](https://toolshed.g2.bx.psu.edu/view/ayllon/)*
    * [tcoffee](https://toolshed.g2.bx.psu.edu/view/ayllon/tcoffee): T-Coffee multiple alignment suite.

* *From [anton](https://toolshed.g2.bx.psu.edu/view/anton/)*
    * [bamtools](https://toolshed.g2.bx.psu.edu/view/anton/bamtools): A collection of tools for manipulation of bamfiles
    * [vcfflatten](https://toolshed.g2.bx.psu.edu/view/anton/vcfflatten): Removes multi-allelic sites by picking the most common alternate

* *From [crs4](https://toolshed.g2.bx.psu.edu/view/crs4/)*
    * [kggseq_variant_selection](https://toolshed.g2.bx.psu.edu/view/crs4/kggseq_variant_selection): Variant selection with KGGSeq

* *From [tyty](https://toolshed.g2.bx.psu.edu/view/tyty/)*
    * [structurefold](https://toolshed.g2.bx.psu.edu/view/tyty/structurefold): StructureFold predicts RNA secondary structures from high throughput RNA structure profiling data

* *From [big-tiandm](https://toolshed.g2.bx.psu.edu/view/big-tiandm/)*
    * [sirna_plant](https://toolshed.g2.bx.psu.edu/view/big-tiandm/sirna_plant): plant siRNA analysis toolkits. siRNA prediction, siRNA annotation, siRNA quantify

* *From [geert-vandeweyer](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/)*
    * [dc_genotyper](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/dc_genotyper): genotyper aimed at finding SNPs in high-ploidy (or pooled) samples sequenced at very high depth in a targeted region.

* *From [galaxyp](https://toolshed.g2.bx.psu.edu/view/galaxyp/)*
    * [fasta_merge_files_and_filter_unique_sequences](https://toolshed.g2.bx.psu.edu/view/galaxyp/fasta_merge_files_and_filter_unique_sequences): Merge FASTA files, keeping only unique sequences
    * [filter_by_fasta_ids](https://toolshed.g2.bx.psu.edu/view/galaxyp/filter_by_fasta_ids): Extract sequences from a FASTA file based on a list of IDs
    * [myrimatch](https://toolshed.g2.bx.psu.edu/view/galaxyp/myrimatch): protein identification via database search using Bumbershoot MyriMatch
    * [ltq_iquant_cli](https://toolshed.g2.bx.psu.edu/view/galaxyp/ltq_iquant_cli): iQuant performs tag based isobaric quantification
    * [idpqonvert](https://toolshed.g2.bx.psu.edu/view/galaxyp/idpqonvert): Bumbershoot idpQonvert, a part of Bumbershoot IDPicker.
    * [directag_and_tagrecon](https://toolshed.g2.bx.psu.edu/view/galaxyp/directag_and_tagrecon): protein identification via Directag and TagRecon.

### Suites

* *From [biomonika](https://toolshed.g2.bx.psu.edu/view/biomonika/)*
    * [suite_linkyx_bundle_main_0_1](https://toolshed.g2.bx.psu.edu/view/biomonika/suite_linkyx_bundle_main_0_1):

* *From [anton](https://toolshed.g2.bx.psu.edu/view/anton/)*
    * [suite_vcflib_tools_3_0](https://toolshed.g2.bx.psu.edu/view/anton/suite_vcflib_tools_3_0): 23 tools for manipulation of VCF datasets

### Packages / Tool Dependency Definitions

* *From [fubar](https://toolshed.g2.bx.psu.edu/view/fubar/)*
    * [package_graphicsmagick_1_3_20](https://toolshed.g2.bx.psu.edu/view/fubar/package_graphicsmagick_1_3_20):

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc/)*
    * [package_graphicsmagick_1_3_20](https://toolshed.g2.bx.psu.edu/view/iuc/package_graphicsmagick_1_3_20):
    * [package_bioperl_1_6](https://toolshed.g2.bx.psu.edu/view/iuc/package_bioperl_1_6): downloads and compiles version 1.6 of Bioperl.
    * [package_onto_perl_1_41](https://toolshed.g2.bx.psu.edu/view/iuc/package_onto_perl_1_41): downloads and compiles version 1.41 of the Ontology toolkit written in perl.

* *From [saket-choudhary](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/)*
    * [package_requests_2_2_1](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_requests_2_2_1): Tool dependency definition of python-requests
    * [package_beautifulsoup4_4_1_0](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_beautifulsoup4_4_1_0): Tool dependency definition for python-bs4

* *From [qfab](https://toolshed.g2.bx.psu.edu/view/qfab/)*
    * [opal2_4_1](https://toolshed.g2.bx.psu.edu/view/qfab/opal2_4_1): Opal Package - GVL

* *From [galaxyp](https://toolshed.g2.bx.psu.edu/view/galaxyp/)*
    * [package_peptides_to_gff_0_1](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_peptides_to_gff_0_1): Installs the peptides_to_gff python script
    * [package_myrimatch](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_myrimatch):
    * [package_directag](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_directag):
    * [package_idpqonvert](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_idpqonvert):
    * [package_ltq_iquant_cli](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_ltq_iquant_cli):
    * [package_mgf_formatter](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_mgf_formatter):
    * [package_tagrecon](https://toolshed.g2.bx.psu.edu/view/galaxyp/package_tagrecon):

* *From [anton](https://toolshed.g2.bx.psu.edu/view/anton/)*
    * [package_vcflib_8a5602bf07](https://toolshed.g2.bx.psu.edu/view/anton/package_vcflib_8a5602bf07): Compiled vcflib binaries for x86_64

* *From [geert-vandeweyer](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/)*
    * [package_igvtools_2_3_32](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/package_igvtools_2_3_32): igvtools binaries, to be used as dependency in other tools.

* *From [lparsons](https://toolshed.g2.bx.psu.edu/view/lparsons/)*
    * [package_rseqc_2_4](https://toolshed.g2.bx.psu.edu/view/lparsons/package_rseqc_2_4): downloads and compiles version 2.4 of RSeQC.

### Updates

* *From [fubar](https://toolshed.g2.bx.psu.edu/view/fubar/)*
    * [toolfactory](https://toolshed.g2.bx.psu.edu/view/fubar/toolfactory): Citations added (thanks John!) and a few more output formats for Alistair Chilcott

* *From [nilesh](https://toolshed.g2.bx.psu.edu/view/nilesh/)*
    * [rseqc](https://toolshed.g2.bx.psu.edu/view/nilesh/rseqc): Upgraded RSeQC to [version 2.4](http://rseqc.sourceforge.net/#release-history)

# Other News

* [Why the three biggest positive contributions to reproducible research are the iPython Notebook, knitr, and Galaxy](http://bit.ly/1rqHVJW) on the [Simply Statistics blog](http://simplystatistics.org/)
* Updated [wiki page about dynamically discovering output datasets at runtime](/admin/tools/multiple-output-files/#number-of-output-datasets-cannot-be-determined-until-tool-run).
* The [Ansible playbook used to update usegalaxy.org](https://github.com/galaxyproject/usegalaxy-playbook) is available in GitHub.
* New [GVL Galaxy Release](https://genome.edu.au/wiki/GVL): Metagenomics Tutorial tools, MACS2, BLAST, MEME, hg38, rn6, and Trinity.
* Galaxy Community UK launches a Twitter channel: [@GalaxyUKFriends](https://twitter.com/GalaxyUKFriends)
* [BOSC 2015 will be in Dublin with ISMB/ECCB 2015](http://news.open-bio.org/news/2014/09/bosc-2015-will-be-in-dublin/).  *We should have voted more often!*
* [Supporting Enhanced Reproducibility for Platforms like Galaxy](https://github.com/Homebrew/homebrew-science/issues/1191), discussion on [GitHub](https://github.com/Homebrew/homebrew-science/issues/1191).
