---
title: April 2017 Galaxy News
---
<div class='right'><a href='/src/galaxy-updates/index.md'><img src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the April 2017 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter then please send it to outreach@galaxyproject.org.


# Usegalaxy.org Passes 100,000 Registered Users

<div class='right'><a href='/src/galaxy-project/statistics/index.md#galaxy-servers'><img src="/src/galaxy-updates/2017-04/100-k-users.jpg" alt="Usegalaxy.org hits 100,000 users" height="60" /></a></div>

[Usegalaxy.org](https://usegalaxy.org), the Galaxy Project's free public server, has surpassed 100,000 registered users. We started with 7 registered users in June 2006, and [have been climbing ever since](/src/galaxy-project/statistics/index.md#galaxy-servers).

Thanks for using Galaxy, all 100,000 of you.



# 2017 Galaxy Community Conference

<div class="right"><a href="https://gcc2017.sciencesconf.org/"><img src="/src/images/logos/gcc2017-logo-300.png' alt='2017 Galaxy Community Conference (GCC2017)' width="220" /></div>

## Registration & abstract submission are open

[Early registration](https://gcc2017.sciencesconf.org/page/registration) and [abstract submission](https://gcc2017.sciencesconf.org/page/submit) for the **[2017 Galaxy Community Conference (GCC2017)](https://gcc2017.sciencesconf.org/)** are open.

**Note: The submission deadline for oral presentations is 15 April.**

GCC2017 will be in Montpellier, France, 26-30 June and will feature two days of presentations, discussions, poster sessions, lightning talks, computer demos, keynotes, and birds-of-a-feather meetups, **all about data-intensive biology and the tools and infrastructure that support it.** GCC2017 also includes data and coding hacks, and **[two days of training](https://gcc2017.sched.com/overview/type/Training)** covering 16 different topics.
 
GCC2017 will be held at [Le Corum Conference Centre](http://www.montpellier-events.com/en/The-Corum/Presentation) in the heart of [Montpellier](https://gcc2017.sciencesconf.org/page/location), just 10km from the Mediterranean. This event will gather several hundred researchers addressing diverse questions and facing common challenges in data intensive life science research. GCC participants work across the tree of life, come from around the world, and work at universities, research organizations, industry, medical schools and research hospitals.

**[Early registration](https://gcc2017.sciencesconf.org/page/registration)** is also open and starts at less than 55€ / day for post-docs and students.  You can also [book low cost conference housing](https://gcc2017.sciencesconf.org/page/lodging) when you register.


## GCC2017 Travel Fellowships

[GCC2017](https://gcc2017.sciencesconf.org/) is a great opportunity to discuss your research with others that are working in and facing similar challenges in data-intenisve life science research.  *But only if you can get there.*  The Galaxy Community Fund is offering [fellowships](/src/news/2017-03-gcc2017-fellowships/) to early career researchers that would be coming from afar to attend GCC2017. 

See the [application](http://bit.ly/gcc2017apply) for full details on what's covered and what we ask you to provide.

**We'll review applications starting on April 5, 2017. Interested?  [Submit an abstract](https://gcc2017.sciencesconf.org/page/submit) and [apply for a fellowship](http://bit.ly/gcc2017apply).**

We hope to see you at [GCC2017](https://gcc2017.sciencesconf.org/)!

Galaxy Community Fund Board


## New GCC2017 Sponsors

We are pleased to welcome [ELIXIR](http://www.elixir-europe.org/) and GigaScience as returning GCC sponsors.

<div class='right'> 
<a href='http://www.elixir-europe.org/'><img src="/src/images/logos/ElixirNoTextLogo.png" alt="Elixer" height="120" /></a>
</div>

ELIXIR is a Gold level sponsor of the conference *and* a Giga sponsor of the GCC2017 Hackathons.  

[ELIXIR](http://www.elixir-europe.org/) unites Europe’s leading life science organisations in managing and safeguarding the increasing volume of data being generated by publicly funded research. It coordinates, integrates and sustains bioinformatics resources across its member states and enables users in academia and industry to access services that are vital for their research. [More...](http://www.elixir-europe.org/about-us)

<div class='left'><a href='https://academic.oup.com/gigascience'><img src="/src/images/logos/GigaScienceLogo.gif" alt="GigaScience" width="120" /></a>
</div>

*[GigaScience](https://academic.oup.com/gigascience)* is a GCC Sponsor for the 5th year in a row. *GigaScience* is an online open access, open data, open peer-review journal. Our focus covers ‘big data’ research from the life and biomedical sciences, including the growing range of work that uses difficult-to-access large-scale data, such as imaging, neuroscience, ecology, cohort, systems biology, and other new types of sharable data.

*GigaScience* is also offering 15% off the article-processing charge for papers submitted and published in *GigaScience* from the conference


----

# G-OnRamp Beta Testers Workshop: Invitation

<div class='right'><a href='https://wustl.edu/'><img src="/src/images/logos/WashULogoWide.jpg" alt="Washington University in St. Louis" width="170" /></a><br /><br />
<a href="http://www.ohsu.edu/"><img src="/src/images/logos/OHSU_Logo_300.png" alt='Oregon Health &amp; Science University' width="170" /></a>
</div>

A collaboration between the [Genomics Education Partnership (GEP)](http://gep.wustl.edu) and the [Galaxy project](https://galaxyproject.org/), the [G-OnRamp Project](http://gonramp.org) aims to enable educators and researchers to create genome browsers for collaborative annotations of eukaryotic genomes. G-OnRamp uses Galaxy workflows to construct evidence tracks (e.g., protein sequence similarity, gene predictions, RNA-Seq, repeats) and display the results on the [UCSC Genome Browser](https://genome.ucsc.edu/) or [JBrowse](http://jbrowse.org/). Educators can use G-OnRamp for hands-on learning in data-intensive biology; researchers can use G-OnRamp for best-practice annotation of novel genomes.

*We will hold two beta testers workshops this summer on June 20-22 and July 25-27, 2017 at [Washington University in St. Louis](https://wustl.edu/)* that will demonstrate how you can use G-OnRamp to create genome browsers for your favorite genomes. To receive future announcements regarding registration for these G-OnRamp workshops, please [register your interest](http://gonramp.org/signup).  Travel and local costs are supported by NIH BD2K grant 1R25GM119157.  Questions can be directed to [Jeremy Goecks](mailto:goecksj@ohsu.edu) (Galaxy), or [Sarah Elgin](mailto:selgin@wustl.edu) (GEP).


# All upcoming events

There are [a plenitude of Galaxy related events coming up](/src/events/index.md) in the next few months:

<table class="table table-striped"><thead>
<tr><th>Date</th><th>Topic/Event</th><th>Venue/Location</th><th>Contact</th>
</tr></thead><tbody>
<tr>
  <td><span class="text-nowrap">April 7th 2017</span></td>
  <td><a href="https://www.biogenouest.org/RNA-seq%2526microarray#Traitement%20de%20données%20de%20séquences%20par%20Galaxy%20-%201%20jour">Traitement de données de séquences par Galaxy</a></td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/>Rennes, France</td>
  <td></td>
</tr>
<tr>
  <td><span class="text-nowrap">April 19th 2017</span></td>
  <td><a href="http://www.canceropole-idf.fr/formation-ngs-dnaseq/">NGS &amp; Cancer : Analyses DNA-Seq</a> [<a href="/src/events/2017-04-cancerpole">hub page</a>]</td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/><a href="http://www.canceropole-idf.fr/">Cancerpole, Paris, Fance</a></td>
  <td>formateurs</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 15th 2017</span></td>
  <td><a href="https://biotexcel.com/event/ngs-2017-glasgow/#workshop">Data Analysis Workshop</a> [<a href="/src/events/2017-05-glasgow-ngs">hub page</a>]</td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/><a href="https://biotexcel.com/event/ngs-2017-glasgow/">NGS 2017 Glasgow, Glasgow, United Kingdom</a></td>
  <td>Julian Augley, Graham Hamilton, David Meltzer and Mani Mudaliar</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 16th 2017</span></td>
  <td><a href="http://www.france-bioinformatique.fr/en/evenements/G4I">Formation Galaxy4Imaging</a></td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/>Nantes, France</td>
  <td>Comité d'organisation</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 22nd 2017</span></td>
  <td><a href="https://docs.google.com/document/d/1dLCL5-2pkWTvmGTz7GyQkb5LEIZ3FBkPvCBoqUPGUOg/edit">ELIXIR/GOBLET/GTN hackathon for Galaxy training material re-use</a></td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/><a href="http://www.cam.ac.uk/">University of Cambridge, United Kingdom</a></td>
  <td>Frederik Coppens</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2017</span></td>
  <td><a href="http://www.france-bioinformatique.fr/fr/evenements/W4E2017">Workflow4Experimenters International Course</a> [<a href="/src/events/2017-05-w4e">hub page</a>]</td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/><a href="http://www.france-bioinformatique.fr/fr">Pasteur Institute, Paris, France</a></td>
  <td><a href="/src/teach/gtn/index.md"><img style="float:right;" alt="Training offered by GTN Member" src="/src/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christophe Caron</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2017</span></td>
  <td><a href="https://bioinformatics.ca/workshops/2017/bioinformatics-cancer-genomics-2017">Bioinformatics for Cancer Genomics</a> [<a href="/src/events/2017-05-cancer-genomics">hub page</a>]</td>
  <td><img style="float:right;" src="/src/images/icons/NA.png"/>Toronto, Ontario, Canada</td>
  <td>organizers</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 4th 2017</span></td>
  <td><a href="http://www.asms.org/conferences/annual-conference/annual-conference-homepage">Metaproteomics For The Future: Democratizing functional analysis of microbiomes via community-based informatics development and dissemination</a></td>
  <td><img style="float:right;" src="/src/images/icons/NA.png"/><a href="http://www.asms.org/conferences/annual-conference/annual-conference-homepage">ASMS 2017, Indianapolis, Indiana, United States</a></td>
  <td>Pratik Jagtap</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 6th 2017</span></td>
  <td><a href="https://lists.galaxyproject.org/archives/list/galaxy-france@lists.galaxyproject.org/thread/PHXFO7BB724GSZ5TXE7WNH543OUYFKGY/#PHXFO7BB724GSZ5TXE7WNH543OUYFKGY">Formation détection et annotation des éléments transposables dans les génomes eucaryote</a></td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/><a href="https://urgi.versailles.inra.fr/">INRA, Versailles, France</a></td>
  <td>Nathalie Choisne, Olivier Inizan</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 20th 2017</span></td>
  <td><a href="/src/news/2017-03-g-on-ramp/index.md">G-OnRamp Beta Testers Workshop</a></td>
  <td><img style="float:right;" src="/src/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/src/teach/gtn/index.md"><img style="float:right;" alt="Training offered by GTN Member" src="/src/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 26th 2017</span></td>
  <td><a href="https://gcc2017.sciencesconf.org/">2017 Galaxy Community Conference (GCC2017)</a> [<a href="/src/events/2017-gcc">hub page</a>]</td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/>Montpellier, Fance</td>
  <td><a href="/src/teach/gtn/index.md"><img style="float:right;" alt="Training offered by GTN Member" src="/src/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Galaxy community</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 2nd 2017</span></td>
  <td><a href="http://www.smbe2017.org/symposia/">Computation and reproducibility in molecular evolution</a></td>
  <td><img style="float:right;" src="/src/images/icons/NA.png"/><a href="http://www.smbe2017.org/">SMBE 2017, Austin, Texas, United States</a></td>
  <td><a href="/src/teach/gtn/index.md"><img style="float:right;" alt="Training offered by GTN Member" src="/src/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Anton Nekrutenko, Sergei Kosakovsky Pond</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 25th 2017</span></td>
  <td><a href="/src/news/2017-03-g-on-ramp/index.md">G-OnRamp Beta Testers Workshop</a></td>
  <td><img style="float:right;" src="/src/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/src/teach/gtn/index.md"><img style="float:right;" alt="Training offered by GTN Member" src="/src/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">September 1st 2017</span></td>
  <td><a href="http://www.earlham.ac.uk/genome-10k-and-genome-science">The Galaxy Bioinformatics Platform: learning to use Galaxy for RNA-seq analysis</a> [<a href="/src/events/2017-genome-10k">hub page</a>]</td>
  <td><img style="float:right;" src="/src/images/icons/EU.png"/><a href="http://www.earlham.ac.uk/genome-10k-and-genome-science">Genome 10K and Genome Science, Earlham Institute, Norwich, United Kingdom</a></td>
  <td><a href="/src/teach/gtn/index.md"><img style="float:right;" alt="Training offered by GTN Member" src="/src/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a></td>
</tr></tbody></table>



See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

# New Publications

**275 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in March. This includes papers that were added as part of the 2016 year end review of alerts. (Often a paper isn't added when an alert first comes out because the DOI isn't working yet, or because the paper is embargoed.) 

Some highlights from the papers added in March:

* [High-resolution TADs reveal DNA sequences underlying genome organization in flies](https://doi.org/10.1101/115063)
   Fidel Ramirez, Vivek Bhardwaj, Jose Villaveces, et al. *bioRxiv* (08 March 2017), 115063, doi:10.1101/115063

* [Enhancing Knowledge Discovery from Cancer Genomics Data with Galaxy](https://doi.org/10.1093/gigascience/gix015)
   Marco A. Albuquerque, Bruno M. Grande, Elie J. Ritch, et al. *GigaScience* (09 March 2017), doi:10.1093/gigascience/gix015

* [Tools for cluster analysis of data from genome-wide association studies](http://urn.nb.no/URN:NBN:no-56327)
  Johanne H. Horn (2016)

* [Secure Genomic Data Processing on the Cloud using TrustStore](https://www.researchgate.net/profile/Sehrish_Kanwal5/publication/286439667_Secure_Genomic_Data_Processing_on_the_Cloud_using_TrustStore/links/5668f88108ae193b5fa13d0d.pdf)
  C. Wise, C. Friedrich, S. Nepal, S. Kanwal, R. Sinnott. In 21st International Congress on Modelling and Simulation (MODSIM2015) (December 2015)

* [Utilisation de Docker en bioinformatique dans le cloud de IFB](https://jobim2016.sciencesconf.org/conference/jobim2016/pages/book_jobim2016.pdf#page=394)
  Sandrine Perrin, Bryan Branco, Jonathan Lorenzo, et al. In Journées Ouvertes en Biologie, Informatique et Mathématiques (JOBIM) (2016)

* [I'll take that to go: Big data bags and minimal identifiers for exchange of large, complex datasets](https://static.aminer.org/pdf/fa/bigdata2016/BigD418.pdf)
  K. Chard, M. D'Arcy, B. Heavner, et al. In 2016 IEEE International Conference on Big Data (Big Data) (Dec 2016), pp. 319-328, doi:10.1109/BigData.2016.7840618

* [Functional and Evolutionary Genomics in Aphids](http://www.crcnetbase.com/doi/abs/10.1201/b19967-4)
  Denis Tagu, Federica Calevro, Stefano Colella, Toni Gabaldón, Akiko Sugio. In Biology and Ecology of Aphids (2016), pp. 52-88

* [Enhancing Access to Media Collections and Archives Using Computational Linguistic Tools](http://ceur-ws.org/Vol-1786/pustejovsky.pdf)
   James Pustejovsky, Marc Verhagen, Nancy Ide, Keith Suderman. In Corpora in the Digital Humanities (CDH), Bloomington, Indiana (2017)

* [DNApod: DNA polymorphism annotation database from next-generation sequence read archives](https://doi.org/10.1371/journal.pone.0172269)
  Takako Mochizuki, Yasuhiro Tanizawa, Takatomo Fujisawa, et al. *PLOS ONE*, Vol. 12, No. 2. (24 February 2017), e0172269, doi:10.1371/journal.pone.0172269

* [Deep and Surface Causality: Global Teaching and Access to HPC Social Science](https://pdfs.semanticscholar.org/b056/c9c5deda9ce754a3d4bf9b45fc46e40ddc9e.pdf)
   Douglas W. Uci, Paul Rodriguez, Eric Blau, et al. (2015)

* [Blood-based omic profiling supports female susceptibility to tobacco smoke-induced cardiovascular diseases](https://doi.org/10.1038/srep42870)
   Aristotelis Chatziioannou, Panagiotis Georgiadis, Dennie G. Hebels, et al. *Scientific Reports*, Vol. 7 (22 February 2017), 42870, doi:10.1038/srep42870

* [Architectural models for deploying and running virtual laboratories in the cloud](https://doi.org/10.1109/MIPRO.2016.7522153)
   E. Afgan, A. Lonie, J. Taylor, K. Skala, N. Goonasekera. In 39th International Convention on Information and Communication Technology, Electronics and Microelectronics (MIPRO) (May 2016), pp. 282-286, doi:10.1109/MIPRO.2016.7522153


## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 120 | [methods](http://www.citeulike.org/group/16008/tag/methods) | | 69 | [workbench](http://www.citeulike.org/group/16008/tag/workbench) | | 37 | [usepublic](http://www.citeulike.org/group/16008/tag/usepublic) | | 27 | [usemain](http://www.citeulike.org/group/16008/tag/usemain) | 
| 23 | [tools](http://www.citeulike.org/group/16008/tag/tools) | | 21 | [uselocal](http://www.citeulike.org/group/16008/tag/uselocal) | | 20 | [refpublic](http://www.citeulike.org/group/16008/tag/refpublic) | | 19 | [other](http://www.citeulike.org/group/16008/tag/other) | 
| 15 | [reproducibility](http://www.citeulike.org/group/16008/tag/reproducibility) | | 14 | [cloud](http://www.citeulike.org/group/16008/tag/cloud) | | 11 | [isgalaxy](http://www.citeulike.org/group/16008/tag/isgalaxy) | | 4 | [unknown](http://www.citeulike.org/group/16008/tag/unknown) | 
| 3 | [visualization](http://www.citeulike.org/group/16008/tag/visualization) | | 3 | [shared](http://www.citeulike.org/group/16008/tag/shared) | | 1 | [howto](http://www.citeulike.org/group/16008/tag/howto) | | 1 | [project](http://www.citeulike.org/group/16008/tag/project) | 
  
----

# Who's Hiring

<div class='right'><br /><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Automated deployment of Galaxy environments using Ansible](http://artbio.fr/?page_id=78), Institut de Biologie Paris Seine, France
* [BioInformaticien](https://talent.limagrain.com/Pages/Offre/detailoffre.aspx?idOffre=1407&idOrigine=2673&LCID=1036), HM.Clause, Business Unit du Groupe Limagrain,  Maine et Loire, La Bohalle, France
* [Scientific Research Programmer](http://goo.gl/cMZVCn), [Sethuraman Lab](http://arunsethuraman.weebly.com/), [California State University San Marcos](https://www.csusm.edu/). Develop model-based population genomics pipelines.
* [Development Scientist (Bioinformatics)](http://chk.tbe.taleo.net/chk05/ats/careers/requisition.jsp?org=NEB&cws=1&rid=272), New England Biolabs, Ipswich, Massachusetts, United States.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />


----

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them in January.

## Want to Mine Scientific Literature? LAPPS Grid Needs You

<div class='right'><a href='/src/news/2017-03-lapps-grid/index.md'><img src="/src/use/lapps-grid/lapps-grid-logo.png" alt="LAPPS Grid" width="120" /></a>
</div>

*The [The Language Applications (LAPPS) Grid](http://www.lappsgrid.org) team intends to seek funding for a project that would create customizable NLP applications that can be used to mine scientific literature,* in response to requests from scientists in several disciplines who want to extract entities, relations, networks, and ontologies from scientific publications and identify articles in the scientific literature that have treated particular topics or entities.

Interested in participating? See the [full call for details](/src/news/2017-03-lapps-grid/index.md).

[LAPPS Grid](http://www.lappsgrid.org) provides an infrastructure for rapid development of natural language processing applications (NLP) that uses the Galaxy platform as its workflow engine. The LAPPS Grid has integrated a wide range of NLP tools and resources into Galaxy and provided for using them interoperably in a “plug-and-play” environment. 

## PhenoMeNal Online Training Material

<div class='right'><a href='http://www.ebi.ac.uk/training/online/course/phenomenal-accessing-metabolomics-workflows-galaxy'><img src="/src/use/phenomenal/phenomenal.png" alt="PhenoMeNal" width="120" /></a>
</div>

New [online training](http://www.ebi.ac.uk/training/online/course/phenomenal-accessing-metabolomics-workflows-galaxy) for the [PhenoMeNal Galaxy server](http://public.phenomenal-h2020.eu/) is available from EMBL-EBI.  "PhenoMeNal (Phenome and Metabolome aNalysis) is a standardised e-infrastructure that supports the data processing and analysis pipelines for molecular phenotype data generated by metabolomics applications.  This course will give you an overview of PhenoMeNal, how to create your cloud research environment, and how to access Galaxy workflows for metabolomics data."

## New Public Galaxy Servers

### Galaxy@GenOuest

<div class='right solid'><a href='https://galaxy.genouest.org'><img src="/src/use/galaxy-genouest/galaxy-genouest.png" alt="Galaxy@GenOuest" /></a></div>

* *Links:*
  * [Galaxy@GenOuest](https://galaxy.genouest.org)
  * [Request a GenOuest account](https://my.genouest.org)
* *Domain/Purpose:*
  * A general purpose Galaxy server
  * Includes tools developped by [Dyliss](http://www.irisa.fr/dyliss/) and [GenScale](https://team.inria.fr/genscale/) bioinformatics research teams in Rennes, France.
* *Comments:*
* *User Support:*
  * [Email](mailto:support AT genouest DOT org)
* *Quotas:*
  * You must be part of an academic research lab and create an account on [GenOuest](https://my.genouest.org).
  * The default quota for new users is 50GB.
* *Sponsor(s):*
  * [GenOuest](http://www.genouest.org/)
  * [IRISA](https://www.irisa.fr/)
  * [Inria](https://www.inria.fr/centre/rennes)


### BIPAA (BioInformatics Platform for Agroecosystem Arthropods)

<div class='right solid'><a href='https://bipaa-galaxy.genouest.org/'><img src="/src/use/bipaa/bipaa-logo.png" alt="BIPAA (BioInformatics Platform for Agroecosystem Arthropods)" width="120" /></a></div>

* *Links:*
  * [BIPAA Galaxy Server](https://bipaa-galaxy.genouest.org)
  * [BIPAA home page](https://bipaa.genouest.org/)
* *Domain/Purpose:*
  * Insect genomics (aphids, parasitoïd wasps, lepidopterans)
* *Comments:*
  * Includes tools for the analysis of insect genomes (annotation, transcriptomics, epigenomics, ...).
  * Linked to genome databases hosted by BIPAA: [AphidBase](http://bipaa.genouest.org/is/aphidbase/), [LepidoDB](http://bipaa.genouest.org/is/lepidodb/) and [ParWaspDB](http://bipaa.genouest.org/is/parwaspdb/)
  * Hosted on the GenOuest architecture (same computing resources).
* *User Support:*
  * [Account creation](http://bipaa.genouest.org/account/)
  * [Contact BIPAA](mailto:bipaa@inra.fr)
* *Quotas:*
  * Default quota is low (<10Go), but can be increased on request.
* *Sponsor(s):*
  * [INRA](http://www.inra.fr)
  * [GenOuest](http://www.genouest.org/)
  * [IRISA](https://www.irisa.fr/)
  * [Inria](https://www.inria.fr/centre/rennes)




<!--

# Galaxy Community Directories

<table class="table table-striped">
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/teach/index.md'><img src="/src/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/logs/index.md'><img src="/src/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/deployments/index.md'><img src="/src/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your <a href='/src/teach/resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/teach/trainers/index.md'>experience</a> <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/logs/index.md#add-a-log-page'>Share your experience</a> <em><a href='/src/community/logs/index.md#add-a-log-page'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/deployments/index.md#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/src/community/deployments/index.md#add-your-galaxy-deployment'>now</a></em> </strong> </td>
  </tr>
</table>

<br />

No new resources were added to community directories in January or February.

-->

----

# Releases

## The all-new CloudLaunch ##

Technically, the [all-new Galaxy CloudLaunch service](https://beta.launch.usegalaxy.org/) has been in public beta since February but keep in mind that it will replace the [current CloudLaunch service](https://launch.usegalaxy.org/launch) eventually so give it a try and let us know how it performs for you.

## Galaxy 17.01 available on the Jetstream cloud ##

A new image of Galaxy with Galaxy release 17.01 is available on the NSF-sponsored academic cloud [Jetstream](http://jetstream-cloud.org/). The updated image comes with an [updated list of tools](https://github.com/ARTbio/GalaxyKickStart/blob/master/extra-files/cloud_setup/usegalaxy_main_tool_list.yml) and all the reference genomes available on Galaxy Main. **Remember that the Jetstream cloud is free to use.** Instructions on how to get started are available [here](https://galaxyproject.org/cloud/jetstream/).

## CloudBridge 0.2.0

[CloudBridge](https://github.com/gvlproject/cloudbridge) aims to provide a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud. It is currently under development and is in an Alpha state.  [Release 0.2.0](https://github.com/gvlproject/cloudbridge/releases/tag/v0.2.0) includes several fixes and enhancements.


## Planemo 0.39 - 0.40

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/src/images/galaxy-logos/planemo-logo.png" alt="" width="200" /></a></div>

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).


## galaxy-lib 17.5.6 - 17.5.7

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).




## Earlier Releases

Other packages that have been released in the prior 4 months.

###  17.01 Galaxy Release

<div class='right'><a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.01 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.01_announce.html)** in February.


#### Highlights

Some highlights:

* **Conda auto initialization is enabled by default**
* **New interface for user preferences**
* **Support for compressed FASTQ formats**

#### Release Notes

For full details on all of the enhancements and fixes in this release, please see the [full release notes](https://docs.galaxyproject.org/en/master/releases/17.01_announce.html).


#### Galaxy Docker Image 17.01

And, thanks to [Björn Grüning](/src/people/bjoern-gruening/index.md), there is also now [a Docker image for Galaxy 17.01](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.01) as well.

### Starforge 0.3

<div class='right'>
<a href='https://github.com/galaxyproject/starforge/blob/master/README.rst'><img src="/src/images/galaxy-logos/starforge_logo_big.png" alt="Starforge" width="180" /></a>
</div>

[Starforge](https://pypi.python.org/pypi/starforge) is a collection of scripts that supports the building of components for Galaxy.  Specifically, with Starforge you can:

* Build [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) dependencies
* Build [Python Wheels](http://pythonwheels.com/) (e.g. for the [Galaxy Wheels Server](http://wheels.galaxyproject.org/))
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.

Documentation can be found at [starforge.readthedocs.org](http://starforge.readthedocs.org/). 

### Pulsar 0.7.4

<div class='right'><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/src/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" /></a></div>

A [Pulsar](https://pypi.python.org/pypi/pulsar-app/) update was released in February.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

[This release](https://github.com/galaxyproject/pulsar/blob/master/HISTORY.rst) contains Conda and flake8 updates.


### sequence_utils

Galaxy's [sequence utilities](https://github.com/galaxyproject/sequence_utils) are a set of Python modules for reading, analyzing, and converting sequence formats.



### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

[Tool Shed contributions from March](/src/toolshed/contributions/2017-03/index.md).


# Other News

* A new [in-depth RNAseq tutorial](http://bit.ly/rnaseq_tut) is available. 
* Greatly improved Cytoscape visualizations are [now available in Galaxy](https://github.com/galaxyproject/galaxy/pull/3817), thanks to [Anup Kumar](https://github.com/anuprulez).
* [EtherCalc integrated into Galaxy](https://github.com/galaxyproject/galaxy/pull/3699) as an interactive environment thanks to [Saskia Hiltemann](https://github.com/shiltemann).
* From [Rob Davey](https://twitter.com/froggleston)
  * Over 400000 jobs have been run on our Galaxy server at [Earlaham Iinstitute](http://www.earlham.ac.uk/) since 2015. Infrastructure win.
* [Getting started with Galaxy on the cloud](/src/cloudman/getting-started/index.md)
* New [step by step guide on howto upgrade a Galaxy Docker image](https://github.com/bgruening/docker-galaxy-stable#Upgrading-images) from [Rafa Hernández](https://github.com/fikipollo).
