---
title: September 2016 Galaxy News
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 24em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/galaxy-updates/'><img src="/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the September 2016 *[Galactic News](/galaxy-updates/)*, a summary of what is going on in the Galaxy community.

If you have anything to include in the next News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org).


# Events

## Galaxy Admin Training, November 7-11, Salt Lake City, Utah

<div class='left solid center'>
<a href='/events/admin-training2016/'><img src="/images/logos/AdminTraining2016-500.png" alt="Galaxy Project Admin Training" width="200" /></a><br />
November 7-11, 2016<br />University of Utah<br />Salt Lake City, Utah<br />United States<br /><br /> [Early registration ends Sept 19](/events/admin-training2016/registration/)
</div>

**Early registration for [Galaxy Admin Training 2016](/events/admin-training2016/) ends September 19.**

[Galaxy Admin Training 2016](/events/admin-training2016/) is a weeklong event offering basic and advanced sessions.  It is being held in Salt Lake City, Utah, November 7-11, 2016, the week before [Supercomputing 16 (SC16)](http://sc16.supercomputing.org/) meets in Salt Lake.  

**[Basics Session: November 7-8](/events/admin-training2016/basics-session/)**

This two day [basics session](/events/admin-training2016/basics-session/) will introduce participants to what you need to know to get a Galaxy server up and running on a standalone server. You'll also learn how to extend your Galaxy with your own tools and tools from the community, and how to define reference data in your server as well.

**[Advanced Session: November 9-11](/events/admin-training2016/advanced-session/)**

The three day [advanced session](/events/admin-training2016/advanced-session/) will build on topics covered in the [basics section](/events/admin-training2016/basics-session/).  This session both extends those topics, such as tool definition, and adds new ones, such as working with compute clusters and heterogeneous resources. The goal of the advanced workshop is to enable Galaxy administrators to create robust, high-performance Galaxy instances that take full advantage of available tools and compute and storage resources.

[Register by September 19](/events/admin-training2016/registration/) and save up to $130 off standard registration rates (and up to $250 if you work in industry).

<br />

<div class='center'>
<a href='http://www.chpc.utah.edu'><img src="/images/logos/UtahCHPCLogo.png" alt="University of Utah Center for High Performance Computing (CHPC)" height="90" /></a> &nbsp;
<a href='http://healthsciences.utah.edu/'><img src="/images/logos/UUtahHealthSciences.png" alt="University of Utah" height="90" /></a> <br />
<a href='http://medicine.utah.edu/dbmi/'><img src="/images/logos/UUtahDBMI.png" alt="University of Utah Department of Biomedical Informatics" width="220" /></a>
<a href='http://medicine.utah.edu/ccts/bmic/'><img src="/images/logos/UUtahCCTS_BMIC.png" alt="University of Utah Clinical & Translational Science Biomedical Informatics Core (CCTS BMIC)" width="220" /></a><br />
<a href='http://ucgd.genetics.utah.edu/'><img src="/images/logos/USTAR-UCGD-Logo.jpg" alt="USTAR Center for Genetic Discovery (UCGD)" height="90" /></a>
</div>

Galaxy Admin Training 2016 is hosted by the [University of Utah Center for High Performance Computing (CHPC)](http://www.chpc.utah.edu/), the [USTAR Center for Genetic Discovery (UCGD)](http://ucgd.genetics.utah.edu/), the  [Department of Biomedical Informatics](http://medicine.utah.edu/dbmi/) and the [Clinical & Translational Science Biomedical Informatics Core (CCTS BMIC)](http://medicine.utah.edu/ccts/bmic/) at the [University of Utah](http://utah.edu).

We hope to see you in Salt Lake!


<div class='right'>
<a href='/events/sg2016/'><img src="/images/logos/SG2016T.V2_logo.png" alt="2016 Swiss German Galaxy Tour" height="120" /></a></div>

## Swiss-German Galaxy Days

The 2016 editions of [Swiss-German Galaxy Days](/events/sg2016/) will be held 20-21 October in Freiburg (Germany).

Registration is free, but space is limited on both days. We recommend you register soon to secure your spot for one or both days of the SG2016Tour: **[Register Now](https://docs.google.com/forms/d/1mRO4uNXQd8jL1I9JNiQ4xd0fnrZKlQPQSORgJCpDeqg)**


## Other upcoming events

There are [a plentitude of Galaxy related events coming up](/events/). Here's what's coming up over the next few months:

<div class='center'>
<a href='http://sites.unica.it/metabolomicaclinica/events/scientific-school-2016/'><img src="/images/logos/UCagliariLogoTrans.png" alt="Metabolomics Sardinian Scientific School: How to bridge metabolomics and genomics" height="80" /></a>
<a href='http://biosb.nl/education/course-portfolio/rna-seq/'><img src="/images/logos/BioSBResearchSchool.png" alt="NGS Course: RNA-seq data analysis" height="80" /></a>
<a href='http://www.jsbi.org/iibmp2016/program_bof/#B10'><img src="/images/logos/IIBMP2016Logo.png" alt="GalaxyによるNGSデータ解析 (Galaxy NGS Data Analysis BoF)" height="80" /></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<a href='/events/admin-training2016/'><img src="/images/logos/AdminTraining2016-500.png" alt="Galaxy Project Admin Training" width="120" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> September 12-16 </th>
    <td> <em><a href='http://sites.unica.it/metabolomicaclinica/events/scientific-school-2016/'>The Workflow4Metabolomics (W4M) online infrastructure for omics analysis with Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://sites.unica.it/metabolomicaclinica/events/scientific-school-2016/'>Metabolomics Sardinian Scientific School: How to bridge metabolomics and genomics</a>, Pula, Italy </td>
    <td> E Thévenot </td>
  </tr>
  <tr>
    <th> September 12-16 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/fr/evenements/summer_school_metagenomics'>Summer School 2016 in Metagenomics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Institut Pasteur, Paris, France </td>
    <td> <a href='http://www.france-bioinformatique.fr/fr/evenements/summer_school_metagenomics'>Contacts</a> </td>
  </tr>
  <tr>
    <th> September 19-20 </th>
    <td> <em><a href='http://bioinformaticstraining.pythonanywhere.com/course/10/'>Exome analysis using Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Milano, Italy </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a>  <a href='http://bioinformaticstraining.pythonanywhere.com/trainers/paolo_uva'>Paolo Uva</a>, <a href='http://bioinformaticstraining.pythonanywhere.com/trainers/gianmauro_cuccuru'>Gianmauro Cuccuru</a> </td>
  </tr>
  <tr>
    <th> September 19-22 </th>
    <td> <em>Streamlined analysis of duplex sequencing data with Du Novo</em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='https://coursesandconferences.wellcomegenomecampus.org/events/item.aspx?e=598'>Genome Informatics 2016</a>, Hinxton, United Kingdom </td>
    <td> <a href='http://nstoler.com/'>Nick Stoler</a> </td>
  </tr>
  <tr>
    <th> September 19-23 </th>
    <td> <em><a href='https://drive.google.com/file/d/0Bw3CPLmTKRT8V1p0QlU5QnYyS28/view'>High Throughput Sequencing (HTS) Data Analyses for Scientists</a></em> <div class='indent red'>This workshop is full</div></td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Freiburg, Germany </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:galaxy@informatik.uni-freiburg.de">Contact</a> </td>
  </tr>
  <tr>
    <th> September 26-28 </th>
    <td> <em><a href='http://biosb.nl/education/course-portfolio/rna-seq/'>NGS Course: RNA-seq data analysis</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Leiden University Medical Center, The Netherlands </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> Peter-Bram ‘t Hoen, Jan Oosting, Celia van Gelder </td>
  </tr>
  <tr>
    <th> September 29 - October 1 </th>
    <td> <em><a href='http://www.jsbi.org/iibmp2016/program_bof/#B10'>GalaxyによるNGSデータ解析 (Galaxy NGS Data Analysis BoF)</a></em> </td>
    <td> <img src='/images/icons/AS.png' alt='Asia' align='right' /> <a href='http://www.jsbi.org/iibmp2016/'>第5回生命医薬情報学連合大会 (IIBMP 2016)</a>, Tokyo, Japan </td>
    <td> Ryota Yamanaka, Tazra Ohta </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th rowspan=2> October 10 </th>
    <td> <em><a href='https://drive.google.com/file/d/0BwK61gB7NIcic1BnVmhxMFJEM3M/view'>Omics Data Services Platform: Supporting bacterial pathogen multi omics analysis</a></em> </td>
    <td rowspan=2> <img src='/images/icons/AU.png' alt='Australia' align='right' />  <a href='https://nectar.org.au/workshop-science-gateways/'>International Workshop on Science Gateways</a>, Melbourne, Australia </td>
    <td> Andrew Lonie </td>
  </tr>
  <tr>
    <td> <em><a href='https://drive.google.com/file/d/0BwK61gB7NIcia1EtUTBkaUgxU28/view'>Microbial Genomics Virtual Laboratory: Supporting bacterial pathogen multi omics analysis</a></em> </td>
    <td> Simon Gladman </td>
  </tr>
  <tr>
    <th> October 10-13 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=83'>Metagenomic: amplicon analysis</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> INRA Auzeville, Auzeville, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> Sarah Maman </td>
  </tr>
  <tr>
    <th> October 18-20 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/en/evenements/galaxy4bioinformatics'>Galaxy4Bioinformatics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Lyon, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='http://www.france-bioinformatique.fr/en/node/17'>Contacts</a> </td>
  </tr>
  <tr>
    <th> October 20 </th>
    <td> <em><a href='/events/switzerland2016/'>(third Swiss) Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='/events/sg2016/'>2016 Swiss-German Galaxy Tour</a> in Freiburg, Germany </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <th> October 21 </th>
    <td> <em><a href='/events/germany2016/'>(second German) Galaxy Developers Day</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='/events/sg2016/'>2016 Swiss-German Galaxy Tour</a> in Freiburg, Germany </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> November 7-8 </th>
    <td> <strong><a href='/events/admin-training2016/basics-session/'>Galaxy Admin Training: Introduction</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Salt Lake City, Utah, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/galaxy-team/'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <th> November 7-9 </th>
    <td> <em><a href='http://www.canceropole-idf.fr/formation-ngs-chipseq'>NGS & Cancer : Analyses ChIP-Seq</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' />  Paris, France </td>
    <td> <a href="mailto:communication@canceropole-idf.fr">Cancéropôle Île-de-France</a> </td>
  </tr>
  <tr>
    <th> November 8-20 </th>
    <td> <em><a href='https://meetings.cshl.edu/courses.aspx?course=c-seqtec&year=16'>Advanced Sequencing Technologies & Applications</a></em> <div class='indent'>Application deadline is July 15</div> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='https://cshl.edu/'>Cold Spring Harbor Laboratory</a>, New York, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/jeremy-goecks/'>Jeremy Goecks</a> </td>
  </tr>
  <tr>
    <th> November 9-11 </th>
    <td> <strong><a href='/events/admin-training2016/advanced-session/'>Galaxy Admin Training: Advanced</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Salt Lake City, Utah, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/galaxy-team/'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <th> November 15-16 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de données RNA-seq sous l’environnement Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> PRABI, Lyon, France </td>
    <td> <a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Contact</a> </td>
  </tr>
  <tr>
    <th> November 20-25 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/evenements/EBA2016'>Ecole Bioinformatique AVIESAN</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.sb-roscoff.fr/'>Station Biologique, Roscoff</a>, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:ecole-bioinfo@aviesan.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 21-23 </th>
    <td> <em><a href='https://urgi.versailles.inra.fr/content/view/full/3794'>Annotation of transposable elements</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> URGI, INRA Versailles, France </td>
    <td> <a href='https://urgi.versailles.inra.fr/content/view/full/3794'>Contact</a> </td>
  </tr>
  <tr>
    <th> November 28 - December 2 </th>
    <td> <em><a href='http://workflow4metabolomics.org/training/w4mcourse2016'>Traitement des donnéesmétabolomiques sous Galaxy</a></em> <div class='indent red'>Les pré-inscriptions sont ouvertes jusqu'au 15 juillet 2016</div> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' />  <a href='http://www.sb-roscoff.fr/'>Station Biologique, Roscoff</a>, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a>  <a href="mailto:contact@workflow4metabolomics.org">Contact</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" text-align: center; background-color: #eef;"> <span style="font-size: larger;"> <strong>2017</strong> </span> </td>
  </tr>
  <tr>
    <th> February 3-9 </th>
    <td> <strong><a href='http://www.embl-abr.org.au/GAMe2017'>Galaxy Australasia Meeting (GAMe 2017)</a></strong> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> University of Melbourne, Victoria, Australia </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://www.embl-abr.org.au/organisers/'>Organisers</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 26-30 </th>
    <td> <strong><a href='https://gcc2017.sciencesconf.org/'>2017 Galaxy Community Conference (GCC2017)</a></strong> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Montpellier, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:gcc2017_organization@groupes.france-bioinformatique.fr">Organisers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='https://training.galaxyproject.org/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

<div class='right'>
<br /><a href='http://f1000research.com/channels/galaxy'><img src="/images/news-graphics/F1000GalaxyChannelThumb.png" alt="F1000Research Galaxy Channel" width="120" /></a>
</div>

# GCC2016 Talks, Posters and Training Slides are on the F1000Research Galaxy Channel

Slides for keynote, accepted and lightning talks, and training sessions at GCC2016 are now available on the  [F1000Research Galaxy Channel](http://f1000research.com/channels/galaxy).   [Slides](http://f1000research.com/channels/galaxy?&selectedDomain=SLIDES) and [posters](http://f1000research.com/channels/galaxy?&selectedDomain=POSTERS) have all bee assigned DOIs.

If you present at Galaxy at an event, please consider [submitting your presentation to the Galaxy channel](http://f1000research.com/channels/galaxy/for-authors/publish-your-research).

# New Papers

**75 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in August.

Some highlights from last month:

* **[Omicron: a Galaxy for reproducible proteogenomics](http://etd.library.vanderbilt.edu/available/etd-08052016-114746/unrestricted/MatthewChambers.pdf)**
    Matt Chambers.

* **[CloudBridge: a Simple Cross-Cloud Python Library](https://s3.amazonaws.com/cloudbridgelib/CloudBridge.pdf)**
    Nuwan Goonasekera, Andrew Lonie, [James Taylor](/people/james-taylor/), [Enis Afgan](/people/enis-afgan/). XSEDE16.

* **[DISMISS: detection of stranded methylation in MeDIP-Seq data](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1158-7)**
    Umar Niazi, Kathrin K Geyer, Martin J Vickers, Karl F Hoffmann, Martin T Swain. *BMC Bioinformatics*, Vol. 17, No. 1. (2016), pp. 1-12, doi:10.1186/s12859-016-1158-7

* **[SEX-DETector: a probabilistic approach to study sex chromosomes in non-model organisms](https://doi.org/10.1093/gbe/evw172)**
    Aline Muyle, Jos Käfer, Niklaus Zemp, Sylvain Mousset, Franck Picard, Gabriel A. B. Marais. *Genome Biology and Evolution* (04 August 2016), evw172, doi:10.1093/gbe/evw172

* **[Galaxy Workflows for Web-based Bioinformatics Analysis of Aptamer High-throughput Sequencing Data](https://doi.org/10.1038/mtna.2016.54)**
    William H. Thiel. *Molecular Therapy — Nucleic Acids*, Vol. 5, No. 8. (01 August 2016), e345, doi:10.1038/mtna.2016.54

* **[RNAcommender: genome-wide recommendation of RNA-protein interactions](https://doi.org/10.1093/bioinformatics/btw517)**
    Gianluca Corrado, Toma Tebaldi, Fabrizio Costa, Paolo Frasconi, Andrea Passerini. *Bioinformatics* (08 August 2016), btw517, doi:10.1093/bioinformatics/btw517

* **[Tools and techniques for computational reproducibility](https://doi.org/10.1186/s13742-016-0135-4)**
    Stephen R. Piccolo, Michael B. Frampton. *GigaScience*, Vol. 5, No. 1. (11 July 2016), doi:10.1186/s13742-016-0135-4

* ** [Streamlined analysis of duplex sequencing data with Du Novo](http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1039-4)**
    Nicholas Stoler, Barbara Arbeithuber, Wilfried Guiblet, Kateryna D. Makova, [Anton Nekrutenko](/people/anton/). *Genome Biology*, 201617:180 DOI: 10.1186/s13059-016-1039-4

## Tagged Publications

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 38 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><br /><img src="/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Post­doctoral Research Position](https://drive.google.com/file/d/0B5_H88SQ1C_7Z3dJY3FzU0hQMUU/view?pref=2&pli=1), Multi­omic Bioinformatics, University of Minnesota
* [Postdoctoral Researcher: Forest Genomics Database and Software Developer](http://compgenomics.lab.uconn.edu/join/), University of Connecticut.  "... data sharing among partner databases through Galaxy modules." <div class='red'>Act fast.</span>  
* [Software developer and Post-docs](http://gehlenborg.com/open-positions/), Gehlenborg Lab, Harvard Medical School, Boston, Massachusetts, United States

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

----


# Public Galaxy Server News

There are [over 80 publicly accessible Galaxy servers](/use/) and [four semi-public Galaxy services](/use/).  Here's what happened with them in August.

## New Public Galaxy Servers

### ImmPort Galaxy

<div class='right solid'><a href='https://www.immportgalaxy.org/'><img src="/use/immport-galaxy/immport.png" alt="ImmPort Galaxy" width="300" /></a></div>
* *Links:*
  * [ImmPort Galaxy](https://www.immportgalaxy.org/)
* *Domain/Purpose:*
  * Flow Cytometry Analysis.
* *Comments:*
  * [ImmPort Galaxy](https://www.immportgalaxy.org/) hosts [ImmPort](http://www.immport.org/)'s flow analysis tools.
  * See [FlowGalaxy: Developing a workflow for Flow Cytometry Analysis in Galaxy](http://sched.co/745G), presented by [Cristel Thomas](http://twitter.com/crstlthms) at [GCC2016](https://gcc16.sched.org/).
  * [ImmPort on Twitter](http://twitter.com/ImmPort)

* *User Support:*
    * [Help page](https://www.immportgalaxy.org/static/immportgalaxyhelp.html)
    * [Videos](https://www.youtube.com/channel/UC8lQSx1Z7sMFm07wfqI-_kA)
    * [Email](mailto:immport-galaxy@immport.org)
    * Tutorials to come.
* *Quotas:*
    * Anyone can use this server. No quotas for now.
* *Sponsor(s):*
    * [ImmPort](http://immport.org/immport-open/public/home/home)
    * [NIAID](http://www.niaid.nih.gov/Pages/default.aspx)
    * [DAIT](https://www.niaid.nih.gov/about/organization/dait/pages/default.aspx)
    * [NIH](https://www.nih.gov/)
    * [HHS](http://www.hhs.gov/)

### Palfinder

<div class='right solid'><a href='https://palfinder.ls.manchester.ac.uk/'><img src="/use/palfinder/palfinder.png" alt="Galaxy Palfinder Service" /></a></div>

* *Links:*
    * [Galaxy Palfinder Service](https://palfinder.ls.manchester.ac.uk/)

* *Domain/Purpose*
    * Microsatellite development.
* *Comments:*
    * See [A Galaxy-based bioinformatics pipeline for optimised, streamlined microsatellite development from Illumina next-generation sequencing data](https://doi.org/10.1007/s12686-016-0570-7), Griffiths, S.M., Fox, G., Briggs, P.J. *et al. Conservation Genetics Resources* (2016). doi:10.1007/s12686-016-0570-7
* *User Support:*
    * [Galaxy Palfinder Service Manual](https://palfinder.ls.manchester.ac.uk/manual)
    * [Email](mailto:peter.briggs@manchester.ac.uk)
* *Quotas:*
    * Accounts have a quota of 20 GB.
    * Any data uploaded or generated are automatically deleted after 30 days (to prevent the build up of large volumes of data onto our servers long-term). Please ensure that you download all outputs that you wish to keep.
* *Sponsor(s):*
    * Palfinder Galaxy is a service is provided by the [Bioinformatics Core Facility](https://www.bmh.manchester.ac.uk/research/facilities/bioinformatics/), [Research Computing](http://www.rcs.manchester.ac.uk/), and [The Faculty of Biology, Medicine and Health](https://www.bmh.manchester.ac.uk/) at the [University of Manchester](https://www.manchester.ac.uk/) in conjunction with the [lab of Richard Preziosi](http://preziosilab.org/) at the University of Manchester.

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

<br />

No new entries were added in August, but a new virtual appliance was added to the VA directory:

* **RNAcommender**
    A tool for genome-wide recommendation of RNA-protein interactions.

----

# Releases

## Galaxy v16.07

<div class='right'><a href='http://getgalaxy.org'><img src="/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The [Galaxy Committers team](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) is pleased to announce the **[July 2016 (v16.07) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.07_announce.html)**.

<span style="font-size: larger;"> Highlights </span>

**Shift of Galaxy tool dependencies to Conda**

Galaxy admins obtained the ability to install Galaxy tool’s dependencies using the Conda package manager. This is a Beta feature and we encourage interested deployers to opt-in by modifying configuration. Documentation that explains this switch and answers FAQ is [available](https://docs.galaxyproject.org/en/master/admin/conda_faq.html).

**Dynamic tool destinations**

Our friends from Canada National Microbiology Laboratory enhanced Galaxy with feature that allows dynamic mapping of tools to destinations based on finer grained admin-specified rules. Please see the [wiki](/admin/config/jobs/#dynamic-destination-mapping). Implemented in [PR #2579](https://github.com/galaxyproject/galaxy/pull/2579).

**Galaxy chat**

Admins can now plug in the included communication server to enable users of their instance to use real-time chat within the Galaxy interface. Please see the [documentation](https://docs.galaxyproject.org/en/master/admin/chat.html) to learn how to activate and use this feature. Implemented in [PR #2515](https://github.com/galaxyproject/galaxy/pull/2515).

<span style="font-size: larger;"> [Github](https://github.com/galaxyproject/galaxy) </span>

**New Galaxy repository**

```
$ git clone -b release_16.07 https://github.com/galaxyproject/galaxy.git
```


**Update of existing Galaxy repository**

```
$ git checkout release_16.07 && git pull --ff-only origin release_16.07
```


See [our wiki](/develop/source-code/) for additional details regarding the source code locations.

<br />

See the [full release notes](https://docs.galaxyproject.org/en/master/releases/16.07_announce.html) for more.

### Galaxy Docker Image 16.07

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 16.07](https://github.com/bgruening/docker-galaxy-stable/releases/tag/16.07) as well.  



## CloudBridge 0.1.1

[CloudBridge](https://github.com/gvlproject/cloudbridge) provides a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud. The latest release adds support for advanced networking management and OpenStack Keystone v3 support (required to use NSF Jetstream cloud).

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/images/galaxy-logos/planemo-logo.png" alt="" width="200" /></a></div>

## Planemo 0.28.0 - 0.29.1

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  Planemo saw several releases in August. See [GitHub for details](https://github.com/galaxyproject/planemo/blob/0.29.1/HISTORY.rst).

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="150" /></a></div>

## Pulsar 0.7.1 - 0.7.2

[Pulsar](https://pypi.python.org/pypi/pulsar-app/) updates were released in August.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

## BioBlend 0.8.0

[BioBlend](http://bioblend.readthedocs.org/) is a Python library for interacting with CloudMan and Galaxy‘s API.  BioBlend makes it possible to script and automate the process of cloud infrastructure provisioning and scaling via CloudMan, and running of analyses via Galaxy.

See the [release notes](https://github.com/galaxyproject/bioblend/releases) for what's new in release 0.8.0.

## galaxy-lib 16.7.10 - 16.10.0

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

The August releases align the library with recent changes to Galaxy.


## Earlier Releases

### blend4php 0.1 alpha

<div class='right'><a href='https://github.com/galaxyproject/blend4php'><img src="/images/galaxy-logos/blend4phpLogo.png" alt="" width="220" /></a></div>

The alpha release of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html) was announced in June. It follows the lead of [BioBlend](https://bioblend.readthedocs.io/en/latest/) which provides a Python package for interacting with Galaxy and CloudMan--hence the use of 'blend' in the name of this package. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases. Integration of Tripal and Galaxy will allow the community research databases to provide next-generation analytical tools to their users using Galaxy. However, this library was created independently of Tripal to support integration of any PHP application with Galaxy.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/docs-v0.1a/html/index.html) page for full information.


### CloudMan 16.04

<div class='right'><a href='/cloudman/'><img src="/images/galaxy-logos/cloudman-logo.jpg" alt="CloudMan" width="300" /></a></div>

An update to Galaxy CloudMan on AWS was released in May. CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

This is a minor update release with the following changes:

* Galaxy 16.04 update
* Availability on Amazon's Ireland region
* A couple of bug fixes

See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a more complete set of changes.

### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

[Tool Shed contributions for August](/toolshed/contributions/2016-08/).

# Other News

* From [Greg Von Kuster](/people/greg-vonkuster/):
    [Galaxy tools for Constructive Solid Geometry (Galaxy CSG flavor)](https://github.com/gregvonkuster/galaxy-csg/blob/master/README.md)

* [How to Build Bioinformatic Pipelines Using Galaxy](http://www.the-scientist.com/?articles.view/articleNo/46670/title/How-to-Build-Bioinformatic-Pipelines-Using-Galaxy), and article by Jeffrey M. Perkel in *The Scientist*.
