---
title: It's a new year in the Galaxy!
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/galaxy-updates/'><img src="/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the January 2016 *[Galactic News](/galaxy-updates/)*, a summary of what is going on in the Galaxy community.

If you have anything to include in the next News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org).

# New Papers

**51** new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in December. Highlights include:

* [Sequence database versioning for command line and Galaxy bioinformatics servers](http://bit.ly/1QRmQEM)
    by Damion Dooley, Aaron Petkau, Gary Van Domselaar, William Hsiao, *Bioinformatics* (12 December 2015), btv724, doi:10.1093/bioinformatics/btv724

* [Dintor: functional annotation of genomic and proteomic data](http://bit.ly/1OzzTvY)
    by Christian X. Weichenberger, Hagen Blankenburg, Antonia Palermo, *et al.* *BMC Genomics*, Vol. 16, No. 1. (21 December 2015), doi:10.1186/s12864-015-2279-5

* [Globus Nexus: A Platform-as-a-Service provider of research identity, profile, and group management](http://bit.ly/1PesQHO)
    by Kyle Chard, Mattias Lidman, Brendan McCollam, *et al.* *Future Generation Computer Systems*, Vol. 56 (March 2016), pp. 571-583, doi:10.1016/j.future.2015.09.006

* [Enhanced reproducibility of SADI web service workflows with Galaxy and Docker](https://doi.org/10.1186/s13742-015-0092-3)
    by Mikel Egaña E. Aranguren, Mark D. Wilkinson *GigaScience*, Vol. 4, No. 1. (3 December 2015), doi:10.1186/s13742-015-0092-3

* [FuMa: reporting overlap in RNA-seq detected fusion genes](http://bit.ly/1Qyy6qT)
    by Youri Hoogstrate, René Böttcher, Saskia Hiltemann, Peter van der Spek, Guido Jenster, Andrew P. Stubbs, *Bioinformatics* (10 December 2015), btv721, doi:10.1093/bioinformatics/btv721

* [ADMIRE: analysis and visualization of differential methylation in genomic regions using the Infinium HumanMethylation450 Assay](https://doi.org/10.1186/s13072-015-0045-1)
    by Jens Preussner, Julia Bayer, Carsten Kuenne, Mario Looso *Epigenetics & chromatin*, Vol. 8, doi:10.1186/s13072-015-0045-1

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 23 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


----

# Events

## GCC2016 Website is online

<div class='right'>
<a href='https://web.archive.org/web/http://gcc2016.iu.edu/'><img src="https://web.archive.org/web/http://gcc2016.iu.edu/img/gcc2016-logo.png" alt="2016 Galaxy Community Conference"  /></a>
</div>

The **[2016 Galaxy Community Conference website is now online](http://gcc2016.iu.edu/)** and includes [registration information](http://gcc2016.iu.edu/registration/index.php) and the full [training schedule](https://gcc16.sched.org/grid-full#2016-06-26).

[GCC2016](http://gcc2016.iu.edu/) will be held June 25-29 at [Indiana University](http://indiana.edu/) in Bloomington, Indiana, United States. This will be the 7th annual gathering of the Galaxy community, and we are expecting over 200 participants again this year.  

[Registration information](http://gcc2016.iu.edu/registration/index.php) describes the different registration packages, rates, and deadlines.  Early registration rates start at less than $40 per day for students and post-docs.

The [training schedule](https://gcc16.sched.org/grid-full#2016-06-26) includes session titles, descriptions, prerequisites, and instructors. Many thanks to everyone who nominated and voted on topics, and especially to the many [instructors](https://gcc16.sched.org/directory/artists) who are contributing their time and effort to make GCC2016 a success.

<div class='right'>
<a href='https://pti.iu.edu/'><img src="https://web.archive.org/web/http://gcc2016.iu.edu/img/PTI_signature_small.jpg" alt="Indiana University Pervasive Technology Institute (PTI)" /></a>
</div>

Thanks also go to **the [Indiana University Pervasive Technology Institute (PTI)](https://pti.iu.edu/)**, who are supporting  GCC2016 as a **[Platinum Sponsor](https://web.archive.org/web/http://gcc2016.iu.edu/sponsors/index.php)**.  PTI is home to the [National Center for Genome Analysis Support](http://ncgas.org/) ([for which NSF funding was recently renewed](https://itnews.iu.edu/articles/2015/iu-genome-analysis-center-wins-national-science-foundation-funding-renewal.php)), and the [Galaxy Trinity instance](https://galaxy.ncgas-trinity.indiana.edu/root), and is a long time supporter of the Galaxy Community. PTI pairs fundamental academic computational research with the widely known strengths of Indiana University through innovations and service delivery in networking and high performance computing.  See the [PTI website](http://pti.iu.edu/) for more information.

### Seeking Sponsors

We continue to seek other sponsors as well and [offer a wide range of sponsorship plans](https://web.archive.org/web/http://gcc2016.iu.edu/sponsors/index.php). If your organization is interested in having a presence at GCC2016, please contact the GCC2016 Exec for more information.

### Registration will open this month

Yeah, we know, we said this last month.  But last month "this month" actually meant next month which is now this month, due to, ummm, failure to compensate for the [2015 leap second](https://en.wikipedia.org/wiki/Leap_second).  Yeah.

Anyway, watch Galaxy channels for registration opening *this month*, really.

## Metagenomics Tools and Workflows Codefest Reports

The [Intergalactic Utilities Commission](/iuc/) held it's [second codefest](https://github.com/galaxyproject/tools-iuc/issues/299) with the aim to improve metagenomic research in Galaxy.

The IUC was available for two days in a Hangout to answers questions and to work together with the Galaxy community to enhance our tools and the overall Galaxy ecosystem. 22 scientists from all over the world participated in this and made it a fantastic experience and a success story, which the IUC would like to repeat more often.  

There are two event reports, See the [one by Björn Grüning](https://gist.github.com/bgruening/734ff541e9d1b02f7b09), and [one by Yvan Le Bras](http://cesgo.genouest.org/groups/guggo/wiki/MEtagenomicsHackathon).

## Upcoming Events

There are [many upcoming events](/events/) in the next few months.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='center'>
<a href='/events/asu2016/'><img src="/images/logos/ASUBioKIC_600.png" alt="Biodiversity Knowledge Integration Center" height="60" /></a>&nbsp;
<a href='/events/pag2016/'><img src="/images/logos/PAGLogo300.png" alt="Galaxy @ Plant and Animal Genome (PAG 2016)" height="90" /></a> &nbsp;
<a href='http://www.festivalofgenomicslondon.com/an-introduction-to-clinical-bioinformatics/'><img src="/images/logos/FoG_London.jpg" alt="An introduction to clinical bioinformatics" height="70" /></a>
<a href='http://www.drylab.nl/teaching-and-trainings/trait-vumc-course-on-galaxy/'><img src="/images/logos/CTMM-TraITLogoSmall.png" alt="NGS Data Analysis in Galaxy (CTMM-TraIT course)" height="90" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> January 8 </th>
    <td> <em><a href='/events/asu2016/'>Introduction to Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://asu.edu/'>Arizona State University</a>, Tempe, Arizona, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://sols.asu.edu/people/nico-franz'>Nico Franz</a>, <a href='https://sols.asu.edu/people/melissa-wilson-sayres'>Melissa A. Wilson Sayres</a>, <a href='/people/anton/'>Anton Nekrutenko</a> </td>
  </tr>
  <tr>
    <th> January 8, 15 </th>
    <td> <em><a href='http://www.vib.be/en/training/research-training/courses/Pages/Introduction-to-NGS-data-analysis.aspx'>Introduction to NGS data analysis</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.vib.be/'>VIB</a>, Leuven, Belgium </td>
    <td> <a href='http://www.vib.be/whoiswho/Pages/Janick-Mathys.aspx?lang=en'>Janick Mathys</a> </td>
  </tr>
  <tr>
    <th> January 9-13 </th>
    <td> <strong><a href='/events/pag2016/'>Plant and Animal Genome (PAG XXIV)</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> San Diego, California, United States </td>
    <td> See <a href='/events/pag2016/'>presenters</a> </td>
  </tr>
  <tr>
    <th> January 19 </th>
    <td> <em><a href='http://www.festivalofgenomicslondon.com/an-introduction-to-clinical-bioinformatics/'>An introduction to clinical bioinformatics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.festivalofgenomicslondon.com/'>Festival of Genomics London</a>, London, United Kingdom </td>
    <td> <a href='http://www.festivalofgenomicslondon.com/speaker/ang-davies/'>Ang Davies</a> </td>
  </tr>
  <tr>
    <th> January 19 </th>
    <td> <em><a href='http://www.drylab.nl/teaching-and-trainings/trait-vumc-course-on-galaxy/'>NGS Data Analysis in Galaxy (CTMM-TraIT course)</a></em> </td>
    <td> VU University Medical Center Amsterdam, The Netherlands </td>
    <td> <a href='https://scholar.google.com/citations?user=BJ7bjCEAAAAJ&hl=en'>Christian Rausch</a>, Youri Hoogstrate, <a href='https://www.linkedin.com/in/ritaazevedo'>Rita Azevedo</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> February 11 </th>
    <td> <em><a href='http://bioteam.net/bioteam-appliance/galaxy-edition/'>BioTeam Galaxy Appliance: Simplify Hosting Your Own Galaxy (Software Demo)</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://www.agbt.org/meetings/agbt-general-meeting/'>AGBT</a>, Orlando, Florida, United States </td>
    <td> <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a></td>
  </tr>
  <tr>
    <th> February 15-19 </th>
    <td> <em><a href='https://www.supagro.fr/web/pages/?idl=19&page=1964&id_page=2666'>Analyse bioinformatique de séquences sous Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='https://www.supagro.fr/web/index.php'>Montpellier SupAgro</a>, Montpellier, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://www.supagro.fr/web/pages/contact.php?id=139'>Yolande Olivier</a>, <a href='https://www.supagro.fr/web/pages/contact.php?id=102'>Dominique Alhinc</a> </td>
  </tr>
  <tr>
    <th> February 20-23 </th>
    <td> <em><a href='https://conf.abrf.org/the-galaxy-platform'>(SW4) The Galaxy Platform for Multi-Omic Data Analysis and Informatics</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' />  <a href='https://conf.abrf.org/abrf-2016'>ABRF 2016</a>, Fort Lauderdale, Florida, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://cbs.umn.edu/contacts/pratik-d-jagtap'>Pratik Jagtap</a>, <a href='https://cbs.umn.edu/contacts/timothy-j-griffinhttps://cbs.umn.edu/contacts/timothy-j-griffin'>Tim Griffin</a>, <a href='/people/dave-clements/'>Dave Clements</a>, <a href='http://experts.umn.edu/en/persons/candace-r-guerrero(c3134cd2-fe98-4d90-b5de-77fce1921d29).html'>Candace Guerrero</a>, <a href='http://www-users.cs.umn.edu/~onsongo/'>Getiria Onsongo</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> March 7-8 </th>
    <td> <strong><a href='https://github.com/galaxyproject/tools-iuc/issues/422'>IUC Contribution Fest - RADSeq Tools and Workflows</a></strong> </td>
    <td> <img src='/images/icons/GL.png' alt='Around the World' align='right' /> <em>Online</em> </td>
    <td> <a href='/iuc/'> IUC </a></td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 25-29 </th>
    <td> <strong><em><a href='/events/gcc2016/'>2016 Galaxy Community Conference (GCC2016)</a></strong></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a>  <a href='https://web.archive.org/web/http://gcc2016.iu.edu/organizers/index.php'>Organizers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='https://training.galaxyproject.org/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Computational Biology post-doc](http://bit.ly/1J1WLla), IMI-eTRIKS consortium, CNRS-EISBM, Lyon France. Contribute "to the development of eTRIKS Galaxy tools and workflows for disease stratification and biomarker discovery from single and multiple omics datasets."
* [Post-doc in Functional Genomics Of Obesity-Related Diseases](http://bit.ly/1QFyDHK), Inserm, Lille, France
* [Bioinformatician Post-doc at SB Roscoff](https://lists.galaxyproject.org/archives/list/galaxy-france@lists.galaxyproject.org/message/IC6W3TYHDUB2BSI2SCJQPU2KK3MA7NQM/) working on parasitic dinoflagellate genomics
* [Software Engineer](/news/software-engineer-ohsu/), Oregon Health Sciences University, Portland, Oregon, United States
* [Software Developer / Bioinformatician](https://ig14.i-grasp.com/fe/tpl_embl01.asp?s=4A515F4E5A565B1A&jobid=54309,8734341283&key=64564906&c=342372129814&pagestamp=seqbdpnrnmbvdpxtgb), European Molecular Biology Laboratory (EMBL), Heidelberg, Germany
    *Extended until 10 January 2016.*
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

----

# New Public Galaxy Servers

[Two new publicly accessible Galaxy servers](/use/) were added in December.  And, while not a new server, the *server formerly known as MIRPIPE* is now the greatly expanded MPI-HLR Bioinformatics Server.

## Dintor

<div class='right solid'><a href='http://dintor.eurac.edu/'><img src="/use/dintor/dintor.png" alt="Dintor: Data Integrator Tool Suite"  /></a></div>

* *Link:*
    * [Dintor: Data Integrator Tool Suite](http://dintor.eurac.edu/)
* *Domain/Purpose:*
    * A "suite of tools that facilitate working with GWA and NGS data" and "offers modules for high level functional annotation of genes and gene products such as gene set prioritization, functional similarity of proteins, or clinical significance of variation data. Each of these tools has been designed to perform a basic task independently."
* *Comments:*
    * Includes "more than thirty modules ready for use by bioinformaticians and biologists working in genomics research."
    * "The Galaxy server has been set up to facilitate access to our **`Dintor`** tools by biologists with little background in bioinformatics. A second, expert mode of invocation is given by command line access to the tool suite, which can be downloaded"
    * [Admin and deployment documentation is available](http://dintor.eurac.edu/doc/doc/index.html).
    * [Dintor: functional annotation of genomic and proteomic data](http://bit.ly/1OzzTvY), Christian X. Weichenberger, Hagen Blankenburg, Antonia Palermo, Yuri D’Elia, Eva König, Erik Bernstein and Francisco S. Domingues, *BMC Genomics* 201516:1081 DOI: 10.1186/s12864-015-2279-5

* *User Support:*
    * [Tutorials](http://dintor.eurac.edu/static/tutorial.html)
    * In addition to help on each of the Galaxy Tool pages, there is [additional help for each Dintor tool here](http://dintor.eurac.edu/doc/doc/tools.html).
* *Quotas:*
    * Anyone can create a login. Anonymous use is also supported.
* *Sponsor(s):*
    * The [Center of Biomedicine (CBM)](http://www.eurac.edu/en/research/health/biomed/Pages) at [EURAC research](http://www.eurac.edu/).

## Erasmus MC

<div class='right solid'><a href='https://bioinf-galaxian.erasmusmc.nl/galaxy/'><img src="/use/erasmus-mc/erasmus-mc-bioinf-300.png" alt="Erasmus MC Bioinformatics Galaxy Server"  /></a></div>

* *Link:*
    * [Erasmus MC Bioinformatics Galaxy Server](https://bioinf-galaxian.erasmusmc.nl/galaxy/)
* *Domain/Purpose:*
    * General purpose genomics analysis, featuring many standard tools plus many additional tools.
* *Comments:*
    * See [FuMa: reporting overlap in RNA-seq detected fusion genes](http://bioinformatics.oxfordjournals.org/content/early/2015/12/09/bioinformatics.btv721.abstract), by Youri Hoogstrate, René Böttcher, Saskia Hiltemann, Peter van der Spek, Guido Jenster and Andrew P Stubbs, *Bioinformatics* (2015) doi: 10.1093/bioinformatics/btv721
* *User Support:*

    *
* *Quotas:*
    * "This Galaxy instance may be used without registration, but guests' histories are deleted nightly. To request an account, please [contact](mailto:a.stubbs@erasmusmc.nl) the Bioinformatics department."
* *Sponsor(s):*
    * [Erasmus Medical Center department of Bioinformatics](http://bioinformatics.erasmusmc.nl/)

----


# Releases

## Galaxy 15.10

<div class='right'><a href='http://getgalaxy.org'><img src="/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The **[October 2015 Galaxy Release (v 15.10)](https://docs.galaxyproject.org/en/master/releases/15.10_announce.html)**
was released on November 30.* Highlights include

 Reports Application
    The reports web application has been greatly enhanced - including improved styling, new landing page, more reports, pagination, and sparklines. Huge thanks to [Daniel Bouchard (@Airistotal)](https://github.com/Airistotal) for these enhancements.

 Upload
    The Galaxy upload widget now features support for composite datatypes and improved styling.

 Data Libraries
    Improved API and UI for data libraries - including support for library folder management and search.

 Deprecations:
    Mercurial is deprecated.<br />
    Old Data Libraries UI is deprecated<br />
    graphview visualization is deprecated<br />
    Direct access to Tool Shed repositories through the Mercurial API is deprecated

 and many smaller updates and bug fixes.

See the [full release notes](https://docs.galaxyproject.org/en/master/releases/15.10_announce.html#deprecation-notices) for details.

**Galaxy Docker Image 15.10**

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 15.10](https://github.com/bgruening/docker-galaxy-stable/releases/tag/15.10) as well.  

<span style="font-size: smaller;"> * Hey, we're the Galaxy Project - we can warp time, dang it! </span>


## galaxy-lib 16.1.0 - 16.1.7

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

It is also quite popular and has been downloaded from PyPi almost 4000 times since it was created on December 20.


<div class='right'><a href='/cloudman/'><img src="/images/logos/CloudManWideBlackLogo.png" alt="CloudMan" width="150" /></a></div>

## CloudMan 15.12

[CloudMan](/cloudman/) is a cloud manager that orchestrates all of the steps required to provision a complete compute cluster environment on a cloud infrastructure; subsequently, it allows one to manage the cluster, all through a web browser.

This minor [CloudMan 15.12 release](https://github.com/galaxyproject/cloudman/releases) includes:

* Update Galaxy to the Galaxy 15.10 release (see above for all the juicy details with the improvements that brings along)
* Preload the GIE IPython Docker container onto the image for faster startup: less than 30 seconds to a fully functional IPython notebook vs. several minutes before!
* For those of you that ran into the problem of the size of your user data overfilling after having added and removed a bunch of worker instances - no worries, that has been fixed now and resource tags are no longer stored in the cluster config to prevent *persistent_data.yaml* from becoming too big.
* The file system archive extraction has been made more resilient and synchronous.
* For those that are building custom images, a more direct method for locating the Nginx configuration directory has been added to help with different versions of the operating system (thanks to [Matthew Ralston](https://github.com/MatthewRalston))
* A number of dependent library versions have been updated to their latest versions.

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="150" /></a></div>

## Pulsar 0.6.0 - 0.6.1

[Pulsar 0.6](https://pypi.python.org/pypi/pulsar-app/) was released in December.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

The 0.6.x release includes these changes:

* Pulsar now depends on the new galaxy-lib Python package instead of manually synchronizing Python files across Pulsar and Galaxy.
* Numerous build and testing improvements.
* Fixed a documentation bug in the code (thanks to @erasche). [e8814ae](https://github.com/galaxyproject/pulsar/commit/e8814ae)
* Remove galaxy.eggs stuff from Pulsar client (thanks to @natefoo). [00197f2](https://github.com/galaxyproject/pulsar/commit/00197f2)
* Add new logo to README (thanks to @martenson). [abbba40](https://github.com/galaxyproject/pulsar/commit/abbba40)
* Implement an optional awknowledgement system on top of the message queue system (thanks to @natefoo). [Pull Request 82](https://github.com/galaxyproject/pulsar/pull/82) [431088c](https://github.com/galaxyproject/pulsar/commit/431088c)
* Documentation fixes thanks to @remimarenco. [Pull Request 78](https://github.com/galaxyproject/pulsar/pull/78), [Pull Request 80](https://github.com/galaxyproject/pulsar/pull/80)
* Fix project script bug introduced this cycle (thanks to @nsoranzo). [140a069](https://github.com/galaxyproject/pulsar/commit/140a069)
* Fix config.py on Windows (thanks to @ssorgatem). [Pull Request 84](https://github.com/galaxyproject/pulsar/pull/84)
* Add a job manager for XSEDE jobs (thanks to @natefoo). [1017bc5](https://github.com/galaxyproject/pulsar/commit/1017bc5)
* Fix pip dependency installation (thanks to @afgane) [Pull Request 73](https://github.com/galaxyproject/pulsar/pull/73)

## Other Releases

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/images/logos/PlanemoLogo.png" alt="" width="150" /></a></div>

 **Planemo 0.21.1**

   [Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  [Planemo 0.21.1](https://pypi.python.org/pypi/planemo/0.21.1#history) is the most recent release. See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).


 **BioBlend 0.7.0**

   [BioBlend version 0.7.0](https://github.com/galaxyproject/bioblend/) was released at the beginning of November. BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/develop/api/).  CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG).


 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and BioCloudCentral.


----


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

<br />

----
<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in December](/toolshed/contributions/2015-12/).


----

# Other News

* Help the [MetaboHub](http://dsv.cea.fr/dsv/english/Pages/Research/national-infrastructures/MetaboHub.aspx) / [Workflow4metabolomics](http://galaxy.workflow4metabolomics.org/) and the [University of Birmingham](http://www.birmingham.ac.uk/) / [Galaxy-M](https://github.com/Viant-Metabolomics/Galaxy-M) teams! Please fill out the [Galaxy Workflows for Metabolomics](https://www.surveymonkey.co.uk/r/metabolomics-galaxy) Questionnaire
* From [GigaScience](https://twitter.com/GigaScience):
    * We've been busy in 2015! Read our [end of year blog](http://bit.ly/1NQl4Tk) and we have more exciting things lined up for 2016!
