---
autotoc: true
slug: events/gcc2024/training/microbiome-analysis
---
## Table of contents

* [Pathogen Detection from (Direct Nanopore) Sequencing Data using Galaxy](#pathogen-detection-from-direct-nanopore-sequencing-data-using-galaxy)
  * [Meet the Instructors](#meet-the-instructors)
    * [Engy Nasr](#engy-nasr)
    * [Paul Zierep](#paul-zierep)

# Pathogen Detection from (Direct Nanopore) Sequencing Data using Galaxy

</div>

Food contamination by pathogens is a global health problem, affecting an estimated 600 million people a year. During investigation of a foodborne outbreak, microbiological analysis of food vehicles is performed to detect and identify responsible pathogens and contamination sources. Using metagenomic approaches, one can get an overview of the genomic composition of the microbial community, including the possible pathogens in samples. Together with Oxford Nanopore sequencing, such metagenomic approaches become faster and easier to apply. A key limitation of these approaches is the lack of accessible, easy-to-use, and openly available pipelines for identifying and tracking pathogens from metagenomic data.
This workshop demonstrates the PathoGFAIR workflow, a series of Galaxy-based FAIR workflows using state-of-the-art tools to detect and track pathogens from metagenomic Nanopore sequencing. PathoGFAIR integrates several visualisations and reports for comprehensible results.

## Meet the Instructors

### Engy Nasr

University of Freiburg, Department of Bioinformatics <br>
I am research assistant and a PhD student at the faculty of engineering, the department of bioinformatics. My main PhD focus is Microbiome analysis using Galaxy Framework

### Paul Zierep

University of Freiburg, European Galaxy Team <br>
Enthusiast about workflow development and microbial research. Main focus on microbiome pipeline development and tool integration. Co-organiser of the microGalaxy community.
