---
title: May 2014 Galaxy Update
---
<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the May 2014 *[Galaxy Update](/galaxy-updates/)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Galaxy Biostar

<div class='right'><a href='https://biostar.usegalaxy.org/'><img src="/images/logos/GalaxyBiostar.png" alt="Galaxy Biostar"  /></a></div>

***Note: Galaxy's support forum has moved to [help.galaxyproject.org](https://help.galaxyproject.org/).***

**Galaxy has teamed up with [Biostar](http://biostars.org) to create [Galaxy Biostar](https://biostar.usegalaxy.org), a Galaxy User support forum.**

Galaxy Biostar is a space where researchers using Galaxy come together and share both scientific advice and practical tool help.  Whether on [usegalaxy.org](https://usegalaxy.org), a CloudMan instance, or any other Galaxy ([public](/use/) or local), if you have something to say about *Using Galaxy*, this is the place to do it.

**Current integration with usegalaxy.org**

* The **whole history of the `galaxy-user@bx.psu.edu`** mailing list was imported into *[Galaxy Biostar](https://biostar.usegalaxy.org)*. Your prior posts are automatically claimed when you login!
* If you access [Galaxy Biostar](https://biostar.usegalaxy.org) from [usegalaxy.org](http://usegalaxy.org) (Menu: **Help &rarr; Galaxy Biostar**) you will be automatically logged in. A Galaxy Biostar account will be created for you if it did not previously exist. To obtain this account’s password please use the [password reset feature](https://biostar.usegalaxy.org/accounts/password/reset/) of Galaxy Biostar.
* When you have a question, search Galaxy Biostar directly from any Galaxy tool page.

Read more about how to get started on the [Biostar wiki page](/support/biostar/) or on the [Biostar](https://biostar.usegalaxy.org/p/7287/) [itself](https://biostar.usegalaxy.org/p/7288/).

**Roll-out phase**

* [Galaxy Biostar](https://biostar.usegalaxy.org) is available at [biostar.usegalaxy.org](http://biostar.usegalaxy.org) and will be our primary avenue for end-user support
* The galaxy-user@bx.psu.edu mailing list will continue to be supported during the transition but starting now please use the [Galaxy Biostar forum](https://biostar.usegalaxy.org) to ask all questions about using Galaxy.
* Please *do not double post* to both [Galaxy Biostar](https://biostar.usegalaxy.org) and galaxy-user@bx.psu.edu
* *Send us feedback in [this Biostar post](https://biostar.galaxyproject.org/p/7288/)* to tell us what you think. *We care.*

**What’s next**

* Notice will be given when the galaxy-user@bx.psu.edu mailing list is retired.
* [Archives of galaxy-user@bx.psu.edu](https://lists.galaxyproject.org/archives/list/galaxy-user@lists.galaxyproject.org/) will remain accessible.

Galaxy Biostar was launched on April 23.  We hope you like the change and look forward to any feedback you may have.

# New Papers

*47 papers* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in April. Some papers that may be particularly interesting to the Galaxy community:

* "[Analysis of Next-Generation Sequencing Data Using Galaxy](https://doi.org/10.1007/978-1-4939-0512-6_2)," [Daniel Blankenberg](/people/dan/) and [Jennifer Hillman-Jackson](/people/jennifer-jackson/), *Stem Cell Transcriptional Networks*, Vol. 1150 (2014), pp. 21-43, doi:10.1007/978-1-4939-0512-6_2

* "[Support for data-intensive computing with CloudMan](http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6596260)," Yousef Kowsar and [Enis Afgan](/people/enis-afgan/),  *[Information & Communication Technology Electronics & Microelectronics (MIPRO), 2013 36th International Convention on](http://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=6588635),* (May 2013), pp. 243-248

* "[Chromatin states reveal functional associations for globally defined transcription start sites in four human cell lines](http://www.biomedcentral.com/1471-2164/15/120)," Morten Rye, Geir Kjetil Sandve, Carsten O Daub, Hideya Kawaji, Piero Carninci, Alistair RR Forrest, Finn Drabløs and the FANTOM consortium, *[BMC Genomics](http://www.biomedcentral.com/bmcgenomics/)*, Vol. 15, No. 1. (26 March 2014), 120, doi:10.1186/1471-2164-15-120

* "[A Model-Based Approach to Identify Binding Sites in CLIP-Seq Data](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0093248)," Tao Wang, Beibei Chen, Minsoo Kim, Yang Xie, and Guanghua Xiao, *[PloS ONE](http://www.plosone.org/)*, Vol. 9, No. 4. (2014)

The new papers were tagged in 14 different areas (the most diverse month we've had):

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 21 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>



# Events

## GCC2014: June 30 - July 2, Baltimore

<div class='right'><a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoTall200.png" alt="GCC2014: June 30 - July 2" width="150" /></a></div>

The [2014 Galaxy Community Conference (GCC2014)](/events/gcc2014/) will be held June 30 through July 2, at the [Homewood Campus](http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/) of [Johns Hopkins University](http://jhu.edu), in [Baltimore, Maryland](http://visitors.baltimorecity.gov/), United States.

### Early Registration Closes May 23

[Early registration closes the month](/events/gcc2014/register/). *Early registration saves more than 70% on registration costs, and [Training Day](/events/gcc2014/training-day/) registration is an additional 55% off if you register for both at the same time.*  This is by far the most affordable option, with early registration fees starting at less than $50 per day.  When you register you can also reserve [lodging at Charles Commons](/events/gcc2014/logistics/#lodging), a very affordable housing option in the same building as the conference.

*[Training Day](/events/gcc2014/training-day/)* is an opportunity to learn about all things Galaxy including using Galaxy, deploying and managing Galaxy, extending Galaxy, and Galaxy internals.  There are 5 parallel tracks, each with 3 sessions, with each of those sessions two and half hours long.  *That's 15 sessions and over 37 hours of workshop material.*

<div class='right'><br /><br /><a href='http://ccb.jhu.edu/people/salzberg/'><img src="/images/people/Salzberg.jpg" alt="Steven Salzberg" width="125" /></a></div>

### Keynote Speaker: Steven Salzberg

We are pleased to announce that [Steven Salzberg](http://ccb.jhu.edu/people/salzberg/) will be the keynote speaker at GCC2014.  Steven is a Professor of Medicine, [Biostatistics](http://www.biostat.jhsph.edu/), and [Computer Science](http://www.cs.jhu.edu/) at the [Johns Hopkins University School of Medicine](http://www.hopkinsmedicine.org/som/) where he is also Director of the [Center for Computational Biology](http://ccb.jhu.edu/) at the [McKusick-Nathans Institute of Genetic Medicine](http://igm.jhmi.edu/).  Steven has made many prominent contributions to open source software, including several of the most popular tools used on Galaxy Platforms.  Recently he was awarded the  [2013 Benjamin Franklin Award for Open Access in the Life Sciences](http://www.bioinformatics.org/forums/forum.php?forum_id=10206), and the [2012 Balles Prize in Critical Thinking](http://www.csicop.org/news/show/skeptic_authors_steven_salzberg_and_joe_nickell_to_receive_balles_prize) for his [science column](http://www.forbes.com/sites/stevensalzberg/) at [Forbes](http://www.forbes.com/).

Steve's GCC2014 talk will be on "Transcriptomes and Exomes: Computational Challenges of NGS Data."  

### Galaxy Hackathon at GCC2014

<div class='left'><a href='/events/gcc2014/hackathon/'><img src="/images/logos/GCC2014HackLogoSquare.png" alt="GCC2014 Hackathon" width="150" /></a></div>

The very first [Galaxy Project Hackathon](/events/gcc2014/hackathon/) will be a three day event taking place at Johns Hopkins immediately preceding [GCC2014](/events/gcc2014/) from June 28th-30th.

Do you have a feature you've always wanted to implement?  Just want to hack on Galaxy (or CloudMan!) with other folks?  The [Galaxy Hackathon](/events/gcc2014/hackathon/) will be a great opportunity to meet and work closely with other community and [Galaxy Team](/galaxy-team/) members over the course of three days, culminating in some really great improvements and new features to show off at the [Galaxy Community Conference](/events/gcc2014/) afterward.

Participation in the hackathon itself is completely free, but there's limited space so if you're interested and would like to participate please go ahead and book both your lodging and hackathon seat at [EventBrite](http://bit.ly/gcc2014hackreg).

To help organize ideas and people into more concrete projects, we've also set up a [hackathon-specific Trello board](https://trello.com/b/3YgU637b/gcc2014-hackathon) that we'd love for everyone to go ahead and [start using it](https://trello.com/b/3YgU637b/gcc2014-hackathon).  The board is public and open to commentary and voting, but to create new cards you’ll need to be added as a member so please note the instructions on the board for that.

<div class='right'>
<a href='https://curoverse.com/'><img src="/images/logos/CuroversLogoBig.png" alt="Curoverse: Open Source developers of Arvados" width="170" /></a>&nbsp;
<a href='https://arvados.org/'><img src="/images/logos/ArvadosLogoBig.png" alt="Arvados: a free and open source bioinformatics platform for genomic and biomedical data" width="140" /></a>
</div>

Finally, we are very happy to have **[Curoverse](https://curoverse.com/) on board as the exclusive [Peta level sponsor](/events/gcc2014/hackathon/#sponsors)** of the hackathon.  If you know of any other group that might be interested in sponsoring at the [Giga level](/events/gcc2014/hackathon/#sponsors) please let us know.

### Abstracts and Program

The deadlines for both oral and poster presentations were in April.  Oral presentation submitters have been contacted and [we've heard back from over half of them](/events/gcc2014/abstracts/talks/), and we will continue to update the [Talk Abstracts page](/events/gcc2014/abstracts/talks/) as we hear from the rest.  If you submitted a poster abstract, they you will be notified at the end of this week if your poster was accepted.  We'll start posting those abstracts online then too.

The conference is still accepting [late abstract submissions](/events/gcc2014/abstracts/).  These will not be considered for constructing the initial list of accepted abstracts, but will be reviewed as cancellations occur and space frees up (*and we have always had a few cancellations*).

Look for a [more detailed draft program](/events/gcc2014/program/) to be posted later this month, once we have heard from all presenters.


### Sponsorships and Exhibitors

<div class='right'>
<a href='http://biolchem.bs.jhmi.edu/'><img src="/images/logos/JohnsHopkinsBioChem.png" alt="Biological Chemistry @ Johns Hopkins" width="260" /></a><br />
<a href='http://epigenetics.jhu.edu/'><img src="/images/logos/JohnHopkinsEpigenetics.jpg" alt="Center for Epigenetics" width="260" /></a>
</div>

We are happy to have the [Center for Epigenetics](http://epigenetics.jhu.edu/), and the [Department of Biological Chemistry](http://biolchem.bs.jhmi.edu/), both of Johns Hopkins University as GCC2014 Sponsors.

There are still [Silver and Bronze sponsorships](/events/gcc2014/sponsor-exhibit/) available for the [GCC2014](/events/gcc2014/) and Giga sponsorships for the [Hackathon](/events/gcc2014/hackathon/). Please contact the [Organizers](/events/gcc2014/organizers/) if your organization would like to help sponsor these events.

In 2014 we are also adding [non-sponsor exhibit spaces](/events/gcc2014/sponsor-exhibit/#exhibit) in addition to the sponsor exhibits.  This will significantly increase the size of the exhibit floor. Please contact the [Organizers](/events/gcc2014/organizers/) if your organization would like to have an exhibit space at GCC2014.

## UK May 2014 Galaxy Tour

<div class='left'>
<a href='/events/uk-may2014/'><img src="/images/logos/UKMay2014Tour.png" alt="UK May 2014 Galaxy Tour" width="170px" /></a>
</div>

<div class='right'>
<div class='right'>
<a href='/events/uk-may2014/tgac-workshop/'><img src="/images/logos/TGACLogo.png" alt="Introduction to Galaxy Workshop" width="180" /></a><br /><br />
<a href='/events/uk-may2014/u-edinburgh/'><img src="/images/logos/UEdinburghWideLogo.png" alt="Introduction to Galaxy Workshop" width="180" /></a></div><br />
<a href='http://www.bioinformatics.ed.ac.uk/events/5th-edinburgh-bioinformatics-meeting'><img src="/images/logos/EdinburghBioinformatics.png" alt="5th Edinburgh Bioinformatics Meeting" width="220" /></a><br />
<div class='right'><a href='http://www.igmm.ac.uk/'><img src="/images/logos/IGMMLogo.jpg" alt="Institute of Genetics and Molecular Medicine" width="110" /></a></div>
</div>

A [Galaxy Tour](/events/uk-may2014/) is happening in the United Kingdom in early May 2014.  If you are anywhere close to Norwich or Edinburgh, then it might be worth your while to attend an event.

First, there will be a talk on *Scaling Galaxy for Big Data* at the [NGS Data after the Gold Rush](http://www.tgac.ac.uk/SeqAhead_Scientific_Meeting_and_Management_Committee_Meeting/) meeting, being held 6-7 May, at [The Genome Access Centre (TGAC)](http://www.tgac.ac.uk/) in Norwich.  This will be followed by a hands-on *[Introduction to Galaxy Workshop](/events/uk-may2014/tgac-workshop/)* on 9 May, also at TGAC.

After that, there will be 3 events in Edinburgh the following week, starting on Monday with a hands on *[Introduction to Galaxy Workshop](/events/uk-may2014/u-edinburgh/)* at the [University of Edinburgh](http://www.ed.ac.uk/) in the morning and a *Galaxy Project Update* talk at the [5th Edinburgh Bioinformatics Meeting](http://www.bioinformatics.ed.ac.uk/events/5th-edinburgh-bioinformatics-meeting), in the afternoon, and also at the University.  Finally, on Tuesday 13 May, there will be a [all-day hands-on Galaxy Workshop](/events/uk-may2014/igmm-workshop/) at the [Institute of Genetics and Molecular Medicine (IGMM)](http://www.igmm.ac.uk/) at Western General Hospital.

You must be affiliated with the University of Edinburgh or the IGMM to register for either of those workshops, but all other events are open to anyone.

## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div>

And don't worry if you are not near Norwich or Edinburgh in May.  There are [at least 17 other Galaxy related events](/events/) in the next 70 days in Norway, France, *online*, Croatia, Thailand, Canada, the US, the Netherlands, and Australia. Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<br />

<div class='center'>
<a href='/events/uk-may2014/'><img src="/images/logos/UKMay2014Tour.png" alt="" height="80" /></a>&nbsp;
<a href='http://gmod.org/wiki/GMOD_Online_Training_2014'><img src="/images/logos/GMODOnlineTraining2014.png" alt="GMOD Online Training" height="80" /></a> &nbsp;
<a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoWide200.png" alt="GCC2014 early registration and abstract submission are now open" height="75" /></a>&nbsp;

</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th rowspan=1> April 29 -<br />May 1 </th>
    <td rowspan=1> <strong><a href='/events/bioitworld2014/'>Two tutorials and at least five talks</a></strong> </td>
    <td rowspan=1> <strong><a href='/events/bioitworld2014/'>BioIT World</a></strong><br />Boston, Massachusetts, United States </td>
    <td> <a href='/events/bioitworld2014/'>See tutorial and talk list</a> </td>
  </tr>
  <tr>
    <th> May 6-7 </th>
    <td> <em>Scaling Galaxy for Big Data</em> </td>
    <td> <a href='http://www.tgac.ac.uk/SeqAhead_Scientific_Meeting_and_Management_Committee_Meeting/'>NGS Data after the Gold Rush</a>, <a href='http://www.tgac.ac.uk/'>TGAC</a>, Norwich, United Kingdom </td>
    <td rowspan=5> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> May 9 </th>
    <td> <em><a href='/events/uk-may2014/tgac-workshop/'>Introduction to Galaxy Workshop</a></em> </td>
    <td> <a href='http://www.tgac.ac.uk/'>The Genome Analysis Centre (TGAC)</a>, Norwich, United Kingdom </td>
  </tr>
  <tr>
    <th rowspan=2> May 12 </th>
    <td> <em><a href='/events/uk-may2014/u-edinburgh/'>Galaxy Workshop</a></em> </td>
    <td> <a href='http://www.ed.ac.uk/'>University of Edinburgh</a>, Edinburgh, UK </td>
  </tr>
  <tr>
    <td> <em>Galaxy Project Update</em> </td>
    <td> <a href='http://www.bioinformatics.ed.ac.uk/events/5th-edinburgh-bioinformatics-meeting'>5th Edinburgh Bioinformatics Meeting</a>, <a href='http://www.ed.ac.uk/'>University of Edinburgh</a>, Edinburgh, UK </td>
  </tr>
  <tr>
    <th> May 13 </th>
    <td> <em><a href='/events/uk-may2014/igmm-workshop/'>Galaxy Workshop</a></em> </td>
    <td> <a href='http://www.igmm.ac.uk/'>Institute of Genetics and Molecular Medicine (IGMM)</a>, Edinburgh, UK </td>
  </tr>
  <tr>
    <th> May 12-14 </th>
    <td> <em><a href='http://www.bioinfo.no/training/rna-and-chipseq-analysis-workshop/'>Short course on RNA-seq and ChIP-seq</a></em> </td>
    <td> University of Bergen, Bergen, Norway </td>
    <td> <a href="mailto:david.fredman@uib.no">David Fredman</a> </td>
  </tr>
  <tr>
    <th> May 16 </th>
    <td> <em><a href='http://abims.sb-roscoff.fr/sites/abims.sb-roscoff.fr/files/files/formation_2014/module_formation_abims-galaxy_initiation.pdf'>Galaxy Initiation</a></em> </td>
    <td> <a href='http://abims.sb-roscoff.fr/formation'>Formation en Bioinformatique Plateforme ABiMS</a>, <a href='http://www.sb-roscoff.fr/'>Station Biologique de Roscoff</a>, France </td>
    <td> <a href="mailto:christophe.caron@sb-roscoff.fr">Christophe Caron</a> </td>
  </tr>
  <tr>
    <th> May 19 </th>
    <td> <em><a href='https://colloque.inra.fr/8_js_rfmf_lyon_2014/layout/set/print/Ateliers/Galaxy'>Initiation au traitement et à l'analyse des données métabolomiques sur la plateforme scientifique web Galaxy IFB-MetaboHUB</a></em> </td>
    <td> <a href='https://colloque.inra.fr/8_js_rfmf_lyon_2014/Inscription'>8e Journées Scientifiques du RFMF</a>, Lyon, France </td>
    <td> <a href="mailto:rfmf@bordeaux.inra.fr">Réseau Français de Métabolomique et Fluxomique</a> </td>
  </tr>
  <tr>
    <th> May 19-23 </th>
    <td> <em>Installing Galaxy Workshop</em><div class='indent red'>Application deadline: April 28</div> </td>
    <td> <a href='http://gmod.org/wiki/GMOD_Online_Training_2014'>GMOD Online Training</a> </td>
    <td> <a href='/people/carl-eberhard/'>Carl Eberhard</a>, <a href='http://gmod.org/wiki/User:Girlwithglasses'>Amelia Ireland</a> </td>
  </tr>
  <tr>
    <th> May 27 </th>
    <td> <em><a href='http://www.mipro.hr/LinkClick.aspx?fileticket=y1%2bUY%2b54jB4%3d&tabid=110&mid=461&language=en-US&forcedownload=true'>Enabling Data Analysis with Galaxy CloudMan workshop</a></em> </td>
    <td> <a href='http://www.mipro.hr/CallForPapers/tabid/176/language/en-US/Default.aspx'>MIPRO 2014</a>, Opatija, Croatia </td>
    <td> <a href='/people/enis-afgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> June 2-3 </th>
    <td> <em><a href='http://www.oaspa-asia-conference.com/program/'>Open (and Big) Data – the next challenge</a></em> </td>
    <td> <a href='http://www.oaspa-asia-conference.com/'>1st Asian Conference on Open Access Scholarly Publishing</a>, Bangkok, Thailand </td>
    <td> Scott Edmunds </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> <em><a href='http://bioinformatics.ca/workshops/2014/informatics-high-throughput-sequencing-data-2014'>Informatics on High Throughput Sequencing Data Workshop</a></em> </td>
    <td> Toronto, Canada </td>
    <td> <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 10 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module17-10juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td rowspan=2> <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2014</a>, <a href='http://migale.jouy.inra.fr/'>INRA Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin AT jouy DOT inra DOT fr">Véronique Martin</a>, <a href='http://genome.jouy.inra.fr/~schbath/'>Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 11 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module8bis-11juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> June 16 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module17-16juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td rowspan=2> <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2014</a>, <a href='http://migale.jouy.inra.fr/'>INRA Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin AT jouy DOT inra DOT fr">Véronique Martin</a>, <a href='http://genome.jouy.inra.fr/~schbath/'>Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 17 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module8bis-17juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> June 16-20 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2014/02/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-june-16-20-2014/'>Using Galaxy for Analysis of High Throughput Sequence Data</a></em> </td>
    <td> <a href='http://training.bioinformatics.ucdavis.edu/'>UC Davis</a>, California, United States </td>
    <td> <a href="mailto:training DOT ucdbio AT gmail DOT com">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th> June 23-34 </th>
    <td> <em><a href='http://abims.sb-roscoff.fr/sites/abims.sb-roscoff.fr/files/files/formation_2014/module_formation_galaxy_initiation_phylogenie.pdf'>Galaxy : initiation à la phylogénie</a></em> </td>
    <td> <a href='http://abims.sb-roscoff.fr/formation'>Formation en Bioinformatique Plateforme ABiMS</a>, <a href='http://www.sb-roscoff.fr/'>Station Biologique de Roscoff</a>, France </td>
    <td> <a href="mailto:christophe.caron@sb-roscoff.fr">Christophe Caron</a> </td>
  </tr>
  <tr>
    <th> June 28-30 </th>
    <td> <a href='/events/gcc2014/hackathon/'>Galaxy Hackathon</a> </td>
    <td rowspan=2> <a href='http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/'>Homewood Campus</a> of <a href='http://jhu.edu'>Johns Hopkins University</a>, <a href='http://visitors.baltimorecity.gov/'>Baltimore, Maryland</a>, United States </td>
    <td> <a href='/events/gcc2014/organizers/'>Organizers</a> </td>
  </tr>
  <tr>
    <th> June 30 -<br />July 2 </th>
    <td> <strong><a href='/events/gcc2014/'>2014 Galaxy Community Conference (GCC2014)</a></strong> </td>
    <td> <a href='/events/gcc2014/organizers/'>Organizers</a> </td>
  </tr>
  <tr>
    <th> July 7-9 </th>
    <td> <em><a href='http://biosb.nl/education/course-portfolio/rna-seq/'>NBIC/BioSB RNA- seq data analysis course</a></em> </td>
    <td> Leiden, the Netherlands </td>
    <td> <a href="mailto:education AT biosb DOT nl">NBIC/BioSB</a> </td>
  </tr>
  <tr>
    <th> July 10 </th>
    <td> <em><a href='http://www.genetics.org.au/annual-meeting-2014/workshops/'>An Introduction to Galaxy with the Genomics Virtual Lab</a></em> </td>
    <td> <a href='http://www.genetics.org.au/annual-meeting-2014/'>Post-GSA 2014 Workshop</a>, Sydney, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Post-doctoral position in computational mass spectrometry](http://bit.ly/1qFobAj) at CEA Saclay, France, Paris area
* [Bioinformatics specialist/Scientific programmer](http://bit.ly/1p7GC35), Ettema Lab, Uppsala University
* [Bioinformatician / Computational Biologist](http://bit.ly/1t6n52s), EMBL Heidelberg, Germany
* [Bioinformatics Technician](http://bit.ly/QaQWb9), Bioinformatics Unit, Core Facilities, CRG, Barcelona, Spain  
* [Experimental Officer in Bioinformatics](http://bit.ly/1imRDs4), NERC Metabolomics Facility, University of Birmingham, UK
* [Statistical Genomics Postdoc](/news/statistical-genomics-post-doc-penn-state/) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

# New Public Servers

Two [public Galaxy servers](/use/) were added to the published list in April:

## Globus Genomics Proteomics

<div class='right solid'><a href='http://proteomics.globusgenomics.org/'><img src="/images/logos/GlobusGenomics.png" alt="Globus Genomics Proteomics Galaxy Server"  /></a></div>

* *Links:*
    * [Globus Genomics Proteomics Galaxy Server](http://proteomics.globusgenomics.org/)
    * *[Building a Low-budget Public Resource for Large-scale Proteomic Analyses](https://depot.galaxyproject.org/hub/attachments/documents/presentations/GlobusGenomicsProteomics.pdf)*, presentation by [Anoop Mayampurath](http://www.ci.uchicago.edu/profile/257) at [Globus World 2014](http://globusworld.org/)
* *Domain/Purpose:*
    The [Globus Genomics Project](http://globusgenomics.org) demonstration Galaxy server. It has a strong emphasis on proteomics
* *Comments:*
    Includes [Globus](http://www.globus.org/) tools and over 20 proteomics tools, as well as many NGS analysis tools and the usual data manipulation tool set.
* *User Support:*

* *Quotas:*
* *Sponsor(s):*
    [Globus Genomics](http://globusgenomics.org), [The Computation Institute](http://ci.uchicago.edu/), [The University of Chicago](http://www.uchicago.edu/), [Argonne National Lab](http://www.anl.gov/), and [Amazon Web Services](http://aws.amazon.com/).

<br />

## SunLab

<div class='right solid'><a href='http://meta.cmb.usc.edu/'><img src="/public-galaxy-servers/archive/sun-lab.png" alt="SunLab Galaxy Server"  /></a></div>

* *Links:*
    [SunLab Galaxy Server](http://meta.cmb.usc.edu/)
* *Domain/Purpose:*
    Provides access to computational tools developed by [Fengzhu Sun's](http://www-rcf.usc.edu/~fsun/) group at [University of Southern California](http://usc.edu), notably tools for local similarity analysis (LSA).
* *Comments:*
* *User Support:*
    [Email](mailto:lxia AT usc DOT edu)
* *Quotas:*
    "Due to the limited computational resources, we refer users not using the tools developed by SunLab to the main public Galaxy site. We also encourage user applying SunLab tools to large data sets to install their standalone version of the specific tools, or install this version of Galaxy server with SunLab tools integrated.
* *Sponsor(s):*
    The [SunLab](http://www-rcf.usc.edu/~fsun/) at the [University of Southern California](http://usc.edu).

# Galaxy Distributions

## April 14, 2014 Galaxy Distribution

<div class='right'>
<table>
  <tr>
    <td> <a href='http://usegalaxy.org'><img src="/images/news-graphics/2014_04_14_trackster-deep.png" alt="Trackster Vis" width="300" /></a> </td>
  </tr>
</table>

    *Trackster **deep coverage** view*</div>
<br />
**[News Brief](http://wiki.galaxyproject.org/DevNewsBriefs/2014-04-14)** **Highlights:**

* Visualization framework and Trackster display enhancements
* Tool Shed upgrades for repos, installs, tests, and docs
* Over 100 genomes with new content on our rsync server
* UI unification of design plus expanded dataset action access
* API functionality additions including new job control/admin abilities
* More features for admin functions, config options, and job controls
* 18 new community contributions incorporated (big thanks!)

<table>
  <tr>
    <td rowspan=3 style=" border: none;"> <a href='http://getgalaxy.org/'><img src="http://galaxy.psu.edu/static/getgalaxy.png" alt="getgalaxy" width=70 /></a> &nbsp;&nbsp; </td>
    <td colspan=2 style=" border: none;"> <strong><a href='http://wiki.galaxyproject.org/Admin/Get%20Galaxy'>getgalaxy.org</a></strong> </td>
  </tr>
  <tr>
    <td style=" border: none;"> <strong><a href='http://galaxy-dist.readthedocs.org'>galaxy-dist.readthedocs.org</a></strong> </td>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> <strong><a href='http://bitbucket.org/galaxy/galaxy-dist'>bitbucket.org/galaxy/galaxy-dist</a></strong> </td>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> <strong>new</strong>: </td>
    <td style=" border: none;"> <code>$ hg clone https://bitbucket.org/galaxy/galaxy-dist#stable </code> </td>
  </tr>
  <tr>
    <td style=" border: none;"> <strong>upgrade</strong>: </td>
    <td style=" border: none;"> <code>$ hg pull </code> <br /> <code>$ hg update release_2014.04.14 </code> </td>
  </tr>
</table>

<br />

## CloudMan and BioBlend

[BioBlend 0.4.3](https://pypi.python.org/pypi/bioblend/0.4.3) was released on April 11, 2014.  

The most recent version of CloudMan was [released in January 2014](/news/cloudman-release201401/).


# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a></div>
<div class='right'><img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style="text-align: center;"> &nbsp;&nbsp; <a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a> </em> &nbsp;&nbsp;<br /> </td>
  </tr>
</table>

<br /><br />

The *[Community Log Board](/news/)* and  *Deployment Catalog* *[Galaxy community hubs](/community/)* were launched in December.  If you have a deployment, or experience you want to share then please publish them.


<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

## New Public Tool Sheds

<div class='left'><a href='http://toolshed.nbic.nl/'><img src="/images/logos/DTLLogo.png" alt="DTL ToolShed" width="150" /></a></div>

The [Dutch Techcentre for Life Sciences (DTL)](http://www.dtls.nl/dtl/) has made its [Galaxy ToolShed](http://toolshed.nbic.nl/) publicly available.  The [DTL ToolShed](http://toolshed.nbic.nl/) has almost 70 tools in it, from [ANNOVAR](http://toolshed.nbic.nl/view/saskia-hiltemann/annovar) to [VCF-2-VariantList](http://toolshed.nbic.nl/view/saskia-hiltemann/virtual_normal_preprocessing).  This ToolShed was originally started at NBIC.

## Galaxy Project ToolShed New Repositories

* [infernal](http://bit.ly/1lmSSsm): Inference of RNA Alignments search DNA sequence DBs for RNA structure/sequence similarities
* [msa_datatypes](http://bit.ly/QOoZpI): Galaxy applicable data formats for Multiple Sequence Alignments
* [taxonomy_krona_chart](http://bit.ly/1r5lrdC): convert metagenomic profiling results into zoomable pie chart using Krona
* [deeptools_workflows](http://bit.ly/1tiIPIt): deepTools workflows to visualize large datasets in a meaningful way
* [mosaik2](http://bit.ly/1r5kXUP): reference-guided aligner for next-generation sequencing technologies.
* [suite_gops_1_0](http://bit.ly/1eIw5a2): Metarepository for the gops tool suite - will install the gops tool suite
* [suite_gatk_1_4](http://bit.ly/QtB2IH): A suite of Galaxy utilities associated with version 1.4 of the GATK package.
* [join](http://bit.ly/1gHkF1G): Join the intervals of two datasets side-by-side
* [compute_q_values](http://bit.ly/PcPq7e): Compute q-values based on multiple simultaneous tests p-values
* [charts](http://bit.ly/1nuzERh): Enable advanced visualization options in Galaxy Charts, a visualization plugin for Galaxy
* [concat](http://bit.ly/1eK0QvF): Concatenate two datasets into one dataset
* [merge](http://bit.ly/1kzaYHh): Merge the overlapping intervals of a dataset
* [coverage](http://bit.ly/1hKkkzC): Coverage of a set of intervals on second set of intervals
* [basecoverage](http://bit.ly/1gBb8Ji): count total bases covered by a set of intervals
* [intersect](http://bit.ly/1hMgJ55): Intersect the intervals of two datasets
* [flanking_features](http://bit.ly/1eRODzf): Fetch closest non-overlapping feature for every interval
* [subtract](http://bit.ly/1paBnQg): Subtract the intervals of two datasets
* [quality_filter](http://bit.ly/1paB5Zx): filter nucleotides in every alignment block of MAF file based on quality/PHRED scores
* [rcve](http://bit.ly/P45Iz8): Compute RCVE (Relative Contribution to Variance) for all possible variable subsets
* [microsats_mutability](http://bit.ly/1eGqPE2): Estimate microsatellite mutability by specified attributes
* [partialr_square](http://bit.ly/1hH1xoL): Compute partial R square
* [linear_regression](http://bit.ly/1hJaij9): uses R 'lm' function to perform linear regression
* [getindels_2way](http://bit.ly/1kYUL0Q): Fetch Indels from pairwise alignments
* [getindelrates_3way](http://bit.ly/1t76Zpg): Estimate Indel Rates for 3-way alignments
* [cluster](http://bit.ly/1h1eLaW): Cluster the intervals of a dataset
* [complement](http://bit.ly/1eGog50): Complement intervals of a dataset
* [subtract_query](http://bit.ly/1gyaa40): Subtract Whole Dataset from another dataset
* [featurecounter](http://bit.ly/1gy9NX9): find the coverage of intervals in the first dataset on intervals in the second dataset
* [logistic_regression_vif](http://bit.ly/RjVLjd): Perform Logistic Regression with vif
* [maf_cpg_filter](http://bit.ly/1iQ9jtW): Mask CpG/non-CpG sites from MAF file
* [get_flanks](http://bit.ly/RjVjBl): find the upstream and/or downstream flanking region(s)
* [count_covariates](http://bit.ly/1m3Ni0a): Count Covariates on BAM files
* [depth_of_coverage](http://bit.ly/1ijA5x5): Depth of Coverage on BAM files at different levels of partitioning and aggregation
* [substitutions](http://bit.ly/RjU6tR): Fetch substitutions from pairwise alignments
* [microsats_alignment_level](http://bit.ly/RjTRz0): Extract Orthologous Microsatellites from pair-wise alignments
* [variant_combine](http://bit.ly/1p4zKUd): Combines VCF records from different sources; supports full merges & set unions
* [best_regression_subsets](http://bit.ly/1eH7Qnf): use regsubsets R function for regression subset selection
* [variant_filtration](http://bit.ly/1qHKOSN): Filter variant calls using user-selectable, parameterizable criteria
* [windowsplitter](http://bit.ly/1hqwPAa): splits intervals into smaller intervals based on the specified window-size and type
* [variants_validate](http://bit.ly/1eh939U): Validates a variants file.
* [variant_select](http://bit.ly/QdPkNi): Select Variants from VCF files
* [table_recalibration](http://bit.ly/1iBUxFP): Second pass in a two-pass BAM processing step, doing a by-read traversal
* [realigner_target_creator](http://bit.ly/1n1d1Xe): Realigner Target Creator for use in local realignment
* [variant_eval](http://bit.ly/OQ2cZ1): General tool for variant evaluation (% in dbSNP, genotype concordance, Ti/Tv ratios, ...)
* [variant_recalibrator](http://bit.ly/QdNPyF): learns a Gaussian mixture model over variant annotations and evaluates the variant
* [unified_genotyper](http://bit.ly/1lVOYbS): Variant caller which unifies approaches of several disparate callers
* [substitution_rates](http://bit.ly/1hGLYZi): Estimate substitution rates for non-coding regions using Jukes-Cantor JC69 model
* [variant_annotator](http://bit.ly/1lVOi6m): Annotate variant calls with context information.
* [weightedaverage](http://bit.ly/QaMbxN): Assign weighted-average of the values of features overlapping an interval
* [tables_arithmetic_operations](http://bit.ly/1hwAxsQ): Arithmetic Operations on tables
* [print_reads](http://bit.ly/1mZYGu6): Dynamically merge multiple BAM files, resulting in merged output sorted in coordinate order
* [variant_apply_recalibration](http://bit.ly/1lNSL8F): Cut vcf to get novel FDR levels specified during Variant Recalibration
* [indel_realigner](http://bit.ly/PVx2AI): local realignment of reads based on misalignments due to the presence of indels
* [analyze_covariates](http://bit.ly/1epA8rQ): Create collapsed recal csv files, call R to plot residual error vs covariates
* [data_manager_gatk_picard_index_builder](http://bit.ly/1mZXSFo): Generate GATK-sorted Picard indexes
* [chartskit](http://bit.ly/1hEnCzn): Enables advanced visualization options in Galaxy Charts
* [column_join](http://bit.ly/1qlIeDF): Join tabular files

# Other News

* The [meeting report and slides for the Galaxy Australasia Workshop](/events/gaw2014/) are now online.
* Galaxy has a [new status interface](http://status.galaxyproject.org). Check [status.galaxyproject.org](http://status.galaxyproject.org) if you suspect downtime.
