---
autotoc: true
title: January 2014 Galaxy Update
---


<div class='right'></div>

<div class='left'><a href='/GalaxyUpdates'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the January 2014 *[Galaxy Update](/GalaxyUpdates)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/GalaxyUpdates)* complement the *[Galaxy Development News Briefs](/DevNewsBriefs)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Galaxy Community Hubs

December saw the launch of two *[Galaxy community hubs](/Community)* - one to describe Galaxy deployments, and one to document admin and deployment experiences with Galaxy.  Thanks to everyone who has described their experiences in the first month.

## Galaxy Community Log Board

<div class='left'><a href='/Community/Logs'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' width="200" /></a></div>

The *[Galaxy Community Log Board](/Community/Logs)* is a place to share how you addressed a particular task in your Galaxy deployment.  Log entries describe specific solutions to particular tasks, such as the [details of what steps were taken to deploy Galaxy on particular platforms](/Community/Log/2013/URGIVirtualisation), or [specific stumbling blocks when configuring Galaxy](/Community/Log/2013/Example).  Solutions can be [described on this wiki](/Community/Log/2013/Example), or just [some metadata and a link to the description elsewhere](http://wiki.galaxyproject.org/Community/Log/2013/UsingBioServicesWithGalaxy).  Log entries are an easy way to help others (and learn from others) by sharing what you've already done.  If you have figured out or documented how to do something, then this is the ideal forum for sharing that information with the community.

<div class='left'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='/Community/Logs#add-a-log-page'>Share your experience</a> </em><a href='/Community/Logs#add-a-log-page'>now</a><em> &nbsp;&nbsp; </th>
  </tr>
  <tr>
    <td style=" border: none; text-align: center;"> <span style="font-size: smaller;"> C'mon, it will make you feel good! </span> </td>
  </tr>
</table>

</div>

The [Galaxy Community Log Board](/Community/Logs) describes how things can be done, and it complements the (also [recently announced](/News/GalaxyDeploymentCatalog)) *[Galaxy Deployments Catalog](/Community/Deployments)*, which describes different Galaxy installations.  Both the [Log Board](/Community/Logs) and the [Deployments Catalog](/Community/Deployments) came out of discussions at the [GCC2013 GalaxyAdmins Birds-of-a-Feather gathering](/Events/GCC2013/BoF/GalaxyAdmins).  Thanks to everyone who attended that BoF and to everyone who taken the time to [log their experiences so far](/Community/Logs).

## Galaxy Deployment Catalog

<div class='left'><a href='/Community/Deployments'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' width="150" /></a></div> 

<div class='right'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='/Community/Deployments#add-your-galaxy-deployment'>Describe your instance</a> </em><a href='/Community/Logs#add-a-log-page'>now</a><em> &nbsp;&nbsp; </th>
  </tr>
  <tr>
    <td style=" border: none; text-align: center;"> <span style="font-size: smaller;"> All the cool researchers are doing it! </span> </td>
  </tr>
</table>

</div>

The [Galaxy Deployment Catalog](/Community/Deployments) is a place to describe individual Galaxy deployments, and to learn how others in the community are implementing Galaxy in production environments.

If you have a Galaxy deployment then please [take a few minutes to describe your instance to the community](/Community/Deployments).
<br />


# 2013 Events Archive

<div class='right'>
<table>
  <tr>
    <td colspan=2 style=" text-align: center; border: none;"> <span style="font-size: larger;"> </strong><a href='/Events/Archive'>2013 Events</a><strong> </span> </td>
  </tr>
  <tr>
    <th style=" text-align: right;"> 200+ </th>
    <td> Talks, workshops, tutorials, ... </td>
  </tr>
  <tr>
    <th style=" text-align: right;"> 90+ </th>
    <td> Events </td>
  </tr>
  <tr>
    <th style=" text-align: right;"> 80% </th>
    <td> with slides/video</td>
  </tr>
  <tr>
    <th style=" text-align: right;"> 75% </th>
    <td> presented by the Galaxy Community</td>
  </tr>
</table>

</div>

The [Galaxy Events](/Events) for 2013 have been moved to the [Events Archive](/Events/Archive), where they can still be accessed.  There are 160 items in the 2013 Events list.  That count *goes well over 200 items* if you follow some links and include the 13 talks at [Galaxy Day](/News/GalaxyDay2013Report); and all the talks (25), posters (26), lightning talks (19) and Training Day Sessions (15) at [GCC2013](/Events/GCC2013); and the 20+ items [at ISMB 2013](/Events/ISMB2013); and ...  Furthermore, *at least 3/4 of those items were presented by people outside the core Galaxy Team.*  

While those numbers are just plain cool, the really useful bit is that there are slides, exercises, and videos for over 4/5 of those presentation.  If you have a question about Galaxy, chances are someone talked about it in 2013, *and that you can find their slides in the [Events Archive](/Events/Archive).*

Many, many thanks to everyone who talked about Galaxy last year and for making their material available online.


# New Papers

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 18 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 20 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>


*52 papers* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in December. Some papers that may be particularly interesting to the Galaxy community:

* "[Galaxy tools to study genome diversity](http://bit.ly/1cQUjwU)" by Bedoya-Reina, *et al.*, *[GigaScience](http://gigasciencejounal.com)*, Vol. 2, No. 1. (2013), 17, doi:10.1186/2047-217x-2-17
* "[Detection of candidate tumor driver genes using a fully integrated Bayesian approach](http://dx.doi.org/10.1002/sim.6066)" by Yang, *et al.*, *Statistics in Medicine* (1 December 2013), doi:10.1002/sim.6066
* "[In silico to in vivo splicing analysis using splicing code models](http://dx.doi.org/10.1016/j.ymeth.2013.11.006)" by Gazzara, *et al.*, *Methods* (December 2013), doi:10.1016/j.ymeth.2013.11.006

This brings the total number of papers to 1330.

# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Integrated Data Analysis for Improved Understanding of Disease Life Sciences studentship](http://www.qmul.ac.uk/lifesciences/phd/data-analysis/index.html), Queen Mary University of London 

* [Stage Master2](http://bit.ly/1hQIpTZ) "Annotation du transcriptome de l’esturgeon" Bordeaux, France 
* [Informatics Software Support Specialist](http://bit.ly/1cdfOpr) at the University of Minnesota, United States 
* [CDD Gif-sur-Yvette NGS et Variation Structurales UMR de Génétique Végétale](http://bit.ly/1exDipD), INRA-Université Paris Sud-CNRS 
* [Statistical Genomics Postdoc](/News/StatisticalGenomicsPostDocPennState) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/GalaxyIsHiring)**, *including positions at George Washington University*.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />

# Events

## GCC2014: June 30 - July 2, Baltimore

<div class='right'><a href='/Events/GCC2014'><img src='/Images/Logos/GCC2014LogoTall200.png' alt='GCC2014: June 30 - July 2' width="150" /></a></div>

The [2014 Galaxy Community Conference (GCC2014)](/Events/GCC2014) will be held June 30 through July 2, at the [Homewood Campus](http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/) of [Johns Hopkins University](http://jhu.edu), in [Baltimore, Maryland](http://visitors.baltimorecity.gov/), United States.

### Training Day Topic Voting, January 6-17

GCC2014 will start with a [Training Day](/Events/GCC2014/TrainingDay) on June 30.  As for [GCC2013](/News/GCC2013TrainingDayNominations), the specific topics offered will be determined by you, the Galaxy Community.  Topic nomination was open December 2 through December 20.  The organizers are now collating the nominations into a single list that the community (yes, that means you) will vote on these topics January 6-17.  The final schedule of topics and instructors will be published no later than February 10, when registration opens.

Watch the [Galaxy Mailing Lists](/MailingLists) for announcements and links to the voting form.

*[Training Day](/Events/GCC2014/TrainingDay)* is an opportunity to learn about all things Galaxy including using Galaxy, deploying and managing Galaxy, extending Galaxy, and Galaxy internals.  There will be 4 parallel tracks, each with 3 sessions, with each of those sessions two and half hours long.  *That's 12 sessions and 30 hours of workshop material.*

<div class='right'><br /><br /><a href='http://www.sgi.com/solutions/genomics/'><img src='/Images/Logos/SGILogo400.jpg' alt='SGI' width="120" /></a></div>

### Sponsorships

<div class='left'><a href='http://www.emc.com/isilon'><img src='/Images/Logos/EMCIsilonLogo.jpg' alt='EMC Isilon' width="200" /></a><br />
</div>
 
We are pleased to announce several new [sponsorships](/Events/GCC2014/SponsorExhibit) for GCC2014.  First, [EMC Isilon](http://www.emc.com/isilon) will be a Gold Sponsor for the third year in a row, and  [SGI](http://www.sgi.com/) will a [Gold Sponsor](/Events/GCC2014/SponsorExhibit) for the first time.

<br />

Also please welcome three new [Silver Sponsors](/Events/GCC2014/SponsorExhibit):
* [Globus Genomics](http://globus.org/genomics), for the 3rd year
* [GigaScience Journal](http://www.gigasciencejournal.com/) and [BioMed Central](http://www.biomedcentral.com/), for the 2nd year
* [ABgenomica](http://www.abgenomica.com/), for the first time.

<table>
  <tr>
    <td style=" border: none;"> <a href='http://globus.org/genomics'><img src='/Images/Logos/GlobusGenomics.png' alt='Globus Genomics' width="140" /></a> </td>
    <td style=" border: none; width: 5%; text-align: center;"> </td>
    <td style=" border: none; text-align: center;"> <a href='http://www.gigasciencejournal.com/'><img src='/Images/Logos/GigaScienceLogo250.png' alt='GigaScience Journal' width="130" /></a> <span style="font-size: smaller;"> <br />&nbsp;<br /> </span> <a href='http://www.biomedcentral.com/'><img src='/Images/Logos/BMCLogo200.png' alt='BioMed Central, The Open Access Publisher' width="130" /></a> </td>
    <td style=" border: none; width: 5%; text-align: center;"> </td>
    <td style=" border: none; text-align: center;"> <a href='http://www.abgenomica.com/'><img src='/Cloud/ABgenomicaLogo.jpg' alt='ABgenomica' width="150" /></a> </td>
  </tr>
</table>


The Platinum, Gold and Silver sponsors will have exhibit booths at the event.  There are still [Silver and Bronze sponsorships](/Events/GCC2014/SponsorExhibit) still available. Please contact the [Organizers](/Events/GCC2014/Organizers) if your organization would like to help sponsor this event. 

### Exhibitors

In 2014 we are also adding [non-sponsor exhibit spaces](/Events/GCC2014/SponsorExhibit#exhibit) in addition to the sponsor exhibits.  This will significantly increase the size of the exhibit floor. Please contact the [Organizers](/Events/GCC2014/Organizers) if your organization would like to have an exhibit space at GCC2014. 


## 2013 Galaxy Day Report

<div class='right'><a href='http://wiki.sb-roscoff.fr/ifb/index.php/Accueil'><img src='/Images/Logos/GalaxyIFB.png' alt='Galaxy Day' width="150" /></a></div>

The [2013 Galaxy Day event](http://wiki.sb-roscoff.fr/ifb/index.php/Accueil) took place at Institut Curie in Paris, 4 December.  Program and slides for all presentations are [available online](http://wiki.sb-roscoff.fr/ifb/index.php/Accueil).  A [meeting summary is also available](/News/GalaxyDay2013Report).


## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif' /></a></div> There are several events of interest in the next few months. Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<br />

<div class='center'> 
<a href='/Events/PAG_GMOD_2014'><img src='/Images/Logos/PAGLogo114.png' alt='Galaxy @ Plant and Animal Genome (PAG 2014)' height="80" /></a>
<a href='http://gmod.org/wiki/Jan_2014_GMOD_Meeting'><img src='/Images/Logos/GMOD2014.png' alt='2014 GMOD Meeting' height="70" /></a> &nbsp;
<a href='http://www.supagro.fr/web/pages/?idl=19&page=498&id_page=2666'><img src='/Images/Logos/SupAgroMontpelierLogo.jpg' alt='Analyse Bioinformatique de séquences pour l'amélioration des plantes' height="60" /></a>
<a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-et-ChIP-seq-sequencage-haut-debit--a-l-aide-d-outils-orientes-vers-un-public-de-biologistes_a39.html'><img src='/Images/Logos/PRABILogo.png' alt='RNA-Seq & ChIP-Seq analysis course using Galaxy' height="60" /></a>

</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th rowspan=3> January 11-15 </th>
    <td> </em><a href='https://pag.confex.com/pag/xxii/webprogram/Session2298.html'>Galaxy for NGS Data Analysis: A Hands-on Computer Demo</a><em> </td>
    <td rowspan=3> </strong><a href='/Events/PAG_GMOD_2014'>Plant and Animal Genome XXII (PAG 2014)</a><strong>, San Diego, California, United States </td>
    <td rowspan=1> <a href='/DaveClements'>Dave Clements</a>, <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a> </td>
  </tr>
  <tr>
    <td> </em><a href='https://pag.confex.com/pag/xxii/webprogram/Session2159.html'>Galaxy Cloudman: A Gentle Introduction to Data Analysis on the Cloud</a><em><div class='indent'>Part of the <a href='https://pag.confex.com/pag/xxii/webprogram/Session2159.html'>GMOD Workshop</a></div> </td>
    <td rowspan=1> <a href='/DaveClements'>Dave Clements</a>, <a href='http://gmod.org/wiki/User:Scott'>Scott Cain</a> </td>
  </tr>
  <tr>
    <td> Plus <a href='/Events/PAG_GMOD_2014'>2 more talks and 4 posters</a> </td>
    <td> See <a href='/Events/PAG_GMOD_2014'>list</a> </td>
  </tr>
  <tr>
    <th> January 16-17 </th>
    <td> <a href='http://gmod.org/wiki/Jan_2014_GMOD_Meeting'>2014 GMOD Meeting</a> </td>
    <td> San Diego, California, United States </td>
    <td> <a href='/DaveClements'>Dave Clements</a>, <a href='http://gmod.org/wiki/User:Scott'>Scott Cain</a> </td>
  </tr>
  <tr>
    <th> February 10-14 </th>
    <td> </em><a href='http://www.supagro.fr/web/pages/?idl=19&page=498&id_page=2666'>Analyse Bioinformatique de séquences pour l'amélioration des plantes</a><em> </td>
    <td> Campus la Gaillarde, Montpellier, France </td>
    <td> <a href="mailto:yolande DOT olivier AT supagro DOT inra DOT fr">Yolande OLIVIER</a> </td>
  </tr>
  <tr>
    <th> February 11-13 </th>
    <td> </em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-et-ChIP-seq-sequencage-haut-debit--a-l-aide-d-outils-orientes-vers-un-public-de-biologistes_a39.html'>RNA-Seq & ChIP-Seq analysis course using Galaxy</a><em> </td>
    <td> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <a href="mailto:navratil AT prabi DOT fr">Navratil V</a>., <a href="mailto:oger AT prabi DOT fr">Oger C.</a>, <a href="mailto:veber AT prabi DOT fr">Veber P.</a>, <a href="mailto:deschamps AT prabi DOT fr">Deschamps C.</a>, <a href="mailto:perriere AT prabi DOT fr">Perriere G.</a> </td>
  </tr>
  <tr>
    <th> February 12-15 </th>
    <td> <a href='http://www.agbt.org/'>Advances in Genome Biology and Technology (AGBT)</a> </td>
    <td> Marco Island, Florida, United States </td>
    <td> <a href='/anton'>Anton Nekrutenko</a> </td>
  </tr>
  <tr>
    <th> February 26-28 </th>
    <td> <a href='http://gmod.org/wiki/GMOD_Malaysia_2014'>GMOD Malaysia 2014</a> </td>
    <td> Kuala Lumpur, Malaysia </td>
    <td> <a href='http://gmod.org/wiki/User:Girlwithglasses'>Amelia Ireland</a> </td>
  </tr>
  <tr>
    <th> February 27 </th>
    <td> </em><a href='http://www.nowgen.org.uk/facilities/events/event.php?id=63'>Introduction to NGS Bioinformatics</a><em> </td>
    <td> <a href='http://www.nowgen.org.uk/'>Nowgen Centre</a>, Manchester, United Kingdom </td>
    <td> <a href="mailto:training AT nowgen DOT org DOT uk">Jan Taylor</a> </td>
  </tr>
  <tr>
    <th> March 24-25 </th>
    <td> </em><a href='https://wiki.galaxyproject.org/Events/GAW2014'>Galaxy Australasia Workshop</a><em> </td>
    <td> <a href='http://www.synchrotron.org.au/'>Australian Synchrotron</a> and <a href='http://mcec.com.au/'>Melbourne Convention Centre</a>, Melbourne, Australia </td>
    <td> <a href='http://australianbioinformatics.net/gaw-2014#'>Registration</a> </td>
  </tr>
  <tr>
    <th> March 26-30 </th>
    <td> </em>Galaxy toolset for Drosophila genomics<em><div class='indent'>and one-on-one help in the Flybase Demonstrations Room</div> </td>
    <td> <a href='http://www.genetics-gsa.org/drosophila/2014/'>Drosophila Research Conference</a>, San Diego, California, United States </td>
    <td> <a href='/anton'>Anton Nekrutenko</a>, <a href='/JenniferJackson'>Jen Jackson</a> </td>
  </tr>
  <tr>
    <th> May 27 </th>
    <td> </em><a href='http://www.mipro.hr/LinkClick.aspx?fileticket=y1%2bUY%2b54jB4%3d&tabid=110&mid=461&language=en-US&forcedownload=true'>Enabling Data Analysis with Galaxy CloudMan workshop</a><em> </td>
    <td> <a href='http://www.mipro.hr/CallForPapers/tabid/176/language/en-US/Default.aspx'>MIPRO 2014</a>, Opatija, Croatia </td>
    <td> <a href='/EnisAfgan'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> </em><a href='http://bioinformatics.ca/workshops/2014/informatics-high-throughput-sequencing-data-2014'>Informatics on High Throughput Sequencing Data Workshop</a><em> </td>
    <td> Toronto, Canada </td>
    <td> <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 2 </th>
    <td> </strong><a href='/Events/GCC2014'>2014 Galaxy Community Conference (GCC2014)</a><strong> </td>
    <td> <a href='http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/'>Homewood Campus</a> of <a href='http://jhu.edu'>Johns Hopkins University</a>, <a href='http://visitors.baltimorecity.gov/'>Baltimore, Maryland</a>, United States </td>
    <td> <a href='/Events/GCC2014/Organizers'>Organizers</a> </td>
  </tr>
</table>

<br />

# New Public Servers

One new server was added to [the list of publicly accessible Galaxy servers](/PublicGalaxyServers) last month.

## CAPER

[CAPER](http://www.bprc.ac.cn/CAPE) is "An Interactive, Configurable, and Extensible Workflow-Based Platform to Analyze Data Sets from the [Chromosome-centric Human Proteome Project](http://www.c-hpp.org/)" at [Beijing Proteome Research Center](http://www.bprc.ac.cn/), China.

The Chromosome-centric Human Proteome Project (C-HPP) aims to map and annotate the entire human proteome by the "chromosome-by-chromosome" strategy. As the C-HPP proceeds, the increasing volume of proteomic data sets presents a challenge for customized and reproducible bioinformatics data analyses for mining biological knowledge. To address this challenge CAPER 2.0 − an interactive, configurable and extensible workflow-based platform is developed for C-HPP data analyses. 

<div class='right solid'><a href='http://www.bprc.ac.cn/CAPE'><img src='/PublicGalaxyServers/CAPER300.png' alt='CAPER'  /></a></div>

In addition to the previous visualization functions of [track-view and heatmap-view](http://pubs.acs.org/doi/abs/10.1021/pr300831z), CAPER 2.0 presents a powerful toolbox for C-HPP data analyses and also integrates Galaxy, allowing users to easily conduct their own C-HPP proteomic data analyses and visualization. CAPER 2.0 presents four specific workflows for finding missing proteins, mapping peptides to chromosomes for genome annotation, integrating peptides with transcription factor binding sites from ENCODE data sets, and functionally annotating proteins.

See "[CAPER 2.0: An Interactive, Configurable, and Extensible Workflow-Based Platform to Analyze Data Sets from the Chromosome-centric Human Proteome Project](http://pubs.acs.org/doi/abs/10.1021/pr400795c)" by Wang, *et al.*, *J. Proteome Res.*, DOI: 10.1021/pr400795c, Publication Date (Web): November 22, 2013.

[Tutorials](http://61.50.138.124/caper2/static/documentation/index.html) and support ([lidong DOT bprc AT foxmail DOT com](Dong Li), [wangdanburnett@163.com](Dan Wang)) are offered.  CAPER 2.0 is sponsored by [Beijing Proteome Research Center (BPRC)](http://www.bprc.ac.cn/), the [Human Liver Proteome Project (HLPP)](http://www.hupo.org/initiatives/human-liver-proteome-project/), and the [Chromosome-centric Human Proteome Project (C-HPP)](http://www.c-hpp.org/)

# Galaxy Distributions

The [most recent Galaxy Distribution was November 04, 2013](/DevNewsBriefs/2013_11_04).

The latest version of [/CloudMan](/CloudMan) was [released in July](/News/CloudManRelease).

<br />

# ToolShed Contributions

<div class='right'><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

## New Repositories in the Galaxy Project ToolShed

* [redup](http://bit.ly/1gAatJQ): Remove exact duplicate reads from paired-end fastq files 
* [rgttable](http://bit.ly/1bFbgd8): Generate FPKM tables from cuffdiff read_group_tracking files 
* [dgidb_annotator](http://bit.ly/IHOqFC): Annotate tabular file with information from [Drug-Gene Interaction DB](http://dgidb.genome.wustl.edu) 
* [gatk2](http://bit.ly/1eYgaAX): The Genome Analysis Toolkit in Version 2
* [table_annovar](http://bit.ly/1bJC382): Annotate VCF file using ANNOVAR annotations to create a tabular file that can be filtered 
* [spectral_hk](http://bit.ly/1d6jJ5Z): Convert !InChIs to spectral hashkeys.  
* [snpeff](http://bit.ly/1f8ZxBP): snpEff tools from Pablo Cingolani 
* [split_file_on_column](http://bit.ly/1be1ybR): Split file on specific column; every group is saved in a separate file.  

# Other News

* Take the [ABRF DNA Sequencing Research Group survey](http://svy.mk/1cclCfs) on nucleic acid quantification methods & best practices 
* The first 30 minutes of the "[Galaxy Code and Storage Architecture](/Events/GCC2013/TrainingDay#galaxy-code-and-storage-architecture)" [GCC2013 Training Day](/Events/GCC2013/TrainingDay) session is now on  [Vimeo](https://vimeo.com/82179199) and [YouTube](http://youtu.be/kphYnONMOP8).  Thanks to [Samuel Lampa](http://saml.rilspace.org/blogs/samuel-lampa) for recording and making this available.
* [http://gmod.org/wiki/|GMOD](http://gmod.org/wiki/|GMOD) is looking into running a GMOD workshop in Malaysia, February 26-28 -- please email help@gmod.org if you're interested in attending.
* Aussie researchers can now launch their own Galaxy instance on Nectar. See the guide at the [Genome Virtual Lab](http://genome.edu.au/wiki/Get)
