---
title: December 2017 Galaxy News
---
[<img class="float-right" src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 />](/src/galaxy-updates/index.md)

Welcome to the December 2017 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# Events

There are [a plenitude of Galaxy related events coming up](/src/events/index.md) in the next few months:

## European Galaxy Administrator Workshop

[<img class="float-right" src="/src/images/logos/ElixirNoTextLogo.png" alt="Galaxy Admin training, this January, in Oslo " width="160" />](https://www.elixir-europe.org/events/european-galaxy-administrator-workshop)

**[Registration](https://skjema.uio.no/galaxy-admin-workshop-2018) opened in November for the [European Galaxy Administrator Workshop](https://www.elixir-europe.org/events/european-galaxy-administrator-workshop)**.  

* Mon 8 January 2018 to Fri 12 January 2018 CET
* Python room, Blindern, UiO campus, [Ole Johan Dahls hus](https://www.uio.no/english/about/getting-around/areas/gaustad/ga06/
), Gaustadalléen 23B, Oslo, 0373, Norway

Although located in Europe, and named to reflect this, the workshop is definitely **open to a world-wide audience.** The workshop is a result of the collaboration between the [ELIXIR Galaxy WG](https://www.elixir-europe.org/about/groups/galaxy-wg) and the Galaxy Project, and so far known, this is the only Galaxy Admin workshop planned for 2018. 

*Participants are to cover their own travel and accommodation, while the workshop participation is free of charge.*

The workshop is already close to full. If you are interested, you are encouraged to apply **right now**.

See the [full event description](https://www.elixir-europe.org/events/european-galaxy-administrator-workshop) for more.

## Galaxy (and GMOD) at Plant and Animal Genome XXVI

[<img class="float-left" src="/src/events/2018-pag/galaxy-at-pag-2018.png" alt="Galaxy at PAG XXVI" width="180" />](/src/events/2018-pag/index.md)

The [26th Plant and Animal Genome Conference](http://intlpag.org/) will be held January 13-17, in San Diego, California.  PAG is the largest ag-genomics meeting in the world and [Galaxy will be there](/src/events/2018-pag/index.md) to help researchers get the most from their data.  There will be a [Galaxy Workshop with 4 talks](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Session/4796), and a Galaxy Community Update talk during the [GMOD session](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Session/4806).  See the [Galaxy @ PAG page](/src/events/2018-pag/index.md) for details.

### GMOD @ PAG

<img class="float-right" src="/src/images/logos/gmod-in-san-diego-300.png" alt="GMOD at PAG XXVI" width="160" />

[GMOD](https://gmod.org/) (of which Galaxy is a part) has a bunch of things going on at PAG that are of interest to the Galaxy Community:

* GMOD will have a [project-wide hackathon on January 11-12](http://gmod.org/wiki/2018_PAG_Hackathon). 
* [Meg Staton](http://gst.tennessee.edu/people/meg-staton/) will [speak about](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Paper/28295) the Tripal-Galaxy bridge 
* There will be a [GMOD workshop](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Session/4806) on the last day of PAG.
* The [Tripal Project](https://tripal.info) will have a user meeting on January 11, and a hackathon on January 12. [Register here](https://docs.google.com/document/d/1klQwUqaT_KxPKIfGYy_3rtGnOMVhJxEoD9dZ4PGc4yg/edit).

If you are interested in supporting better integration of Galaxy with other GMOD components then please give the two hackathons a look.

### Pre-PAG Intro to Galaxy Workshop at CSUSM

[<img class="float-right" src="/src/images/logos/csusm-logo.gif" alt="Introduction to Galaxy Workshop, CSU San Marcos, January 12" width="160" />](/src/events/2018-01-csusm-workshop/index.md)

If you are at [Cal State San Marcos](https://csusm.edu/), or anywhere nearby, there will be an [Introduction to Galaxy Workshop](/src/events/2018-01-csusm-workshop/index.md) in San Marcos, California on Friday January 12.  The workshop is free, but space is limited, so you are encouraged to [register now](http://bit.ly/gxy-csusm-2018-reg).

## 2018 Big Genomics Data Skills Training course

[<img class="float-left" src="/src/images/logos/jax-logo-no-text-200.png" alt="2018 Big Genomics Data Skills Training course" width="160" />](https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors)

The [Jackson Lab (JAX)](https://www.jax.org/) is pleased to open the application period for the 2018 Big Genomics Data Skills Training course, funded by the NIH BD2K initiative: [Big Genomic Skills Training info and online application](https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors). The program provides training and hands-on experience working with genomic scale data and will enable participants to integrate genomic data analysis into their courses or launch new courses. Topics covered include RNAseq, Whole Exome Sequencing and variant detection, ChIPseq and network analysis. Participants will gain experience using Galaxy, R and Python.
 
As a grant funded activity housing is included; however you will need to cover your own travel.  If you teach at an HBCU or Minority-serving institution (by Dept. of Education criteria) JAX may be able to provide a modest travel award.
 
*The course is for faculty who primarily teach undergraduate students* and will be held at JAX Genomic Medicine in Farmington CT, May 21-25.

# All Upcoming events

<table class="table table-striped">
<thead>
<tr>
  <th>Date</th>
  <th>Topic/Event</th>
  <th>Venue/Location</th>
  <th>Contact</th>
</tr>
</thead>
<tbody>
<tr>
  <td><span class="text-nowrap">January 8th 2018</span></td>
  <td><a href="https://www.elixir-europe.org/events/european-galaxy-administrator-workshop">European Galaxy Administrator Workshop</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>Oslo, Norway</td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Instructors</td>
</tr>
<tr>
  <td><span class="text-nowrap">January 12th 2018</span></td>
  <td><a href="/events/2018-01-csusm-workshop">Introduction to Galaxy Workshop</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://csusm.edu/">Cal State University San Marcos, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Arun Sethuraman, Dave Clements</td>
</tr>
<tr>
  <td><span class="text-nowrap">January 13th 2018</span></td>
  <td><a href="/events/2018-pag">Plant &amp; Animal Genome XXVI (PAG 2018)</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://www.intlpag.org/">San Diego, California</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>See web page</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 12th 2018</span></td>
  <td><a href="http://www.triconference.com/Converged-IT-and-the-Cloud/#Day1">Supporting Highly Scalable Scientific Data Analysis with Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://www.triconference.com/">Converged IT &amp; the Cloud, San Francisco, California, United States</a></td>
  <td>James Taylor</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/RNA_Sequencing_2018.pdf">RNA Sequencing and Differential Expression</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 26th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/460-galaxy5">5th Galaxy high-throughput sequencing (HTS) data analysis workshop</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>University of Freiburg, Freiburg Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Freiburg Galaxy Team</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 7th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/469-introduction-to-rna-seq-data-analysis-with-galaxy">Introduction to RNA-Seq data analysis with Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>Am Botanischen Garten, Kiel, Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Andrea Bagnacani, Markus Wolfien</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/intro-galaxy-gvl/">Introduction to Galaxy and the GVL</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/rna-seq-dge-analysis-galaxy-gvl/#more-1720">RNA-Seq Differential Gene Expression Analysis using Galaxy and the GVL</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/Alignment_2018.pdf">Alignment, Visualisation and Variant Calling</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/variant-calling-galaxy-gvl/#more-1012">Variant Calling with Galaxy &amp; the GVL for beginners</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 22nd 2018</span></td>
  <td><a href="http://conf.abrf.org/sw2-galaxy-platform-accessible-core-laboratory-solution-proteogenomic-analysis-and-informatics">The Galaxy Platform as an Accessible, Core Laboratory Solution for Proteogenomic Analysis and Informatics</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://conf.abrf.org/">ABRF 2018, Myrtle Beach, South Carolina, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Pratik Jagtap</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors">Big Genomics Data Skills Training course</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://www.jax.org/about-us/locations/farmington">JAX Genomic Medicine, Farmington, Connecticut, United States</a></td>
  <td>Charlie Wray</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 25th 2018</span></td>
  <td><a href="https://gccbosc2018.sched.com">GCC2018 &amp; BOSC 2018</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="/events/gccbosc2018/venue/">Reed College, Portland, Oregon, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
</tbody>
</table>

See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

# New Galactic Blog Entries

[<img class="float-right" src="/src/images/galaxy-logos/galactic-blog-400.png" alt="The Galactic Blog is born" width="200" />](/src/blog/index.md)

We've got two new *[Galactic Blog](/src/blog/index.md)* entries:

* [Setup your own instance of the Galaxy Bioinformatics platform on DigitalOcean](https://vimal.io/setup-your-own-galaxy-bioinformatics-instance-on-digitalocean/), by [Vimalkumar Velayudhan](https://vimal.io/)
* [Galaxy Docker 17.09](/src/blog/2017-11-galaxy-docker-17.09/index.md), by [Björn Grüning](/src/people/bjoern-gruening/index.md)

The *Galactic Blog* is a place for longer-form posts that are relevant to Galaxy. (See [Galaxy News](/src/news/index.md) for shorter-form items.) 

## The Galactic Blog needs you!

The Galaxy community needs to know what you are doing.  If you have done something that others would benefit from knowing then please create a *Galactic Blog* post describing your experiences.

Blog posts can either be directly created on the Hub, or can be posted on your favorite blogging platform and linked to from the [Blog home page](/src/blog/index.md).  See [the CONTRIBUTING page](https://github.com/galaxyproject/galaxy-hub/blob/master/CONTRIBUTING.md#blog-metadata) for how.

Questions? [Ask us](mailto:outreach@galaxyproject.org).

# Publications

**140 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in November.

## Highlighted Publications

The *Galactic* and *Stellar* publications added in October were:

* [Using bio.tools to generate and annotate workbench tool descriptions](https://f1000research.com/articles/6-2074/v1), Kenzo-Hugo Hillion, Ivan Kuzmin, Anton Khodak, [E. Rasche](/src/people/helena-rasche/index.md), Michael Crusoe, Hedi Peterson, Jon Ison, Hervé Ménager. *F1000Research* 2017, 6(ELIXIR):2074 (doi: 10.12688/f1000research.12974.1)
* [Galaksio, a user friendly workflow-centric front end for Galaxy](http://journal.embnet.org/index.php/embnetjournal/article/view/897), Tomas Klingström, Rafael Hernández-de-Diego, Théo Collard, Erik Bongcam-Rudloff. *EMBnet.journal*, [S.l.], v. 23, p. e897, nov. 2017. ISSN 2226-6089.
  * [Data integration and handling](https://pub.epsilon.slu.se/14669/), Klingström, Tomas.  Doctoral thesis, Uppsala : Sveriges lantbruksuniv., Acta Universitatis agriculturae Sueciae, 1652-6880 ; 2017:99, ISBN 978-91-7760-088-6, eISBN 978-91-7760-089-3
* [TED toolkit: a comprehensive approach for convenient transcriptomic profiling as a clinically-oriented application](https://peerj.com/preprints/3385/), Ali T, Kim B, Lijeron C, Ogunwobi OO, Mazumder R, Krampis K. (2017). *PeerJ Preprints* 5:e3385v1 https://doi.org/10.7287/peerj.preprints.3385v1
* [An Accessible Proteogenomics Informatics Resource for Cancer Researchers](http://cancerres.aacrjournals.org/content/77/21/e43), Matthew C. Chambers, Pratik D. Jagtap, James E. Johnson, Thomas McGowan, Praveen Kumar, Getiria Onsongo, Candace R. Guerrero, Harald Barsnes, Marc Vaudel, Lennart Martens, [Björn Grüning](/src/people/bjoern-gruening/index.md), Ira R. Cooke, Mohammad Heydarian, Karen L. Reddy and Timothy J. Griffin.
*Cancer Research,* DOI: 10.1158/0008-5472.CAN-17-0331 Published November 2017
* [A Galaxy Implementation of Next-Generation Clustered Heatmaps for Interactive Exploration of Molecular Profiling Data](http://cancerres.aacrjournals.org/content/77/21/e23.article-info), Bradley M. Broom, Michael C. Ryan, Robert E. Brown, Futa Ikeda, Mark Stucky, David W. Kane, James Melott, Chris Wakefield, Tod D. Casasent, Rehan Akbani and John N. Weinstein. *Cancer Research,* DOI: 10.1158/0008-5472.CAN-17-0318 Published November 2017

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 83 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Methods) | | 29 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Workbench) | | 28 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UsePublic) | | 13 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseLocal) | 
| 12 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+RefPublic) | | 8 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Tools) | | 8 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseMain) | | 6 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+IsGalaxy) | 
| 3 | [+Unknown](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Unknown) | | 3 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Reproducibility) | | 3 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Shared) | | 2 | [+Other](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Other) | 
| 1 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+HowTo) | | 1 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Cloud) | | | | | | | |

----
# Who's Hiring

<div class='right'><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Développement de scripts R pour la mise en place d’une stratégie de traitement de données métabolomiques, et interfaçage avec la plate-forme web Galaxy](https://www.sfbi.fr/content/d%C3%A9veloppement-de-scripts-r-pour-la-mise-en-place-d%E2%80%99une-strat%C3%A9gie-de-traitement-de-donn%C3%A9es--0), Plateforme Metabolomique BioMeT, Marseille, France
* [Classification des micropolluants dans l'environnement : amélioration de l'algorithme online TyPol](https://www.sfbi.fr/content/classification-des-micropolluants-dans-lenvironnement-am%C3%A9lioration-de-lalgorithme-online-typ), INRA-LBE, Narbonne, France
* [Bioinformaticien immunologie](https://www.sfbi.fr/content/bioinformaticien-immunologie), Gustave Roussy Institute, Villejuif, France
* [Bioinformaticien Gustave Roussy](https://www.sfbi.fr/content/bioinformaticien-gustave-roussy), Villejuif, France
* The [Blankenberg](/src/people/dan/index.md) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/src/news/2017-09-hiring-at-cc/index.md).
* [Galaxy Project is hiring software engineers and postdocs at Johns Hopkins](/src/news/2017-08-hiring-at-hopkins/index.md), Baltimore, Maryland, United States

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them in November.

## MiModD 0.1.8 Released

And it's available on the [MiModD NacreousMap](http://mapping-by-sequencing.vm.uni-freiburg.de:8080/) public Galaxy Server.  MiModD is a comprehensive software package for the identification and annotation of mutations in the genomes of model organisms from whole-genome sequencing (WGS) data.

## Public Servers in November's Publications

As part of the move to Zotero, we have started tagging papers that use, mention, implement or extend public Galaxy Servers with the server name.  Here are the counts for November's papers.

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 15 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Huttenhower) | | 9 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RepeatExplorer) | | 5 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Galaxy-P) | | 2 | [>Orione](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Orione) | 
| 2 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Workflow4Metabolomics) | | 1 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Cistrome) | | 1 | [>AGEseq](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>AGEseq) | | 1 | [>ARGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>ARGalaxy) | 
| 1 | [>GIO](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GIO) | | | | | | | | | | |


----

# Tools

## ToolShed Contributions

<div class='right'>[<img src="/src/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in November](/src/toolshed/contributions/2017-11/index.md).

# Releases

## 17.09 Galaxy Release

<div class="right">
<a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a><br /><br />
</div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.09 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html)** at the end of October.

Highlights include

* **Singularity** Tool execution using the HPC-friendly container technology Singularity is now supported. 
* **Download entire collection** Downloading whole collections is now possible from the history interface. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Switch tool versions in workflows** You can now select exactly what version of tool you want to use when building workflows. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Security patches**
* Several features were **deprecated**

See the [17.09 release announcement](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html) for details.

### Galaxy Docker Image 17.09

The [Galaxy Docker project has seen a matching release](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.09), for Galaxy 17.05. Major features include
* much improved documentation about using Galaxy Docker and an external cluster ([@rhpvorderman](https://github.com/rhpvorderman))
* CVMFS support - mounting in 4TB of pre-build reference data ([@chambm](https://github.com/chambm))
* Singularity support and tests (compose only)
* more work on K8s support and testing ([@jmchilton](https://github.com/jmchilton))
* using .env files to configure the compose setup for SLURM, Condor, K8s, SLURM-Singularity, Condor-Docker

And

* The [Galaxy Docker Project](https://github.com/bgruening/docker-galaxy-stable) has reached more than 31k downloads on [Dockerhub](https://hub.docker.com/r/bgruening/galaxy-stable/) - not counting quay.io and all flavors 

## galaxy-lib 17.9.10

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

This revision:

* Added docs for using mulled-build with your own quay.io account (thanks to @jerowe).
* Catch errors in Conda search if nothing is found (preventing planemo-monitor from functioning properly) (thanks to @bgruening).
* Make multi-requirement container building via mulled more stable (thanks to @bgruening).

### Planemo 0.47.0

[<img class="float-right" src="/src/images/galaxy-logos/planemo-logo.png" alt="Planemo" width="200" />](https://pypi.python.org/pypi/planemo/)

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

## Earlier Releases

Other packages that have been released in the prior 4 months.

### sequence_utils 1.1.2

Galaxy's [sequence utilities](https://github.com/galaxyproject/sequence_utils) are a set of Python modules for reading, analyzing, and converting sequence formats.  See the [release notes](https://github.com/galaxyproject/sequence_utils/blob/master/HISTORY.rst) for what's new this month.

### StarForge 0.3.1-5

[StarForge](https://github.com/galaxyproject/starforge) help build Galaxy things in virtualization:

* Build Galaxy Tool Shed dependencies
* Build Python Wheels (e.g. for the Galaxy Wheels Server)
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.  StarForge has had several updates this fall.  Fixes and new features include:

* Support xz/lzma tarballs for wheel builds Pull Request 166
* Native support for auditwheel and delocate. (#160)
* Do not build sdists with the wheel subcommand by default. (#155)
* Fix a bug where the wrong working directory was set when building wheels with multiple sources. (#154)
* Fix a bug with sudo and brew install on macOS. (#151).
* Short circuit platform caching on OS X (#150).

### BioBlend 0.10.0

[BioBlend](http://bioblend.readthedocs.org/) is a Python library for interacting with CloudMan and Galaxy‘s API.  BioBlend makes it possible to script and automate the process of cloud infrastructure provisioning and scaling via CloudMan, and running of analyses via Galaxy.

See the [release notes](https://github.com/galaxyproject/bioblend/releases) for what's new in release 0.10.0.


### Pulsar 0.8.0

[<img class="float-right" src="/src/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" />](https://pypi.python.org/pypi/pulsar-app/)

A [Pulsar](https://pypi.python.org/pypi/pulsar-app/) update was released in September.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

[This release](https://github.com/galaxyproject/pulsar/blob/master/HISTORY.rst) adds several features and 4 bug fixes.

### nebulizer 0.4.2

[Command-line utilities](https://pypi.python.org/pypi/nebulizer/) to help with managing users, data libraries and tools in a Galaxy instance, using the Galaxy API via the BioBlend library.

- [Documentation](https://nebulizer.readthedocs.io)
- [Code](https://github.com/pjbriggs/nebulizer)

*CAVEAT nebulizer is still a work in progress; please exercise caution when attempting irreversible operations especially against production Galaxy instances (for example when creating users or data libraries).*

### CloudBridge 0.3.3

[CloudBridge](https://github.com/gvlproject/cloudbridge) aims to provide a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud. It is currently under development and is in an Alpha state.  

## And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

