---
title: 'March 2019 Galactic News'
tease: 'GalaxyAdmins, GCC2019 Childcare sponsors, Platforms, Pubs, Jobs, Blog!, Training, Tools, Releases and more!'
date: "2019-03-04"
---

[<img class="float-right" src="/src/news/2019-03-galaxy-update/2019-03-01-galaxy-news-image.png" alt="Galaxy News" width="350" />](/src/galaxy-updates/index.md>)

The *March  2019 Galactic News* is here! This is a summary of what is going on in the Galaxy community.

* Event News
    * **[eLife and F1000Research sponsor childcare at GCC2019](/src/news/2019-03-galaxy-update/index.md#gcc2019-sponsors)**
    * **[The GalaxyAdmins](/src/news/2019-03-galaxy-update/index.md#galaxyadmins)** meetup is back March 21!
* [184 new publications](/src/news/2019-03-galaxy-update/index.md#publications), great resources lead to great insight. 
* Some most excellent [Galaxy Platform News](/src/news/2019-03-galaxy-update/index.md#galaxy-platforms-news), including metagenomics, predictive model building, epigenome-wide association studies, immunogenomics, DNA methylation analysis, and more!
* An interview on [The Galactic Blog](/src/news/2019-03-galaxy-update/index.md#new-galactic-blog-post) with [Nate Coraor](/src/people/nate/index.md) about CVMFS & Galaxy. 
* At least [15 Open positions](/src/news/2019-03-galaxy-update/index.md#whos-hiring) in five countries on two continents. 
* [A burst of Training Updates](/src/news/2019-03-galaxy-update/index.md#doc-hub-and-training-updates).
* [ToolShed contributions](/src/news/2019-03-galaxy-update/index.md#toolshed-contributions).
* [Galaxy 19.01 released](/src/news/2019-03-galaxy-update/index.md#releases)!
* And [a bunch of other news](/src/news/2019-03-galaxy-update/index.md#other-news) too.

If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# GCC2019 Sponsors

We are very pleased to announce that **[eLife](https://elifesciences.org/)** and **[F1000Research](https://f1000research.com/)** are first two [sponsors for GCC2019](/src/events/gcc2019/sponsors/index.md). *And they are both sponsoring childcare.*  **Yes, there will be childcare at GCC2019!**

## eLife

[<img class="float-right img-fluid mx-auto" style="max-width: 240px" src="/src/events/gccbosc2018/sponsors/elife-logo-wide.jpg" alt="eLife" />](https://elifesciences.org/) 

***[eLife](https://elifesciences.org/)*** is a non-profit open access journal, publishing research from across the life and biomedical sciences. We also invest heavily in software development, new product design, collaboration and outreach. This includes the development of open-source tools, with extensible capabilities, that can be used, adopted and modified by any interested party to help move towards an ecosystem that serves science and scientists as efficiently and as cost-effectively as possible.

Have you got an idea that will bring cutting-edge technology to open research communication? We are here to help. 

eLife’s **Innovation Initiative** was set up to identify and support innovative, small-to-medium-scale open-source projects aiming to improve the sharing and reuse of scientific discoveries. The primary outputs of the Initiative are open tools, technologies and processes aimed at improving the discovery, sharing, consumption and evaluation of important scientific research.

Through this Initiative, we’re always on the lookout for opportunities to engage with the best emerging talent and ideas at the interface of research and technology. You can find out more about some of these engagements [on eLife Labs](https://elifesciences.org/labs), or contact our Innovation Community Manager for more information (innovation@elifesciences.org).

## The Galaxy Gateway on F1000Research

[<img class="float-right img-fluid mx-auto" style="max-width: 200px" src="/src/events/gccbosc2018/sponsors/f1000-logo-wide.png" alt="F1000Research: Open for Science" />](https://f1000research.com/) 

[F1000Research](https://f1000research.com/) is proud this year to be sponsoring childcare for GCC2019, which we feel is vital for inclusivity. F1000Research is an Open Access publishing platform for the life sciences that publishes a range of different article types, from traditional research articles, to method articles and software tools. The platform is particularly well suited to publishing software tool articles as our versioning system allows small updates at any time, we have proper syntax highlighting and we can support LaTeX and markdown submissions.

Our [Galaxy Gateway](https://f1000research.com/gateways/galaxy) on the platform brings together articles, posters and slides from Galaxy events. The Gateway allows authors to quickly share their work with the community, boosting visibility and giving researchers proper credit for their contribution. 

We’re pleased to be running a call for papers for GCC2019 and **would like to invite the Galaxy community to submit their articles to the [Galaxy Gateway](https://f1000research.com/gateways/galaxy) by 20<sup>th</sup> July 2019.** These can be any type of article, such as a new Galaxy workflow or an update to an existing method or software tool. This is a great opportunity to shine a light on your research, improve its impact and be credited with an academic paper for your work. Please submit via the Galaxy gateway and if you have any questions at all do not hesitate to get in touch at publishers@f1000.com.

## Your Organization/Vendor Here!

[GCC2019](/src/events/gcc2019/index.md) is looking for [sponsors](/src/events/gcc2019/sponsors/index.md)!  If your organization wants to help put on the premier Galaxy event of the year, then please contact the [organizers](/src/events/gcc2019/sponsors/index.md)). And please encourage your vendors to consider sponsoring as well.  Sponsors are potential partners for participants and their contributions make GCC affordable (and maybe even possible).

# GalaxyAdmins 

<div class="float-right">
[<img class="img-fluid mx-auto" src="/src/images/galaxy-logos/GalaxyAdmins_64.png" alt="GalaxyAdmins" />](/src/community/galaxy-admins/index.md) <br /><br />
<img class="img-fluid mx-auto" src="/src/images/logos/kubernetes-logo-text-wide.png" alt="Kubernetes" style="max-width: 160px" />
</div>

After a two year hiatus, the [online GalaxyAdmins meetups](/src/community/galaxy-admins/meetups/index.md) will return on [March 21](/src/community/galaxy-admins/meetups/2019-03-21/index.md), with a community discussion, and a presentation from [Pablo Moreno](/src/blog/2018-08-cotm-pablo-moreno/index.md) on *Running Galaxy on Kubernetes*, online at 15:00 GMT (see [your local time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=March+2019+GalaxyAdmins+Meetup&iso=20190321T15&p1=1234&ah=1)).  See the [meetup page for links](/src/community/galaxy-admins/meetups/index.md)).

[GalaxyAdmins](/src/community/galaxy-admins/index.md) is a discussion group for Galaxy community members who are responsible for Galaxy installations. Our online meetups are around an hour long and feature a presentation followed by an open discussion.  It's a great place to catch up on what your fellow admins are thinking about.

# Upcoming events

These and [other Galaxy related events](/src/events/index.md) are coming up:

<table class="table table-striped">
<thead>
  <tr>
      <th>Date</th>
      <th>Topic/Event</th>
      <th>Venue/Location</th>
      <th>Contact</th>    
    </tr>
</thead>
<tbody>
  <tr>
       <td><span class="text-nowrap">March 6th 2019</span></td>
       <td><a href="https://www.denbi.de/training/532-galaxy-for-linking-bisulfite-sequencing-with-rna-sequencing">Galaxy for linking bisulfite sequencing with RNA sequencing</a></td>
       <td><img style="float:right;" src="/images/icons/EU.png"/>University of Rostock, Rostock, Germany</td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>destair [at] leibniz-fli.de</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">March 18th 2019</span></td>
       <td><a href="http://meetings.ipk-gatersleben.de/grc2019-abc/">Gatersleben Research Conference: Applied Bioinformatics for Crops</a></td>
       <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://meetings.ipk-gatersleben.de/grc2019-abc/accomodation/">Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany</a></td>
       <td>Björn Grüning</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">March 20th 2019</span></td>
       <td><a href="http://meetings.ipk-gatersleben.de/compbiostarter/index.php/program/">Introduction into Galaxy and how to design your own analysis workflow</a></td>
       <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://meetings.ipk-gatersleben.de/compbiostarter/">Computational Biology Starter, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany</a></td>
       <td>Anne Fiebig</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">March 21st 2019</span></td>
       <td><a href="/community/galaxy-admins/meetups/2019-03-21/">GalaxyAdmins February 2019 Meetup</a></td>
       <td><img style="float:right;" src="/images/icons/GL.png"/>Online</td>
       <td>Helena Rasche, Dave Clements</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">March 21st 2019</span></td>
       <td><a href="http://artbio.fr/?page_id=341">RNAseq analysis with Galaxy</a></td>
       <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.google.com/maps/d/viewer?mid=1hOogl-2tKCtwgsyf0v7ztigLBU0&amp;ll=48.847069261929576%2C2.3571523499999785&amp;z=17">Sorbonne Université, Paris, France</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>https://gitter.im/ARTbio/Hotline</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 1st 2019</span></td>
       <td><a href="https://qfab.org/training#NGS_Workshops">Getting started with Galaxy and NGS Platforms</a></td>
       <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 2nd 2019</span></td>
       <td><a href="https://qfab.org/training#NGS_Workshops">Genome assembly using Galaxy</a></td>
       <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 3rd 2019</span></td>
       <td><a href="https://qfab.org/training#NGS_Workshops">Variant detection using Galaxy</a></td>
       <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 4th 2019</span></td>
       <td><a href="https://qfab.org/training#NGS_Workshops">RNA Seq analysis using Galaxy </a></td>
       <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 5th 2019</span></td>
       <td><a href="https://qfab.org/training#NGS_Workshops">Metagenomics analysis using Galaxy </a></td>
       <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 15th 2019</span></td>
       <td><a href="https://faes.org/content/advanced-studies-bioinformatics-and-data-science">Genome assembly and annotation using Galaxy</a></td>
       <td><img style="float:right;" src="/images/icons/NA.png"/>National Institutes of Health (NIH), Bethesda, Maryland, United States</td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>mo@galaxyproject.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 23rd 2019</span></td>
       <td><a href="https://www.france-bioinformatique.fr/fr/evenements/elixir-galaxy-community-workshop-roscofffrance">Elixir Galaxy Community - Workshop at Roscoff/France</a></td>
       <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.google.com/maps?ll=48.726635,-3.987479&amp;z=14&amp;t=m&amp;hl=en&amp;gl=US&amp;mapclient=embed&amp;cid=7841494204688338171">Station Biological De Roscoff, Roscoff, France</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Gildas Le Corguillé</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">April 29th 2019</span></td>
       <td><a href="https://ittraining.iu.edu/explore-topics/titles/denovo/index.html">de novo assembly of transcriptomes using HPC resources</a></td>
       <td><img style="float:right;" src="/images/icons/NA.png"/>Indiana University, Bloomington, Indiana, United States</td>
       <td>help@ncgas.org</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">May 13th 2019</span></td>
       <td><a href="http://sciencegateways.org/bootcamp">Science Gateways Bootcamp</a></td>
       <td><img style="float:right;" src="/images/icons/NA.png"/>Indianapolis, Indiana, United States'</td>
       <td>help@sciencegateways.org.</td>
  </tr>
  <tr>
       <td><span class="text-nowrap">July 1st 2019</span></td>
       <td><a href="/events/gcc2019/">2019 Galaxy Community Conference (GCC2019)</a></td>
       <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.konzerthaus.freiburg.de/">Freiburg, Germany</a></td>
       <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
  </tr>
</tbody>
</table>

----

# Publications

**184 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in February.

## Highlighted Publications

*[Galactic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/%2BGalactic)* and *[Stellar](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/%2BStellar)* publications from February:

* **[Développement d’outils et de méthodes pour l’analyse de données épigénomiques](https://savoirs.usherbrooke.ca/handle/11143/14918?show=full)**, Jonathan Laperle, M. Sc thesis, Université de Sherbrooke, 2019
* **[GCAC: galaxy workflow system for predictive model building for virtual screening](https://doi.org/10.1186/s12859-018-2492-8)**, Deepak R. Bharti, Anmol J. Hemrom and Andrew M. Lynn. *BMC Bioinformatics* 2019 19 (Suppl 13) :550 doi:10.1186/s12859-018-2492-8
* **[EWAS-Galaxy: a tools suite for population epigeneticsintegrated into Galaxy](https://doi.org/10.1101/553784 )**, Katarzyna Murat, [Björn Grüning](/src/people/bjoern-gruening/index.md), Paulina Wiktoria Poterlowicz, Gillian Westgate, Desmond J Tobin, *Krzysztof Poterlowicz. *bioRxiv* 553784; doi:doi.org/10.1101/553784 "
* **[Statistical Mitogenome Assembly with Repeats](https://dna.engr.uconn.edu/bibtexmngr/upload/AM18.pdf)**, Fahad Alqahtani and Ion I. Mandoiu. 2018 IEEE 8th International Conference on Computational Advances in Bio and Medical Sciences (ICCABS)
* **[SMAGEXP: a galaxy tool suite for transcriptomics data meta-analysis](https://academic.oup.com/gigascience/article/8/2/giy167/5304063)**, Samuel Blanck and Guillemette Marot. *GigaScience*, Volume 8, Issue 2, 1 February 2019, giy167, doi:10.1093/gigascience/giy167
* **[Enrichment of phosphate solubilizing bacteria during late developmental stages of eggplant (*Solanum melongena* L.)](https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiz023/5319216)**, Huixiu Li, Xiaoyan Ding, Chen Chen, Xiangnan Zheng, Hui Han, Chennan Li, Jingyang Gong, Ting Xu, Qing X Li, Guo-chun Ding, Ji Li. *FEMS Microbiology Ecology*, fiz023, doi:10.1093/femsec/fiz023
* **[Optimized Storing of Workflow Outputs through Mining Association Rules](https://doi.org/10.1109/BigData.2018.8622351)**,  Debasish Chakroborti, Manishankar Mondal, Banani Roy, Chanchal K. Roy, Kevin A. Schneider.  2018 IEEE International Conference on Big Data (Big Data), doi:10.1109/BigData.2018.8622351
* **[Biomolecular Reaction & Interaction Dynamics Global Environment (BRIDGE)](https://doi.org/10.1093/bioinformatics/btz107)**, Tharindu Senapathi, Simon Bray, Christopher B. Barnett, [Björn Grüning](/src/people/bjoern-gruening/index.md), Kevin J Naidoo. *Bioinformatics*, btz107, doi:10.1093/bioinformatics/btz107

Five of the eight highlighted publications are open access.

## Publication Topics

### New Tag: Education

We added a new tag to the [Galaxy Publication Library](/src/publication-library/index.md) this month: *[Education](https://www.zotero.org/groups/1732893/galaxy/items/tag/%2BEducation)*.  This tag highlights papers that focus on training, teaching, and/or education in general.  Galaxy is well suited for teaching bioinformatics and this has been reflected in publications for several years.  There are three new pubs this month about education. We also selectively back-curated several papers.  *Education* is the first new tag added to the library in six years.

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 124 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Methods) | | 50 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UsePublic) | | 27 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Workbench) | | 26 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseMain) | 
| 16 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseLocal) | | 12 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+RefPublic) | | 8 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Tools) | | 7 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+IsGalaxy) | 
| 7 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Reproducibility) | | 4 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Cloud) | | 3 | [**+Education**](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Education) | | 2 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+HowTo) | 
| 2 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Shared) | | 1 | [+Other](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Other) | | | | | | | |

----

# Galaxy Platforms News

The [Galaxy Platform Directory](/src/use/index.md) lists resources for easily running your analysis on Galaxy, including publicly available servers, cloud services, and containers and VMs that run Galaxy.  A lot was added in February:

## FreeBioInfo

[<img class="float-left" style="max-width: 400px" src="/src/use/freebioinfo/freebioinfologo.jpg" alt="FreeBioInfo" />](/src/use/freebioinfo/index.md)

[FreeBioinfo](http://www.freebioinfo.org/) is a public server for metagenomic analysis.  The server supports a wide range of metagenomics and other tools. Ther is a [Cookbook](http://www.freebioinfo.org/galaxy/static/protocol/galaxyhelphuixiuli20170425.pdf) for the site (although it is in Chinese). FreeBioInfo is hosted at the [China Agricultural University](https://cau.edu.cn/), in Beijing, China.

[<img class="float-right" style="max-width: 300px" src="/src/use/gcac/gcac-landing.png" alt="GCAC" />](/src/use/gcac/index.md)

## GCAC

[GCAC : Galaxy for Compound Activity Classification](https://github.com/LynnLab-JNU/gcac-galaxy-tools) is available as a [public server](http://ccbb.jnu.ac.in/gcac), as a [virtual machine](https://drive.google.com/file/d/1ZQM5x-9J49rvPy9SoWOrM3Xm1MHZdr1I/view?usp=sharing), and as a [tool suite in the Galaxy Toolshed](https://toolshed.g2.bx.psu.edu/repository?repository_id=351af44ceb587e54). GCAC is a web enabled paltform for predictive model building using R and caret package for virtual screening. GCAC is a one stop solution for data generation, data preparation, feature selection, model building, prediction and retrieval of potential lead molecules. The [GCAC Manual](https://drive.google.com/open?id=1eWJT05zSFxFNwjaTjjPFE0zZp6g7fO9mxutRtSbc9zU) describes how to use the service.  GCAC is supported by [Jawaharlal Nehru University](https://www.jnu.ac.in/main/).

## EWAS-Galaxy

[EWAS-Galaxy](https://github.com/kpbioteam/docker_ewas) is a Docker image for population epigenetics data analysis. Epigenome-wide association studies (EWAS) analyse genome-wide activity of epigenetic marks in cohorts of different individuals to find associations between epigenetic variation and phenotype. EWAS-Galaxy is supported by the [University of Bradford](https://bradford.ac.uk/external/).

[<img class="float-right" style="max-width: 300px" src="/src/use/mandoiu/mandoiu-landing-500.png" alt="Mandoiu Lab" />](/src/use/mandoiu/index.md)

## Mandoiu Lab

The [Mandoiu Lab Galaxy server](https://neo.engr.uconn.edu/) is a customized installation of the Galaxy framework deploying bioinformatic tools and pipelines developed in the [Mandoiu Lab](http://dna.engr.uconn.edu/) at the [University of Connecticut](http://www.uconn.edu/) for high-throughput sequencing and immunogenomics data analysis.  Source code and relevant publications for these and other tools developed in the lab are available on the [Mandoiu lab software page](https://dna.engr.uconn.edu/?page_id=40).  

## DocMethyl 

[DocMethyl EpiMOLAS](https://hub.docker.com/r/lsbnb/docmethyl) is Docker image  for DNA methylation analysis in WGBS. This Galaxy docker container have tested with demo dataset under the Ubuntu 16.04 64 bit system equipped with four-core CPU, 32GB RAM, 400GB storage.  The image is maintained by the [Institute of Information Science, Academia Sinica](http://eln.iis.sinica.edu.tw), in Taipei, Taiwan.

## Trinity

[<img class="float-left" style="max-width: 300px" src="/src/use/trinity-ctat/trinity-pti-logo.jpg" alt="Trinity CTAT" />](/src/use/trinity-ctat/index.md)

[Trinity CTAT Galaxy](https://galaxy.ncgas-trinity.indiana.edu/) is a [public server](https://galaxy.ncgas-trinity.indiana.edu/) and [Docker container](https://github.com/NCIP/Trinity_CTAT_galaxy/wiki/Intro) providing users with access to large memory resources required for *de novo* transcriptome assembly and downstream analysis.  Funded by the National Cancer Institute, the Trinity Cancer Transcriptome Analysis Toolkit provides a suite of software for the analysis of cancer transcriptome data. The nature of cancer data requires different assumptions and approaches to analysis, above and beyond the normal challenges of dealing with RNA-seq. View [more information on the tools available here](https://github.com/NCIP/Trinity_CTAT/wiki). The resource has slides available for tasks such as moving data, and help can be reached at trinity@ncgas.org. There are no quotas, however, there is an aggressive purge policy for the disk usage. Files over 60 days old are removed by the system. An account is needed to access the resource and users are asked to fill out a registration form about your research (this helps us stay funded).

## Galaxy Platforms in Publications

We tag papers that use, mention, implement or extend public Galaxy platforms (servers, services, clouds, containers...).  Here are the counts for the past month's publications:

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 21 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Huttenhower) | | 6 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RepeatExplorer) | | 5 | [>APOSTL](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>APOSTL) | | 5 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Workflow4Metabolomics) | 
| 3 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Cistrome) | | 2 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Galaxy-P) | | 2 | [>Globus Genomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Globus Genomics) | | 1 | [>ABiMS](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>ABiMS) | 
| 1 | [>CPT](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>CPT) | | 1 | [>epiGeEC](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>epiGeEC) | | 1 | [>EWAS-Galaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>EWAS-Galaxy) | | 1 | [>FreeBioinfo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>FreeBioinfo) | 
| 1 | [>G-OnRamp](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>G-OnRamp) | | 1 | [>GalaxEast](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GalaxEast) | | 1 | [>GCAC](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GCAC) | | 1 | [>GVL-Unspecified](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GVL-Unspecified) | 
| 1 | [>Langille](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Langille) | | 1 | [>Mandoiu](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Mandoiu) | | 1 | [>MGEScan](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>MGEScan) | | 1 | [>Nebula](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Nebula) | 
| 1 | [>Palfinder](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Palfinder) | | 1 | [>Pasteur](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Pasteur) | | 1 | [>PhenoMeNal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PhenoMeNal) | | 1 | [>RiboGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RiboGalaxy) | 
| 1 | [>Sciensano](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Sciensano) | | 1 | [>SMAGEXP](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>SMAGEXP) | | 1 | [>UseGalaxy.eu](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>UseGalaxy.eu) | | 1 | [>VirAmp](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>VirAmp) | 
| 1 | [>ZBIT](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>ZBIT) | | | | | | | | | | |


----

[<img class="float-right" src="/src/images/galaxy-logos/galactic-blog-400.png" alt="Galactic Blog" style="max-width: 220px" />](/src/blog/index.md)

# New Galactic Blog Post

This month we have a *[Galactic Blog](/src/blog/index.md)* post where [Nate Coraor was interviewed](https://galaxyproject.org/blog/2019-02-cvmfs/) about what CVMFS actually is and what it does for Galaxy. 

----

# Who's Hiring

<div class='float-right'><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The *dark energy* of irreproducible research is threatening the science universe! Please help the Galaxy push it back!

* [Cloud Engineer V](https://careers-fhcrc.icims.com/jobs/12290/cloud-engineer-v/job?mobile=false&width=960&height=500&bga=true&needsRedirect=false&jan1offset=-480&jun1offset=-420), Fred Hutchinson Cancer Research Center, Seattle, Washington, United States. 
* ELIXIR Belgium has **four Galaxy-related openings** in Ghent:
  * [ELIXIR Open Science Community Manager](https://vibvzw.jobsoid.com/j/17164/elixir-open-science-community-manager), VIB-UGent Center for Plant Systems Biology
  * [ELIXIR Software developer data management tools](https://vibvzw.jobsoid.com/jobs/35bba14f-79b7-4388-b015-a4fb9d97ee41/17168/software-developer-data-management-tools), VIB-UGent Center for Plant Systems Biology
  * [ELIXIR Bioinformatics Trainer](https://vibvzw.jobsoid.com/j/17026/elixir-bioinformatics-trainer), VIB Bioinformatics Core
  * [ELIXIR Scientific Cloud Coordinator](https://vibvzw.jobsoid.com/j/17165/elixir-scientific-cloud-coordinator), VIB-UGent Center for Plant Systems Biology
* [Software Developer](https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25240&siteid=5341&PageType=JobDetails&jobid=1433370#jobDetails=1433370_5341), Harvard T.H. Chan School of Public Health, Boston, Massachusetts, United States. "Basic automated analysis workflows using Galaxy for 16S marker gene, metagenomic, and metatranscriptomic data leveraging existing software." 
* [Cloud Computing Bioinformatics Programmer](https://bioinformatics.ca/job-postings/0fd00bb0-9f21-11e8-a370-3ba76904a08d/#/?&order=desc) working with [IRIDA](http://www.irida.ca/), Simon Fraser University, Vancouver, Canada
* [Senior IT DevOps Data Engineer](https://adaptimmunellc.applytojob.com/apply/jobs/details/OsmMz9jkwv?) and [Senior IT Data Engineer (IT Bioinformatician)](https://adaptimmunellc.applytojob.com/apply/jobs/details/W1CcPTXteJ?),  Adaptimmune, Abingdon, Oxfordshire, United Kingdom. *Required: Experience with following bioinformatic pipeline tools: Galaxy…*
* [Specialist Solutions Architect -- Biological Sciences](https://www.amazon.jobs/en/jobs/728182/specialist-solutions-architect-biological-sciences), Amazon Web Services, United States
* [Principal Technical Business Development Manager: AWS Research in Biomedical](https://www.amazon.jobs/en/jobs/745693/principal-technical-business-development-manager-aws-research-in-biomedical), Amazon Web Services, United States
* [Bioinformatics Scientist](https://www.indeed.com/viewjob?jk=5288546935e35605), ResearchDx, Irvine, California, United States
* [NGS Scientific Applications Specialist](https://careers-idtdna.icims.com/jobs/4112/ngs-scientific-applications-specialist/job), Integrated DNA technologies, Iowa City, Iowa, United States
* The [The European Galaxy Team has open positions](https://usegalaxy-eu.github.io/posts/2019/01/10/openpositions/), Freiburg, Germany
  * Software engineer, system analysts/administrators, data analyst, and a comunnity and/or research manager
* The [Blankenberg](/src/people/dan/index.md) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/src/news/2017-09-hiring-at-cc/index.md).
* [Galaxy Project is hiring two software engineers at Johns Hopkins University](/src/news/2017-08-hiring-at-hopkins/index.md), Baltimore, Maryland, United States.

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

---

# Doc, Hub, and Training Updates

Updates from the [Galaxy Training Materials](https://galaxyproject.github.io/training-material/) in February, and the Hub as well:

## GTN Training Materials

[<img class="float-right img-responsive" src="/src/images/galaxy-logos/GTNLogoTrans300.png" alt="Galaxy Training Materials" />](https://galaxyproject.github.io/training-material/)

* New [Plates, Batches, and Barcodes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html#1) slides by [Mehmet Tekman](https://github.com/mtekman).
* New [Understanding Barcodes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html) tutorial by [Mehmet Tekman](https://github.com/mtekman).
* New [Mass spectrometry imaging: Examining the spatial distribution of analytes](https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/msi-analyte-distribution/tutorial.html) tutorial by [Melanie Föll](https://github.com/foellmelanie).
* New [Pre-processing of Single-Cell RNA Data](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.html) tutorial by [Mehmet Tekman](https://github.com/mtekman).
* Addition of [data-library.yaml file](https://github.com/galaxyproject/training-material/pull/1255) for describing sample data of tutorials by [Bert Droesbeke](https://github.com/bedroesb).
* Update to [Visualization of RNA-Seq results with CummeRbund](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.html) by [Pavankumar Videm](https://github.com/pavanvidem).
* Update to [Making sense of a newly assembled genome](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/ecoli_comparison/tutorial.html) by [Ekaterina Polkh](https://github.com/polkhe).
* Updates to [Reference-based RNA-Seq data analysis](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html) by [Andrea Bagnacani](https://github.com/bagnacan).
* Update to [De Bruijn Graph Assembly](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/debruijn-graph-assembly/tutorial.html), by [Ekaterina Polkh](https://github.com/polkhe).
* Update to [CLIP-Seq data analysis from pre-processing to motif detection](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/clipseq/tutorial.html) by [Florian-H-Lab](https://github.com/Florian-H-Lab).
* Update to [GO Enrichment Analysis](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/goenrichment/tutorial.html) by [Daniel Sobral](https://github.com/dsobral).
* Updates to [Mapping and molecular identification of phenotype-causing mutations](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.html) and [Analysis of metagenomics data - The global picture](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/general-tutorial/tutorial.html) by [Anna Syme](https://github.com/AnnaSyme).

There was a tremendous amount of content added to the GTN while our fearless instructors of the [2019 Galaxy Admin Training](/src/events/2019-admin-training/index.md) faced below freezing temperatures in the unforgiving mountains of Pennsylvania. A _warm_ thanks to [Helena Rasche](https://github.com/erasche), [John Chilton](https://github.com/jmchilton), [Marten Cech](https://github.com/martenson), [Nate Coraor](https://github.com/natefoo), and [Simon Gladman](https://github.com/Slugger70) for adding to the growing Galaxy Server Administration section of the GTN. See these training materials below:
* New [Ansible for installing Galaxy](https://training.galaxyproject.org/training-material/topics/admin/tutorials/ansible/tutorial.html) tutorial.
* New [Galaxy Tool Management](https://training.galaxyproject.org/training-material/topics/admin/tutorials/tool-management/tutorial.html) tutorial and slides.
* New [Reference Data with CVMFS](https://training.galaxyproject.org/training-material/topics/admin/tutorials/cvmfs/tutorial.html) tutorial.
* New [Upstream Authentication](https://github.com/jmchilton) tutorial. 
* New [Empathy](https://training.galaxyproject.org/training-material/topics/admin/tutorials/empathy/slides.html#1) for admins slides.
* Update to [Docker and Galaxy](https://training.galaxyproject.org/training-material/topics/admin/tutorials/galaxy-docker/slides.html) slides.
* Update to [Connecting Galaxy to a compute cluster](https://training.galaxyproject.org/training-material/topics/admin/tutorials/connect-to-compute-cluster/tutorial.html) tutorial and slides.
* Update to [Server Maintenance](https://training.galaxyproject.org/training-material/topics/admin/tutorials/maintenance/slides.html#1) slides.
* Update to [Galaxy Reports](https://training.galaxyproject.org/training-material/topics/admin/tutorials/reports/tutorial.html) tutorial.
* Update to [External Authentication](https://training.galaxyproject.org/training-material/topics/admin/tutorials/external-auth/slides.html) slides.


---

# ToolShed Contributions

<div class='right'>[<img src="/src/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in February  2019](/src/toolshed/contributions/2019-02/index.md) (there were a lot).


# Releases

New additions to the Galaxy Ecosystem.

## Galaxy Release 19.01

We are pleased to announce the **[release of Galaxy 19.01](https://docs.galaxyproject.org/en/release_19.01/releases/19.01_announce.html)**. This release includes the revised UI style, addition of colorful tags, extensive workflow enhancements, and enhanced support for Singularity. 

Please see the [release notes](https://docs.galaxyproject.org/en/release_19.01/releases/19.01_announce.html) for additional information. 

## galaxy-lib 19.5.0

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

----

# Other News

* From [QCIF](https://www.qcif.edu.au/), both highlighting [UseGalaxy.org.au](https://usegalaxy.org.au/)
  * "[Data Enhanced Virtual Labs: Better the DEVL you know](https://www.qcif.edu.au/index.php/news/532-data-enhanced-virtual-labs-better-the-devl-you-know)"
  * "[Galaxy Australia usage on the rise ](https://www.qcif.edu.au/index.php/news/535-galaxy-australia-usage-on-the-rise)"
* We got an [early look at the future of Galaxy management](https://twitter.com/EnisAfgan/status/1098026864948465664) with Kubernates from Enis Afgan, stay tuned friends. 
