<div class="title">June 2016 Galaxy News</div>

<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 24em"> TABLE_OF_CONTENTS </td>
  </tr>
</table>

</div>

<div class='left'><a href='/GalaxyUpdates/'><img src='/Images/GalaxyLogos/GalaxyNews.png' alt='Galaxy News' width=150 /></a></div>

Welcome to the June 2016 *[Galactic News](/GalaxyUpdates)*, a summary of what is going on in the Galaxy community. 

If you have anything to include in the next News, please send it to [outreach AT galaxyproject DOT org](Galaxy Outreach). 


# GCC2016

<div class='center'>
<a href='https://gcc2016.iu.edu/'><img src='https://gcc2016.iu.edu/img/gcc2016-logo.png' alt='2016 Galaxy Community Conference' width="300" /></a>
</div>

## Complete Schedule Online

<div class='right'>
<a href='https://gcc2016.iu.edu'><img src='/Images/Screenshots/GCC2016SchedHomeThumb.png' alt='2016 Galaxy Community Conference complete schedule' width="170" /></a>
</div>

[GCC2016](http://gcc2016.iu.edu/) will be held June 25-29 at [Indiana University](http://indiana.edu/) in Bloomington, Indiana, United States. This will be the 7th annual gathering of the Galaxy community, and we are expecting over 200 participants again this year.  The 2016 Galaxy Community Conference includes [2 days of hackathons](https://gcc2016.iu.edu/hacks/index.php), [2 days of training](http://gcc16.sched.org/overview/type/D.x+Training+-+All), and [a two day meeting](http://gcc16.sched.org/overview/type/E.x+Conference+-+All) featuring [accepted presentations and keynotes](http://gcc16.sched.org/overview/type/E.1+Conference+-+Talks), **[poster sessions](http://gcc16.sched.org/overview/type/E.2+Conference+-+Posters), the new [Visualization Showcase and Software Demo sessions](http://gcc16.sched.org/overview/type/E.3+Conference+-+Demos)**, [lightning talks](http://gcc16.sched.org/overview/type/E.5+Conference+-+Lightning), [birds-of-a-feather meetups](/Events/GCC2016/BoFs), and [plenty of networking](http://gcc16.sched.org/overview/type/B.+Networking+%26+Sustenance). 

## Registration: June 4

<div class='left'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='https://gcc2016.iu.edu/registration/'>Register Now</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

**[On-time registration for GCC2016](https://gcc2016.iu.edu/registration/) ends June 4**.  *(After that costs go up 100%.  You don't want that.)* Registration rates depend on which events you register for, and your career stage & affiliation. On-time registration starts at less than $65 / day for students and postdocs, and at less than $90 / day for attendees from non-profits.

You can also sign up for [conference housing](https://gcc2016.iu.edu/location) during registration.  

You are strongly encouraged to review the [training](https://gcc16.sched.org/grid-full#2016-06-26) and [housing](https://gcc2016.iu.edu/location) options before beginning the registration process.

## Posters: There is still space!

<div class='right'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://bit.ly/gcc2016abssub'>Submit an abstract</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

The deadline for [abstract submission](http://bit.ly/gcc2016abssub) has passed. However **there are still some poster spaces left and poster abstracts will be considered until we run out of space.**

Other types of late abstract submissions can also be submitted and will be considered if and when cancellations happen.

And, please remember that all accepted presentations are eligible for consideration for publication in the journal *[GigaScience](http://gigascience.biomedcentral.com/)'s* [Galaxy series](http://www.gigasciencejournal.com/series/Galaxy). **Published papers will receive a 15% discount in the [article-processing charge](http://gigascience.biomedcentral.com/frequently-asked-questions)** if you flag GCC2016 on submission.

Topics should be of interest to those working in high-throughput data analysis and research. Presentations that are Galaxy-centric are encouraged, but not required.  Please see the [abstracts page](https://gcc2016.iu.edu/abstracts/index.php) for full details.

<div class='right'><br />
<a href='http://jetstream-cloud.org/'><img src='/Images/Logos/JetstreamRectClipped.jpg' alt='Jetstream' width="240" /></a><br />
<a href='http://ncgas.org/'><img src='/Images/Logos/NCGAS_Logo300.png' alt='National Center for Genome Analysis Support' width="240" /></a>
</div>

## Opening Reception Added

[Jetstream](http://jetstream-cloud.org/), IU's newest National Science Foundation-funded project (and in which Galaxy is a partner), and the [National Center for Genome Analysis Support](http://ncgas.org/) at IU are sponsoring an [opening reception on Monday evening](http://sched.co/72bN) at the IU Cyberinfrastructure Building.  The first ever GCC opening reception will feature local wine/beer, morsels from local eateries, and demonstrations of the 15 million+ pixel IQ-Wall, IU's Data Center, Science on a Sphere, and other IU-centric IT.

## Sponsors

We continue to seek other sponsors as well and [offer a wide range of sponsorship plans](https://gcc2016.iu.edu/sponsors/index.php). If your organization is interested in having a presence at GCC2016, please contact the [gcc2016-exec@lists.galaxyproject.org](GCC2016 Exec) for more information.

### Bloomington Life Sciences Partnership

<div class='right'><a href='http://bloomingtonlifesciences.com/'><img src='/Images/Logos/BloomingtonLifeSciencesPartnership.png' alt='Bloomington Life Sciences Partnership (BLSP)' width="200" /></a>
</div>

Please welcome the **[Bloomington Life Sciences Partnership (BLSP)](http://bloomingtonlifesciences.com/)** as a [GCC2016 Sponsor](https://gcc2016.iu.edu/sponsors/index.php). BLSP is a collaborative effort dedicated to continued life sciences business growth in greater Bloomington, Indiana - a nationally recognized leading metro area in medical devices, contract pharma, biotechnology, basic research and life sciences workforce development. The BLSP continues to carry out its mission of attracting quality jobs to Bloomington and Monroe County. We are fortunate to have a vibrant, diverse and accepting community with a strong academic, advanced manufacturing, life sciences and technology base that add to and support our extraordinary quality of life. With assets like Indiana University and Ivy Tech Community College, we will continue to attract the highly skilled and talented workforce our local employers and community need. The BLSP posts open life sciences jobs, information on local resources and equipment for public use at the university, and posts news and events in the area.

----

# Upcoming Events

## Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data

<div class='left'><a href='https://registration.genomecenter.ucdavis.edu/events/June_2016_Bioinformatics_Galaxy_Workshop/'><img src='/Images/Logos/UCDavisGenomeCenter_Bioinf.png' alt='Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data' width="200" /></a>
</div>

The [UC Davis Bioinformatics Training Program](http://training.bioinformatics.ucdavis.edu/), a [GTN member](/Teach/Trainers/#uc-davis-genome-centers-bioinformatics-core-facility), will be presenting the workshop **[Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data](https://registration.genomecenter.ucdavis.edu/events/June_2016_Bioinformatics_Galaxy_Workshop/)** on June 13-17, at UC Davis in Davis, California, United States.  

This workshop will include a rich collection of lectures and hands-on sessions, covering both theory and tools. We will explore the basics of high throughput sequencing technologies, focusing on Illumina data for hands-on exercises. Participants will explore software and protocols, create and modify workflows, and diagnose/treat problematic data, utilizing computing power of the Amazon Cloud.

*Space is limited and this workshop is already more than 50% full.*

## GalaxyAdmins June Meetups

<div class='right'>
<a href='/Community/GalaxyAdmins/Meetups/2016_06_16/'><img src='/Images/GalaxyLogos/GalaxyAdmins.png' alt='16 June GalaxyAdmins Web Meetup' height="60" /></a>
</div>

There will be two [GalaxyAdmins](/Community/GalaxyAdmins) [meetups](/Community/GalaxyAdmins/Meetups) in June:

1. **An [online meetup on June 16](/Community/GalaxyAdmins/Meetups/2016_06_16)** featuring   [Evan Bollig](https://www.msi.umn.edu/staff/evan) and [JJ Johnson](https://www.msi.umn.edu/staff/jj) of the [Minnesota Supercomputing Institute](https://www.msi.umn.edu/) speaking on *Galaxy in Production at the University of Minnesota*.  

1. **[An in person birds-of-a-feather meetup](http://sched.co/7KLn)** at [GCC2016](https://gcc2016.iu.edu/) in Bloomington, Indiana.

## June, July, and August Events

There are [a plentitude of Galaxy related events coming up](/Events). Here's what's coming up over the summer:

<div class='center'> 
<a href='/Events/ASMS2016/'><img src='/Images/Logos/ASMSLogo.png' alt='64th ASMS Conference on Mass Spectrometry and Allied Topics' height="80" /></a>
<a href='https://www.vlsci.org.au/events/new-round-workshops-announced'><img src='/Images/Logos/VLSCILogo.png' alt='GVL and Galaxy Workshops' height="80" /></a>
<a href='http://bioinformatics.ca/workshops/2016/informatics-high-throughput-sequencing-data-2016'><img src='/Images/Logos/CanadianBioinfWorkshopsLog.png' alt='Informatics on High-throughput Sequencing Data Workshop' height=80 /></a>
<a href='/Community/GalaxyAdmins/Meetups/2016_06_16/'><img src='/Images/GalaxyLogos/GalaxyAdmins.png' alt='16 June GalaxyAdmins Web Meetup' height="60" /></a>
<a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Jun16.html'><img src='/Images/Logos/GlasgowPolyomics.jpg' alt='Introduction to Galaxy Workshop' height="80" /></a>
&nbsp;&nbsp;&nbsp; 
<a href='https://gcc2016.iu.edu/index.php'><img src='/Images/Logos/GCC2016LogoSquareBig.png' alt='2016 Galaxy Community Conference (GCC2016)' width="120" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> June 5-9 </th>
    <td> <em><a href='/Events/ASMS2016/'>64th ASMS Conference on Mass Spectrometry and Allied Topics</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) San Antonio, Texas, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='/Events/ASMS2016/'>Presenters</a> </td>
  </tr>
  <tr>
    <th> June 7 </th>
    <td> <em><a href='https://www.eventbrite.com.au/e/introduction-to-the-gvl-and-galaxy-registration-25420716046'>Introduction to the GVL and Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Australia) Melbourne, Australia </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='https://www.vlsci.org.au/page/contacts'>VLSCI</a> </td>
  </tr>
  <tr>
    <th> June 8 </th>
    <td> <em><a href='https://www.eventbrite.com.au/e/variant-calling-with-galaxy-and-the-gvl-for-beginners-registration-25420916646'>Variant Calling with Galaxy and the GVL for beginners</a></em> </td>
    <td> INCLUDE(/Events/Badges/Australia) Melbourne, Australia </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='https://www.vlsci.org.au/page/contacts'>VLSCI</a> </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> <em><a href='http://bioinformatics.ca/workshops/2016/informatics-high-throughput-sequencing-data-2016'>Informatics on High-throughput Sequencing Data</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Montreal, Quebec, Canada </td>
    <td> INCLUDE(/Teach/GTN/Badge16)  <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 13-17 </th>
    <td> <em><a href='https://registration.genomecenter.ucdavis.edu/events/June_2016_Bioinformatics_Galaxy_Workshop/'>Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Davis, California, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href="mailto:training.bioinformatics@ucdavis.edu">UC Davis Bioinformatics Training Program</a> </td>
  </tr>
  <tr>
    <th> June 14 </th>
    <td> <em><a href='https://www.eventbrite.com.au/e/rna-seq-differential-gene-expression-analysis-using-the-gvl-and-galaxy-registration-25500085442'>RNA-Seq Differential Gene Expression Analysis using the GVL and Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Australia) Melbourne, Australia </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='https://www.vlsci.org.au/page/contacts'>VLSCI</a> </td>
  </tr>
  <tr>
    <th> June 16 </th>
    <td> <em><a href='/Community/GalaxyAdmins/Meetups/2016_06_16/'>Galaxy in Production at the University of Minnesota</a></em> </td>
    <td> INCLUDE(/Events/Badges/World) <a href='/Community/GalaxyAdmins/Meetups/2016_06_16/'>GalaxyAdmins Web Meetup</a>, <em><a href='/Community/GalaxyAdmins/Meetups/2016_06_16/'>Online</a></em> </td>
    <td> <a href='https://www.msi.umn.edu/staff/evan'>Evan Bollig</a>, <a href='https://www.msi.umn.edu/staff/jj'>JJ Johnson</a>, <a href='/HansrudolfHotz/'>Hans-Rudolf Hotz</a>, <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> June 20 </th>
    <td> <em><a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Jun16.html'>Introduction to Galaxy Workshop</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) University of Glasgow, Glasgow, United Kingdom </td>
    <td> <a href='http://www.polyomics.gla.ac.uk/biog-davidmeltzer.html'>David Meltzer</a> </td>
  </tr>
  <tr>
    <th> June 25-29 </th>
    <td> </strong><em><a href='https://gcc2016.iu.edu/'>2016 Galaxy Community Conference (GCC2016)</a><strong></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16)  <a href='https://gcc2016.iu.edu/organizers'>Organizers</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 1 </th>
    <td> <em><a href='http://gmod.org/wiki/Jun_2016_GMOD_Meeting'>GMOD Meeting</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> <a href='http://gmod.org/wiki/Scott'>Scott Cain</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> July 4-8 </th>
    <td> <em><a href='http://bioinformatics.org.au/ws16/program/'>An introduction to Galaxy with the NeCTAR Genomics Virtual Laboratory Workshop</a></em> </td>
    <td> INCLUDE(/Events/Badges/Australia) <a href='http://bioinformatics.org.au/ws16/'>2016 Winter School in Mathematical & Computational Biology</a>, University of Queensland, Brisbane, Australia </td>
    <td> <a href='https://www.rcc.uq.edu.au/genomics-virtual-lab'>Igor Makunin, Derek Benson</a> </td>
  </tr>
  <tr>
    <th> July 6-7 </th>
    <td> <em><a href='https://www.open-bio.org/wiki/Codefest_2016'>BOSC Codefest 2016</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Orlando, Florida, United States </td>
    <td> Brad Chapman </td>
  </tr>
  <tr>
    <th> July 8-9 </th>
    <td> <em><a href='/Events/ISMB2016/'>BOSC 2016</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica)  <a href='http://www.iscb.org/ismb2016'>ISMB 2016</a>, Orlando, Florida, United States </td>
    <td> See <a href='/Events/ISMB2016/'>presenters</a> </td>
  </tr>
  <tr>
    <th> July 8-12 </th>
    <td> <em><a href='/Events/ISMB2016/'>ISMB 2016</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica)  Orlando, Florida, United States </td>
    <td> See <a href='/Events/ISMB2016/'>presenters</a> </td>
  </tr>
  <tr>
    <th> July 13-17 </th>
    <td> <em><a href='http://www.genetics-gsa.org/genetics/2016/cgi-bin/tagc16soe.pl#Sat'>An Introduction to Using Galaxy for Genetic Data Analysis</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.genetics2016.org/index.htm'>The Allied Genetics Conference 2016 (TAGC)</a>, Orlando, Florida, United States </td>
    <td> <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> July 15 </th>
    <td> <em><a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Jul16.html'>Introduction to Galaxy Workshop</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) University of Glasgow, Glasgow, United Kingdom </td>
    <td> <a href='http://www.polyomics.gla.ac.uk/biog-davidmeltzer.html'>David Meltzer</a> </td>
  </tr>
  <tr>
    <th> July 21 </th>
    <td> <em><a href='/Events/Cornell2016/'>Introduction to Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Cornell University, Ithaca, New York, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> August 1-4 </th>
    <td> <em><a href='http://icbo.cgrb.oregonstate.edu/node/41'>Integrating Ontological and Genomic Analysis with Galaxy Workshop</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://icbo.cgrb.oregonstate.edu/'>ICBO 2016</a>, Corvallis, Oregon, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 12 </th>
    <td> <em><a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Aug16.html'>Introduction to Galaxy Workshop</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) University of Glasgow, Glasgow, United Kingdom </td>
    <td> <a href='http://www.polyomics.gla.ac.uk/biog-davidmeltzer.html'>David Meltzer</a> </td>
  </tr>
  <tr>
    <th> August 29 - September 2 </th>
    <td> <em><a href='https://registration.genomecenter.ucdavis.edu/events/Aug_2016_Bioinformatics_Galaxy_Workshop/'>Using Galaxy for Analysis of RNA-Seq, Exome-Seq, and Variants</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Davis, California, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href="mailto:training.bioinformatics@ucdavis.edu">UC Davis Bioinformatics Training Program</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none"> <a href='/Teach/Trainers/'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none"> <span style="font-size: smaller;"> <a href='/Teach/Trainers/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

# New Papers

**76 new papers** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in May.

Some highlights from May:

* [LimiTT: link miRNAs to targets](http://dx.doi.org/10.1186/s12859-016-1070-1) 
  * by Julia Bayer, Carsten Kuenne, Jens Preussner, Mario Looso. *BMC Bioinformatics*, Vol. 17, No. 1. (11 May 2016), doi:10.1186/s12859-016-1070-1

* [From Scientific Workflow Patterns to 5-star Linked Open Data](https://www.usenix.org/conference/tapp16/workshop-program/presentation/gaignard)
    by Alban Gaignard, Hala Skaf-Molli, and Audrey Bihouée. In 8th USENIX Workshop on the Theory and Practice of Provenance (TaPP 16) (June 2016)

* [Computational analysis of fitness landscapes and evolutionary networks from in vitro evolution experiments](http://dx.doi.org/10.1016/j.ymeth.2016.05.012)
    by Ramon Xulvi-Brunet, Gregory W. Campbell, Sudha Rajamani, José I. Jiménez, Irene A. Chen. *Methods* (May 2016), doi:10.1016/j.ymeth.2016.05.012

and ...

## The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update

The [project and the developer community](/GalaxyTeam) has published a new paper, 

  [The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2016 update](http://dx.doi.org/10.1093/nar/gkw343), *Nucleic Acids Research* (2 May 2016), doi:10.1093/nar/gkw343

that describes recent updates to the Galaxy platform in general and to the project's [usegalaxy.org Galaxy server](/Main) in particular.  Check it out.

## Tagged Publications

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 11 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 16 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 43 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><br />![Please Help! Yes you!](/Images/GalaxyIsExpandingCloud.png)</div>

The Galaxy is expanding! Please help it grow.
* [Postdoctoral Researcher: Forest Genomics Database and Software Developer](http://compgenomics.lab.uconn.edu/join/), University of Connecticut.  "... data sharing among partner databases through Galaxy modules."  <<span(red)>>Deadline June 10.<<span>> 
* [Galaxy Administrator & Developer](http://www.uni-freiburg.de/verwaltung/stellenboerse/?erf=y&stellenid=9693&layout=v4), University of Freiburg, Freiburg, Germany.
* [Bioinformatics Web Application Developer-Biology](https://jobs.wustl.edu/) (Job ID 32899), Washington University in St. Louis, Missouri, United States
* [Software developer and Post-docs](http://gehlenborg.com/open-positions/), Gehlenborg Lab, Harvard Medical School, Boston, Massachusetts, United States
* [Postdoctoral Research Positions](https://chk.tbe.taleo.net/chk01/ats/careers/requisition.jsp?org=BCM&cws=42&rid=8453), Molecular and Cellular Biology Department at Baylor College of Medicine, Houston, Texas, United States 

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />

----


# New Public Galaxy Servers

No [new publicly accessible Galaxy servers](/PublicGalaxyServers) were added to [the list](/PublicGalaxyServers) in May.  To make up for this sad state of affairs, we created the ...

<div class='right'><br />
<div class='center'>
<a href='https://launch.genome.edu.au/launch'><img src='/PublicGalaxyServers/GenomicsVirtualLab300.png' alt='GVL' width="80" /></a> &nbsp; <a href='https://galaxy.plgrid.pl'><img src='/SemiPublicGalaxyServers/PL-Grid_Logo.png' alt='PL-Grid Galaxy' width="80" /></a><br />
<a href='https://www.genap.ca/'><img src='/SemiPublicGalaxyServers/GenAP_Logo_Trans_250.png' alt='GenAP' width="120" /></a><br />
<a href='/SemiPublicGalaxyServers/#australian-access-federation-aaf'><img src='/SemiPublicGalaxyServers/AAF_Logo_wide.png' alt='AAF on GVL' width="200" /></a>
</div></div>

## New Semi-Public Galaxy Servers Directory

The [Semi-Public Galaxy Servers Directory](/SemiPublicGalaxyServers) lists Galaxy servers that aren't [fully publicly accessible](/PublicGalaxyServers), but that are accessible to a larger group than just the host institution's members.  (Most Galaxy instances are *internal* to an organization - they can only be accessed by members of the host institution.)

Servers are classified by whom they are accessible to.  This can be based on [research domain](/SemiPublicGalaxyServers/#domain-based), [Geography Based|geography (regional or national)](/SemiPublicGalaxyServers), or by [membership in a multi-organization association](/SemiPublicGalaxyServers/#association-based) that provides Galaxy Servers. 

**To add your semi-public Galaxy server to this list**, please either just add it (hey, *it's a wiki*), or [describe the server here](https://docs.google.com/forms/d/14i8FNHB7pwXrXmhvUkxXTh2VKQ-2BKsnMubXsCZadjs/viewform) and we'll post it this directory.

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Teach/'><img src='/Images/GalaxyLogos/GTNLogo300.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Logs/'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Deployments/'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong>Share your <a href='/Teach/Resources/#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers/#add-a-trainer'>experience</a> <em>now</em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Logs/#add-a-log-page'>Share your experience</a> <em><a href='/Community/Logs/#add-a-log-page'>now</a></em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Deployments/#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/Community/Deployments/#add-your-galaxy-deployment'>now</a></em> <strong> </td>
  </tr>
</table>

<br />

One new training organization joined the [Galaxy Training Network](/Teach/GTN):

* [Geneclat Technologies](/Teach/Trainers/#geneclat-technologies) is based in Bengaluru, Karnataka, India, and is available globally (and "ready to travel!") 

## New Commercial Support Directory

Ever wanted commercial support for your Galaxy needs?  There is now a [Commercial Galaxy Support directory](/Support/Commercial) you can consult if you are looking to find service providers and consultants.  The initial list includes consultants, server providers, cloud providers, and commercial trainers.

Want to add your company to that list?  Send an email to [outreach AT galaxyproject.org](outreach AT galaxyproject.org).

----

# Releases

## Galaxy v16.04

<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GalaxyNewLogo_GalaxyProject_Trans.png' alt='GalaxyProject' width="200" /></a></div>
 
The [Galaxy Committers team](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) is pleased to announce the **[April 2016 (v16.04) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.04_announce.html)**.

*Please note that Python 2.6 is no longer supported as of this release. See the [16.04 release announcement](https://wiki.galaxyproject.org/News/2016_04_GalaxyRelease) for details.*

<span style="font-size: larger;"> **Highlights** </span>

**Tool Profile Versions**

Tools may now [declare which version](http://planemo.readthedocs.io/en/latest/galaxy_changelog.html#tool-profile-version-pr-1688) of Galaxy they require. Tools requiring 16.04 or newer will have new default behaviors (such as using exit code for error detection) that should simplify tool development. See [PR #1688](https://github.com/galaxyproject/galaxy/pull/1688).

**Embedded Pulsar Job Runner**

Galaxy can now start a Pulsar application embedded within the Galaxy process itself. This allows using Pulsar’s job staging and isolation without requiring a RESTful web service or a message queue. This is enabling usegalaxy.org to run jobs to on the new [JetStream cloud](http://jetstream-cloud.org/). See [PR #2057](https://github.com/galaxyproject/galaxy/pull/2057).

**New Chemical Datatypes**

Galaxy now detects and supports many molecular datatypes. See [PR 1941](https://github.com/galaxyproject/galaxy/pull/1941). Thanks to [Björn Grüning (@bgruening)](https://github.com/bgruening).


<table>
  <tr>
    <td style=" border: none"> <span style="font-size: larger;"> Github </span> <br /><br /></strong>New<strong><br /><code>% git clone -b master https://github.com/galaxyproject/galaxy.git</code><br /><br />U</strong>pdate to latest stable release<strong><br /><code>% git checkout master && git pull --ff-only origin master</code><br /><br /></strong>Update to exact version<strong><br /><code>% git checkout v16.04</code> </td>
    <td style=" vertical-align: top; border: none; width: 10%"> </td>
    <td style=" vertical-align: top; border: none"> <span style="font-size: larger;"> !BitBucket </span> <br /><br /></strong>Upgrade<strong><br /><code>% hg pull</code><br /><code>% hg update latest_16.04</code> </td>
  </tr>
  <tr>
    <td colspan=3 style=" border: none"> See the <a href='/Admin/GetGalaxy/'>Get Galaxy</a> page for additional details regarding the source code locations. </td>
  </tr>
</table>



### Galaxy Docker Image 16.04

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 16.04](https://github.com/bgruening/docker-galaxy-stable/releases/tag/16.04) as well.  

## CloudMan 16.04

<div class='right'><a href='/CloudMan/'><img src='/Images/GalaxyLogos/cloudman-logo.jpg' alt='CloudMan' width="300" /></a></div>

**We just released an update to Galaxy !CloudMan on AWS.** CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

This is a minor update release with the following changes:
* Galaxy 16.04 update
* Availability on Amazon's Ireland region 
* A couple of bug fixes




See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG) for a more complete set of changes.

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src='/Images/Logos/PlanemoLogo.png' alt='' width="200" /></a></div>

## Planemo 0.25.0 - 0.26.0

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  There was a lot of activity in the May releases of Planemo. Some highlights: 

* Implement `Engine` and `Runnable` abstractions - Planemo now has beta support for testing Galaxy workflows and CWL tools with Galaxy and any CWL artifact with [cwltool](https://github.com/common-workflow-language/cwltool/). [Pull Request 454](https://github.com/galaxyproject/planemo/pull/454), [7be1bf5](https://github.com/galaxyproject/planemo/commit/7be1bf5)
* Introduce profile commands (`profile_create, profile_delete`, and `profile_list`) and profile improvements (automatic postgres database creation support). [Pull Request 480](https://github.com/galaxyproject/planemo/pull/480), [a87899b](https://github.com/galaxyproject/planemo/commit/a87899b)
* Rework Galaxy test reporting to use structured data instead of XUnit data. [4d29bf1](https://github.com/galaxyproject/planemo/commit/4d29bf1)
* Refactor Galaxy configuration toward support for running Galaxy in docker-galaxy-stable. [Pull Request 479](https://github.com/galaxyproject/planemo/pull/479)
* Implement Galaxy "profiles" - the ability to configure perisistent, named environments for `serve` and `test`. [5d08b67](https://github.com/galaxyproject/planemo/commit/5d08b67)
* Greatly improved `serve` command - make `test-data` available as an FTP folder, (on 16.07) automatically log in an admin user, and many more options (such as those required for "profiles" and a `--daemon` mode).
* Target draft 3 of [CWL](http://www.commonwl.org/) instead of draft 2. [775bf49](https://github.com/galaxyproject/planemo/commit/775bf49)
* Add IUC standards to Planemo docs. [2ae2b49](https://github.com/galaxyproject/planemo/commit/2ae2b49)
* Revise command-line handling framework for consistency and extension - allow extra options to be configured as defaults `~/.planemo.yml` including `--job_config_file` and conda configuration options. [e769118](https://github.com/galaxyproject/planemo/commit/e769118), [26e378e](https://github.com/galaxyproject/planemo/commit/26e378e)

See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).

## galaxy-lib 16.7.1 - 16.7.7

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

Highlights of May releases include:

* Fixes to CWL and Docker libraries for Planemo.
* Updates to cwltool_deps for Planemo.
* Updates to tool parsing and linting for Planemo.
* Update against the latest development branch of Galaxy.

## Earlier Releases

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src='/Images/GalaxyLogos/pulsar_transparent.png' alt='Pulsar' width="150" /></a></div>

**Pulsar 0.7.0**

[Pulsar 0.7](https://pypi.python.org/pypi/pulsar-app/) was released in April.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.


**!CloudBridge 0.1.0**

The Galaxy Team is proud to be part of the development team for a new cross-cloud library called !CloudBridge. *!CloudBridge is a Python library providing a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud.* The library is generally applicable to any domain wishing to run cloud-independent applications. There is already support for Amazon and !OpenStack clouds with support for Google’s Compute Engine in development. 

The first version of !CloudBridge was released earlier this month and it comes with [detailed user documentation](http://cloudbridge.readthedocs.org/). The source code is [available on Github](https://github.com/gvlproject/cloudbridge).



**Starforge 0.1**

<div class='right'>
<a href='https://github.com/galaxyproject/starforge/blob/master/README.rst'><img src='/Images/GalaxyLogos/starforge_logo_big.png' alt='Starforge' width="180" /></a>
</div>

[Starforge](https://github.com/galaxyproject/starforge/blob/master/README.rst) is a collection of scripts that supports the building of components for Galaxy.  Specifically, with Starforge you can:

* Build [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) dependencies
* Build [Python Wheels](http://pythonwheels.com/) (e.g. for the [Galaxy Wheels Server](http://wheels.galaxyproject.org/))
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.

Documentation can be found at [starforge.readthedocs.org](http://starforge.readthedocs.org/). 


**!BioBlend 0.7.0**

   [BioBlend version 0.7.0](https://github.com/galaxyproject/bioblend/) was released at the beginning of November. !BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/Learn/API).  CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG).


**blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and !BioCloudCentral.


----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

Please welcome [Anthony Brétaudeau](https://www6.rennes.inra.fr/igepp/Listes-des-personnes/B/Bretaudeau-Anthony) from [BIPAA](https://www6.rennes.inra.fr/igepp/L-IGEPP/Plateformes/Plateforme-bioinformatique-pour-les-arthropodes-des-agroecosystemes) at [INRA](https://www6.rennes.inra.fr/igepp/) to the [/IUC](/IUC).

[ToolShed contributions for April and May](/ToolShed/Contributions/2016_06).

# Other News

* [Metavisitor Galaxy workflows in MyExperiment](http://www.myexperiment.org/workflows?filter=TAG_ID%28%224598%22%29): rapid detection & discovery of viruses
* 200+ [forks of Galaxy](https://github.com/galaxyproject/galaxy)!
* *Galaxy on the Cloud* is now available in the AWS EU-Ireland region - directly from https://launch.usegalaxy.org.
