---
autotoc: true
title: December 2012 Galaxy Update
---
<div class='right'></div>



<div class='left'><a href='/GalaxyUpdates'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the December 2012 edition of *[Galaxy Update](/src/GalaxyUpdates/index.md)*, a monthly summary of what is going on in the Galaxy community.  *[Galaxy Updates](/src/GalaxyUpdates/index.md)* complement the *[Galaxy Development News Briefs](/src/DevNewsBriefs/index.md)* which accompany new Galaxy releases and focus on Galaxy code updates.

<br />

## New Papers

These papers may be of interest to the Galaxy community:

<div class='left'>
<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 20 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>methods</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/other'>other</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>tools</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>project</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>isgalaxy</a> </td>
  </tr>
</table>

</div>

* [Enis Afgan](/src/EnisAfgan/index.md), Brad Chapman and [James Taylor](/JamesTaylor), "[CloudMan as a platform for tool, data, and analysis distribution](http://www.biomedcentral.com/1471-2105/13/315/abstract)." *[BMC Bioinformatics](http://www.biomedcentral.com/bmcbioinformatics/)* 2012, 13:315 <br /><br />
* [Jeremy Goecks](/src/JeremyGoecks/index.md), [Nate Coraor](/nate), [The Galaxy Team](/GalaxyTeam), [Anton Nekrutenko](/anton) & [James Taylor](/JamesTaylor), "[NGS analyses by visualization with Trackster](http://bit.ly/VMUz7p)."  *[Nature Biotechnology](http://www.nature.com/nbt/index.html)* 30, 1036–1039 (2012) <br /><br />
* Samantha Baldwin, Roopashree Revanna, Susan Thomson, *et al.*, "[A Toolkit for bulk PCR-based marker design from next-generation sequence data: application for development of a framework linkage map in bulb onion (Allium cepa L.)](http://dx.doi.org/10.1186/1471-2164-13-637)," *BMC Genomics*, Vol. 13, No. 1. (2012), 637 <br /><br />
* Jeremy C. Morgan, Robert W. Chapman, Paul E. Anderson, "[A next generation sequence processing and analysis platform with integrated cloud-storage and high performance computing resources](http://dx.doi.org/10.1145/2382936.2383033).  Proceedings of the ACM Conference on Bioinformatics, Computational Biology and Biomedicine <br /><br /> 
* Bo Liu, Borja Sotomayor, Ravi Madduri, Kyle Chard, "[Deploying Bioinformatics Workflows on Clouds with Galaxy and Globus Provision](http:// bit.ly/UPBbRI)."  [Third International Workshop on Data Intensive Computing in the Clouds (DataCloud 2012)](http://www.cse.buffalo.edu/faculty/tkosar/DataCloud2012/)

These papers were among [37 papers added to the Galaxy CiteULike group](http://www.citeulike.org/group/16008/library) since the last [Galaxy Update](/src/GalaxyUpdates/index.md).  


<br />

## New Public Galaxy Servers

<div class='right'><a href='http://galaxy.nbic.nl/'><img src='/Images/Logos/NBICLogo.png' alt='NBIC' height="50" /></a></div>

* **[Andromeda](http://galaxy.nbic.nl/):** The [Netherlands Bioinformatics Centre (NBIC)](http://www.nbic.nl/) and [BiG Grid/SARA](http://www.biggrid.nl/) have launched  [a public and fully populated Galaxy instance](http://galaxy.nbic.nl/) that runs on a cloud infrastructure.  See [the announcement](http://www.nbic.nl/about-nbic/news-press/bioinformatics-news/detail/article/galaxy-on-cloud/) for more details.

<div class='right'><a href='http://w3lamc.umbr.cas.cz/lamc/'><img src='/Images/Logos/PlantMolBioCZLogo.png' alt='Laboratory of Molecular Cytogenetics, Laboratory of Molecular Cytogenetics, Institute of Plant Molecular Biology' height="80" /></a></div>

* **[RepeatExplorer](http://repeatexplorer.umbr.cas.cz/):** Features utilities for [Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data](http://www.biomedcentral.com/1471-2105/11/378) and tools for the detection of transposable element protein coding domains.  [RepeatExplorer](http://repeatexplorer.umbr.cas.cz/) is hosted by the [Laboratory of Molecular Cytogenetics](http://w3lamc.umbr.cas.cz/lamc/), [Institute of Plant Molecular Biology](http://www.umbr.cas.cz/), [Biology Centre ASCR](http://www.bc.cas.cz/en/)

See [the list of public Galaxy servers](/src/PublicGalaxyServers/index.md) for more open and accessible Galaxy instances.

## Who's Hiring

<div class='right'>![Please Help!  Yes you!](/src/Images/Icons/PointingFinger.png)</div>
The Galaxy is expanding!  Please help it grow.

* [Research Fellow (Molecular Biologist)](http://bit.ly/UZzlOd) @ [MRC Human Genetics Unit](http://www.hgu.mrc.ac.uk/) at the [MRC Institute of Genetics and Molecular Medicine](http://www.igmm.ac.uk/), Edinburgh, UK
* [Offre de stage M2 PRO](http://bit.ly/114Xd9p), [Mathématique, Informatique et Génome (MIG)](http://mig.jouy.inra.fr/) est une unité de l'INRA de Jouy-en-Josas
* [Engineer position in bioinformatics: structural polymorphism analysis from NGS data](http://bit.ly/TEQvzd) @ [UMR de Génétique Végétale, INRA-Université Paris Sud-CNRS](http://moulon.inra.fr/index.php/fr/equipestransversales/atelier-de-bioinformatique)
* [R/Bioconductor and Genomics Expert](http://bit.ly/T1QOZP) @ the [Friedrich Miescher Institute](http://www.fmi.ch/), affiliated with the University of Basel.
* [Computational Biologist](http://bit.ly/R2ZJoW) @ the [Harvard Stem Cell Institute's (HSCI)](http://www.hsci.harvard.edu/)[Center for Stem Cell Bioinformatics](http://www.hsci.harvard.edu/research/center-stem-cell-bioinformatics)
* The [Galaxy Project is hiring](/src/GalaxyIsHiring/index.md) post-docs @ Penn State and Emory

Got a Galaxy-related opening?  Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

## Upcoming Events and Deadlines

<div class='center'>
<a href='https://pag.confex.com/pag/xxi/webprogram/Paper7241.html'><img src='/Images/Logos/PAGLogo114.png' alt='Plant and Animal Genome (PAG 2013)' height="80" /></a>
<a href='/Community/GalaxyAdmins/Meetups/2013_01_16'><img src='/Images/Logos/GalaxyAdmins.png' alt='January Galaxy Admins Web Meetup' height="70" /></a>
<a href='http://conf.abrf.org/'><img src='/Images/Logos/ABRF2013LogoSquare300.png' alt='ABRF 2013' height="80" /></a>
<a href='/Events/GCC2013'><img src='/Images/Logos/GCC2013Logo200.png' alt='2013 Galaxy Community Conference (GCC2013)' width="135px" /></a>
</div>

<br />
<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif' /></a></div> 

See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on these and other events.

### Events

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> January 12-16 </th>
    <td> <em><a href='https://pag.confex.com/pag/xxi/webprogram/Paper7241.html'>Galaxy</a></em> </td>
    <td> Part of the <a href='https://pag.confex.com/pag/xxi/webprogram/Session1537.html'>GMOD Workshop</a> at </strong><a href='http://www.intlpag.org/'>Plant and Animal Genome (PAG 2013)</a><strong>, San Diego, California, United States </td>
    <td> <a href='/JenniferJackson'>Jennifer Jackson</a> </td>
  </tr>
  <tr>
    <th> January 16 </th>
    <td> <em><a href='/Community/GalaxyAdmins/Meetups/2013_01_16'>Deploying Production Galaxy Environments on OpenStack with CloudBioLinux and CloudMan</a></em> </td>
    <td> <a href='/Community/GalaxyAdmins/Meetups/2013_01_16'>January GalaxyAdmins Web Meetup</a> </td>
    <td> <a href='https://www.msi.umn.edu/users/chilton'>John Chilton</a>, Srinivas Maddhi, <a href='/DaveClements'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> January 14-18 </th>
    <td> <em><a href='http://bit.ly/RhQFPm'>Ecole de bioinformatique : Initiation au traitement des données de génomique obtenues par séquençage à haut débit</a></em> </td>
    <td> Station Biologique de Roscoff, France </td>
    <td> <a href="mailto:ecole-bioinfo AT aviesan DOT fr">ecole-bioinfo AT aviesan DOT fr</a> </td>
  </tr>
  <tr>
    <th> January 28 - February 1 </th>
    <td> </strong><em><a href='http://bit.ly/PmKgUI'>Phylotastic Hackathon</a><strong></em>  </td>
    <td> <a href='http://bio5.arizona.edu/about-bio5'>Tucson, Arizona, United States</a> </td>
    <td> Arlin Stoltzfus </td>
  </tr>
  <tr>
    <th> March 2-5 </th>
    <td> <em><a href='http://conf.abrf.org/index.cfm/page/satellite_workshops.htm'>SW1: Application of NGS Platforms for Whole Transcriptome and Genome Analysis</a></em> and <br /><em><a href='http://conf.abrf.org/index.cfm/page/index/2013ConfProg.htm'>W6: Community Resource Solutions to Analyzing Large Genomic Data Sets</a></em> </td>
    <td> </strong><a href='http://conf.abrf.org/'>ABRF 2013</a><strong><br />Palm Springs, California, United States </td>
    <td> <a href='/DaveClements'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> April 5-6 </th>
    <td> <em><a href='http://gmod.org/wiki/April 2013 GMOD Meeting'>2013 GMOD Meeting</a></em> </td>
    <td> Cambridge, United Kingdom, immediately prior to <a href='http://www.ebi.ac.uk/biocuration2013/'>Biocuration 2013</a> </td>
    <td> <a href='/DaveClements'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> April 7-10 </th>
    <td> <em><a href='http://wiki.geneontology.org/index.php/GO_Galaxy_Workshop_at_Biocuration_2013'>GO Galaxy Workshop</a></em> </td>
    <td> <a href='http://www.ebi.ac.uk/biocuration2013/home'>Biocuration 2013</a>,  Cambridge, United Kingdom </td>
    <td> <a href='/DaveClements'>Dave Clements</a>, <a href='http://www.berkeleybop.org/person/suzanna-lewis'>Suzanna Lewis</a> </td>
  </tr>
  <tr>
    <th> June 30 - July&nbsp;2 </th>
    <td> </strong><em><a href='/Events/GCC2013'>2013 Galaxy Community Conference (GCC2013)</a><strong></em> </td>
    <td> University of Oslo, Oslo, Norway </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>


### Deadlines

We only know of one deadline coming up in December:
* [Genomic Standards Council Meeting (GSC 15)](http://gensc.org/gc_wiki/index.php/GSC_15) (Dec 20: Deadline for submission of abstracts)

However, there are a few high-profile meetings:
* [Rocky 2012](http://www.iscb.org/rocky2012)
* [BeNeLux Bioinformatic Conference (BBC) 2012](http://www.nbic.nl/bbc2012)
* [ISCB-Asia/SCCG 2012](http://www.iscb.org/iscb-asia2012) 


## Source Code Documentation

<div class='right'><a href='http://galaxy-dist.readthedocs.org'><img src='/Images/NewsGraphics/2012_11_14_dist-readthedocs.png' alt='Galaxy-Dist at ReadTheDocs' width="320" /></a><br /><a href='http://readthedocs.org'><img src='/Images/Logos/ReadTheDocsLogo.png' alt='Read the Docs' width="125" /></a><a href='http://sphinx-doc.org/'><img src='/Images/Logos/SphinxLogo333.png' alt='Sphinx Python Documentation Generator' width="200" /></a></div>

The Galaxy Project is now using *[Sphinx](http://sphinx-doc.org/)* hosted at [Read the Docs](http://readthedocs.org) to document the Galaxy code base.  Two versions are available:
* **[galaxy-dist](http://galaxy-dist.readthedocs.org):** Code documentation for latest official release of Galaxy.
* **[galaxy-central](http://galaxy-central.readthedocs.org):** Code documentation for the latest committed version of the code.  This should never lag behind the Bitbucket checkins by more than an hour.

This documentation is a work in program and should make incremental improvements with each release.

## New Galaxy Distributions

**[November 11, 2012](/src/DevNewsBriefs/2012_11_14/index.md)**

<div class='indent'>
**Highlights:**
* **NGS: Picard (beta)** tools have moved from the **[Galaxy distribution](https://bitbucket.org/galaxy/galaxy-dist)** to the **[Galaxy Main Tool Shed](http://toolshed.g2.bx.psu.edu/)**.
* The **[Intergalactic Utilities Commission](http://wiki.g2.bx.psu.edu/ReviewingToolShedRepositories)** will soon begin reviewing repositories in the **[Galaxy Main Tool Shed](http://toolshed.g2.bx.psu.edu/)** (better repos, better tools)
* **[Tool Shed](http://toolshed.g2.bx.psu.edu/)** **"best practice"** advice: [a single tool or a suite of tools per repository?](http://wiki.galaxyproject.org/AToolOrASuitePerRepository)
* **[Tophat](http://tophat.cbcb.umd.edu/)**, **[Tophat2](http://tophat.cbcb.umd.edu/manual.html)**, and **[Cuffdiff](http://cufflinks.cbcb.umd.edu/manual.html#cuffdiff)** **updated** to accept gene annotations in **[GFF3](http://wiki.galaxyproject.org/Learn/Datatypes#GFF3)** format.
* Multiple **enhancements** to the **[API](/src/DevNewsBriefs/2012_11_14/index.md#api)** targeting user and history actions.
* Plus **updates to** **[CloudLaunch](http://usegalaxy.org/cloud)**, new **[Security Fixes](/src/DevNewsBriefs/2012_11_14/index.md#security_fixes)**, and several usability enhancements for **Datasets**, **Datatypes**, **Tools**, and **Tool Shed** functions.

```
new:     $ hg clone http://www.bx.psu.edu/hg/galaxy galaxy-dist
upgrade: $ hg pull -u -r 5dcbbdfe1087
```

</div>

## New Galaxy CloudMan Release

CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.

This update brings a large number of updates and new features, the most prominent ones being:
* Support for Eucalyptus cloud middleware. Thanks to Alex Richter. Also, CloudMan can now run on the HPcloud in basic mode (note that there is no public image available on the HPcloud at the moment and one would thus need to be built by you).
* Added a new file system management interface on the CloudMan Admin page, allowing control and providing insight into each available file system
* Added quite a few new user data options. See the [UserData page](/src/CloudMan/UserData/index.md) for details. Thanks to John Chilton.
* Galaxy can now be run in multi-process mode. Thanks to John Chilton.
* Added Galaxy Reports app as a CloudMan service. Thanks to John Chilton.
* Introduced a new format for cluster configuration persistence, allowing more flexibility in how services are maintained
* Added a new file system service for instance's transient storage, allowing it to be used across the cluster over NFS. The file system is available at /mnt/transient_nfs just know that any data stored there will not be preserved after a cluster is terminated.
* Support for Ubuntu 12.10
* Worker instances are now also SGE submit hosts

This update comes as a result of 175 code changesets; for a complete list of changes, see the [commit messages](https://bitbucket.org/galaxy/cloudman/changesets/tip/3a63b9a40331%3A35baec1). 

**Any new cluster will automatically start using this version of CloudMan. Existing clusters will be given an option to do an automatic update once the main interface page is refreshed.**

## Tool Shed Contributions

Several new repositories were added to, and existing repositories updated in the [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) in the past month.

* `bsmap`: A short read mapping tool for bisulfite sequencing reads
* `bowtie_wrappers, bowtie_color_wrappers, lastz, lastz_paired_ends`: Wrappers all created by the [Galaxy Team](/src/GalaxyTeam/index.md)
* `semweb_tools`: A collection of Semantic Web tools, including a (pure python) SPARQL-to-tabular format tool
* `ea_utils`: FASTQ processing utilities; sam-stats added
* `regex_replace`: Regular expression replacement using the Python 
* `blast_datatypes` and `ncbi_blast_plus`: Added BLAST database support
* `homer`: motif discovery and next generation sequencing analysis
* `meme_chip`: Motif based sequence analysis
* `SVDetect`:  detect genomic structural variations from paired-end and mate-pair sequencing data
* `cloudmap_in_silico_complementation`: Perform in silico complementation analysis on multiple tabular snpEff output files 
* `sniploid`: compares SNP detected from a polyploid to those derived from its parental genomes
* `tabular_label_convert`: Takes a tabular format file of numerical values and converts the labels of the  rows or columns using an alias map
* `tabular_edit`: Edit the contents and row/column labels of a tabular file using python statements


## Other News

<div class='right'><a href='/Events/GCC2013/Promotion'><img src='/Images/Logos/GCC2013Logo200.png' alt='2013 Galaxy Community Conference (GCC2013)' width="175px" /></a><br />
<a href='/Events/GCC2013/TrainingDay'><img src='/Images/Logos/GCC2013TrainingDayLogo200.png' alt='2013 Galaxy Community Conference (GCC2013) Training Day' width="175px" /></a>
</div>

* [GCC2013](/src/Events/GCC2013/index.md) is coming.  Help [get the word out](/src/Events/GCC2013/Promotion/index.md)!
* [Training Day Topic Nominations](/src/Events/GCC2013/TrainingDay/index.md) for [GCC2013](/src/Events/GCC2013/index.md) will open in December.  Please start thinking of ideas now.
* Slides and Screencast from [November GalaxyAdmins Meetup](/src/Community/GalaxyAdmins/Meetups/2012_11_14/index.md) are online.  The next [GalaxyAdmins Meetup](/Community/GalaxyAdmins/Meetups) will be on [January 16](/Community/GalaxyAdmins/Meetups/2013_01_16) and feature [John Chilton](/src/JohnChilton/index.md) discussing "Deploying Galaxy on !OpenStack with !CloudBioLinux & CloudMan"
* New Tutorial: *[Analysis of ChIP-seq data in Galaxy](http://jura.wi.mit.edu/bio/education/hot_topics/galaxy/GalaxyNov2012_ChIP-seq_toPost.pdf)* from the [BaRC, Whitehead Institute](http://jura.wi.mit.edu/bio/education/hot_topics/)
* [A short "Getting started with JGalaxy"](http://bit.ly/SkW2yU) document (with screenshots), by [John Chilton](/src/JohnChilton/index.md)
* [生物人的電腦教室：高通量定序分析一次搞定](http://bit.ly/Sc4LDN) including Galaxy, by Eric Lee 
* [Batch Workflow starting using the Galaxy API : Practical Example](http://bit.ly/TjTj6X) by Geert Vandeweyer
 
