---
autotoc: true
title: The April 2015 Galactic News, !
---


<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 30em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/GalaxyUpdates'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the April 2015 *[Galactic News](/src/GalaxyUpdates/index.md)*, a summary of what is going on in the Galaxy community. *[These newsletters](/src/GalaxyUpdates/index.md)* complement the *[Galaxy Development News Briefs](/src/DevNewsBriefs/index.md)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# New Papers

68 new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in March, bring the total to 2200 publications. Some highlights:

* [Transcriptionally Active Regions Are the Preferred Targets for Chromosomal HPV Integration in Cervical Carcinogenesis](http://dx.doi.org/10.1371/journal.pone.0119566), by Irene Kraus K. Christiansen, Geir Kjetil K. Sandve, Martina Schmitz, Matthias Dürst, Eivind Hovig; *PloS One*, Vol. 10, No. 3. (2015)
  
* [Reproducible Analysis of Sequencing-Based RNA Structure Probing Data with User-Friendly Tools](http://dx.doi.org/10.1016/bs.mie.2015.01.014), by Lukasz J. Kielpinski, Nikolaos Sidiropoulos, Jeppe Vinther, *Methods in Enzymology* (2015), doi:10.1016/bs.mie.2015.01.014

* [The Globus Galaxies Platform: Delivering Science Gateways as a Service](http://1.usa.gov/1I223bq), by Madduri, R, Chard, K, Chard, R, Lacinski, L, Rodriguez, A, Sulakhe, D, Kelly, D, Dave, U, Foster, I, Argonne National Laboratory Report, 2015

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 19 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 33 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


# Events

## April GalaxyAdmins Meetup

<div class='right'><a href='/Community/GalaxyAdmins/Meetups/2015_04_16'><img src='/Images/Logos/GalaxyAdmins.png' alt='GalaxyAdmins meetup April 16' width="150" /></a></div>

The next [GalaxyAdmins online meetup](/src/Community/GalaxyAdmins/Meetups/2015_04_16/index.md) will be [16 April](/src/Community/GalaxyAdmins/Meetups/2015_04_16/index.md). [Carrie Ganote](http://www.researchgate.net/profile/Carrie_Ganote) from the [National Center for Genome Analysis Support (NCGAS)](http://ncgas.org/) and [Pervasive Technology Institute](http://pti.iu.edu/) at [Indiana University](http://iu.edu/) will talk about her Galaxy work with [Trinity](http://trinityrnaseq.github.io/), IU Galaxy, and the [Open Science Grid](http://www.opensciencegrid.org/).

See the [meetup page](/src/Community/GalaxyAdmins/Meetups/2015_04_16/index.md) for more.

[GalaxyAdmins](/src/Community/GalaxyAdmins/index.md) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  

## Galaxy Workshop Tokyo, April 28

<div class='left'>
<a href='/Events/Tokyo2015'><img src='/Events/Tokyo2015/WST2015.png' alt='' width="170" /></a>
</div>

The [Galaxy Workshop Tokyo 2015](/src/Events/Tokyo2015/index.md) is a full day of hands-on training, keynotes, lightning talks and discussions all about ways of using Galaxy for high-throughput biology, specially for human genome sequencing. This workshop will consist of two parts: The morning session is a hands-on training to learn how to run [Pitagora-Galaxy](http://www.pitagora-galaxy.org/about_en), our *Galaxy preconfigured virtual machine*, on your laptop or on AWS cloud. The afternoon session includes keynotes and lightning sessions to explain actual workflows, which *you can try immediately* on Pitagora-Galaxy.<br /><br />

## GCC2015: 6-8 July, Norwich UK

<div class='right'><a href='http://gcc2015.tsl.ac.uk/'><img src='/Images/Logos/GCC2015LogoWide600.png' alt='Sponsor GCC2015' width="170" /></a></div>

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is the Galaxy community's annual gathering of users, developers, and administrators.  [Previous GCC's](/src/Events/GCC2014/index.md) have drawn over 200 participants, and we expect that to happen again in 2015.  GCC2015 is being hosted by [The Sainsbury Lab](http://tsl.ac.uk/) in Norwich, UK, immediately before [BOSC](http://www.open-bio.org/wiki/BOSC_2015) and [ISMB/ECCB](http://www.iscb.org/ismbeccb2015) in Dublin.

There are a lot of events going on at GCC2015, including:

* [Code Hackathon](#code-hackathon), 4-5 July
* [Data Wrangling Hackathon](#data-wrangling-hackathon), 4-5 July (*new*)
* [Training SunDay](#training-sunday), 5 July (*new*)
* [Training Day](#training-monday), 6 July
* [GCC Meeting](#abstract-submission-is-now-open), 7-8 July

### Code Hackathon

An [intense two-day hands-on collaboration](http://gcc2015.tsl.ac.uk/Hackathon) to *develop working code that is useful to the Galaxy community*. If you know how to code, and want to contribute to one of the most successful open source projects in the life sciences, then please consider attending. See the [Code Hackathon](http://gcc2015.tsl.ac.uk/Hackathon) home page for more.

### Data Wrangling Hackathon

An [intense two-day hands-on collaboration](http://gcc2015.tsl.ac.uk/Data-Hackathon) to *develop cutting edge analysis pipelines that are useful to the Galaxy community*. If you know data analysis, we would love to have you here to help us beat back those seemingly unsurmountable analysis challenges. See the [Data Wrangling Hackathon home page](http://gcc2015.tsl.ac.uk/Data-Hackathon) for more.

### Training SunDay

Something new for GCC2015 is *[Training SunDay](http://gcc2015.tsl.ac.uk/training-day/)*, an additional day of training offered the day before its older sibling [Training Day](http://gcc2015.tsl.ac.uk/training-day/), and featuring a single track with the most in-demand topics. You can attend both Training Days, or just one.  Training !SunDay features these three topics:

* [Introduction to Galaxy](http://gcc2015.tsl.ac.uk/training-day/#Introduction_to_Galaxy)
* [RNA-Seq Analysis with Galaxy](http://gcc2015.tsl.ac.uk/training-day/#RNA-Seq_Analysis_with_Galaxy)
* [Visualisation of NGS Data](http://gcc2015.tsl.ac.uk/training-day/#Visualisation_of_NGS_Data)

These three topics are also offered on Monday as well.  You can register for one or both Training Days.

<div class='right'><br /><a href='http://gcc2015.tsl.ac.uk/training-day/'><img src='/Images/Logos/GCC2015_TrainingDay128.png' alt='Training Day Schedules set' width="100" /></a>
</div>

### Training (Mon)Day

The schedule for [Training Day, Monday, 6 July is available](http://gcc2015.tsl.ac.uk/training-day/). Training Day featuring five parallel tracks, each with three, two and a half hour workshops. There are topics on [using Galaxy](http://gcc2015.tsl.ac.uk/training-day/#Using_Galaxy), [interacting with it programmatically](http://gcc2015.tsl.ac.uk/training-day/#Using_Galaxy_Programmatically), and [deploying, administering, and extending Galaxy](http://gcc2015.tsl.ac.uk/training-day/#Deploying_Administering_and_Wrapping_Tools_for_Galaxy). No matter what you do with Galaxy, there are workshops for you.

### Early Registration opens ...

*Early* [registration](http://gcc2015.tsl.ac.uk/abstracts/registration/) (save *heaps)* will open in April, we promise. Early registration is very affordable and starts at less than £40 per day for students and postdocs. If you work in data-intensive life science research, then it is hard find a meeting more relevant than GCC2015.  We look forward to seeing you there.

### Paper Abstract Submission Extended to April 20

[Abstract submission for Oral and Poster Presentations](http://gcc2015.tsl.ac.uk/Abstracts) at the [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is now open.

[Abstract submission](http://gcc2015.tsl.ac.uk/Abstracts) for oral presentations closes ~~10~~ 20 April, while poster submission closes 1 May. Poster authors will be notified of acceptance status within two weeks of submission, while oral presentation authors will be notified no later than 4 May.  *Please consider presenting your work. If you are dealing with big biological data, then this meeting wants to hear about it.*

 
### GCC2015 Sponsorships

<div class='center'> 
<a href='https://www.sgi.com/solutions/research/'><img src='/Images/Logos/SGI_logo_platinum_400.png' alt='SGI' height="80" /></a> &nbsp;&nbsp;
<a href='http://www.intel.com/'><img src='/Images/Logos/IntelLogo600.png' alt='Intel' height="70" /></a> &nbsp;&nbsp;
<a href='http://www.kelway.com/'><img src='/Images/Logos/KelwayLogo700.png' alt='Kelway' height="58" /></a>
</div>
<br />

We are pleased to announce a joint GCC2015 [Platinum Sponsorship](http://gcc2015.tsl.ac.uk/sponsors/) from [SGI](https://www.sgi.com/solutions/research/), [Intel](http://www.intel.com/), and [Kelway](http://www.kelway.com/).  Please welcome Intel and Kelway to the community, and welcome SGI back!

#### Call for Sponsors

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is still accepting [Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/).  Your organisation can play a prominent part in the Galaxy community by sponsoring GCC2015.  Sponsorship is an excellent way to raise your organization’s visibility.

Several sponsorship levels are available, including two levels of premier sponsorships that include presentations.  Premium sponsorships are limited, however, so you are encouraged to act soon.

Please let the [organisers|GCC2015 Organising Committee](http://gcc2015.tsl.ac.uk/) ([gcc2015-org AT lists DOT galaxyproject DOT org](gcc2015-org AT lists DOT galaxyproject DOT org)) know if you are interested in helping make this event a success.


## Other Events

<div class='center'> 
<a href='https://www.e-biogenouest.org/events/details/45/'><img src='/Images/Logos/eBiogenouestLogo.png' alt='Fourth GUGGO meeting' height="80" /></a> &nbsp;
<a href='/Community/GalaxyAdmins/Meetups/2015_04_16'><img src='/Images/Logos/GalaxyAdmins.png' alt='16 April GalaxyAdmins Web Meetup' height="56" /></a> &nbsp;
<a href='http://www.qfab.org/events/upcoming/'><img src='/Images/Logos/QFABLogo.png' alt='QFAB Workshops' height="80" /></a>
<a href='http://www.bio-itworldexpo.com/'><img src='/Images/Logos/Bio-ITWorld2015.jpg' alt='Bio-IT World 2015' height="60" /></a>
<a href='http://bioinformatics.ca/workshops/2015/high-throughput-biology-sequence-networks-2015'><img src='/Images/Logos/BioinformaticsCA_NYC.jpg' alt='High-throughput Biology: From Sequence to Networks 2015' height="70" /></a>
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif' /></a></div>

There are [upcoming events](/src/Events/index.md) in 8 countries on 4 continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> April 7 </th>
    <td> </em><a href='https://www.e-biogenouest.org/events/details/45//'>5th GUGGO meeting</a><em> </td>
    <td> <div class='right'><a href='/Teach/Trainers#galaxy_user_group_grand_ouest_28guggo29'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div><a href='https://www.e-biogenouest.org/groups/guggo'>Galaxy User Group Grand Ouest (GUGGO), Rennes, France</a> </td>
    <td> <a href='https://www.e-biogenouest.org/members/1005'>Cyril Monjeaud, Yvan Le Bras</a> </td>
  </tr>
  <tr>
    <th> April 14-15 </th>
    <td> </em><a href='https://www.globusworld.org/program'>Swift Parallel Scripting for Science, Engineering and Data Analysis</a><em> </td>
    <td> <a href='http://www.globusworld.org/'>GlobusWorld 2015</a>, Chicago, Illinois, United States </td>
    <td> <a href='http://www.ci.anl.gov/profile/342'>Mike Wilde</a> </td>
  </tr>
  <tr>
    <th> April 16 </th>
    <td> </em><a href='/Community/GalaxyAdmins/Meetups/2015_04_16'>GalaxyAdmins April 2015 Meetup</a><em> </td>
    <td> </em>Online<em> </td>
    <td> <a href='/HansrudolfHotz'>Hans-Rudolf Hotz</a>, <a href='/DaveClements'>Dave Clements</a>, Carrie Ganote </td>
  </tr>
  <tr>
    <th> April 20-21 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-2/'>Workshop: Extended RNA-Seq analysis</a><em> </td>
    <td> <div class='right'><a href='/Teach/Trainers#qfab-bioinformatics'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th rowspan=3> April 21 </th>
    <td> </em><a href='http://www.bio-itworldexpo.com/Next-Gen-Sequencing-Informatics/'>Large-Scale NGS Analysis Using Globus Genomics: Challenges and User Success Stories</a><em> </td>
    <td rowspan=4> <a href='http://www.bio-itworldexpo.com/'>Bio-IT World 2015</a>, Boston, Massachusetts, United States </td>
    <td> <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a> </td>
  </tr>
  <tr>
    <td> </em><a href='http://www.bio-itworldexpo.com/Bio-It_Expo_Content.aspx?id=144286'>Workshop: Large Scale NGS Analysis Using Globus Genomics</a><em> </td>
    <td> Paul Davé, <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a>, Alex Rodriguez, Dinanath Sulakhe </td>
  </tr>
  <tr>
    <td> </em><a href='http://www.bio-itworldexpo.com/Bioinformatics/'>Extending Galaxy with External Microbiome Databases</a><em> </td>
    <td> Bob Brown </td>
  </tr>
  <tr>
    <th> April 23 </th>
    <td> </em><a href='http://www.bio-itworldexpo.com/Data-Visualization/'>Visualization Tools for the Refinery Platform</a><em> </td>
    <td> <a href='http://gehlenborg.com/'>Nils Gehlenborg</a> </td>
  </tr>
  <tr>
    <th> April&nbsp;27 - May 3 </th>
    <td> </em><a href='http://bioinformatics.ca/workshops/2015/high-throughput-biology-sequence-networks-2015'>High-throughput Biology: From Sequence to Networks 2015</a><em> </td>
    <td> <div class='right'><a href='/Teach/Trainers#bioinformaticsca'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> New York City, New York, United States </td>
    <td> <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> April 28 </th>
    <td> </em><a href='/Events/Tokyo2015'>Galaxy Workshop Tokyo 2015</a><em> </td>
    <td> <a href='http://www.rcast.u-tokyo.ac.jp/index_en.html'>RCAST, The University of Tokyo</a>, Japan </td>
    <td> <a href='http://wiki.pitagora-galaxy.org/wiki/index.php/Galaxy_Workshop_Tokyo_2015#Contact'>Ryota Yamanaka</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> May 12 </th>
    <td> </em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module17-12mai.pdf'>Initiation à l’utilisation de Galaxy</a><em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
    <td rowspan=2> <a href='http://www.jouy.inra.fr/'>Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin@jouy.inra.fr">Veronique Martin</a>, <a href="mailto:sophie.schbath@jouy.inra.fr">Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> May 13 </th>
    <td> </em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module8bis-13mai.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a><em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
  </tr>
  <tr>
    <th> May 13 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-variant-detection-using-galaxy-3/'>Workshop: Variant detection using Galaxy</a><em> </td>
    <td> <div class='right'><a href='/Teach/Trainers#qfab-bioinformatics'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> May 25-29 </th>
    <td> <a href='http://www.mipro.hr/MIPRO2015.DCVIS/ELink.aspx'>MIPRO</a> </td>
    <td> Opatija, Croatia </td>
    <td> <a href='/EnisAfgan'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> May 26-28 </th>
    <td> <a href='http://w3lamc.umbr.cas.cz/repeatexplorer/?page_id=14'>Workshop on the Application of Next Generation Sequencing to Repetitive DNA Analysis in Plants</a> </td>
    <td> České Budějovice, Czech Republic </td>
    <td> <a href="mailto:macas@umbr.cas.cz">Jiri Macas</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 2 </th>
    <td> </em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module17-2juin.pdf'>Initiation à l’utilisation de Galaxy</a><em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
    <td rowspan=3> <a href='http://www.jouy.inra.fr/'>Jouy-en-Josas</a>, France </td>
    <td rowspan=3> <a href="mailto:veronique.martin@jouy.inra.fr">Veronique Martin</a>, <a href="mailto:sophie.schbath@jouy.inra.fr">Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 3 </th>
    <td> </em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module8bis-3juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a><em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
  </tr>
  <tr>
    <th> June 4 </th>
    <td> </em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module18.pdf'>Traitement bioinformatique des données RNA-Seq sous Galaxy</a><em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
  </tr>
  <tr>
    <th> June 3 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-rna-seq-analysis-using-galaxy-2/'>Workshop: RNA-Seq analysis using Galaxy</a><em> </td>
    <td> <div class='right'><a href='/Teach/Trainers#qfab-bioinformatics'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> June 22 </th>
    <td> </em>Introduction to variant calling for pathologists and laboratory managers<em> </td>
    <td> <div class='right'><a href='/Teach/Trainers'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div> Part of <a href='http://www.rcpa.edu.au/Events/Event-Details/2015/June/Short-Course-in-Medical-Genetics-and-Genetic-Patho'>Short Course in Medical Genetics and Genetic Pathology 2015</a>, Sydney, Australia </td>
    <td> <a href='/RossLazarus'>Ross Lazarus</a>, <a href='https://www.vlsci.org.au/researcher/alonie'>Andrew Lonie</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> </strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a><strong> </td>
    <td> <div class='right'><a href='/Teach/Trainers'><img src='/Images/Icons/GTN32.png' alt='Training offered by GTN Member' width="16" /></a></div>   <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
  <tr>
    <th> July 10-14 </th>
    <td> </em><a href='http://www.iscb.org/ismbeccb2015-program/ismbeccb2015-akes#akes4'>Using Biological Cyberinfrastructure to Scale Science and People – Applications in Data Storage, HPC, Cloud Analysis, and Bioinformatics Training</a><em> </td>
    <td> <a href='http://www.iscb.org/ismbeccb2015'>ISMB / ECCB 2015</a>, <a href='http://www.open-bio.org/wiki/BOSC_2015'>BOSC 2015</a>, Dublin, Ireland </td>
    <td> <a href='/DaveClements'>Dave Clements</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/Teach/Trainers'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/Teach/Trainers'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


---

# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Graph-based Genomes: PhDs in Oslo](/src/News/GraphBasedGenomeOpeningsOslo/index.md)
* [Computational Metabolomics Professor](https://psu.jobs/job/55363), Penn State University, Pennsylvania, United States
* [Software Developer for the Refinery Team](https://jobs.brassring.com/tgwebhost/jobdetails.aspx?jobId=1118216&PartnerId=25240&SiteId=5341&type=mail&JobReqLang=1&recordstart=1&JobSiteId=5341&JobSiteInfo=1118216_5341&gqid=0), Boston, Massachusetts, United States
* **The [Galaxy Project is hiring software engineers and post-docs](/src/GalaxyIsHiring/index.md)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

---

# New Public Servers

One new [public Galaxy server](/src/PublicGalaxyServers/index.md) was added in March:

## Vinther Lab

<div class='right solid'><a href='http://galaxy.bio.ku.dk/'><img src='/PublicGalaxyServers/VintherLab300.png' alt='Vinther Lab: User-Friendly Tools for Sequencing-Based RNA Structure Probing Data'  /></a></div>

* *Links:*
  * [Vinther Lab server](http://galaxy.bio.ku.dk/)
  * [Reproducible Analysis of Sequencing-Based RNA Structure Probing Data with User-Friendly Tools](http://www.sciencedirect.com/science/article/pii/S0076687915000713) by Lukasz Jan Kielpinski, Nikolaos Sidiropoulos, Jeppe Vinther, * Methods in Enzymology*, DOI: 10.1016/bs.mie.2015.01.014:
* *Domain/Purpose:*
  * RNA structure-probing data analysis to "improve the prediction of RNA secondary and tertiary structure and allow structural changes to be identified and investigated." 
* *Comments:*
  * From [Kielpinski, et al.](http://www.sciencedirect.com/science/article/pii/S0076687915000713):
      a collection of tools, which allow raw sequencing reads to be converted to normalized probing values using different published strategies. In addition, we also provide tools for visualization of the probing data in the UCSC Genome Browser and for converting RNA coordinates to genomic coordinates and vice versa. The collection is implemented as functions in the R statistical environment and as tools in the Galaxy platform, making them easily accessible for the scientific community. 
* *User Support:*
* *Quotas:* 
  * Must have a login to use the server; anyone can create a login.
* *Sponsor(s):*
  * [Vinther Lab](http://www1.bio.ku.dk/english/research/bi/rna_biologi/vinther/), [Department of Biology](http://cms.ku.dk/nat-sites/bio-sites/bio/english/), [University of Copenhagen](http://www.ku.dk/english/).
  * [Danish Council for Strategic Research](http://ufm.dk/en/research-and-innovation/councils-and-commissions/the-danish-council-for-strategic-research/the-danish-council-for-strategic-research) [Center for Computational and Applied Transcriptomics](http://rna.dk/)

---

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Teach'><img src='/Images/Logos/GTNLogo600.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Community/Logs'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Community/Deployments'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong>Share your <a href='/Teach/Resources#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers#add-a-trainer'>experience</a> </em>now<em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/Community/Logs#add-a-log-page'>Share your experience</a> </em><a href='/Community/Logs#add-a-log-page'>now</a><em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/Community/Deployments#add-your-galaxy-deployment'>Describe your instance</a> </em><a href='/Community/Deployments#add-your-galaxy-deployment'>now</a><em> <strong> </td>
  </tr>
</table>

<br />

One new [Community Log Board](/src/Community/Logs/index.md) entry was added in March:

* [Installing ANNOVAR in Galaxy](/src/Community/Log/2015/InstallingANNOVAR/index.md)
    by [Peter Briggs](http://www.ls.manchester.ac.uk/people/profile/?alias=briggsp)

---

# Releases

## March 2015 Galaxy Release (v 15.03)

<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GetGalaxyOrg.png' alt='width=175' /></a></div>

<span style="font-size: larger;">[Complete Release Notes](http://wiki.galaxyproject.org/DevNewsBriefs/2015_03) </span>
<br /><br />
<span style="font-size: larger;"> Highlights </span>

 **[Release Versioning](/src/DevNewsBriefs/2015_03/index.md#release_versioning)**

  Starting with this distribution, an updated Galaxy release versioning system has been implemented. The versioning scheme is [Ubuntu-style](https://wiki.ubuntu.com/Releases).

 **[Github](/src/DevNewsBriefs/2015_03/index.md#github)**

  Galaxy development has moved to Github, but stable/release changes are mirrored to Bitbucket. Deployers can continue to use Bitbucket as they have done in the past. Release branches discussed in the full release notes.

 **[Tool Redesign](/src/DevNewsBriefs/2015_03/index.md#tool_redesign)**

  Much of Galaxy’s core tool set has been redesigned. Several contain new functionality. These tools are included in the [Tool Shed](http://usegalaxy.org/toolshed) and many are ready for use on Galaxy [/Main](/src/Main/index.md).

<span style="font-size: larger;"> Get the Galaxy Release </span>
<br />
<table>
  <tr>
    <td rowspan=3 style=" border: none;"> <a href='http://getgalaxy.org/'><img src='http://galaxy.psu.edu/static/getgalaxy.png' alt='getgalaxy' width=70 /></a> &nbsp;&nbsp; </td>
    <td colspan=2 style=" border: none;"> </strong><a href='http://wiki.galaxyproject.org/Admin/Get%20Galaxy'>getgalaxy.org</a><strong> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong><a href='http://galaxy-dist.readthedocs.org'>galaxy-dist.readthedocs.org</a><strong> </td>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong><a href='http://bitbucket.org/galaxy/galaxy-dist'>bitbucket.org/galaxy/galaxy-dist</a><strong> </td>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong>new<strong>: </td>
    <td style=" border: none;"> <code>$ hg clone https://bitbucket.org/galaxy/galaxy-dist#stable </code> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong>upgrade<strong>: </td>
    <td style=" border: none;"> <code>$ hg pull </code> <br /> <code>$ hg update latest_15.03</code> </td>
  </tr>
</table>


<br /><br />
*Thanks for using Galaxy!* <br />
[The Galaxy Team](/src/GalaxyTeam/index.md)

## BioBlend v0.5.3 Released

<div class='right'><a href='/CloudMan'><img src='/Images/Logos/CloudManWideBlackLogo.png' alt='CloudMan' width="200" /></a></div>

[BioBlend](https://github.com/afgane/bioblend) [v0.5.3](https://github.com/galaxyproject/bioblend/blob/master/CHANGELOG.md) has been released.  [BioBlend](https://github.com/galaxyproject/bioblend/blob) is a python library for interacting with CloudMan and the [Galaxy API](/src/Learn/API/index.md).  (CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)

This is mostly an incremental bug fix release with the following summary of changes:

* Project source moved to new URL - https://github.com/galaxyproject/bioblend 
* Huge improvements to automated testing, tests now run against Galaxy release_14.02 and all later versions to ensure backward compatibility (see [travis.yml](https://github.com/galaxyproject/bioblend/blob/master/.travis.yml) for details).
* Many documentation improvements (thanks to [Eric Rasche](/src/EricRasche/index.md)).
* Add Galaxy clients for the tool data tables, the roles, and library folders (thanks to Anthony Bretaudeau).
* Add method to get the standard error and standard output for the job corresponding to a Galaxy dataset (thanks to Anthony Bretaudeau).
* Add `get_state()` method to `JobsClient`.
* Add `copy_from_dataset()` method to `LibraryClient`.
* Add `create_repository()` method to `ToolShedClient` (thanks to [Eric Rasche)](/src/EricRasche/index.md).
* Fix `DatasetClient.download_dataset()` for certain proxied Galaxy deployments.
* Make `LibraryClient._get_root_folder_id()` method safer and faster for Galaxy release_13.06 and later.
* Deprecate and ignore invalid deleted parameter to `WorkflowClient.get_workflows()`.
* CloudMan: Add method to fetch instance types.
* CloudMan: Update cluster options to reflect change to SLURM.
* [BioBlend.objects](http://bioinformatics.oxfordjournals.org/content/30/19/2816.abstract): Deprecate and ignore invalid deleted parameter to `ObjWorkflowClient.list()`.
* !BioBlend.objects: Add `paste_content()` method to `History` objects.
* !BioBlend.objects: Add `copy_from_dataset()` method and `root_folder` property to `Library` objects.
* !BioBlend.objects: Add `container` and `deleted` attributes to `Folder` objects.
* !BioBlend.objects: Set the `parent` attribute of a `Folder` object to its parent folder object (thanks to John M. Eppley).
* !BioBlend.objects: Add `deleted` parameter to `list()` method of libraries and histories.
* !BioBlend.objects: Add `state` and `state_details` attributes to `History` objects (thanks to Gianmauro Cuccuru).
* !BioBlend.objects: Rename `upload_dataset()` method to `upload_file()` for History objects.
* !BioBlend.objects: Rename `input_ids and output_ids` attributes of `Workflow objects` to `source_ids` and `sink_ids` respectively.
* Add `run_bioblend_tests.sh` script (useful for Continuous Integration testing).

Enjoy and please let us know what you think,

[Enis](/src/EnisAfgan/index.md) & [John](/JohnChilton) & [Nicola Soranzo](/NicolaSoranzo) & Simone Leo & [Eric Rasche](/src/EricRasche/index.md)

## Planemo 0.6.0

[Planemo 0.6.0](https://pypi.python.org/pypi/planemo/0.6.0) was released in March.  The Release Notes:


* Many enhancements to the tool building documentation - descriptions of macros, collections, simple and conditional parameters, etc…
* Fix tool_init to quote file names (thanks to Peter Cock). [Pull Request 98](https://github.com/galaxyproject/planemo/pull/98).
* Allow ignoring file patterns in .shed.yml (thanks to [Björn Grüning](/src/BjoernGruening/index.md)). [Pull Request 99](https://github.com/galaxyproject/planemo/pull/98)
* Add --macros flag to tool_init command to generate a macro file as part of tool generation. [ec6e30f](https://github.com/galaxyproject/planemo/commit/ec6e30f)
* Add linting of tag order for tool XML files. [4823c5e](https://github.com/galaxyproject/planemo/commit/4823c5e)
* Add linting of stdio tags in tool XML files. [8207026](https://github.com/galaxyproject/planemo/commit/8207026)
* More tests, much higher test coverage. [0bd4ff0](https://github.com/galaxyproject/planemo/commit/0bd4ff0)

Planemo is a set of command-line utilities to assist in building tools for the Galaxy project 

## CloudMan and blend4j

New versions [CloudMan](/src/GalaxyUpdates/2014_09/index.md#august-2014-cloudman-release), and [blend4j](/src/GalaxyUpdates/2014_09/index.md#blend4j-011-release) were released in August.

---

# Other News

* From [Björn Grüning](/src/BjoernGruening/index.md): 
    [New release of our Galaxy Docker container](https://biostar.usegalaxy.org/p/11376/) with a lot of new features 
* You can browse our tutorials section by visiting this [biostar.usegalaxy.org/t/Tutorial/|Bostar post](https://biostar.usegalaxy.org/t/Tutorial/). 
* [Should Galaxy use Trello or Github for issue tracking?](http://trello.com/c/iddlQT2p)
* Every contribution and every pull request to the Galaxy repo is now [being publicly built on Travis-CI](https://travis-ci.org/galaxyproject/galaxy/builds).
* [Galaxy Training Network (GTN) Joins GOBLET](/src/News/GTNJoinsGOBLET/index.md)

---

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

A [best practices for creating Galaxy Tools](/src/Tools/BestPractices/index.md) is now available on this wiki.  Thanks to the many contributors who created it.

## Galaxy Project ToolShed Repos

Note: Starting with the May news, this list will be placed on a separate page and linked to from here.  This section is just getting too long (which is the kind of problem we want to have :-).

### Suites

* *From [biomonika](https://toolshed.g2.bx.psu.edu/view/biomonika):*
  * [suite_linkyx_1_0](https://toolshed.g2.bx.psu.edu/view/biomonika/suite_linkyx_1_0): Metapackage for the installation of linkyx suite of tools.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [suite_vcflib_14_08](https://toolshed.g2.bx.psu.edu/view/devteam/suite_vcflib_14_08): A collection of Galaxy wrappers for tools for manipulation of VCF files from 8/2014

### Tools

* *From [galaxyp](https://toolshed.g2.bx.psu.edu/view/galaxyp):*
  * [ms_wiff_loader](https://toolshed.g2.bx.psu.edu/view/galaxyp/ms_wiff_loader): Loads AB Sciex wiff files from URLs into a Galaxy Wiff Composite dataset
  * [ms_data_converter](https://toolshed.g2.bx.psu.edu/view/galaxyp/ms_data_converter): Converts WIFF format MS data to mzML or MGF using AB SCIEX MS Data Converter

* *From [rnateam](https://toolshed.g2.bx.psu.edu/view/rnateam):*
  * [kinwalker](https://toolshed.g2.bx.psu.edu/view/rnateam/kinwalker): The Kinwalker algorithm performs cotranscriptional folding of RNAs, starting at a user a specified structure (default: open chain) and ending at the minimum free energy structure. Folding events are performed between transcription of additional bases and are regulated by barrier heights between the source and target structure
  * [vienna_rna](https://toolshed.g2.bx.psu.edu/view/rnateam/vienna_rna): The ViennaRNA Package consists of several stand-alone programs for the prediction and comparison of RNA secondary structures.   RNA secondary structure prediction through energy minimization is the most used function in the package. We provide three kinds of dynamic programming algorithms for structure prediction: the minimum free energy algorithm of (Zuker & Stiegler 1981) which yields a single optimal structure, the partition function algorithm of (!McCaskill 1990) which calculates base pair probabilities in the thermodynamic ensemble, and the suboptimal folding algorithm of (Wuchty et.al 1999) which generates all suboptimal structures within a given energy range of the optimal energy. For secondary structure comparison, the package contains several measures of distance (dissimilarities) using either string alignment or tree-editing (Shapiro & Zhang 1990). Finally, we provide an algorithm to design sequences with a predefined structure (inverse folding).
      In case you are using our software for your publications you may want to cite:

     Lorenz, Ronny and Bernhart, Stephan H. and H\u00f6ner zu Siederdissen, Christian and Tafer, Hakim and Flamm, Christoph and Stadler, Peter F. and Hofacker, Ivo L.<br />
     ViennaRNA Package 2.0, *Algorithms for Molecular Biology,* 6:1 26, 2011, doi:10.1186/1748-7188-6-26 

* *From [izsam](https://toolshed.g2.bx.psu.edu/view/izsam):*
  * [phylogeny_converter](https://toolshed.g2.bx.psu.edu/view/izsam/phylogeny_converter): Converts different file formats (FASTA, !GenBank, phylip, nexus) to allow data-exchange from different phylogeny tools.

* *From [biomonika](https://toolshed.g2.bx.psu.edu/view/biomonika):*
  * [trinityrnaseq](https://toolshed.g2.bx.psu.edu/view/biomonika/trinityrnaseq): De novo assembly of RNA-Seq data Using Trinity Contains Trinity only. See [trinityrnaseq in Test Tool Shed](https://testtoolshed.g2.bx.psu.edu/view/jjohnson/trinityrnaseq) for additional tools.

* *From [jjkoehorst](https://toolshed.g2.bx.psu.edu/view/jjkoehorst):*
  * [sapp](https://toolshed.g2.bx.psu.edu/view/jjkoehorst/sapp): GBK2RDF Semantic Annotation Platform for Prokaryotes. It might take a while but I will try to make for each module in the SAPP paper a galaxy tool shed module.

* *From [fastaptamer](https://toolshed.g2.bx.psu.edu/view/fastaptamer):*
  * [fastaptamer_cluster](https://toolshed.g2.bx.psu.edu/view/fastaptamer/fastaptamer_cluster): Cluster closely-related sequences using Levenshtein edit distance. FASTAptamer-Cluster uses the Levenshtein algorithm to cluster together closely-related sequences based on a user-defined edit distance (the minimum number of insertions, deletions, or subsitutions required to transform one string into another). 

* [fastaptamer_count](https://toolshed.g2.bx.psu.edu/view/fastaptamer/fastaptamer_count): Count, rank, sort and normalize sequence reads in a selection population. FASTAptamer-Count serves as the gateway to the FASTAptamer suite of bioinformatics tools for combinatorial selections (aptamers, (deoxy)ribozymes, phage display, direct mutagenesis, etc.). For a given FASTQ input file, FASTAptamer-Count will determine the number of times each sequence was read, normalize sequence frequency to reads per million, and rank and sort sequences by decreasing total reads.
* [fastaptamer_compare](https://toolshed.g2.bx.psu.edu/view/fastaptamer/fastaptamer_compare): Compare sequence distribution between two populations. FASTAptamer-Compare facilitates statistical analysis of two populations by rapidly generating a tab-delimited output file that lists each unique sequence along with RPM (reads per million) in each population file (if available) and log(2) of the ratio of their RPM values in each population.  RPM data for both populations can be utilized to generate an XY-scatter plot of sequence distribution across two populations.  FASTAptamer-Compare also facilitates the generation of a histogram of the sequence distribution by creating 102 bins for the log(2) values.  This histogram can provide a quick visual comparison of the two populations: distributions centered around 0 indicate similar populations, while distributions shifted to the left or right indicate overall enrichment or depletion.
* [fastaptamer_search](https://toolshed.g2.bx.psu.edu/view/fastaptamer/fastaptamer_search): Degenerate nucleotide motif searching. FASTAptamer-Search searches for degenerate nucleotide motifs within a FASTA file.

* [fastaptamer_enrich](https://toolshed.g2.bx.psu.edu/view/fastaptamer/fastaptamer_enrich): Calculate fold-enrichment of each sequence across populations. FASTAptamer-Enrich rapidly calculates "fold-enrichment" values for each sequence across two or three input files.  Output is provided as a tab-delimited file and is formatted to include sequence composition, length, rank, reads, reads per million (RPM), cluster information (if available) and enrichment values for each sequence. 


* *From [bgruening](https://toolshed.g2.bx.psu.edu/view/bgruening):*
  * [diamond](https://toolshed.g2.bx.psu.edu/view/bgruening/diamond): DIAMOND is a new high-throughput program for aligning a file of short reads against a protein reference database such as NR, at 20,000 times the speed of BLASTX, with high sensitivity

     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/bgruening/galaxytools
* [text_processing](https://toolshed.g2.bx.psu.edu/view/bgruening/text_processing): High performance text processing tools using the GNU coreutils, sed, awk and friends. That repository contains all kind of different text processing tools. 

* awk - The AWK programmning language ( http://www.gnu.org/software/gawk/ )
* sed - Stream Editor ( http://sed.sf.net )
* grep - Search files ( http://www.gnu.org/software/grep/ )
* sort_columns - Sorting every line according to there columns
* GNU Coreutils programs ( http://www.gnu.org/software/coreutils/ ):
  * sort - sort files
  * join - join two files, based on common key field.
  * cut  - keep/discard fields from a file
  * unsorted_uniq - keep unique/duplicated lines in a file
  * sorted_uniq - keep unique/duplicated lines in a file
  * head - keep the first X lines in a file.
  * tail - keep the last X lines in a file.

     Originally known as "Unix Tools" and developed from Assaf Gordon @ Greg Hannon's lab ( http://hannonlab.cshl.edu ) in Cold Spring Harbor Laboratory, it is now hosted under https://github.com/bgruening/galaxytools/tree/master/unix_tools and open for contributions. It will also replace several smaller sed, sort and uniq wrappers, developed over the time.<br />
     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/bgruening/galaxytools

* [data_manager_diamond_database_builder](https://toolshed.g2.bx.psu.edu/view/bgruening/data_manager_diamond_database_builder): Diamond data manager 
* [find_genes_located_nearby_workflow](https://toolshed.g2.bx.psu.edu/view/bgruening/find_genes_located_nearby_workflow): Galaxy workflow for the identification of candidate genes clusters This approach screens two proteins against all nucleotide sequence from the NCBI nt database within hours on our cluster, leading to all organisms with an interesting gene structure for further investigation. As usual in Galaxy workflows every parameter, including the proximity distance, can be changed and additional steps can be easily added. For example additional filtering to refine the initial BLAST hits, or inclusion of a third query sequence.
* [find_three_genes_located_nearby_workflow](https://toolshed.g2.bx.psu.edu/view/bgruening/find_three_genes_located_nearby_workflow): Galaxy workflow for the identification of candidate genes clusters with three known genes This approach screens three proteins against a given genome sequence, leading to a genome position were all three genes are located nearby. As usual in Galaxy workflows every parameter, including the proximity distance, can be changed and additional steps can be easily added. For example additional filtering to refine the initial BLAST hits, or inclusion of a third query sequence.
* [find_subsequences](https://toolshed.g2.bx.psu.edu/view/bgruening/find_subsequences): Searches for a subsequence in a larger sequence. For example to get all restriction enzymes for !BamH1. Searches for a subsequence in a larger sequence. For example to get all restriction enzymes for !BamH1.

     This tool is based on biopython: 10.1093/bioinformatics/btp163 <br />
     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/bgruening/galaxytools

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [macs2](https://toolshed.g2.bx.psu.edu/view/iuc/macs2): MACS - Model-based Analysis of ChIP-Seq.  With the improvement of sequencing techniques, chromatin immunoprecipitation followed by high throughput sequencing (ChIP-Seq) is getting popular to study genome-wide protein-DNA interactions. To address the lack of powerful ChIP-Seq analysis method, we present a novel algorithm, named Model-based Analysis of ChIP-Seq (MACS), for identifying transcript factor binding sites. MACS captures the influence of genome complexity to evaluate the significance of enriched ChIP regions, and MACS improves the spatial resolution of binding sites through combining the information of both sequencing tag position and orientation. MACS can be easily used for ChIP-Seq data alone, or with control sample with the increase of specificity.

     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/iuc/galaxytools
* [seqtk](https://toolshed.g2.bx.psu.edu/view/iuc/seqtk): toolkit for processing FASTA and FASTQ files. Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.<br />

     https://github.com/lh3/seqtk/ <br />
     Repository-Maintainer: [Eric Rasche](/src/EricRasche/index.md)<br />
     Repository-Development: https://github.com/galaxy-iuc/tool_shed

* *From [hogart](https://toolshed.g2.bx.psu.edu/view/hogart):*
  * [unafold](https://toolshed.g2.bx.psu.edu/view/hogart/unafold): Galaxy Tool wrapper for UNAFold This is the Galaxy wrapper for the UNAFold (http://mfold.rna.albany.edu/?q=DINAMelt/software). UNAFold software was developed for nucleic acid folding and hybridization prediction (doi: 10.1007/978-1-60327-429-6_1, doi: 10.1093/nar/gki591)
   
     Note: the UNAFold requires a big amount of RAM - e.g. the folding of 43 kb RNA eats near 30 GB of memory. So, after the installation of this wrapper you will need to modify the job_conf.xml of your Galaxy instance properly. Also, please be sure that zip datatype is registered as binary datatype in your Galaxy instance.

* *From [steffen](https://toolshed.g2.bx.psu.edu/view/steffen):*
  * [covenntree](https://toolshed.g2.bx.psu.edu/view/steffen/covenntree): !CoVennTree compares up to three rooted trees at the same time. !CoVennTree (Comparative weighted Venn Tree) is a software to analyze and compare up to three datasets. Unlike other methods, !CoVennTree correlates data on the leaf level and transfers this information to the root node. !CoVennTree works with numbers to compute weighted Venn diagrams for each node in the graph (rooted tree). Therefore any kind of input data can be processed as long as the data structure will be taken into account.

     http://journal.frontiersin.org/Journal/10.3389/fgene.2015.00043/abstract

* *From [ngsplot](https://toolshed.g2.bx.psu.edu/view/ngsplot):*
  * [ngsplot](https://toolshed.g2.bx.psu.edu/view/ngsplot/ngsplot): ngs.plot is a program that allows you to easily visualize your next-generation sequencing (NGS) samples at functional genomic regions. This galaxy implementation of ngs.plot has been tested to work with ngs.plot v2.47.1. For instructions on the system installation of ngs.plot, please see https://github.com/shenlab-sinai/ngsplot.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [vcfhethom](https://toolshed.g2.bx.psu.edu/view/devteam/vcfhethom): Count the number of heterozygotes and alleles, compute het/hom ratio. This tool perfoms three basic calculations:
    1. Computes the number of heterozygotes                                                                                                                              
    1. Computes the ratio between heterozygotes and homozygotes                                                                                                               1. Computes the total number of alleles in the input dataset               
  * [vcfselectsamples](https://toolshed.g2.bx.psu.edu/view/devteam/vcfselectsamples): Select samples from a VCF file Allows selecting samples from a VCF dataset. This tool combines vcfkeepsamples and vcfremovesamples from VCFlib package into a single utility.
  * [vcfleftalign](https://toolshed.g2.bx.psu.edu/view/devteam/vcfleftalign): Left-align indels and complex variants in VCF dataset Left-aligns variants in VCF dataset.  Window size is determined dynamically according to the entropy of the regions flanking the indel.  These must have entropy > 1 bit/bp, or be shorter than ~5kb.
  * [vcfannotategenotypes](https://toolshed.g2.bx.psu.edu/view/devteam/vcfannotategenotypes): Annotate genotypes in a VCF dataset using genotypes from another VCF dataset. Annotates genotypes in the first file with genotypes in the second adding the genotype as another flag to each sample filed in the first file. Annotation-tag is the name of the sample flag which is added to store the annotation.  also adds a 'has\_variant' flag for sites where the second file has a variant.     
  * [vcfbreakcreatemulti](https://toolshed.g2.bx.psu.edu/view/devteam/vcfbreakcreatemulti): Break multiple alleles into multiple records, or combine overallpoing alleles into a single record This tool breaks or creates multiallelic VCF records based on user selection.
    * Breaking = If multiple alleles are specified in a single record, break the record into multiple lines, preserving allele-specific INFO fields.
    * Creation = If overlapping alleles are represented across multiple records, merge them into a single record.   
  * [vcfgenotypes](https://toolshed.g2.bx.psu.edu/view/devteam/vcfgenotypes): Convert numerical representation of genotypes to allelic. Converts numerical representation of genotypes (standard in GT field) to the alleles provided in the call's ALT/REF fields. 
  * [vcfaddinfo](https://toolshed.g2.bx.psu.edu/view/devteam/vcfaddinfo): Adds info fields from the second dataset which are not present in the first dataset. Adds info fields from the second dataset which are not present in the first dataset.
  * [vcfsort](https://toolshed.g2.bx.psu.edu/view/devteam/vcfsort): Sort VCF dataset by coordinate This tools uses native UNIX sort command to order VCF dataset in coordinate order. 
  * [vcfbedintersect](https://toolshed.g2.bx.psu.edu/view/devteam/vcfbedintersect): Intersect VCF and BED datasets Computes intersection between a VCF dataset and a set of genomic intervals defined as either a BED dataset (http://genome.ucsc.edu/FAQ/FAQformat.html#format1) or a manually typed interval (in the form of chr:start-end).  
  * [vcf2tsv](https://toolshed.g2.bx.psu.edu/view/devteam/vcf2tsv): Converts VCF files into tab-delimited format Converts stdin or given VCF file to tab-delimited format, using null string to replace empty values in the table. Specifying -g will output one line per sample with genotype information. A part of the vcflib utilities developed by Erik Garrison (https://github.com/ekg/vcflib).
  * [vcfcheck](https://toolshed.g2.bx.psu.edu/view/devteam/vcfcheck): Verify that the reference allele matches the reference genome Verifies that the VCF REF field matches the reference as described
  * [vcffixup](https://toolshed.g2.bx.psu.edu/view/devteam/vcffixup): Count the allele frequencies across alleles present in each record in the VCF file. Uses genotypes from the VCF file to correct AC (alternate allele count), AF (alternate allele frequency), NS (number of called), in the VCF records.
  * [data_manager_bwa_mem_index_builder](https://toolshed.g2.bx.psu.edu/view/devteam/data_manager_bwa_mem_index_builder): Data Manager for building BWA (0.6+) indexes Data Manager for building BWA (0.6+) indexes.
  * [vcfrandomsample](https://toolshed.g2.bx.psu.edu/view/devteam/vcfrandomsample): Randomly sample sites from VCF dataset Randomly sample sites from an input VCF file. Scale the sampling probability by the field specified by --scale-by (see advanced controls).  This may be used to provide uniform sampling across allele frequencies, for instance (AF field in this case).
  * [vcfgeno2haplo](https://toolshed.g2.bx.psu.edu/view/devteam/vcfgeno2haplo): Convert genotype-based phased alleles into haplotype alleles Convert genotype-based phased alleles within a window size specified by -w option into haplotype alleles. Will break haplotype construction when encountering non-phased genotypes on input. 
  * [data_manager_fetch_genome_dbkeys_all_fasta](https://toolshed.g2.bx.psu.edu/view/devteam/data_manager_fetch_genome_dbkeys_all_fasta): Allows optionally defining a new DBKEY and retrieves a FASTA file and populate the all_fasta.loc data table. 
  * [samtools_bedcov](https://toolshed.g2.bx.psu.edu/view/devteam/samtools_bedcov): Calculate read depth on BAM files This tool uses the SAMTools toolkit to produce read depth per BED region.
  * [vcfcommonsamples](https://toolshed.g2.bx.psu.edu/view/devteam/vcfcommonsamples): Output records belonging to samples commong between two datasets. Outputs each record in the first file, removing samples not present in the second.
  * [vcfvcfintersect](https://toolshed.g2.bx.psu.edu/view/devteam/vcfvcfintersect): Intersect two VCF datasets Computes intersections and unions for two VCF datasets. Unifies equivalent alleles within window-size bp.
  * [vcfcombine](https://toolshed.g2.bx.psu.edu/view/devteam/vcfcombine): Combine multiple VCF datasets Combines VCF files positionally, combining samples when sites and alleles are identical. Any number of VCF files may be combined. The INFO field and other columns are taken from one of the files which are combined when records in multiple files match. Alleles must have identical ordering to be combined into one record. If they do not, multiple records will be emitted.
  * [vcfdistance](https://toolshed.g2.bx.psu.edu/view/devteam/vcfdistance): Calculate distance to the nearest variant. Adds a value to each VCF record indicating the distance to the nearest variant in the file.     The dataset used as input to this tool must be coordinate sorted. This can be achieved by either using the VCFsort utility or Galaxy's general purpose sort tool (in this case sort on the first and the second column in ascending order). 
  * [vcfflatten](https://toolshed.g2.bx.psu.edu/view/devteam/vcfflatten): Removes multi-allelic sites by picking the most common alternate Removes multi-allelic sites by picking the most common alternate. Requires allele frequency specification 'AF' and use of 'G' and 'A' to specify the fields which vary according to the Allele or Genotype.
  * [vcfprimers](https://toolshed.g2.bx.psu.edu/view/devteam/vcfprimers): Extract flanking sequences for each VCF record For each VCF record, extract the flanking sequences, and write them to stdout as FASTA records suitable for alignment.  This tool is intended for use in designing validation experiments.  Primers extracted which would flank all of the alleles at multi-allelic sites.
  * [vcffilter](https://toolshed.g2.bx.psu.edu/view/devteam/vcffilter): Tool for filtering VCF files A vcflib-based tool for flexible filtering of VCF datasets on a variety of tags. This is a galaxy wrapper for vcffilter utility from vcflib package.
  * [vcfallelicprimitives](https://toolshed.g2.bx.psu.edu/view/devteam/vcfallelicprimitives): Splits alleleic primitives (gaps or mismatches) into multiple VCF lines If multiple alleleic primitives (gaps or mismatches) are specified in a single VCF record, this tools splits the record into multiple lines, but drops all INFO fields.  "Pure" MNPs are split into multiple SNPs unless the -m flag is provided.  Genotypes are phased where complex alleles have been decomposed, provided genotypes in the input.     
  * [vcfannotate](https://toolshed.g2.bx.psu.edu/view/devteam/vcfannotate): Intersect VCF records with BED annotations Intersects the records in the VCF file with targets provided in a BED file. Intersections are done on the reference sequences in the VCF file.

* *From [damion](https://toolshed.g2.bx.psu.edu/view/damion):*
  * [blast_reporting](https://toolshed.g2.bx.psu.edu/view/damion/blast_reporting): Provides filtered, sorted HTML and tabular reports of Blast XML format search results NCBI BLAST+ searches can output in a range of formats, but in the past only the XML format included fields like sequence description. This tool converts the NCBI BLAST XML report into 12, 24, 26 or custom column tabular and HTML reports.  It is used as a command-line tool or via its Galaxy tool.

     The tool allows almost complete control over which fields are displayed and filtered, how columns are named, and how the HTML report on each query is sectioned.  Search result records can be filtered out based on values in numeric or textual fields.  Matches (by accession id) to a selection of reference databases can be shown, and this can include a description of the matched sequence.

* [ffp_phylogeny](https://toolshed.g2.bx.psu.edu/view/damion/ffp_phylogeny): calculating Feature Frequency Profiles (FFP) from fasta sequence and text data. FFP (Feature frequency profile) is an alignment free comparison tool for phylogenetic analysis and text comparison. It can be applied to nucleotide sequences, complete genomes, proteomes and even used for text comparison. This tool calculates FFP on one or more fasta sequence or text datasets.  It prepares a mini pipeline consisting of [ffpry | ffpaa | ffptxt] > [ ffpfilt | ffpcol > ffprwn] > ffpjsd > ffptree
   
     The original command line ffp-phylogeny code is at http://ffp-phylogeny.sourceforge.net/ . This tool uses Aaron Petkau's modified version: https://github.com/apetkau/ffp-3.19-custom.

* *From [yokofakun](https://toolshed.g2.bx.psu.edu/view/yokofakun):*
  * [jvarkit](https://toolshed.g2.bx.psu.edu/view/yokofakun/jvarkit): Tools from jvarkit https://github.com/lindenb/jvarkit Java utilities for Bioinformatics - Pierre Lindenbaum / @yokofakun https://github.com/lindenb/jvarkit 

* *From [kosrou](https://toolshed.g2.bx.psu.edu/view/kosrou):*
  * [ngs_plot](https://toolshed.g2.bx.psu.edu/view/kosrou/ngs_plot): ngs plot Novel tool to visualise next generation sequencing data around TSS, gene bodies etc

* *From [dereeper](https://toolshed.g2.bx.psu.edu/view/dereeper):*
  * [admixture](https://toolshed.g2.bx.psu.edu/view/dereeper/admixture): admixture: fast ancestry estimation ADMIXTURE is a software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets. It uses the same statistical model as STRUCTURE but calculates estimates much more rapidly using a fast numerical optimization algorithm. 
  * [snpeff_from_gff_vcf](https://toolshed.g2.bx.psu.edu/view/dereeper/snpeff_from_gff_vcf): snpeff v4.0 from VCF, fasta reference and GFF files snpeff v4.0 from VCF, fasta reference and GFF files
  * [sniplay](https://toolshed.g2.bx.psu.edu/view/dereeper/sniplay): SNiPlay3: a package for exploration and large scale analyses of SNP polymorphisms (filtering, density, vcftools, diversity, linkagedisequilibrium, GWAS) SNiPlay3: a package for exploration and large scale analyses of SNP polymorphisms (filtering, density, vcftools, diversity, linkagedisequilibrium, GWAS)
  * [tassel5](https://toolshed.g2.bx.psu.edu/view/dereeper/tassel5): Software to evaluate traits associations, evolutionary patterns, and linkage disequilibrium. Software to evaluate traits associations, evolutionary patterns, and linkage disequilibrium.

* *From [okorol](https://toolshed.g2.bx.psu.edu/view/okorol):*
  * [itsx](https://toolshed.g2.bx.psu.edu/view/okorol/itsx): ITSx -- Identifies ITS sequences and extracts the ITS region ITSx is an open source software utility to extract the highly variable ITS1 and ITS2 subregions from ITS sequences, which is commonly used as a molecular barcode for e.g. fungi.

* *From [iracooke](https://toolshed.g2.bx.psu.edu/view/iracooke):*
  * [protk_proteogenomics](https://toolshed.g2.bx.psu.edu/view/iracooke/protk_proteogenomics): Docker support and update for protk 1.4 Tools for mapping peptides and proteins to genomic coordinates 

* *From [wolma](https://toolshed.g2.bx.psu.edu/view/wolma):*
  * [mimodd_workflows](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_workflows): Some example workflows for use with MiModD The workflows defined here let you automate much of the tutorial analyses from the MiModD documentation (see http://mimodd.readthedocs.org/en/latest/tutorial.html).  These example workflows should be easy to customize for your own needs.
  * [mimodd](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd): MiModD - Identify Mutations from Whole-Genome Sequencing Data installs the MiModD suite of tools for the analysis of genome-wide sequencing data from model organisms along with their Galaxy tool wrappers.

#### Tool Dependency Definitions

* *From [dereeper](https://toolshed.g2.bx.psu.edu/view/dereeper):*
  * [package_admixture_1_23](https://toolshed.g2.bx.psu.edu/view/dereeper/package_admixture_1_23): package_admixture_1_23 package_admixture_1_23
  * [package_tassel_5_0](https://toolshed.g2.bx.psu.edu/view/dereeper/package_tassel_5_0): package to evaluate traits associations, evolutionary patterns, and linkage disequilibrium. 
  * [package_plink_1_07](https://toolshed.g2.bx.psu.edu/view/dereeper/package_plink_1_07): package_plink_1_07 

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [package_cofold_0_0_1](https://toolshed.g2.bx.psu.edu/view/iuc/package_cofold_0_0_1): Contains a tool dependency definition that downloads and compiles !CoFold. A tool for prediction of RNA secondary structure that takes co-transcriptional folding into account.

     http://www.e-rna.org/cofold/ <br />
     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/bgruening/galaxytools/
* [package_samtools_1_2](https://toolshed.g2.bx.psu.edu/view/iuc/package_samtools_1_2): Contains a tool dependency definition that downloads and installs version 1.2 of the SAMTools package.  samtools \u2212 Utilities for the Sequence Alignment/Map (SAM) format

     Samtools is a set of utilities that manipulate alignments in the BAM format. It imports from and exports to the SAM (Sequence Alignment/Map) format, does sorting, merging and indexing, and allows to retrieve reads in any regions swiftly.
     
     Samtools is designed to work on a stream. It regards an input file \u2018-\u2019 as the standard input (stdin) and an output file \u2018-\u2019 as the standard output (stdout). Several commands can thus be combined with Unix pipes. Samtools always output warning and error messages to the standard error output (stderr).
     
     Samtools is also able to open a BAM (not SAM) file on a remote FTP or HTTP server if the BAM file name starts with \u2018ftp://\u2019 or \u2018http://\u2019. Samtools checks the current working directory for the index file and will download the index upon absence. Samtools does not retrieve the entire alignment file unless it is asked to do so.

     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
* [package_gengetopt_2_22_6](https://toolshed.g2.bx.psu.edu/view/iuc/package_gengetopt_2_22_6): Contains a tool dependency definition that downloads and compiles version 2.22.6 of GNU gengetopt Gengetopt is a tool to write command line option parsing code for C programs. 

     http://www.gnu.org/software/gengetopt/gengetopt.html <br />
     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/galaxyproject/tools-iuc
* [package_rnastructure_5_7](https://toolshed.g2.bx.psu.edu/view/iuc/package_rnastructure_5_7): Contains a tool dependency definition that downloads and compiles version 5.7 of RNAstructure RNAstructure is a complete package for RNA and DNA secondary structure prediction and analysis. It includes algorithms for secondary structure prediction, including facility to predict base pairing probabilities. It also can be used to predict bimolecular structures and can predict the equilibrium binding affinity of an oligonucleotide to a structured RNA target. This is useful for siRNA design. It can also predict secondary structures common to two, unaligned sequences, which is much more accurate than single sequence secondary structure prediction. Finally, RNAstructure can take a number of different types of experiment mapping data to constrain or restrain structure prediction. These include chemical mapping, enzymatic mapping, NMR, and SHAPE data.

     http://rna.urmc.rochester.edu/RNAstructure.html
* [package_vcflib_8a5602bf07](https://toolshed.g2.bx.psu.edu/view/iuc/package_vcflib_8a5602bf07): Compiled vcflib binaries for x86_64 Binary files in this package are compiled from source code with SHA: 8a5602bf07.

     This is package dependency for tools relying on VCFlib toolkit developed by Erik Garrison (https://github.com/ekg/vcflib). This package is distributed as x86_64 binaries only as it is difficult to compile on other platforms. These binaries should work on any of the supported linux platforms other than RHEL/CentOS 5.
* [package_vienna_rna_2_1](https://toolshed.g2.bx.psu.edu/view/iuc/package_vienna_rna_2_1): Contains a tool dependency definition that downloads and compiles version 2.1 of the Vienna RNA package.  The Vienna RNA Package consists of a C code library and several stand-alone programs for the prediction and comparison of RNA secondary structures.

     http://www.tbi.univie.ac.at/RNA/ <br />
     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/bgruening/galaxytools
* [package_diamond_0_6_13](https://toolshed.g2.bx.psu.edu/view/iuc/package_diamond_0_6_13): Contains a tool dependency definition that downloads and compiles version 0.6.13 of DIAMOND DIAMOND is a new high-throughput program for aligning a file of short reads against a protein reference database such as NR, at 20,000 times the speed of BLASTX, with high sensitivity

     Repository-Maintainer: [Bjoern Gruening](/src/BjoernGruening/index.md)<br />
     Repository-Development: https://github.com/bgruening/galaxytools
* [package_stringtie_1_0_1](https://toolshed.g2.bx.psu.edu/view/iuc/package_stringtie_1_0_1): tool dependency definition. Contains a tool dependency definition that downloads and installs version 1.0.1 of the stringtie RNA-seq assembler. !StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. It uses a novel network flow algorithm as well as an optional de novo assembly step to assemble and quantitate full-length transcripts representing multiple splice variants for each gene locus. Its input can include not only the alignments of raw reads used by other transcript assemblers, but also alignments longer sequences that have been assembled from those reads.To identify differentially expressed genes between experiments, !StringTie's output can be processed either by the Cuffdiff or Ballgown programs. 

* *From [biomonika](https://toolshed.g2.bx.psu.edu/view/biomonika):*
  * [package_samtools_0_1_19_custom](https://toolshed.g2.bx.psu.edu/view/biomonika/package_samtools_0_1_19_custom): custom compilation of samtools 0.1.19 using -fPIC flag custom compilation of samtools 0.1.19 using -fPIC flag

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [package_freebayes_0_9_20_b040236](https://toolshed.g2.bx.psu.edu/view/devteam/package_freebayes_0_9_20_b040236): Contains a tool dependency definition that downloads and compiles version 0.9.20 of !FreeBayes. Program: freebayes (Bayesian haplotype-based polymorphism discovery and genotyping.)

     Version: 0.9.20 (b040236)

* *From [jjohnson](https://toolshed.g2.bx.psu.edu/view/jjohnson):*
  * [package_trinityrnaseq_2013_08_14](https://toolshed.g2.bx.psu.edu/view/jjohnson/package_trinityrnaseq_2013_08_14): Contains a tool dependency definition that downloads and compiles version 2013_08_14 of trinity RNA-Seq De novo Assembly Using Trinity
      adds Trinity.pl to PATH environment variable, set TRINITY_HOME to the installation directory, http://trinityrnaseq.sourceforge.net/index.html

    Requires perl compiled to use threads, bioperl, and perl modules: PerlIO::gzip and Bio::DB::Sam


### Select Updates

#### Tools

* *From [iracooke](https://toolshed.g2.bx.psu.edu/view/iracooke):*
  * [proteomics_datatypes](https://toolshed.g2.bx.psu.edu/view/iracooke/proteomics_datatypes): Add trafo and qcml, =improved readme
  * [sixframe_translate](https://toolshed.g2.bx.psu.edu/view/iracooke/sixframe_translate): Docker support and update for protk 1.4
  * [xtandem](https://toolshed.g2.bx.psu.edu/view/iracooke/xtandem): Docker support and update for protk 1.4
  * [make_protein_decoys](https://toolshed.g2.bx.psu.edu/view/iracooke/make_protein_decoys): Docker support and update for protk 1.4
  * [make_protein_decoys](https://toolshed.g2.bx.psu.edu/view/iracooke/make_protein_decoys): Docker support and update for protk 1.4
  * [omssa](https://toolshed.g2.bx.psu.edu/view/iracooke/omssa): Docker support and update for protk 1.4
  * [mascot](https://toolshed.g2.bx.psu.edu/view/iracooke/mascot): Docker support and update for protk 1.4
  * [mascot](https://toolshed.g2.bx.psu.edu/view/iracooke/mascot): Make Trypsin default
  * [tpp_prophets](https://toolshed.g2.bx.psu.edu/view/iracooke/tpp_prophets): Docker support and update for protk 1.4
  * [msgfplus](https://toolshed.g2.bx.psu.edu/view/iracooke/msgfplus): Docker support and update for protk 1.4


* *From [geert-vandeweyer](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer):*
  * [coverage_report](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/coverage_report): Added (default) option to collapse repetitive BED files, and added BED format check before collapsing regions.
  * [vcf_to_variantdb](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/vcf_to_variantdb): Added support for 23andMe VCF files generated by !ArrogantRobot

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/peterjc):*
  * [blast2go](https://toolshed.g2.bx.psu.edu/view/peterjc/blast2go): v0.0.9, embed citation, updated README
  * [sample_seqs](https://toolshed.g2.bx.psu.edu/view/peterjc/sample_seqs): v0.2.0, adds desired count mode
  * [sample_seqs](https://toolshed.g2.bx.psu.edu/view/peterjc/sample_seqs): v0.2.1, fixed missing test file, more tests.

* *From [crs4](https://toolshed.g2.bx.psu.edu/view/crs4):*
  * [edge_pro](https://toolshed.g2.bx.psu.edu/view/crs4/edge_pro): Add support for paired collection of FASTQ (thanks to Inge Alexander Raknes).

* *From [george-weingart](https://toolshed.g2.bx.psu.edu/view/george-weingart):*
  * [micropita](https://toolshed.g2.bx.psu.edu/view/george-weingart/micropita): Updated version that suppresses the future warnings option that was causing a problem

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [stringtie](https://toolshed.g2.bx.psu.edu/view/iuc/stringtie): updated tool wrapper for stringtie 1.0.1
