---
title: July 2016 Galaxy News
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 24em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/galaxy-updates/'><img src="/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the July 2016 *[Galactic News](/galaxy-updates/)*, a summary of what is going on in the Galaxy community.

If you have anything to include in the next News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org).


# GCC2016 is done!

<div class='right'>
<a href='https://web.archive.org/web/http://gcc2016.iu.edu/'><img src="/images/logos/GCC2016LogoTallBig.png" alt="2016 Galaxy Community Conference" width="120" /></a>
</div>

And it was again a smashing success!  For the 5th year in a row we had over 200 participants signed up for GCC events, including [hackathons](https://web.archive.org/web/http://gcc2016.iu.edu/hacks/index.php), [training](http://gcc16.sched.org/overview/type/D.x+Training+-+All), and [the two day meeting](http://gcc16.sched.org/overview/type/E.x+Conference+-+All). The [meeting](http://gcc16.sched.org/overview/type/E.x+Conference+-+All) featured [accepted presentations and keynotes](http://gcc16.sched.org/overview/type/E.1+Conference+-+Talks), [posters](http://gcc16.sched.org/overview/type/E.2+Conference+-+Posters), the new [Visualization Showcase and Software Demo sessions](http://gcc16.sched.org/overview/type/E.3+Conference+-+Demos), [lightning talks](http://gcc16.sched.org/overview/type/E.5+Conference+-+Lightning), and [birds-of-a-feather meetups](/events/gcc2016/bofs/).

[GCC2016](http://gcc2016.iu.edu/) was held June 25-29 at [Indiana University](http://indiana.edu/) in Bloomington, Indiana, United States. This was the 7th annual gathering of the Galaxy community.  

Slides for [accepted](http://gcc16.sched.org/overview/type/E.1+Conference+-+Talks) and [lightning](https://gcc16.sched.org/overview/type/E.5+Conference+-+Lightning) talks are online, as are [most posters](http://gcc16.sched.org/overview/type/E.2+Conference+-+Posters).  Remaining slides and video of training and talks will also be posted online this month.

Thanks to everyone for making GCC2016 a success, and looking forward to seeing you at GCC2017!

# Upcoming Events

Several upcoming events were announced / highlighted at GCC2016:

## GCC2017: 26-30 June 2017, Montpellier, France

<div class='right'><a href='https://gcc2017.sciencesconf.org/'><img src="/images/logos/GCC2107.png" alt="2017 Galaxy Community Conference (GCC2017)" width="240" /></a></div>

*GCC returns to Europe in 2017!*

The [2017 Galaxy Community Conference (GCC2017)](https://gcc2017.sciencesconf.org/) will be held 26-30 June in [Montpellier, France](http://www.montpellier-france.com/). Montpellier is home to three universities, and is the agronomy hotspot in France.  The city is home to half a million people is the third largest city of France's Mediterranean coast.

GCC2017 will feature hackathons, two days of training and a two day meeting highlighting the best that data intensive biology has to offer.

All conference events will be at the [Le Corum Conference Center](http://www.montpellier-events.com/Le-Corum/Presentation), located in central Montpellier, next to the train station, and a 15 minute ride from the Montpellier International Airport.  GCC2017 is hosted by the [French Institute of Bioinformatics (IFB)](http://www.france-bioinformatique.fr/).

<div class='right'><br /><a href='/events/sg2016/'><img src="/images/logos/SG2016T.V2_logo.png" alt="2016 Swiss German Galaxy Tour" width="230" /></a>
</div>

## Swiss-German Galaxy Days

There will be a Swiss-German Galaxy Tour this fall featuring a range of events over 2 days in October.  Events will held in Freiburg (Germany).

Registration is free, but space is limited on both days. We recommend you register soon to secure your spot for one or both days of the SG2016Tour: **[Register Now](https://docs.google.com/forms/d/1mRO4uNXQd8jL1I9JNiQ4xd0fnrZKlQPQSORgJCpDeqg)**

## Galaxy Admin Training, November 7-11, Salt Lake City, Utah

<div class='left'><a href='/events/admin-training2016/'><img src="/images/logos/AdminTraining2016-500.png" alt="Galaxy Project Admin Training" width="140" /></a></div>

The first ever [Galaxy Project Admin Training workshop](/events/admin-training2016/) will be held November 7-11 in Salt Lake City Utah.  The workshop features a 2 day basics session followed by a 3 day advanced session. There is a separate registration for each session, and participants may sign up for one or both.  We are working with the Utah High Performance Computing Center at the University of Utah for cluster access during the advanced session.

The introductory session will be at the Salt Lake City Library, and the advanced session will be on the University of Utah (the "U") campus.  Workshop housing is on the U as well.

The curriculum will be published and registration will open later this month.  Watch this space.  This workshop is the week before [Supercomputing'16](http://sc16.supercomputing.org/) meets in Salt Lake.

## European Galaxy Developer Knowledge Exchange

<div class='right'>
<a href='https://www.elixir-europe.org/about/groups/galaxy-wg'><img src="/images/logos/ElixirNoTextLogo.png" alt="ELIXIR" width="140" /></a>
</div>

[ELIXIR](https://www.elixir-europe.org/), the European life-science Infrastructure for Biological Information, aims to orchestrate the collection, quality control, share-ability and archiving of large amounts of biological data produced by life science experiments and the resources to compute with this data.

Galaxy is widely used in the ELIXIR community. The [French ELIXIR node (IFB)](http://www.france-bioinformatique.fr/) and the [ELIXIR Galaxy Working Group](https://www.elixir-europe.org/about/groups/galaxy-wg) will *organize a knowledge exchange workshop in January 2017 (date tbc) in Strasbourg (France)*. This workshop will be for developers that want to set up a Galaxy instance, wrap tools and use Galaxy for data analysis and visualisation.


## Galaxy AustralAsia Meeting (GAMe 2017), 3-9 February, Melbourne

<div class='right'><a href='http://www.embl-abr.org.au/GAMe2017'><img src="/images/logos/GAMeLogo200.png" alt="Galaxy Australasia Meeting (GAMe 2017)" width="140" /></a></div>

The **[Galaxy Australasia Meeting (GAMe 2017)](http://www.embl-abr.org.au/GAMe2017)** will be held 3-9 February in Melbourne Australia. The schedule has been set:

  *3 February*
     **[Using Galaxy Training](https://www.embl-abr.org.au/training-day/)**<br />
     Morning and afternoon sessions covering 4 topics in two parallel tracks
  *4-5 February*
     **Meeting**<br />
     A two day meeting of the Galaxy AustralAsian communities featuring accepted and invited talks, posters and panel discussions on best practices, data intensive research, and the computational infrastructure to support it.
  *6-9 February*
     **Admin Training**<br />
     Four days of intense hands-on training on Galaxy Administration, based on the [Galaxy Admin Training](/events/admin-training2016/) offered in the US in November.

Instructors and speakers will include experts from Australia and Asia, as well as members of the Galaxy Team.  Training sessions and instructors will be announced shortly.  Registration and abstract submission will open in August.  Watch this space.

## Other upcoming events

There are [a plentitude of Galaxy related events coming up](/events/). Here's what's coming up over the next fe months:

<div class='center'>
<a href='/events/ismb2016/'><img src="/images/logos/BOSC_logo.png" alt="BOSC 2015" width="120" /></a>
<a href='/events/ismb2016/'><img src="/images/logos/ISMB_2016_Logo.jpg" alt="ISMB 2016" width="140" /></a>
<a href='http://www.genetics-gsa.org/genetics/2016/cgi-bin/tagc16soe.pl#Sat'><img src="/images/logos/TAGC_logo_squarish.png" alt="An Introduction to Using Galaxy for Genetic Data Analysis @ TAGC" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> July 4-8 </th>
    <td> <em><a href='http://bioinformatics.org.au/ws16/program/'>An introduction to Galaxy with the NeCTAR Genomics Virtual Laboratory Workshop</a></em> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> <a href='http://bioinformatics.org.au/ws16/'>2016 Winter School in Mathematical & Computational Biology</a>, University of Queensland, Brisbane, Australia </td>
    <td> <a href='https://www.rcc.uq.edu.au/genomics-virtual-lab'>Igor Makunin, Derek Benson</a> </td>
  </tr>
  <tr>
    <th> July 6-7 </th>
    <td> <em><a href='https://www.open-bio.org/wiki/Codefest_2016'>BOSC Codefest 2016</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Orlando, Florida, United States </td>
    <td> Brad Chapman </td>
  </tr>
  <tr>
    <th> July 8-9 </th>
    <td> <em><a href='/events/ismb2016/'>BOSC 2016</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' />  <a href='http://www.iscb.org/ismb2016'>ISMB 2016</a>, Orlando, Florida, United States </td>
    <td> See <a href='/events/ismb2016/'>presenters</a> </td>
  </tr>
  <tr>
    <th> July 8-12 </th>
    <td> <em><a href='/events/ismb2016/'>ISMB 2016</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' />  Orlando, Florida, United States </td>
    <td> See <a href='/events/ismb2016/'>presenters</a> </td>
  </tr>
  <tr>
    <th> July 13-17 </th>
    <td> <em><a href='http://www.genetics-gsa.org/genetics/2016/cgi-bin/tagc16soe.pl#Sat'>An Introduction to Using Galaxy for Genetic Data Analysis</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://www.genetics2016.org/index.htm'>The Allied Genetics Conference 2016 (TAGC)</a>, Orlando, Florida, United States </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> July 15 </th>
    <td> <em><a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Jul16.html'>Introduction to Galaxy Workshop</a></em> <div class='indent red'>This workshop is full</div> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> University of Glasgow, Glasgow, United Kingdom </td>
    <td> <a href='http://www.polyomics.gla.ac.uk/biog-davidmeltzer.html'>David Meltzer</a> </td>
  </tr>
  <tr>
    <th> July 19-20 </th>
    <td> <em><a href='/events/rochester2016/'>Introduction to Galaxy</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> University of Rochester, Rochester, New York, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> July 21 </th>
    <td> <em><a href='/events/cornell2016/'>Introduction to Galaxy</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Cornell University, Ithaca, New York, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> August 1-4 </th>
    <td> <em><a href='http://icbo.cgrb.oregonstate.edu/node/41'>Integrating Ontological and Genomic Analysis with Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://icbo.cgrb.oregonstate.edu/'>ICBO 2016</a>, Corvallis, Oregon, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 9-10 </th>
    <td> <em>Introduction to Galaxy Workflows</em> </td>
    <td> <img src='/images/icons/GL.png' alt='Around the World' align='right' /> <a href='https://sites.google.com/a/illinois.edu/workflows-workshop/home'>Workflows Workshop</a>, <em><a href='https://sites.google.com/a/illinois.edu/workflows-workshop/home/call-for-host-sites'>Distributed</a></em> </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 12 </th>
    <td> <em><a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Aug16.html'>Introduction to Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> University of Glasgow, Glasgow, United Kingdom </td>
    <td> <a href='http://www.polyomics.gla.ac.uk/biog-davidmeltzer.html'>David Meltzer</a> </td>
  </tr>
  <tr>
    <th> August 18 </th>
    <td> <a href='/community/galaxy-admins/meetups/2016-08-18/'>GalaxyAdmins Web Meetup</a> </td>
    <td> <img src='/images/icons/GL.png' alt='Around the World' align='right' />  <em><a href='/community/galaxy-admins/meetups/2016-08-18/'>Online</a></em> </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a>, <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 29 - September 2 </th>
    <td> <em><a href='https://registration.genomecenter.ucdavis.edu/events/Aug_2016_Bioinformatics_Galaxy_Workshop/'>Using Galaxy for Analysis of RNA-Seq, Exome-Seq, and Variants</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Davis, California, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:training.bioinformatics@ucdavis.edu">UC Davis Bioinformatics Training Program</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> September 3-7 </th>
    <td> <em><a href='http://www.eccb2016.org/programme/tutorials/t2/'>Tutorial: Scientific workflows under Galaxy. Use case: multiple genome comparison</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.eccb2016.org/'>European Conference on Computational Biology (ECCB 2016)</a>, The Hague, Netherlands </td>
    <td> <a href='http://www.ac.uma.es/~ots/'>Oswaldo Trelles Salazar</a> </td>
  </tr>
  <tr>
    <th> September 12-16 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/fr/evenements/summer_school_metagenomics'>Summer School 2016 in Metagenomics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Institut Pasteur, Paris, France </td>
    <td> <a href='http://www.france-bioinformatique.fr/fr/evenements/summer_school_metagenomics'>Contacts</a> </td>
  </tr>
  <tr>
    <th> September 19-23 </th>
    <td> <em><a href='https://drive.google.com/file/d/0Bw3CPLmTKRT8V1p0QlU5QnYyS28/view'>High Throughput Sequencing (HTS) Data Analyses for Scientists</a></em> <div class='indent red'>This workshop is full</div></td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Freiburg, Germany </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:galaxy@informatik.uni-freiburg.de">Contact</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> October 18-20 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/en/evenements/galaxy4bioinformatics'>Galaxy4Bioinformatics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Lyon, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='http://www.france-bioinformatique.fr/en/node/17'>Contacts</a> </td>
  </tr>
  <tr>
    <th> October 20 </th>
    <td> <em><a href='/events/switzerland2016/'>(third Swiss) Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='/events/sg2016/'>2016 Swiss-German Galaxy Tour</a> in Freiburg, Germany </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <th> October 21 </th>
    <td> <em><a href='/events/germany2016/'>(second German) Galaxy Developers Day</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='/events/sg2016/'>2016 Swiss-German Galaxy Tour</a> in Freiburg, Germany </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> November 7-8 </th>
    <td> <strong><a href='/events/admin-training2016/'>Galaxy Admin Training: Introduction</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Salt Lake City, Utah, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/galaxy-team/'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <th> November 9-11 </th>
    <td> <strong><a href='/events/admin-training2016/'>Galaxy Admin Training: Advanced</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Salt Lake City, Utah, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/galaxy-team/'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" text-align: center; background-color: #eef;"> <span style="font-size: larger;"> <strong>2017</strong> </span> </td>
  </tr>
  <tr>
    <th> February 3-9 </th>
    <td> <strong><a href='http://www.embl-abr.org.au/GAMe2017'>Galaxy Australasia Meeting (GAMe 2017)</a></strong> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> University of Melbourne, Victoria, Australia </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://www.embl-abr.org.au/organisers/'>Organisers</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 26-30 </th>
    <td> <strong><a href='https://gcc2017.sciencesconf.org/'>2017 Galaxy Community Conference (GCC2017)</a></strong> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Montpellier, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:gcc2017_organization@groupes.france-bioinformatique.fr">Organisers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='https://training.galaxyproject.org/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

# New Papers

**82 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in June.

Some highlights from last month:

* [Integration and Visualization of Translational Medicine Data for Better Understanding of Human Diseases](https://doi.org/10.1089/big.2015.0057)
    by Venkata Satagopam, Wei Gu, Serge Eifes *et al.* *Big Data*, Vol. 4, No. 2. (1 June 2016), pp. 97-108, doi:10.1089/big.2015.0057

* [biosigner: a new method for the discovery of significant molecular signatures from omics data](https://doi.org/10.3389/fmolb.2016.00026)
    by Philippe Rinaudo, Samia Boudah, Christophe Junot, Etienne A. Thevenot, *Frontiers in Molecular Biosciences,* Vol. 3, No. 26. (2016), doi:10.3389/fmolb.2016.00026

* [CART – a chemical annotation retrieval toolkit](https://doi.org/10.1093/bioinformatics/btw233)
    by Samy Deghou, Georg Zeller, Murat Iskar *et al.,* *Bioinformatics* (02 June 2016), btw233, doi:10.1093/bioinformatics/btw233

* [Experiences in integrated data and research object publishing using GigaDB](https://doi.org/10.1007/s00799-016-0174-6)
    by Scott C Edmunds, Peter Li, Christopher I Hunter *et al.*, In *International Journal on Digital Libraries* (2016), pp. 1-13, doi:10.1007/s00799-016-0174-6

## Tagged Publications

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 15 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 47 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 22 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><br /><img src="/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [PhD student or postdoctoral researcher](http://www.uni-freiburg.de/verwaltung/stellenboerse/?erf=y&stellenid=9837&layout=v4), University of Freiburg, Freiburg, Germany.
* [Postdoctoral Researcher: Forest Genomics Database and Software Developer](http://compgenomics.lab.uconn.edu/join/), University of Connecticut.  "... data sharing among partner databases through Galaxy modules."  <div class='red'>Deadline July 31.</span>
* [Software developer and Post-docs](http://gehlenborg.com/open-positions/), Gehlenborg Lab, Harvard Medical School, Boston, Massachusetts, United States

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

----


# New Public Galaxy Servers

One [new publicly accessible Galaxy servers](/use/) was added to [the list](/use/) in June:

## Chemical Annotation Retrieval Toolkit (CART)

<div class='right solid'><a href='http://cart.embl.de/'><img src="/use/cart/cart.png" alt="Chemical Annotation Retrieval Toolkit (CART) Galaxy" width="300" /></a></div>

* *Link:*
    * [Server](http://cart.embl.de/)
* *Domain/Purpose:*
    * Retrieves annotations on biological effects of chemicals and determines which ones are enriched.
* *Comments:*
    * From "[CART – a chemical annotation retrieval toolkit](http://bioinformatics.oxfordjournals.org/content/early/2016/06/02/bioinformatics.btw233.abstract)" by Samy Deghou, Georg Zeller, Murat Iskar, Marja Driessen, Mercedes Castillo, Vera van Noort, and Peer Bork. *Bioinformatics* (2016), doi: 10.1093/bioinformatics/btw233
      "it matches an input list of chemical names into a comprehensive reference space to assign unambiguous chemical identifiers. In this unified space, bioactivity annotations can be easily retrieved from databases covering a wide variety of chemical effects on biological systems. Subsequently, CART can determine annotations enriched in the input set of chemicals and display these in tabular format and interactive network visualizations, thereby facilitating integrative analysis of chemical bioactivity data."
    * Source code and an easy-to-install command line tool are also available.
* *User Support:*
    * See the "Tutorial" and "Contact" tabs on the [website](http://cart.embl.de/).
* *Quotas:*
    * Supports anonymous access and creation of user logins.
* *Sponsor(s):*
    * [Structural and Computational Biology Unit](http://www.embl.de/research/units/scb/), [European Molecular Biology Laboratory](http://www.embl.de/index.php), Heidelberg, Germany

# New Semi-Public Galaxy Servers

And one [new set of Semi-Public Galaxy server](/use/) was added in June.  Semi-public servers aren't [fully publicly accessible](/use/), but that are accessible to a larger group than just the host institution's members.  

## Norway: NeLS

<div class='right solid'><a href='https://nels.bioinfo.no'><img src="https://galaxy-ntnu.bioinfo.no/www/NeLS_logo_300x100.png" alt="NeLS Portal"  /></a></div>

* *Links:*
    * **[NeLS Galaxy server at UiO](https://galaxy-uio.bioinfo.no)**
    * **[NeLS Galaxy server at UiB](https://galaxy-uib.bioinfo.no)**
    * **[NeLS Galaxy server at UiT](https://galaxy-uit.bioinfo.no)**
    * **[NeLS Galaxy server at NTNU](https://galaxy-ntnu.bioinfo.no)**
    * **[NeLS Galaxy server at NMBU](https://galaxy-nmbu.bioinfo.no)**
    * **[Lifeportal server at UiO](https://lifeportal.uio.no)**
    * [Apply for a NeLS account](https://nels.bioinfo.no/pages/registration.xhtml)
* *Eligibility:*
    * Life Science researchers from Norwegian academic institutions can log in with their FEIDE accounts (federation of home institution credentials). Other research collaborators can apply for a NeLS account.
* *Comments:*
    * The Norwegian e-Infrastructure for Life Sciences (NeLS) consists of a network of Galaxy servers at five Norwegian universities accessible via single sign-on.
    * All the NeLS Galaxy servers are connected to the [NeLS Portal](https://nels.bioinfo.no/) which provides centralized storage and sharing capabilities.
    * NeLS develops and maintains several [analysis pipelines](https://galaxy-ntnu.bioinfo.no/www/nels_pipelines.html) offered as Galaxy workflows.
    * The NeLS infrastructure is operated by the Norwegian node of [Elixir](https://www.elixir-europe.org).

* *User Support:*
    * The [Norwegian Bioinformatics Platform](http://www.bioinfo.no) has a help desk that can be reached by emailing contact@bioinfo.no
* *Quotas:*
    * NeLS Galaxy instances have some temporary storage available, while the bulk volumes of data are managed through the NeLS centralized storage and the Norwegian national storage services.
* *Sponsor(s):*
    * [University of Oslo](http://www.uio.no/)
    * [University of Bergen](http://www.uib.no/)
    * [Norwegian University of Science and Technology (NTNU)](http://www.ntnu.no/)
    * [Norwegian University of Life Sciences (NMBU)](http://www.nmbu.no/)
    * [UiT The Arctic University of Norway](http://www.uit.no/)
    * [Elixir Norway](http://www.elixir-norway.org/)
    * [The Research Council of Norway](http://www.forskningsradet.no/en/Home_page/1177315753906)

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

<br />

One new [community log board entry](/news/) was added:

* [Running jobs outside the Galaxy Docker container](https://github.com/bgruening/docker-galaxy-stable/blob/dev/docs/Running_jobs_outside_of_the_container.md)
    by [Marius van den Beek](https://github.com/mvdbeek).

----

# Releases

## blend4php 0.1 alpha

<div class='right'><a href='https://github.com/galaxyproject/blend4php'><img src="/images/galaxy-logos/blend4phpLogo.png" alt="" width="220" /></a></div>

**We are pleased to announce the alpha release of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html).** It follows the lead of [BioBlend](https://bioblend.readthedocs.io/en/latest/) which provides a Python package for interacting with Galaxy and CloudMan--hence the use of 'blend' in the name of this package. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases. Integration of Tripal and Galaxy will allow the community research databases to provide next-generation analytical tools to their users using Galaxy. However, this library was created independently of Tripal to support integration of any PHP application with Galaxy.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/docs-v0.1a/html/index.html) page for full information.


<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/images/galaxy-logos/planemo-logo.png" alt="" width="200" /></a></div>

## Planemo 0.27.0

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  Planemo 0.27.0 was released in June. Some highlights:

* Use ephemeris to handle syncing shed tools for workflow actions. [1c6cfbb](https://github.com/galaxyproject/planemo/commit/1c6cfbb)
* More planemo testing enhancements for testing artifacts that aren’t Galaxy tools. [Pull Request 491](https://github.com/galaxyproject/planemo/pull/491)
* Implement `docker_galaxy` engine type. [eb039c0](https://github.com/galaxyproject/planemo/commit/eb039c0), [Issue 15](https://github.com/galaxyproject/planemo/issues/15)
* Enhance profiles to be Dockerized Galaxy-aware. [Pull Request 488](https://github.com/galaxyproject/planemo/pull/488)
* Add linter for DOI type citation - thanks to [@mvdbeek](https://github.com/mvdbeek). [Pull Request 484](https://github.com/galaxyproject/planemo/pull/484)

See the [release history](http://planemo.readthedocs.org/en/latest/history.html#history).


## galaxy-lib 16.7.8 - 16.7.9

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

Highlights of June releases include:

* Updates for recent changes to Galaxy and cwltool.
* Updates to include Galaxy library for verifying test outputs and the latest dev changes to Galaxy.

## Earlier Releases

### Galaxy v16.04

<div class='right'><a href='http://getgalaxy.org'><img src="/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The [Galaxy Committers team](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) is pleased to announce the **[April 2016 (v16.04) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.04_announce.html)**.

*Please note that Python 2.6 is no longer supported as of this release. See the [16.04 release announcement](https://wiki.galaxyproject.org/News/2016-04_GalaxyRelease) for details.*

<span style="font-size: larger;"> **Highlights** </span>

**Tool Profile Versions**

Tools may now [declare which version](http://planemo.readthedocs.io/en/latest/galaxy_changelog.html#tool-profile-version-pr-1688) of Galaxy they require. Tools requiring 16.04 or newer will have new default behaviors (such as using exit code for error detection) that should simplify tool development. See [PR #1688](https://github.com/galaxyproject/galaxy/pull/1688).

**Embedded Pulsar Job Runner**

Galaxy can now start a Pulsar application embedded within the Galaxy process itself. This allows using Pulsar’s job staging and isolation without requiring a RESTful web service or a message queue. This is enabling usegalaxy.org to run jobs to on the new [JetStream cloud](http://jetstream-cloud.org/). See [PR #2057](https://github.com/galaxyproject/galaxy/pull/2057).

**New Chemical Datatypes**

Galaxy now detects and supports many molecular datatypes. See [PR 1941](https://github.com/galaxyproject/galaxy/pull/1941). Thanks to [Björn Grüning (@bgruening)](https://github.com/bgruening).


<table>
  <tr>
    <td style=" border: none;"> <span style="font-size: larger;"> Github </span> <br /><br /><strong>New</strong><br /><code>% git clone -b master https://github.com/galaxyproject/galaxy.git</code><br /><br />U<strong>pdate to latest stable release</strong><br /><code>% git checkout master && git pull --ff-only origin master</code><br /><br /><strong>Update to exact version</strong><br /><code>% git checkout v16.04</code> </td>
    <td style=" vertical-align: top; border: none; width: 10%;"> </td>
    <td style=" vertical-align: top; border: none;"> <span style="font-size: larger;"> BitBucket </span> <br /><br /><strong>Upgrade</strong><br /><code>% hg pull</code><br /><code>% hg update latest_16.04</code> </td>
  </tr>
  <tr>
    <td colspan=3 style=" border: none;"> See the <a href='/admin/get-galaxy/'>Get Galaxy</a> page for additional details regarding the source code locations. </td>
  </tr>
</table>



#### Galaxy Docker Image 16.04

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 16.04](https://github.com/bgruening/docker-galaxy-stable/releases/tag/16.04) as well.  

### CloudMan 16.04

<div class='right'><a href='/cloudman/'><img src="/images/galaxy-logos/cloudman-logo.jpg" alt="CloudMan" width="300" /></a></div>

**We just released an update to Galaxy CloudMan on AWS.** CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

This is a minor update release with the following changes:

* Galaxy 16.04 update
* Availability on Amazon's Ireland region
* A couple of bug fixes

See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a more complete set of changes.

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="150" /></a></div>

### Pulsar 0.7.0

[Pulsar 0.7](https://pypi.python.org/pypi/pulsar-app/) was released in April.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.


### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

Did not get there this month.  Look for a double edition next month.

# Other News

* [biojs2galaxy](http://biojs.io/d/biojs2galaxy): Continuous integration of BioJS components into Galaxy
    A fast tool to convert BioJS components into Galaxy visualization plugins. *Warning: This tool is in a ALPHA stage. Use at your own risk.*
* [v1.0.0 release of ASaiM framework, an open-source Galaxy-based opinionated framework dedicated to microbiota sequence analyses](https://zenodo.org/record/56282#.V3wBgI4hrsE)
