---
autotoc: true
title: May 2016 Galaxy News
---


<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/src/GalaxyUpdates/index.md'><img src="/src/Images/GalaxyLogos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the May 2016 *[Galactic News](/src/GalaxyUpdates/index.md)*, a summary of what is going on in the Galaxy community. 

If you have anything to include in the next News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org). 


# GCC2016

<div class='left'>
<a href='https://gcc2016.iu.edu/'><img src="https://gcc2016.iu.edu/img/gcc2016-logo.png" alt="2016 Galaxy Community Conference" width="250" /></a>
</div>

[GCC2016](http://gcc2016.iu.edu/) will be held June 25-29 at [Indiana University](http://indiana.edu/) in Bloomington, Indiana, United States. This will be the 7th annual gathering of the Galaxy community, and we are expecting over 200 participants again this year.  The 2016 Galaxy Community Conference includes 2 days of hackathons, [2 days of training](https://gcc16.sched.org/grid-full#2016-06-26), and a two day meeting featuring accepted presentations, keynotes, poster sessions, the new Visualization Showcase and Software Demo sessions, lightning talks, birds-of-a-feather meetups, and plenty of networking. 

## Deadlines!

### Poster & Demo Abstracts: May 20

<div class='left'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://bit.ly/gcc2016abssub'>Submit an abstract</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

**The deadline for [poster and computer demo abstracts](http://bit.ly/gcc2016abssub) is May 20**, or when we run out of space, *whichever comes first.*  Abstracts are reviewed on a rolling basis and submitters are notified of acceptance status no later than two weeks after submission. You may submit similar content for oral presentations, posters, and demos. 

Topics should be of interest to those working in high-throughput data analysis and research. Presentations that are Galaxy-centric are encouraged, but not required.  Please see the [abstracts page](https://gcc2016.iu.edu/abstracts/index.php) for full details.

(And we are also still accepting late oral abstracts, that will be considered if we have cancellations.)

### Early Registration: May 20

<div class='right'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='https://gcc2016.iu.edu/registration/'>Register Now</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

**[Early registration for GCC2016](https://gcc2016.iu.edu/registration/) ends May 20**. 
Registration costs depend on which events you register for, your career stage & affiliation, and when you register.  Early bird registration ends May 20 and is up to 40% less than regular registration rates.  Early bird registration starts at less than $45 / day for students and postdocs, and at $65 / day for other attendees from non-profits.

You can also sign up for [conference housing](https://gcc2016.iu.edu/location) during registration.  

You are strongly encouraged to review the [training](https://gcc16.sched.org/grid-full#2016-06-26) and [housing](https://gcc2016.iu.edu/location) options before beginning the registration process.

### Scholarships: May 1

<div class='right'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='https://docs.google.com/forms/d/1IbMoNcA6z31a8hvqjSdfQRIQ92wo-HxXvA5uk0UilnQ/viewform'>Submit an application</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

<div class='left'><a href='https://docs.google.com/forms/d/1IbMoNcA6z31a8hvqjSdfQRIQ92wo-HxXvA5uk0UilnQ/viewform'><img src="/src/Images/GalaxyLogos/GalaxyScholarshipsBig.png" alt="Scholarships: Apply now" width="220" /></a></div>

We are pleased to offer [scholarships](https://docs.google.com/forms/d/1IbMoNcA6z31a8hvqjSdfQRIQ92wo-HxXvA5uk0UilnQ/viewform) for the [2016 Galaxy Community Conference](http://gcc2016.iu.edu/), being held in Bloomington, Indiana, United States, June 25-29.  Scholarships are available to students and post-docs in historically under-represented groups, and to those from or based in Low and Lower-Middle Income Economies, [as defined by the World Bank](http://bit.ly/1nO82wD).  If this describes you or one of your students then we hope to receive an application.

Scholarships cover [registration](http://gcc2016.iu.edu/registration) and [lodging](http://gcc2016.iu.edu/location) during the GCC Meeting, and for any Training or Hackathon events the applicant chooses to attend.  Scholarships do not cover travel or other expenses. **The application deadline is May 1 for members of historically underrepresented groups.**

See the [full announcement](/src/News/GCC2016Scholarships/index.md) for details.

## Sponsors

We continue to seek other sponsors as well and [offer a wide range of sponsorship plans](https://gcc2016.iu.edu/sponsors/index.php). If your organization is interested in having a presence at GCC2016, please contact the [GCC2016 Exec](mailto:gcc2016-exec@lists.galaxyproject.org) for more information.

### GigaScience

<div class='right'><a href='http://gigascience.biomedcentral.com/'><img src="/src/Images/Logos/GigaScienceLogo.gif" alt="GigaScience" /></a>
</div>

Please welcome the journal ***[GigaScience](http://gigascience.biomedcentral.com/)*** as a [GCC Silver Sponsor](https://gcc2016.iu.edu/sponsors/index.php) for the 4th year in a row.  *GigaScience* aims to revolutionize reproducibility of analyses, data dissemination, organization, understanding, and use. 

All accepted oral presentations are eligible for consideration for publication in the journal *[GigaScience](http://gigascience.biomedcentral.com/)'s* [Galaxy series](http://www.gigasciencejournal.com/series/Galaxy). **Published papers will receive a 15% discount in the [article-processing charge](http://gigascience.biomedcentral.com/frequently-asked-questions)** if you flag GCC2016 on submission. As an open access and open-data journal focussing on reproducibility, GigaScience publishes all research objects (including data, software tools, workflows, VMs and containers) from 'big data' studies across the entire spectrum of life and biomedical sciences. GigaScience submissions utilize a novel format, where all of the supporting research objects are hosted and integrated into accepted papers using independently citable [digital object identifiers](http://en.wikipedia.org/wiki/Digital_object_identifier) from the journal's [GigaGalaxy server](http://gigagalaxy.net/) and [GigaDB database](http://gigadb.org/). See the [Galaxy series page](http://www.gigasciencejournal.com/series/Galaxy) for examples of work coming from previous GCC meetings. 
 
### GenomeWeb

<div class='left'><a href='https://www.genomeweb.com/'><img src="/src/Images/Logos/GenomeWebLogo300.jpg" alt="GenomeWeb" width="250" /></a>
</div>

Please welcome ***[GenomeWeb](https://www.genomeweb.com/)*** as a [GCC Silver Sponsor](https://gcc2016.iu.edu/sponsors/index.php) for the third year in a row.  *[GenomeWeb](https://www.genomeweb.com/)* is an independent online news organization that provides in-depth coverage of the scientific and economic ecosystem spurred by high-throughput genome sequencing. We are the leading information source for scientists, executives, and clinicians who use and develop advanced life science tools.

### EMC

<div class='right'><a href='http://www.emergingtechsolutions.com/life-science'><img src="/src/Images/Logos/EMCLogo.png" alt="EMC" width="220" /></a>
</div>

Finally, we are delighted to have **[EMC](http://www.emergingtechsolutions.com/life-science)** again as a [GCC sponsor](https://gcc2016.iu.edu/sponsors/index.php) for the 4th time.  EMC is a Peta Sponsor for [both GCC2016 Hacakthons](https://gcc2016.iu.edu/hacks/index.php).

[EMC Emerging Technologies Division (ETD)](http://www.emergingtechsolutions.com/life-science) is a global leader and trusted partner in Life Science storage solutions. We deliver powerful yet versatile solutions for healthcare and life science organizations that want to manage clinical and genomics data. ETD storage solutions are simple to install, manage and scale, at any size, across the R&D data lifecycle.  As a leader and trusted partner at hundreds of Life Science organizations worldwide, ETD storage solutions provide the security, ease of management,high availability, and scalability needed to manage Life Science workflows today and in the future.

----

# Upcoming Events

<div class='right'><br /><a href='/src/Events/ASMS2016/index.md'><img src="/src/Images/Logos/ASMSLogo.png" alt="64th ASMS Conference on Mass Spectrometry and Allied Topics" width="120" /></a></div>

## Galaxy at ASMS 2016

Galaxy will have a [strong presence](/src/Events/ASMS2016/index.md) at the [64th ASMS Conference on Mass Spectrometry and Allied Topics](http://www.asms.org/conferences/annual-conference/annual-conference-homepage), being held June 5-9 in San Antonio, Texas, United States. There will be [one workshop and one talk](/src/Events/ASMS2016/index.md) (both from the [GalaxyP Project](https://usegalaxyp.org/)), and [at least 7 posters](/src/Events/ASMS2016/index.md) on using Galaxy for proteomics.

*If you are interested, **[register now](http://www.asms.org/conferences/annual-conference/registration) as early registration closes April 30.***

## Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data

<div class='left'><a href='https://registration.genomecenter.ucdavis.edu/events/June_2016_Bioinformatics_Galaxy_Workshop/'><img src="/src/Images/Logos/UCDavisGenomeCenter_Bioinf.png" alt="Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data" width="200" /></a>
</div>

The [UC Davis Bioinformatics Training Program](http://training.bioinformatics.ucdavis.edu/), a [GTN member](/src/Teach/Trainers/index.md#uc-davis-genome-centers-bioinformatics-core-facility), will be presenting the workshop **[Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data](https://registration.genomecenter.ucdavis.edu/events/June_2016_Bioinformatics_Galaxy_Workshop/)** on June 13-17, at UC Davis in Davis, California, United States.  

This workshop will include a rich collection of lectures and hands-on sessions, covering both theory and tools. We will explore the basics of high throughput sequencing technologies, focusing on Illumina data for hands-on exercises. Participants will explore software and protocols, create and modify workflows, and diagnose/treat problematic data, utilizing computing power of the Amazon Cloud.

*Space is limited and this workshop is already more than 50% full.*

## May, June and July Events

There are [a staggering 14 known Galaxy related events and presentations in May](/src/Events/index.md).  These are spread over 4 countries on 3 continents.  June and July are filling up too.

See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='center'> 
<a href='http://www.qfab.org/training/'><img src="/src/Images/Logos/QFABLogo.png" alt="QFAB Workshops" height="80" /></a>
<a href='http://www.canceropole-idf.fr/formation-ngs-rnaseq'><img src="/src/Images/Logos/Cancerpole_logo.jpg" alt="NGS & Cancer : Analyses RNA-Seq" height="70" /></a>
<a href='https://www.molmed.nl/courses/CourseDetail.asp?backpage=courses.asp&courseID=1448'><img src="/src/Images/Logos/MolecularMedicinePostGraduateSchoolLogo.jpg" alt="RNA-Seq and DNA-Seq Cancer Analyses" height="70" /></a>
<a href='http://bioinfo.genotoul.fr/index.php?id=10'><img src="/src/Images/Logos/GenoToulLogo.png" alt="Cycle d'aprentissage sous Galaxy" height="60" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> May 5 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-genome-assembly-using-gvl/'>De novo genome assembly using Genomics Virtual Lab Workshop</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> University of Queensland, St Lucia, Australia </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:training@qfab.org">QFAB Training</a>, <a href='http://www.qfab.org/xin-yi-chua/'>Xin-Yi Chua</a>, <a href='http://www.qfab.org/michael-thang/'>Mike Thang</a> </td>
  </tr>
  <tr>
    <th> May 6 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-variant-detection-using-galaxy-2-2/'>Variant detection using Galaxy Workshop</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> University of Queensland, St Lucia, Australia </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:training@qfab.org">QFAB Training</a> </td>
  </tr>
  <tr>
    <th> May 10-14 </th>
    <td> <em>EDGY—Export of data from Galaxy to Yabi, automated workflow transfer to command line tools</em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> <a href='https://meetings.cshl.edu/meetings.aspx?meet=genome&year=16'>The Biology of Genomes</a>, Cold Spring Harbor Laboratory, New York, United States </td>
    <td> <a href='http://david.molik.co/'>David Molik</a> </td>
  </tr>
  <tr>
    <th> May 11-13 </th>
    <td> <em><a href='http://www.canceropole-idf.fr/formation-ngs-rnaseq'>NGS & Cancer : Analyses RNA-Seq</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Paris, France </td>
    <td> <a href="mailto:communication@canceropole-idf.fr">Cancéropôle Île-de-France</a> </td>
  </tr>
  <tr>
    <th> May 12-13 </th>
    <td> <em><a href='https://www.molmed.nl/courses/CourseDetail.asp?backpage=courses.asp&courseID=1448'>RNA-Seq and DNA-Seq Cancer Analyses</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Rotterdam, The Netherlands </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:y DOT hoogstrate AT erasmusmc.nl">Youri Hoogstrate</a> </td>
  </tr>
  <tr>
    <th> May 17 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=80'>Galaxy First Step</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Part of <a href='http://bioinfo.genotoul.fr/index.php?id=10'>Cycle d'aprentissage sous Galaxy</a>, INRA, Auzeville, France </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> Sarah Maman </td>
  </tr>
  <tr>
    <th> May 18-19 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=81'>Galaxy: Reads alignment and SNP calling</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Part of <a href='http://bioinfo.genotoul.fr/index.php?id=10'>Cycle d'aprentissage sous Galaxy</a>, INRA, Auzeville, France </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> Philippe Bardou </td>
  </tr>
  <tr>
    <th> May 19-20 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=82'>Galaxy : RNAseq alignment and transcripts assemblies</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Part of <a href='http://bioinfo.genotoul.fr/index.php?id=10'>Cycle d'aprentissage sous Galaxy</a>, INRA, Auzeville, France </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> Celine Noirot, Cédric Cabau </td>
  </tr>
  <tr>
    <th> May 18-20 </th>
    <td> <em><a href='https://www.elixir-europe.org/events/technical-hackathon-tools-workflows-and-workbenches'>ELIXIR Technical Hackathon: Tools, Workflows and Workbenches</a></em><div class='indent red'>Full</div> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> <a href='https://www.eventbrite.fr/e/technical-hackathon-tools-workflows-and-workbenches-tickets-22797525011#listing-organizer'>Institut Pasteur, Paris, France</a> </td>
    <td> <a href='https://www.eventbrite.fr/e/technical-hackathon-tools-workflows-and-workbenches-tickets-22797525011#listing-organizer'>Organisers</a> </td>
  </tr>
  <tr>
    <th> May 23-24 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module16.pdf'>Analyse statistique de données RNA-Seq sous Galaxy -Recherche des régions d'intérêt différentiellement exprimées</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> May 26 </th>
    <td> <em><a href='http://www.bioplatforms.com/rna-seq-using-galaxy-may-2016/'>RNA-Seq Analysis Using Galaxy</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> <a href='http://www.cmri.org.au/'>Children’s Medical Research Institute</a>, Westmead, NSW, Australia </td>
    <td> <a href="mailto:kchamp@bioplatforms.com">Katherine Champ</a> </td>
  </tr>
  <tr>
    <th> May 27 </th>
    <td> <em><a href='http://www.bioplatforms.com/variant-detection-using-galaxy-may-2016/'>Variant Detection using Galaxy</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> <a href='http://www.cmri.org.au/'>Children’s Medical Research Institute</a>, Westmead, NSW, Australia </td>
    <td> <a href="mailto:kchamp@bioplatforms.com">Katherine Champ</a> </td>
  </tr>
  <tr>
    <th> May 30 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module17-15mars.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> May 31 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module8bis-31mai.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 5-9 </th>
    <td> <em><a href='/src/Events/ASMS2016/index.md'>64th ASMS Conference on Mass Spectrometry and Allied Topics</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> San Antonio, Texas, United States </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href='/src/Events/ASMS2016/index.md'>Presenters</a> </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> <em><a href='http://bioinformatics.ca/workshops/2016/informatics-high-throughput-sequencing-data-2016'>Informatics on High-throughput Sequencing Data</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> Montreal, Quebec, Canada </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div>  <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 13-17 </th>
    <td> <em><a href='https://registration.genomecenter.ucdavis.edu/events/June_2016_Bioinformatics_Galaxy_Workshop/'>Using Galaxy for Analysis of RNA-Seq and ChIP-Seq Data</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> Davis, California, United States </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:training.bioinformatics@ucdavis.edu">UC Davis Bioinformatics Training Program</a> </td>
  </tr>
  <tr>
    <th> June 16 </th>
    <td> <em><a href='/src/Community/GalaxyAdmins/Meetups/2016_06_16/index.md'>GalaxyAdmins Web Meetup</a></em> </td>
    <td> <img src='/src/images/Icons/World40.png' alt='Around the World' align='right' /> <em><a href='/src/Community/GalaxyAdmins/Meetups/2016_06_16/index.md'>Online</a></em> </td>
    <td> <a href='https://www.msi.umn.edu/staff/evan'>Evan Bollig</a>, <a href='https://www.msi.umn.edu/staff/jj'>JJ Johnson</a>, <a href='/src/HansrudolfHotz/index.md'>Hans-Rudolf Hotz</a>, <a href='/src/DaveClements/index.md'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> June 25-29 </th>
    <td> <strong><em><a href='https://gcc2016.iu.edu/'>2016 Galaxy Community Conference (GCC2016)</a></strong></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src='/src/images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div>  <a href='https://gcc2016.iu.edu/organizers'>Organizers</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 1 </th>
    <td> <em><a href='http://gmod.org/wiki/Jun_2016_GMOD_Meeting'>GMOD Meeting</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> <a href='http://gmod.org/wiki/Scott'>Scott Cain</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> July 4-8 </th>
    <td> <em><a href='http://bioinformatics.org.au/ws16/program/'>An introduction to Galaxy with the NeCTAR Genomics Virtual Laboratory Workshop</a></em> </td>
    <td> <img src='/src/images/Icons/Australia40.png' alt='Austalia' align='right' /> <a href='http://bioinformatics.org.au/ws16/'>2016 Winter School in Mathematical & Computational Biology</a>, University of Queensland, Brisbane, Australia </td>
    <td> <a href='https://www.rcc.uq.edu.au/genomics-virtual-lab'>Igor Makunin, Derek Benson</a> </td>
  </tr>
  <tr>
    <th> July 6-7 </th>
    <td> <em><a href='https://www.open-bio.org/wiki/Codefest_2016'>BOSC Codefest 2016</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> Orlando, Florida, United States </td>
    <td> Brad Chapman </td>
  </tr>
  <tr>
    <th> July 8-9 </th>
    <td> <em><a href='https://www.open-bio.org/wiki/BOSC_2016'>BOSC 2016</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' />  <a href='http://www.iscb.org/ismb2016'>ISMB 2016</a>, Orlando, Florida, United States </td>
    <td> </td>
  </tr>
  <tr>
    <th> July 8-12 </th>
    <td> <em><a href='http://www.iscb.org/ismb2016'>ISMB 2016</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' />  Orlando, Florida, United States </td>
    <td> </td>
  </tr>
  <tr>
    <th> July 13-17 </th>
    <td> <em><a href='http://www.genetics2016.org/index.htm'>The Allied Genetics Conference 2016 (TAGC)</a></em> </td>
    <td> <img src='/src/images/Icons/NorthAmerica40.png' alt='North America' align='right' /> Orlando, Florida, United States </td>
    <td> <a href='/src/DaveClements/index.md'>Dave Clements</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/src/Teach/Trainers/index.md'><img src="/src/Images/Icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/src/Teach/Trainers/index.md'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


# Past Events

## April GalaxyAdmins Slides & Video

<div class='right'>
<a href='/src/Community/GalaxyAdmins/index.md'><img src="/src/Images/GalaxyLogos/GalaxyAdmins.png" alt="GalaxyAdmins" width="160" /></a> 
</div>

[Slides and video](/src/Community/GalaxyAdmins/Meetups/2016_04_21/index.md) from the [April 2016 GalaxyAdmins meetup are now available](/src/Community/GalaxyAdmins/Meetups/2016_04_21/index.md). [Ivar Grytten](http://www.mn.uio.no/ifi/english/people/aca/ivargry/) and  [Geir Kjetil Sandve](http://www.mn.uio.no/ifi/english/people/aca/geirksa/) from the [University of Oslo](http://www.uio.no/) discussed [The Galaxy Portal: Accessing Galaxy from Mobile Devices](http://bioinformatics.oxfordjournals.org/content/early/2016/02/17/bioinformatics.btw042.full) ([Slides](PLACEHOLDER_ATTACHMENT_URL/src/Documents/Presentations/201604_Admins_Portal.pdf)) and [John Chilton](/src/JohnChilton/index.md) covered [Tool Development Developments](http://planemo.readthedocs.org/en/latest/galaxy_changelog.html).

## Conda Dependency Codefest Report

<div class='left'><a href='https://github.com/galaxyproject/tools-iuc/issues/700'><img src="/src/Images/Logos/Conda_480.png" alt="Conda Dependencies Codefest" width="160" /></a>
</div>

A [Conda Dependencies Dodefest](https://github.com/galaxyproject/tools-iuc/issues/700) was held on Monday April 4, and involved 8 participants.  It was designed to be beginner friendly, which increased contribution from the community. 4 members of the galaxy community were [added as contributors](https://github.com/bioconda/bioconda-recipes/issues/1#issuecomment-205241333) to the [bioconda-recipe repository](https://github.com/bioconda/bioconda-recipes) as a result of this hackathon.  The main aim of the codefest was to get community members familiar with the Conda-Galaxy integration, and to remove tools from testing blacklist. See the [full codefest report](/src/News/CondaDependencyCodefestReport/index.md) for details.



----

# Galaxy on Jetstream Cloud

<div class='right'><a href='http://jetstream-cloud.org/'><img src="https://www.nsf.gov/news/mmg/media/images/jetstream_logo_f_fe60185c-962f-4c1f-99bf-e6ec82d54c21.jpg" alt="" width="300" /></a></div>

**Want your own Galaxy server, for free?** You can now easily create Galaxy servers on the new [NSF Jetstream cloud](http://jetstream-cloud.org/). Each server comes preconfigured with hundreds of tools and commonly used reference datasets. It only takes a couple of minutes to start one. Once running, you can use it or change it up any way you like.

**How do I get access?**

You must be a US-based academic to access Jetstream cloud. Access is free but it is necessary to have an XSEDE account (go to https://www.xsede.org/ to sign up) and have an active resource allocation. Getting the resource allocation is matter of writing a summary of your research in less than 100 words and waiting ~24 hrs for the application to get approved. Go to http://jetstream-cloud.org/allocations.php → "Submit and manage allocation requests" to get started; choose *Startup* type of allocation.

** How do I launch my own Galaxy server? **

After you have your XSEDE account and an active allocation:
1. Visit https://use.jetstream-cloud.org/
2. Browse the available images and choose "Galaxy 16.01 Standalone"
3. Follow the prompts on the screen to launch an instance
4. In less than 5 minutes, you should have your own, fully configured Galaxy server

More documentation about the process can be found [here](/src/Cloud/Jetstream/index.md).

# New Papers

**72 new papers** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in April. 

Some April highlights:

* [Unlocking Large-Scale Genomics](http://bit.ly/PiredduThesis)
    by Luca Pireddu

* [qpMerge: Merging different peptide isoforms using a motif centric strategy](http://biorxiv.org/content/early/2016/04/05/047100)
    by Matthew M Hindle, Thierry Le Bihan, Johanna Krahmer, Sarah F Martin, Zeenat B Noordally, T. Ian Simpson, Andrew J. Millar, doi: http://dx.doi.org/10.1101/047100 

* [Metavisitor, a suite of Galaxy tools for simple and rapid detection and discovery of viruses in deep sequence data](http://biorxiv.org/content/early/2016/04/20/048983)
    by Guillaume Carissimo, Marius van den Beek, Juliana Pegoraro, Kenneth D Vernick, Christophe Antoniewski, doi: http://dx.doi.org/10.1101/048983 

* [MutSpec: a Galaxy toolbox for streamlined analyses of somatic mutation spectra in human and mouse cancer genomes](http://dx.doi.org/10.1186/s12859-016-1011-z)
    by Maude Ardin, Vincent Cahais, Xavier Castells, *et al.* *BMC Bioinformatics*, Vol. 17, No. 1. (18 April 2016), doi:10.1186/s12859-016-1011-z

* [deepTools2: a next generation web server for deep-sequencing data analysis](http://dx.doi.org/10.1093/nar/gkw257)
    by Fidel Ramírez, Devon P. Ryan, Björn Grüning, *et al.* *Nucleic Acids Research* (13 April 2016), gkw257, doi:10.1093/nar/gkw257

* [META-pipe - Pipeline Annotation, Analysis and Visualization of Marine Metagenomic Sequence Data](http://arxiv.org/abs/1604.04103)
    by Espen M. Robertsen, Tim Kahlke, Inge A. Raknes, et al.

* [MGEScan: a Galaxy based system for identifying retrotransposons in genomes](http://dx.doi.org/10.1093/bioinformatics/btw157)
    by Hyungro Lee, Minsu Lee, Wazim M. Ismail, *et al.* *Bioinformatics* (07 April 2016), btw157, doi:10.1093/bioinformatics/btw157

* [R-Syst::diatom: an open-access and curated barcode database for diatoms and freshwater monitoring](http://dx.doi.org/10.1093/database/baw016)
    by Frédéric Rimet, Philippe Chaumeil, François Keck, *et al.* *Database*, Vol. 2016 (1 January 2016), doi:10.1093/database/baw016

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 17 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 42 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


----

# New Tutorials and Video

<div class='left'><a href='https://github.com/nekrut/galaxy/wiki/Diploid-variant-calling'><img src="/src/Images/Screenshots/AntonsDiploidVariantTutorial1.png" alt="Diploid variant calling" width="190" /></a></div>

There are two new comprehensive online tutorials from [Anton Nekrutenko](/src/anton/index.md):

## Diploid variant calling

Variant calling is a complex field that was significantly propelled by advances in DNA sequencing and efforts of large scientific consortia such as the [1000 Genomes](http://www.1000genomes.org/). **[This tutorial](https://github.com/nekrut/galaxy/wiki/Diploid-variant-calling)** summarizes basic ideas central to Genotype and Variant calling. 

<div class='right'><a href='https://github.com/nekrut/galaxy/wiki/Reference-based-RNA-seq'><img src="/src/Images/Screenshots/AntonsRnaSeqTutorial1.png" alt="Reference based RNA seq" width="190" /></a></div>

Much of Galaxy-related features described in this tutorial have been developed by [Björn Grüning](/src/BjoernGruening/index.md) ([@bgruening](https://github.com/bgruening)) and configured by [Dave Bouvier](/src/DaveBouvier/index.md) ([@davebx](https://github.com/davebx)).

## Reference based RNA seq

**[This tutorial](https://github.com/nekrut/galaxy/wiki/Reference-based-RNA-seq)** is inspired by an [exceptional RNAseq course](http://chagall.med.cornell.edu/RNASEQcourse/) at the Weill Cornell Medical College compiled by Friederike Dündar, Luce Skrabanek, and Paul Zumbo and by tutorials produced by Björn Grüning (@bgruening) for Freiburg Galaxy instance. Much of Galaxy-related features described in this section have been developed by [Björn Grüning](/src/BjoernGruening/index.md) ([@bgruening](https://github.com/bgruening)) and configured by [Dave Bouvier](/src/DaveBouvier/index.md) ([@davebx](https://github.com/davebx)).

## Dataset Collections

Dataset collections help analyzing multiple datasets in just a few clicks. This [short video](https://t.co/FslNV1GoLQ) shows how. If you prefer reading to watching, then check out [this tutorial](http://github.com/nekrut/galaxy/wiki/Processing-many-samples-at-once).

<br /> 

----

# Who's Hiring

<div class='right'><br /><img src="/src/Images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.
* [Galaxy Administrator & Developer](http://www.uni-freiburg.de/verwaltung/stellenboerse/?erf=y&stellenid=9693&layout=v4), University of Freiburg, Freiburg, Germany.
* [Postdoctoral researcher](https://www.uni-freiburg.de/administration/stellenboerse/?erf=y&stellenid=9661), University of Freiburg, Freiburg, Germany.
* [Bioinformatics Web Application Developer-Biology](https://jobs.wustl.edu/) (Job ID 32899), Washington University in St. Louis, Missouri, United States
* [Software developer and Post-docs](http://gehlenborg.com/open-positions/), Gehlenborg Lab, Harvard Medical School, Boston, Massachusetts, United States
* [Postdoctoral Research Positions](https://chk.tbe.taleo.net/chk01/ats/careers/requisition.jsp?org=BCM&cws=42&rid=8453), Molecular and Cellular Biology Department at Baylor College of Medicine, Houston, Texas, United States 
* [Software Engineer](/src/News/SoftwareEngineerOHSU/index.md), Oregon Health Sciences University, Portland, Oregon, United States

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

----


# New Public Galaxy Servers

There are [two new publicly accessible Galaxy servers](/src/PublicGalaxyServers/index.md):

## MGEScan

<div class='right solid'><a href='http://mgescan.readthedocs.org/en/latest/server.html'><img src="/src/PublicGalaxyServers/MGEScanVideoStart.png" alt="MGEScan on Galaxy Workflow System" width="300px" /></a></div>

* *Link(s):*
  * [MGEScan on Galaxy Workflow System](http://mgescan.readthedocs.org/en/latest/server.html)
* *Domain/Purpose:*
  * Identifying long terminal repeats (LTR) and non-LTR retroelements in eukaryotic genomic sequences. 
* *Comments:*
  * ENA Browser or local storage is used to obtain input genome sequences including a traditional file upload. HMMER 3.1b1 is applied to gain speed boosts compared to a previous version HMMER 2+. In addition Generic Feature Format Version 3 is used for visualization of genome sequence data via a web-based genome browser e.g. UCSC Genome Browser or Ensembl Genome Browser. 
  * MGESCan is also [accessible through Amazon Cloud (EC2)](http://mgescan.readthedocs.org/en/latest/aws.html), [Galaxy Tool Shed](http://mgescan.readthedocs.org/en/latest/toolshed.html) or [Published Workflow](https://usegalaxy.org/u/hyungro-lee/w/retrotminer-imported-from-uploaded-file) on the public galaxy server (usegalaxy.org)
  * [MGEScan: a Galaxy based system for identifying retrotransposons in genomes](http://bioinformatics.oxfordjournals.org/content/early/2016/04/07/bioinformatics.btw157) by Hyungro Lee1, Minsu Lee, Wazim Mohammed Ismail, Mina Rho, Geoffrey Fox, Sangyoon Oh, and Haixu Tang, *Bioinformatics* (2016) doi: 10.1093/bioinformatics/btw157
* *User Support:*
  * [MGEScan on Galaxy QuickStart Tutorial](http://mgescan.readthedocs.org/en/latest/tutorial.html)
  * [MGEScan documentation](http://mgescan.readthedocs.org/en/latest/)
* *Quotas:* 
  * MGEScan can be used anonymously or with a login. Anyone can create a login.
* *Sponsor(s):*
  * [School of Informatics and Computing](http://www.soic.indiana.edu/), [Indiana University](http://www.indiana.edu/)

## Koslicki Lab

<div class='right solid'><a href='http://math-galaxy.cgrb.oregonstate.edu:8080/'><img src="/src/Images/Logos/OregonStateU.jpg" alt="Koslicki Lab Server" height="200" /></a></div>

* *Link:*
  * [Koslicki Lab Galaxy server](http://math-galaxy.cgrb.oregonstate.edu:8080/)
* *Domain/Purpose:*
  * Web based access to tools created by the [Koslicki Lab](http://www.math.oregonstate.edu/~koslickd/) at [Oregon State University](http://www.oregonstate.edu/):
    * [ARK](https://github.com/dkoslicki/ARK) and [Quikr](https://github.com/dkoslicki/Quikr)/[SEK](https://github.com/dkoslicki/SEK) for 16S rRNA based taxonomic profiling
    * [MetaPalette](https://github.com/dkoslicki/MetaPalette) for whole genome shotgun based taxonomic profiling
* *Comments:*
* *User Support:*
* *Quotas:*   
* *Sponsor(s):*
  * [Koslicki Lab](http://www.math.oregonstate.edu/~koslickd/) at [Oregon State University](http://www.oregonstate.edu/)

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/Teach/index.md'><img src="/src/Images/GalaxyLogos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/Community/Logs/index.md'><img src="/src/Images/Logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/Community/Deployments/index.md'><img src="/src/Images/Logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your <a href='/src/Teach/Resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/Teach/Trainers/index.md#add-a-trainer'>experience</a> <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/Community/Logs/index.md#add-a-log-page'>Share your experience</a> <em><a href='/src/Community/Logs/index.md#add-a-log-page'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/Community/Deployments/index.md#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/src/Community/Deployments/index.md#add-your-galaxy-deployment'>now</a></em> </strong> </td>
  </tr>
</table>

<br />

One new [training resource](/src/Teach/Resources/index.md) was added in April:

* [Freiburg hands-on training collection](/src/Teach/Resource/FreiburgHandsOnTraining/index.md).

----

# Releases

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/src/Images/Logos/PlanemoLogo.png" alt="" width="200" /></a></div>

## Planemo 0.24.2

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  April releases features these updates:

* Fix test summary report. [Pull Request 429](https://github.com/galaxyproject/planemo/pull/429)
* Improve error reporting when running shed_test. [ce8e1be](https://github.com/galaxyproject/planemo/commit/ce8e1be)
* Improved code comments and tests for shed related functionality. [89674cb](https://github.com/galaxyproject/planemo/commit/89674cb)
* Rev galaxy-lib dependency to 16.4.1 to fix wget usage in newer versions of wget. [d76b489](https://github.com/galaxyproject/planemo/commit/d76b489)
* Revert "check `.shed.yml` owner against credentials during shed creation", test was incorrect and preventing uploads. [Pull Request 425](https://github.com/galaxyproject/planemo/pull/425), [Issue 246](https://github.com/galaxyproject/planemo/issues/246)

See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).

## galaxy-lib 16.7.0

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/src/Images/GalaxyLogos/pulsar_transparent.png" alt="Pulsar" width="150" /></a></div>

## Pulsar 0.7.0

[Pulsar 0.7](https://pypi.python.org/pypi/pulsar-app/) was released in April.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

## Earlier Releases

**Galaxy v16.01**

<div class='right'><a href='http://getgalaxy.org'><img src="/src/Images/Logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The [January 2016 (v16.01) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.01_announce.html) features

* Interactive Tours
* Wheels
* Nested Workflows

See the [announcement](https://docs.galaxyproject.org/en/master/releases/16.01_announce.html) for full details.

<div class='red'>
Galaxy administrators should also be aware of [these security announcements](/src/GalaxyUpdates/2016_05/index.md#security-announcements) below.
* [Read/write arbitrary filesystem paths, arbitrary code execution](http://dev.list.galaxyproject.org/Galaxy-Security-Vulnerability-Read-write-arbitrary-filesystem-paths-arbitrary-code-execution-td4668936.html)
* [Tool Shed Security Vulnerability - Read/write arbitrary filesystem paths](http://dev.list.galaxyproject.org/Galaxy-Tool-Shed-Security-Vulnerability-Read-write-arbitrary-filesystem-paths-td4668935.html)

</div>


**Galaxy Docker Image 16.01**

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 16.01](https://github.com/bgruening/docker-galaxy-stable/releases/tag/16.01) as well.  

**[CloudMan](/src/CloudMan/index.md) 16.03 **

<div class='right'><a href='/src/CloudMan/index.md'><img src="/src/Images/GalaxyLogos/cloudman-logo.jpg" alt="CloudMan" width="180" /></a></div>

**We just released an update to Galaxy CloudMan on AWS.** CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

Most notable changes include:
* Galaxy 16.01 release
* A fine-grained control over auto-scaling options
* Several fixes to cluster sharing and cloning

See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a more complete set of changes.


**CloudBridge 0.1.0**

The Galaxy Team is proud to be part of the development team for a new cross-cloud library called CloudBridge. *CloudBridge is a Python library providing a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud.* The library is generally applicable to any domain wishing to run cloud-independent applications. There is already support for Amazon and OpenStack clouds with support for Google’s Compute Engine in development. 

The first version of CloudBridge was released earlier this month and it comes with [detailed user documentation](http://cloudbridge.readthedocs.org/). The source code is [available on Github](https://github.com/gvlproject/cloudbridge).



**Starforge 0.1**

<div class='right'>
<a href='https://github.com/galaxyproject/starforge/blob/master/README.rst'><img src="/src/Images/GalaxyLogos/starforge_logo_big.png" alt="Starforge" width="180" /></a>
</div>

[Starforge](https://github.com/galaxyproject/starforge/blob/master/README.rst) is a collection of scripts that supports the building of components for Galaxy.  Specifically, with Starforge you can:

* Build [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) dependencies
* Build [Python Wheels](http://pythonwheels.com/) (e.g. for the [Galaxy Wheels Server](http://wheels.galaxyproject.org/))
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.

Documentation can be found at [starforge.readthedocs.org](http://starforge.readthedocs.org/). 


**BioBlend 0.7.0**

   [BioBlend version 0.7.0](https://github.com/galaxyproject/bioblend/) was released at the beginning of November. BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/src/Learn/API/index.md).  CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG).


**blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and BioCloudCentral.


----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/Images/Logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

Sorry. Ran out of time.  Look for a double batch in the June News.
