---
autotoc: true
title: December 2013 Galaxy Update
---


<div class='right'></div>

<div class='left'><a href='/src/GalaxyUpdates/index.md'><img src="/src/Images/Logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the December 2013 *[Galaxy Update](/src/GalaxyUpdates/index.md)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/src/GalaxyUpdates/index.md)* complement the *[Galaxy Development News Briefs](/src/DevNewsBriefs/index.md)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# 50+ Public Servers

Three new servers were added to [the list of publicly accessible Galaxy servers](/src/PublicGalaxyServers/index.md) last month.  The number of servers has more than *doubled* in the past 12 months, with the new additions bringing the total to 51. 

The server list has also been reorganized from a single alphabetical list into three distinct depending on server breadth of coverage. From generality to specificity the 3 lists are:

 **[General Purpose Servers](/src/PublicGalaxyServers/index.md#general-purpose-servers)**
    These servers support analysis across a wide range of biology.
 **[Domain Servers](/src/PublicGalaxyServers/index.md#domain-servers)**
    These servers are dedicated to a particular domain, either within the tree of life, or in a particular type of analysis.
 **[Tool Publishing Servers](/src/PublicGalaxyServers/index.md#tool-publishing-servers)**
    These servers' primary function is to enable researchers to use a specific set of locally developed tools, without having to install them locally and use command line interfaces.

The newly added servers are:

## OSDDlinux LiveGalaxy

<div class='right solid'><a href='http://osddlinux.osdd.net:8001/'><img src="/src/PublicGalaxyServers/OSDDLinuxLiveGalaxyHomePage300.png" alt="OSDDlinux LiveGalaxy"  /></a></div>

* *Links:*
  * [OSDDlinux LiveGalaxy](http://osddlinux.osdd.net:8001/), part of the [OSDD Portal](http://sysborg2.osdd.net).
  * *Customized Galaxy with Applications as Web Services and on the Grid for Open Source Drug Discovery*, presentation by Anshu Bhardwaj and Chintalapati Janaki at [GCC2011](http://wiki.galaxyproject.org/Events/GCC2011) ([Slides](PLACEHOLDER_ATTACHMENT_URL/src/Events/GCC2011/OpenSourceDrugDiscovery.pdf), [Video](http://vimeo.com/24870112)).
* *Domain/Purpose:* 
    [OSDDlinux LiveGalaxy](http://osddlinux.osdd.net:8001/) exposes the tools that are included in [OSDDlinux](http://osddlinux.osdd.net/), "a customized linux operating system for drug discovery that integrates open source software, libraries, workflows and web services in linux for creating environment for drug discovery."   OSDDLinux contains a Galaxy instance with applications for Drug Discovery.
* *Comments:*
    [OSDDlinux LiveGalaxy](http://osddlinux.osdd.net:8001/) integrates GPSR1.0, GPSR2.0 along with the [webservers](http://osddlinux.osdd.net/livew.php) and the [standalone software](http://osddlinux.osdd.net/stand.php) supported by OSDDlinux.  See the [complete list of tools and services](http://osddlinux.osdd.net/liveg) included in OSDDlinux !LiveGalaxy. 
* *User Support:*
    [Web form](http://crdd.osdd.net/osddlinux/feed.php) or [mailto:osddlinux AT gmail DOT com](Email)
* *Quotas:* 
* *Sponsor(s):*
    The [Open Source Drug Discovery Project](http://www.osdd.net/) and the [OSDD Linux Project](http://osddlinux.osdd.net/) under the direction of [Dr. GPS Raghava](http://www.imtech.res.in/raghava/about.html) of the [CSIR-Institute of Microbial Technology (IMTECH)](http://www.imtech.res.in/).

## Dostie Lab Galaxy

<div class='right solid'><a href='http://galaxy.bci.mcgill.ca/'><img src="/src/Images/Logos/McGillUniversityLogo.jpg" alt="Dostie Lab Galaxy @ McGill University"  /></a></div>

* *Link:*
    [Dostie Lab Galaxy](http://galaxy.bci.mcgill.ca/)
* *Domain/Purpose:*
    Chromosome Conformation Capture (3C) and 3C Carbon Copy (5C) analysis tools.
* *Comments:*
    "Recent studies indicate that genomes are organized into dynamic three-dimensional physical networks, which are required for correct gene expression. The functional (physical) connectivity between genes and regulatory DNA elements must be mapped to understand how correct gene expression is obtained and ultimately how the genome functions." These tools  quantitatively measure the intensity of individual DNA contacts within the genome.
* *User Support:*

* *Quotas:* 
    Must create a login to run tools.  Anyone can create a login.
* *Sponsor(s):*
    [Canadian Institute of Health Research](http://www.cihr-irsc.gc.ca/), and the [Canadian Cancer Society](http://www.cancer.ca/research/)

## PreSTIGE

<div class='right solid'><a href='http://prestige.case.edu/'><img src="/src/PublicGalaxyServers/PreSTIGE.png" alt="PreSTIGE" width="300" /></a></div>

* *Link:*
    [PreSTIGE](http://prestige.case.edu/)
* *Domain/Purpose:*
    "PreSTIGE (Predicting Specific Tissue Interactions of Genes and Enhancers) predicts cell-line specific gene-enhancer interactions for a user-provided cell line as well as 12 PreSTIGE cell lines."
* *Comments:*
  * Users can generate predictions for any cell type of interest for which !H3K4me1 ChIP-seq and RNA-seq data are available.
  * See "[Combinatorial effects of multiple enhancer variants in linkage disequilibrium dictate levels of gene expression to confer susceptibility to common traits](http://genome.cshlp.org/content/early/2013/11/06/gr.164079.113.abstract)" by Corradin, *et al.*, *[Genome Research](http://genome.cshlp.org/)* (2013)
* *User Support:*
    [PreSTIGE User Guide](http://genetics.case.edu/prestige/misc/Scacheri_PreSTIGE_User_Guide.pdf), [mailto:olivia DOT corradin AT case DOT edu](Email)
* *Quotas:* 
    Available for academic use only
* *Sponsor(s):*
    [Scacheri Lab](http://genetics.case.edu/page.php?page_id=5&LN=Scacheri&FN=Peter) at [Case Western Reserve University](http://www.case.edu/), [Department of Genetics and Genome Sciences](http://genetics.case.edu/)

<br />
# New Papers

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 18 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 26 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 11 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>


*61 papers* (a new record) were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in November. Some papers that may be particularly interesting to the Galaxy community:

* "[BS-Seeker2: a versatile aligning pipeline for bisulfite sequencing data](http://dx.doi.org/10.1186/1471-2164-14-774)" by Guo, *et al.,* *BMC Genomics*, Vol. 14, No. 1. (10 November 2013), 774, doi:10.1186/1471-2164-14-774

* "[Ten recommendations for creating usable bioinformatics command line software](http://dx.doi.org/10.1186/2047-217x-2-15)" by Torsten Seemann, *!GigaScience*, Vol. 2, No. 1. (2013), 15, doi:10.1186/2047-217x-2-15

<br />

# Who's Hiring

<div class='right'><a href='/src/GalaxyIsHiring/index.md'><img src="/src/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Stage Master2](http://bit.ly/1hQIpTZ) "Annotation du transcriptome de l’esturgeon" Bordeaux, France 
* [Sysadmin/Computing Officer](http://bit.ly/1hAHXfa) at Queen Mary University of London
* [Informatics Software Support Specialist](http://bit.ly/1cdfOpr) at the University of Minnesota, United States 
* [CDD Gif-sur-Yvette NGS et Variation Structurales UMR de Génétique Végétale](http://bit.ly/1exDipD), INRA-Université Paris Sud-CNRS 
* [CDD en bioinformatique : recherche de variants structuraux par reséquençage (NGS)](http:////bit.ly/1h1yQnf)
* [Statistical Genomics Postdoc](/src/News/StatisticalGenomicsPostDocPennState/index.md) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/src/GalaxyIsHiring/index.md)**, *including positions at George Washington University*.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

# Events

## GCC2014: June 30 - July 2, Baltimore

<div class='right'><a href='/src/Events/GCC2014/index.md'><img src="/src/Images/Logos/GCC2014LogoTall200.png" alt="GCC2014: June 30 - July 2" width="150" /></a></div>

The [2014 Galaxy Community Conference (GCC2014)](/src/Events/GCC2014/index.md) will be held June 30 through July 2, at the [Homewood Campus](http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/) of [Johns Hopkins University](http://jhu.edu), in [Baltimore, Maryland](http://visitors.baltimorecity.gov/), United States.

### Training Day Nominations, December 2-20

GCC2014 will start with a [Training Day](/src/Events/GCC2014/TrainingDay/index.md) on June 30.  As for [GCC2013](/src/News/GCC2013TrainingDayNominations/index.md), the specific topics offered will be determined by you, the Galaxy Community.  Topic nomination will open December 2 and remain open through December 20.  Nominated topics will be posted to the [Training Day page](/src/Events/GCC2014/TrainingDay/index.md) as they come in.  The organizers will then collate the nominations into a single list that the community (yes, that means you) will vote on the nominations in January.  The final schedule of topics and instructors will be published no later than February 10, when registration opens.

Watch the [Galaxy Mailing Lists](/src/MailingLists/index.md) for announcements and links to the nomination form.

*[Training Day](/src/Events/GCC2014/TrainingDay/index.md)* is an opportunity to learn about all things Galaxy including using Galaxy, deploying and managing Galaxy, extending Galaxy, and Galaxy internals.  There will be 4 parallel tracks, each with 3 sessions, with each of those sessions two and half hours long.  *That's 12 sessions and 30 hours of workshop material.*

### Sponsorships

<div class='left'><a href='http://lifetech.com/'><img src="/src/Images/Logos/IonTorrentLogo340.png" alt="Ion Torrent by Life Technologies" width="170" /></a><br />
</div>

We are pleased to announce several [sponsorships](/src/Events/GCC2014/SponsorExhibit/index.md) for GCC2014.  First, [Ion Torrent](http://lifetech.com/) will again be the exclusive Platinum Sponsor for the annual Galaxy meeting.  This is the third year in a row that Ion Torrent has been the Platinum Sponsor of this event. 

<br />
<div class='right'><a href='http://bioteam.net/'><img src="/src/Images/Logos/BioTeamLogo154.gif" alt="BioTeam" width="120" /></a></div>

We are also pleased to announce that [BioTeam](http://bioteam.net) will again be a [Gold Sponsor](/src/Events/GCC2014/SponsorExhibit/index.md) for this annual event.  This is the second year in a row that !BioTeam has been a GCC Gold Sponsor.

There are still [Gold, Silver and Bronze sponsorships](/src/Events/GCC2014/SponsorExhibit/index.md) still available. Please contact the [Organizers](/src/Events/GCC2014/Organizers/index.md) if your organization would like to help sponsor this event. 

### Exhibitors

In 2014 we are also adding [non-sponsor exhibit spaces](/src/Events/GCC2014/SponsorExhibit/index.md#exhibit) in addition to the sponsor exhibits.  This will significantly increase the size of the exhibit floor. Please contact the [Organizers](/src/Events/GCC2014/Organizers/index.md) if your organization would like to have an exhibit space at GCC2014. 

## UC Davis Bioinformatics Boot Camps

<div class='left'><a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'><img src="/src/Images/Logos/UCDavisGenomeCenterLogo.jpg" alt="Boot Camps @ UC Davis Bioinformatics Core" width="60" /></a><br />
&nbsp;<img src="/src/Images/Logos/galaxyLogoSymbol.png" alt="Galaxy" width="55" /></div>

**There is still space in 3 of the 4 [UC Davis Bioinformatics Bootcamps](https://training.bioinformatics.ucdavis.edu/bootcamps/) this month.**

The [next offering of UC Davis Bioinformatics Bootcamps](https://training.bioinformatics.ucdavis.edu/bootcamps/) will be held on the [UC Davis](http://www.ucdavis.edu/) campus December 10-13. 

These focused one-day courses are for researchers looking to get up to speed quickly on the latest technologies and techniques in bioinformatics. Students will work on their own laptops and have continued access to software and example data used in the exercises through a [public Amazon Web Services virtual machine](http://bioinformatics.ucdavis.edu/software/). The first three bootcamps will use the Galaxy platform, and the final bootcamp will use both Galaxy and the command-line. The Alignment and Assembly bootcamps (Dec. 11th & 12th) require you to know Galaxy, so if you are unfamiliar with Galaxy, you should also take the Introduction bootcamp on Dec. 10th.

<div class='right'><a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'><img src="/src/Images/Logos/UCDavisLogoSmall.gif" alt="UC Davis"  /></a></div>

   **Tuesday, December 10:**
      **[Introduction to Next-Generation Sequence Analysis with Galaxy](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-next-generation-sequence-analysis-with-galaxy-december-10-2013/)**<br />
      <div class='red'>This Boot Camp is full.</span>

   **Wednesday, December 11:**
      **[Next-Generation Sequence Alignment and Variant Discovery](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-next-generation-sequence-alignment-and-variant-discovery-december-11-2013/)**

   **Thursday, December 12:**
      **[Genome Assembly using Next-Generation Sequence Data](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-genome-assembly-using-next-generation-sequence-data-december-12-2013/)**

   **Friday, December 13:**
      **[Introduction to the Amazon Cloud for Galaxy and the Command-Line](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-the-amazon-cloud-for-galaxy-and-the-command-line-december-13-2013/)**

Enrollment for each bootcamp is capped at 24 students. Please enroll early to be assured of a seat, as these bootcamps usually fill up quickly! More information, including full descriptions of each bootcamp is [available online](https://training.bioinformatics.ucdavis.edu/bootcamps/). The cost for each bootcamp is $200 (academic/government) or $250 (non-academic/industry).

## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src="/src/Images/Icons/CalendarIcon.gif" /></a></div> There is a lot going on in the next three months. Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<br />

<div class='center'> 
<a href='http://www.qfab.org/events/upcoming/'><img src="/src/Images/Logos/QFABLogo.png" alt="QFAB Workshops" height="80" /></a>
<a href='http://www.ifb-galaxy.org/4dec2013.html'><img src="/src/Images/Logos/GalaxyIFB.png" alt="Galaxy Day" height="60" /></a>
<a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'><img src="/src/Images/Logos/UCDavisGenomeCenterLogo.jpg" alt="UC Davis Bioinformatics Bootcamps" height="70" /></a>
<a href='http://www.intlpag.org/'><img src="/src/Images/Logos/PAGLogo114.png" alt="Plant and Animal Genome (PAG 2014)" height="80" /></a>
<a href='http://gmod.org/wiki/Jan_2014_GMOD_Meeting'><img src="/src/Images/Logos/GMOD2014.png" alt="2014 GMOD Meeting" height="70" /></a>
</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> December 4 </th>
    <td> </em><a href='http://www.ifb-galaxy.org/4dec2013.html'>Galaxy Day</a><em><div class='indent red'>Les inscriptions au Galaxy Day sont closes</div> </td>
    <td> Paris (Institut Curie), France </td>
    <td> <a href="mailto:ifb DOT galaxy AT sb DASH roscoff DOT fr">Organisateurs</a> </td>
  </tr>
  <tr>
    <th> December 9 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-an-introduction-to-de-novo-genome-assembly-using-galaxy/'>Workshop : An Introduction to De Novo Genome Assembly using Galaxy</a><em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> December 10 </th>
    <td> </em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-next-generation-sequence-analysis-with-galaxy-december-10-2013/'>Introduction to Next-Generation Sequence Analysis with Galaxy</a><em> <div class='indent red'>Session is full but waiting list is open</div></td>
    <td rowspan=4> </strong><a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'>UC Davis Bioinformatics Bootcamps</a><strong>, Davis, California, United States </td>
    <td rowspan=4> <a href="mailto:training DOT ucdbio AT gmail DOT com">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th> December 11 </th>
    <td> </em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-next-generation-sequence-alignment-and-variant-discovery-december-11-2013/'>Next-Generation Sequence Alignment and Variant Discovery</a><em> </td>
  </tr>
  <tr>
    <th> December 12 </th>
    <td> </em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-genome-assembly-using-next-generation-sequence-data-december-12-2013/'>Genome Assembly using Next-Generation Sequence Data</a><em> </td>
  </tr>
  <tr>
    <th> December 13 </th>
    <td> </em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-the-amazon-cloud-for-galaxy-and-the-command-line-december-13-2013/'>Introduction to the Amazon Cloud for Galaxy and the Command-Line</a><em> </td>
  </tr>
  <tr>
    <th> December 11 </th>
    <td> </em><a href='https://wiki.nci.nih.gov/display/CBIITSpeakers/2013/11/14/December+11%3A+Dr.+Jeremy+Goecks%2C+Using+Galaxy+to+Understand+Cancer+Genomics'>Using Galaxy to Understand Cancer Genomics</a><em> </td>
    <td> <a href='http://www.cancer.gov/aboutnci/ncishadygrove'>National Cancer Institute</a>, Rockville, Maryland, United States </td>
    <td> <a href='/src/JeremyGoecks/index.md'>Jeremy Goecks</a> </td>
  </tr>
  <tr>
    <th colspan=4> <span style="font-size: larger;"> 2014 </span> </th>
  </tr>
  <tr>
    <th rowspan=2> January 11-15 </th>
    <td> </em><a href='https://pag.confex.com/pag/xxii/webprogram/Session2298.html'>Galaxy for NGS Data Analysis: A Hands-on Computer Demo</a><em> </td>
    <td rowspan=2> </strong><a href='http://www.intlpag.org/'>Plant and Animal Genome XXII (PAG 2014)</a><strong>, San Diego, California, United States </td>
    <td rowspan=1> <a href='/src/DaveClements/index.md'>Dave Clements</a>, <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a> </td>
  </tr>
  <tr>
    <td> </em><a href='https://pag.confex.com/pag/xxii/webprogram/Session2159.html'>Galaxy Cloudman: A Gentle Introduction to Data Analysis on the Cloud</a><em><div class='indent'>Part of the <a href='https://pag.confex.com/pag/xxii/webprogram/Session2159.html'>GMOD Workshop</a></div> </td>
    <td rowspan=1> <a href='/src/DaveClements/index.md'>Dave Clements</a>, <a href='http://gmod.org/wiki/User:Scott'>Scott Cain</a> </td>
  </tr>
  <tr>
    <th> January 16-17 </th>
    <td> <a href='http://gmod.org/wiki/Jan_2014_GMOD_Meeting'>2014 GMOD Meeting</a> </td>
    <td> San Diego, California, United States </td>
    <td> <a href='/src/DaveClements/index.md'>Dave Clements</a>, <a href='http://gmod.org/wiki/User:Scott'>Scott Cain</a> </td>
  </tr>
  <tr>
    <th> February 10-14 </th>
    <td> </em><a href='http://www.supagro.fr/web/pages/?idl=19&page=498&id_page=2666'>Analyse Bioinformatique de séquences pour l'amélioration des plantes</a><em> </td>
    <td> Campus la Gaillarde, Montpellier, France </td>
    <td> <a href="mailto:yolande DOT olivier AT supagro DOT inra DOT fr">Yolande OLIVIER</a> </td>
  </tr>
  <tr>
    <th> May 27 </th>
    <td> </em><a href='http://www.mipro.hr/LinkClick.aspx?fileticket=y1%2bUY%2b54jB4%3d&tabid=110&mid=461&language=en-US&forcedownload=true'>Enabling Data Analysis with Galaxy CloudMan workshop</a><em> </td>
    <td> <a href='http://www.mipro.hr/CallForPapers/tabid/176/language/en-US/Default.aspx'>MIPRO 2014</a>, Opatija, Croatia </td>
    <td> <a href='/src/EnisAfgan/index.md'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 2 </th>
    <td> </strong><a href='/src/Events/GCC2014/index.md'>2014 Galaxy Community Conference (GCC2014)</a><strong> </td>
    <td> <a href='http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/'>Homewood Campus</a> of <a href='http://jhu.edu'>Johns Hopkins University</a>, <a href='http://visitors.baltimorecity.gov/'>Baltimore, Maryland</a>, United States </td>
    <td> <a href='/src/Events/GCC2014/Organizers/index.md'>Organizers</a> </td>
  </tr>
</table>

<br />

# Galaxy Distributions

**[Nov 04, 2013 Galaxy Distribution](/src/DevNewsBriefs/2013_11_04/index.md)**

<div class='right'>
<a href='/src/http:usegalaxy.org/index.md'><img src="/src/Images/NewsGraphics/2013_11_04_usegalaxy.org.jpg" alt="usegalaxy.org" width=300 /></a><br />**usegalaxy.org**</div>
<br />
**Highlights:**

* A ***security vulnerability*** with filter tools trapped and fixed: **[Upgrade or Patch NOW](/src/DevNewsBriefs/2013_11_04/index.md#security_fix_now)**
* Galaxy Tool Migration: [48 tools migrated to Tool Shed](http://wiki.galaxyproject.org/DevNewsBriefs/2013_11_04#New_Tool_Migrations) for a leaner distribution.
* Improved [tools for administrators](/src/DevNewsBriefs/2013_11_04/index.md#admin) (email verification, reports).
* The framework for [Tools](/src/DevNewsBriefs/2013_11_04/index.md#tools), [Visualizations](/src/DevNewsBriefs/2013_11_04/index.md#visualizations), and the [Core](/src/DevNewsBriefs/2013_11_04/index.md#core) have been upgraded for performance and scalability. 
* Come see! A whopping [26 pull requests incorporated](/src/DevNewsBriefs/2013_11_04/index.md#pull_requests_merged)!. Many thanks to our open source community!
* Numerous [Tool Shed enhancements and upgrades](/src/DevNewsBriefs/2013_11_04/index.md#tool_shed): API, `READMEs`, Functional Tests, easier installs, and much much more.
* Plus enhancements to [Workflows](/src/DevNewsBriefs/2013_11_04/index.md#workflows), [API](/src/DevNewsBriefs/2013_11_04/index.md#api), [CloudLaunch](/src/DevNewsBriefs/2013_11_04/index.md#cloudlaunch), [UI](/src/DevNewsBriefs/2013_11_04/index.md#ui), and [Bug Fixes](/src/DevNewsBriefs/2013_11_04/index.md#bug_fixes).

[http://getgalaxy.org](http://getgalaxy.org)

[http://bitbucket.org/galaxy/galaxy-dist](http://bitbucket.org/galaxy/galaxy-dist)

[http://galaxy-dist.readthedocs.org](http://galaxy-dist.readthedocs.org)

```
new:       $ hg clone https://bitbucket.org/galaxy/galaxy-dist#stable

upgrade:   $ hg pull 
           $ hg update release_2013.11.04
```


**[CloudMan](/src/News/CloudManRelease/index.md)**

A new version of [CloudMan](/src/CloudMan/index.md) was [released in July](/src/News/CloudManRelease/index.md).

<br />

# ToolShed Contributions

<div class='right'><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/Images/Logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

## New Public ToolSheds

This is the first time we've highlighted publicly accessible !ToolSheds.  We will continue to advertise this as we become aware of them.

### GenOuest ToolShed

The [GenOuest public ToolShed](http://toolshed.genouest.org/) contains over 60 valid tools in 36 repositories in 9 categories.

## Repositories in the Galaxy Project ToolShed

There were many...

* [fastx_barcode_splitter_enhanced](http://bit.ly/1euqt1G): allows barcodes in a separate index read, loads results into a history 
* [shear](http://bit.ly/HNDViX): Sample Heterogeneity Estimation and Assembly by Reference 
* [dimont_motif_discovery](http://bit.ly/184et5t): A general approach for de-novo motif discovery from high-throughput data 
* [arts](http://bit.ly/1dbHluC): Automated Randomization of multiple Traits for Study design  
* [samtools_idxstats](http://bit.ly/1io30Sm): BAM mapping statistics (using samtools idxstats) 
* [vcftools](http://bit.ly/gxyshed)_{annotate, compare, isec, merge, slice, subset} and all_vcftools  
* [filter_tools](http://bit.ly/183Vdou): Various Imaging Filters including Median, Emboss, Blur, Contour, Detail, ... 
* [image_tools](http://bit.ly/InsBed): General Image Tools: Image Filmify, Image Tileify, Image Rotate, Image Resize 
* and ...

## deepTools

<div class='right solid'><img src="/src/GalaxyUpdates/2013_12/deepToolsCollage.png" alt="deepTools" width="500" /></div>

[deepTools](http://toolshed.g2.bx.psu.edu/view/bgruening/deeptools) is a suite of tools that close the software gap between the very first steps of data processing of deeply sequenced data and biologically insightful downstream analyses. deepTools comprise efficient and reliable programs that enable users to perform quality controls and to generate bigWig files of normalised read coverages with different scaling approaches. Furthermore, the tools offer fast computation of coverages for user-specified genomic regions including their visualisation as meta-profiles and heatmaps. deepTools can be installed via the [ToolShed](http://toolshed.g2.bx.psu.edu/view/bgruening/deeptools) and source is at [GitHub](https://github.com/fidelram/deepTools/).

# Other News

* [bioblend 0.4.1](http://bit.ly/IhF6s2) CloudMan and Galaxy API library released 
* New accounts on [Main](/src/Main/index.md) now have to activate using the link sent to email address or they can't run jobs (includes upload).  Want to enable this on your own Galaxy instance?  See the [User Accounts Administration](/src/Admin/UserAccounts/index.md) page.

