---
title: The October 2015 Galactic News!
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/galaxy-updates/'><img src="/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the October 2015 *[Galactic News](/galaxy-updates/)*, a summary of what is going on in the Galaxy community. It's been, and will be, a busy few months.  If you have anything to include in the October News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org).

# New Papers

**60** new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in September. Highlights include:

* [WiseScaffolder: an algorithm for the semi-automatic scaffolding of Next Generation Sequencing data](https://doi.org/10.1186/s12859-015-0705-y)
    *BMC Bioinformatics*, Vol. 16, No. 1. (03 September 2015), 281, doi:10.1186/s12859-015-0705-y, by Gregory K. Farrant, Mark Hoebeke, Frédéric Partensky, Gwendoline Andres, Erwan Corre, Laurence Garczarek

* [Discriminating somatic and germline mutations in tumor DNA samples without matching normals](https://doi.org/10.1101/gr.183053.114)
    *Genome Research*, Vol. 25, No. 9. (01 September 2015), pp. 1382-1390, doi:10.1101/gr.183053.114. Saskia Hiltemann, Guido Jenster, Jan Trapman, Peter van der Spek, Andrew Stubbs

* [VING: a software for visualization of deep sequencing signals](https://doi.org/10.1186/s13104-015-1404-5)
    *BMC Research Notes*, Vol. 8, No. 1. (07 September 2015), 419, doi:10.1186/s13104-015-1404-5. Marc Descrimes, Yousra B. Zouari, Maxime Wery, Rachel Legendre, Daniel Gautheret, Antonin Morillon

* [Hammock: A Hidden Markov model-based peptide clustering algorithm to identify protein-interaction consensus motifs in large datasets](https://doi.org/10.1093/bioinformatics/btv522)
    *Bioinformatics* (05 September 2015), btv522, doi:10.1093/bioinformatics/btv522. Adam Krejci, Theodore Hupp, Matej Lexa, Borivoj Vojtesek, Petr Muller

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 46 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


----

# Events

## October GalaxyAdmins Meetup

<div class='right'><a href='/community/galaxy-admins/meetups/2015-10-15/'><img src="/images/logos/GalaxyAdmins.png" alt="GalaxyAdmins meetup October 15" width="175" /></a><br /><br />
<a href='http://www.analome.com/'><img src="/images/logos/Analome.png" alt="Analome" width="175" /></a>
</div>

The [next GalaxyAdmins meetup on October 15](/community/galaxy-admins/meetups/2015-10-15/) will feature [David Kovalic](https://www.linkedin.com/in/davidkovalic) speaking on *Galaxy at [Analome](http://analome.com/) – Outreach, Opportunity and Challenges*:
 At Analome Galaxy has found another home and we like it for numerous reasons. In this talk we will present the multifaceted role Galaxy plays at Analome and how it supports ongoing bioinformatics projects while at the same time providing opportunities for social good.

[GalaxyAdmins](/community/galaxy-admins/) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  

Hope to see you there.

----

<div class='left'><br />
<a href='/events/gcc2016/'><img src="/events/gcc2016/GCC2016TrainingLogo400.png" alt="GCC2016 Training" width="150" /></a><br />
<table>
  <tr>
    <th> &nbsp; <a href='http://bit.ly/gcc2016tdnom'>Nominate a topic</a> &nbsp; </th>
  </tr>
</table>

</div>

## GCC2016 Call for Training Topic Nominations

The [2016 Galaxy Community Conference (GCC2016)](/events/gcc2016/) features two full days of training on June 26-27.  Each
day will have multiple tracks, with each track featuring several sessions throughout the day.  GCC2016 will be held at [Indiana University](http://indiana.edu) in Bloomington, Indiana, United States.

**GCC2016 Training topics are [nominated by you](http://bit.ly/gcc2016tdnom), the Galaxy Community.** Please give some thought to what topics you would like to see offered, or that you would like to offer, and then [submit them as topic nominations](http://bit.ly/gcc2016tdnom).  Any topic of interest to the Galaxy Community can be nominated and you are encouraged to nominate more than one topic. If you are looking for ideas, see

* [Topics nominated for GCC2015](http://bit.ly/gcc2015vote)
* [Topics nominated for GCC2014](http://bit.ly/1s6NtMN)
* [Topics nominated for GCC2013](http://bit.ly/1i2j1gN)
* The [Galaxy Events page](/events/), which lists what training the community is offering outside of GCCs.
* Topics that have already been nominated for GCC2016

Nominated topics will be published as they come in.  **Topic nomination closes Friday, October, 16.**.  Topics will be compiled into a uniform list by the GCC2016 Organizing Committee, and topics will be voted on by the Galaxy Community from October 26 through November 6.

Topics will then be selected and scheduled based on topic interest, and the organisers' ability to confirm instructors for each session.  Some very popular sessions may be scheduled more than once. The final schedule will be posted before registration opens.

----

## Call for GCC2017 Hosts

<table>
  <tr>
    <td style=" text-align: center; border: none;"> <a href='https://galaxyproject.org/gcc2011/'><img src="/events/GCC2011Logo400.png" alt="" width="120" /></a>&nbsp;&nbsp; <a href='/events/gcc2012/'><img src="/images/galaxy-logos/GCC2012Logo200.png" alt="2012 Galaxy Community Conference" width="85" /></a>&nbsp;&nbsp;&nbsp; <a href='/events/gcc2013/'><img src="/images/logos/GCC2013Logo200.png" alt="2013 Galaxy Community Conference (GCC2013)" width="100px" /></a>&nbsp;&nbsp;&nbsp; <a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoWide200.png" alt="GCC2014, June 30-July2" width="130" /></a>&nbsp;&nbsp;&nbsp; <a href='http://gcc2015.tsl.ac.uk/'><img src="/images/logos/GCC2015LogoWide600.png" alt="2015 Galaxy Community Conference (GCC2015)" width="100" /></a> &nbsp;&nbsp; <a href='http://galaxyproject.org/GCC2016'><img src="/events/gcc2016/GCC2016LogoFull_big.png" alt="2016 Galaxy Community Conference (GCC2016)" width="110px" /></a> </td>
  </tr>
  <tr>
    <td colspan=6 style=" text-align: center; border: none;"> <div class='solid red'> <span style="font-size: larger;"> <strong>GCC2017 needs a host!</strong><br /></span> <span style="font-size: smaller;">(and a logo)</span> </td>
  </tr>
</table>



**We are now [seeking proposals](https://depot.galaxyproject.org/hub/attachments/documents/GCC2016CallForHosts.pdf) from organizations interested in hosting GCC2017.**

GCC draws 200+ participants from the full range of data-intensive life science research.  Participants come from around the world, from all career stages, and do research spanning the tree of life.  Universities, hospitals and medical schools, research organizations, and industry are all represented, including several of the largest and most influential research organizations in the world.  

Hosts get copious (and enthusiastic) organizational support from Galaxy's Outreach Team (who have been helping to organize GCC since 2011), and even more copious gratitude (even *adulation!)* from the Galaxy Community.  

See the **[full call for host proposals](https://depot.galaxyproject.org/hub/attachments/documents/GCC2016CallForHosts.pdf)** for more details on what should be covered in proposals.  Proposals are due by the end of ~~30 October~~ **30 November 2015**. If you are interested in possibly hosting GCC2017 then please contact [Galaxy Outreach](mailto:outreach@galaxyproject.org) with any questions.

Hoping to work with you in 2017!

[Dave Clements](/people/dave-clements/) and the [Galaxy Team](/galaxy-team/)

----

## IUC Tools & Collections Hackathon Report

<div class='right'><a href='/news/iuc-collections-codefest/'><img src="/images/icons/DatasetCollectionIconInForm.png" alt="" width="150" /></a>
</div>

A [Galaxy tool developers to hack on dataset collections](/news/iuc-collections-codefest/) hackathon was held on September 17<sup>th</sup> and 18<sup>th</sup>.

Dataset collections enable MapReduce style workflows in Galaxy and have come a long way over the last year+. Several groups are using dataset collections and newer tools to express workflows of various degrees of complexity that were not possible in Galaxy before.

See the [full report](/news/iuc-collections-codefest/) details.

----

## Upcoming Events

<div class='left'>
<a href='http://bit.ly/gxycal'><img src="/images/logos/GalxyEventsCalThumb.png" alt="Galaxy Events Calendar" width="100" /></a>
</div>
<div class='right'>
<a href='http://bit.ly/gxyothercal'><img src="/images/logos/GalxyOtherEventsCalThumb.png" alt="Galaxy Other Events Calendar" width="100" /></a>
</div>

There are [upcoming events](/events/) in 4 countries on 4 continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='center'>
<a href='https://cnrsformation.cnrs.fr/stage.php?stage=15274&axe=77'><img src="/images/logos/CGFB_bioinformatics.png" alt="Analyse avancée de séquences" height="80" /></a>
<a href='/community/galaxy-admins/meetups/2015-10-15/'><img src="/images/galaxy-logos/GalaxyAdmins.png" alt="October Galaxy Admins Meetup" height="70" /></a>
<a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-4/'><img src="/images/logos/QFABLogo.png" alt="Workshop: Extended RNA-Seq analysis" height="80" /></a>
<a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'><img src="/images/logos/PRABILogo.png" alt="Analyse de données RNA-seq sous l’environnement Galaxy" height="70" /></a>
<a href='http://bit.ly/1NuRJ07'><img src="/images/logos/GalaxyIFB.png" alt="Galaxy Day 2015" height="70" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> October 1 </th>
    <td> <em>Proteogenomics and Metaproteomics using the GalaxyP Platform</em> </td>
    <td> <img src='/images/icons/AS.png' alt='Asia' align='right' /> <a href='http://www.igib.res.in/'>CSIR-Institute of Genomics & Integrative Biology (IGIB)</a>, New Dehli, India </td>
    <td> <a href='https://cbs.umn.edu/contacts/pratik-d-jagtap'>Pratik Jagtap</a> </td>
  </tr>
  <tr>
    <th> October 9 </th>
    <td> <em><a href='http://www.ashg.org/2015meeting/pages/ticketed_events.shtml#collapsesub19'>Introduction to Integrative Analysis with GenomeSpace Workshop</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' />  <a href='http://www.ashg.org/2015meeting/'>ASHG 2015</a>, Baltimore, Maryland, United States </td>
    <td> <a href="mailto:gs-help@genomespace.org">GenomeSpace</a> </td>
  </tr>
  <tr>
    <th> October<br />12-14 </th>
    <td> <em><a href='https://cnrsformation.cnrs.fr/stage.php?stage=15274&axe=77'>Analyse avancée de séquences</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.cbib.u-bordeaux2.fr/'>Bordeaux Bioinformatics Center</a>, University Bordeaux Segalen, France </td>
    <td> <a href="mailto:cfe.contact@cnrs.fr">Email</a> </td>
  </tr>
  <tr>
    <th> October 15 </th>
    <td> <em><a href='/community/galaxy-admins/meetups/2015-10-15/'>October Galaxy Admins Meetup</a></em> </td>
    <td> <img src='/images/icons/GL.png' alt='Around the World' align='right' /> <em>Online</em> </td>
    <td> <a href='https://www.linkedin.com/in/davidkovalic'>David Kovalic</a>, <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a>, <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> October 28-29 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-4/'>Workshop: Extended RNA-Seq analysis</a></em> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> October 28 - November 3 </th>
    <td> <em><a href='http://meetings.cshl.edu/courses.aspx?course=c-ecg&year=15'>Computational & Comparative Genomics Course</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Cold Spring Harbor Laboratory, New York, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/james-taylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th rowspan=2> November 3 </th>
    <td> <em><a href='http://www.festivalofgenomicscalifornia.com/workshops/galaxy-for-the-analysis-of-very-large-biomedical-data/'>Galaxy for the Analysis of Very Large Biomedical Data Workshop</a></em> </td>
    <td rowspan=2> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://www.festivalofgenomicscalifornia.com/'>Festival of Genomics California</a>, San Mateo, California, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a>  <a href='/people/anton/'>Anton Nekrutenko</a>, <a href='/people/james-taylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.festivalofgenomicscalifornia.com/workshops/single-cell-sequencing-analysis-visualisation-of-copy-number-variations/'>Single-Cell Sequencing: Analysis & Visualization of Copy Number Variations Workshop</a></em> </td>
    <td> <a href='https://www.linkedin.com/in/robertaboukhalil'>Robert Aboukhalil</a>, <a href='https://www.linkedin.com/in/tylergarvin'>Tyler Garvin</a> </td>
  </tr>
  <tr>
    <th> November 15-16 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de données RNA-seq sous l’environnement Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTN16.png" alt="Training offered by GTN Member" /></a> </div> <a href="mailto:contact@fc3bio.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 17-18 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de données RNA-seq sous l’environnement Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTN16.png" alt="Training offered by GTN Member" /></a> </div> <a href="mailto:contact@fc3bio.fr">Email</a> </td>
  </tr>
  <tr>
    <th> November 19 </th>
    <td> <a href='http://bit.ly/1NuRJ07'>Galaxy Day 2015</a> <div class='indent red'>Soumissions à envoyer avant le 5 octobre</div> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.pasteur.fr/'>Institut Pasteur</a>, Paris, France </td>
    <td> <a href="mailto:gtgalaxy@groupes.france-bioinformatique.fr">Email</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> December<br />7-11 </th>
    <td> <em><a href='http://bis15.amsi.org.au/galaxy/'>Galaxy and the Genomics Virtual Lab</a></em> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> <a href='http://bis15.amsi.org.au/'>BioInfoSummer 2015</a>, University of Sydney, NSW, Australia </td>
    <td> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='https://training.galaxyproject.org/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Support engineer for Galaxy at the CIB in the Institut Pasteur](https://c3bi.pasteur.fr/cib-team-cdd-job-application/), Paris, France

* [Assistant Computational Scientist](http://bit.ly/1OLwPbW) at [VIB](http://www.vib.be/), Gent, Belgium
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

----

# New Public Galaxy Servers

[1 new publicly accessible Galaxy servers](/use/) was added in September, bringing the total dangerously close to 80.

## ABiMS Tools

<div class='float-right'><a href='http://webtools.sb-roscoff.fr/'><img src="/images/logos/ABiMSLogo.png" alt="ABiMS Tools Galaxy Server"  /></a></div>

* *Link:*
    * [ABiMS Tools Server](http://webtools.sb-roscoff.fr/)
* *Domain/purpose:*
    * "Hosts tools/resources developed at the [Station Biologique de Roscoff](http://www.sb-roscoff.fr/) in collaboration with [ABiMS](http://abims.sb-roscoff.fr/)."
* *Comments:*
    * Available tools:
        * [MicRhoDE workflow](http://application.sb-roscoff.fr/micrhode): places query sequences into MicRhoDE tree (type-1 rhodopsin phylogeny).
        * [Renkan Blast](http://renkan.sb-roscoff.fr/): blast query sequences on 18S and 28S radiolaria databases (blastn).
        * [WiseScaffolder](http://abims.sb-roscoff.fr/wisescaffolder): genome scaffolding of pre-assembled contigs using mate-pair data.
        * [HECTAR](http://www.sb-roscoff.fr/hectar/): HEterokont subCellular TARgeting is a statistical prediction method designed to assign proteins to five different categories of subcellular targeting: Signal peptides, type II signal anchors, chloroplast transit peptides, mitochondrion transit peptides and proteins which do not possess any N-terminal target peptide.
* *User support*:
    * For any question or request for tools or account, send an email at [ABiMS Support](mailto:support.abims@sb-roscoff.fr)
* *Sponsor(s):*
    * [ABiMS](http://abims.sb-roscoff.fr/)
    * [Station Biologique de Roscoff](http://www.sb-roscoff.fr/)

<br />


----

# Galaxy Training Network News

There are two active discussions happening on the [Galaxy-Training mailing list](https://lists.galaxyproject.org/archives/list/galaxy-training@lists.galaxyproject.org/).

## Software & Data Carpentry Train the Trainer Workshop at GCC2016?

<div class='center'>
<a href='http://www.datacarpentry.org/'><img src="/images/logos/DataCarpentryLogo.png" alt="Data Carpentry" width="200" /></a> &nbsp;&nbsp;&nbsp;&nbsp;
<a href='https://software-carpentry.org/'><img src="/images/logos/SoftwareCarpentryLogo.png" alt="Software Carpentry" width="240" /></a>
</div>
<br />

We are considering offering a [Software Carpentry](https://software-carpentry.org/) / [Data Carpentry](http://www.datacarpentry.org/) *Train the Trainer* workshop immediately before [GCC2016](/events/gcc2016/).  This workshop would run for two days, and accommodate 24 people.  

This idea came up at the [GCC2015 GTN BoF](/events/gcc2015/bofs/galaxy-training-network/).  Software and Data Carpentry (S/DC) teach scientists core essentials of programming and data management. Their model is to train and certify trainers, and then provide a common curriculum and teaching infrastructure that enables those trainers to teach scientists how to program and manage data well.

If you think we should (or should not) offer this training at GCC2016, please contribute to the [discussion on the mailing list.

## Galaxy Training Network and GOBLET

<div class='left'>
<a href='http://mygoblet.org/about-us/goblet-events/goblet-agm-2015-cape-town'><img src="/images/logos/GobletLogo.png" alt="2015 GOBLET Meeting" height="100" /></a>
</div>

[GTN](https://training.galaxyproject.org/) became a member of the [Global Organisation for Bioinformatics & Training (GOBLET)](http://mygoblet.org) [earlier this year](/news/gtn-joins-goblet/).  The [2015 GOBLET Meeting](http://mygoblet.org/about-us/goblet-events/goblet-agm-2015-cape-town) will be held 18-20 November, in Cape Town, South Africa.  Fourie Joubert of the Centre for Bioinformatics and Computational Biology at the University of Pretoria will be representing GTN at GOBLET 2015.

If you have anything you'd like to have discussed at the meeting, then please [join this conversation](http://bit.ly/1iM6T5r).

----

# Galaxy Developer Code of Conduct

The Galaxy Project now has a [Developer Code of Conduct](https://github.com/galaxyproject/galaxy/blob/6edaf458e89b3db7115c65f0242b781df7c77faa/CODE_OF_CONDUCT.md#galaxy-project-code-of-conduct).

From the [introduction](https://github.com/galaxyproject/galaxy/blob/6edaf458e89b3db7115c65f0242b781df7c77faa/CODE_OF_CONDUCT.md#galaxy-project-code-of-conduct):
 This code of conduct outlines our expectations for participants within the Galaxy community, as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.

The Code of Conduct includes

* Guidelines on interaction,
* Commitment to diversity,
* and procedures for addressing issues if they arrise.

Thanks to everyone who helped put these together.

# Releases

## July 2015 Galaxy Release (v 15.07)

<div class='right'><a href='http://getgalaxy.org'><img src="/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

**The July 2015 Galaxy Release is out!**
**[Release Notes v 15.07](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html)**

**Release Highlights **

* **Interactive Environments** - The interactive environments (IE) framework features several updates this release and Galaxy is now distributed with a new IE for RStudio implemented by [Helena Rasche](/people/helena-rasche/).
* **Workflow Editor Enhancements** - The workflow editor has been significantly revamped - it now uses newer tool form components and allow construction of workflows with tools explicitly producing output collections.
* **Policies for Committers and Pull Requests** - The process for adding committers to the project and fielding pull requests has been formalized and documented in the source code. Three new committers have been added to the project - [Björn Grüning](/people/bjoern-gruening/), [Nicola Soranzo](/people/nicola-soranzo/), and [Helena Rasche](/people/helena-rasche/).
* And *[much more](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html)*

## Mid 2015 Galaxy CloudMan Release

<div class='right'><a href='/cloudman/'><img src="/images/logos/CloudManWideBlackLogo.png" alt="CloudMan" width="300" /></a></div>

**We just released an update to Galaxy CloudMan.** CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

This is the most complete update of the Galaxy on the Cloud platform to date with all the cloud components being built from the ground up. The following is a summary of the major changes:

* *Galaxy on the Cloud*: all new galaxy and indices file systems, with [Galaxy 15.07 release](/news/2015-07-galaxy-release/), [~200 preinstalled Galaxy tools](https://github.com/galaxyproject/galaxy-cloudman-playbook/blob/master/files/shed_tool_list.yaml), and nearly half a terrabyte of updated reference genome data.
* *Galaxy on the Cloud*: switched to a different Cloud Launch application, available at https://launch.usegalaxy.org/
* *Galaxy on the Cloud*: Integrated Galaxy IPython Interactive Environments
* CloudMan: Creating cluster shares is now fast (~30 seconds vs. hours)
* Created a wealth of new documentation, including [a description of the all new automated method for building private/custom instances](/cloudman/building/).

See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a more complete set of changes or just give it a try: see the all new [Getting Started](/cloudman/getting-started/) documentation.

## Security Advisories

Three security vulnerabilities were announced and patched in August.  The Galaxy Committers have released a number of fixes which have been rolled in to affected versions of Galaxy dating back to the 14.10 release.

* **[Galaxy XSS Reflection Vulnerability](https://lists.galaxyproject.org/archives/list/galaxy-announce@lists.galaxyproject.org/message/EVXN3G5FZUPOEA4OYCC6V25TWGYBBWDV/)**
* **[Galaxy LDAP Authentication Vulnerability](https://lists.galaxyproject.org/archives/list/galaxy-announce@lists.galaxyproject.org/message/GOZKUUTSB2AQSBXIGCFC6RVH37MDBQQZ/)**
* **[Tool Shed Unauthorized Upload Vulnerability](https://lists.galaxyproject.org/archives/list/galaxy-announce@lists.galaxyproject.org/message/KNJIV2QCT52SPLXMCENLA2N5747ALSSK/)**

## Other Releases

<div class='right'><a href='http://getgalaxy.org'><img src="/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

 **BioBlend 0.6.0 and 0.6.1**
    [BioBlend version 0.6.1](https://github.com/galaxyproject/bioblend/) was released in July. BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/develop/api/)([CloudMan](/cloudman/) offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG):

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/images/logos/PlanemoLogo.png" alt="" width="150" /></a></div>

 **Planemo 0.13.2**
   Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.  [Planemo 0.13.2](https://pypi.python.org/pypi/planemo/0.13.2#history) is the most recent release. See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).

 **Pulsar**
   [Pulsar 0.5.0](https://pypi.python.org/pypi/pulsar-app/0.5.0) was released in May.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

 **[CloudMan](/cloudman/) **
   The [most recent edition of CloudMan](/galaxy-updates/2014-09/#august-2014-cloudman-release) was released in August.

 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and BioCloudCentral.


----


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

<br />

1 new **[Galaxy Training Network member](https://training.galaxyproject.org/)** was added in September:

* **The Centre for Bioinformatics and Computational Biology, University of Pretoria**
    The Centre offers training in general usage, analysing and comparing protein and nucleic acid sequences, annotating genomes, gene expression patterns, investigating molecular structures, and simulating metabolic pathways.  Please welcome the first GTN member in Africa.  *(Now we just need Antartica.)*

----
<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in September](/toolshed/contributions/2015-09/).

----

# Other News

* From [Robert M Flight](https://twitter.com/rmflight):
    [Southeast Center for Integrated Metabolomics (SECIM)](http://secim.ufl.edu) has their [Galaxy Project tools on GitHub](https://github.com/secimTools/GalaxyTools). Can follow development, add to own.
* From [Brian Nosek](https://twitter.com/BrianNosek) and many others:
    Center for Open Science gives grant to [Titus Brown](https://twitter.com/ctitusbrown) and team to connect Galaxy Project and the [Open Science Framework](https://osf.io/). See the [full announcement](http://cos.io/pr/2015-09-24/).
* From [Globus](https://www.globus.org/genomics):
    Love (or like!) Globus? Vote for us in the [HPCWire Readers' Choice Awards](http://www.hpcwire.com/2015-hpcwire-readers-choice-awards/) (#16 - Best Data-Intensive System)
* From [Björn Grüning](/people/bjoern-gruening/):
    Brand new [Galaxy Interactive Environment documentation](/admin/gies/): Enterprise Deployments, Docker on a different host...
* From [Keywan Hassani-Pak](https://twitter.com/KeywanHP):
    We encountered [a big problem using bwa-eXpress in Galaxy](https://github.com/Rothamsted/AppliedBioinformatics/wiki/Transcript-abundance-estimation-in-Galaxy) for transcript abundance estimation!
* From [Saket Choudhary](https://twitter.com/saketkc):
    [GalDrive: Pipeline for comparative identification of driver mutations using Galaxy framework](http://bit.ly/GalDrive)
* From [Keith Bradnam](https://twitter.com/ACGT_blog)
    Slides from a short talk:
      [13 questions you may have about using Galaxy](http://buff.ly/1QhSwk1)
* From Markus Wolfien
    [TRAPLINE: A standardized and automated pipeline for RNA sequencing data analysis and evaluation](https://usegalaxy.org/u/mwolfien/p/trapline---manual)
* From [GigaScience](https://twitter.com/GigaScience)
    Supporting Galaxy BLAST+ wrappers, archived at the point of publication hosted in GigaDB [dx.doi.org/10.5524/100149](http://gigadb.org/dataset/100149)
* From [Björn Grüning](/people/bjoern-gruening/):
    [New Galaxy Docker images](https://github.com/bgruening/docker-galaxy-stable)! Enjoy all the new goodies!

<div class='left'>
<br />
<a href='https://storify.com/tnabtaf/galaxy-turns-10'><img src="/images/10Candles.jpg" alt="Galaxy Turns 10" width="100" /></a>
</div>

## Galaxy Turns 10

[People remembered](https://storify.com/tnabtaf/galaxy-turns-10)!  Many thanks to [Keith Bradnam](http://www.acgt.me/) for [highlighting this milestone](http://www.acgt.me/blog/2015/9/2/a-long-time-ago-in-a-galaxy-browser-far-far-away).
