---
title: May 2019 Galactic News
date: '2019-05-06'
tease: GCC2019 registration / Poster/Demo abstracts / sponsors, GTN CoFest, Platforms,
  Pubs, Jobs, Blog!, Training, Tools, Releases and more!
tags: [newsletter, galaxy]
subsites: [all-eu, global, us]
---

[<img class="float-right" src="/news/2019-05-galaxy-update/03may19-galaxy-update-image.png" alt="Galaxy News" width="350" />](/galaxy-updates/)

The *May 2019 Galactic News* is here! This is a summary of what is going on in the Galaxy community.

* Events
    * **[GCC2019](/news/2019-05-galaxy-update/#gcc2019-registration):**
        * **Early registration ends 17 May**
        * Poster and Demo Abstract Submission is open
        * Travel Fellowships: 2nd round ends this Friday
        * *GigaScience* returns as a GCC sponsor!
    * [GTN CoFest and Community Call: 16 May](/news/2019-05-galaxy-update/#gtn-cofest-and-community-call)
    * Plus [17 other upcoming events](/news/2019-05-galaxy-update/#upcoming-events) in the next 90 days
* [265 new publications](/news/2019-05-galaxy-update/#publications), great resources lead to great insight.
* Some most excellent [Galaxy Platform News](/news/2019-05-galaxy-update/#galaxy-platforms-news), including a single cell 'omics workbench, network analysis, subcellular spatial organization modeling, and more!
* A new entry to [The Galactic Blog](/news/2019-05-galaxy-update/#new-galactic-blog-post), Galaxy hearts GCP.  
* At least [11 Open positions](/news/2019-05-galaxy-update/#whos-hiring) in six countries on two continents.
* [Updates to training materials](/news/2019-05-galaxy-update/#doc-hub-and-training-updates).
* [New releases](/news/2019-05-galaxy-update/#releases).
* And [some other news](/news/2019-05-galaxy-update/#other-news) too.

If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# GCC2019 Registration

**[Registration](/events/gcc2019/registration/) for the [2019 Galaxy Community Conference (GCC2019)](/events/gcc2019/) is open! Discounted registration rates are available until 17MAY19.**

[<img class="float-right" style="max-width: 300px" src="/events/gcc2019/gcc2019-logo-big.png" alt="GCC2019" />](/events/gcc2019/)

GCC2019 will be held 1-6 July in Freiburg, Germany.  The *tenth* [GCC](/events/gcc2019/) will have many familiar features from earlier years, including accepted and lightning talks, posters and demos, birds-of-a-feather gatherings (BoFs), training, and a CollaborationFest.  GCC2019 brings the [most significant conference program update](https://gcc2019.sched.com/) in several years:

* some training will be integrated with the main conference,
* three days of conference instead of two,
* several parallel sessions,
* and a strong emphasis on organizing content by domain.

GCC2019, like every GCC before it, will be built around **community**.  Training topics are nominated, selected, and presented by the community.  Presentations will  feature the full spectrum of Galaxy applications, enhancements and deployments from the community as well.

If you are working in data intensive life science research then there will not be a better place to share your work, learn from others, and find new collaborators.

Please find additional information on GCC2019 [here](/events/gcc2019/).

## Poster & Demo Abstract Submission is Still Open

The talk submission abstract deadline was last week, but poster and demo submissions remain open!  Got something to show? [Submit an abstract](/events/gcc2019/abstracts).

## Fellowships: 2nd Round Due this Friday

[Travel fellowships to attend GCC2019](/news/2019-04-gcc-fellowships/) are available.  If you are interested, **apply by this Friday, 10 May**.


## Sponsors

We are delighted to have *GigaScience* as a returning sponsor:

### GigaScience

[<img class="mx-auto float-right" src="/images/logos/GigaScienceLogo250.png" alt="GigaScience Journal" />](https://academic.oup.com/gigascience)

*[GigaScience](https://academic.oup.com/gigascience)* is an online open access, open data, open peer-review journal published by [Oxford University Press](https://academic.oup.com/) and [BGI](https://www.bgi.com/us/). The journal offers ‘big data’ research from the life and biomedical sciences, and on top of 'Omics research includes the growing range of work that uses difficult-to-access large-scale data, such as imaging, neuroscience, ecology, systems biology, and other new types of shareable data. *GigaScience* is unique in the publishing industry as it publishes all research objects (data, software tools, source code, workflows, containers and other elements related to the work underpinning the findings in the article). Promoting Open Science, all published software needs to be under an OSI-license, all supporting data must be available and open, and all peer review is carried out transparently. Presenting workflows via our [GigaGalaxy.net](http://gigagalaxy.net/) server, **novel work presented at the meeting utilising Galaxy is eligible to a 15% APC if it is submitted to our [Galaxy series](https://academic.oup.com/gigascience/pages/galaxy_series_data_intensive_reproducible_research)**.

## GTN CoFest and community call

The Galaxy Training Network has developed an infrastructure to deliver interactive training based on Galaxy: [one central place](https://training.galaxyproject.org) to aggregate training materials covering many current research topics. Each topic is supported by different tutorials developed and maintained by the community via a [GitHub repository](https://github.com/galaxyproject/training-material).

<img class="float-right" style="max-width: 200px;" src="/images/galaxy-logos/gtn-cofests.png" alt="GTN cofest logo"/>

The Galaxy Training Network is organizing regular online CoFests (Collaboration/Contribution Fests) every 3 months on the 3rd Thursday for a day of the collaborative work on the training content.

**The next one will be on the 16th of May.** Anyone who would like to contribute on any other topics is very welcome to join.

We will coordinate via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.4k1elbsb4dutc8e1lu17030nqs) (drop-in channel we will keep open the whole day) and [the GTN Gitter channel](https://gitter.im/Galaxy-Training-Network/Lobby).

During the day, we plan to have **GTN community calls** to discuss the recent changes and future directions. To accommodate different time zones and schedules, we will have 2 calls (with similar discussions) at [8am (UTC + 2)](http://arewemeetingyet.com/Berlin/2019-05-16/08:00/GTN%20meeting) and [4pm (UTC + 2)](http://arewemeetingyet.com/Berlin/2019-05-16/16:00/GTN%20meeting) via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.4k1elbsb4dutc8e1lu17030nqs).

*If you want to join to help or to talk and learn about the project, let us know on the [Gitter channel](https://gitter.im/Galaxy-Training-Network/Lobby) or by leaving a comment on the [issue on GitHub](https://github.com/galaxyproject/training-material/issues/1339)*

# Upcoming events

These and [other Galaxy related events](/events/) are coming up:

<table class="table table-striped">
<thead>
<tr>
<th>Date</th>
<th>Topic/Event</th>
<th>Venue/Location</th>
<th>Contact</th>
</tr>
</thead>
<tbody>
<tr>
  <td><span class="text-nowrap">May 7th 2019</span></td>
  <td><a href="https://meetings.cshl.edu/abstracts.aspx?meet=GENOME&amp;year=19">Streamlining accessibility and computability of large-scale genomic datasets with the NHGRI Genomic Data Science Analysis, Visualization, and Informatics Lab-Space (AnVIL)</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://meetings.cshl.edu/meetings.aspx?meet=GENOME&amp;year=19">Biology of Genomes, CSHL, Cold Spring Harbor, New York, United States</a></td>
  <td>James Taylor</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 13th 2019</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2019/module17.pdf">Initiation à l’utilisation de Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2019, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Véronique Martin</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 13th 2019</span></td>
  <td><a href="http://sciencegateways.org/bootcamp">Science Gateways Bootcamp</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/>Indianapolis, Indiana, United States</td>
  <td>help@sciencegateways.org.</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2019</span></td>
  <td><a href="https://cnrsformation.cnrs.fr/stage-19019-Analyse-avancee-de-sequences.html?axe=98">Analyse avancée de séquences</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>Bordeaux, France</td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Macha Nikolski</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2019</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2019/module8bis.pdf">Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2019, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Véronique Martin</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 16th 2019</span></td>
  <td><a href="/events/2019-05-gtn">GTN CoFest and community call</a></td>
  <td><img style="float:right;" src="/images/icons/GL.png"/>Online</td>
  <td>Bérénice Batut, Saskia Hiltemann</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 17th 2019</span></td>
  <td><a href="https://southgreenplatform.github.io/trainings//files/formation-plateformeSG-2019.pdf#page=9">Initiation aux analyses de données transcriptomiques</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://southgreenplatform.github.io/trainings/trainings2019/">IRD, Montpellier, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Formateurs</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 23rd 2019</span></td>
  <td><a href="https://southgreenplatform.github.io/trainings//files/formation-plateformeSG-2019.pdf#page=10">Initiation aux analyses de données metabarcoding</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://southgreenplatform.github.io/trainings/trainings2019/">IRD, Montpellier, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Formateurs</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 2nd 2019</span></td>
  <td><a href="/events/2019-asms">Galaxy @ ASMS 2019</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://www.asms.org/conferences/annual-conference/annual-conference-homepage">ASMS 2019, Atlanta, Georgia, United States</a></td>
  <td>Tim Griffin, Pratik Jagtap</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 11th 2019</span></td>
  <td><a href="https://faes.org/biotech82june">Bioinformatics for Beginners</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/>National Institutes of Health (NIH), Bethesda, Maryland, United States</td>
  <td>training@faes.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 12th 2019</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2019/module23.pdf">Traitement bioinformatique et analyse différentielle de données d'expression RNA-seq sous Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2019, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Véronique Martin</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 1st 2019</span></td>
  <td><a href="/events/gcc2019/">2019 Galaxy Community Conference (GCC2019)</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.konzerthaus.freiburg.de/">Freiburg, Germany</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 3rd 2019</span></td>
  <td><a href="http://bioinformatics.org.au/winterschool/program/">An introduction to Galaxy with the NeCTAR Genomics Virtual Laboratory</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="http://bioinformatics.org.au/winterschool/">2019 Winter School in Mathematical &amp; Computational Biology, Brisbane, Queensland, Australia</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Igor Makunin</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 29th 2019</span></td>
  <td><a href="https://qfab.org/training#NGS_Workshops">Getting started with Galaxy and NGS Platforms</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 30th 2019</span></td>
  <td><a href="https://qfab.org/training#NGS_Workshops">Genome assembly using Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 31st 2019</span></td>
  <td><a href="https://qfab.org/training#NGS_Workshops">Variant detection using Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">August 1st 2019</span></td>
  <td><a href="https://qfab.org/training#NGS_Workshops">RNA Seq analysis using Galaxy </a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">August 1st 2019</span></td>
  <td><a href="/events/2019-asmcue">&quot;Teaching undergraduate genomics with Galaxy” and “Resources and recommendations for teaching undergraduate genomics with Galaxy&quot;</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://www.asm.org/Events/ASM-Conference-for-Undergraduate-Educators/Home">ASMCUE 2019, Tyson, Virginia, United States</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Mohammad Heydarian, Anton Nekrutenko</td>
</tr>
<tr>
  <td><span class="text-nowrap">August 2nd 2019</span></td>
  <td><a href="https://qfab.org/training#NGS_Workshops">Metagenomics analysis using Galaxy </a></td>
  <td><img style="float:right;" src="/images/icons/AU.png"/><a href="https://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
</tbody>
</table>

----

# Publications

**265 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in the last month, pushing the total number of pubs past 7500.

## Highlighted Publications

There were 15 *[Galactic](https://www.zotero.org/groups/1732893/galaxy/tags/%2BGalactic)* and *[Stellar](https://www.zotero.org/groups/1732893/galaxy/tags/%2BStellar)* publications added.  Here are the open access ones.

* **[Tackling “Big Data” with Biology Undergrads: A Simple RNA-seq Data Analysis Tutorial Using Galaxy](https://www.coursesource.org/courses/tackling-big-data-with-biology-undergrads-a-simple-rna-seq-data-analysis-tutorial-using)**, Matthew Anthony Escobar, William Morgan, Irina Makarevitch, Sabrina Robertson. *CourseSource*. https://doi.org/10.24918/cs.2019.13
* **[Containerization of Galaxy Workflows increases Reproducibility](https://publikationen.uni-tuebingen.de/xmlui/handle/10900/83810)**, Felix Bartusch, Maximilian Hanussek,  Jens Krüger. Proceedings of the 4th bwHPC Symposium
* **[Desarrollo y despliegue de un workflow para el análisis genómico de *Campylobacter Jejuni*](https://repositorio.uam.es/handle/10486/687147)**, José Antonio Barbero Aparicio, thesis, Universidad Autónoma de Madrid
* **[NetworkAnalyst 3.0: a visual analytics platform for comprehensive gene expression profiling and meta-analysis](https://doi.org/10.1093/nar/gkz240)**, Guangyan Zhou, Othman Soufan, Jessica Ewald, Robert E W Hancock, Niladri Basu, Jianguo Xia. *Nucleic Acids Research*, gkz240, doi: 10.1093/nar/gkz240
* **[Comparative Dissection of Three Giant Genomes: *Allium cepa, Allium sativum*, and *Allium ursinum*](https://doi.org/10.3390/ijms20030733)**, Vratislav Peška, Terezie Mandáková, Veronika Ihradská, Jiří Fajkus. *Int. J. Mol. Sci.* 2019, 20, 733.
* **[A rapid and simple method for assessing and representing genome sequence relatedness](https://www.biorxiv.org/content/10.1101/569640v2)**, *bioRxiv*, M Briand, M Bouzid, G Hunault, M Legeay, M Fischer-Le Saux, M Barret, doi: 10.1101/569640
* **[Identification of genetic relationships and subspecies signatures in *Xylella fastidiosa*](https://doi.org/10.1186/s12864-019-5565-9)**, Nicolas Denancé, Martial Briand, Romain Gaborieau, Sylvain Gaillard, Marie-Agnès Jacques. *BMC Genomics* 201920:239, doi: 10.1186/s12864-019-5565-9
* **[Genotyping by sequencing can reveal the complex mosaic genomes in gene pools resulting from reticulate evolution: a case study in diploid and polyploid citrus](https://doi.org/10.1093/aob/mcz029)**,
Dalel Ahmed, Aurore Comte, Franck Curk, Gilles Costantino, François Luro, Alexis Dereeper, Pierre Mournet, Yann Froelicher, Patrick Ollitrault. * Annals of Botany*, mcz029, doi: 10.1093/aob/mcz029. Published: 29 March 2019
* **[Genomic Variants Among Threatened *Acropora* Corals](https://doi.org/10.1534/g3.119.400125)**, Sheila A. Kitchen, Aakrosh Ratan, Oscar C. Bedoya-Reina, Richard Burhans, Nicole D. Fogarty, Webb Miller, Iliana B. Baums. *G3: Genes, Genomes, Genetics*, Early online March 26, 2019; doi: 10.1534/g3.119.400125

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag |
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 162 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/tags/+Methods) | | 66 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/tags/+UsePublic) | | 36 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/tags/+Workbench) | | 26 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/tags/+RefPublic) |
| 19 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/tags/+UseMain) | | 18 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/tags/+Reproducibility) | | 13 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/tags/+UseLocal) | | 9 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/tags/+IsGalaxy) |
| 8 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/tags/+Cloud) | | 6 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/tags/+Shared) | | 5 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/tags/+HowTo) | | 5 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/tags/+Tools) |
| 4 | [+Other](https://www.zotero.org/groups/1732893/galaxy/tags/+Other) | | 2 | [+Education](https://www.zotero.org/groups/1732893/galaxy/tags/+Education) | | 1 | [+Unknown](https://www.zotero.org/groups/1732893/galaxy/tags/+Unknown) | | | | |

----

# Galaxy Platforms News

The [Galaxy Platform Directory](/use/) lists resources for easily running your analysis on Galaxy, including publicly available servers, cloud services, and containers and VMs that run Galaxy.  A lot was added in March:


## Single Cell Omics Galaxy workbench

[<img class="float-right" style="max-width: 110px" src="/use/singlecell/logo_single_cell.svg" alt="scworkbench" />](/use/singlecell/)

The [Single Cell Omics workbench](https://singlecell.usegalaxy.eu/) is a comprehensive set of analysis tools and consolidated workflows for single cell analysis. The current implementation comprises more than 20 bioinformatics tools dedicated to different research areas of single cell biology.

## NetworkAnalyst

[<img class="float-right" style="max-width: 170px" src="/news/2019-05-galaxy-update/networkanalyst.png" alt="networkanalyst" />](https://www.networkanalyst.ca)

[NetworkAnalyst](https://www.networkanalyst.ca/) is an online analysis environment for transcriptome profiling, network analysis, and meta analysis for gene expression data. NetworkAnalyst has developed and made available a [public Galaxy server](https://galaxy.networkanalyst.ca/) to support raw RNAseq processing for 17 species covering mammals, birds, bacteria, plants and parasites.

## CellOrganizer

[<img class="float-right" style="max-width: 170px" src="/use/cell-organizer/cell-organizer-logo2-250.jpg" alt="cellorg" />](/use/cell-organizer/)

[CellOrganizer](http://www.cellorganizer.org/) creates generative models of the spatial organization of cells from microscope images and automatically provides geometries for spatial simulations of cell processes and behaviors. A [Galaxy](http://galaxy.compbio.cs.cmu.edu:8080/) server provides access to CellOrganizer tools for users who do not have the resources to run the Matlab or Docker versions.

## CIRM-CFBP

[<img class="float-right" style="max-width: 200px" src="/use/cirm-cfbp/cirm-cfbp-banner.png" alt="cirmcfbp" />](/use/cirm-cfbp/)

[International Center for Microbial Resources - Plant Associated Bacteria (CIRM-CFBP)](https://www6.inra.fr/cirm_eng/CFBP-Plant-Associated-Bacteria) is providing web based access to tools ([SkIf](https://sourcesup.renater.fr/wiki/skif/) and [KI-S](https://sourcesup.renater.fr/wiki/ki-s/)) of IRHS-EmerSys lab on the [CIRM-CFBP Galaxy](https://iris.angers.inra.fr/galaxypub-cfbp) instance.

[<img class="float-right" style="max-width: 200px" src="/use/gcac/gcac-landing.png" alt="gcac" />](/use/gcac/)

## Galaxy for Compound Activity Classification

[Galaxy for Compound Activity Classification (GCAC)](https://github.com/LynnLab-JNU/gcac-galaxy-tools)is a one stop solution for data generation, data preparation, feature selection, model building, prediction and retrieval of potential molecules. [GCAC](/use/gcac/) is available as a [public Galaxy server](http://ccbb.jnu.ac.in/gcac), a [VM](https://drive.google.com/file/d/1ZQM5x-9J49rvPy9SoWOrM3Xm1MHZdr1I/view?usp=sharing), and an installable [Galaxy tool suite](https://toolshed.g2.bx.psu.edu/repository?repository_id=351af44ceb587e54) from the toolshed.

## MGS-FAST

<img class="float-right" style="max-width: 200px" src="/use/archive/mgs-fast/mgs-fast-fig2.png" alt="mgsfast" />

[MGS-Fast](https://github.com/BCIL/MGS-Fast) is an analysis approach to rapidly map a metagenomics shotgun sequencing (MGS) data set and assign known functions to reads originating from microbial genes, producing counts of KEGG gene orthologs as output. This rapid annotation method is freely available as a [Galaxy workflow within a Docker image](https://hub.docker.com/r/bcil/metagenome).


## Galaxy Platforms in Publications

We tag papers that use, mention, implement or extend public Galaxy platforms (servers, services, clouds, containers...).  Here are the platforms that were referenced at least twice in the past month's publications:

| # | Tag | | # | Tag | | # | Tag | | # | Tag |
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 19 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/tags/>Huttenhower) | | 11 | [>ARGs-OAP](https://www.zotero.org/groups/1732893/galaxy/tags/>ARGs-OAP) | | 9 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/tags/>RepeatExplorer) | | 7 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/tags/>Workflow4Metabolomics) |
| 5 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/tags/>Galaxy-P) | | 4 | [>Orione](https://www.zotero.org/groups/1732893/galaxy/tags/>Orione) | | 4 | [>UseGalaxy.eu](https://www.zotero.org/groups/1732893/galaxy/tags/>UseGalaxy.eu) | | 4 | [>UseGalaxy.org.au](https://www.zotero.org/groups/1732893/galaxy/tags/>UseGalaxy.org.au) |
| 3 | [>epiGeEC](https://www.zotero.org/groups/1732893/galaxy/tags/>epiGeEC) | | 2 | [>APOSTL](https://www.zotero.org/groups/1732893/galaxy/tags/>APOSTL) | | 2 | [>ARGalaxy](https://www.zotero.org/groups/1732893/galaxy/tags/>ARGalaxy) | | 2 | [>CIRM-CFBP](https://www.zotero.org/groups/1732893/galaxy/tags/>CIRM-CFBP) |
| 2 | [>deepTools](https://www.zotero.org/groups/1732893/galaxy/tags/>deepTools) | | 2 | [>Pasteur](https://www.zotero.org/groups/1732893/galaxy/tags/>Pasteur) | | 2 | [>SymD](https://www.zotero.org/groups/1732893/galaxy/tags/>SymD) | | | | |
----

[<img class="float-right" src="/news/2019-04-gcp/ggcp.png" alt="ggcp" style="max-width: 120px" />](/news/)

# New Galactic Blog Post

This month we have a *[Galactic Blog](/news/)* post where [Enis Afgan](https://github.com/afgane) and [Vahid Jalili](https://github.com/VJalili) describe recent efforts to [Build up support for the Google Cloud Platform in Galaxy](/news/2019-04-gcp/).

----

# Who's Hiring

<div class='float-right'><img src="/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The *dark energy* of irreproducible research is threatening the science universe! Please help the Galaxy push it back!

* [Post-doc position in Computational Biology / Genomics / Transcriptomics / Bioinformatics](/news/2019-04-eth/), ETH Zurich, Switzerland
* [Senior IT Systems Administrator](https://jobs.rothamsted.ac.uk/vacancies/246/senior-it-systems-administrator.html), [Rothamsted Research](http://www.rothamsted.ac.uk/), Harpenden, United Kingdom
* [Experienced Galaxy user / programmer / administrator for consulting work](/news/2019-05-sarepta/), [Sarepta Therapeutics](https://www.sarepta.com/)
* [Genome Stability of Adult Stem Cells](https://science.curie.fr/wp-content/uploads/2017/04/Bardin_Servant-Bioinformatician-ad-2019.pdf), Institut Curie, Dept. of Genetics and Developmental Biology, Paris, France
* [Scientist - Molecular R&D](https://jobs.vibrantm.com/emd/job/Rockville-Scientist-Molecular-R&D-MD-20850-3304/518778201/), MilliporeSigma, Rockville, Maryland, United States
* [Cloud Architect](https://careers-fhcrc.icims.com/jobs/12290/cloud-engineer-v/job?mobile=false&width=960&height=500&bga=true&needsRedirect=false&jan1offset=-480&jun1offset=-420), Fred Hutchinson Cancer Research Center, Seattle, Washington, United States.
* ELIXIR Belgium has **three Galaxy-related openings** in Ghent:
    * [ELIXIR Open Science Community Manager](https://vibvzw.jobsoid.com/j/17164/elixir-open-science-community-manager), VIB-UGent Center for Plant Systems Biology
    * [ELIXIR Software developer data management tools](https://vibvzw.jobsoid.com/jobs/35bba14f-79b7-4388-b015-a4fb9d97ee41/17168/software-developer-data-management-tools), VIB-UGent Center for Plant Systems Biology
    * [ELIXIR Scientific Cloud Coordinator](https://vibvzw.jobsoid.com/j/17165/elixir-scientific-cloud-coordinator), VIB-UGent Center for Plant Systems Biology
* [Software Developer](https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25240&siteid=5341&PageType=JobDetails&jobid=1433370#jobDetails=1433370_5341), Harvard T.H. Chan School of Public Health, Boston, Massachusetts, United States. "Basic automated analysis workflows using Galaxy for 16S marker gene, metagenomic, and metatranscriptomic data leveraging existing software."
* The [The European Galaxy Team has open positions](https://usegalaxy-eu.github.io/posts/2019/01/10/openpositions/), Freiburg, Germany
    * Software engineer, system analysts/administrators, data analyst, and a community and/or research manager

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

---

# Doc, Hub, and Training Updates

Updates from the [Galaxy Training Materials](https://galaxyproject.github.io/training-material/) in April:

## Hub

The [User Authentication and Authorization](/authnz/) docs explain all of Galaxy's features that (a) enable users to login to Galaxy using their social and institutional identities (e.g., using their Google account), and (b) allows them to securely authorize Galaxy to access their private cloud-based resources. Both (a) and (b) are described for [admins (how to configure Galaxy for that feature)](/authnz/config/) and [users (how to use that feature)](/authnz/use/) separately. Many thanks to [Vahid Jalili](https://github.com/VJalili), [Kjell Petersen](https://github.com/kjellp), and [Enis Afgan](https://github.com/afgane) for their contributions.

Also see [Vahid Jalili](https://github.com/VJalili)'s slides on [User Authentication and Cloud Authorization in the Galaxy project](https://www.slideshare.net/jalilivahid/user-authentication-and-cloud-authorization-in-the-galaxy-project)

## GTN Training Materials

[<img class="float-right img-responsive" style="max-width: 15em;" src="/images/galaxy-logos/GTNLogoTrans300.png" alt="Galaxy Training Materials" />](https://galaxyproject.github.io/training-material/)

* New [Mass spectrometry imaging: Finding differential analytes](https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.html), by [Melanie Foell](https://github.com/foellmelanie)
* New [Creating a new tutorial - Writing content in Markdown](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html) tutorial by [Bérénice Batut](https://github.com/bebatut), [Björn Grüning](https://github.com/bgruening), [Saskia Hiltemann](https://github.com/shiltemann), and [Helena Rasche](https://github.com/hexylena).
* New [Name tags for following complex histories](https://training.galaxyproject.org/training-material/topics/galaxy-data-manipulation/tutorials/name-tags/tutorial.html) tutorial by [Helena Rasche](https://github.com/hexylena).
* New [Bibliography](https://github.com/galaxyproject/training-material/pull/1324/files) feature for tutorials by [Saskia Hiltemann](https://github.com/shiltemann) and [Helena Rasche](https://github.com/hexylena).
* New [RNA-Seq reads to counts](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html) tutorial by [Maria Doyle](https://github.com/mblue9).
* New [RNA-seq counts to genes](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-genes/tutorial.html) tutorial by [Maria Doyle](https://github.com/mblue9).
* New [RNA-seq genes to pathways](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-genes-to-pathways/tutorial.html)tutorial by [Maria Doyle](https://github.com/mblue9).
* Update to [Understanding UMIs](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html) tutorial by [Mehmet Tekman](https://github.com/mtekman).

---

# ToolShed Contributions

<div class='right'>[<img src="/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

We didn't get to this this month.  Look forward to a double dose of tools next month.

# Releases

New additions to the Galaxy Ecosystem.

## Galaxy-parsec 1.12.1

[Parsec](https://pypi.org/project/galaxy-parsec/) is a set of automatically generated wrappers for BioBlend functions. I found myself writing a large number of small / one-off scripts that invoked simple bioblend functions. These scripts were impossible to compose and use in a linux-friendly manner. I copied and pasted code between all of these utility scripts.

Parsec is the answer to all of these problems. It extracts all of the individual functions I was writing as separate CLI commands that can be piped together, run in parallel, etc.

## Pulsar 0.9.1

[<img class="float-right" src="/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" />](https://pypi.python.org/pypi/pulsar-app/)

[Pulsar](https://pypi.python.org/pypi/pulsar-app/) is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.


## galaxy-lib 19.5.2

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

----

# Other News

* The [ELIXIR Galaxy Community](https://elixir-europe.org/communities/galaxy) held a training and tools workshop in Roscoff, France. Check out the [meeting summary](https://galaxyproject.eu/posts/2019/04/26/elixir-galaxy-roscoff-ws/).
* [SGCI Consultation Allows CloudLaunch to Benefit Researchers Across a Wide Variety of Fields](https://sciencegateways.org/-/sgci-consultation-allows-cloudlaunch-to-benefit-researchers-across-a-wide-variety-of-fields), by Nayiri Mullinix
* [Galaxy-ML](https://github.com/ohsu-comp-bio/Galaxy-ML): a GitHub repo from [Qiang Gu](https://github.com/qiagu) of [OHSU](https://ohsu.edu/), providing a set of Galaxy tools wrapped with cutting-edge machine learning algorithms.

