---
autotoc: true
---
<div class="title">November 2013 Galaxy Update</div>


<div class='left'><a href='/GalaxyUpdates/'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the November 2013 *[Galaxy Update](/GalaxyUpdates)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/GalaxyUpdates)* complement the *[Galaxy Development News Briefs](/DevNewsBriefs)* which accompany new Galaxy releases and focus on Galaxy code updates.  

<br /><br />

# New Public Servers

Two new servers were added to [the list of publicly accessible Galaxy servers](/PublicGalaxyServers) in October, include one that covers some radically new territory for Galaxy:

## CoSSci: Complex Social Science Gateway

<div class='right'><div class='solid'><a href='http://socscicompute.ss.uci.edu/'><img src='/PublicGalaxyServers/CoSSciMap.png' alt='CoSSci; Women's work map: cycles of lines contain local autocorrelation regions' width="275" /></a></div>
   
*Tools for solving [Galton's problem](http://en.wikipedia.org/wiki/Galton's_problem) in Comparative Research and complex network problems in Social Science.*

The [CoSSci Gateway](http://socscicompute.ss.uci.edu/) is a public Galaxy server portal linked to a [SDSC XSEDE Gateway](https://www.xsede.org/gateways-listing) for R-based analysis of multiple social science databases. CoSSci is hosted by the [Institute for Mathematical Behavioral Science](http://intersci.ss.uci.edu/) at UC Irvine.  The initial six databases are anthropological (cross-cultural, foragers, etc.) and will expand to include cross-national (political, psychological, economic), regional, comparative urban and kinship networks, and solutions to identifying network-based structural cohesion and its consequences. 

The initial modeling software solves for autocorrelation effects (spatial, phylogenetic histories, common language, common environment, larger polities and religion, etc. *Dow-Eff* software performs autocorrelation regression.  *logit* provides multiple imputations of missing data for single or multiple overlapping datasets, maps, and also incorporates Bayesian and graphical approaches to testing for causal inferences and controlling for biases.

A [Visual Manual](http://intersci.ss.uci.edu/wiki/index.php/Visual_Manual) is available

<div class='right'><div class='solid'><a href='http://http://avispa.biociphers.org/'><img src='/PublicGalaxyServers/BioCiphersLandingPage300.png' alt='BioCiphers Galaxy' width="275" /></a></div>

## BioCiphers Lab Galaxy

The [BioCiphers Lab Galaxy](http://avispa.biociphers.org/) provides a user friendly interface for analysis tools developed by the [BioCiphers Lab](http://biociphers.org/) at the University of Pennsylvania.  This includes the AVISPA alternative splicing prediction and analysis tool.  See Barash, *et al.,* "[AVISPA: a web tool for the prediction and analysis of alternative splicing](http://genomebiology.com/2013/14/10/R114)" *[Genome Biology](http://genomebiology.com/)* 2013, 14:R114.  BCL Galaxy offers [email support](/Public Galaxy Servers/#biociphers-lab-galaxy).  It is open to anyone from an educational organization.

<br />
# New Papers

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 29 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>



A record *53 new papers* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in October. Some papers that may be particularly interesting to the Galaxy community:

* Li, *et al.,* "[Expanding roles in a library-based bioinformatics service program: a case study](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3794686/)," *[Journal of the Medical Library Association](http://www.ncbi.nlm.nih.gov/pmc/journals/93/latest/)* : JMLA, Vol. 101, No. 4. (October 2013), pp. 303-309, doi:10.3163/1536-5050.101.4.012
* Nagasaki, *et al.* "[DDBJ Read Annotation Pipeline: A Cloud Computing-Based Pipeline for High-Throughput Analysis of Next-Generation Sequencing Data](http://dx.doi.org/10.1093/dnares/dst017)," *[DNA Research](http://dnaresearch.oxfordjournals.org/)*, Vol. 20, No. 4. (1 August 2013), pp. 383-390, doi:10.1093/dnares/dst017

And ...

## Ten Simple Rules for Reproducible Computational Research

[Geir Kjetil Sandve](http://www.mn.uio.no/ifi/english/people/aca/geirksa/), [Anton Nekrutenko](/anton), [James Taylor](/JamesTaylor), and [Eivind Hovig](http://www.mn.uio.no/ifi/english/people/aca/ehovig/index.html)'s paper "[Ten Simple Rules for Reproducible Computational Research](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285)" was published in *[PLoS Computational Biology](http://www.ploscompbiol.org/)* in October.  

<div class='right'><a href='http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285'><img src='/Images/Logos/PlosCompBioLogo.png' alt='PLoS Computational Biology: Ten Simple Rules for Reproducible Computational Research' width="240" /></a></div>

Those 10 rules are (in our opinion) worth repeating:

1. [For Every Result, Keep Track of How It Was Produced](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s2)

1. [Avoid Manual Data Manipulation Steps](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s3)

1. [Archive the Exact Versions of All External Programs Used](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s4)

1. [Version Control All Custom Scripts](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s5)

1. [Record All Intermediate Results, When Possible in Standardized Formats](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s6)

1. [For Analyses That Include Randomness, Note Underlying Random Seeds](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s7)

1. [Always Store Raw Data behind Plots](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s8)

1. [Generate Hierarchical Analysis Output, Allowing Layers of Increasing Detail to Be Inspected](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s9)

1. [Connect Textual Statements to Underlying Results](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s10)

1. [Provide Public Access to Scripts, Runs, and Results](http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003285#s11)

If you care about reproducible research, then take a look.

<br />

# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring/'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [CDD en bioinformatique : recherche de variants structuraux par reséquençage (NGS)](http:////bit.ly/1h1yQnf)
* [Stage Master 2](http://bit.ly/18Qd2Eg) : Développement et intégrationd’outils pour la bioanalyse dans l’environnement Galaxy 
* [Statistical Genomics Postdoc](/News/StatisticalGenomicsPostDocPennState) opening in the Makova lab at Penn State
* [Computational biology opening](/News/ComputationalBiologyParis) at University Pierre-et-Marie-Curie, Paris
* **The [Galaxy Project is hiring software engineers and post-docs](/GalaxyIsHiring)**.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />

# Events

## GCC2014: June 30 - July 2, Baltimore

<div class='right'><a href='/Events/GCC2014/'><img src='/Images/Logos/GCC2014LogoTall200.png' alt='GCC2014: June 30 - July 2' width="130" /></a></div>

The [2014 Galaxy Community Conference (GCC2014)](/Events/GCC2014) will be held June 30 through July 2, at the [Homewood Campus](http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/) of [Johns Hopkins University](http://jhu.edu), in [Baltimore, Maryland](http://visitors.baltimorecity.gov/), United States.

Galaxy Community Conferences are an opportunity to participate in presentations, discussions, poster sessions, lightning talks and breakouts, all about high-throughput biology and the tools that support it.  The conference will also includes a *Training Day* offering in-depth topic coverage, across several concurrent sessions.  See [the GCC2013 web site](/Events/GCC2013) for an idea of what happens at a Galaxy Community Conference.

## Galaxy Day, December 4, Paris

<div class='left'><a href='http://www.ifb-galaxy.org/4dec2013.html'><img src='/Images/Logos/GalaxyIFB.png' alt='Galaxy Day' height="55" /></a></div>

The [IFB (French Bioinformatic Institute) Galaxy working group](http://www.ifb-galaxy.org/index.html) is organizing [a seminar around the Galaxy platform](http://www.ifb-galaxy.org/4dec2013.html) which will takes place in Paris on December 4 (@Institut Curie - 9h30-17h).

The main aim of [this meeting](http://www.ifb-galaxy.org/4dec2013.html) is to share experiences from laboratories and platforms working daily with Galaxy. Thirteen talks will have been selected to present insights about installations and releases of Galaxy platforms, new usages / domains, and technological developments.

[Registration](http://www.ifb-galaxy.org/formations_formulaire.html) is open and free, but limited to 90 people. 

<div class='center'>
Sponsored and organized by [URGI](http://urgi.versailles.inra.fr/), [GenoToul](http://bioinfo.genotoul.fr/), [MIGALE](http://migale.jouy.inra.fr/), [PFEM](http://metabohub.fr/), [SouthGreen](http://southgreen.cirad.fr/), [Institut Curie](http://curie.fr/), [ABiMS](http://abims.sb-roscoff.fr/)

<a href='http://urgi.versailles.inra.fr/'><img src='/Images/Logos/URGILogo.png' alt='URGI' width="70" /></a>
<a href='http://bioinfo.genotoul.fr/'><img src='/Images/Logos/GenoToulLogo.png' alt='GenoToul' width="100" /></a>
<a href='http://migale.jouy.inra.fr/'><img src='/Images/Logos/MIGALELogo.png' alt='MIGALE' width="120" /></a>
<a href='http://metabohub.fr/'><img src='/Images/Logos/MetaboHubLogo.jpg' alt='PFEM' width="120" /></a>
<a href='http://southgreen.cirad.fr/'><img src='/Images/Logos/SouthGreenLogo.png' alt='SouthGreen' width="120" /></a>
<a href='http://curie.fr/'><img src='/Images/Logos/CurieLogo.gif/' alt='Institut Curie' width="80" /></a>
<a href='http://abims.sb-roscoff.fr/'><img src='/Images/Logos/ABiMSLogo.png' alt='ABiMS' width="70" /></a>
</div>

## UC Davis Bioinformatics Boot Camps

**Registration is now open for [Bioinformatics Bootcamps in December](https://training.bioinformatics.ucdavis.edu/bootcamps/)!**

<div class='left'><a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'><img src='/Images/Logos/UCDavisGenomeCenterLogo.jpg' alt='Introduction to Galaxy Boot Camp @ UC Davis Bioinformatics Core'  /></a><br />
&nbsp;![Galaxy](/Images/Logos/galaxyLogoSymbol.png)</div>

The [next offering of UC Davis Bioinformatics Bootcamps](https://training.bioinformatics.ucdavis.edu/bootcamps/) will be held on the [UC Davis](http://www.ucdavis.edu/) campus December 10-13. 

These focused one-day courses are for researchers looking to get up to speed quickly on the latest technologies and techniques in bioinformatics. Students will work on their own laptops and have continued access to software and example data used in the exercises through a [public Amazon Web Services virtual machine](http://bioinformatics.ucdavis.edu/software/). The first three bootcamps will use the Galaxy platform, and the final bootcamp will use both Galaxy and the command-line. The Alignment and Assembly bootcamps (Dec. 11th & 12th) require you to know Galaxy, so if you are unfamiliar with Galaxy, you should also take the Introduction bootcamp on Dec. 10th.

<div class='right'><a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'><img src='/Images/Logos/UCDavisLogoSmall.gif/' alt='UC Davis'  /></a></div>

   **Tuesday, December 10:**
      **[Introduction to Next-Generation Sequence Analysis with Galaxy](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-next-generation-sequence-analysis-with-galaxy-december-10-2013/)**

   **Wednesday, December 11:**
      **[Next-Generation Sequence Alignment and Variant Discovery](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-next-generation-sequence-alignment-and-variant-discovery-december-11-2013/)**

   **Thursday, December 12:**
      **[Genome Assembly using Next-Generation Sequence Data](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-genome-assembly-using-next-generation-sequence-data-december-12-2013/)**

   **Friday, December 13:**
      **[Introduction to the Amazon Cloud for Galaxy and the Command-Line](http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-the-amazon-cloud-for-galaxy-and-the-command-line-december-13-2013/)**

Enrollment for each bootcamp will be capped at 24 students. Please enroll early to be assured of a seat, as these bootcamps usually fill up quickly! More information, including full descriptions of each bootcamp is [available online](https://training.bioinformatics.ucdavis.edu/bootcamps/). The cost for each bootcamp is $200 (academic/government) or $250 (non-academic/industry).

## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif/' /></a></div> There is a lot going on in the next three months. Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<br />

<div class='center'> 
<a href='http://www.qfab.org/events/upcoming/'><img src='/Images/Logos/QFABLogo.png' alt='QFAB Workshops' height="80" /></a>
<a href='http://bit.ly/16tBdIZ'><img src='/Images/Logos/DDBJLogoSmall.png' alt='NGS解析Galaxy特集とDDBJ' height="60" /></a>
<a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=43'><img src='/Images/Logos/GenoToulLogo.png' alt='Galaxy Training Days' height="60" /></a> &nbsp;
<a href='/Community/GalaxyAdmins/Meetups/2013_11_20/'><img src='/Images/Logos/GalaxyAdmins.png' alt='GalaxyAdmins Meetups' height="55" /></a>
<a href='http://www.ifb-galaxy.org/4dec2013.html'><img src='/Images/Logos/GalaxyIFB.png' alt='Galaxy Day' height="55" /></a>
</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> November 2-6 </th>
    <td> <em><a href='https://apha.confex.com/apha/141am/webprogramadapt/Paper287924.html'>A zebrafish cloud computational resource for environmental health science research</a></em> </td>
    <td> <a href='http://www.apha.org/meetings/AnnualMeeting/'>141st American Public Health Association (APHA) Annual Meeting</a>, Boston, Massachusetts, United States </td>
    <td> <a href='https://apha.confex.com/apha/141am/webprogramadapt/Person216879.html'>Peter Tonellato</a> </td>
  </tr>
  <tr>
    <th> November 4 </th>
    <td> <em><a href='http://www.qfab.org/event/rna-seq-analysis-using-galaxy-workshop/'>RNA-Seq Analysis using Galaxy workshop</a></em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 6 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-an-introduction-to-variant-detection-using-galaxy/'>Workshop: An Introduction to Variant Detection using Galaxy</a></em> </td>
    <td> Children’s Medical Research Institute, Westmead, New South Wales, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 6-8 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=43'>Galaxy Training Days</a></em> <div class='indent'><div class='red'>Workshop is full, but there is space on the waiting list</div> </td>
    <td> <a href='http://bioinfo.genotoul.fr/index.php?id=4#c93'>INRA Auzeville</a>, France </td>
    <td> <a href="mailto:sarah DOT maman AT toulouse AT inra AT fr">Sarah Maman</a> </td>
  </tr>
  <tr>
    <th> November 7 </th>
    <td> <em><a href='http://www.qfab.org/event/rna-seq-analysis-using-galaxy-4/'>RNA-Seq Analysis using Galaxy workshop</a></em> </td>
    <td> Children’s Medical Research Institute, Westmead, New South Wales, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 19 </th>
    <td> <em><a href='http://bit.ly/16tBdIZ'>NGS解析Galaxy特集とDDBJ</a></em> </td>
    <td> 新潟大学 <a href='http://www1.niigata-u.ac.jp/tokimate/outline.html'>駅南キャンパス「ときめいと」</a> Japan </td>
    <td> <a href='http://www.ddbj.nig.ac.jp/addresses-j.html#to-ddbj'>お問い合わせの</a> </td>
  </tr>
  <tr>
    <th> November 20 </th>
    <td> <em><a href='/Community/GalaxyAdmins/Meetups/2013_11_20/'>November GalaxyAdmins Meetup</a></em> </td>
    <td> <a href='/Community/GalaxyAdmins/Meetups/2013_11_20/'>Online</a> </td>
    <td> <a href='/DaveClements/'>Dave Clements</a>, Srinivas Maddhi </td>
  </tr>
  <tr>
    <th> November 22 </th>
    <td> <em><a href='http://www.qfab.org/event/variant-detection-using-galaxy-workshop/'>Variant Detection using Galaxy workshop</a></em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> December 4 </th>
    <td> <em><a href='http://www.ifb-galaxy.org/4dec2013.html'>Galaxy Day</a></em><div class='indent'><div class='red'>Soumissions dû 10 Octobre</div> </td>
    <td> Paris (Institut Curie), France </td>
    <td> <a href="mailto:ifb DOT galaxy AT sb DASH roscoff DOT fr">Organisateurs</a> </td>
  </tr>
  <tr>
    <th> December 9 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-an-introduction-to-de-novo-genome-assembly-using-galaxy/'>Workshop : An Introduction to De Novo Genome Assembly using Galaxy</a></em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> December 10 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-next-generation-sequence-analysis-with-galaxy-december-10-2013/'>Introduction to Next-Generation Sequence Analysis with Galaxy</a></em> </td>
    <td rowspan=4> </strong><a href='https://training.bioinformatics.ucdavis.edu/bootcamps/'>UC Davis Bioinformatics Bootcamps</a><strong>, Davis, California, United States </td>
    <td rowspan=4> <a href="mailto:training DOT ucdbio AT gmail DOT com">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th> December 11 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-next-generation-sequence-alignment-and-variant-discovery-december-11-2013/'>Next-Generation Sequence Alignment and Variant Discovery</a></em> </td>
  </tr>
  <tr>
    <th> December 12 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-genome-assembly-using-next-generation-sequence-data-december-12-2013/'>Genome Assembly using Next-Generation Sequence Data</a></em> </td>
  </tr>
  <tr>
    <th> December 13 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2013/10/16/bootcamp-introduction-to-the-amazon-cloud-for-galaxy-and-the-command-line-december-13-2013/'>Introduction to the Amazon Cloud for Galaxy and the Command-Line</a></em> </td>
  </tr>
  <tr>
    <th colspan=4> <span style="font-size: larger;"> 2014 </span> </th>
  </tr>
  <tr>
    <th> January 11-15 </th>
    <td> <a href='http://www.intlpag.org/'>Plant and Animal Genome XXII (PAG 2014)</a> </td>
    <td> San Diego, California, United States </td>
    <td> <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> January 16-17 </th>
    <td> <a href='http://gmod.org/wiki/Jan_2014_GMOD_Meeting'>2014 GMOD Meeting</a> </td>
    <td> San Diego, California, United States </td>
    <td> <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> May 27 </th>
    <td> <a href='http://www.mipro.hr/LinkClick.aspx?fileticket=y1%2bUY%2b54jB4%3d&tabid=110&mid=461&language=en-US&forcedownload=true'>Enabling Data Analysis with Galaxy CloudMan workshop</a> </td>
    <td> <a href='http://www.mipro.hr/CallForPapers/tabid/176/language/en-US/Default.aspx'>MIPRO 2014</a>, Opatija, Croatia </td>
    <td> <a href='/EnisAfgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 2 </th>
    <td> </strong><a href='/Events/GCC2014/'>2014 Galaxy Community Conference (GCC2014)</a><strong> </td>
    <td> <a href='http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/'>Homewood Campus</a> of <a href='http://jhu.edu'>Johns Hopkins University</a>, <a href='http://visitors.baltimorecity.gov/'>Baltimore, Maryland</a>, United States </td>
    <td> <a href='/Events/GCC2014/Organizers/'>Organizers</a> </td>
  </tr>
</table>

<br />

# Lifeportal at the University of Oslo

<div class='right'><a href='https://lifeportal.uio.no/'><img src='/Images/Logos/LifeportalLeaf.jpg' alt='Lifeportal' /></a></div>

The University of Oslo (UiO), the hosts of this year's [Galaxy Community Conference](/Events/GCC2013), launched [Lifeportal](https://lifeportal.uio.no/) in October.  While Lifeportal is *not* a [public Galaxy server](/PublicGalaxyServers), it is [available to all research institutions in Norway and their collaborators abroad](http://www.uio.no/english/services/it/research/hpc/lifeportal/start-using/).

From the [Lifeportal website](https://lifeportal.uio.no/):
 Lifeportal gives you easy access to the High Performance Computing cluster Abel at the University of Oslo. The Galaxy based Lifeportal has a continuously growing list of services, and among them the most widely used tools from the former Bioportal.

The [official opening event](http://www.uio.no/english/services/it/research/hpc/lifeportal/news/opening-of-the-lifeportal.html) was on October 9 and included talks and demonstrations by Nils Christophersen, Rector Ole Petter Ottersen, Nikolay Vazov, Katerina Michalickova, Sveinung Gundersen, and Geir Kjetil Sandve, all of whom spoke and/or presented their work at [GCC2013](/Events/GCC2013/Program). Rector Ole Petter Ottersen commented that "The slogan of the university is 'reaching for the stars.' Great to open an entire galaxy today."

See [this article](http://bit.ly/GDSjK4) for more on Lifeportal and why UIO chose Galaxy.

# Galaxy Distributions

The most recent [Galaxy distribution was August 12](/DevNewsBriefs/2013_08_12).

A new version of [/CloudMan](/CloudMan) was [released in July](/News/CloudManRelease).

<br />

# Tool Shed Contributions

<div class='right'><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy Tool Shed' width=150 /></a></div>

There were many...

* [spp_tool](http://bit.ly/19W9VKq): Cross-correlation analysis package (SPP) 
* [micomplete](http://bit.ly/1b3zQQG): Completeness report for (single-cell amplified) genomes 
* [BLAT](http://bit.ly/1b3zARw) a very fast sequence alignment tool similar to BLAST  
* [sequel](http://bit.ly/GWziCQ): correct errors (i.e., insertions, deletions, substitutions) in contigs output from assembly 
* [gatk_2_7](http://bit.ly/1cK0P5I) 
* [macs2](http://bit.ly/1gXjSMI): Model-based Analysis of ChIP-Seq (macs2) 
* [sample](http://bit.ly/1ejVjay): sample records from the input file(s). Supports paired data if paired files are in sync. 
* [interproscan5](http://bit.ly/1cLUTvJ): functional annotations/predictions 
* [cuffmerge](http://bit.ly/Gzzqsc), [cuffcompare](http://bit.ly/17oDXYf), [cuffdiff](http://bit.ly/19V3Eyh), and [cufflinks](http://bit.ly/GzJSj4)
* [muscle](http://bit.ly/1aJ0fVO): multiple alignment tool 
* [CLC Assembly Cell (CLCbio)](http://bit.ly/1aTY66l) (under development and looking for comments)
