---
title: 'James Peter Taylor | 1979 - 2020'
---

<div class="text-right">

*Good ideas don’t have owners—they belong to everyone (@jxtx)*

</div>

<br />

---

<br />


James Taylor started his professional path at the University of Vermont, where he received a BS in Computer Science in 2000. In 2003, after working as a software engineer in the private sector, he found that his real purpose in life was elsewhere. That year he began his graduate studies in Computer Science at Penn State and joined the nascent Center for Comparative Genomics and Bioinformatics. He used to refer to the Center as ‘BX’—a contraction of BioinformatiX. The name stuck; ‘BX’ is still a part of many URLs (e.g. the original Galaxy URL at http://g2.bx.psu.edu) and is the origin for the name of the [bx-python software package](https://github.com/bxlab/bx-python). The interdisciplinary faculty of the Center included Francesca Chiaromonte, Ross Hardison, Kateryna Makova, Webb Miller, and Anton Nekrutenko.  This group became the core of James’ academic family, a family that has now lost its most brilliant and prolific child.

<figure class="figure figure-img img-fluid rounded">

[![James Taylor at the Galaxy Community Conference. July 8, 2019. Freiburg, Germany. Photo by Bérénice Batut](/jxtx/james-at-gcc2019.jpg)](/jxtx/james-at-gcc2019.jpg)

<figcaption class="figure-caption">James Taylor at the Galaxy Community Conference. July 8, 2019. Freiburg, Germany. Photo by Bérénice Batut.</figcaption>
</figure>

James concluded his PhD work under the supervision of Webb Miller and Francesca Chiaromointe in just three years—during which he blossomed as a methods developer and as a scientist. He published extensively on comparative genome analysis, gene regulation and molecular evolution—a total of eleven papers<sup>1–11</sup> which were hugely instrumental in advancing the research of the Center.  Ross Hardison, former head of the Center, remembers:

<blockquote class="blockquote">
I may have been on James’ thesis committee, but I learned far more from him than he did from me. In those days, the major data sources for functional genomics were alignments of genome sequences from different species—a specialty of Webb Miller. James was an early developer of machine learning methods to find signals in those multi-species alignments that were predictive of gene regulatory regions. He realized that those methods required substantial dimensional reduction—a specialty of Francesca Chiaromonte—to be effective. He had an amazing command of the statistical, computational, and biological frameworks in which he was working. With his high energy and creativity, he generated effective, publicly released predictions of regulatory elements by 2006. An enduring highlight of my career was my experimental lab testing many of those predictions, and finding that an impressive portion of them did affect levels of gene expression in transfected cells. James and I continued to work together on many projects, always with the goal of finding out what is important in our genomes. It is hard to think of continuing on without him.
</blockquote>

Midway through his graduate studies James became entangled in a messy project that was given the name “GalaXY”. This name was coined by Robert Harris, a fellow graduate student and LASTZ<sup>12</sup> developer, by adding ‘X’ and ‘Y’ to the name of Galaxy’s predecessor, GALA<sup>13</sup>. GALA was the brainchild of Ross Hardison and Webb Miller, who wanted to link genome alignments and annotations and provide tools for operating on those data. They convinced James and Anton Nekrutenko to take it on as a potentially interesting side project. It is hard to pinpoint the exact time when James and Anton started working on Galaxy together, but most likely it was in the middle of a long night of random pub traversals in Glasgow, Scotland, during the 2004 annual meeting of the International Society for Computational Biology (ISCB).

After graduating from Penn State, James gained post-doctoral experience as a visiting member of the Courant Institute of Mathematical Sciences at the New York University (2006 - 2008). During these two years, and ever since, Galaxy became one of his principal projects. Its now iconic features such as the three-pane interface, the “noodly” workflow editor, and the dynamic genome browser Trackster, were built by James while at NYU.

James’ move to Emory University as an assistant professor in 2008 coincided with the rapid emergence of cloud computing, and it became obvious that the future of Galaxy was cloud-dependent. Together with his new team—Enis Afgan, Dannon Baker, Kanwei Li, and Jeremy Goecks—James quickly adapted Galaxy to cloud infrastructure, making it the first comprehensive data analysis resource in Life Sciences to cross this bridge<sup>14,15</sup>.

From the very beginning, James’ vision for the future of Galaxy was about creating and supporting a community of developers and users. This took shape with the Galaxy Developer Conference held at Cold Spring Harbor Laboratory in the summer of 2010, which has since become a yearly event that draws more than 200 attendees. He believed that to be successful and, most importantly, useful, a project must be open and cannot be owned by a single lab or a PI<sup>16</sup>. He was firmly dedicated to this idea. Today Galaxy is not associated with a particular group or an institution. It is a community effort supported by hundreds of developers worldwide. This, perhaps more than anything else, ensures that James’ legacy will endure.

James once said that he would change his middle name to Reproducibility if it would help move the cause forward. He was an advocate for reproducible science long before it became fashionable, and he made it a central tenet of Galaxy.  He could be downright evangelical about it. Martin Morgan of Bioconductor, a collaborator on the AnVIL Project, [shared](https://support.bioconductor.org/p/129652/):

<blockquote class="blockquote">
We in the Bioconductor community knew James as a Galaxy project leader, a strong advocate for open and reproducible science, and an enthusiastic and inspirational colleague. James recognized Bioconductor as a kindred spirit, and bridged the relationship between Galaxy and Bioconductor through his leadership of the NHGRI AnVIL project. James' leadership has had deep consequences for the way Bioconductor now navigates toward large-scale cloud-based computing.
</blockquote>

James was particularly concerned with the inadequate state of quantitative education in life sciences. He developed many of the early training materials that became the foundation of the Galaxy Training Network<sup>17</sup>. He continued to make important contributions to biomedical education after he joined the Biology Department at Johns Hopkins University (JHU) in 2014. At JHU he participated in the development of massive open online courses (MOOCs), and he taught data analysis and scientific computing to biologists.

Vince Hilser, chair of the Johns Hopkins Biology department, [describes](https://bio.jhu.edu/2020/04/03/in-memoriam-professor-james-taylor/) James as a bedrock of the department:

<blockquote class="blockquote">
He came in 2014, and it was transformational. He was this catalyst for change, with a huge positive impact. His presence in the department opened up many areas for research, as he was able to help other faculty members uncover new insights by revealing similarities between the proteins they were studying and those in other organisms.
</blockquote>

People who worked for James thought the world of him, and many people in his group followed him from Emory to Johns Hopkins, and are still in his group today. When a new employee asked him and Anton whether it was "better to ask for permission or forgiveness" James's response was "Don't let us be a bottleneck." That was typical—always thinking of a third option where everyone else only saw two.  Enis Afgan, [in his memoriam](/jxtx/stories/#a-man-who-wanted-to-live-forever-james-taylor), describes James:

<blockquote class="blockquote">
He ran his lab not as a boss but as a friend and a colleague that inspired each individual to do their best work. He was an enabler. He was someone you tried your best not to disappoint.
</blockquote>

While we celebrate James’ transformative vision of computational biology and bioinformatics, he was also just a lot of fun to be with. He had a finely tuned palate for good beer, he was an energetic and inquisitive conversationalist, and he had an uncanny ability to stay up late talking about science, politics, and all things related, but still be at his best the next morning.
James met his future wife, Meredith Greif, when they were graduate students at Penn State. They were married in 2009. They have pursued their careers in a mutually supportive manner, him holding a faculty appointment at Emory University, and Meredith at Georgia State University. Both then moved to Johns Hopkins University. We extend our deepest sympathy to her on this shattering loss.

This memoriam can only hint at how remarkable James was as a human being. We could not capture who he was even if we were given volumes to fill. Comments have come in ([and continue to come in](/jxtx/we-miss-you/) from all over the world, and they touch on both his humanity and his intellect. Melissa Wilson, an associate professor at Arizona State and a fellow PhD student, captures this perfectly:

<blockquote class="blockquote">
He is leaving a hole that cannot be filled. I met James when I started grad school at Penn State. I had no idea what to expect. We were both students, but he was clearly in a league of his own. James was one of the smartest people I've ever known, in a way that wasn't (as) intimidating. Talking with James was easy and enlightening. He advocated and put in the work for better science, in all aspects. James supported my science and my career in ways I don't think I can ever fully communicate. As a colleague, he believed in me. As a friend, he expected me to do good. He supported my growth, selflessly. Most of all, James was an advocate. He advocated for his students, for trainees, for open science, for open data, for making this world a better place.
</blockquote>

James hated self-promotion. For him, the biggest human flaw was baseless arrogance. He believed that everything should be based on merit, which for him meant top-notch science, technical perfection, and a healthy dose of idealism. After all, he was the brightest, most unapologetic, and kindest idealist that the Galaxy project, his students, and his many colleagues and friends will ever know. This post from Karen Reddy, a colleague at Johns Hopkins, sums it all up:

<blockquote class="blockquote">
James was, simply put, an amazing person. He was certainly an amazing scientist, but so much more than that. He was what we all hope to be as scientists, mentors, colleagues, and friends. We will miss you James.
</blockquote>

Enis Afgan, Dannon Baker, Francesca Chiaromonte, Dave Clements, Nate Coraor, Jeremy Goecks, Ross Hardison, Anton Nekrutenko and all of Galaxy Team and Galaxy Community.

April 5, 2020

1. Gibbs, R. A. et al. Genome sequence of the Brown Norway rat yields insights into mammalian evolution. Nature 428, 493–521 (2004).
2. Kolbe, D. et al. Regulatory potential scores from genome-wide three-way alignments of human, mouse, and rat. Genome Res. 14, 700–707 (2004).
3. Hillier, L. W. et al. Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution. Nature 423, (2014).
4. Taylor, J. Clues to function in gene deserts. Trends Biotechnol. 23, 269–271 (2005).
5. King, D. C. et al. Evaluation of regulatory potential and conservation scores for detecting cis-regulatory modules in aligned mammalian genome sequences. Genome Res. 15, 1051–1060 (2005).
6. Chimpanzee Sequencing and Analysis Consortium. Initial sequence of the chimpanzee genome and comparison with the human genome. Nature 437, 69–87 (2005).
7. Hillier, L. W. et al. Generation and annotation of the DNA sequences of human chromosomes 2 and 4. Nature 434, 724–731 (2005).
8. Giardine, B. et al. Galaxy: A platform for interactive large-scale genome analysis. Genome Res. 15, (2005).
9. Wang, H. et al. Experimental validation of predicted mammalian erythroid cis-regulatory modules. Genome Res. 16, 1480–1492 (2006).
10. Taylor, J., Tyekucheva, S., Zody, M., Chiaromonte, F. & Makova, K. D. Strong and weak male mutation bias at different sites in the primate genomes: insights from the human-chimpanzee comparison. Mol. Biol. Evol. 23, 565–573 (2006).
11. Taylor, J. et al. ESPERR: learning strong and weak signals in genomic sequence alignments to identify functional elements. Genome Res. 16, 1596–1604 (2006).
12. Harris, B. lastz. (Github).
13. Giardine, B. et al. GALA, a database for genomic sequence alignments and annotations. Genome Res. 13, 732–741 (2003).
14. Afgan, E. et al. Galaxy CloudMan: delivering cloud compute clusters. BMC Bioinformatics 11 Suppl 1, S4 (2010).
15. Afgan, E. et al. Harnessing cloud computing with Galaxy Cloud. Nat. Biotechnol. 29, 972–974 (2011).
16. Grüning, B. et al. Practical Computational Reproducibility in the Life Sciences. Cell Syst 6, 631–635 (2018).
17. Batut, B. et al. Community-Driven Data Analysis Training for Biology. Cell Syst 6, 752–758.e1 (2018).

