---
title: March 2018 Galaxy News
---
[<img class="float-left" src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 />](/src/galaxy-updates/index.md)

Welcome to the March 2018 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# GCCBOSC 2018
[<img class="float-right" src="/src/images/logos/gcc-bosc-2018-logo-boxed-300.png" width="160" alt="GCCBOSC 2018" />](https://gccbosc2018.sched.com/)

## Abstract submission, registration & housing are open

We are pleased to announce that abstract submission and early registration for [GCCBOSC2018](https://gccbosc2018.sched.com/) are now open.

* **[Submit an abstract](https://easychair.org/conferences/?conf=gccbosc2018) (oral presentation abstracts due March 16)**
* [Register](https://gccbosc2018.eventbrite.com)
* [Book conference housing](https://gccbosc2018housing.eventbrite.com)

[GCCBOSC 2018](https://gccbosc2018.sched.com/) will be held 25-30 June in Portland, Oregon, United States. This brings the 2018 Galaxy Community Conference and the Bioinformatics Open Source Conference together into a unified week-long event. If you work in open source life science or data-intensive biomedical research, then there is no better place than GCCBOSC 2018 to present your work and to learn from others.

[GCCBOSC](https://gccbosc2018.sched.com/) starts with [two days of training](https://gccbosc2018.sched.com/overview/type/C.x+Training+-+All) with a wide range of topics nominated and selected by our communities.  Training is followed by a two day meeting, with joint and parallel tracks, featuring oral presentations, posters, demos, lightning talks, birds-of-a-feather and [invited keynotes](/src/events/gccbosc2018/keynotes/index.md).  The week finishes with *[CollaborationFest](/src/events/gccbosc2018/collaboration/index.md) Core* and *Encore*, two or four days of collaborative work on code, documentation, training and challenging data analysis problems.

## Training schedule set

A big thank you to everyone who nominated topics, voted on topics, helped create workshop descriptions, and to the instructors who will be presenting in June.  All your efforts and input have helped us create [two full days of training at GCCBOSC](https://gccbosc2018.sched.com/overview/type/C.x+Training+-+All).

The first day has two tracks about Galaxy, each with three sessions:

* Intro to Using Galaxy
* Intro to Galaxy Administration

The second day has five parallel track, and covers a full spectrum of topics of interest to those working in and with open source bioinformatics:

* [Setting up for success: Everything you need to know when planning for an RNA-seq analysis (2 sessions)](http://sched.co/Doda)
* [GATK4: What's new and how to run it](http://sched.co/DnB1) 
* [Data Carpentry Genomics Workshop: Data Organization and Automation with Shell (3 sessions)](http://sched.co/Dn8R) 
* [Handling integrated biological data using Python (or R) and InterMine](http://sched.co/Dn4G)
* [Bioinformatics Training and Education with the Galaxy Training Network](http://sched.co/Drp9) 
* [Practical use of the Galaxy API command line tools](http://sched.co/Dn9n) 
* [Command line workflow management systems: Snakemake and Nextflow](http://sched.co/DnAi)
*  [WDL, the Workflow Description Language](http://sched.co/DnAs)
* [Community built analyses that run everywhere with bcbio](http://sched.co/Dn3v) 
* [Introduction to Common Workflow Language](http://sched.co/Dn9R) 
* [Conda and Containers](http://sched.co/Dn3G) 
* [Galaxy Interactive Environments](http://sched.co/DnAc)
* [Galaxy Architecture](http://sched.co/DnB7) 

## Keynote speakers

We are pleased to announce two of our keynote speakers:

[<img class="float-right" src="/src/events/gccbosc2018/keynotes/perez.png" alt="Fernando Pérez"  width="110"  />](https://bids.berkeley.edu/people/fernando-perez)

**BOSC / GCC Joint Keynote: [Fernando Pérez](/src/events/gccbosc2018/keynotes/index.md#bosc-gcc-joint-keynote-fernando-perez)**

* *[Berkeley Institute for Data Science](https://bids.berkeley.edu/), [University of California Berkeley](https://berkeley.edu/)*
* *[Project Jupyter](http://jupyter.org/)*

[Dr. Pérez](https://bids.berkeley.edu/people/fernando-perez) is an assistant professor in Statistics at UC Berkeley and a Faculty Scientist in the Department of Data Science and Technology at Lawrence Berkeley National Laboratory.  He also created IPython while a graduate student and co-founded its successor, [Project Jupyter](http://jupyter.org/).

[<img class="float-right" src="/src/events/gccbosc2018/keynotes/teal.jpg" alt="Tracy Teal" width="110" />](http://www.datacarpentry.org/people/)

**BOSC 2018 Keynote: [Tracy K. Teal](http://www.datacarpentry.org/people/)**

* *The Carpentries*
* *[Journal for Open Source Software](http://joss.theoj.org/)*

[Dr. Teal](http://www.datacarpentry.org/people/) is the Executive Director of The Carpentries.  She co-founded [Data Carpentry](http://www.datacarpentry.org/) to develop and scale this training, and Data Carpentry has now joined with [Software Carpentry](https://software-carpentry.org/) to form The Carpentries and more broadly meet training and community needs.


## GCCBOSC 2018 Sponsors

We are happy to have several confirmed [GCCBOSC 2018 sponsors](/src/events/gccbosc2018/sponsors/index.html). This month we are highlighting *F1000Research* and the *Technology Association of Oregon*:

### F1000Research

[<img class="float-right" src="/src/images/logos/F1000research-logo.jpg" alt="F1000Research" width="200" />](https://f1000research.com/)

For the first time, the *Galaxy Community Conference* and *Bioinformatics Open Source Conference* are joining together for a unified event. To mark this occasion, the [BOSC Collection](https://f1000research.com/collections/BOSC) and the [Galaxy Gateway](https://f1000research.com/gateways/galaxy) on [F1000Research](https://f1000research.com/) will be publishing a series of articles in the run up to the conference in June and are now inviting contributions.  The submission deadline is April 30<sup>th</sup>, to ensure papers are published before GCCBOSC 2018 starts. If you are interested in contributing or have any other questions, please email [research@f1000.com](mailto:research@f1000.com).


### Technology Association of Oregon

[<img class="float-right" src="/src/events/gccbosc2018/sponsors/tao-logo.png" alt-"Technology Association of Oregon" width="200" />](http://www.techoregon.org/)

[Technology Association of Oregon](http://www.techoregon.org/) is the force behind the Oregon tech industry. Supporting entrepreneurs, connecting peers and decision makers, and helping grow the Northwest as a global hub for innovation. We are proud to be a leader in the regional technology community and offer many ways for individuals, companies, and community organizations to partner with us in Oregon and SW Washington.


<div class="float-right">
<br />
[<img src="/src/events/gccbosc2018/sponsors/summary-thumb.png" alt="Download the one page summary." />](https://depot.galaxyproject.org/hub/attachments/documents/gccbosc2018_sponsorship_summary_20170906.pdf) [<img src="/src/events/gccbosc2018/sponsors/prospectus-thumb.png" alt="Download the full prospectus." />](https://depot.galaxyproject.org/hub/attachments/documents/gccbosc2018_sponsorship_prospectus_20170906.pdf)  </div>




### Call for sponsors!

[Sponsors](/src/events/gccbosc2018/sponsors/index.html) are a key part of [GCCBOSC 2018](https://gccbosc2018.sched.com/).  Is your organization interested in playing a prominent role in the first joint gathering of the Galaxy and BOSC communities? Then [become a GCCBOSC 2018 sponsor](/src/events/gccbosc2018/sponsors/index.md#interested) and raise your organization's visibility in these active and engaged communities.


# ELIXIR Galaxy Community Kickoff and Meeting, March, Freiburg

<div class="float-left">
[<img src="/src/images/logos/ElixirNoTextLogo.png" alt="ELIXIR Galaxy Community Kickoff and Meeting" width="160" />](https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference) <br />
[<img src="/src/images/logos/embl-australia-logo.png" alt="EMBL-Australia" width="120" />](http://www.emblaustralia.org/)
</div>

The ELIXIR Galaxy community will have a [kickoff meeting](https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference) 14-16th March 2018, in Freiburg/Germany. We will combine this with a Galaxy User Conference and the **official launch of the usegalaxy.eu server.**

This event will also be coordinated with the [EMBL Australia](http://www.emblaustralia.org/) to also **launch usegalaxy.org.au at the same time.**

Wednesday (the 14th) will be dedicated to the ELIXIR Galaxy community, to discuss and plan our roadmap in 2018 and 2019. Thursday and Friday (15th, 16th) will be a conference where Galaxy users talk about their research and use-cases. However, a few spots we will reserved for official talks from ELIXIR.

*[Register now](https://docs.google.com/forms/d/e/1FAIpQLSfqKqyPwW-s6m64ObleUSv4B8RbGxCRIddBY1cpt3Za7af6XQ/viewform) as space is limited.*

# Galaxy Africa, 3-5 April, Cape Town

[<img class="float-right" src="/src/events/2018-04-galaxy-africa/galaxy-africa-logo.svg" alt="Galaxy Africa" width="170" />](http://galaxyafrica.sanbi.ac.za/)

[Galaxy Africa](http://galaxyafrica.sanbi.ac.za/) will be held 3-5 April in Cape Town, South Africa at [SANBI](http://www.sanbi.ac.za/) on the [Universtiy of the Western Cape](https://www.uwc.ac.za/Pages/default.aspx) campus.

The inaugural Galaxy Africa meeting represents the start of building a data intensive analytics network in Africa. This annual meeting is intended be the networking space for bioinformatics and system administrators to strengthen tool development and research.

**[Register now](https://www.quicket.co.za/events/39729-galaxy-africa-2018/)**.

## Conference Fellowships

If you would like to attend, but need financial assistance to get there, then you are encouraged to [apply for a conference fellowship](http://bit.ly/gxyafrica2018-apply).  Fellowship application review will start 10 March.  Fellowships are provided by the Galaxy Community Fund.

# Upcoming events

These and [other Galaxy related events](/src/events/index.md)  coming up in the next few months:

<table class="table table-striped"><thead>
<tr><th>Date</th><th>Topic/Event</th><th>Venue/Location</th><th>Contact</th></tr></thead><tbody>
<tr>
  <td><span class="text-nowrap">March 7th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/469-introduction-to-rna-seq-data-analysis-with-galaxy">Introduction to RNA-Seq data analysis with Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Am Botanischen Garten, Kiel, Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Andrea Bagnacani, Markus Wolfien</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 14th 2018</span></td>
  <td><a href="https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference">Galaxy community kickoff meeting and Galaxy User Conference</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="https://www.openstreetmap.org/?mlat=48.00189&amp;mlon=7.84628#map=17/48.00189/7.84628">Freiburg, Germany</a></td>
  <td>ELIXIR Galaxy Working Group</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/rna-seq-dge-analysis-galaxy-gvl/#more-1720">RNA-Seq Differential Gene Expression Analysis using Galaxy and the GVL</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/intro-galaxy-gvl/">Introduction to Galaxy and the GVL</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/Alignment_2018.pdf">Alignment, Visualisation and Variant Calling</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/variant-calling-galaxy-gvl/#more-1012">Variant Calling with Galaxy &amp; the GVL for beginners</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 3rd 2018</span></td>
  <td><a href="http://galaxyafrica.sanbi.ac.za/">Galaxy Africa</a></td>
  <td><img style="float:right" src="/images/icons/AF.png"/><a href="http://www.sanbi.ac.za/">SANBI, University of the Western Cape, Cape Town, South Africa</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>galaxyafrica@sanbi.ac.za</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 10th 2018</span></td>
  <td><a href="http://formationcontinue.agrocampus-ouest.fr/infoglueDeliverLive/toutes-sessions/programme?idModule=733&amp;nomModule=Galaxy--Traitement-de-donnees-de-sequences-par-Galaxy-module-7/8">Galaxy : Traitement de données de séquences par Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://formationcontinue.agrocampus-ouest.fr/infoglueDeliverLive/zoom?contentId=12699">AGROCAMPUS OUEST, Campus de Rennes, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Yvan Le Bras, Sandrine LAGARRIGUE, formco@agrocampus-ouest.fr</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 22nd 2018</span></td>
  <td><a href="http://conf.abrf.org/sw2-galaxy-platform-accessible-core-laboratory-solution-proteogenomic-analysis-and-informatics">The Galaxy Platform as an Accessible, Core Laboratory Solution for Proteogenomic Analysis and Informatics</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="http://conf.abrf.org/">ABRF 2018, Myrtle Beach, South Carolina, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Pratik Jagtap</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 23rd 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/513-4th-de-nbi-genomics-training-course">4th de.NBI Genomics training course</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.uni-giessen.de/index.html">Justus-Liebig-Universität, Gießen, Freiburg Germany</a></td>
  <td>denbi_genomics @ computational.bio</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2018</span></td>
  <td><a href="https://wwz.ifremer.fr/Recherche-Technologie/Departements-scientifiques/Departement-Infrastructures-de-Recherche-et-Systemes-d-Information/Activites/Bioinformatique/Formation-metabarcoding-2018">Formation metabarcoding 2018</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.sb-roscoff.fr/">Station Biologique de Roscoff, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Intervenants</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2018</span></td>
  <td><a href="https://sciencegateways.org/engage/bootcamp">Science Gateways Bootcamp</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.tacc.utexas.edu/">TACC, Austin, Texas, United States</a></td>
  <td>help@sciencegateways.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors">Big Genomics Data Skills Training course</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.jax.org/about-us/locations/farmington">JAX Genomic Medicine, Farmington, Connecticut, United States</a></td>
  <td>Charlie Wray</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2018</span></td>
  <td><a href="http://cnrsformation.cnrs.fr/stage-18239-Analyse-avancee-de-sequences.html?axe=65">Bioinformatique - Analyse avancée de séquences</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.labri.fr/">Bordeaux, France</a></td>
  <td>Intervenants</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 12th 2018</span></td>
  <td><a href="http://gonramp.wustl.edu/?page_id=724">G-OnRamp Workshop</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 25th 2018</span></td>
  <td><a href="https://gccbosc2018.sched.com">GCCBOSC 2018</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="/events/gccbosc2018/venue/">Reed College, Portland, Oregon, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 16th 2018</span></td>
  <td><a href="http://gonramp.wustl.edu/?page_id=724">G-OnRamp Workshop</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
</tbody></table>


----

# New Galactic Blog Entry: Coding in the Winter Wonderland: Galaxy Admin Training

<div class="float-right" >
[<img src="/src/images/galaxy-logos/galactic-blog-400.png" alt="Galactic Blog" width="200" />](/src/blog/index.md)
</div>

February had one new *[Galactic Blog](/src/blog/index.md)* entry:

* [Coding in the Winter Wonderland: Galaxy Admin Training in Oslo, 2018](https://arundecano.wordpress.com/2018/02/01/coding-in-the-winter-wonderland-galaxy-admin-training-in-oslo-2018/), by Arun Decano

# Publications

**133 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in February.

## Highlighted Publications

The *Galactic* and *Stellar* publications added in February were:

* [GeneSeqToFamily: a Galaxy workflow to find gene families based on the Ensembl Compara GeneTrees](https://doi.org/10.1093/gigascience/giy005), Anil S Thanki, Nicola Soranzo, Wilfried Haerty, Robert P Davey. *GigaScience*, giy005, doi: 10.1093/gigascience/giy005
* [Disseminating Metaproteomic Informatics Capabilities and Knowledge Using the Galaxy-P Framework](https://doi.org/10.3390/proteomes6010007), Clemens Blank, Caleb Easterly, [Bjoern Gruening](/src/people/bjoern-gruening/index.md), James Johnson, Carolin A. Kolmeder, Praveen Kumar, Damon May, Subina Mehta, Bart Mesuere, Zachary Brown, Joshua E. Elias, W. Judson Hervey, Thomas McGowan, Thilo Muth, Brook L. Nunn, Joel Rudney, Alessandro Tanca, Timothy J. Griffin and Pratik D. Jagtap. *Proteomes* 2018, 6(1), 7; doi:10.3390/proteomes6010007
* [Mind your gaps: Overlooking assembly gaps confounds statistical testing in genome analysis](https://doi.org/10.1101/252973), Diana Domanska, Chakravarthi Kanduri, Boris Simovski, Geir Kjetil Sandve. *bioRxiv* 252973; doi: 10.1101/252973
* [MAJIQ-SPEL: web-tool to interrogate classical and complex splicing variations from RNA-Seq data](https://doi.org/10.1093/bioinformatics/btx565), Christopher J Green, Matthew R Gazzara, Yoseph Barash. *Bioinformatics*, Volume 34, Issue 2, 15 January 2018, Pages 300–302, doi: 10.1093/bioinformatics/btx565
* [Low-pass single-chromosome sequencing of human small supernumerary marker chromosomes (sSMCs) and *Apodemus* B chromosomes](https://link.springer.com/article/10.1007/s00412-018-0662-0), Makunin, A.I., Rajičić, M., Karamysheva, T.V. et al. *Chromosoma* (2018). doi: 10.1007/s00412-018-0662-0

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 93 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Methods) | | 37 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UsePublic) | | 21 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Workbench) | | 20 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseMain) | 
| 12 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+RefPublic) | | 12 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseLocal) | | 4 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Shared) | | 4 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Tools) | 
| 4 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Reproducibility) | | 2 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+HowTo) | | 2 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Cloud) | | 2 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+IsGalaxy) | 
| 1 | [+Visualization](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Visualization) | | 1 | [+UseCloud](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseCloud) | | | | | | | |

----
# Who's Hiring

<div class='right'><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Software Engineer - 8 months contract](https://www.sfbi.fr/content/software-engineer-8-months-contract-institut-pasteur), Institut Pasteur, Paris, France
* [Développement d'outils bioinformatiques et statistiques pour l'analyse de l'air expiré en médecine par spectrométrie de masse en temps-réel](https://www.sfbi.fr/content/d%C3%A9veloppement-doutils-bioinformatiques-et-statistiques-pour-lanalyse-de-lair-expir%C3%A9-en-m%C3%A9dec), CEA Saclay, France
* [Ingénieur en développement de composants Galaxy pour la protéomique](https://www.sfbi.fr/content/ing%C3%A9nieur-en-d%C3%A9veloppement-de-composants-galaxy-pour-la-prot%C3%A9omique-0), MaIAGE , INRA, Jouy-en-Josas, France
* The [Freiburg Galaxy Team](https://usegalaxy-eu.github.io/galaxy-freiburg/people/) is [looking for 2 Postdoctoral researchers](https://usegalaxy-eu.github.io/galaxy-freiburg/2018/01/18/openpositions.html)
* The [Blankenberg](/src/people/dan/index.md) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/src/news/2017-09-hiring-at-cc/index.md).
* [Galaxy Project is hiring software engineers and postdocs at Johns Hopkins](/src/news/2017-08-hiring-at-hopkins/index.md), Baltimore, Maryland, United States

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them last month, including a new public server:

## Metaproteomics Gateway Server

[<img class="float-right" src="/src/use/metaproteomics-gateway/metaproteomics-gateway-landing-400.png" width="250" alt="Metaproteomics Gateway Server" />](http://z.umn.edu/metaproteomicsgateway)

The [Galaxy-P Metaproteomics Gateway](http://z.umn.edu/metaproteomicsgateway) provides access to documentation and other instructional materials, and an opportunity for hands-on training using example datasets and optimized metaproteomics workflows. The main goal of this server is to provide documentation to facilitate training and mastery of these software and workflows.

[Step-by-step training instructions and related information](http://z.umn.edu/supps1) and a [Metaproteomics Tutorial Video](http://z.umn.edu/mpvideo2018) are available.  The server can be accessed anonymously or you can create an account.  It is maintained by [@galaxyproteomics](https://github.com/galaxyproteomics).

See [Disseminating Metaproteomic Informatics Capabilities and Knowledge Using the Galaxy-P Framework](https://doi.org/10.3390/proteomes6010007) by Clemens Blank *et al.* for more.

## deepTools 3.0 is live

deepTools 3.0 is now available on the [deepTools Galaxy Server](https://deeptools.ie-freiburg.mpg.de/). Changes include a completely revamped normalization interface (and new methods), CRAM support, interactive plots,  a more homogenized interface, new tools and better MultiQC integration

## ARGs-OAP v2.0 published

In addition to being referenced in several papers last month (see below), the [ARGs-OAP server](/src/use/args-oap/index.md) was upgraded as well:

* [ARGs-OAP v2.0 with an expanded SARG database and Hidden Markov Models for enhancement characterization and quantification of antibiotic resistance genes in environmental metagenomes](https://doi.org/10.1093/bioinformatics/bty053), Xiaole Yin, Xiao-Tao Jiang, Benli Chai, Liguan Li, Ying Yang, James R Cole, James M Tiedje, Tong Zhang. *Bioinformatics*, bty053, doi: 10.1093/bioinformatics/bty053

## Public Servers in Publications

We tag papers that use, mention, implement or extend public Galaxy Servers.  Here are the counts for February's publications.

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 15 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Huttenhower) | | 7 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RepeatExplorer) | | 6 | [>ARGs-OAP](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>ARGs-OAP) | | 3 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Workflow4Metabolomics) | 
| 3 | [>Orione](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Orione) | | 2 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Cistrome) | | 2 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Galaxy-P) | | 2 | [>RNA-Rocket](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RNA-Rocket) | 
| 1 | [>PhenoMeNal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PhenoMeNal) | | 1 | [>Osiris](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Osiris) | | 1 | [>BioCiphers](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>BioCiphers) | | 1 | [>Metaproteomics Gateway](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Metaproteomics Gateway) | 
| 1 | [>Mississippi](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Mississippi) | | 1 | [>BIPAA](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>BIPAA) | | 1 | [>Genomic Hyperbrowser](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Genomic Hyperbrowser) | | 1 | [>RiboGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RiboGalaxy) | 
| 1 | [>APOSTL](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>APOSTL) | | 1 | [>PreSTIGE](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PreSTIGE) | | 1 | [>GenAP](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GenAP) | | 1 | [>GVL-Unspecified](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GVL-Unspecified) | 
| 1 | [>Pasteur](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Pasteur) | | 1 | [>GalaxEast](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GalaxEast) | | | | | | | |


----

# Tools

## ToolShed Contributions

<div class='right'>[<img src="/src/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in February](/src/toolshed/contributions/2018-02/index.md).

# Releases

## Planemo 0.48.0

[<img class="float-right" src="/src/images/galaxy-logos/planemo-logo.png" alt="Planemo" width="200" />](https://pypi.python.org/pypi/planemo/)

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

## Pulsar 0.8.1-3

[<img class="float-right" src="/src/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" />](https://pypi.python.org/pypi/pulsar-app/)

[Pulsar](https://pypi.python.org/pypi/pulsar-app/) updates were [released in February](https://github.com/galaxyproject/pulsar/blob/master/HISTORY.rst).  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

## galaxy-lib 17.9.11-18.5.2

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).


## Earlier Releases

Other packages released in the prior 4 months.

### 17.09 Galaxy Release

<div class="right">
<a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a><br /><br />
</div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.09 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html)** in October.

Highlights include

* **Singularity** Tool execution using the HPC-friendly container technology Singularity is now supported. 
* **Download entire collection** Downloading whole collections is now possible from the history interface. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Switch tool versions in workflows** You can now select exactly what version of tool you want to use when building workflows. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Security patches**
* Several features were **deprecated**

See the [17.09 release announcement](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html) for details.

### Galaxy Docker Image 17.09

The [Galaxy Docker project has seen a matching release](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.09), for Galaxy 17.05. Major features include
* much improved documentation about using Galaxy Docker and an external cluster ([@rhpvorderman](https://github.com/rhpvorderman))
* CVMFS support - mounting in 4TB of pre-build reference data ([@chambm](https://github.com/chambm))
* Singularity support and tests (compose only)
* more work on K8s support and testing ([@jmchilton](https://github.com/jmchilton))
* using .env files to configure the compose setup for SLURM, Condor, K8s, SLURM-Singularity, Condor-Docker

And

* The [Galaxy Docker Project](https://github.com/bgruening/docker-galaxy-stable) has reached more than 31k downloads on [Dockerhub](https://hub.docker.com/r/bgruening/galaxy-stable/) - not counting quay.io and all flavors 

### ephemeris 0.8.0

Ephemeris is a small Python library and set of scripts for managing the bootstrapping of Galaxy plugins - tools, index data, and workflows.

* Free software: Academic Free License version 3.0
* Documentation: https://ephemeris.readthedocs.org.
* Code: https://github.com/galaxyproject/ephemeris

### blend4php 0.1 beta

[<img class='float-right' src="/src/images/galaxy-logos/blend4phpLogo.png" alt="blend4php" width="220" />](https://github.com/galaxyproject/blend4php)

The beta version of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html), was released in December. It provides a PHP package for interacting with Galaxy and CloudMan. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/) page for full information.

## And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

# Other News

* From Björn Grüning 
  * New Galaxy tutorial by Simon Gladman and Torsten Seeman about [Microbial Variant Calling using snippy](https://galaxyproject.github.io/training-material/topics/variant-analysis/tutorials/microbial-variants/tutorial.html). 
  * This is big: [Aysam Guerler](/src/people/guerler/index.md) has reworked the Galaxy visualization plugin architecture.
