---
autotoc: true
title: Galaxy Screencasts and Demos
---


PLACEHOLDER_INCLUDE(/src/Learn/LinkBox/index.md)
<div class='left'></div>

Screencasts are one of several ways to **[learn](/src/Learn/index.md)** how to use Galaxy.  These video tutorials cover many aspects of Galaxy, from simple tasks like uploading data, to complex analysis.

The current screencasts use a variety of technologies, including QuickTime (you may need to download the [QuickTime player](http://www.apple.com/quicktime/download/)).

Watch some of our screencasts at **[Galaxy Project on YouTube](http://www.youtube.com/user/nekrut/videos)**.
<br />
<br />
**NEW**!! Watch many videos now here at **[http://vimeo.com/galaxyproject](http://vimeo.com/galaxyproject)**
*Including the latest tutorials plus talks from GCC 2013.*
<br />
<br />
## Getting Started

* **[Galaxy 101](http://screencast.g2.bx.psu.edu/galaxy101/)**

## Using Galaxy

<div class='left'> 
<table>
  <tr>
    <td> <strong><a href='http://vimeo.com/galaxyproject/accounts'>Accounts on Main</a></strong><br /> registration, openIDs</td>
    <td> <strong><a href='http://vimeo.com/galaxyproject/datasets1'>Datasets 1</a></strong> and <strong>Datasets 2</strong> (pending) <br /> icons, functions, <br /> content, metadata, <br /> basic workflows </td>
    <td> <strong><a href='http://vimeo.com/galaxyproject/sharepublish'>Share or Publish</a></strong> <br /> histories, workflows, <br /> visualizations, pages, <br /> publication support </td>
  </tr>
  <tr>
    <td> <strong><a href='http://vimeo.com/galaxyproject/upload'>Get Data: Upload File</a></strong> <br /> browse, url, paste, <a href='http://screencast.g2.bx.psu.edu/quickie_17_ftp_upload/flow.html'>ftp</a> </td>
    <td> <strong><a href='http://vimeo.com/galaxyproject/learning'>Learning Resources</a></strong> <br /> self-serve help </td>
    <td> <strong><a href='http://vimeo.com/galaxyproject/managehistories'>Managing Histories</a></strong> <br /> saved histories, <a href='/src/Main/index.md'>disk quotas</a> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/quickie_17_ftp_upload/flow.html'>Using FTP</a></strong> <br /> step-by-step, <a href='/src/FTPUpload/index.md'> wiki</a> </td>
    <td> <strong><a href='http://vimeo.com/galaxyproject/fastqprep'>FASTQ Prep Illumina</a></strong> <br /> jobs, FASTQ Groomer, <br /> edit attributes, FastQC </td>
    <td> <strong><a href='http://vimeo.com/galaxyproject/customgenome'>Custom Genomes</a></strong> <br /> format, how to load, <br /> <a href='/src/Learn/CustomGenomes/index.md'>read more on wiki...</a></td>
  </tr>
</table>

</div>
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
## Analysis

<div class='left'>
<table>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/flash/TAF1.html'>TAF1: Which genes' promoter regions contain these sites?</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/flash/mg_screencast_1.html'>Metagenomics 1</a> and <a href='http://screencast.g2.bx.psu.edu/flash/mg_screencast_2.html'>Metagenomics 2</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/bushman_access/flow.html'>Bushman Data Access</a> and <a href='http://screencast.g2.bx.psu.edu/bushman_analysis/flow.html'>Bushman Data Analysis</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/quickie8_solid_single_end/flow.html'>SOLiD Single End</a></strong> and <strong><a href='http://screencast.g2.bx.psu.edu/quickie9_solid_mate_pair/flow.html'>SOLiD Mate Pair</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/quickie10_custom_genome/flow.html'>Mapping Against Custom Genome</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/quickie_11_illumina_se/flow.html'>Illumina Single Ends</a></strong> and <strong><a href='http://screencast.g2.bx.psu.edu/quickie12_illumina_pe/flow.html'>Illumina Paired Ends</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/quickie_15_lastz_frag/flow.html'>454 Mapping Single End</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://screencast.g2.bx.psu.edu/DNaseIHSS/'>DNase I hypersensitive sites</a></strong> </td>
  </tr>
  <tr>
    <td> <strong><a href='http://www.youtube.com/watch?v=kMME658xOu4&feature=youtu.be'>Using PhyloTastic Tools inside Galaxy</a></strong> by Rutger Vos, created 2012/06/11 </td>
  </tr>
  <tr>
    <td> <strong><a href='http://main.g2.bx.psu.edu/u/galaxyproject/p/using-galaxy-2012'>CPB-2012</a>:</strong> <br /> prot1 <strong><a href='http://vimeo.com/galaxyproject/cbp-usinggalaxy1'>Finding Human Coding Exons with Highest SNP Density</a></strong> <br /> prot2 <strong><a href='http://vimeo.com/galaxyproject/cbp-usinggalaxy2'>Loading Data and Understanding Datatypes</a></strong> <br /> prot3 <strong><a href='http://vimeo.com/galaxyproject/cbp-usinggalaxy3'>Calling Peaks for ChIP-seq Data</a></strong> <br /> prot5 <strong><a href='http://vimeo.com/galaxyproject/cbp-usinggalaxy5'>Working with Multiple Sequence Alignments</a></strong> </td>
  </tr>
</table>

</div>
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
<br />
## Tutorials

* **[Using UCSC from Galaxy](http://screencast.g2.bx.psu.edu/UCSC2Galaxy/)**
* **[Sequences as Tab Delimited Data](http://screencast.g2.bx.psu.edu/quickie1_TabSeq/flow.html)** 
* **[Filtering Data](http://screencast.g2.bx.psu.edu/microFilterTabData/)**
* **[Grouping](http://screencast.g2.bx.psu.edu/quickie2_Grouping/flow.html)** 
* **[Intervals](http://screencast.g2.bx.psu.edu/quickie3_Intervals/flow.html)**
* **[Joining Features](http://screencast.g2.bx.psu.edu/quickie5_join/flow.html)**
* **[http://screencast.g2.bx.psu.edu/MAF_alignExtractStitch/Stitching Multiple Alignments](http://screencast.g2.bx.psu.edu/MAF_alignExtractStitch/Stitching Multiple Alignments)**
* **[Filtering Multiple Alignments and Converting Formats](http://screencast.g2.bx.psu.edu/MAF_alignFilterConvert/)**
* **[Manipulation of Multiple Alignments](http://screencast.g2.bx.psu.edu/MAF_manipulation)**
* **[Sharing your work](http://screencast.g2.bx.psu.edu/flash/Sharing.html)**
* **[Create workflow from history](http://screencast.g2.bx.psu.edu/flash/WorkflowFromHistory.html)** 
* **[Create workflow from Scratch](http://screencast.g2.bx.psu.edu/flash/WorkflowFromScratch.html)**
* **[Import Workflow](http://screencast.g2.bx.psu.edu/mt-workflowImport/)**
* **[Edit Workflow](http://screencast.g2.bx.psu.edu/GR_WF_4B/)**
* **[Convert Workflow in a Tool](http://screencast.g2.bx.psu.edu/GR_WF_6/)**
* **[Extracting Sequences and Alignments](http://screencast.g2.bx.psu.edu/flash/SeqAlign.html)**
* **[Basic FASTQ Manipulation](http://screencast.g2.bx.psu.edu/quickie_13_fastq_basic/flow.html)** 
* **[Advanced FASTQ Manipulation](http://screencast.g2.bx.psu.edu/quickie_14_fastq_adv/flow.html)** 
* **[Copy History Items](http://screencast.g2.bx.psu.edu/quickie_16_copyingHistoryItems/index.html)**
* **[BioMart](http://screencast.g2.bx.psu.edu/BioMart2Galaxy/)**

## Interval Operations tutorial

Also see [Learn/Interval Operations](/src/Learn/Interval Operations/index.md).

* **[Preparing Data](http://screencast.g2.bx.psu.edu/GOPS_PrepareData/)**
* **[Intersect and Subtract](http://screencast.g2.bx.psu.edu/GOPS_IntersectSubtract/)**
* **[Merge and Concatenate](http://screencast.g2.bx.psu.edu/GOPS_MergeConcatenate/)**
* **[Base Coverage and Compliment](http://screencast.g2.bx.psu.edu/GOPS_BaseCoverageComplement/)**
* **[Feature Coverage](http://screencast.g2.bx.psu.edu/GOPS_Coverage/)**
* **[Finding Clusters](http://screencast.g2.bx.psu.edu/GOPS_Cluster/)**
* **[Joining Features](http://screencast.g2.bx.psu.edu/quickie5_join/flow.htm)** 

## Sample Tracking

* **[Sample Tracking Admin](http://screencast.g2.bx.psu.edu/sample_tracking/admin_tasks.html)**
* **[Sample Tracking Create Forms](http://screencast.g2.bx.psu.edu/sample_tracking/create_forms.html)**
* **[Sample Tracking Request Admin](http://screencast.g2.bx.psu.edu/sample_tracking/create_request_admin.html)**
* **[Sample Tracking Request](http://screencast.g2.bx.psu.edu/sample_tracking/create_request.html)**
* **[Sample Tracking Sequencer Config](http://screencast.g2.bx.psu.edu/sample_tracking/create_seq_conf.html)**
* **[Sample Tracking Transfer Datasets](http://screencast.g2.bx.psu.edu/sample_tracking/create_transfer_datasets.html)**
* **[PacBio Demo](http://screencast.g2.bx.psu.edu/pacBio-demo/)**

## Developers How To

* **[Tool integration](http://screencast.g2.bx.psu.edu/toolIntegration/)**
* **[Galaxy in the Cloud using AWS](http://screencast.g2.bx.psu.edu/cloud/)**
* **[Getting Started with the CloudMan Platform Using BioCloudCentral](http://www.youtube.com/watch?v=AKu_CbbgEj0)**
* **[PDC Cloud Screencast for using Galaxy on Amazon AWS with SNIC Accounts](http://www.youtube.com/watch?v=Jzp0D2dTbbo)** - Created by the SNIC/PDC group.  Generic, non-SNIC content starts at 2:00 in.

## Archives

* [Galaxy2ENCODE](/src/Learn/Screencasts/GR2007 Screencasts/index.md) - A framework for collaborative analysis of ENCODE data: Making large-scale analyses biologist-friendly ([Genome Research 2007](http://www.genome.org/cgi/content/full/17/6/960))
* [Current Protocols in BioInformatics](/src/Learn/Screencasts/Current Protocols/index.md) - Galaxy: Interactive Analyses of Genomic Data by Example
* [Screencasts detailing analysis of data generated by Affymetrix tiling arrays](/src/Learn/Screencasts/Affy Screencasts/index.md)

