---
title: Map and identify phenotypic mutations in model organism genomes
date: '2018-03-08'
tags: [tools]
subsites: [eu, freiburg]
main_subsite: freiburg
---

The MiModD suite of tools, developed in the
[Baumeister lab](https://celegans.de/mimodd/) at the Institute for Biology 3
here in Freiburg, is now available on the server.
MiModD is for geneticists who have isolated phenotypic mutants of their
favourite organism, *e.g.*, through forward genetic screens, and who now wish
to map the underlying mutations and eventually identify them molecularly.

Provided you have followed a mutant phenotype through a cross between the
mutant line and a suitable mapping strain and have sequenced recombinant
offspring genomes, you can use the MiModD tools to identify marker SNPs
(inherited from either parent) and perform a linkage analysis between these
markers and the observed phenotype. You can then continue with MiModD to
identify and annotate candidate mutations that fall into the linked genomic
region.

MiModD can derive linkage patterns from a variety of different crossing
schemes and has been used successfully for species as diverse as the ciliate
*Tetrahymena thermophila*, the roundworm *C. elegans*, and zebrafish.

A Galaxy-based tutorial for the tools is available
[here](https://galaxyproject.github.io/training-material/topics/variant-analysis/tutorials/mapping-by-sequencing/tutorial.html) and there is also a shared
[workflow to reproduce the analysis from the tutorial](https://usegalaxy.eu/u/wolfgang-maier/w/identify-phenotypic-mutation-in-arabidopsis).
Additional example analyses can be found in the
[MiModD documentation](https://mimodd.readthedocs.io/en/default/tutorial.html).

