---
title: Galactic December News!
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/src/galaxy-updates/index.md'><img src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the December 2015 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community. 

If you have anything to include in the next News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org). 

# New Papers

**81** new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in November. Highlights include:

* [Genomics Virtual Laboratory: A Practical Bioinformatics Workbench for the Cloud](https://doi.org/10.1371/journal.pone.0140829)
    by [Enis Afgan](/src/people/enis-afgan/index.md), Clare Sloggett, Nuwan Goonasekera *et al.*, *PloS One*, Vol. 10, No. 10. (2015)

* [Cost-Aware Cloud Provisioning](https://doi.org/10.1109/escience.2015.67)
    by Ryan Chard, Kyle Chard, Kris Bubendorfer, Lukasz Lacinski, Ravi Madduri, Ian Foster, In e-Science (e-Science), 2015 IEEE 11th International Conference on (August 2015), pp. 136-144, doi:10.1109/escience.2015.67

* [DNA data bank of Japan (DDBJ) progress report](https://doi.org/10.1093/nar/gkv1105)
    by Jun Mashima, Yuichi Kodama, Takehide Kosuge, *et al.* *Nucleic Acids Research* (17 November 2015), gkv1105, doi:10.1093/nar/gkv1105

* [antaRNA – Multi-objective inverse folding of pseudoknot RNA using ant-colony optimization](https://doi.org/10.1186/s12859-015-0815-6)
    by Robert Kleinkauf, Torsten Houwaart, Rolf Backofen, Martin Mann, *BMC Bioinformatics*, Vol. 16, No. 1. (18 November 2015), 389, doi:10.1186/s12859-015-0815-6

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 23 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 37</td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 11 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


----

# Events

## Spanish Galaxy Community Meetup: 18 December, Barcelona

<div class='right'>
<a href='http://scb.iec.cat/wp-content/uploads/2015/11/jdB2015_anunci_.pdf'><img src="/src/images/logos/BioinformaticsBarcelona.jpg" alt="Galaxy Session at III Bioinformatics and Computational Biology Symposium" height="120" /></a>
</div>

There will be a session at the end of the [III Bioinformatics and Computational Biology Symposium](http://scb.iec.cat/wp-content/uploads/2015/11/jdB2015_anunci_.pdf) being held in Barcelona on 18 December:

 where users and developers can, not only share information and code, but also plan how to coordinate for future Galaxy workshops and other activities. Chairs of this session are [Toni Espinosa](http://grupsderecerca.uab.cat/hpca4se/en/content/antonio-espinosa-morales) ([UAB](http://www.uab.cat/)) and [Gonzalo Vera (CRAG)](http://www.cragenomica.es/staff/detail/gonzalo-vera).

See the [announcement PDF](http://scb.iec.cat/wp-content/uploads/2015/11/jdB2015_anunci_.pdf) and the [announcement email](http://bit.ly/1Xsad8D) for further details.

## Galaxy Day 2015 Presentations

<div class='left'>
<a href='http://www.france-bioinformatique.fr/?q=fr/galaxy-2'><img src="/src/images/logos/IFB_galaxywg.png" alt="Galaxy Day 2015" height="70" /></a>
</div>

[Galaxy Day 2015](http://www.france-bioinformatique.fr/?q=fr/galaxy-2) was held in Paris on 19 November. It was preceded by a Galaxy Hackathon.  Both events were well attended and the events were yet another successful gathering of the French Galaxy community.  Presentations from Galaxy Day and a summary of the Hackathon are now available [online](http://www.france-bioinformatique.fr/?q=fr/galaxy-2) (click on "Archives Galaxy").

## GCC2016 Registration will open this month

GCC2016 registration will open this month. The full training schedule will also be posted at that time.  We'll announce this on all Galaxy channels - you won't miss it.

## Upcoming Events

There are [many upcoming events](/src/events/index.md) in the next few months.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='center'> 
<a href='http://www.bioplatforms.com/current-training-courses/'><img src="/src/images/logos/BioplatformsAustralia.png" alt="Galaxy Training from Bioplatforms Australia and Intersect" height="40" /></a>
<a href='http://scb.iec.cat/wp-content/uploads/2015/11/jdB2015_anunci_.pdf'><img src="/src/images/logos/BioinformaticsBarcelona.jpg" alt="Galaxy Session at III Bioinformatics and Computational Biology Symposium" height="80" /></a>&nbsp;
<a href='/src/events/asu2016/index.md'><img src="/src/images/logos/ASUBioKIC_600.png" alt="Biodiversity Knowledge Integration Center" height="60" /></a>&nbsp;
<a href='/src/events/pag2016/index.md'><img src="/src/images/logos/PAGLogo300.png" alt="Galaxy @ Plant and Animal Genome (PAG 2016)" height="90" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> November 30 - December 1 </th>
    <td> <a href='https://github.com/galaxyproject/tools-iuc/issues/299'>IUC Contribution Fest - Metagenomics Tools and Workflows</a> </td>
    <td> <img src='/src/images/icons/GL.png' alt='Around the World' align='right' /> <em>Online</em> </td>
    <td> <a href='IUC'>/src/iuc/index.md</a>, <a href='/src/people/john-chilton/index.md'>John Chilton</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> December 3 </th>
    <td> <em><a href='http://www.bioplatforms.com/rna-seq-using-galaxy-dec15/'>RNA-Seq Analysis Using Galaxy</a></em> </td>
    <td rowspan=2> <img src='/src/images/icons/AU.png' alt='Australia' align='right' /> University of Newcastle, Newcastle, NSW, Australia </td>
    <td rowspan=2> <a href='/src/teach/trainers/index.md'><img src='/src/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> Mark Crowe, <a href="mailto:kchamp@bioplatforms.com">Katherine Champ</a> </td>
  </tr>
  <tr>
    <th> December 4 </th>
    <td> <em><a href='http://www.bioplatforms.com/variant-detection-using-galaxy-dec15/'>Variant Detection Using Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> December 18 </th>
    <td> <em><a href='http://scb.iec.cat/wp-content/uploads/2015/11/jdB2015_anunci_.pdf'>Galaxy Session</a></em> </td>
    <td> <img src='/src/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://scb.iec.cat/wp-content/uploads/2015/11/jdB2015_anunci_.pdf'>III Bioinformatics and Computational Biology Symposium</a>, Barcelona, Spain </td>
    <td> <a href="mailto:scb@iec.cat">Email</a> </td>
  </tr>
  <tr>
    <td> </td>
    <td colspan=4 style=" background-color: #eef; text-align: center;"> <strong>2016</strong> </td>
  </tr>
  <tr>
    <th> January 8 </th>
    <td> <em><a href='/src/events/asu2016/index.md'>Introduction to Galaxy Workshop</a></em> </td>
    <td> <img src='/src/images/icons/NA.png' alt='North America' align='right' /> <a href='http://asu.edu/'>Arizona State University</a>, Tempe, Arizona, United States </td>
    <td> <a href='/src/teach/trainers/index.md'><img src='/src/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://sols.asu.edu/people/nico-franz'>Nico Franz</a>, <a href='https://sols.asu.edu/people/melissa-wilson-sayres'>Melissa A. Wilson Sayres</a>, <a href='/src/people/anton/index.md'>Anton Nekrutenko</a> </td>
  </tr>
  <tr>
    <th> January 9-13 </th>
    <td> <strong><a href='/src/events/pag2016/index.md'>Plant and Animal Genome (PAG XXIV)</a></strong> </td>
    <td> <img src='/src/images/icons/NA.png' alt='North America' align='right' /> San Diego, California, United States </td>
    <td> See <a href='/src/events/pag2016/index.md'>presenters</a> </td>
  </tr>
  <tr>
    <th> January 19 </th>
    <td> <em><a href='http://www.festivalofgenomicslondon.com/an-introduction-to-clinical-bioinformatics/'>An introduction to clinical bioinformatics</a></em> </td>
    <td> <img src='/src/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.festivalofgenomicslondon.com/'>Festival of Genomics London</a>, London, United Kingdom </td>
    <td> <a href='http://www.festivalofgenomicslondon.com/speaker/ang-davies/'>Ang Davies</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> February 11 </th>
    <td> <em><a href='http://bioteam.net/bioteam-appliance/galaxy-edition/'>BioTeam Galaxy Appliance: Simplify Hosting Your Own Galaxy (Software Demo)</a></em> </td>
    <td> <img src='/src/images/icons/NA.png' alt='North America' align='right' /> <a href='http://www.agbt.org/meetings/agbt-general-meeting/'>AGBT</a>, Orlando, Florida, United States </td>
    <td> <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a></td>
  </tr>
  <tr>
    <th> February 15-19 </th>
    <td> <em><a href='https://www.supagro.fr/web/pages/?idl=19&page=1964&id_page=2666'>Analyse bioinformatique de séquences sous Galaxy</a></em> </td>
    <td> <img src='/src/images/icons/EU.png' alt='Europe' align='right' /> <a href='https://www.supagro.fr/web/index.php'>Montpellier SupAgro</a>, Montpellier, France </td>
    <td> <a href='/src/teach/trainers/index.md'><img src='/src/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://www.supagro.fr/web/pages/contact.php?id=139'>Yolande Olivier</a>, <a href='https://www.supagro.fr/web/pages/contact.php?id=102'>Dominique Alhinc</a> </td>
  </tr>
  <tr>
    <th> February 20-23 </th>
    <td> <em><a href='https://conf.abrf.org/the-galaxy-platform'>(SW4) The Galaxy Platform for Multi-Omic Data Analysis and Informatics</a></em> </td>
    <td> <img src='/src/images/icons/NA.png' alt='North America' align='right' />  <a href='https://conf.abrf.org/abrf-2016'>ABRF 2016</a>, Fort Lauderdale, Florida, United States </td>
    <td> <a href='/src/teach/trainers/index.md'><img src='/src/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://cbs.umn.edu/contacts/pratik-d-jagtap'>Pratik Jagtap</a>, <a href='https://cbs.umn.edu/contacts/timothy-j-griffinhttps://cbs.umn.edu/contacts/timothy-j-griffin'>Tim Griffin</a>, <a href='/src/people/dave-clements/index.md'>Dave Clements</a>, <a href='http://experts.umn.edu/en/persons/candace-r-guerrero(c3134cd2-fe98-4d90-b5de-77fce1921d29).html'>Candace Guerrero</a>, <a href='http://www-users.cs.umn.edu/~onsongo/'>Getiria Onsongo</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 25-29 </th>
    <td> <strong><em><a href='/src/events/gcc2016/index.md'>2016 Galaxy Community Conference (GCC2016)</a></strong></em> </td>
    <td> <img src='/src/images/icons/NA.png' alt='North America' align='right' /> <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> <a href='/src/teach/trainers/index.md'><img src='/src/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a>  <a href='https://gcc2016.iu.edu/organizers/'>Organizers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/src/teach/trainers/index.md'><img src="/src/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/src/teach/trainers/index.md'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><a href='/src/galaxy-is-hiring/index.md'><img src="/src/galaxy-is-hiring/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [intégration d'outils phylogénétiques dans Galaxy](http://bit.ly/1NYrSdI), [ANSES Ploufragan](https://www.anses.fr/en/node/50284), Unité de Génétique Virale et Biosécurité, France
* [Computational Biologist / Bioinformatician](http://www.jobs.ac.uk/job/AMJ391/computational-biologist-bioinformatician/), [The Sainsbury Laboratory](http://www.tsl.ac.uk/), Norwich, UK.
* [Software Engineer](/src/news/software-engineer-ohsu/index.md), Oregon Health Sciences University, Portland, Oregon, United States
* [Software Developer / Bioinformatician](https://ig14.i-grasp.com/fe/tpl_embl01.asp?s=4A515F4E5A565B1A&jobid=54309,8734341283&key=64564906&c=342372129814&pagestamp=seqbdpnrnmbvdpxtgb), European Molecular Biology Laboratory (EMBL), Heidelberg, Germany
    Closes 13 December 2015.
* **The [Galaxy Project is hiring software engineers and post-docs](/src/galaxy-is-hiring/index.md)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

----

# New Public Galaxy Servers

[Two new publicly accessible Galaxy servers](/src/use/index.md) were added in November, bringing the total precipitously close to 80.

## Galaxy Integrated Omics (GIO)

<div class='right solid'><a href='http://gio.sbcs.qmul.ac.uk/'><img src="/src/use/galaxy-integrated-omics-gio/gio-logo.png" alt="Galaxy Integrated Omics (GIO)" width="200" /></a></div>

* *Link:*
  * [GIO Server](http://gio.sbcs.qmul.ac.uk/)
* *Domain/Purpose:*
  * "Proteomics Informed by Transcriptomics (PIT) methodology described in [Evans et al. 2012](http://www.ncbi.nlm.nih.gov/pubmed/23142869), and selection of surrogate peptides for targeted proteomics."
* *Comments:*
  * "Galaxy-based Integrated Omics (GIO) is a curated collection of new and pre-existing open source tools brought together for proteomics applications."
  * See [Galaxy Integrated Omics: Web-based standards-compliant workflows for proteomics informed by transcriptomics](http://www.mcponline.org/content/14/11/3087), Jun Fan, Shyamasree Saha, Gary Barker, Kate J. Heesom, Fawaz Ghali, Andrew R. Jones, David A. Matthews and Conrad Bessant, *Molecular & Cellular Proteomics*, 14, 3087-3093. 
* *User Support:*
  * [Tutorial 1:Getting started with GIO](http://gio.sbcs.qmul.ac.uk/static/Tutorial1.pdf)
  * [Tutorial 2:Getting started with GIO workflows](http://gio.sbcs.qmul.ac.uk/static/Tutorial2.pdf)
  * [Tutorial 3:Proteomics Informed by Transcriptomics (PIT)](http://gio.sbcs.qmul.ac.uk/static/Tutorial3.pdf)
  * [Tutorial 4:Targeted proteomics](http://gio.sbcs.qmul.ac.uk/static/Tutorial4.pdf)
  * For draft documentation please visit the [GIO repository](https://code.google.com/p/gio-repository/). 
* *Quotas:* 
* *Sponsor(s):*
  * "This project is led by [Conrad Bessant](http://www.bessantlab.org/) at Queen Mary and [David Matthews](http://www.bristol.ac.uk/infection-immunity/people/person/37949) at Bristol, with additional contributions from the groups of [Andy Jones](http://pcwww.liv.ac.uk/~jonesar/jonesar.html) at Liverpool and [Simon Hubbard](http://www.manchester.ac.uk/research/simon.hubbard/) at Manchester. GIO development was supported by BBSRC TRDF2 grants BB/L018438/1 (Proteomics Goes Viral), BB/K016075/1 (Galaxy Workflows for Proteomics Informed by Transcriptomics) and BB/K004123/1 (Integrating Genomes and Proteomes on the Cloud). Lead developer is [Jun Fan](http://www.sbcs.qmul.ac.uk/staff/drjunfan.html)."

## RNA-Seq Portal

<div class='right solid'><a href='http://weizhongli-lab.org/RNA-seq/'><img src="/src/public-galaxy-servers/archive/rna-seq-portal.png" alt="RNA-Seq Portal - Analyzing RNA-seq Data for Agricultural Animal Species" width="300" /></a></div>

* *Links:*
  * [RNA-Seq Portal](http://weizhongli-lab.org/RNA-seq/)
  * [Galaxy server](http://weizhongli-lab.org:8088/)

* *Domain/Purpose:*
  * Analyzing RNA-seq Data for Agricultural Animal Species.
* *Comments:*
  * From the [portal landing page](http://weizhongli-lab.org/RNA-seq/):
      The goal of this project is to develop a web portal with integrated tools for RNA-seq based gene expression analysis for agricultural animals.
      * improve genome annotation of agricultural animal species, including (but not limiting to) cattle, pig, chicken, turkey, horse, sheep, and goat as well as catfish;
      * develop and integrate needed bioinformatics tools and pipelines, visualization interfaces, and statistical methods;
      * build a web portal that enable RNA-seq based transcriptomics analysis in aforementioned animal species.
* *User Support:*
  * [Example usage](http://weizhongli-lab.org:8088/) (see center panel)
  * [Web support](http://weizhongli-lab.org/RNA-seq/contact/)
* *Quotas:* 

* *Sponsor(s):*
  * The project is developed by [Weizhong Li's lab](http://weizhongli-lab.org/) at [J. Craig Venter Institute](http://jcvi.org/) with support from NIFA (award #2013-67015-21428). 

----


# Releases

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/src/images/logos/PlanemoLogo.png" alt="" width="150" /></a></div>

## Planemo 0.19.0 - 0.21.1

[Planemo](https://pypi.python.org/pypi/planemo) saw a regular Oort cloud of activity in November with 3 releases.

Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.  [Planemo 0.21.1](https://pypi.python.org/pypi/planemo/0.21.1#history) is the most recent release. See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).

## Other Releases

<div class='right'><a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

 **[July 2015 Galaxy Release (v 15.07)](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html)**
   [Galaxy Release v 15.07](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html) includes enhancements to Interactive Environments and the Workflow Editor, an update of the Policies for Committers and Pull Requests and [much more](http://galaxy.readthedocs.org/en/master/releases/15.07_announce.html).<br /><br />

 **BioBlend 0.7.0**

   [BioBlend version 0.7.0](https://github.com/galaxyproject/bioblend/) was released at the beginning of November. BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/src/develop/api/index.md).  CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG).


 **Mid 2015 Galaxy CloudMan Release**

<div class='right'><a href='/src/cloudman/index.md'><img src="/src/images/logos/CloudManWideBlackLogo.png" alt="CloudMan" width="150" /></a></div>

<div class='indent'>
   This is the most complete update of the Galaxy on the Cloud platform to date with all the cloud components being built from the ground up. See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a complete set of changes or just give it a try: see the all new [Getting Started](/src/cloudman/getting-started/index.md) documentation. 
</div>

 **Security Advisories**

   Three security vulnerabilities were announced and patched in August.  The Galaxy Committers have released a number of fixes which have been rolled in to affected versions of Galaxy dating back to the 14.10 release.

<div class='indent'><div class='indent'>
* **[Galaxy XSS Reflection Vulnerability](https://lists.galaxyproject.org/archives/list/galaxy-announce@lists.galaxyproject.org/message/EVXN3G5FZUPOEA4OYCC6V25TWGYBBWDV/)**
* **[Galaxy LDAP Authentication Vulnerability](https://lists.galaxyproject.org/archives/list/galaxy-announce@lists.galaxyproject.org/message/GOZKUUTSB2AQSBXIGCFC6RVH37MDBQQZ/)**
* **[Tool Shed Unauthorized Upload Vulnerability](https://lists.galaxyproject.org/archives/list/galaxy-announce@lists.galaxyproject.org/message/KNJIV2QCT52SPLXMCENLA2N5747ALSSK/)**

</div></div>


 **Pulsar**
   [Pulsar 0.5.0](https://pypi.python.org/pypi/pulsar-app/0.5.0) was released in May.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.
 

 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and BioCloudCentral.


----


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/teach/index.md'><img src="/src/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/blog/index.md'><img src="/src/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/deployment/index.md'><img src="/src/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your <a href='/src/teach/resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/teach/trainers/index.md#add-a-trainer'>experience</a> <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/blog/index.md'>Share your experience</a> <em><a href='/src/blog/index.md'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/deployment/index.md'>Describe your instance</a> <em><a href='/src/community/deployment/index.md'>now</a></em> </strong> </td>
  </tr>
</table>

<br />

----
<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in October and November](/src/toolshed/contributions/2015-11/index.md).


----

# Other News

* From [Guy Reeves](https://biostar.usegalaxy.org/u/4670/):
    'Hidden' Galaxy features which are really useful. For Scratchbook see [this video](http://vimeo.com/120839821).
* Ever wondered who writes Galaxy? 
    [These people do](https://github.com/galaxyproject/galaxy/blob/dev/CONTRIBUTORS.md).
* See the [executive summary](https://galaxyproject.github.io) of free open source software the Galaxy community provides. (Big thanks to E. Rasche.)
* From [Nate Coraor](https://github.com/natefoo)
    An Ansible role for installing Galaxy Interactive Environments 
* [Expanding the Galaxy Platform](https://www.msi.umn.edu/content/expanding-galaxy-platform) at the [Minnesota Supercomputing Institute](https://www.msi.umn.edu/).
* From [Björn Grüning](https://twitter.com/bjoerngruening):
    The [Galaxy Docker Image](https://github.com/bgruening/docker-galaxy-stable/tree/dev) has gained TravisCI testing. BioBlend will test all functionality and tool installation. 
* Updated documentation:
  * [The source code of Galaxy](/src/develop/source-code/index.md)
  * [Galaxy Job Configuration](/src/admin/config/jobs/index.md)
  * [Datatypes](/src/learn/datatypes/index.md)
