---
autotoc: true
title: September 2014 Galaxy Update
---


<div class='right'></div>

<div class='left'><a href='/GalaxyUpdates'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the September 2014 *[Galaxy Update](/src/GalaxyUpdates/index.md)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/src/GalaxyUpdates/index.md)* complement the *[Galaxy Development News Briefs](/src/DevNewsBriefs/index.md)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Galaxy-UK Community Launched

[Galaxy-UK](http://galaxy-community.org.uk/), a new [Galaxy Community](/src/Community/index.md) was launched in August.  The [Galaxy-UK Community](http://galaxy-community.org.uk/) aims to:  

* Bring the Galaxy community in the United Kingdom closer together
* Identify and address the needs of the community
* Encourage interaction and collaboration.

Galaxy-UK is also an information hub for events such as:
* [UK based Galaxy training courses](http://galaxy-community.org.uk/category/galaxy-timeline/training/)
* [UK based talks involving Galaxy](http://galaxy-community.org.uk/news/)
* Information on the [location of UK Galaxy servers](http://galaxy-community.org.uk/galaxy-servers/)
* Anything else that might be pertinent to bring the UK Galaxy users/admins/trainers together as a community

The community will also have both online meetings and physical meetings, so keep an eye on the [site](http://galaxy-community.org.uk/) for these events. 


## Other Galaxy Communities

Don't fret if you want to join a community but you are not in the UK.  [Galaxy-UK](http://galaxy-community.org.uk/) is just one of several [Galaxy communities](/src/Community/index.md) you can join.  (And there are rumors of a German language community in the works.)

# Events

## Galaxy Events in Europe, Fall 2014

There are a wealth of Galaxy related events in Europe this fall. Events include a [large Galaxy presence at ECCB'14](/src/Events/ECCB2014/index.md), the [Fourth Galaxy User Group Grand Ouest (GUGGO) meeting](http://www.biogenouest.org/agenda/evenement/groupe-de-travail-des-utilisateurs-galaxy-du-grand-ouest-guggo), the [2014 Swiss-German Galaxy Tour](/src/Events/SG2014/index.md), and several training events in Italy, the United Kingdom, Croatia, Norway, and France.

These events are a great way to meet other Galaxy users and developers and learn and share best practices. If you're in Europe and are interested in learning more about Galaxy and/or the community, then please give these a look.

<div class='center'> 
<a href='/Events/ECCB2014'><img src='/Images/Logos/ECCB2014Logo.png' alt='Galaxy at ECCB' height="80" /></a>
<a href='http://go4bioinformatics.genouest.org/integration-doutils-dans-la-plateforme-web-galaxy/'><img src='/Images/Logos/eBiogenouestLogo.png' alt='Tools integration on Galaxy' height="80" /></a> &nbsp;
<a href='http://www.bioinformatica.crs4.it/training'><img src='/Images/Logos/CRS4Logo.jpg' alt='Analisi dati Next Generation Sequencing con Galaxy' width="130" /></a> &nbsp;
<a href='/Events/SG2014'><img src='/Images/Logos/SG2014Logo400.png' alt='2014 Swiss German Galaxy Tour' height="90" /></a>
</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> September 6-10 </th>
    <td> <a href='/Events/ECCB2014'>At least one tutorial, a panel of European Public Galaxy Instances, and 5 posters</a> </td>
    <td> </strong><a href='/Events/ECCB2014'>European Conference on Computational Biology (ECCB'14)</a><strong>, Strasbourg, France </td>
    <td> <a href='/Events/ECCB2014'>Presenters</a> </td>
  </tr>
  <tr>
    <th> September 11 </th>
    <td> </em><a href='http://go4bioinformatics.genouest.org/integration-doutils-dans-la-plateforme-web-galaxy/'>Tools integration on Galaxy</a><em><div class='indent'>This event is full.</div> </td>
    <td> <a href='https://www.e-biogenouest.org/groups/guggo'>Galaxy User Group Grand Ouest</a>, Rennes, France </td>
    <td> <a href='https://www.e-biogenouest.org/members/1005'>Cyril Monjeaud</a>, <a href='https://www.e-biogenouest.org/members/1003'>Yvan Le Bras</a> </td>
  </tr>
  <tr>
    <th> September 15 </th>
    <td> </em><a href='http://www.biogenouest.org/agenda/evenement/groupe-de-travail-des-utilisateurs-galaxy-du-grand-ouest-guggo'>Fourth GUGGO meeting</a><em> </td>
    <td> <a href='https://www.e-biogenouest.org/groups/guggo'>Galaxy User Group Grand Ouest</a>, Rennes, France </td>
    <td> <a href='https://www.e-biogenouest.org/members/1005'>Cyril Monjeaud</a>, <a href='https://www.e-biogenouest.org/members/1003'>Yvan Le Bras</a> </td>
  </tr>
  <tr>
    <th> September 23-25 </th>
    <td> </em><a href='http://www.bioinformatica.crs4.it/next_generation_sequencing_course_with_galaxy_september_2014'>Analisi dati Next Generation Sequencing con Galaxy</a><em> </td>
    <td> Cagliari, Italy </td>
    <td> <a href="mailto:ngs-courses@crs4.it">CRS4</a> </td>
  </tr>
  <tr>
    <th> September 24 </th>
    <td> </em><a href='http://training.csx.cam.ac.uk/event/1222231'>Introduction to Galaxy - Data Manipulation and Visualisation</a><em> </td>
    <td> University of Cambridge, United Kingdom </td>
    <td> Anne Pajon, Jing Su </td>
  </tr>
  <tr>
    <th> September 25 </th>
    <td> </em><a href='http://go4bioinformatics.genouest.org/analyse-de-donnees-de-type-rad-par-le-pipeline-stacks-sous-la-plateforme-web-danalyse-de-donnees-galaxy/'>RADseq analysis using STACKS on Galaxy</a><em> </td>
    <td> <a href='https://www.e-biogenouest.org/groups/guggo'>Galaxy User Group Grand Ouest</a>, Rennes, France </td>
    <td> <a href='https://www.e-biogenouest.org/members/1003'>Yvan Le Bras</a>, <a href='https://www.e-biogenouest.org/members/1005'>Cyril Monjeaud</a> </td>
  </tr>
  <tr>
    <th> September 30 </th>
    <td> </em><a href='http://www.irb.hr/eng/Research/Divisions-and-Centers/Centre-for-Informatics-and-Computing/News/CIR-Interactive-Workshop-Introduction-to-bioinformatics-analysis-with-Galaxy-application'>CIR Interactive Workshop - Introduction to bioinformatics analysis with Galaxy application</a><em> </td>
    <td> RBI, Zagreb, Croatia </td>
    <td> <a href='https://wiki.galaxyproject.org/EnisAfgan'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th rowspan=3> September 30 - October 2 </th>
    <td> <a href='/Events/Switzerland2014/trainingday'>Galaxy Training and Demo Day</a> </td>
    <td rowspan=3> </strong><a href='/Events/SG2014'>2014 Swiss-German Galaxy Tour</a><strong> with events in </strong>Bern, Switzerland<strong> and </strong>Freiburg, Germany<strong> </td>
    <td rowspan=3> <a href='/HansrudolfHotz'>Hans-Rudolf Hotz</a> and <a href='/BjoernGruening'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <td> </em><a href='/Events/Switzerland2014'>(second Swiss) Galaxy Workshop</a><em> </td>
  </tr>
  <tr>
    <td> </em><a href='/Events/Germany2014'>German Galaxy Developers Day</a><em> </td>
  </tr>
  <tr>
    <th> October 6 - 24 </th>
    <td> </em><a href='http://wiki.uio.no/projects/clsi/index.php/INF-BIOX121_H14'>Intensive course in High Throughput Sequencing technologies and bioinformatics analysis</a><em> </td>
    <td> University of Oslo, Oslo, Norway </td>
    <td> <a href="mailto:lex.nederbragt AT ibv.uio.no">Lex Nederbragt</a>, <a href="mailto:karin.lagesen AT medisin.uio.no">Karin Lagesen</a> </td>
  </tr>
  <tr>
    <th> November 26-28 </th>
    <td> </em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-et-ChIP-seq-sequencage-haut-debit--a-l-aide-d-outils-orientes-vers-un-public-de-biologistes_a39.html'>RNA-Seq & ChIP-Seq analysis course using Galaxy</a><em> </td>
    <td> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <a href="mailto:navratil AT prabi DOT fr">Navratil V</a>., <a href="mailto:oger AT prabi DOT fr">Oger C.</a>, <a href="mailto:veber AT prabi DOT fr">Veber P.</a>, <a href="mailto:deschamps AT prabi DOT fr">Deschamps C.</a>, <a href="mailto:perriere AT prabi DOT fr">Perriere G.</a> </td>
  </tr>
</table>


<br />

## The Great GigaScience and Galaxy Workshop

<div class='right'><a href='http://www.gigasciencejournal.com/'><img src='/Images/Logos/GigaScienceLogo.gif' alt='GigaScience Journal' width="200" /></a><br />
<a href='http://www.vlsci.org.au/'><img src='/Images/Logos/VLSCILogo.png' alt='VLSCI' width="200" /></a><br />
<a href='http://australianbioinformatics.net/'><img src='/Images/Logos/ABNLogoWText200.jpg' alt='Australian Bioinformatics Network' width="200" /></a>
</div>

The [The Great GigaScience and Galaxy (G3) Workshop](http://australianbioinformatics.net/great-gigascience-galaxy-wkshp) will be held, Friday 19 September 2014 at The University of Melbourne from 8:45-5pm.
 
The day's theme is *Turning data—big data—into research impact*

**Morning Session of Talks (8:45am - 12:00pm)**
* Theatre 1, 207 Bouverie Street, Parkville.
  * Featuring talks from [David Vaux](https://theconversation.com/from-fraud-to-fair-play-australia-must-support-research-integrity-15733), A Speaker on Galaxy, Representative from NHMRC on open access and the [GigaScience Editorial Team](http://www.gigasciencejournal.com/about). 

**Afternoon Workshops (2:00pm - 5:00pm)**
* Workshop Stream 1 - [VLSCI Boardroom, 187 Grattan Street, Parkville](https://www.google.com.au/maps/place/187+Grattan+St,+University+of+Melbourne,+Carlton+VIC+3053/@-37.8003045,144.9626346,17z/data=!3m1!4b1!4m2!3m1!1s0x6ad642d318691715:0xbb29ca3420505539)
  * Setting up your own personal galaxy instance and basic galaxy usage. Galaxy 101.
* Workshop Stream 2 - [B117, 207 Bouverie Street, Parkville](https://www.google.com.au/maps/preview?q=B118,+207+Bouverie+Street,+Parkville&ie=UTF-8&hq&hnear=0x6ad642d2c404d12d:0xde07ecf4a4d913ad,b118/207+Bouverie+St,+University+of+Melbourne,+Carlton+VIC+3053&gl=au&ei=HMjHU5OrB8H48QW8j4GgAQ&ved=0CB0Q8gEwAA)
  * [Authorea](http://www.authorea.com/) software carpentry workshop

See the [event page](http://australianbioinformatics.net/great-gigascience-galaxy-wkshp) for registration and contact links, and additional information.


## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif' /></a></div> 

And don't worry, Europe does not have a complete lock on upcoming Galaxy related events.  There is also things going on in North America, and a few more in Australia too.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> October 29 - November 4 </th>
    <td> <a href='http://meetings.cshl.edu/courses/2014/c-ecg14.shtml'>Computational & Comparative Genomics Course</a> <div class='indent'>Application Deadline: July 15</div> </td>
    <td> <a href='http://www.cshl.edu/'>Cold Spring Harbor Laboratories (CSHL)</a>, New York, United States </td>
    <td> <a href='http://faculty.virginia.edu/wrpearson/'>William Pearson</a>, <a href='http://mcb.illinois.edu/faculty/profile/ljstubbs'>Lisa Stubbs</a> </td>
  </tr>
  <tr>
    <th rowspan=3> November 2-5 </th>
    <td> </em><a href='https://www.acsmeetings.org/tours-workshops#Intro to'>Introduction to Bioinformatics Analysis with Galaxy Workshop</a><em> </td>
    <td rowspan=3> </strong><a href='https://www.acsmeetings.org/home'>ASA, CSSA, and SSSA International Annual Meeting</a><strong>, Long Beach, California, United States </td>
    <td rowspan=3> <a href="mailto:outreach AT galaxyproject DOT org">Galaxy Outreach</a> </td>
  </tr>
  <tr>
    <td> </em><a href='https://www.acsmeetings.org/tours-workshops#SNP'>SNP/Variant Analysis with Galaxy Workshop</a><em> </td>
  </tr>
  <tr>
    <td> </em><a href='https://www.acsmeetings.org/tours-workshops#A Gentle Intro'>A Gentle Introduction to Cloud Computing: Setting up your own Galaxy Server Workshop</a><em> </td>
  </tr>
  <tr>
    <th> November 19-20 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-1920-november-brisbane/'>Workshop: Extended RNA-Seq analysis</a><em> </td>
    <td> The University of Queensland, Brisbane, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th colspan=4> 2015 </th>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> </strong>2015 Galaxy Community Conference (GCC2015)<strong> </td>
    <td> <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>


# New Papers

*44 papers* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in August, including this one:

* [Using galaxy-P to leverage RNA-Seq for the discovery of novel protein variations](http://bit.ly/1AV6hP5), by Gloria Sheynkman, James Johnson, Pratik Jagtap, *et al.;* *BMC Genomics,* Vol. 15, No. 1. (22 August 2014), 703, doi:10.1186/1471-2164-15-703

* [Integrating UIMA with Alveo, a human communication science virtual laboratory](http://bit.ly/1zkQwhL), by Dominique Estival, Steve Cassidy, Karin Verspoor, Andrew !MacKinlay, and Denis Burnham; *Proceedings of the Workshop on Open Infrastructures and Analysis Frameworks for HLT*, pages 12–22, Dublin, Ireland, August 23rd 2014.

The new papers were tagged in many different areas:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 30 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Galaxy Workflow Developer](http://bit.ly/1t1PKpO), John Innes Centre, Norwich UK. Applications Close: 12 Sept 
* [Computational Science Developer I](http://bit.ly/1lLkAMR), Cold Spring Harbor Laboratory (CSHL), New York, United States 
* [Galaxy Platform Development Officer](http://bit.ly/1q5iXPL), The Genome Analysis Centre (TGAC), Norwich, United Kingdom.  **Closes 18 September.**
* [Bioinformatician position](http://www.xenbase.org/community/job.do?id=633&method=View+Job) at [Xenbase](http://xenbase.org), University of Calgary.
* [Statistical Genomics Postdoc](/src/News/StatisticalGenomicsPostDocPennState/index.md) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/src/GalaxyIsHiring/index.md)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

# New Releases

August was an eventful month for releases.  New versions of [Galaxy](#august-11-2014-galaxy-distribution), [CloudMan](#august-2014-cloudman-release), [BioBlend](#bioblend-051-release), and [blend4j](#blend4j-011-release) were all released.

## August 11, 2014 Galaxy Distribution

<div class='left'><a href='http://getgalaxy.org'><img src='/Images/Logos/GetGalaxyOrg.png' alt=' ' width=150 /></a></div>
<div class='right'><a href='http://wiki.galaxyproject.org/Admin/Tools/Docker'><img src='/Images/Logos/DockerInGalaxyAnnotated.png' alt=' ' width=280 /></a></div>
<br />
**[Complete News Brief](http://wiki.galaxyproject.org/DevNewsBriefs/2014_08_11)**
<br /><br /><br />
*Highlights:* <br />
<br />  • [Security alert](http://tinyurl.com/nhgmbc5) from July 31st, upgrade now
<br />  • [Citations](http://wiki.galaxyproject.org/Admin/Tools/ToolConfigSyntax#A.3Ccitations.3E_tag_set): DOIs, `BibTeX`, and much much more
<br />  • [Docker](http://wiki.galaxyproject.org/Admin/Tools/Docker): You voted, we've *got it*, with a little help from our friends (you!)
<br />  • Significant Workflow, API, Job, Tool Shed, and Dataset management updates
<br />  • Fixes, tunings, plus just a drop of &rarr; Gossip
<br /><br />
<table>
  <tr>
    <td rowspan=3 style=" border: none;"> <a href='http://getgalaxy.org/'><img src='http://galaxy.psu.edu/static/getgalaxy.png' alt='getgalaxy' width=70 /></a> &nbsp;&nbsp; </td>
    <td colspan=2 style=" border: none;"> </strong><a href='http://wiki.galaxyproject.org/Admin/Get%20Galaxy'>getgalaxy.org</a><strong> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong><a href='http://galaxy-dist.readthedocs.org'>galaxy-dist.readthedocs.org</a><strong> </td>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong><a href='http://bitbucket.org/galaxy/galaxy-dist'>bitbucket.org/galaxy/galaxy-dist</a><strong> </td>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong>new<strong>: </td>
    <td style=" border: none;"> <code>$ hg clone https://bitbucket.org/galaxy/galaxy-dist#stable </code> </td>
  </tr>
  <tr>
    <td style=" border: none;"> </strong>upgrade<strong>: </td>
    <td style=" border: none;"> <code>$ hg pull </code> <br /> <code>$ hg update latest_2014.08.11</code> </td>
  </tr>
</table>


## August 2014 CloudMan Release

<div class='right'><a href='/CloudMan'><img src='/Images/Logos/CloudManWideBlackLogo.png' alt='CloudMan' width="300" /></a></div>

This is mostly an incremental bug fix release with the following summary of changes:

* On AWS, updated galaxyFS snapshot (snap-e6e1c04a), which includes the June 2, 2014 Galaxy release with the July 30th security fix. All the tools installed via the Tool Shed have been updated and a number of new tools added, most notably: Tophat2, Bowtie2, FastQC, several FASTQ manipulation tools, several QC tools.
* For AWS, added support for VPC
* For [OpenStack](http://www.openstack.org/) clouds, added the ability to automatically recover worker instances on cluster reboot
* Added support for creating a file system based on a downloadable archive
* Do not run Galaxy with multiple processes by default. This is because Tool Shed installs do not work properly in the multi-process mode. This feature can be enabled by setting user data option `configure_multiple_galaxy_processes` to `True` when launching an instance.
* Set SGE slots in each queue to be equal to the number of cores on the instance
* Set instance IP in the Galaxy's FTP data upload tool message
* Added support for Nginx v1.4 and allow it (with the PAM module) to used as the authentication mechanism when accessing Galaxy Reports app
* Fixed cluster deletion when performed via the API
* No longer automatically start Hadoop and HTCondor services
* On manually-invoked instance reboots, do not increment the instance reboot count that otherwise eventually leads to instance termination
* Limit the size of the log message buffer used in the UI to 1000 lines. Long-running instances had issues with this log growing large and that led to poor UI performance. The complete log is still available from the Admin page (or the command line).
* Automatically delete the bucket/container for Test type (*i.e*, 'SGE only') clusters on cluster termination

For complete details on implemented changes, please see the [source code commits](https://bitbucket.org/galaxy/cloudman/commits/all?search=835%3A903).

CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.

## BioBlend 0.5.1 Release

[BioBlend version 0.5.1](https://pypi.python.org/pypi/bioblend/0.5.1) was released on August 19.  From the [CHANGELOG](https://github.com/afgane/bioblend/blob/master/CHANGELOG.md):

* Fixed url joining problem described in [issue #82](https://github.com/afgane/bioblend/issues/82)
* Enabled [Travis Continuous Integration](https://travis-ci.org/) testing
* Added script to create a user and get its API key
* Deprecated `create_user()` method in favor of clearer `create_remote_user()`. Added `create_local_user()`.
* Skip instead of fail tests when `BIOBLEND_GALAXY_URL` and `BIOBLEND_GALAXY_API_KEY` environment variables are not defined.
* Added export and download to objects API
* Added export/download history
* !GalaxyClient: changed `make_put_request` to return whole `requests` response object
* Added Tool wrapper to `BioBlend.objects` plus methods to list tools and get one
* Added `show_tool()` method to `ToolClient` class
* Added `name`, `in_panel` and `trackster` filters to `get_tools()`
* Added `upload_dataset()` method to `History` class.
* Removed `DataInput` and `Tool` classes for workflow steps. `Tool` is to be used for running single tools.

## blend4j 0.1.1 Release

[blend4j version 0.1.1](http://search.maven.org/#artifactdetails%7Ccom.github.jmchilton.blend4j%7Cblend4j%7C0.1.1%7Cjar) was released on August 27th. Some key features from the [CHANGELOG](https://github.com/jmchilton/blend4j/blob/master/CHANGELOG.md):

* Dataset collection support by Aaron Petkau. Among other things the [histories client](http://jmchilton.github.io/blend4j/apidocs/com/github/jmchilton/blend4j/galaxy/HistoriesClient.html)
  can now [create](http://jmchilton.github.io/blend4j/apidocs/com/github/jmchilton/blend4j/galaxy/HistoriesClient.html#createDatasetCollection(java.lang.String%2C%20com.github.jmchilton.blend4j.galaxy.beans.collection.request.CollectionDescription)) and [return](http://jmchilton.github.io/blend4j/apidocs/com/github/jmchilton/blend4j/galaxy/HistoriesClient.html#showDatasetCollection(java.lang.String%2C%20java.lang.String)) information about collections and the [workflows client](http://jmchilton.github.io/blend4j/apidocs/com/github/jmchilton/blend4j/galaxy/WorkflowsClient.html) can [specify](http://jmchilton.github.io/blend4j/apidocs/com/github/jmchilton/blend4j/galaxy/beans/WorkflowInputs.InputSourceType.html) dataset collections as inputs.
* Documentation overhaul - [API documentation](http://jmchilton.github.io/blend4j/apidocs/) is now available online.
* Update tool shed client defaults to reflect the fact main tool shed is now being served over HTTPS.

# Galaxy IPython

<div class='right'><a href='http://ipython.org/'><img src='/Images/Logos/IPythonSqaure.png' alt='IPython'  /></a></div>

*And ...* Björn Grüning and Eric Rasche also released **[Galaxy IPython](https://github.com/bgruening/galaxy-ipython)**:

<div class='indent'>

We proudly present the first release of the [Galaxy IPython](https://github.com/bgruening/galaxy-ipython) project.

Galaxy IPython is a visualization plugin which should enable Galaxy users with coding skills to easily process their data in the most flexible way. With this plugin, it is possible to analyse and post-process data without downloading datasets or entire histories. One of our aims was to make Galaxy more attractive and accessible to bioinformaticians and programmers, and we hope that this project will build some bridges.

Disclaimer: Even though the [Ipython](http://ipython.org/) notebooks can be stored and reused, this plugin will break the Galaxy philosophy of reproducibility, I feel personally bad about that, but I also think it is a great opportunity to get more bioinformaticians into Galaxy, and to get Galaxy used more often as a teaching resource. By being able to teach not only about workflows but also about data analysis tasks often necessary with Bioinformatics, Galaxy will be significantly more useful in teaching environments.

Keep in mind to write a nice Tool Shed Tool if you catch yourself using IPython in Galaxy to often for the same task.

<div class='right'><a href='https://www.youtube.com/watch?v=jQDyTuYnn1k'><img src='/Images/Screenshots/IPythonVideo.png' alt='Galaxy IPython Screencast' width="200" /></a>
<div class='center'>[Screencast](https://www.youtube.com/watch?v=jQDyTuYnn1k)</div>
</div>

A few features we have up and running:
* Use IPython directly in the main window or in the Scratchbook
* Completely encapsulated IPython environment with matplotlib, biopython, pandas and friends already installed.
* IPython runs completely self-contained within a [Docker](https://www.docker.com/) container, separate from your Galaxy data
* Easy access to datasets from your current history via pre-defined IPython functions
* Manipulate and plot data as you like and export your new files back into your Galaxy history
* Save IPython Notebooks across analysis sessions in your Galaxy history with the click of a button.
* View saved IPython Notebooks directly in HTML format, or re-open them to continue your analysis.
* Self-closing and self-cleaning IPython Docker container
* Notebooks are secure, only accessible to the intended user

Please follow the installation instruction on our [project page](https://github.com/bgruening/galaxy-ipython).

The underlying IPython Notebook (+Galaxy sugar) is stored at [Github](https://github.com/bgruening/docker-ipython-notebook) and the [Docker Registry](https://registry.hub.docker.com/u/bgruening/docker-ipython-notebook/).

You can also install a ipynb datatype:
* [Github](https://github.com/bgruening/galaxytools/tree/master/datatypes/json)
* [Test ToolShed](https://testtoolshed.g2.bx.psu.edu/view/iuc/datatyp_ipynb)

Comments welcome!<br />
Happy research!<br />
Eric & Björn
</div>



# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/Community/Logs'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a></div>
<div class='right'><a href='/Community/Deployments'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style=" class="blue"  text-align: center;"> &nbsp;&nbsp; <a href='/Community/Logs#add-a-log-page'>Share your experience</a> </em><a href='/Community/Logs#add-a-log-page'>now</a> <em> &nbsp;&nbsp;<br /><a href='/Community/Deployments#add-your-galaxy-deployment'>Describe your instance</a> </em><a href='/Community/Deployments#add-your-galaxy-deployment'>now</a><em> </td>
  </tr>
</table>

<br /><br />

**There were no new [Log Board](/src/Community/Logs/index.md) or [Deployment Catalog](/src/Community/Deployments/index.md) entries in August!**  Eek!  Please don't let this happen again!  

The *[Community Log Board](/src/Community/Logs/index.md)* and  *[Deployment Catalog](/src/Community/Deployments/index.md)* *[Galaxy community hubs](/Community)* were launched last your.  If you have a [Galaxy deployment](/src/Community/Deployments/index.md), or [experience](/src/Community/Logs/index.md) you want to share then please publish them **this month.**

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

## Galaxy Project ToolShed Repos

Here are new contributions for the past *two* months.

In no particular order: 

### Tools

* *From [lparsons](https://toolshed.g2.bx.psu.edu/view/lparsons/)*
  * [sam_stats](https://toolshed.g2.bx.psu.edu/view/lparsons/sam_stats): generates basic sam/bam stats (from [ea-utils package](https://code.google.com/p/ea-utils/))
  * [fastq_join](https://toolshed.g2.bx.psu.edu/view/lparsons/fastq_join): merge overlapping paired-end reads (from ea-utils package)

* *From [saskia-hiltemann](https://toolshed.g2.bx.psu.edu/view/saskia-hiltemann/)*
  * [ireport](https://toolshed.g2.bx.psu.edu/view/saskia-hiltemann/ireport): create interactive HTML reports from galaxy outputs. 

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc)*
  * [stringtie](https://toolshed.g2.bx.psu.edu/view/iuc/stringtie): fast and highly efficient assembler of RNA-Seq alignments into potential transcripts.
  * [vt_variant_tools](https://toolshed.g2.bx.psu.edu/view/iuc/vt_variant_tools): VT: a variant tool set that discovers short variants from Next Generation Sequencing data.
  * [bcftools](https://toolshed.g2.bx.psu.edu/view/iuc/bcftools): Utilities for variant calling and manipulating VCFs and BCFs
  * [gemini](https://toolshed.g2.bx.psu.edu/view/iuc/gemini): GEMINI: a flexible framework for exploring genome variation

* *From [urgi-team](https://toolshed.g2.bx.psu.edu/view/urgi-team/)*
  * [tandem_repeats_finder](https://toolshed.g2.bx.psu.edu/view/urgi-team/tandem_repeats_finder): locate and display tandem repeats in DNA sequences

* *From [bgruening](https://toolshed.g2.bx.psu.edu/view/bgruening)*
  * [antismash](https://toolshed.g2.bx.psu.edu/view/bgruening/antismash): AntiSMASH - rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters

* *From [agordon](https://toolshed.g2.bx.psu.edu/view/agordon/):*
  * [datamash_wrapper](https://toolshed.g2.bx.psu.edu/view/agordon/datamash_wrapper): grouping and summarizing tool on tabular data files

* *From [david-hoover](https://toolshed.g2.bx.psu.edu/view/david-hoover/):*
  * [mirdeep2_and_targetspy_dh](https://toolshed.g2.bx.psu.edu/view/david-hoover/mirdeep2_and_targetspy_dh): Finding miRNA in NGS data and finding the targets for those miRNA

* *From [big-tiandm](https://toolshed.g2.bx.psu.edu/view/big-tiandm/):*
  * [mirplant2](https://toolshed.g2.bx.psu.edu/view/big-tiandm/mirplant2): plant microRNA analysis tools

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam/):*
  * [correlation](https://toolshed.g2.bx.psu.edu/view/devteam/correlation): computes the matrix of correlation coefficients between numeric columns.
  * [vcf2pgsnp](https://toolshed.g2.bx.psu.edu/view/devteam/vcf2pgsnp): Convert from VCF to pgSnp format
  * [pgsnp2gd_snp](https://toolshed.g2.bx.psu.edu/view/devteam/pgsnp2gd_snp): onverts a pgSnp dataset to gd_snp format, either starting a new dataset or appending to an old one.
  * [ucsc_custom_track](https://toolshed.g2.bx.psu.edu/view/devteam/ucsc_custom_track): Build custom track for UCSC genome browser
  * [dna_filtering](https://toolshed.g2.bx.psu.edu/view/devteam/dna_filtering): Filter on ambiguities in polymorphism datasets
  * [mine](https://toolshed.g2.bx.psu.edu/view/devteam/mine): Applies the Maximal Information-based Nonparametric Exploration strategy to an input dataset.
  * [pearson_correlation](https://toolshed.g2.bx.psu.edu/view/devteam/pearson_correlation): Computes Pearson's correlation coefficient between any two numerical columns. Column numbers start at 1.
  * [generate_pc_lda_matrix](https://toolshed.g2.bx.psu.edu/view/devteam/generate_pc_lda_matrix): generate a matrix to be used for running the Linear Discriminant Analysis as described in Carrel et al., 2006 (PMID: 17009873)
  * [count_gff_features](https://toolshed.g2.bx.psu.edu/view/devteam/count_gff_features): Counts the number of features in a GFF dataset.
  * [column_maker](https://toolshed.g2.bx.psu.edu/view/devteam/column_maker): computes an expression for every row of a dataset and appends the result as a new column (field).
  * [scatterplot](https://toolshed.g2.bx.psu.edu/view/devteam/scatterplot): creates a simple scatter plot between two variables containing numeric values of a selected dataset. 
  * [plot_from_lda](https://toolshed.g2.bx.psu.edu/view/devteam/plot_from_lda): generates a Receiver Operating Characteristic (ROC) plot that shows LDA classification success rates for different values of the tuning parameter tau as Figure 3 in Carrel et al., 2006 (PMID: 17009873).
  * [histogram](https://toolshed.g2.bx.psu.edu/view/devteam/histogram): computes a histogram of the numerical values in a column of a dataset.
  * [lda_analysis](https://toolshed.g2.bx.psu.edu/view/devteam/lda_analysis): Perform Linear Discriminant Analysis
  * [snpfreq](https://toolshed.g2.bx.psu.edu/view/devteam/snpfreq): basic analysis of bi-allelic SNPs in case-control data, using the R statistical environment and Fisher's exact test to identify SNPs with a significant difference in the allele frequencies between the two groups

* *From [arkarachai-fungtammasan](https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/):*
  * [microsatellite_ngs](https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/microsatellite_ngs): Pipeline to profile and genotype microsatellites from short read data

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/peterjc):*
  * [blast_rbh](https://toolshed.g2.bx.psu.edu/view/peterjc/blast_rbh): BLAST Reciprocal Best Hits (RBH) from two FASTA files
   
* *From [nikos](https://toolshed.g2.bx.psu.edu/view/nikos/):*
  * [bigwig_to_wig](https://toolshed.g2.bx.psu.edu/view/nikos/bigwig_to_wig): Converts a bigWig file to Wiggle (WIG) format

* *From [mcharles](https://toolshed.g2.bx.psu.edu/view/mcharles/):
  * [genephys](https://toolshed.g2.bx.psu.edu/view/mcharles/genephys): 

* *From [alan-blakely](https://toolshed.g2.bx.psu.edu/view/alan-blakely/):
  * [filter_by_substring_match](https://toolshed.g2.bx.psu.edu/view/alan-blakely/filter_by_substring_match): Allows for partial sequences to match lines containing a larger, more complete sequence.

* *From [anton](https://toolshed.g2.bx.psu.edu/view/anton/):*
  * [bamtools_filter](https://toolshed.g2.bx.psu.edu/view/anton/bamtools_filter): Filter BAM files on a variety of attributes


### Data Managers

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc/):*
  * [data_manager_gemini_database_downloader](https://toolshed.g2.bx.psu.edu/view/iuc/data_manager_gemini_database_downloader): Manage GEMINI databases. This tool will retrieve all files for the use in GEMINI.


### Suites

* *From [anton](https://toolshed.g2.bx.psu.edu/view/anton)*
  * [suite_vcflib_tools_2_0](https://toolshed.g2.bx.psu.edu/view/anton/suite_vcflib_tools_2_0): tools for manipulation of VCF files


### Packages / Tool Dependency Definitions

* *From [iracooke](https://toolshed.g2.bx.psu.edu/view/iracooke)*
  * [package_peptideshaker_0_31](https://toolshed.g2.bx.psu.edu/view/iracooke/package_peptideshaker_0_31): Installs tool dependencies for !PeptideShaker 0.31
  * [package_searchgui_1_19](https://toolshed.g2.bx.psu.edu/view/iracooke/package_searchgui_1_19): Installs tool dependencies for !SearchGUI 1.19

* *From [[https://toolshed.g2.bx.psu.edu/view/anton|]]*
  * [vcflib_86723982aa](https://toolshed.g2.bx.psu.edu/view/anton/vcflib_86723982aa): Compiled binary files for vcflib toolkit 86723982aa
  * [package_bamtools](https://toolshed.g2.bx.psu.edu/view/anton/package_bamtools): bamtools - a collection of utilities for processing of bam files

* *From [lparsons](https://toolshed.g2.bx.psu.edu/view/lparsons/)*
  * [package_ea_utils_1_1_2_484](https://toolshed.g2.bx.psu.edu/view/lparsons/package_ea_utils_1_1_2_484): Command-line tools for processing biological sequencing data. Barcode demultiplexing, adapter trimming, etc. See [ea-utils](https://code.google.com/p/ea-utils/).
  * [package_rseqc_2_3_9](https://toolshed.g2.bx.psu.edu/view/lparsons/package_rseqc_2_3_9): downloads and compiles version 2.3.9 of RSeQC.

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc/)*
  * [package_stringtie_0_97](https://toolshed.g2.bx.psu.edu/view/iuc/package_stringtie_0_97): 
  * [package_vcftools_0_1_12b](https://toolshed.g2.bx.psu.edu/view/iuc/package_vcftools_0_1_12b): downloads and compiles version 0.1.12b of the vcftools suite.
  * [package_vt_5c735ab14b5603d9f14da6ee0e63d86ba3779934](https://toolshed.g2.bx.psu.edu/view/iuc/package_vt_5c735ab14b5603d9f14da6ee0e63d86ba3779934): a variant tool set that discovers short variants from Next Generation Sequencing data.
  * [package_htseq_0_6](https://toolshed.g2.bx.psu.edu/view/iuc/package_htseq_0_6): downloads and compiles version 0.6 of the htseq package.
  * [package_pandas_0_14](https://toolshed.g2.bx.psu.edu/view/iuc/package_pandas_0_14): downloads and compiles version 0.14 of the python library [pandas](http://pandas.pydata.org/), an open source, BSD-licensed library providing high-performance, easy-to-use data structures and data analysis tools for the Python programming language.
  * [package_bedtools_2_19](https://toolshed.g2.bx.psu.edu/view/iuc/package_bedtools_2_19): downloads and compiles version 2.19 of bedtools, a swiss army knife for genome arithmetic.
  * [package_grabix_0_1_3](https://toolshed.g2.bx.psu.edu/view/iuc/package_grabix_0_1_3): downloads and compiles version 0.1.3 of grabix. grabix leverages the fantastic BGZF library in samtools to provide random access into text files that have been compressed with bgzip
  * [package_gemini_0_10_0](https://toolshed.g2.bx.psu.edu/view/iuc/package_gemini_0_10_0): downloads and compiles version 0.10.0 of GEMINI. GEMINI (GEnome MINIng) is designed to be a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome.
  * [package_r_ggplot2_0_9_3](https://toolshed.g2.bx.psu.edu/view/iuc/package_r_ggplot2_0_9_3): downloads and compiles verion 0.9.3.x from gglot2 the R package.  ggplot2 is a plotting system for R, based on the grammar of graphics, which tries to take the good parts of base and lattice graphics and none of the bad parts.

* *From [jjohnson](https://toolshed.g2.bx.psu.edu/view/jjohnson/):*
  * [package_shear_0_2_11](https://toolshed.g2.bx.psu.edu/view/jjohnson/package_shear_0_2_11): Download and install SHEAR version 0.2.11 (Sample Heterogeneity Estimation and Assembly by Reference)

* *From [agordon](https://toolshed.g2.bx.psu.edu/view/agordon/):*
  * [package_datamash_1_0_5](https://toolshed.g2.bx.psu.edu/view/agordon/package_datamash_1_0_5): grouping and summarizing tool on tabular data files

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam/):*
  * [package_mine_1_0_1](https://toolshed.g2.bx.psu.edu/view/devteam/package_mine_1_0_1): downloads and installs version 1.0.1 of the MINE .jar package.


# Other News

* [Basic variant calling in Galaxy](http://bit.ly/V7Efj6) online tutorial by Cynthia Gibas at Genome Intelligence 
* From ron Horst:
    You can now BLAST search DNA & Proteins on [galaxy-qld.genome.edu.au](http://galaxy-qld.genome.edu.au), using BRAEMBL resources. Coming soon to more Gernomics Virtual Lab VMs
