---
title: New Paper "Biomolecular Reaction and Interaction Dynamics Global Environment
  (BRIDGE)
date: '2019-03-15'
tags: [paper]
doi: 10.1093/bioinformatics/btz107
supporters:
- denbi
- elixir
- ViCE
subsites: [eu, freiburg]
main_subsite: freiburg
---

In collaboration with our cooperation partner at the Scientific Computing Research Unit, University of Cape Town, we have recently published ["Biomolecular Reaction and Interaction Dynamics Global Environment (BRIDGE)"](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz107/5317160). This publication provides a basis for performing molecular dynamics simulations and analysis through Galaxy, using a number of commonly used packages, such as NAMD, GROMACS and CHARMM (for simulation) and Bio3D and MDAnalysis (for analysis).

Concurrently, we are publishing a new flavor of Galaxy for computational chemistry (located [here](https://cheminformatics.usegalaxy.eu/)) which we will continue to develop by adding further molecular dynamics and cheminformatics tools. We hope you find it useful - any feedback is appreciated!

## Abstract

>**Motivation**
>
>The pathway from genomics through proteomics and onto a molecular description of biochemical processes makes the discovery of drugs and biomaterials possible. A research framework common to genomics and proteomics is needed to conduct biomolecular simulations that will connect biological data to the dynamic molecular mechanisms of enzymes and proteins. Novice biomolecular modelers are faced with the daunting task of complex setups and a myriad of possible choices preventing their use of molecular simulations and their ability to conduct reliable and reproducible computations that can be shared with collaborators and verified for procedural accuracy.
>
>**Results**
>
>We present the foundations of Biomolecular Reaction and Interaction Dynamics Global Environment (BRIDGE) developed on the Galaxy platform that makes possible fundamental molecular dynamics of proteins through workflows and pipelines via commonly used packages, such as NAMD, GROMACS and CHARMM. BRIDGE can be used to set up and simulate biological macromolecules, perform conformational analysis from trajectory data and conduct data analytics of large scale protein motions using statistical rigor. We illustrate the basic BRIDGE simulation and analytics capabilities on a previously reported CBH1 protein simulation.
>
>**Availability and implementation**
>
>Publicly available at https://github.com/scientificomputing/BRIDGE and https://usegalaxy.eu
>
>**Supplementary information**
>
>Supplementary data are available at Bioinformatics online.


