<div class="title">March 2016 Galaxy News</div>

<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em"> TABLE_OF_CONTENTS </td>
  </tr>
</table>

</div>

<div class='left'><a href='/GalaxyUpdates.md'><img src='/Images/GalaxyLogos/GalaxyNews.png' alt='Galaxy News' width=150 /></a></div>

Welcome to the March 2016 *[Galactic News](/GalaxyUpdates)*, a summary of what is going on in the Galaxy community. 

If you have anything to include in the next News, please send it to [outreach AT galaxyproject DOT org](Galaxy Outreach). 


# GCC2016

<div class='left'>
<a href='https://gcc2016.iu.edu/'><img src='https://gcc2016.iu.edu/img/gcc2016-logo.png' alt='2016 Galaxy Community Conference' width="250" /></a>
</div>

[GCC2016](http://gcc2016.iu.edu/) will be held June 25-29 at [Indiana University](http://indiana.edu/) in Bloomington, Indiana, United States. This will be the 7th annual gathering of the Galaxy community, and we are expecting over 200 participants again this year.  The 2016 Galaxy Community Conference includes 2 days of hackathons, [2 days of training](https://gcc16.sched.org/grid-full#2016-06-26), and a two day meeting featuring accepted presentations, keynotes, poster sessions, the new Visualization Showcase and Software Demo sessions, lightning talks, birds-of-a-feather meetups, and plenty of networking. 

## GCC2016 Early Registration

<div class='right'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='https://gcc2016.iu.edu/registration/'>Register Now</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

**[Early registration for GCC2016](https://gcc2016.iu.edu/registration/) is now open**. 
Registration costs depend on which events you register for, your career stage & affiliation, and when you register.  Early bird registration ends May 20 and is up to 40% less than regular registration rates.  Early bird registration starts at less than $45 / day for students and postdocs, and at $65 / day for other attendees from non-profits.

You can also sign up for [conference housing](https://gcc2016.iu.edu/location) during registration.  

You are strongly encouraged to review the [training](https://gcc16.sched.org/grid-full#2016-06-26) and [housing](https://gcc2016.iu.edu/location) options before beginning the registration process.

## GCC2016 Abstract Submission Deadlines!

<div class='left'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://bit.ly/gcc2016abssub'>Submit an abstract</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

**The deadline for [oral presentation abstracts](http://bit.ly/gcc2016abssub) is March 25, 2016**.  Abstracts will be reviewed and submitters will be notified of acceptance status no later than April 11, 2016.

**[Poster and Demo abstract submission](http://bit.ly/gcc2016abssub) closes May 20, 2016** (or earlier depending on poster space availability).  Poster and demo submissions are reviewed on a rolling basis, and submitters will be notified of acceptance status no later than two weeks after the abstract is submitted.  You may submit similar content for oral presentations, posters, and demos. 

Topics should be of interest to those working in high-throughput data analysis and research. Presentations that are Galaxy-centric are encouraged, but not required.  Please see the [abstracts page](https://gcc2016.iu.edu/abstracts/index.php) for full details.

## Scholarships: International application deadline is March 20

<div class='right'>
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='https://docs.google.com/forms/d/1IbMoNcA6z31a8hvqjSdfQRIQ92wo-HxXvA5uk0UilnQ/viewform'>Submit an application</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

We are pleased to offer [scholarships](https://docs.google.com/forms/d/1IbMoNcA6z31a8hvqjSdfQRIQ92wo-HxXvA5uk0UilnQ/viewform) for the [2016 Galaxy Community Conference](http://gcc2016.iu.edu/), being held in Bloomington, Indiana, United States, June 25-29.  Scholarships are available to students and post-docs in historically under-represented groups, and to those from or based in Low and Lower-Middle Income Economies, [as defined by the World Bank](http://bit.ly/1nO82wD).  If this describes you or one of your students then we hope to receive an application.

Scholarships cover [registration](http://gcc2016.iu.edu/registration) and [lodging](http://gcc2016.iu.edu/location) during the GCC Meeting, and for any Training or Hackathon events the applicant chooses to attend.  Scholarships do not cover travel or other expenses. **The application deadline is May 1 for members of historically underrepresented groups and March 20 for those from Low and Lower-Middle Income Economies.**

See the [full announcement](/News/GCC2016Scholarships) for details.


## Sponsors

<div class='left'><br /><a href='http://bioteam.net/'><img src='/Images/Logos/BioTeamLogo154.gif.md' alt='BioTeam' width="130" /></a></div>

We are pleased to announce that **[BioTeam](http://bioteam.net)** will again be a [sponsor](https://gcc2016.iu.edu/sponsors) for this annual event.  !BioTeam is a professional services and products company at the intersection of science and information technology. !BioTeam solves analytics, big data and scientific computing challenges for organizations by mapping the right technologies to their unique scientific needs. !BioTeam has a well-established history of providing forward-thinking solutions to the Life Sciences.   

The **[BioTeam Appliance Galaxy Edition](http://bioteam.net/bioteam-appliance/galaxy-edition/)** is a push-button solution that let’s researchers get up and running quickly with Galaxy.  The Galaxy Appliance comes preinstalled with a production instance of Galaxy, bioinformatics tools, and reference datasets. This powerful system is specifically configured for computationally intensive scientific workloads. Most importantly, the Galaxy Appliance is an open system so researchers can install whatever tools they need and use the server as their own high-performance informatics infrastructure along side Galaxy.  !BioTeam provides ongoing support for the Galaxy Appliance, enabling researchers to minimize their IT burden. The Galaxy Appliance is used by researchers around the world for metagenomic, ChIP-Seq, RNA-Seq analysis and more.  For more information:  http://bioteam.net/bioteam-appliance/galaxy-edition/

This is the fourth year in a row that !BioTeam has been a GCC Sponsor.



We continue to seek other sponsors as well and [offer a wide range of sponsorship plans](https://gcc2016.iu.edu/sponsors/index.php). If your organization is interested in having a presence at GCC2016, please contact the [gcc2016-exec@lists.galaxyproject.org](GCC2016 Exec) for more information.

----

# New Papers

54 new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in February, bringing the total number of papers to over 3000. Highlights from last month include:

* [MetaPalette: A $k$-mer painting approach for metagenomic taxonomic profiling and quantification of novel strain variation](http://arxiv.org/abs/1602.05328)
    David Koslicki, Daniel Falush, arXiv:1602.05328 [q-bio.GN] (17 Feb 2016)

* [Colib’read on galaxy: a tools suite dedicated to biological information extraction from raw NGS reads](http://dx.doi.org/10.1186/s13742-015-0105-2)
    by Yvan Le Bras, Olivier Collin, Cyril Monjeaud, Vincent Lacroix, Éric Rivals, Claire Lemaitre, Vincent Miele, Gustavo Sacomoto, Camille Marchet, Bastien Cazaux, Amal Zine El Aabidine, Leena Salmela, Susete Alves-Carvalho, Alexan Andrieux, Raluca Uricaru and Pierre Peterlongo, *!GigaScience*, Vol. 5, No. 1. (11 February 2016), doi:10.1186/s13742-015-0105-2

* [Impact of soil heat on reassembly of bacterial communities in the rhizosphere microbiome and plant disease suppression](http://dx.doi.org/10.1111/ele.12567)
    Menno van der Voort, Marcel Kempenaar, Marc van Driel, Jos M. Raaijmakers, Rodrigo Mendes, *Ecology Letters* (January 2016), pp. n/a-n/a, doi:10.1111/ele.12567

* [RiboGalaxy: a browser based platform for the alignment, analysis and visualization of ribosome profiling data](http://dx.doi.org/10.1080/15476286.2016.1141862)
    Audrey M. Michel, James P. A. Mullan, Vimalkumar Velayudhan, Patrick B. F. O'Connor, Claire A. Donohue, Pavel V. Baranov, *RNA Biology* (29 January 2016), pp. 00-00, doi:10.1080/15476286.2016.1141862


The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 29 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 11 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


The [Galaxy CiteULike group](http://www.citeulike.org/group/16008/) surpassed 3000 pubs in February. 49% of them use Galaxy in their methods.  

----

# Events

## March 7-8: Online IUC Contribution Fest - RADSeq Tools and Workflows

<div class='left'>
<a href='https://github.com/galaxyproject/tools-iuc/issues/422'><img src='/Images/Logos/RADSeqHack.png' alt='remote Contribution Fest will be held 7th and 8th of March for developers to work on Galaxy RADSeq tools' width="200" /></a>
</div>

A [remote Contribution Fest will be held 7th and 8th of March for developers to work on Galaxy RADSeq tools](https://github.com/galaxyproject/tools-iuc/issues/422).

RADSeq is a cheap sequencing technology that is used by many resource-limited groups who would benefit a lot from easy-to-use galaxy tools. Indeed there has been quite some interest in analyzing RADSeq with Galaxy. Currently there is a wrapper for stacks and little more to help with RAD specific analysis (though many other galaxy tools are useful with RADs - bwa, cap3, gatk, velvet, ...).

If you are interested in participating in the hackathon but not interested in actual tool development - we will assemble a list of smaller, manageable Python and !JavaScript tasks to work on and certainly documentation is a chronically lacking for collections so we could use help there and no actual coding would be required.

We encourage ideas or advice about how to organize this so please let us know. A core group will be available on IRC all day and we will have google hangouts across those days to organize, answer questions, and report progress.

We will do our best to coordinate and make this hackathon a nice and productive experience and we would like to especially focus on working reasonable hours and discourage overnighters.

**All forms of contribution are welcome!**

## GMOD Meeting, June 30 - July 1

<div class='right'>
<a href='/gmod:Jun_2016_GMOD_Meeting.md'><img src='/Images/Logos/GMODLogo.png' alt='2016 GMOD meeting' width="160" /></a>
</div>

The [2016 GMOD meeting](http://gmod.org/wiki/Jun_2016_GMOD_Meeting) will be held **immediately following GCC2016 and in the same venue as GCC2016**.  [http://gmod.org/wiki/|GMOD](http://gmod.org/wiki/|GMOD) is a consortium of open-source software project, including Galaxy, that address common challenges with biological data.  Other projects in the GMOD consortium include 
* [Tripal](http://gmod.org/wiki/Tripal): A web front end for [Chado](http://gmod.org/wiki/Chado) databases.  Galaxy is working with the Tripal project to make Galaxy be Tripal's analysis engine.
* [JBrowse](http://gmod.org/wiki/JBrowse): A client-side genome browser and successor to the venerable [http://gmod.org/wiki/GBrowse](http://gmod.org/wiki/GBrowse).  [JBrowse as a Galaxy Tool](http://gcc2015.tsl.ac.uk/Lightning/#JBrowse_as_a_Galaxy_Tool) was presented by Eric Rasche at [GCC2015](http://gcc2015.tsl.ac.uk/). Ian Holmes, the JBrowse PI, has put JBrowse-Galaxy integration  at "top of the list" for JBrowse's infrastructure upcoming infrastructure work.
* [MAKER](http://gmod.org/wiki/MAKER): A genome annotation pipeline that integrates several gene annotation engines, and combines them to produce annotation that is better than any individual tool produces.  A [MAKER-Galaxy](ATTACHMENT_URLDocuments/Posters/ISMB2014_Kandalaft_GenomeAnnotation.pdf) by Agriculture and Agri-Food Canada was presented at ISMB 2014.
* [InterMine](http://gmod.org/wiki/InterMine) and [BioMart](http://gmod.org/wiki/BioMart): These are both popular data sources that are integrated with Galaxy.

Early bird registration ends May 21. For those who would like to present a talk or poster, the [meeting registration form](https://gmod2016.eventbrite.com/) includes a section for submitting the presentation title and abstract. *Note: GCC2016 and GMOD have separate event registration, but are sharing housing - you won't even have to change rooms.*

## Galaxy in Google Summer of Code

<div class='left'><a href='http://gmod.org/wiki/GSOC_Project_Ideas_2016'><img src='/Images/Logos/GSoC_2016_logo.png' alt='Galaxy in GSoC 2016 as part of the Open Genome Informatics group' width="100" /></a>
</div>

We are delighted to announce that Galaxy will be participating in [Google Summer of Code](https://summerofcode.withgoogle.com/) this year, along with [http://gmod.org/wiki/|GMOD](http://gmod.org/wiki/|GMOD) and [Reactome](http://reactome.org), as part of the [Open Genome Informatics](http://gmod.org/wiki/GSOC_Project_Ideas_2016) initiative.

The current list of project ideas is [on this project ideas page](http://gmod.org/wiki/GSOC_Project_Ideas_2016).  There is a bit more time to firm up and add new project ideas. It is also possible for students to suggest their own ideas and we will try hard to find them a mentor. If you add (or have) new ideas to the proposal, please let our GSoC coordinators [help@reactome.org](Robin Haw), [help@gmod.org](Scott Cain), and [Marc Gillespie](http://www.stjohns.edu/academics/bio/marc-gillespie) know.

Between now and 13 March, would-be student participants will start to discuss application ideas with this group.
Please let Robin know if you have any questions about GSoC.

<div class='right'><br />
<a href='/Community/GalaxyAdmins/Meetups/2016_02_18.md'><img src='/Images/NewsGraphics/AnsibleWheels.png' alt='February 2016 GalaxyAdmins meetup are now available' width="150" /></a>
</div>

## February GalaxyAdmins Slides & Video

Slides and video from the [February 2016 GalaxyAdmins meetup are now available](/Community/GalaxyAdmins/Meetups/2016_02_18).



## Upcoming Events

There are [many upcoming events](/Events) in the next few months.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='center'> 
<a href='https://github.com/galaxyproject/tools-iuc/issues/422'><img src='/Images/Logos/IUC_RADSeq_Hack.png' alt='IUC Contribution Fest - RADSeq Tools and Workflows' height="80" /></a>
<a href='http://migale.jouy.inra.fr/?q=formations'><img src='/Images/Logos/MIGALELogo.png' alt='Le cycle "Bioinformatique par la pratique" 2016' height="50" /></a> 
<a href='http://go4bioinformatics.genouest.org/analyse-de-donnees-rnaseq-sous-galaxy/'><img src='/Images/Logos/eBiogenouestLogo.png' alt='Introduction à l'analyse de données RNAseq sous Galaxy' height="80" /></a> &nbsp;
<a href='http://icmexico2016.soibio.org/product/icwmexico2016-galaxy-workshop/'><img src='/Images/Logos/SolBio2016.png' alt='Workshop on Galaxy: Practical Course' height="60" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> March 7-8 </th>
    <td> </strong><a href='https://github.com/galaxyproject/tools-iuc/issues/422'>IUC Contribution Fest - RADSeq Tools and Workflows</a><strong> </td>
    <td> INCLUDE(/Events/Badges/World) <em>Online</em> </td>
    <td> <a href='/IUC.md'>/IUC.md</a> </td>
  </tr>
  <tr>
    <th> March 15 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module17-15mars.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> March 16 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module8bis.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> March 17 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module18bis.pdf'>Traitement bioinformatique des données RNA-Seq sous Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> March 29 </th>
    <td> <em><a href='http://go4bioinformatics.genouest.org/analyse-de-donnees-rnaseq-sous-galaxy/'>Introduction à l'analyse de données RNAseq sous Galaxy</a></em> </td>
    <td> <div class='right'><a href='/Teach/Trainers.md#galaxy_user_group_grand_ouest_28guggo29-bioinformatics'><img src='/Images/Icons/Europe40.png' alt='' /></a></div><a href='https://www.e-biogenouest.org/groups/guggo'>Galaxy User Group Grand Ouest (GUGGO), Rennes, France</a> </td>
    <td> <div class='right'><a href='/Teach/Trainers.md#galaxy_user_group_grand_ouest_28guggo29-bioinformatics'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div><a href='https://www.e-biogenouest.org/members/1003'>Yvan Le Bras</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> April 6 </th>
    <td> <em><a href='http://www.bio-itworldexpo.com/data-visualization/'>Approaches for the Integration of Visual and Computational Analysis of Biomedical Data</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.bio-itworldexpo.com/'>BioIT World 2016</a>, Boston, Massachusetts, United States </td>
    <td> Nils Gehlenborg </td>
  </tr>
  <tr>
    <th> April 22-26 </th>
    <td> <em><a href='http://icmexico2016.soibio.org/product/icwmexico2016-galaxy-workshop/'>Workshop on Galaxy: Practical Course</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://icmexico2016.soibio.org/'>SolBio International Conference 2016</a>, Riviera Maya, México </td>
    <td> <a href='http://www.ac.uma.es/~ots/'>Oswaldo Trelles</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> May 11-13 </th>
    <td> <em><a href='http://www.canceropole-idf.fr/formation-ngs-rnaseq'>NGS & Cancer : Analyses RNA-Seq</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Paris, France </td>
    <td> <a href="mailto:communication@canceropole-idf.fr">Cancéropôle Île-de-France</a> </td>
  </tr>
  <tr>
    <th> May 23-24 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module16.pdf'>Analyse statistique de données RNA-Seq sous Galaxy -Recherche des régions d'intérêt différentiellement exprimées</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> May 30 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module17-15mars.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> May 31 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2016/module8bis-31mai.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
    <td> INCLUDE(/Events/Badges/Europe) Part of <a href='http://migale.jouy.inra.fr/?q=formations'>Le cycle "Bioinformatique par la pratique" 2016</a>, Jouy-en-Josas, France </td>
    <td> <a href="mailto:formation.migale AT jouy.inra.fr">Formation Migale</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef"> </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> <em><a href='http://bioinformatics.ca/workshops/2016/informatics-high-throughput-sequencing-data-2016'>Informatics on High-throughput Sequencing Data</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Montreal, Quebec, Canada </td>
    <td> INCLUDE(/Teach/GTN/Badge16)  <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 13-17 </th>
    <td> <em><a href='http://bioinformatics.ucdavis.edu/2016-bioinformatics-training-program-preview/'>Using Galaxy For Analysis of High Throughput Sequence Data</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) Davis, California, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16) <a href="mailto:training.bioinformatics@ucdavis.edu">UC Davis Bioinformatics Training Program</a> </td>
  </tr>
  <tr>
    <th> June 25-29 </th>
    <td> </strong><em><a href='https://gcc2016.iu.edu/'>2016 Galaxy Community Conference (GCC2016)</a><strong></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> INCLUDE(/Teach/GTN/Badge16)  <a href='https://gcc2016.iu.edu/organizers'>Organizers</a> </td>
  </tr>
  <tr>
    <th> June 30 - July 1 </th>
    <td> <em><a href='http://gmod.org/wiki/Jun_2016_GMOD_Meeting'>GMOD Meeting</a></em> </td>
    <td> INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.iu.edu/'>Indiana University</a>, Bloomington, Indiana, United States </td>
    <td> <a href='http://gmod.org/wiki/Scott'>Scott Cain</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none"> <a href='/Teach/Trainers.md'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none"> <span style="font-size: smaller;"> <a href='/Teach/Trainers.md'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'>![Please Help! Yes you!](/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png)</div>

The Galaxy is expanding! Please help it grow.
* [Bioinformatics Web Application Developer-Biology](http://bit.ly/1LQQ2qz), Washington University in St. Louis, Missouri, United States
* [Software developer and Post-docs](http://gehlenborg.com/open-positions/), Gehlenborg Lab, Harvard Medical School, Boston, Massachusetts, United States
* [Postdoctoral Research Positions](http://bit.ly/1KYf1rg), Molecular and Cellular Biology Department at Baylor College of Medicine, Houston, Texas, United States 
* [Bioinformaticist](/News/JHUAPLBioinformaticist), Johns Hopkins University Applied Physics Laboratory (APL)
* [Software Engineer](/News/SoftwareEngineerOHSU), Oregon Health Sciences University, Portland, Oregon, United States

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />



----

# New Public Galaxy Servers

[Three new publicly accessible Galaxy servers](/PublicGalaxyServers) were listed in February:

## BISTRO

<div class='right'><div class='solid'><a href='http://galaxy.bioinfo-bistro.fr/galaxy/'><img src='/PublicGalaxyServers/Bistro.png' alt='BISTRO Galaxy' width="300" /></a></div>

* *Link:*
  * [BISTRO Galaxy](http://galaxy.bioinfo-bistro.fr/galaxy/)
* *Domain/Purpose:* 
  * Taxonomic studies of environmental microbial communities 
* *Comments:*
  * Publishes the [PipeAlign workflow](http://galaxy.bioinfo-bistro.fr/galaxy/u/kress/w/pipealign-2) and supporting tools. These tools include ballast, RASCAL, LEON, and MACSIMS.
* *User Support:*
  * [akress@unistra.fr](Email support)
* *Quotas:* 
  * Supports anonymous access and creation of user logins. 
* *Sponsor(s):*
  * This instance of Galaxy is managed by the [BISTRO platform](http://bioinfo-bistro.fr/bioinfo-bistro/).

## MiModD NacreousMap

<div class='right'><div class='solid'><a href='http://mapping-by-sequencing.vm.uni-freiburg.de:8080/'><img src='/PublicGalaxyServers/Baumeister.png' alt='Baumeister Lab Galaxy featuring MiModD' width="300" /></a></div>

* *Link:*
  * [MiModD NacreousMap](http://mapping-by-sequencing.vm.uni-freiburg.de:8080/)
* *Domain/Purpose:* 
  * This server exposes the !NacreousMap mapping/plotting tool of MiModD for users of MiModD who do not want to install the required dependencies (R and rpy2) for graphical output from this tool on their local system. [MiModD](http://www.celegans.de/en/mimodd) is a comprehensive software package for the identification and annotation of mutations in the genomes of model organisms from whole-genome sequencing (WGS) data. 
* *Comments:*
  * [CloudMap](https://usegalaxy.org/cloudmap) users can replot data produced with the Hawaiian Variant Mapping tool using the !NacreousMap engine to obtain optimized (much smaller files that display faster) plots.
  * To install the complete MiModD package for use as a command line tool or for integration into any local Galaxy follow the [installation instructions](http://mimodd.readthedocs.org/en/latest/INSTALL.html) in the [MiModD User Guide](http://mimodd.readthedocs.org/en/latest/).
* *User Support:*
  * [MiModD User Guide](http://mimodd.readthedocs.org/en/latest/index.html#)
  * [mimodd@googlegroups.com](MiModD Google Group)
* *Quotas:* 
  * The quota for anonymous usage is 50MB, registered users have 250MB available.
  * You can analyze/plot data in VCF format or the "Per variant report" format generated by local runs of the MiModD !NacreousMap tool. The latter has the advantage of being up to 20 times smaller than the corresponding VCF file.
* *Sponsor(s):*
  * [The Baumeister Laboratory](http://www.celegans.de/) at the [Institute for Biology 3](http://www.biologie.uni-freiburg.de/data/bio3/) of the [Albert-Ludwig University](http://www.uni-freiburg.de/) in Freiburg/Breisgau, Germany.

## NIOO-KNAW PhyloProfiler

<div class='right'><div class='solid'><a href='https://galaxy.bioinf.nioo.knaw.nl/public/'><img src='/PublicGalaxyServers/NIOO-KNAW-Logo.jpg' alt='NIOO-KHAN Public Galaxy Server featuring PhyloPofiler' width="300" /></a></div>

* *Link:*
  * [NIOO-KHAN Public Galaxy Server featuring PhyloPofiler](https://galaxy.bioinf.nioo.knaw.nl/public/)
* *Domain/Purpose:*
  * [PhyloProfiler](https://galaxy.bioinf.nioo.knaw.nl/public/) consists of several filtering and reporting options for analyzing [PhyloChip](http://www.secondgenome.com/solutions/services/phylochip/) data consisting of OTUs with hybridization scores (!HybScores).
* *Comments:*
  * See [Impact of soil heat on reassembly of bacterial communities in the rhizosphere microbiome and plant disease suppression](http://onlinelibrary.wiley.com/doi/10.1111/ele.12567/abstract) by van der Voort, *et al.*, in *Ecology Letters* doi: 10.1111/ele.12567
* *User Support:*
  * See the [Galaxy PhyloChip analysis pipeline PhyloProfiler User’s Manual](https://galaxy.bioinf.nioo.knaw.nl/public/static/pdf/PhyloProfiler_Manual_v2.pdf) 
* *Quotas:* 
  * Anonymous access is supported.
* *Sponsor(s):*
  * [BRS group of the Netherlands Bioinformatics Centre (NBIC)](http://www.nbic.nl/support/brs/group-members) in collaboration with the [Laboratory of Phytopathology of the Wageningen University](http://www.wageningenur.nl/en/Expertise-Services/Chair-groups/Plant-Sciences/Laboratory-of-Phytopathology.htm)/[NIOO-KNAW](https://nioo.knaw.nl/).


# Releases

## Galaxy v16.01

<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GalaxyNewLogo_GalaxyProject_Trans.png' alt='GalaxyProject' width="200" /></a></div>

The [Galaxy Committers team](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) is pleased to announce the [January 2016 (v16.01) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.01_announce.html). 

**Galaxy administrators should also be aware of the<<span(red)>> [security announcements](#security-announcements) <<span>> below.**

<span style="font-size: larger;"> **Highlights** </span>

**Interactive Tours**

The interactive tours framework allows developers and deployers to build interactive tutorials for users superimposed on the actual Galaxy web front end. Unlike video tutorials, these will not become stale and are truly interactive (allowing users to actually navigate and interact with Galaxy). Galaxy 16.01 ships with two example tours and new ones can easily be added by creating a small YAML file describing the tour. Try the Galaxy UI tour on Main.

**Wheels**

Galaxy’s Python dependencies have traditionally been distributed as eggs using custom dependency management code to enable Galaxy to distribute binary dependencies (enabling quick downloads and minimal system requirements). With this release all of that infrastructure has been replaced with a modern Python infrastructure based on pip and wheels. Work done as part of this to enable binary dependencies on Linux has been included with the recently released pip 8.

Detailed documentation on these changes and their impact under a variety of Galaxy deployment scenarios can be found in the Galaxy Framework Dependencies section of the Admin documentation.

**Nested Workflows**

Workflows may now run other workflows as a single abstract step in the parent workflow. This allows for reusing or subworkflows in your analyses.


<table>
  <tr>
    <td style=" border: none"> <span style="font-size: larger;"> Github </span> <br /><br /></strong>New<strong><br /><code>% git clone -b master https://github.com/galaxyproject/galaxy.git</code><br /><br />U</strong>pdate to latest stable release<strong><br /><code>% git checkout master && pull --ff-only origin master</code><br /><br /></strong>Update to exact version<strong><br /><code>% git checkout v16.01</code> </td>
    <td style=" vertical-align: top; border: none; width: 10%"> </td>
    <td style=" vertical-align: top; border: none"> <span style="font-size: larger;"> !BitBucket </span> <br /><br /></strong>Upgrade<strong><br /><code>% hg pull</code><br /><code>% hg update latest_16.01</code> </td>
  </tr>
  <tr>
    <td colspan=3 style=" border: none"> See the <a href='/Admin/GetGalaxy.md'>Get Galaxy</a> page for additional details regarding the source code locations. </td>
  </tr>
</table>



<span style="font-size: larger;"> Deprecation Notices </span>

Barring a strong outcry from deployers, 16.01 will be the last release of Galaxy to support Python 2.6. For more information, see [Galaxy Github Issue #1596](https://github.com/galaxyproject/galaxy/issues/1596).


<div class='red'>
## Security Announcements

<div class='indent'>

### Read/write arbitrary filesystem paths, arbitrary code execution

Multiple security vulnerabilities were recently discovered in Galaxy that allow malicious actors to read and write files on the Galaxy server. Additionally, Galaxy servers on which a rarely used feature has been enabled are vulnerable to an arbitrary code execution exploit.  

*This issue affects all known releases of Galaxy in at least the last 3 years.*  See the [full announcement](http://dev.list.galaxyproject.org/Galaxy-Security-Vulnerability-Read-write-arbitrary-filesystem-paths-arbitrary-code-execution-td4668936.html) for details.

### Tool Shed Security Vulnerability - Read/write arbitrary filesystem paths

Multiple security vulnerabilities were recently discovered in the Tool Shed that allow malicious actors to read and write files on the Tool Shed server outside of normal Tool Shed repository directories. 

*This issue affects all known releases of Galaxy in at least the last 3 years.*  See the [full announcment](http://dev.list.galaxyproject.org/Galaxy-Tool-Shed-Security-Vulnerability-Read-write-arbitrary-filesystem-paths-td4668935.html) for details.
<br /><br /></div></div>

## galaxy-lib 16.4.0

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

## CloudBridge 0.1.0

The Galaxy Team is proud to be part of the development team for a new cross-cloud library called !CloudBridge. **!CloudBridge is a Python library providing a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud.** The library is generally applicable to any domain wishing to run cloud-independent applications. There is already support for Amazon and !OpenStack clouds with support for Google’s Compute Engine in development. 

The first version of !CloudBridge was released earlier this month and it comes with detailed user documentation: http://cloudbridge.readthedocs.org/. The source code is available on Github: https://github.com/gvlproject/cloudbridge.

## Other Releases


**Galaxy Docker Image 15.10**

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 15.10](https://github.com/bgruening/docker-galaxy-stable/releases/tag/15.10) as well.  



<div class='right'><a href='/CloudMan.md'><img src='/Images/Logos/CloudManWideBlackLogo.png' alt='CloudMan' width="150" /></a></div>

**CloudMan 15.12**

CloudMan is a cloud manager that orchestrates all of the steps required to provision a complete compute cluster environment on a cloud infrastructure; subsequently, it allows one to manage the cluster, all through a web browser. 

This minor [CloudMan 15.12 release](https://github.com/galaxyproject/cloudman/releases) includes:

* Update Galaxy to the Galaxy 15.10 release (see above for all the juicy details with the improvements that brings along)
* Preload the GIE IPython Docker container onto the image for faster startup: less than 30 seconds to a fully functional IPython notebook vs. several minutes before!
* For those of you that ran into the problem of the size of your user data overfilling after having added and removed a bunch of worker instances - no worries, that has been fixed now and resource tags are no longer stored in the cluster config to prevent *persistent_data.yaml* from becoming too big.
* The file system archive extraction has been made more resilient and synchronous.
* For those that are building custom images, a more direct method for locating the Nginx configuration directory has been added to help with different versions of the operating system (thanks to [Matthew Ralston](https://github.com/MatthewRalston))
* A number of dependent library versions have been updated to their latest versions.

**Starforge 0.1**

<div class='right'>
<a href='https://github.com/galaxyproject/starforge/blob/master/README.rst'><img src='/Images/GalaxyLogos/starforge_logo_big.png' alt='Starforge' width="180" /></a>
</div>

[Starforge](https://github.com/galaxyproject/starforge/blob/master/README.rst) is a collection of scripts that supports the building of components for Galaxy.  Specifically, with Starforge you can:

* Build [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) dependencies
* Build [Python Wheels](http://pythonwheels.com/) (e.g. for the [Galaxy Wheels Server](http://wheels.galaxyproject.org/))
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.

Documentation can be found at [starforge.readthedocs.org](http://starforge.readthedocs.org/). 


<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src='/Images/GalaxyLogos/pulsar_transparent.png' alt='Pulsar' width="150" /></a></div>

**Pulsar 0.6.1**

[Pulsar 0.6](https://pypi.python.org/pypi/pulsar-app/) was released in December.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.
 
The 0.6.x release includes these changes:

* Pulsar now depends on the new galaxy-lib Python package instead of manually synchronizing Python files across Pulsar and Galaxy.
* Numerous build and testing improvements.
* Fixed a documentation bug in the code (thanks to @erasche). [e8814ae](https://github.com/galaxyproject/pulsar/commit/e8814ae)
* Remove galaxy.eggs stuff from Pulsar client (thanks to @natefoo). [00197f2](https://github.com/galaxyproject/pulsar/commit/00197f2)
* Add new logo to README (thanks to @martenson). [abbba40](https://github.com/galaxyproject/pulsar/commit/abbba40)
* Implement an optional awknowledgement system on top of the message queue system (thanks to @natefoo). [Pull Request 82](https://github.com/galaxyproject/pulsar/pull/82) [431088c](https://github.com/galaxyproject/pulsar/commit/431088c)
* Documentation fixes thanks to @remimarenco. [Pull Request 78](https://github.com/galaxyproject/pulsar/pull/78), [Pull Request 80](https://github.com/galaxyproject/pulsar/pull/80)
* Fix project script bug introduced this cycle (thanks to @nsoranzo). [140a069](https://github.com/galaxyproject/pulsar/commit/140a069)
* Fix config.py on Windows (thanks to @ssorgatem). [Pull Request 84](https://github.com/galaxyproject/pulsar/pull/84)
* Add a job manager for XSEDE jobs (thanks to @natefoo). [1017bc5](https://github.com/galaxyproject/pulsar/commit/1017bc5)
* Fix pip dependency installation (thanks to @afgane) [Pull Request 73](https://github.com/galaxyproject/pulsar/pull/73)


<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src='/Images/Logos/PlanemoLogo.png' alt='' width="150" /></a></div>

**Planemo 0.21.1**

   [Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  [Planemo 0.21.1](https://pypi.python.org/pypi/planemo/0.21.1#history) is the most recent release. See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).


**!BioBlend 0.7.0**

   [BioBlend version 0.7.0](https://github.com/galaxyproject/bioblend/) was released at the beginning of November. !BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/Learn/API).  CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG).


**blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and !BioCloudCentral.


----


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Teach.md'><img src='/Images/GalaxyLogos/GTNLogo300.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Logs.md'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%"> <a href='/Community/Deployments.md'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong>Share your <a href='/Teach/Resources.md#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers.md#add-a-trainer'>experience</a> <em>now</em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Logs.md#add-a-log-page'>Share your experience</a> <em><a href='/Community/Logs.md#add-a-log-page'>now</a></em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none"> </strong><a href='/Community/Deployments.md#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/Community/Deployments.md#add-your-galaxy-deployment'>now</a></em> <strong> </td>
  </tr>
</table>

<br />

----
<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in February](/ToolShed/Contributions/2016_02).


----

# Other News
 
* From [Peter Amstutz (@peteramstutz)](https://twitter.com/peteramstutz)
  * [Common Workflow Language (CWL) draft-3](https://common-workflow-language.github.io/draft-3/) is now stable!
