---
date: '2018-09-05'
title: "Contributor of the Month: Carrie Ganote"
authors: "Björn Grüning"
image: "/news/2018-09-cotm-carrie-ganote/carrie-ganote.png"
---

The Galaxy ecosystem is a direct result of the Galaxy community and the contributions and involvement of community members.  The community makes it all work in innumerable ways, including support, training and tutorial development, talks, posters, documentation, issue reporting, feature requests, tool publishing, coding, and Galaxy server, cloud, and appliance support.

To help highlight our community we will interview a member of the community every month and feature him/her in the *Galactic Blog*.

## Carrie Ganote

This month we welcome Carrie Ganote (Twitter [@clg_cganote](https://twitter.com/clg_cganote), GitHub: [@cganote](https://github.com/cganote)) as our Galaxy contributor of the Month! Carrie is developing and maintaining the Galaxy instance of the Indiana University Bloomington, contributes regularly to Galaxy and Galaxy tools and organises great break-out events during GCCs.

----

Carrie, thanks for doing this interview, we are looking forward to learn more about you and your work!

### Can you tell us a little about yourself (hobbies, education, background, etc.)?

I was born in Indiana, in the Midwest part of the US. I was always an outdoorsy kid, but I also loved computers, reading, and art. When it came time for college, I couldn’t decide between Biology and Computer Science, so I did both. I figured I’d have a bright future building prosthetic limbs or androids or some such. This was before bioinformatics was really a word that people said out loud. After college, I moved out into the country where I run a small hobby farm, raising turkeys, chickens, ducks, peafowl, geese, and pigs. I am working part-time on a PhD in bioinformatics currently.

### Why did you start using Galaxy?

I inherited a running Galaxy instance when I started my current job with the [National Center for Genome Analysis Support](https://ncgas.org/) at the end of 2012. It was my job to go through the tools, make sure they all worked, keep it running, pare down things I thought we didn’t need, and add things that were needed. I worked on some collaborations with other teams here at Indiana University, and we managed to hook our Galaxy instance into [inCommon](https://www.incommon.org/) authentication systems and launch jobs onto the [Open Science Grid](http://opensciencegrid.org/), a collection of machines that agreed to make spare cycles available to whoever wants to use them.

### How did you get into the Galaxy community, what (and when) was your first contribution?

Not long after I started this job, we were going to the [Plant and Animal Genome conference](http://www.intlpag.org/) in San Diego. We were going to show off Galaxy on iPads for the conference, but the best we could scrounge together were first generation iPads - and it browsers on these devices totally broke scrolling on Galaxy. I figured out a way to get it working on the iPad 1’s out of necessity!

### Which are your favorite Galaxy tools?

FastQC might be my favorite tool on Galaxy. The fact that you have to run this tool in order to inaugurate pretty much any NGS project makes it critical for biologists. The html output is hard to get at in a normal HPC environment - you either have to tunnel with X11, or download the FastQC results to a laptop for viewing. Seeing the results on the browser is very convenient!

### What is your favorite Galaxy feature?

The history panel is my favorite feature - it really helps keep track of what was being done to the data. I use the history-to-workflow tool sometimes to get a better idea of the structure of the history, but it can be a little hard to use.

### What projects are you working on now?

Lots - Dockerizing a Galaxy instance, running 2 production servers plus 2 dev ones, installing new tools and features. I’d like to get back to some pet projects of running Galaxy jobs across institutions and moving files to RAM while running. This is in addition to my other work, annotating the Coffee transcriptome, providing user support in general, and developing RNA-seq workshop materials.

### You have been at many GCCs, what do you like at this conference and what could be improved?

I really like how the developer team makes themselves accessible to the community! I feel like I can go ask questions, find out the latest plans, and my voice is heard. I think text and email is fine as a way to communicate but nothing beats face-to-face.

### What advice would you give to new admins/developers/contributors?

Keep up the good work, listening to the needs of admins and users. I was very pleased to hear that the Galaxy UI was going to be a focus on development efforts in the coming months. I think, as software engineers and developers, we get excited or caught up in the technical aspects and challenges - can I make this run this fast, can I integrate this other technology, or run in this scenario, things like that. I think the end user just wants the thing to work smoothly and with minimal pain. I think spending a bit more time watching users flail around on the system would help devs understand where the pain points are.

### What is your vision for the Galaxy project for the next years?

I’m excited to see Galaxy continue to evolve, adding more features to the user interface and continuing to be a critical piece in the workflow for biologists who would otherwise be unable to approach large-scale data analysis.

----

Thanks for doing the interview!


