---
title: November 2014 Galaxy Update
---
<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the November 2014 *[Galaxy Update](/galaxy-updates/)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Thanks!

Thanks to everyone who took the Galaxy Questionnaires in October.  We received 155 responses, which is an amazing number.  We will use this input to help prepare the next grant for the Galaxy Project.  Thanks again for your time and thoughtful responses.

# IRC Channel is Now Publicly Archived

<div class='right'><a href='https://botbot.me/freenode/galaxyproject/'><img src="/images/logos/BotBotLogo.png" alt="Galaxy IRC Channel archive at BotBot" width="60" /></a></div>

The #galaxyproject IRC channel now has an [online public archive](https://botbot.me/freenode/galaxyproject/).  These archives have also been included in the [Galaxy search engine](/search/).  The archive started on October 22, 2014.

This was [proposed](/news/irc-public-log-proposal/) and then [discussed, and approved on Galaxy Biostar](http://bit.ly/gxybs8804).

# Galaxy Training Network

<div class='left'><a href='https://training.galaxyproject.org/'><img src="/images/logos/GTNLogo600.png" alt="Galaxy Training Network" width="200" /></a></div>

The *[Galaxy Training Network (GTN)](https://training.galaxyproject.org/)* was [launched October 16](/news/galaxy-training-network/) with 16 charter member organizations.  The GTN is a network of trainers who teach bioinformatics using Galaxy, or teach about Galaxy itself.  The GTN aims to make it easy to find Galaxy trainers, and to share and discover the wealth of training resources available for Galaxy.  This includes training materials, a trainer directory, best practices, and guidance on computing platforms for teaching with Galaxy.  The Galaxy Training Network is accessible to the entire community.  

If you teach with Galaxy, then please consider adding your organization, materials, and best practices. Since the GTN launched two weeks ago, 4 new organizations have joined:

* Alberta Children's Hospital Research Institute, Calgary, Canada
* Memorial Sloan Kettering Cancer Center, Rätsch Laboratory, New York City, United States
* MMG@IICT, Hyderabad, India
* South Green Platform, Montpellier, France

This brings the total to 20 training organizations on 5 continents.

# Events

## Galaxy Days: 2-3 December, Paris

The [French Working Group GALAXY-IFB (Institut Français de Bioinformatique)](http://www.ifb-galaxy.org/index.html) is organizing a second session around the Galaxy portal.  The event will be at Institut Curie in Paris over two days.  This year, we want to involve two communities: *biologists* (also known as Galaxy 'users') and *bioinformaticians* (Galaxy 'developers').  The goal is to present user experience around the portal, from a single user to a wider community:

* Dec 2 (13:30-17:30): Galaxy's user experiences, and discussion on how the platform is (or is not) useful for building analysis.
* Dec 3 (09:00-17:00): Technology talks (new environment, Galaxy in production, ...)

Interested? Please contact [ifb DOT galaxy AT sb DASH roscoff DOT fr](mailto:ifb DOT galaxy AT sb DASH roscoff DOT fr) for more information.

The [French IFB Galaxy Working Group](http://www.ifb-galaxy.org/index.html):

[URGI](http://urgi.versailles.inra.fr/), [GenoToul](http://bioinfo.genotoul.fr/), [MIGALE](http://migale.jouy.inra.fr/), [PFEM](http://metabohub.fr/), [SouthGreen](http://southgreen.cirad.fr/), [Institut Curie](http://curie.fr/), [ABiMS](http://abims.sb-roscoff.fr/)

<div class='center'>
<a href='http://urgi.versailles.inra.fr/'><img src="/images/logos/URGILogo.png" alt="URGI" width="70" /></a> &nbsp;
<a href='http://bioinfo.genotoul.fr/'><img src="/images/logos/GenoToulLogo.png" alt="GenoToul" width="100" /></a> &nbsp;
<a href='http://migale.jouy.inra.fr/'><img src="/images/logos/MIGALELogo.png" alt="MIGALE" width="120" /></a> &nbsp;
<a href='http://metabohub.fr/'><img src="/images/logos/MetaboHubLogo.jpg" alt="PFEM" width="120" /></a> &nbsp;
<a href='http://southgreen.cirad.fr/'><img src="/images/logos/SouthGreenLogo.png" alt="SouthGreen" width="120" /></a>
<a href='http://curie.fr/'><img src="/images/logos/CurieLogo.gif" alt="Institut Curie" width="80" /></a> &nbsp;
<a href='http://abims.sb-roscoff.fr/'><img src="/images/logos/ABiMSLogo.png" alt="ABiMS" width="70" /></a>
</div>

## Swiss German Galaxy Tour 2014 Report

<div class='right'><br /><br /><a href='/events/sg2014/'><img src="/images/logos/SG2014Logo400.png" alt="Swiss-German Galaxy Tour 2014" width="120" /></a></div>

After the big success of the [first Swiss Galaxy Workshop two years ago](/events/switzerland2012/), we decided to [organize a similar event](/events/sg2014/) again this fall. This time, we added a [training day](/events/switzerland2014/trainingday/) prior to the [workshop](/events/switzerland2014/), and a [developer day](/events/germany2014/) after the workshop. The first two days were held in Bern (Switzerland), and the third in Freiburg (Germany). Hence we called the whole event: "[Swiss German Galaxy Tour 2014](/events/sg2014/)".  

More than 40 people registered for the event, signing up for one, two or all three days, ...<br />
*- [read more](/news/s-g2014-report/) -*


## Fall 2014 GUGGO Events Report

<div class='right'><a href='http://www.biogenouest.org/agenda/evenement/groupe-de-travail-des-utilisateurs-galaxy-du-grand-ouest-guggo'><img src="/images/logos/eBiogenouestLogo.png" alt="Galaxy User Group Grand Ouest (GUGGO)" width=120 /></a></div>

Three events were sponsored by the [Galaxy User Group Grand Ouest (GUGGO)](https://www.e-biogenouest.org/groups/guggo) in western France earlier this fall.  Summaries of all 3 events are now available online.

The *[Tools integration on Galaxy Workshop](http://go4bioinformatics.genouest.org/integration-doutils-dans-la-plateforme-web-galaxy/)* was held 11 September. The [summary](https://www.e-biogenouest.org/resources/1312/about) includes ...<br />
*- [read more](/news/guggo-report-fall2014/) -*

## Other Events

<div class='center'>
<a href='http://galaxy4bioinformatics.sb-roscoff.fr/'><img src="/images/logos/SB-RoscoffBig.JPG" alt="Des bonnes pratiques d'intégration d'outils sous Galaxy" height="60" /></a> &nbsp;
<a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=62'><img src="/images/logos/GenoToulLogo.png" alt="Galaxy Training Days" height="60" /></a> &nbsp;
<a href='http://www.bioinformatica.crs4.it/next_generation_sequencing_course_with_galaxy_november_2014'><img src="/images/logos/CRS4Logo.jpg" alt="Analisi dati Next Generation Sequencing in Galaxy" width="130" /></a> &nbsp;
<a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-1920-november-brisbane/'><img src="/images/logos/QFABLogo.png" alt="Workshop: Extended RNA-Seq analysis" height="80" /></a>
<a href='http://www.prabi.fr/spip.php?article215'><img src="/images/logos/PRABILogo.png" alt="RNA-Seq & ChIP-Seq analysis course using Galaxy" height="60" /></a>
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div>

There are [upcoming events](/events/) in France, Germany, Australia, Italy, and the United States.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> November 3-5 </th>
    <td> <em><a href='http://galaxy4bioinformatics.sb-roscoff.fr/'>Des bonnes pratiques d'intégration d'outils sous Galaxy</a></em><div class='indent'>Workshop full, but you can get on the waiting list</div> </td>
    <td> <a href='http://www.sb-roscoff.fr/'>Station Biologique de Roscoff</a>, France </td>
    <td> <a href="mailto:christophe.caron@sb-roscoff.fr">Christophe Caron</a> </td>
  </tr>
  <tr>
    <th> November 3-5 </th>
    <td> <em><a href='http://www.helmholtz-muenchen.de/en/agv/research/research-groups/viral-latency/index.html'>Galaxy NGS Training in the Group of Prof. Dr. Bettina Kempkes</a></em><div class='indent'>Workshop full, but you can get on the waiting list</div> </td>
    <td> Helmholtz Zentrum München, Germany </td>
    <td> <a href="mailto:bjoern.gruening@gmail.com">Björn Grüning</a> </td>
  </tr>
  <tr>
    <th> November 3-6 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=62'>Galaxy training days</a></em><div class='indent'>Workshop full, but you can get on the waiting list</div> </td>
    <td> <a href='http://bioinfo.genotoul.fr/index.php?id=4#c93'>INRA de Toulouse Midi-Pyrénées</a>, France </td>
    <td> <a href='http://bioinfo.genotoul.fr/index.php?id=74'>GenoToul Bioinformatics Team</a> </td>
  </tr>
  <tr>
    <th rowspan=2> November 5-8 </th>
    <td> <em>Rapidly bringing software to biologists with Galaxy and Docker</em> </td>
    <td rowspan=2> <a href='http://meetings.cshl.edu/meetings/2014/data14.shtml'>Biological Data Science</a>, Cold Spring Harbor Laboratory, New York, United States </td>
    <td> <a href='/people/john-chilton/'>John Chilton</a> </td>
  </tr>
  <tr>
    <td> <em>Building Galaxy Japan community</em> <div class='indent'> (See <a href='http://www.pitagora-galaxy.org/'>Pitagora Galaxy</a>)</td>
    <td> Ryota Yamanaka </td>
  </tr>
  <tr>
    <th> November 16 </th>
    <td> <em>Deciphering Big Data Stacks: An Overview of Big Data Tools</em> </td>
    <td> <a href='http://web.ornl.gov/sci/knowledgediscovery/CloudComputing/PDAC-SC14/'>Workshop on Big Data Analytics: Challenges, and Opportunities (BDAC-14)</a>, <a href='http://sc14.supercomputing.org/'>Supercomputing 2014 (SC14)</a>, New Orleans, Louisiana, United States </td>
    <td> <a href='/people/enis-afgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> November 18-20 </th>
    <td> <em><a href='http://www.bioinformatica.crs4.it/training'>Analisi dati Next Generation Sequencing con Galaxy</a></em> </td>
    <td> Cagliari, Italy </td>
    <td> <a href="mailto:ngs-courses@crs4.it">CRS4</a> </td>
  </tr>
  <tr>
    <th> November 19-20 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-1920-november-brisbane/'>Workshop: Extended RNA-Seq analysis</a></em> </td>
    <td> The University of Queensland, Brisbane, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 21 </th>
    <td> <em>Galaxy Cluster to Cloud - Genomics at Scale</em> </td>
    <td> <a href='http://sciencegateways.org/upcoming-events/gce14/'>GCE: The 9th Gateway Computing Environments Workshop</a>, <a href='http://sc14.supercomputing.org/'>Supercomputing 2014 (SC14)</a>, New Orleans, Louisiana, United States </td>
    <td> <a href='/people/enis-afgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> November 26-28 </th>
    <td> <em><a href='http://www.prabi.fr/spip.php?article215'>RNA-Seq & ChIP-Seq analysis course using Galaxy</a></em> </td>
    <td> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <a href="mailto:navratil AT prabi DOT fr">Navratil V</a>., <a href="mailto:oger AT prabi DOT fr">Oger C.</a>, <a href="mailto:veber AT prabi DOT fr">Veber P.</a>, <a href="mailto:deschamps AT prabi DOT fr">Deschamps C.</a>, <a href="mailto:perriere AT prabi DOT fr">Perriere G.</a> </td>
  </tr>
  <tr>
    <th> December 2-3 </th>
    <td> <em><a href='http://bit.ly/ZWVjM1'>Galaxy Day</a></em> </td>
    <td> Institut Curie, Paris, France </td>
    <td> <a href="mailto:ifb DOT galaxy AT sb DASH roscoff DOT fr">IFB Galaxy</a> </td>
  </tr>
  <tr>
    <th> December 5-8 </th>
    <td> <em><a href='http://manuals.bioinformatics.ucr.edu/workshops'>Next Generation Data Analysis Workshop</a></em> </td>
    <td> UC Riverside, Riverside, California, United States </td>
    <td> <a href="mailto:rkaundal AT ucr DOT edu">Rakesh Kaundal</a> </td>
  </tr>
  <tr>
    <th> December 9-11 </th>
    <td> <em><a href='http://www.pf-bird.univ-nantes.fr/85361781/0/fiche___pagelibre/&RH=1357829686766'>Microarray data analysis on Galaxy</a></em> </td>
    <td> <a href='http://www.pf-bird.univ-nantes.fr/'>BIRD IFB core facility</a> Nantes University/INSERM, Nantes, France </td>
    <td> <a href="mailto:raluca DOT teusan AT inserm DOT fr">Raluca Teusan</a>, Audrey Bihouée, Edouard Hirchaud</td>
  </tr>
  <tr>
    <th> December 16-19 </th>
    <td> <em><a href='https://training.bioinformatics.ucdavis.edu/2014/10/10/rna-seq-and-chip-seq-analysis-with-galaxy-december-16-19-2014/'>RNA-Seq and ChIP-Seq Analysis with Galaxy</a></em> </td>
    <td> <a href='http://training.bioinformatics.ucdavis.edu/'>UC Davis</a>, California, United States </td>
    <td> <a href="mailto:bioinformatics DOT core AT ucdavis DOT edu">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th colspan=4> 2015 </th>
  </tr>
  <tr>
    <th> January 10-14 </th>
    <td> <em>Galaxy for SNP and Variant Data Analysis</em> </td>
    <td> <a href='http://www.intlpag.org/'>Plant and Animal Genome XXIII (PAG2014)</a>, San Diego, California, United States </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> February 9-13 </th>
    <td> <em><a href='http://bit.ly/1q2L2cv'>Analyse bioinformatique de séquences sous Galaxy</a></em> </td>
    <td> Montpellier, France </td>
    <td> <a href="mailto:jean-francois.dufayard@cirad.fr">J.F. Dufayard</a> </td>
  </tr>
  <tr>
    <th> February 16-18 </th>
    <td> <em>Managing and Disseminating Tools and Data in Galaxy</em> </td>
    <td> <a href='http://www.triconference.com/Sequencing-Analysis/'>Genome and Transcriptome Analysis</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='/people/james-taylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> <strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a></strong> </td>
    <td> <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>




# New Papers

38 papers were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in October, including:

* [Flexible and accessible workflows for improved proteogenomic analysis using the Galaxy framework](https://doi.org/10.1021/pr500812t), by Pratik D. Jagtap, James E. Johnson, Getiria Onsongo, *et al.,* *J. Proteome Res.* (10 October 2014), doi:10.1021/pr500812t


* [iReport: a generalised Galaxy solution for integrated experimental reporting](https://doi.org/10.1186/2047-217x-3-19), by Saskia Hiltemann, Youri Hoogstrate, Peter van der Spek, Guido Jenster, Andrew Stubbs, *GigaScience*, Vol. 3, No. 1. (2014), 19, doi:10.1186/2047-217x-3-19

* [ExomeAI: Detection of recurrent Allelic Imbalance in tumors using whole Exome sequencing](https://doi.org/10.1093/bioinformatics/btu665), by Javad Nadaf, Jacek Majewski, Somayyeh Fahiminiya, *Bioinformatics* (08 October 2014), btu665, doi:10.1093/bioinformatics/btu665


The new papers were tagged in these areas:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 19 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Bioinformatician](/news/bioinformatician-freiburg2014/), Max Planck Institute of Immunobiology and Epigenetics, Freiburg, Germany
* [Research Specialist](http://bit.ly/13pqd0T), Michigan State University, United States
* [Galaxy Workflow Developer](https://www.jic.ac.uk/training-careers/vacancies/2014/10/galaxy-workflow-developer/#), John Innes Centre, Norwich, United Kingdom.  <div class='red'>Closes Nov 5.</span>
* [Computational Science Developer I](http://bit.ly/1lLkAMR), Cold Spring Harbor Laboratory (CSHL), New York, United States
* [Statistical Genomics Postdoc](/news/statistical-genomics-post-doc-penn-state/) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

# New Public Servers

Two new [public Galaxy servers](/use/) were added to the published list in October:

## Majewski Lab Galaxy

<div class='right solid'><a href='http://genomequebec.mcgill.ca/exomeai'><img src="/images/use/majewski-lab/mcgill-qgic.png" alt="Majewski Lab: ExomeAI" width="250px" /></a></div>

The [Majewski Lab ExomeAI Server](http://genomequebec.mcgill.ca/exomeai) supports detection of recurrent allelic imbalance in tumors using whole exome sequencing data, using ExomeAI, a free web-based application for detection of recurrent AI/LOH segments in tumor samples. Support is provided in the [ExomeAI Manual](http://bioinformatics.oxfordjournals.org/content/suppl/2014/10/07/btu665.DC1/suppl_data.zip), and via [email](mailto:Javad DOT Nadaf AT gmail DOT com).  See Nadaf J, Majewski J, Fahiminiya S. (2014). [ExomeAI: Detection of recurrent Allelic Imbalance in tumors using whole Exome sequencing data](http://www.ncbi.nlm.nih.gov/pubmed/25297069). *Bioinformatics*. 2014 Oct 8.

The [Majewski Lab ExomeAI Server](http://genomequebec.mcgill.ca/exomeai) is supported by the [McGill University and Génome Québec Innovation Centre](http://gqinnovationcenter.com/index.aspx)

## OSDD Molecular Property Diagnostic Suite (MPDS)

<div class='right solid'><a href='http://mpds.osdd.net/'><img src="/images/use/mpdstb/mpdstb.png" alt="Molecular Property Diagnostic Suite (MPDS)" width="200" /></a></div>

The [OSDD Molecular Property Diagnostic Suite (MPDS)](http://mpds.osdd.net/) Galaxy server is an OSDD Chemoinformatics Portal.  MPDS exposes a software toolset that rationally diagnoses (druggable) molecules.  MPDS 1.0 consists of six modules covering informatics (DataBases, File format conversion), structure and analogue based drug design approaches (Property calculation, QSAR, Docking). [Support](http://mpds.osdd.net/galaxy/static/contact.htm) is available.

[MPDS](http://mpds.osdd.net/) is developed under the broad initiative of [OSDD (Open Source Drug Discovery)](http://www.osdd.net/) of [CSIR (Council of Scientific and Industrial Research, Govt. of India)](http://www.csir.res.in/home.asp). The site is being hosted from [IICT](http://www.iictindia.org/), Hyderabad, India.

# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a></div>
<div class='right'><img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style="text-align: center;"> &nbsp;&nbsp; <a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a> </em> &nbsp;&nbsp;<br /> </td>
  </tr>
</table>

<br /><br />

**There were no new [Log Board](/news/) or Deployment Catalog entries in August!**  Eek!  Please don't let this happen again!  

The *[Community Log Board](/news/)* and  *Deployment Catalog* *[Galaxy community hubs](/community/)* were launched last your.  If you have a Galaxy deployment, or [experience](/news/) you want to share then please publish them **this month.**

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# New Releases

[BioBlend v0.5.2 was released in October](/news/bio-blend-v0-5-2/).  [BioBlend](https://github.com/afgane/bioblend) is a python library for interacting with CloudMan and the [Galaxy API](/develop/api/).  

New versions of [Galaxy](/galaxy-updates/2014-09/#august-11-2014-galaxy-distribution), [CloudMan](/galaxy-updates/2014-09/#august-2014-cloudman-release), and [blend4j](/galaxy-updates/2014-09/#blend4j-011-release) were all released in August.

Look for a new Galaxy distribution in November.

# ToolShed Contributions

## Galaxy Project ToolShed Repos

Here are new contributions for the past *two* months.

In no particular order:

### Tools

* *From [crs4](https://toolshed.g2.bx.psu.edu/view/crs4):*
    * [seal_galaxy](https://toolshed.g2.bx.psu.edu/view/crs4/seal_galaxy): Galaxy wrappers for [Seal](http://github.com/crs4/seal)

* *From [arkarachai-fungtammasan](https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan):*
    * [microsatellite_ngs](https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/microsatellite_ngs): Pipeline to profile and genotype microsatellites from short read data. This repository contains these sets of tools: 1 create microsatellite length profile, 2 correct for sequencing errors and report genotype, 3 estimate minimum sequencing read depth, 4 convert informative read depth to locus specific/genome wide sequencing depth.

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/peterjc):*
    * [mummer](https://toolshed.g2.bx.psu.edu/view/peterjc/mummer): v0.0.1, essentially a preview (previously only on the TestToolShed). A simple wrapper allowing MUMmer to be used to draw dotplots from within Galaxy using mummer, mucmer, or promer with mummerplot. No tests yet, no gnuplot or ps2pdf dependency yet.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [picard_plus](https://toolshed.g2.bx.psu.edu/view/devteam/picard_plus): Picard wrappers for version 122 and up. New set of Picard wrappers that do not rely on external scripts and deal with all aspects of picard management and UI via tool XML.

* *From [saket-choudhary](https://toolshed.g2.bx.psu.edu/view/saket-choudhary):*
    * [fathmm_web](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/fathmm_web): Calls FATHMM webservice  at http://fathmm.biocompute.org.uk
    * [mutationassessor_web](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/mutationassessor_web): Call [Mutation Assessor webservice](http://mutationassessor.org)
    * [replace_delimiters](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/replace_delimiters): Allows replacing any delimiter in the input to any other delimiter. This tool is similar to Galaxy's default 'Convert delimiter' tool, but allows conversion from any given type(comma, dash, pipe etc)
    * [inchlib_clust](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/inchlib_clust): a python script that performs data clustering and prepares input data for InCHlib. inchlib_clust can be used both from command line or from Python code. Data for clustering are supplied to inchlib_clust as a csv file.
    * [vep_rest](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/vep_rest): Variant Effect Predictor Webservice Package to interact with the GRCh37 (ONLY!). Variant Effect Predictor webservice at http://grch37.rest.ensembl.org
    * [chasm_webservice](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/chasm_webservice): Calls CHASM webserice at www.cravat.us
    * [polyphen2_web](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/polyphen2_web): Calls Polyphen2 webservice at http://genetics.bwh.harvard.edu/pph2/
    * [merge_columns_with_delimiter](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/merge_columns_with_delimiter): Modified merge_columns to allow merging columns separated by a delimiter. This tool allows merging columns separated by a delimiter (two or multiple columns). It is similar to the Galaxy's default too; 'Merge Columns' but also allows merging them separated by a specified delimiter.

* *From [galaxyp](https://toolshed.g2.bx.psu.edu/view/galaxyp):*
    * [pepxml_to_xls](https://toolshed.g2.bx.psu.edu/view/galaxyp/pepxml_to_xls): Convert PepXML to Tabular
    * [protxml_to_xls](https://toolshed.g2.bx.psu.edu/view/galaxyp/protxml_to_xls): Convert ProtXML to Tabular
    * [blastxml_to_tabular_selectable](https://toolshed.g2.bx.psu.edu/view/galaxyp/blastxml_to_tabular_selectable): Converts blast xml file to a tabular with options for unmatched queries, and number of hits to convert.  The unmatched queries can be useful for finding novel peptides.
    * [blast_plus_remote_blastp](https://toolshed.g2.bx.psu.edu/view/galaxyp/blast_plus_remote_blastp): NCBI BLAST+ remote blastp NCBI BLAST+ blastp with additional optional arguments.

### Workflows

* *From [bgruening](https://toolshed.g2.bx.psu.edu/view/bgruening):*
    * [chemicaltoolbox_library_hole_filling_workflow](https://toolshed.g2.bx.psu.edu/view/bgruening/chemicaltoolbox_library_hole_filling_workflow): Uploaded Given one library, it extends all molecules by similar molecules of an other library and thus fill gaps in an automatic manner. Given one library, it extends all molecules by similar molecules of an other library and thus fill gaps in an automatic manner. This workflow is part of case study demonstrating the capability of the chemicaltoolbox. For further information please have a look at the chemicaltoolbox: https://github.com/bgruening/galaxytools/tree/master/chemicaltoolbox

### Packages / Tool Dependency Definitions

* *From [takadonet](https://toolshed.g2.bx.psu.edu/view/takadonet):*
    * [package_tbl2asn_23_7](https://toolshed.g2.bx.psu.edu/view/takadonet/package_tbl2asn_23_7): Contains a tool dependency definition that downloads the binary version 23.7 of tbl2asn. tbl2asn is an automated bulk submission program.
    * [package_minced_0_1_6](https://toolshed.g2.bx.psu.edu/view/takadonet/package_minced_0_1_6): Contains a tool dependency definition that downloads version 0.1.6 of minced, a CRISPR finder. MinCED is a program to find Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) in full genomes or environmental datasets such as metagenomes, in which sequence size can be anywhere from 100 to 800 bp. MinCED runs from the command-line and was derived from CRT (http://www.room220.com/crt/): Charles Bland ''et al.'', CRISPR Recognition Tool (CRT): a tool for automatic detection of clustered regularly interspaced palindromic repeats, BMC Bioinformatics 8, no. 1 (2007): 209.
    * [package_barrnap_0_5](https://toolshed.g2.bx.psu.edu/view/takadonet/package_barrnap_0_5): Contains a tool dependency definition that downloads and compiles version 0.4 of the barrnap. Barrnap predicts the location of 5S, 16S and 23S ribosomal RNA genes in Bacterial genome sequ It takes FASTA DNA sequence as input, and write GFF3 as output. https://github.com/Victorian-Bioinformatics-Consortium/barrnap

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
    * [package_numpy_1_9](https://toolshed.g2.bx.psu.edu/view/iuc/package_numpy_1_9): Contains a tool dependency definition that downloads and compiles version 1.9 of the the python numpy package. NumPy is the fundamental package for scientific computing with Python.
    * [package_blast_plus_2_2_30](https://toolshed.g2.bx.psu.edu/view/iuc/package_blast_plus_2_2_30): first version, based on BLAST+ 2.2.29 definition. NCBI BLAST+ 2.2.30 (binaries only)  This Tool Shed package is intended to be used as a dependency of the Galaxy wrappers for NCBI BLAST+ and any other tools which call the BLAST+ binaries internally.
    * [package_matplotlib_1_4](https://toolshed.g2.bx.psu.edu/view/iuc/package_matplotlib_1_4): Contains a tool dependency definition that downloads and compiles version 1.4.x of the the python matplotlib package. matplotlib is a python 2D plotting library which produces publication quality figures. This is the version 1.2.x of matplotlib. www.matplotlib.org/
    * [package_networkx_1_9](https://toolshed.g2.bx.psu.edu/view/iuc/package_networkx_1_9): Contains a tool dependency definition that downloads and compiles version 1.9.x of the python library networkx. NetworkX is a Python language software package for the creation, manipulation, and study of the structure, dynamics, and functions of complex networks. http://networkx.github.io/
    * [package_scipy_0_14](https://toolshed.g2.bx.psu.edu/view/iuc/package_scipy_0_14): Contains a tool dependency definition that downloads and compiles version 0.14 of the the scipy python library. SciPy is open-source software for mathematics, science, and engineering. The SciPy library is built to work with NumPy arrays, and provides many user-friendly and efficient numerical routines such as routines for numerical integration and optimization.  http://www.scipy.org/
    * [package_dill_0_2](https://toolshed.g2.bx.psu.edu/view/iuc/package_dill_0_2): Contains a tool dependency definition that downloads and compiles version 1.9.x of the python library dill. Dill extends python's 'pickle' module for serializing and de-serializing python objects to the majority of the built-in python types. Serialization is the process of converting an object to a byte stream, and the inverse of which is converting a byte stream back to on python object hierarchy. http://trac.mystic.cacr.caltech.edu/project/pathos/wiki/dill
    * [package_scikit_learn_0_15](https://toolshed.g2.bx.psu.edu/view/iuc/package_scikit_learn_0_15): Contains a tool dependency definition that downloads and compiles version 0.15.x of the the scikit-learn package. Easy-to-use and general-purpose machine learning in Python.  Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world, building upon numpy, scipy, and matplotlib. As a machine-learning module, it provides versatile tools for data mining and analysis in any field of science and engineering. It strives to be simple and efficient, accessible to everybody, and reusable in various contexts. http://scikit-learn.org/

* *From [saket-choudhary](https://toolshed.g2.bx.psu.edu/view/saket-choudhary):*
    * [package_xlrd_0_9_3](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_xlrd_0_9_3): Tool dependency definition of python-xlrd
    * [package_scikit_learn_0_15](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_scikit_learn_0_15): Tool dependency package for scikit-learn-0.15
    * [package_fastcluster_1_1_13](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_fastcluster_1_1_13): Tool dependency definition of python-fastcluster
    * [package_blas_3_5_0](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_blas_3_5_0): Tool dependency package for blas
    * [package_pyvcf_0_6_7](https://toolshed.g2.bx.psu.edu/view/saket-choudhary/package_pyvcf_0_6_7): Tool dependedency definition for PyVCF

* *From [lparsons](https://toolshed.g2.bx.psu.edu/view/lparsons):*
    * [package_cutadapt_1_6](https://toolshed.g2.bx.psu.edu/view/lparsons/package_cutadapt_1_6): Initial version Contains a tool dependency definition that downloads and compiles cutadapt version 1.6 trim adapters from high-throughput sequencing reads

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [package_picard_122](https://toolshed.g2.bx.psu.edu/view/devteam/package_picard_122): Picard 1.122 package definition This picard package dependency is retrieved directly from https://github.com/broadinstitute/picard/releases

### Workflows

* *From [bgruening](https://toolshed.g2.bx.psu.edu/view/bgruening):*
    * [chemicaltoolbox_library_hole_filling_workflow](https://toolshed.g2.bx.psu.edu/view/bgruening/chemicaltoolbox_library_hole_filling_workflow): Uploaded Given one library, it extends all molecules by similar molecules of an other library and thus fill gaps in an automatic manner. Given one library, it extends all molecules by similar molecules of an other library and thus fill gaps in an automatic manner. This workflow is part of case study demonstrating the capability of the chemicaltoolbox. For further information please have a look at the chemicaltoolbox: https://github.com/bgruening/galaxytools/tree/master/chemicaltoolbox

### Select Updates

* *From [lparsons](https://toolshed.g2.bx.psu.edu/view/lparsons):*
    * [cutadapt](https://toolshed.g2.bx.psu.edu/view/lparsons/cutadapt): Updated to version 1.6
    * [htseq_count](https://toolshed.g2.bx.psu.edu/view/lparsons/htseq_count): Deleted accidentally added file

* *From [saskia-hiltemann](https://toolshed.g2.bx.psu.edu/view/saskia-hiltemann):*
    * [ireport](https://toolshed.g2.bx.psu.edu/view/saskia-hiltemann/ireport): fixed dependencies and added MarkDown support

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [quality_filter](https://toolshed.g2.bx.psu.edu/view/devteam/quality_filter): tool definition that does not fail on stderr output.

* *From [crs4](https://toolshed.g2.bx.psu.edu/view/crs4):*
    * [sspace](https://toolshed.g2.bx.psu.edu/view/crs4/sspace): Update Orione citation. Update dependency to SSPACE Basic v2.1 . Add <citations>.
    * [prokka](https://toolshed.g2.bx.psu.edu/view/crs4/prokka): Use <stdio> because prokka writes some warnings on stderr. Update Orione citation. Update Prokka citation. Support Prokka 1.10. Upgrade dependencies to package_minced_0_1_6, package_barrnap_0_5 and package_tbl2asn_23_7. Added --proteins option. Add <citations>.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [package_galaxy_ops_1_0_0](https://toolshed.g2.bx.psu.edu/view/devteam/package_galaxy_ops_1_0_0): tool dependency definition that uses pip to install gops.

* *From [miller-lab](https://toolshed.g2.bx.psu.edu/view/miller-lab):*
    * [package_quicktree_1_1](https://toolshed.g2.bx.psu.edu/view/miller-lab/package_quicktree_1_1): tool_dependencies.xml with a URL that works for urllib.

# Other News

* Have you ever wanted to improve a Galaxy tool? Large number of tools are [now awaiting your pull requests at GitHub](https://github.com/galaxyproject/tools-devteam).
* [On the awesomeness of the BOSC/OpenBio Codefest 2014](http://etalog.blogspot.com/2014/10/on-awesomeness-of-boscopenbio-codefest.html)
* [Björn Grüning's](/people/bjoern-gruening/) [Galaxy Docker image](https://github.com/bgruening/docker-galaxy-stable/commits/master) now bundles FTP & SLURM and allows targeting external Galaxy directories.
* An IPython notebook demonstrating an [interactive cummeRbund analysis embedded in Galaxy](http://nbviewer.ipython.org/github/jmchilton/ipython_notebooks/blob/master/cummerbund_galaxy_1.ipynb).
* [GUGGO tutorial](http://bit.ly/ZQ7mdu): how to activate Docker functionality in Galaxy; create a Docker image and install Stacks on that image; and integrate a Stacks Galaxy tool using that image.
