---
title: The May 2015 Galactic News!
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 30em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the May 2015 *[Galactic News](/galaxy-updates/)*, a summary of what is going on in the Galaxy community. *[These newsletters](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.

# New Papers

66 new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in April. Highlights include:

* [The ReproGenomics Viewer: an integrative cross-species toolbox for the reproductive science community](https://doi.org/10.1093/nar/gkv345)
    Thomas A. Darde, Olivier Sallou, Emmanuelle Becker, Bertrand Evrard, Cyril Monjeaud, Yvan Le Bras, Bernard Jégou, Olivier Collin, Antoine D. Rolland and Frédéric Chalmel, *Nucleic Acids Research* (16 April 2015), gkv345, doi:10.1093/nar/gkv345

* [VirAmp: a galaxy-based viral genome assembly pipeline](http://www.gigasciencejournal.com/content/4/1/19)
    Yinan Wan1, Daniel W Renner2, Istvan Albert and Moriah L Szpara, *GigaScience* 2015, 4:19  doi:10.1186/s13742-015-0060-y

* [ClusTrack: Feature Extraction and Similarity Measures for Clustering of Genome-Wide Data Sets](https://doi.org/10.1371/journal.pone.0123261)
    Halfdan Rydbeck, Geir Kjetil K. Sandve, Egil Ferkingstad, Boris Simovski, Morten Rye, Eivind Hovig, *PloS one*, Vol. 10, No. 4. (2015)

* [StructureFold: Genome-wide RNA secondary structure mapping and reconstruction in vivo](https://doi.org/10.1093/bioinformatics/btv213)
    Yin Tang, [Emil Bouvier](/people/dave-bouvier/), Chun Kit Kwok, Yiliang Ding, [Anton Nekrutenko](/people/anton/), Philip C. Bevilacqua, and Sarah M. Assmann, *Bioinformatics* (16 April 2015), btv213, doi:10.1093/bioinformatics/btv213

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 41 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


# Events

## GCC2015: 4-8 July, Norwich UK

<div class='right'><a href='http://gcc2015.tsl.ac.uk/'><img src="/images/logos/GCC2015LogoWide600.png" alt="GCC2015" width="170" /></a><br />
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://gcc2015.tsl.ac.uk/registration/'>Register Now</a> &nbsp;&nbsp; </th>
  </tr>
</table>

</div>

The **[2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/)** is the Galaxy community's annual gathering of users, developers, and administrators.  [Previous GCC's](/events/gcc2014/) have drawn over 200 participants, and we expect that to happen again in 2015.  GCC2015 is being hosted by [The Sainsbury Lab](http://tsl.ac.uk/) in Norwich, UK, immediately before [BOSC](http://www.open-bio.org/wiki/BOSC_2015) and [ISMB/ECCB](http://www.iscb.org/ismbeccb2015) in Dublin.

There are a lot of events going on at GCC2015, including:

* **[Code Hackathon](http://gcc2015.tsl.ac.uk/Hackathon/)**, 4-5 July
* **[Data Wrangling Hackathon](http://gcc2015.tsl.ac.uk/data-hackathon/)**, 4-5 July (*new*)
* **[Training SunDay](http://gcc2015.tsl.ac.uk/Training-Day/)**, 5 July (*new*)
* **[Training Day](http://gcc2015.tsl.ac.uk/Training-Day/)**, 6 July
* **[GCC Meeting](http://gcc2015.tsl.ac.uk/Conference/)**, 7-8 July

### Early Registration is Open!

**But it closes 22 May!**

**Early [registration](http://gcc2015.tsl.ac.uk/abstracts/registration/) (save *heaps)* is now open**. Early registration is very affordable and starts at less than £40 per day for students and postdocs. If you work in data-intensive life science research, then it is hard find a meeting more relevant than GCC2015.  We look forward to seeing you there.

**[Register now](http://gcc2015.tsl.ac.uk/registration/).**

### Poster Abstract Deadline is TODAY

On-time [poster abstract submission](http://gcc2015.tsl.ac.uk/Abstracts) **closes today**.  Poster authors will be notified of acceptance status within two weeks. *Please consider presenting your work. If you are dealing with big biological data, then this meeting wants to hear about it.*

### Registration and Lodging Scholarships: Applications due TODAY

The [Galaxy Community Fund Board](mailto:community-fund@lists.galaxyproject.org) is pleased to offer [registration](http://gcc2015.tsl.ac.uk/registration) and [lodging](http://gcc2015.tsl.ac.uk/logistics/#Lodging) scholarships for [GCC2015](http://gcc2015.tsl.ac.uk/) for students and post-docs who are based in the United States, and who are members of historically underrepresented groups in life science research.

See the [full announcement](/news/gcc2015-scholarships/) for further details.

<div class='left'><br />
<a href='http://gcc2015.tsl.ac.uk/keynote-speaker/'><img src="/events/gcc2015/oliver_stegle.jpg" alt="Oliver Stegle will present the keynote address at GCC2015" width="100" /></a></div>

### Keynote Speaker: Oliver Stegle

The [GCC2015 Scientific Committee](http://gcc2015.tsl.ac.uk/organisers) is pleased to announce that the [GCC2015 Keynote Speaker](http://gcc2015.tsl.ac.uk/keynote-speaker/) is [Oliver Stegle](https://www.ebi.ac.uk/about/people/oliver-stegle) of the [Statistical genomics and systems genetic research group](http://www.ebi.ac.uk/stegle) at the [European Bioinformatics Institute (EBI)](http://www.ebi.ac.uk/).  Dr Stegle will speak on his work in *[Modeling molecular heterogeneity between individuals and single cells](http://gcc2015.tsl.ac.uk/keynote-speaker/).*

Dr Stegle is featured in a [Young Investigator Profile](https://www.genomeweb.com/informatics/young-investigator-profile-oliver-stegle) this month on [GenomeWeb](https://www.genomeweb.com/) (which also happens to be a GCC2015 Sponsor).
<br />

### Other Events Near GCC2015

Looking to extend your trip?  There are also **[at least 7 other events](/news/near-gcc2015/)** going on in the British Isles in the week before and week after GCC2015, including events

<div class='right'><a href='/news/near-gcc2015/'><img src="/news/near-gcc2015/MatrixSnap.png" alt="Events nearby GCC2015" width="150" /></a></div>

 In Norwich

* [iPlant Tools & Services Workshop](http://www.tgac.ac.uk/361_Division/training-programme/courses-workshops/tgac-events/iplant-tools-and-services/)
* [BioJS Hackathon](http://biojs.net/)

 In Cambridge

* [Exploring Human Host-Microbiome Interactions in Health and Disease](https://registration.hinxton.wellcome.ac.uk/display_info.asp?id=480)
* [Exploring Plant Variation Data](http://www.ebi.ac.uk/training/course/mining-plant-variation-data)
* [Human Genome Analysis: Genetic Analysis of Multifactorial Diseases](http://www.wellcome.ac.uk/Education-resources/Courses-and-conferences/Advanced-Courses-and-Scientific-Conferences/Advanced-Courses/WTX026851.htm)

 In Edinburgh

* [Introduction to NGS Data Analysis with Galaxy](https://genomics.ed.ac.uk/introduction-ngs-data-analysis-galaxy) *(get a jump on Training Day)*

 In Dublin

* [OpenBio Codefest 2015](http://www.open-bio.org/wiki/Codefest_2015)
* [ISMB / EECB 2015](http://www.iscb.org/ismbeccb2015) including [BOSC 2015](http://www.open-bio.org/wiki/BOSC_2015) and [BioVis 2015](http://www.biovis.net/)

See the [Events Aplenty Near GCC2015](/news/near-gcc2015/) news item for dates and deadlines.

### GCC2015 Sponsorships

We are pleased to announce new GCC2015 [Silver Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/) from [ELIXIR](https://www.elixir-europe.org/) and [GigaScience](http://www.gigasciencejournal.com/).

<div class='right'>
<a href='http://www.elixir-europe.org/'><img src="/images/logos/ElixirNoTextLogo.png" alt="Elixer" height="120" /></a> &nbsp;&nbsp;&nbsp;
</div>
[ELIXIR](https://www.elixir-europe.org/) is building a sustainable European infrastructure for biological information, supporting life science research and its translation to medicine, agriculture, bioindustries and society.  ELIXIR unites Europe’s leading life science organisations in managing and safeguarding the massive amounts of data being generated every day by publicly funded research. It is a pan-European research infrastructure for biological information.  This is the second time ELIXIR has sponsored a GCC.  Please welcome them back!

<div class='left'>
<a href='http://www.gigasciencejournal.com/'><img src="/images/logos/GigaScienceLogo250.png" alt="GigaScience Journal" height="70" /></a>
</div>

*[GigaScience](http://www.gigasciencejournal.com/)* journal is co-published as a collaboration between [BGI Shenzhen](http://www.genomics.cn/en/index) and [BioMed Central](http://www.biomedcentral.com/) focused on studies utilizing large-scale datasets and workflows. This is the third GCC that GigaScience has sponsored.  Please welcome them back!

*And ...* Any oral or poster presentation at GCC2015 will also be eligible for consideration for publication in the *[GigaScience](http://www.gigasciencejournal.com/)* "[Galaxy: Data Intensive and Reproducible Research](http://www.gigasciencejournal.com/series/Galaxy)" series (first announced for GCC2013).  *[GigaScience](http://www.gigasciencejournal.com/)* is pleased to offer a 15% discount on article publishing charges for accepted papers from GCC2015.


#### Call for Sponsors

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is still accepting [Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/).  Your organisation can play a prominent part in the Galaxy community by sponsoring GCC2015.  Sponsorship is an excellent way to raise your organization’s visibility.

Several sponsorship levels are available, including two levels of premier sponsorships that include presentations.  Premium sponsorships are limited, however, so you are encouraged to act soon.

Please let the [organisers](http://gcc2015.tsl.ac.uk/) know if you are interested in helping make this event a success.

## April GalaxyAdmins Meetup: Slides and Video

<div class='right'><a href='/community/galaxy-admins/meetups/2015-04-16/'><img src="/images/logos/GalaxyAdmins.png" alt="GalaxyAdmins meetup April 16" width="150" /></a></div>

[Slides and video](/community/galaxy-admins/meetups/2015-04-16/) from the [April GalaxyAdmins online meetup](/community/galaxy-admins/meetups/2015-04-16/) are now available. [Carrie Ganote](http://www.researchgate.net/profile/Carrie_Ganote) from the [National Center for Genome Analysis Support (NCGAS)](http://ncgas.org/) and [Pervasive Technology Institute](http://pti.iu.edu/) at [Indiana University](http://iu.edu/) presented her Galaxy work with [Trinity](http://trinityrnaseq.github.io/), IU Galaxy, and the [Open Science Grid](http://www.opensciencegrid.org/).  [Sam Guerler](/people/guerler/) of the Galaxy Project talked about Galaxy'a new tool form support.

*Please join us for the [next GalaxyAdmins meetup on June 18](/community/galaxy-admins/meetups/2015-06-18/)* when [Peter Briggs](http://galacticengineer.blogspot.co.uk/) of the [University of Manchester](http://www.manchester.ac.uk/) will speak on *Galactic Engineering: Experiences deploying Galaxy and developing tools within the Bioinformatics Core Facility at the University of Manchester*.  GalaxyAdmins will also have an in-person meetup at GCC2015.

[GalaxyAdmins](/community/galaxy-admins/) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  

## Other Events

<div class='center'>
<a href='http://migale.jouy.inra.fr/?q=formations'><img src="/images/logos/MIGALELogo.png" alt='Cycle "Bioinformatique par la pratique" 2015' height="50" /></a>
<a href='http://www.qfab.org/event/workshop-variant-detection-using-galaxy-3/'><img src="/images/logos/QFABLogo.png" alt="QFAB Workshop: Variant detection using Galaxy" height="80" /></a>
[[<a href='http://genomecenter.ucdavis.edu/2015/04/28/getting-bioinformatics-done-with-galaxy-a-free-bgi-uc-davis-workshop-taught-by-bioinformatics-core/'><img src="/images/logos/UCDavisGenomeCenter.png" alt="Getting Bioinformatics Done with Galaxy" height="50" /></a> &nbsp;
<a href='/events/meetups/psu/2015-05/'><img src="/events/meetups/p-s-u/PSUMeetupLogo600.png" alt="Galaxy Meetups @ Penn State" height="64" /></a> &nbsp;
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div>

There are [upcoming events](/events/) in 7 countries on 3 continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> May 12 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module17-12mai.pdf'>Initiation à l’utilisation de Galaxy</a></em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
    <td rowspan=2> <a href='http://www.jouy.inra.fr/'>Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin@jouy.inra.fr">Veronique Martin</a>, <a href="mailto:sophie.schbath@jouy.inra.fr">Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> May 13 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module8bis-13mai.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
  </tr>
  <tr>
    <th> May 13 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-variant-detection-using-galaxy-3/'>Workshop: Variant detection using Galaxy</a></em> </td>
    <td> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> May 13 </th>
    <td> <em><a href='http://genomecenter.ucdavis.edu/2015/04/28/getting-bioinformatics-done-with-galaxy-a-free-bgi-uc-davis-workshop-taught-by-bioinformatics-core/'>Getting Bioinformatics Done with Galaxy</a></em> </td>
    <td> <a href='http://research.ucdavis.edu/research/bgi-uc-davis/'>BGI @ UC Davis</a> workshop series, Davis, California, United States </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnUCDavis'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href="mailto:bioinformatics.core@ucdavis.edu">UC Davis Bioinformatics</a> </td>
  </tr>
  <tr>
    <th> May 15 </th>
    <td> <em><a href='/events/meetups/psu/2015-05/'>RNA-Seq Galaxy Workshop</a></em> </td>
    <td> <a href='/events/meetups/psu/2015-05/#where'>Penn State University</a>, State College, Pennsylvania, United States </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='/people/marten/'>Martin Cech</a>, <a href='/people/anton/'>Anton Nekrutenko</a> </td>
  </tr>
  <tr>
    <th> May 22, 29 </th>
    <td> <em><a href='http://www.vib.be/en/training/research-training/courses/Pages/Introduction-to-the-analysis-of-NGS-data.aspx'>Introduction to the analysis of NGS data</a></em> </td>
    <td> Leuven, Belgium </td>
    <td> <a href='http://www.vib.be/whoiswho/Pages/Janick-Mathys.aspx?lang=en'>Janick Mathys</a> </td>
  </tr>
  <tr>
    <th> May 25-29 </th>
    <td> <a href='http://www.mipro.hr/MIPRO2015.DCVIS/ELink.aspx'>MIPRO</a> </td>
    <td> Opatija, Croatia </td>
    <td> <a href='/people/enis-afgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> May&nbsp;26-28 </th>
    <td> <a href='http://w3lamc.umbr.cas.cz/repeatexplorer/?page_id=14'>Workshop on the Application of Next Generation Sequencing to Repetitive DNA Analysis in Plants</a> </td>
    <td> České Budějovice, Czech Republic </td>
    <td> <a href="mailto:macas@umbr.cas.cz">Jiri Macas</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 2 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module17-2juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
    <td rowspan=3> <a href='http://www.jouy.inra.fr/'>Jouy-en-Josas</a>, France </td>
    <td rowspan=3> <a href="mailto:veronique.martin@jouy.inra.fr">Veronique Martin</a>, <a href="mailto:sophie.schbath@jouy.inra.fr">Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 3 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module8bis-3juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
  </tr>
  <tr>
    <th> June 4 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2015/module18.pdf'>Traitement bioinformatique des données RNA-Seq sous Galaxy</a></em> <div class='indent'>Partie du <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2015</a></div> </td>
  </tr>
  <tr>
    <th> June 3 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-rna-seq-analysis-using-galaxy-2/'>Workshop: RNA-Seq analysis using Galaxy</a></em> </td>
    <td> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> June 6-9 </th>
    <td> <em><a href='https://www.eshg.org/684.0.html'>WS02: NGS Variant Analysis with Galaxy</a></em> </td>
    <td> <a href='https://www.eshg.org/eshg2015.0.html'>ESHG 2015</a>, Glasgow, United Kingdom </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> June 18 </th>
    <td> <em><a href='/community/galaxy-admins/meetups/2015-06-18/'>Galactic Engineering: Experiences deploying Galaxy and developing tools within the Bioinformatics Core Facility at the University of Manchester</a></em> </td>
    <td> <a href='/community/galaxy-admins/meetups/2015-06-18/'>June 2015 GalaxyAdmins Web Meetup</a>, <em>Online</em> </td>
    <td> <a href='http://galacticengineer.blogspot.co.uk/'>Peter Briggs</a>, <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> </td>
  </tr>
  <tr>
    <th> June 22 </th>
    <td> <em>Introduction to variant calling for pathologists and laboratory managers</em> </td>
    <td> Part of <a href='http://www.rcpa.edu.au/Events/Event-Details/2015/June/Short-Course-in-Medical-Genetics-and-Genetic-Patho'>Short Course in Medical Genetics and Genetic Pathology 2015</a>, Sydney, Australia </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='/people/fubar/'>Ross Lazarus</a>, <a href='https://www.vlsci.org.au/researcher/alonie'>Andrew Lonie</a> </td>
  </tr>
  <tr>
    <th> June 29-30 </th>
    <td> <em><a href='https://genomics.ed.ac.uk/introduction-ngs-data-analysis-galaxy'>Introduction to NGS Data Analysis with Galaxy</a></em> </td>
    <td> <a href='http://www.ed.ac.uk/home'>University of Edinburgh</a>, Edinburgh, United Kingdom </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href="mailto:bert.overduin@ed.ac.uk">Bert Overduin</a>, <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> July 4-8 </th>
    <td> <strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a></strong> </td>
    <td rowspan=6> <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> GCC2015 Organisers </td>
  </tr>
  <tr>
    <th rowspan=2> July 4-5 </th>
    <td> <div class='indent'><strong><a href='http://gcc2015.tsl.ac.uk/data_hackathon'>Data Wrangling Hackathon</a></div></strong> </td>
    <td> Data Hackathon Organisers </td>
  </tr>
  <tr>
    <td> <div class='indent'><strong><a href='http://gcc2015.tsl.ac.uk/hackathon'>Coding Hackathon</a></div></strong> </td>
    <td> Coding Hackathon Organisers </td>
  </tr>
  <tr>
    <th> July 5 </th>
    <td> <div class='indent'><strong><a href='http://gcc2015.tsl.ac.uk/training_day'>Training SunDay</a></div></strong> </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> GCC2015 Organisers  </td>
  </tr>
  <tr>
    <th> July 6 </th>
    <td> <div class='indent'><strong><a href='http://gcc2015.tsl.ac.uk/training_day'>Training (Mon)Day</a></div></strong> </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> GCC2015 Organisers  </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> <div class='indent'><strong><a href='http://gcc2015.tsl.ac.uk/'>GCC2015 Community Meeting</a></strong></div> </td>
    <td> GCC2015 Organisers </td>
  </tr>
  <tr>
    <th> July 6-10 </th>
    <td> <em><a href='http://bioinformatics.org.au/ws15/wp-content/uploads/ws14/sites/9/2015/02/Galaxy-Workshop-Information-Sheet.pdf'>An Introduction to Galaxy with the Genomics Virtual Lab: workshop</a></em> </td>
    <td> <a href='http://bioinformatics.org.au/ws15/registration/'>Winter School in Mathematical and Computational Biology</a>, University of Queensland, Brisbane, Queensland, Australia </td>
    <td> <div class='right'><a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> July 10-14 </th>
    <td> <em><a href='http://www.iscb.org/ismbeccb2015-program/ismbeccb2015-akes#akes4'>Using Biological Cyberinfrastructure to Scale Science and People – Applications in Data Storage, HPC, Cloud Analysis, and Bioinformatics Training</a></em> </td>
    <td> <a href='http://www.iscb.org/ismbeccb2015'>ISMB / ECCB 2015</a>, <a href='http://www.open-bio.org/wiki/BOSC_2015'>BOSC 2015</a>, Dublin, Ireland </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='https://training.galaxyproject.org/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Post-Doc CDD-IR CEA-Institut de Génomique](http://bit.ly/1Pbd9gk), Evry, France
* [Software Developer / Bioinformatician](http://bit.ly/1HVpxk3) at EMBL, Heidelberg. Closes 25 May
* [Computational Metabolomics Professor](https://psu.jobs/job/55363), Penn State University, Pennsylvania, United States
* [Software Developer for the Refinery Team](https://jobs.brassring.com/tgwebhost/jobdetails.aspx?jobId=1118216&PartnerId=25240&SiteId=5341&type=mail&JobReqLang=1&recordstart=1&JobSiteId=5341&JobSiteInfo=1118216_5341&gqid=0), Boston, Massachusetts, United States
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

----

# New Public Galaxy Server: The ReproGenomics Viewer

<div class='right solid'><a href='http://rgv.genouest.org/galaxy/'><img src="/public-galaxy-servers/archive/repro-genomics-viewer.png" alt="The ReproGenomics Viewer"  /></a></div>

* *Link:*
    * [The ReproGenomics Viewer working environment](http://rgv.genouest.org/galaxy/)
    * [The ReproGenomics Viewer: an integrative cross-species toolbox for the reproductive science community](http://bit.ly/1PZFfgV), by Thomas A. Darde, Olivier Sallou, Emmanuelle Becker, Bertrand Evrard, Cyril Monjeaud, Yvan Le Bras, Bernard Jégou, Olivier Collin, Antoine D. Rolland and Frédéric Chalmel1; *Nucleic Acids Research* (2015) doi: 10.1093/nar/gkv345
    * [The ReproGenomics Viewer project home page](http://rgv.genouest.org/)
* *Domain/Purpose:*
    * "The ReproGenomics Viewer (RGV) is a cross-species genomic toolbox for the reproductive community. The system is based on the implementation of a JBrowse genome browser and a Galaxy bioinformatics workflow environment. It was developed to provide a one-stop genomic working environment and aims to assist scientists in the analysis and the mining of a wide range of high-throughput repro-genomics data, including sequencing data.<br /><br />RGV allows hosting, visualization, and direct comparison of users’ data to published genomics studies as well as to relevant genetic variations linked to reproduction. One way it does this is by enabling various genomic file format conversions.<br /><br />The ReproGenomics Viewer currently hosts datasets that are oriented mainly towards testis biology and spermatogenesis. In the near future, these will extend to other areas of reproduction, including gonad development, urogenital cancers, and reproductive toxicology."
* *Comments:*
* *User Support:*
    * [Email](mailto:support@genouest.org)
* *Quotas:*
* *Sponsor(s):*
    * [INSERM](http://www.inserm.fr), [IRSET](http://www.irset.fr), [FRM](http://www.frm.fr), [ANSES](http://www.anses.fr), [Biogenouest](http://www.biogenouest.org), [IRISA](http://www.irisa.fr), [INRIA](http://www.inria.fr/), [CNRS](http://www.cnrs.fr/), [Université de Rennes 1](http://www.univ-rennes1.fr/), [Région Bretagne](http://www.bretagne.fr/), FEDER]]

----

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/logos/GTNLogo600.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

<br />

One new [Community Log Board](/news/) entry was added in April:

* [Using GALAXY_SLOTS with multithreaded Galaxy tools ](http://galacticengineer.blogspot.co.uk/2015/04/using-galaxyslots-for-multithreaded_22.html)
    by [Peter Briggs](http://www.ls.manchester.ac.uk/people/profile/?alias=briggsp)

And one new Training Resource:

* Differential Expression with Tuxedo
    by [Cynthia Gibas](http://bioinformatics.uncc.edu/directory/gibas-cynthia)

----

# Releases

## Pulsar 0.4.0

[Pulsar 0.4.0 and 0.3.0](https://pypi.python.org/pypi/pulsar-app/0.4.0) were released in April.  These included:

* Python 3 support. [Pull Request 62](https://github.com/galaxyproject/pulsar/pull/62)
* Add pulsar-run script for issues commands against a Pulsar server (experimental). [3cc7f74](https://github.com/galaxyproject/pulsar/commit/3cc7f74)
* Changed the name of project to Pulsar, moved to [Github](https://github.com/galaxyproject/pulsar/).
* New RESTful web services interface.
* SCP and Rsync file staging options added by [Helena Rasche](/people/helena-rasche/). [Pull Request](https://github.com/galaxyproject/pulsar/pull/34)
* Allow YAML based configuration.
* Support for more traditional pip/setup.py-style installs.

And dozens of smaller bugfixes and documentation updates.

Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

## Planemo 0.8.4

Planemo saw [two major and several minor](https://pypi.python.org/pypi/planemo/0.8.4#history) releases this past month.  The most recent is [v0.8.4](https://pypi.python.org/pypi/planemo/0.8.4).

Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.

## Others

 **Galaxy v15.03**
   The [most recent Galaxy release is V 15.03](/archive/dev-news-briefs/2015-03/), released in March.

 **[CloudMan](/cloudman/) **
   The [most recent edition of CloudMan](/galaxy-updates/2014-09/#august-2014-cloudman-release) was released in August.

 **BioBlend V0.5.3**
   [BioBlend v0.3](/news/bio-blend-v0-5-3/) was also released in March.<br />
   [BioBlend](https://github.com/galaxyproject/bioblend/blob) is a python library for interacting with CloudMan and the [Galaxy API](/develop/api/).  ([CloudMan](/cloudman/) offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)

 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and BioCloudCentral.


----

# Other News

* [QFAB Bioinformatics Training passes the 1000 participant mark](http://www.qfab.org/2015/04/27/qfab-bioinformatics-training-1000-and-counting/)
* A [proposal to](http://ivory.idyll.org/blog/2015-osf-galaxy-dibs-funding.html) integrate the [Center for Open Science’s](http://centerforopenscience.org/) [Open Science Framework (OSF)](https://osf.io/) with Galaxy as a remote data store [has been funded](http://ivory.idyll.org/blog/2015-osf-galaxy-dibs-funding.html).
* [How Disruptive Are MOOCs? Hopkins Genomics MOOC Launches In June](http://www.forbes.com/sites/stevensalzberg/2015/04/13/how-disruptive-are-moocs-hopkins-genomics-mooc-launches-in-june/), By Steven Salzberg, *Forbes*
* [NIAID-led Partnership Seeks to Establish Bioinformatics Centers in Africa](https://www.genomeweb.com/informatics/niaid-led-partnership-seeks-establish-bioinformatics-centers-africa) by Uduak Grace Thomas, in *GenomeWeb*

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in April](/toolshed/contributions/2015-04/).
