---
title: 'Viral pathogen data analysis beyond BY-COVID'
date: '2024-09-16'
tease: "A look back at the BY-COVID project and ahead to upcoming developments"
authors: Wolfgang Maier
subsites: [global, eu, freiburg]
tags: [COVID-19, virology, community]
supporters: [unifreiburg, by-covid]
---
# Galaxy Europe and BY-COVID

<img style="float: left; width: 150px;" alt="BY-COVID banner" src="https://avatars.githubusercontent.com/by-covid?s=100" />

In autumn 2021 the EU-funded [BY-COVID project](https://by-covid.org/) was launched to support the response to COVID-19 and our preparedness for future infectious disease outbreaks. Over the past 3 years, the SARS-CoV-2 data analysis efforts of the Galaxy Europe Team have been carried out under the umbrella of BY-COVID with remarkable results.

<div style="clear: left;"></div>

Among other things, we have:

1. built an **automated, scalable SARS-CoV-2 genome surveillance system around Galaxy**, which has become a [showcase](https://www.infectious-diseases-toolkit.org/showcase/covid19-galaxy) project within the [Infectious Diseases Toolkit (IDTk)](https://www.infectious-diseases-toolkit.org/) developed under BY-COVID, and have used it to analyze nearly 500,000 public SARS-CoV-2 sequencing datasets.

2. collaborated with researchers from the KwaZulu-Natal Research Innovation and Sequencing Platform (KRISP), South Africa, and international colleagues on **a first characterization of the omicron variant of concern** when it was first observed in late 2021. Read the [original blog post](https://galaxyproject.org/news/2021-11-29-omicron-and-galaxy/) and the resulting [publication](https://doi.org/10.1093/molbev/msac061) for more.

3. participated in the **NIH-led [ACTIV TRACE](https://www.nih.gov/research-training/medical-research-initiatives/activ/tracking-resistance-coronavirus-evolution-trace) initiative**, under which we helped improve and standardize SARS-CoV-2 genome sequencing pipelines of government agencies and industry. Read the [publication](https://doi.org/10.3390/v16030430).

4. **extended our analysis support to additional viruses** beyond SARS-CoV-2 and have created solutions for [influenza](https://usegalaxy.eu/published/page?id=a04ab8d6ecb698fa) and [pox virus](https://usegalaxy.eu/published/page?id=9172077f52d9d49f) sequencing data analysis.

5. offered **SARS-CoV-2/One Health/Pathogen data analysis tracks** as part of the **GTN Smörgåsbord** in [2022](https://gallantries.github.io/video-library/events/smorgasbord2/tapas.html) and [2023](https://gallantries.github.io/video-library/events/smorgasbord3/), and the [2024 Galaxy Training Academy](https://training.galaxyproject.org/training-material/events/galaxy-academy-2024.html) in October will feature a dedicated BY-COVID track.

   <a href="https://training.galaxyproject.org/training-material/events/galaxy-academy-2024.html" target="_blank"><img style="width: 500px; margin-top: 1rem;" alt="Galaxy Training Academy 2024 announcement" src="./banner_gta_1000x300.png" /></a>

# Looking ahead

<a href="https://virology.usegalaxy.eu" target="_blank"><img style="float: right; width: 200px; margin-left: 1rem;" alt="Virology subdomain logo" src="https://usegalaxy-eu.github.io/assets/media/virologyeu_logo.png" /></a>


BeYond-COVID will reach the end of its funding period in October, but keeping with the project's full name, members of the Galaxy Project and the Galaxy Europe Team will continue their efforts to support high-quality viral pathogen data analysis on the Galaxy platform through the development of tools, workflows, training material and through maintenance of data resources.

To make it easier for you to discover what is available and keep up with future developments, we have set up the subdomain **https://virology.usegalaxy.eu** earlier this year, which we hope you will find useful. See you there and thanks for using Galaxy!

