---
title: 'February 2019 Galactic News'
tease: 'Events (GCC Training, GalaxyAdmins), Platforms, Pubs, Jobs, COTM!, Doc, Training, Tools, Releases and more'
date: "2019-02-01"
subsites: [global, us]
---

[<img class="float-right" src="/news/2019-02-galaxy-update/2019-02-news-summary.png" alt="Galaxy News" width="350" />](/galaxy-updates/)

The *February  2019 Galactic News* is here!  This is a summary of what is going on in the Galaxy community.

* [Event News](/news/2019-02-galaxy-update/#event-news)
    * **[Voting on GCC2019 training topics open through February 4th!](/news/2019-02-galaxy-update/#gcc2019-training-topic-voting-is-open)**.
    * **[ELIXIR Galaxy Community Workshop](/news/2019-02-galaxy-update/#elixir-galaxy-community-workshop)** pre-registration is open!
* [GalaxyAdmins needs your topic suggestions](/news/2019-02-galaxy-update/#galaxyadmins-needs-your-topic-suggestions).
* [160 new publications](/news/2019-02-galaxy-update/#publications), great resources lead to great insight.
* Some most excellent [Galaxy Platform News](/news/2019-02-galaxy-update/#galaxy-platforms-news), including proteo-transcriptomics, RNA analyis tools, Genome Browsers, and more!
* The Tutorial of the Month is ... [Genome annotation with Prokka](/news/2019-02-galaxy-update/#new-galactic-blog-posts)!
* At least [23 Open positions](/news/2019-02-galaxy-update/#whos-hiring) in 6 countries on 2 continents.
* [Doc, Hub, and Training Updates](/news/2019-02-galaxy-update/#doc-hub-and-training-updates), including *GO enrichment analysis*  and *Age prediction using machine learning*.
* [ToolShed contributions](/news/2019-02-galaxy-update/#toolshed-contributions).
* [New releases](/news/2019-02-galaxy-update/#releases) of galaxy-lib and Planemo.
* And [a bunch of other news](/news/2019-02-galaxy-update/#other-news) too.

If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# Event News

## GCC2019 Training Topic Voting Is Open

[<img class="float-right" src="/news/2018-12-gcc2019-training-nomination/training_gcc_logo_blue_bg.png" alt="GCC2019" style="max-width: 300px"/>](https://docs.google.com/forms/d/1fBPjEmw7Td52rhBnC0SvflnQCD7Ue5IMJ9RkM0SjnYg/)

The [**2019 Galaxy Community Conference (GCC2019)**](/events/gcc2019/) will be held in **Freiburg, Germany, 1-6 July**. Like [previous Galaxy Community Conferences](/gcc/), GCC2019 will feature invited keynotes, accepted talks, posters, demos, birds-of-a-feather gatherings, and **training**. The format of GCC2019 will be a bit different than the previous years: it starts with 1 day of training and then 3 days with more specialized training sessions aligned with talks. *You can read more about the new format on the [conference FAQ](/events/gcc2019/faq/#what-will-be-the-format-for-gcc-in-2019).*

The GCC2019 Organizing Committee compiled [training topics nominated by the community](https://galaxyproject.org/events/gcc2019/training/#training-topic-votes) and expanded this list with community requests and topics from the previous GCC to offer a broad range of training sessions to be voted on by the community for GCC2019. Voting closes February 4th.

**Training topics that are offered at GCC2019 are determined by *the community* so *[BY YOU!](https://docs.google.com/forms/d/e/1FAIpQLSd75S6n0GwYTrAp66h1mwL8DbnYsum71ffoLVxzKSIR34eL_w/viewform)***

You can vote for as many topics as you want, but please note that the more topics you vote for the less your vote for each one counts. What? Here's an example. If Moni votes for 4 topics then each of her votes counts for 1/4 of a point. If Dave votes for 22 topics then each of his votes counts for 1/22 of a point.

Please take a moment and [**vote**](https://docs.google.com/forms/d/e/1FAIpQLSd75S6n0GwYTrAp66h1mwL8DbnYsum71ffoLVxzKSIR34eL_w/viewform) on which training topics you would like to see at GCC2019, your vote matters!

## ELIXIR Galaxy Community Workshop

[<img class="img-fluid mx-auto float-right" style="max-width: 160px" src="/images/logos/ElixirNoTextLogo.png" alt="GalaxyAdmins" />](/community/galaxy-admins/)

Join the Galaxy Community of Elixir for a [three day workshop](https://www.france-bioinformatique.fr/fr/evenements/elixir-galaxy-community-workshop-roscofffrance) at Station Biological de Roscoff, France to participate in workshops and hackathons focused on tool implementation in Galaxy, development of training materials for the Galaxy Training Network (GTN), and how to teach using GTM materials.

During the first 1 1/2 days, the attendees will learn about the integration of high-quality tools within Galaxy with their dependencies (Bioconda, Planemo) using the IUC best practice guidelines. They will also learn how to use Galaxy as a training tool and develop training material for the Galaxy Training Network. The second half of the workshop will be dedicated to Hackathon sessions where attendees will be able to bring their own projects around tool integration and/or training material and develop them collaboratively, with the support of community experts. Additional information can be found [here](https://www.france-bioinformatique.fr/fr/evenements/elixir-galaxy-community-workshop-roscofffrance).

Find pre-registration **[here](https://www.france-bioinformatique.fr/fr/evenements/elixir-galaxy-community-workshop-roscofffrance)**.


## GalaxyAdmins needs your topic suggestions

[<img class="img-fluid mx-auto float-right" src="/images/galaxy-logos/GalaxyAdmins_64.png" alt="GalaxyAdmins" />](/community/galaxy-admins/)

After a two year hiatus, the [bimonthly online GalaxyAdmins meetups](/community/galaxy-admins/meetups/) will return on [February 28](/community/galaxy-admins/meetups/2019-03-21/). [GalaxyAdmins](/community/galaxy-admins/) is a discussion group for Galaxy community members who are responsible for Galaxy installations. Our online meetups are around an hour long and feature a presentation followed by an open discussion.  It's a great place to catch up on what your fellow admins are thinking about.

**But, before we start meeting this month, [we need to know what you care about](https://docs.google.com/forms/d/e/1FAIpQLSfv257YcdQQO7gp3K4USoBzqkd-UDEblsto5UhU34gFlXTiPQ/viewform).**  What do you want to learn?  Who do you want to hear from?  What's causing you trouble?  Or, what have you worked that your fellow GalaxyAdmins would benefit from knowing?

Got some ideas? **[Nominate them](https://docs.google.com/forms/d/e/1FAIpQLSfv257YcdQQO7gp3K4USoBzqkd-UDEblsto5UhU34gFlXTiPQ/viewform)**

## Upcoming events

These and [other Galaxy related events](/events/) are coming up in the next few months:

<table class="table table-striped">
<thead>
<tr>
   <th>Date</th>
   <th>Topic/Event</th>
   <th>Venue/Location</th>
   <th>Contact</th>

</tr>
</thead>
<tbody>
<tr>
  <td><span class="text-nowrap">January 31st 2019</span></td>
  <td><a href="https://github.com/manabuishii/workflow-meetup/wiki/20190131">12th Workflow Meetup</a></td>
  <td><img style="float:right;" src="/images/icons/AS.png"/>理化学研究所, Osaka and Tokyo, Japan</td>
  <td>Manabu Ishii</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 4th 2019</span></td>
  <td><a href="http://formationcontinue.agrocampus-ouest.fr/infoglueDeliverLive/toutes-sessions/programme?idModule=733">Traitement de données de séquences par Galaxy</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.agrocampus-ouest.fr/ecole/implantations/campus-de-rennes">AGROCAMPUS OUEST, Rennes, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sandrine Lagarrigue</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 21st 2019</span></td>
  <td><a href="/events/2019-02-gtn">GTN CoFest and call on Galaxy Training Material</a></td>
  <td><img style="float:right;" src="/images/icons/GL.png"/>Online</td>
  <td>Bérénice Batut, Saskia Hiltemann</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 25th 2019</span></td>
  <td><a href="https://usegalaxy-eu.github.io/event/2018-11-22-Galaxy-workshop-February2019/">Galaxy HTS data analysis workshop</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://usegalaxy-eu.github.io/event/2018-11-22-Galaxy-workshop-February2019/#venue">University of Freiburg, Freiburg, Germany</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>contact at usegalaxy.eu</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 28th 2019</span></td>
  <td><a href="/community/galaxy-admins/meetups/2019-03-21/">GalaxyAdmins February 2019 Meetup</a></td>
  <td><img style="float:right;" src="/images/icons/GL.png"/>Online</td>
  <td>Helena Rasche, Dave Clements</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 6th 2019</span></td>
  <td><a href="https://www.denbi.de/training/532-galaxy-for-linking-bisulfite-sequencing-with-rna-sequencing">Galaxy for linking bisulfite sequencing with RNA sequencing</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>University of Rostock, Rostock, Germany</td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>destair [at] leibniz-fli.de</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 18th 2019</span></td>
  <td><a href="http://meetings.ipk-gatersleben.de/grc2019-abc/">Gatersleben Research Conference: Applied Bioinformatics for Crops</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://meetings.ipk-gatersleben.de/grc2019-abc/accomodation/">Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany</a></td>
  <td>Björn Grüning</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2019</span></td>
  <td><a href="http://meetings.ipk-gatersleben.de/compbiostarter/index.php/program/">Introduction into Galaxy and how to design your own analysis workflow</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://meetings.ipk-gatersleben.de/compbiostarter/">Computational Biology Starter, Leibniz Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany</a></td>
  <td>Anne Fiebig</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 23rd 2019</span></td>
  <td><a href="https://www.france-bioinformatique.fr/fr/evenements/elixir-galaxy-community-workshop-roscofffrance">Elixir Galaxy Community - Workshop at Roscoff/France</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.google.com/maps?ll=48.726635,-3.987479&amp;z=14&amp;t=m&amp;hl=en&amp;gl=US&amp;mapclient=embed&amp;cid=7841494204688338171">Station Biological De Roscoff, Roscoff, France</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Gildas Le Corguillé</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 1st 2019</span></td>
  <td><a href="/events/gcc2019/">2019 Galaxy Community Conference (GCC2019)</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.konzerthaus.freiburg.de/">Freiburg, Germany</a></td>
  <td><a href="https://training.galaxyproject.org/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
</tbody>
</table>

----

# Publications

**160 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in January.

## Highlighted Publications

*[Galactic](https://www.zotero.org/groups/1732893/galaxy/tags/%2BGalactic)* and *[Stellar](https://www.zotero.org/groups/1732893/galaxy/tags/%2BStellar)* publications from January:

* **[G-OnRamp: A Galaxy-based platform for creating genome browsers for collaborative genome annotation](https://doi.org/10.1101/499558)**, Yating Liu, Luke Sargent, Wilson Leung, Sarah C.R. Elgin, [Jeremy Goecks](/people/jeremy-goecks/). *bioRxiv* 499558; doi: https://doi.org/10.1101/499558
* **[Galaxy mothur Toolset (GmT): a user-friendly application for 16S rRNA gene sequencing analysis using mothur](https://doi.org/10.1093/gigascience/giy166)**, Saskia D Hiltemann, Stefan A Boers, Peter J van der Spek, Ruud Jansen, John P Hays, Andrew P Stubbs. *GigaScience*, giy166, doi:10.1093/gigascience/giy166
* **[MEGARes: an antimicrobial resistance database for high throughput sequencing](https://doi.org/10.1093/nar/gkw1009)**, Steven M. Lakin, Chris Dean, Noelle R. Noyes, Adam Dettenwanger, Anne Spencer Ross, Enrique Doster, Pablo Rovira, Zaid Abdo, Kenneth L. Jones, Jaime Ruiz, Keith E. Belk, Paul S. Morley, Christina Boucher. *Nucleic Acids Research*, Volume 45, Issue D1, 4 January 2017, Pages D574–D580, doi: 10.1093/nar/gkw1009
* **[Peptimapper: proteogenomics workflow for the expert annotation of eukaryotic genomes](https://doi.org/10.1186/s12864-019-5431-9)**, Laetitia Guillot, Ludovic Delage, Alain Viari, Yves Vandenbrouck, Emmanuelle Com, Andrés Ritter, Régis Lavigne, Dominique Marie, Pierre Peterlongo, Philippe Potin and Charles Pineau. *BMC Genomics* 201920:56 doi:10.1186/s12864-019-5431-9
* **[Growing and cultivating the forest genomics database, TreeGenes](https://doi.org/10.1093/database/bay084)**, Taylor Falk, Nic Herndon, Emily Grau, Sean Buehler, Peter Richter, Sumaira Zaman, Eliza M Baker, Risharde Ramnath, Stephen Ficklin, Margaret Staton, Frank A Feltus, Sook Jung, Doreen Main, Jill L Wegrzyn. *Database*, Volume 2018, 1 January 2018, bay084, doi: 10.1093/database/bay084
* **[Exploring the Effects of Spaceflight on Mouse Physiology using the Open Access NASA GeneLab Platform](https://www.jove.com/video/58447/exploring-effects-spaceflight-on-mouse-physiology-using-open-access)**, Afshin Beheshti, Yasaman Shirazi-Fard, Sungshin Choi, Daniel Berrios, Samrawit G. Gebre, Jonathan M. Galazka, Sylvain V. Costes. Journal of Visual Experiments (143), e58447, doi:10.3791/58447 (2019)
* **[Mind the gaps: overlooking inaccessible regions confounds statistical testing in genome analysis](https://doi.org/10.1186/s12859-018-2438-1)**, Diana Domanska, Chakravarthi Kanduri, Boris Simovski and Geir Kjetil Sandve. *BMC Bioinformatics* 201819:481, doi: 10.1186/s12859-018-2438-1
* **[VAPPER: High-throughput Variant Antigen Profiling in African trypanosomes](https://doi.org/10.1101/492074)**, Sara Silva Pereira, John Heap, Andrew R Jones, Andrew P. Jackson. *bioRxiv* 492074; doi: 10.1101/492074
* **[PrediTALE: A novel model learned from quantitative data allows for new perspectives on TALE targeting](https://doi.org/10.1101/522458)**, Annett Erkes, Stefanie Mücke, Maik Reschke, Jens Boch, Jan Grau. *bioRxiv* 522458; doi: 10.1101/522458
* **[Enhancing Access to Digital Media: The Language Application Grid in the HTRC Data Capsule](https://doi.org/10.1145/3093338.3104171)**, James Pustejovsky, Marc Verhagen, Keongmin Rim, Yu Ma, Liang Ran, Samitha Liyanage,  Jaimie Murdock, Robert H. McDonald, Beth Plale. In PEARC17 Proceedings of the Practice and Experience in Advanced Research Computing 2017 on Sustainability, Success and Impact, Article No. 60. doi: 10.1145/3093338.3104171
* **[Predicting runtimes of bioinformatics tools based on historical data: Five years of Galaxy usage](https://doi.org/10.1093/bioinformatics/btz054)**, Anastasia Tyryshkina, [Nate Coraor](/people/nate/), [Anton Nekrutenko](/people/anton/). *Bioinformatics*, btz054, doi: 10.1093/bioinformatics/btz054
* **[Tools for Understanding miRNA–mRNA Interactions for Reproducible RNA Analysis](https://link.springer.com/protocol/10.1007%2F978-1-4939-8982-9_8)**, Andrea Bagnacani, Markus Wolfien, Olaf Wolkenhauer. In: Lai X., Gupta S., Vera J. (eds) Computational Biology of Non-Coding RNA. Methods in Molecular Biology, vol 1912. Humana Press, New York, NY
* **[QuanTP: A Software Resource for Quantitative Proteo-Transcriptomic Comparative Data Analysis and Informatics](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00727)**, Praveen Kumar, Priyabrata Panigrahi, James Johnson, Wanda J. Weber, Subina Mehta, Ray Sajulga, Caleb  Easterly, Brian A. Crooker, Mohammad Heydarian, Krishanpal Anamika, Timothy J. Griffin , and Pratik D. Jagtap. *Journal of Proteome Research*, Article ASAP
DOI: 10.1021/acs.jproteome.8b00727
* **[*miCloud:* A Plug-n-Play, Extensible, On-Premises Bioinformatics Cloud for Seamless Execution of Complex Next-Generation Sequencing Data Analysis Pipelines](https://www.liebertpub.com/doi/10.1089/cmb.2018.0218)**, Baekdoo Kim, Thahmina Ali, Changsu Dong, Carlos Lijeron, Raja Mazumder, Claudia Wultsch, and Konstantinos Krampis. *Journal of Computational Biology* doi: 10.1089/cmb.2018.0218

Ten of the fourteen highlighted publications are open access.

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag |
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 111 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/tags/+Methods) | | 47 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/tags/+UsePublic) | | 23 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/tags/+UseMain) | | 16 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/tags/+Workbench) |
| 11 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/tags/+UseLocal) | | 10 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/tags/+RefPublic) | | 10 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/tags/+IsGalaxy) | | 9 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/tags/+Tools) |
| 6 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/tags/+Reproducibility) | | 3 | [+Other](https://www.zotero.org/groups/1732893/galaxy/tags/+Other) | | 2 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/tags/+Cloud) | | 2 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/tags/+HowTo) |
| 2 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/tags/+Shared) | | 2 | [+Project](https://www.zotero.org/groups/1732893/galaxy/tags/+Project) | | | | | | | |

----

# Galaxy Platforms News

The [Galaxy Platform Directory](/use/) lists resources for easily running your analysis on Galaxy, including publicly available servers, cloud services, and containers and VMs that run Galaxy.  Here's what's been added since last month:

[<img class="float-right" style="max-width: 300px" src="/use/quantp/quantp-fig3-400.jpg" alt="QuanTP" />](/use/quantp/)

## QuanTP

[QuanTP](https://z.umn.edu/quantpdocker) is a Docker image containing a fully-operational Galaxy instance with pre-installed QuanTP, a suite of tools and functionalities critical for proteo-transcriptomics, including statistical algorithms for assessing the correlation between single transcript–protein pairs as well as across two cohorts, outlier identification and clustering, along with a diverse set of results visualizations.

QuanTP is available in this Docker image, and on the [Proteogenomics Gateway](/use/proteogenomics-gateway/) and [UseGalaxy.eu](https://usegalaxy.eu/).  It has an [Installation guide](https://jraysajulga.github.io/quanTP-docker/), and [Supporting information](http://z.umn.edu/quantpsi), including *Instructions for accessing the QuanTP tool on Jetstream*, *Instructions for accessing and running the Mouse data*, and *Accessing the QuanTP on the usegalaxy.eu.* QuanTP is supported by the University of Minnesota, Minneapolis.

## miCloud

[miCloud](http://github.com/BCIL/Personal-NGS-Cloud) is a Docker Image that leverages cloud technologies to provide an easy-to-use streamlined NGS data analysis using local computational hardware.  It includes three preconfigured and ready-to-execute NGS pipelines: two for single or paired-end ChIP-Seq data and one for paired-end RNA-Seq data. miCloud can also provide access to a large selection of ready-to-execute bioinformatics pipelines, available from online Docker repositories such as BioContainers, DockStore, or BioShadock.  It provides a workflow-centric interface to a Galaxy engine, using BioBlend.

miCloud is supported  by Weill Cornell Medicine and Department of Biological Sciences, Hunter College of the City University of New York.

## RNA Workbench

[<img class="float-right" style="max-width: 300px" src="/use/rna-workbench/rna-workbench-logo.png" alt="RNA Workbench" />](/use/rna-workbench/)

[RNA Workbench](/use/rna-workbench/), in addition to being available as a [Docker container](https://github.com/bgruening/galaxy-rna-workbench), is also available as [public server](https://rna.usegalaxy.eu/) hosted at UseGalaxy.eu.  RNA Workbench has a collection of over 50 tools dedicated to different research areas of RNA biology, including RNA structure analysis, RNA alignment, RNA annotation, RNA-protein interaction, ribosome profiling, RNA-Seq analysis, and RNA target prediction.  This server has been up for a while, but we are just adding it to the platforms directory.

The workbench is developed by the RNA Bioinformatics Center (RBC). This center is one of the eight service units of the [German Network for Bioinformatics Infrastructure](http://www.denbi.de), running the German [ELIXIR Node](https://www.elixir-europe.org/).

## Hi-C Explorer

[Hi-C Explorer](https://hicexplorer.usegalaxy.eu) is another server that has been up for a while, but is also new to the [Galaxy Platform Directory](/use/).   It facilitates the study of the 3D conformation of chromatin by allowing Hi-C data processing, analysis and visualization.  See the "[Hi-C analysis of Drosophila melanogaster cells using HiCExplorer](https://galaxyproject.github.io/training-material/topics/epigenetics/tutorials/hicexplorer/tutorial.html) tutorial and [example history](https://hicexplorer.usegalaxy.eu/u/joachim-wolff/h/drosophila-melanogaster-hi-c-training).

Hi-C Explorer is also hosted at UseGalaxy.eu.

## G-OnRamp

[<img class="float-right" style="max-width: 300px" src="/images/logos/g-onramp-logo-trans.png" alt="G-OnRamp" />](/use/g-onramp/)

[G-OnRamp](http://gonramp.wustl.edu/?page_id=1031) provides a user-friendly, web-based platform for collaborative, end-to-end annotation of eukaryotic genomes using UCSC Assembly Hubs and JBrowse/Apollo genome browsers with evidence tracks derived from sequence alignments, ab initio gene predictors, RNA-Seq data, and repeat finders. It is available as a [virtual machine](https://wustl.app.box.com/v/g-onramp-vm-v1) that can be deployed locally, and as an Amazon Machine Image that can be run on AWS and launched with [CloudLaunch](https://launch.usegalaxy.org/catalog).

Use the G-OnRamp output to visualize large genomics datasets, and can utilize the output to drive collaborative genome annotation projects in both research and educational settings. It has documentation:

* [How to use G-OnRamp](http://gonramp.wustl.edu/?page_id=384/#use)
* [Learn G-OnRamp](http://old-gep.wustl.edu/~galaxy/gonramp/?page_id=32/#materials)
* [Virtual Machine Installation Walkthrough](https://wustl.app.box.com/s/9626q6n2mjnd3vuas26j20w419f5v0fc)
* [CloudLaunch Deployment Walkthrough](https://wustl.app.box.com/s/rg7xaezf22p75d8yardsooa2izbdlkd5) (using the AMI)

<img class="float-right" style="max-width: 200px" src="/use/archive/amrplusplus/amrplusplus-flowchart.png" alt="AmrPlusPlus" />

G-OnRamp is supported by [Washington University in St. Louis](https://wustl.edu/) and [Oregon Health & Science University](https://ohsu.edu/)

## AmrPlusPlus

[AmrPlusPlus](http://megares.meglab.org/amrplusplus/latest/html/) is a Docker container with a Galaxy-based metagenomics pipeline that is intuitive and easy to use.  AmrPlusPlus takes advantage of current and new tools to help identify and characterize resistance genes within metagenomic sequence data. The pipeline can be used under a local instance of Galaxy and installed via Galaxy's Main Tool Shed. It is also available as a Galaxy-based Docker Image, using base images developed by [Björn Grüning](/people/bjoern-gruening/) at the University of Freiburg. It has [Documentation](http://megares.meglab.org/amrplusplus/latest/html/) and [Email support](http://megares.meglab.org/amrplusplus/latest/html/contact.html).  AmrPlusPlus is supported by [Colorado State University](https://www.colostate.edu/).

## Lifeportal

[<img class="float-left" style="max-width: 200px" src="/use/lifeportal-oslo/lifeportal-logo.png" alt="LifePortal" />](/use/lifeportal-oslo/)

[Lifeportal](https://lifeportal.uio.no) is a Galaxy server available to any researcher in Norway (and possibly any other user willing to use the service). LifePortal implements about 400 applications. It runs Galaxy version 18.09 and jobs are executed on the [Abel cluster](https://www.uio.no/english/services/it/research/hpc/abel/more/) (~650 nodes) using the slurm-drmaa library.  User support is available [via email](mailto:lifeportal-help@usit.uio.no).

Every user with their own project in Lifeportal can manage it through the menus built in the Galaxy GUI. Lifeportal is supported by [University of Oslo](http://www.uio.no/), [Norwegian Bioinformatics Platform](https://bioinfo.no/), [MLS UiO](https://www.uio.no/english/research/interfaculty-research-areas/mls/), and [UninetT Sigma2](https://www.uninett.no/en/sigma)


## LAPPS Grid

**THIS CONTENT HAS BEEN UPDATED SINCE ORIGINAL PUBLICATION.**

And from this cool writeup (that has nothing to do with life science): [Learning from data: Using technology to gain deeper meaning from archives](https://works.bepress.com/emily-mercer/1/), by Emily Mercer of San Jose State University, describing work by [James Pustejovsky](http://www.pusto.com/) of Brandeis University:

<blockquote class="blockquote">
One example is the collaboration between the American Archive of Public Broadcasting and James Pustejovsky, professor at Brandeis University. In this collaboration James and his research team have created a software package using the Galaxy Workflow Engine and Language Application Grid ([LAPPS Grid](/use/lapps-grid/)) for automatic extraction of metadata from video assets using machine learning methodologies (Oomen et al., 2018; [Pustejovsky et al., 2017](https://doi.org/10.1145/3093338.3104171)). What was once a time exhaustive process of real-time viewing and transcribing of film and video is now as simple as uploading a video file and applying a workflow. Processing takes seconds and afterwards the user has access to the metadata, for example, the text captions on the screen. This metadata can be exported as PBCore, a metadata standard specific to public broadcasting (Oomen et al., 2018; Pustejovsky et al., 2017).
</blockquote>

There's also some ~~[Jetstream](#)~~ in the mix too.

## Galaxy Platforms in Publications

We tag papers that use, mention, implement or extend public Galaxy platforms (servers, services, clouds, containers...).  Here are the counts for the past month's publications:

| # | Tag | | # | Tag | | # | Tag | | # | Tag |
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 23 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/tags/>Huttenhower) | | 6 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/tags/>RepeatExplorer) | | 5 | [>ARGs-OAP](https://www.zotero.org/groups/1732893/galaxy/tags/>ARGs-OAP) | | 3 | [>ABiMS](https://www.zotero.org/groups/1732893/galaxy/tags/>ABiMS) |
| 3 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/tags/>Cistrome) | | 3 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/tags/>Galaxy-P) | | 3 | [>UseGalaxy.eu](https://www.zotero.org/groups/1732893/galaxy/tags/>UseGalaxy.eu) | | 2 | [>LAPPS Grid](https://www.zotero.org/groups/1732893/galaxy/tags/>LAPPS Grid) |
| 2 | [>RNA Workbench](https://www.zotero.org/groups/1732893/galaxy/tags/>RNA Workbench) | | 2 | [>UseGalaxy.org](https://www.zotero.org/groups/1732893/galaxy/tags/>UseGalaxy.org) | | 1 | [>AmrPlusPlus](https://www.zotero.org/groups/1732893/galaxy/tags/>AmrPlusPlus) | | 1 | [>ASH](https://www.zotero.org/groups/1732893/galaxy/tags/>ASH) |
| 1 | [>G-OnRamp](https://www.zotero.org/groups/1732893/galaxy/tags/>G-OnRamp) | | 1 | [>GeneLab](https://www.zotero.org/groups/1732893/galaxy/tags/>GeneLab) | | 1 | [>Genomic Hyperbrowser](https://www.zotero.org/groups/1732893/galaxy/tags/>Genomic Hyperbrowser) | | 1 | [>GIO](https://www.zotero.org/groups/1732893/galaxy/tags/>GIO) |
| 1 | [>GmT](https://www.zotero.org/groups/1732893/galaxy/tags/>GmT) | | 1 | [>GVL-MEL](https://www.zotero.org/groups/1732893/galaxy/tags/>GVL-MEL) | | 1 | [>GVL-QLD](https://www.zotero.org/groups/1732893/galaxy/tags/>GVL-QLD) | | 1 | [>Halogen Bonding](https://www.zotero.org/groups/1732893/galaxy/tags/>Halogen Bonding) |
| 1 | [>HiCExplorer](https://www.zotero.org/groups/1732893/galaxy/tags/>HiCExplorer) | | 1 | [>Martin Luther](https://www.zotero.org/groups/1732893/galaxy/tags/>Martin Luther) | | 1 | [>miCloud](https://www.zotero.org/groups/1732893/galaxy/tags/>miCloud) | | 1 | [>Peptimapper](https://www.zotero.org/groups/1732893/galaxy/tags/>Peptimapper) |
| 1 | [>PRABI](https://www.zotero.org/groups/1732893/galaxy/tags/>PRABI) | | 1 | [>QuanTP](https://www.zotero.org/groups/1732893/galaxy/tags/>QuanTP) | | 1 | [>SouthGreen](https://www.zotero.org/groups/1732893/galaxy/tags/>SouthGreen) | | 1 | [>TDPortal](https://www.zotero.org/groups/1732893/galaxy/tags/>TDPortal) |
| 1 | [>TreeGenes](https://www.zotero.org/groups/1732893/galaxy/tags/>TreeGenes) | | 1 | [>Trinity](https://www.zotero.org/groups/1732893/galaxy/tags/>Trinity) | | 1 |  [>UseGalaxy.org](https://www.zotero.org/groups/1732893/galaxy/tags/>UseGalaxy.org) | | 1 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/tags/>Workflow4Metabolomics)  | |

----

# New Galactic Blog Posts

<div class="float-right">
[<img class="img-fluid mx-auto" style="max-width: 90px;" src="/news/2019-02-galaxy-update/marsupial.png" alt="Genome annotation with Prokka" />](/news/2019-01-totm/)
</div>

There are two new *[Galactic Blog](/news/)* entries:

* **[Tutorial of the Month: Genome annotation with Prokka, selected by Simon Gladman](/news/2019-01-totm/):**
    * read about using Prokka for annotation of draft genome sequences
    * by Bérénice Batut
* **[Hey Galaxy People! Want easy proteo-transcriptomics!??!](https://proteomicsnews.blogspot.com/2018/12/hey-galaxy-people-want-easy-proteo.html)**
    * an enthusiastic post by [Ben Orsburn](https://proteomicsnews.blogspot.com/p/contact.html) about [QuanTP](/news/2019-02-galaxy-update/#quantp)

----

# Who's Hiring

<div class='float-right'><img src="/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The *dark energy* of irreproducible research is threatening the science universe! Please help the Galaxy push it back!

* [Genomic data analyst](/news/2019-01-artbio-postdoc/), [ARTbio](http://artbio.fr/), Institut de Biology Paris Seine, campus Jussieu of the Sorbonne-Université, Paris, France
* [Cloud Engineer V](https://careers-fhcrc.icims.com/jobs/12290/cloud-engineer-v/job?mobile=false&width=960&height=500&bga=true&needsRedirect=false&jan1offset=-480&jun1offset=-420), Fred Hutchinson Cancer Research Center, Seattle, Washington, United States.
* ELIXIR Belgium has **four Galaxy-related openings** in Ghent:
    * [ELIXIR Open Science Community Manager](https://vibvzw.jobsoid.com/j/17164/elixir-open-science-community-manager), VIB-UGent Center for Plant Systems Biology
    * [ELIXIR Software developer data management tools](https://vibvzw.jobsoid.com/jobs/35bba14f-79b7-4388-b015-a4fb9d97ee41/17168/software-developer-data-management-tools), VIB-UGent Center for Plant Systems Biology
    * [ELIXIR Bioinformatics Trainer](https://vibvzw.jobsoid.com/j/17026/elixir-bioinformatics-trainer), VIB Bioinformatics Core
    * [ELIXIR Scientific Cloud Coordinator](https://vibvzw.jobsoid.com/j/17165/elixir-scientific-cloud-coordinator), VIB-UGent Center for Plant Systems Biology
* [Software Developer](https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25240&siteid=5341&PageType=JobDetails&jobid=1433370#jobDetails=1433370_5341), Harvard T.H. Chan School of Public Health, Boston, Massachusetts, United States. "Basic automated analysis workflows using Galaxy for 16S marker gene, metagenomic, and metatranscriptomic data leveraging existing software."
* [M2 - bio-informatique](https://www.sfbi.fr/content/m2-bio-informatique-laboratoire-de-virologie-h%C3%B4pital-st-louis), Laboratoire de Virologie , Hôpital St Louis, Paris, France.
* [Bioinformatics Scientist](https://www.providenceiscalling.jobs/portland-or/bioinformatics-scientist/E4987CF3970846FDBB268E76297287DC/job/), Providence Health Services, Portland, Oregon, United States. Looking for someone "fluent in Galaxy or willing to learn"
* [Etude et galaxyfication d’un pipeline de traitement de données en lipidomique](https://www.sfbi.fr/content/etude-et-galaxyfication-d%E2%80%99un-pipeline-de-traitement-de-donn%C3%A9es-en-lipidomique), Centre INRA Auvergne-Rhône-Alpes, Theix, France
* [Cloud Computing Bioinformatics Programmer](https://bioinformatics.ca/job-postings/0fd00bb0-9f21-11e8-a370-3ba76904a08d/#/?&order=desc) working with [IRIDA](http://www.irida.ca/), Simon Fraser University, Vancouver, Canada
* [Senior IT DevOps Data Engineer](https://adaptimmunellc.applytojob.com/apply/jobs/details/OsmMz9jkwv?) and [Senior IT Data Engineer (IT Bioinformatician)](https://adaptimmunellc.applytojob.com/apply/jobs/details/W1CcPTXteJ?),  Adaptimmune, Abingdon, Oxfordshire, United Kingdom. *Required: Experience with following bioinformatic pipeline tools:   Galaxy…*
* [Specialist Solutions Architect -- Biological Sciences](https://www.amazon.jobs/en/jobs/728182/specialist-solutions-architect-biological-sciences), Amazon Web Services, United States
* [Principal Technical Business Development Manager: AWS Research in Biomedical](https://www.amazon.jobs/en/jobs/745693/principal-technical-business-development-manager-aws-research-in-biomedical), Amazon Web Services, United States
* [Bioinformatics Scientist](https://www.indeed.com/viewjob?jk=5288546935e35605), ResearchDx, Irvine, California, United States
* [NGS Scientific Applications Specialist](https://careers-idtdna.icims.com/jobs/4112/ngs-scientific-applications-specialist/job), Integrated DNA technologies, Iowa City, Iowa, United States
* The [The European Galaxy Team has open positions](https://usegalaxy-eu.github.io/posts/2019/01/10/openpositions/), Freiburg, Germany
    * Software engineer, system analysts/administrators, data analyst, and a comunnity and/or research manager
* The [Blankenberg](/people/dan/) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/news/2017-09-hiring-at-cc/).
* [Galaxy Project is hiring two software engineers at Johns Hopkins University](/news/2017-08-hiring-at-hopkins/), Baltimore, Maryland, United States.

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

---

# Doc, Hub, and Training Updates

Updates from the [Galaxy Training Materials](https://galaxyproject.github.io/training-material/) in January, and the Hub as well:

## GTN Training Materials

[<img class="float-right img-responsive" src="/images/galaxy-logos/GTNLogoTrans300.png" alt="Galaxy Training Materials" />](https://galaxyproject.github.io/training-material/)

* New [Age prediction using machine learning](https://galaxyproject.github.io/training-material/topics/statistics/tutorials/age-prediction-with-ml/tutorial.html), by [Anup Kumar](https://github.com/anuprulez).
* New [GO enrichment analysis](https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/goenrichment/tutorial.html), by [igcbioinformatics](https://github.com/igcbioinformatics), [Maria Doyle](https://github.com/mblue9), and [dsobral](https://github.com/dsobral).
* Join the February 2019 GTN CoFest. Held on 21 February, the CoFest will be coordinated via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.7q82sbcupg0j59li0fqo9pots6) (drop-in channel will be open the whole day) and [the GTN Gitter channel](https://gitter.im/Galaxy-Training-Network/Lobby).


## Hub

* [Configure Cloud Authorizatoin for Azure](/authnz/cloud/azure/), by [Vahid Jalili](/people/vahid-jalili/)


---

# ToolShed Contributions

<div class='right'>[<img src="/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in January  2019](/toolshed/contributions/2019-01/).


# Releases

New additions to the Galaxy Ecosystem.

## galaxy-lib 18.9.2

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

## Planemo 0.58.1

[<img class="float-right" src="/images/galaxy-logos/planemo-logo.png" alt="Planemo" width="200" />](https://pypi.python.org/pypi/planemo/)

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

----

# Other News

* The [Galaxy statistics page](/galaxy-project/statistics/) has received it's semi-annual update.
* From [James Taylor](/people/james-taylor/):
    * [Visualizing scRNA-Seq data](https://twitter.com/jxtx/status/1088580033512390657/video/1) using [cellxgene](https://github.com/chanzuckerberg/cellxgene/blob/master/README.md) inside Galaxy.

## PAG 2019 Presentations and Posters are Available

Copies of most of the Galaxy related presentations and posters form the Plant and Animal Genome Conference [are now online](/events/2019-pag/).  Many thanks to the all the presenters for sharing their during and after the conference.
