---
title: '[GTN news] New Tutorial: Mutation calling, viral genome reconstruction and
  lineage/clade assignment from SARS-CoV-2 sequencing data'
date: '2021-06-30'
tags: [training, gtn-news, covid-19, workflow]
supporters:
- denbi
- elixir
- gallantries
external_url: https://training.galaxyproject.org/training-material/news/2021/06/30/tutorial-sars-cov-2-variant-discovery.html
subsites: [eu, pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
main_subsite: eu
---

<p>Effectively monitoring global infectious disease crises, such as the COVID-19 pandemic, requires capacity to generate and analyze large volumes of sequencing data in near real time. These data have proven essential for monitoring the emergence and spread of new variants, and for understanding the evolutionary dynamics of the virus.</p>

<p>The Galaxy community has developed high-quality analysis workflows to support</p>

<ul>
  <li>sensitive identification of SARS-CoV-2 allelic variants (AVs) starting with allele frequencies as low as 5% from deep sequencing reads</li>
  <li>generation of user-friendly reports for batches of results</li>
  <li>reliable and configurable consensus genome generation from called variants</li>
</ul>

<p>The new tutorial “Mutation calling, viral genome reconstruction and lineage/clade assignment from SARS-CoV-2 sequencing data” will teach you how to obtain, run and combine these workflows appropriately for different types of input data, be it:</p>

<ul>
  <li>Single-end data derived from Illumina-based RNAseq experiments</li>
  <li>Paired-end data derived from Illumina-based RNAseq experiments</li>
  <li>Paired-end data generated with Illumina-based Ampliconic (ARTIC) protocols, or</li>
  <li>ONT FASTQ files generated with Oxford nanopore (ONT)-based Ampliconic (ARTIC) protocols</li>
</ul>

<p>We would be happy to have your feedback or contributions for this new tutorial. That would help us improving it!</p>


