---
title: May 2017 Galaxy News
---
<div class='right'><a href='/src/galaxy-updates/index.md'><img src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the May 2017 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.


# 2017 Galaxy Community Conference

<div class="right"><a href="https://gcc2017.sciencesconf.org/"><img src="/src/images/logos/gcc2017-logo-300.png" alt='2017 Galaxy Community Conference (GCC2017)' width="220" /></a></div>

GCC2017 will be in Montpellier, France, 26-30 June and will feature two days of presentations, discussions, poster sessions, lightning talks, computer demos, keynotes, and birds-of-a-feather meetups, **all about data-intensive biology and the tools and infrastructure that support it.** GCC2017 also includes data and coding hacks, and **[two days of training](https://gcc2017.sched.com/overview/type/Training)** covering 16 different topics.
 
GCC2017 will be held at [Le Corum Conference Centre](http://www.montpellier-events.com/en/The-Corum/Presentation) in the heart of [Montpellier](https://gcc2017.sciencesconf.org/page/location), just 10km from the Mediterranean. This event will gather several hundred researchers addressing diverse questions and facing common challenges in data intensive life science research. GCC participants work across the tree of life, come from around the world, and work at universities, research organizations, industry, medical schools and research hospitals.

## Early registration closes May 15

**[Early registration](https://gcc2017.sciencesconf.org/page/registration) ends May 15.**  Early registration  starts at less than 55€ / day for post-docs and students.  You can also [book low cost conference housing](https://gcc2017.sciencesconf.org/page/lodging) when you register.

## Poster and Demo submission closes May 27 

**[Abstract submission](https://gcc2017.sciencesconf.org/page/submit)** for posters and demos closes May 27.  Note that you can also still submit late oral prentations abstracts.  Late abstracts are considered as cancellations occur.  Lightning talk submissions close June 23.

## New GCC2017 Sponsors

We are pleased to welcome [BioTeam](https://bioteam.net/products/galaxy-appliance/) and [GenomeWeb](https://www.genomeweb.com/?utm_source=galaxyproject_logo&utm_medium=parterships&utm_campaign=may) as returning GCC sponsors.


### GenomeWeb

<div class='left'><a href='https://www.genomeweb.com/?utm_source=galaxyproject_logo&utm_medium=parterships&utm_campaign=may'><img src="/src/images/logos/GenomeWebLogo300.jpg" alt="GenomeWeb" width="200" /></a>
</div>

***[GenomeWeb](https://www.genomeweb.com/?utm_source=galaxyproject_blog&utm_medium=parterships&utm_campaign=may)*** is sponsoring [GCC](https://gcc2017.sciencesconf.org/) for the fourth year in a row. *[GenomeWeb](https://www.genomeweb.com/?utm_source=galaxyproject_blog&utm_medium=parterships&utm_campaign=may)* is an independent online news organization that provides in-depth coverage of the scientific and economic ecosystem spurred by high-throughput genome sequencing. *GenomeWeb* news is the leading information source for scientists, executives, and clinicians who use and develop advanced life science tools. ***As a conference attendee, GenomeWeb is pleased to offer you a three month trial to access their premium news content*** by contacting [customerservice@genomeweb.com](mailto:customerservice@genomeweb.com) and indicating you are a Galaxy Conference 2017 attendee.  If you are an academic professional or non-profit researcher, you automatically have access to *GenomeWeb* premium content and can [register here](https://www.genomeweb.com/user/register?utm_source=galaxyproject_attendee_mailing&utm_medium=parterships&utm_campaign=may) to join using your work email address.

<br />

### BioTeam

<div class='right'> 
<a href='https://bioteam.net/products/galaxy-appliance/'><img src="/src/images/logos/bioteam-appliance-white.png" alt="BioTeam Appliance Galaxy Edition" width="250" /></a>
</div>

**[BioTeam](https://bioteam.net/)** is pleased to announce special pricing for the **[BioTeam Appliance Galaxy Edition](https://bioteam.net/products/galaxy-appliance/)** now through Jun 30, 2017 (the last day of GCC2017).  The BioTeam Appliance Galaxy Edition allows the researcher to get up and running fast with Galaxy.  The Galaxy Appliance comes preinstalled with a production instance of Galaxy, bioinformatics tools, and reference datasets.  "Science on Day One"  To request more information [click here](https://bioteam.net/products/galaxy-appliance/galaxy-appliance-information-request/).

| Model | Price |
| ---- | ---- |
| Galaxy Appliance Base Model 384/32	     | $29,995 USD |
| Galaxy Appliance Base Model 384/96		 | $45,495 USD |
| Galaxy Appliance Enhanced Model 512/32 | $32,995 USD |
| Galaxy Appliance Enhanced Model 512/96 | $48,995 USD |

Technology Financing is available for qualified institutions/companies.  Will ship globally.


## GCC2017 Call for BoFs!

There is no better place than a Galaxy Community Conference to meet and learn from others doing data-intensive biology.  [GCC2017](https://gcc2017.sciencesconf.org/) continues this tradition by again including *[Birds of a Feather (BoF)](http://en.wikipedia.org/wiki/Birds_of_a_feather_%28computing%29)* meetups.  Birds of a Feather meetups are informal gatherings where participants group together based on common interests.

BoF meetups are encouraged throughout GCC2017.  **The [current list of BoFs is available on the conference web site](http://bit.ly/gcc2017bofs).** BoFs are posted as soon as they are proposed.  The exact time and schedule for each will be set in mid to late June.

**Want to plan a BoF?**

To launch a Birds of a Feather meetup **[submit your BoF idea here](https://docs.google.com/forms/d/e/1FAIpQLSdRCs97OWt47qmMxWZAtSV3TTLJ24IhW0m2lHGPLiqJU0Xkzw/viewform).** Once submitted, the conference organizers will create the [BoF in the conference schedule](http://bit.ly/gcc2016BOFS), and give you edit access.

**BoF Support**

The conference *enables and encourages* BoFs by

* Encouraging conference participants to organize BoFs
* Promote BoFs to all conference participants, over and over, both before and during the conference.
* Coordinate and reserve rooms for BoFs, and allocate time in the schedule for them too.

# Galaxy Tutorial @ ISMB/ECCB 2017

<div class="right"><a href="https://www.iscb.org/ismbeccb2017-program/tutorials#p6"><img src="/src/images/logos/ismb-2017.png" alt="Galaxy Tutorial at ISMB/ECCB 2017: Making Galaxy Work for You" width="170" /></div>

Ther will be a half day tutorial on [Making Galaxy Work for You](https://www.iscb.org/ismbeccb2017-program/tutorials#p6) at [ISMB/ECCB 2017](https://www.iscb.org/ismbeccb2017) in Prague, Czechia. The tutorial will provide a practical, hands-on guide to adapting the Galaxy platform to the specific needs of individuals attending the ISMB.

Participants will learn to

- Create Galaxy compatible tool and workflow definitions that are publicly accessible and that make it easy for any instance administrator to add your work to their server. 
- Deploy Galaxy and scale it up to target production-ready resources such as a Postgres database, NGINX webserver, and distributed resource managers such SLURM or PBS.

Interested? Sign up when you [register](https://www.iscb.org/ismbeccb2017-registration#tutorials) for ISMB/ECCB (and register before the tutorial fills up).

We also expect a lot of other Galaxy-related content at ISMB/ECCB 2017 and [BOSC 2017](https://www.open-bio.org/wiki/BOSC_2017). We'll add the presentations to the [Galaxy @ ISMB/ECCB 2017 & BOSC 2017](/src/events/2017-ismb-bosc/index.md) event page as the schedules are posted.  (And, [please let us know](mailto:outreach@galaxyproject.org) if you have anything on the program.)

See you in Prague!


# Gateways 2017 Call for Participation (1st deadline: June 5)

<div class='right'> 
<a href='http://sciencegateways.org/gateways2017/call'><img src="/src/images/logos/sgci-logo-trans-big.png" alt="Gateways 120" height="90" /></a>
</div>

[Gateways 2017](http://sciencegateways.org/gateways2017/) (October 23-25, at the University of Michigan, Ann Arbor) is now accepting submissions of papers, demos, tutorials, and panels (2-4 pages) on the topic of science or engineering gateways, which are integrated, user-friendly interfaces to scientific computing, data, and other domain-specific resources to support research and education.

Submissions may focus on design, use, impact, development processes, sustainability, best practices, or any other aspect that you think fellow gateway creators or users will find interesting to learn. We also welcome educational topics directed toward the next generation of gateway creators.

The primary submission DEADLINE IS JUNE 5, 2017, and a poster session deadline (open to all) will be September 8. Read more details in the [Call for Participation](http://sciencegateways.org/gateways2017/call).

----

# All upcoming events

There are [a plenitude of Galaxy related events coming up](/src/events/index.md) in the next few months:

<table class="table table-striped"><thead>
<tr>
<th>Date</th><th>Topic/Event</th><th>Venue/Location</th><th>Contact</th>
</tr>
</thead><tbody>
<tr>
  <td><span class="text-nowrap">May 4th 2017</span></td>
  <td><a href="/events/2017-05-arabic">Introduction to Genomics in Medicine Using Galaxy  Platform</a></td>
  <td><img style="float:right;" src="/images/icons/AF.png"/><a href="http://med.tanta.edu.eg/">Tanta University Faculty of Mediciene, Tanta, Egypt</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Abdelazeem_Abdelhameed2015505 @ yahoo.com</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 15th 2017</span></td>
  <td><a href="https://biotexcel.com/event/ngs-2017-glasgow/#workshop">Data Analysis Workshop</a> [<a href="/events/2017-05-glasgow-ngs">hub page</a>]</td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://biotexcel.com/event/ngs-2017-glasgow/">NGS 2017 Glasgow, Glasgow, United Kingdom</a></td>
  <td>Julian Augley, Graham Hamilton, David Meltzer and Mani Mudaliar</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 16th 2017</span></td>
  <td><a href="http://www.france-bioinformatique.fr/en/evenements/G4I">Formation Galaxy4Imaging</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>Nantes, France</td>
  <td>Comité d'organisation</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 22nd 2017</span></td>
  <td><a href="https://docs.google.com/document/d/1dLCL5-2pkWTvmGTz7GyQkb5LEIZ3FBkPvCBoqUPGUOg/edit">ELIXIR/GOBLET/GTN hackathon for Galaxy training material re-use</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.cam.ac.uk/">University of Cambridge, United Kingdom</a></td>
  <td>Frederik Coppens</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2017</span></td>
  <td><a href="https://bioinformatics.ca/workshops/2017/bioinformatics-cancer-genomics-2017">Bioinformatics for Cancer Genomics</a> [<a href="/events/2017-05-cancer-genomics">hub page</a>]</td>
  <td><img style="float:right;" src="/images/icons/NA.png"/>Toronto, Ontario, Canada</td>
  <td>organizers</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2017</span></td>
  <td><a href="http://www.france-bioinformatique.fr/fr/evenements/W4E2017">Workflow4Experimenters International Course</a> [<a href="/events/2017-05-w4e">hub page</a>]</td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.france-bioinformatique.fr/fr">Pasteur Institute, Paris, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christophe Caron</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 4th 2017</span></td>
  <td><a href="http://www.asms.org/conferences/annual-conference/annual-conference-homepage">Galaxy @ ASMS 2017</a> [<a href="/events/2017-asms">hub page</a>]</td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://www.asms.org/conferences/annual-conference/annual-conference-homepage">ASMS 2017, Indianapolis, Indiana, United States</a></td>
  <td>Tim Griffin, Pratik Jagtap</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 6th 2017</span></td>
  <td><a href="https://lists.galaxyproject.org/archives/list/galaxy-france@lists.galaxyproject.org/message/PHXFO7BB724GSZ5TXE7WNH543OUYFKGY/">Formation détection et annotation des éléments transposables dans les génomes eucaryote</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://urgi.versailles.inra.fr/">INRA, Versailles, France</a></td>
  <td>Nathalie Choisne, Olivier Inizan</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 20th 2017</span></td>
  <td><a href="/news/2017-03-g-on-ramp/">G-OnRamp Beta Testers Workshop</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 26th 2017</span></td>
  <td><a href="https://gcc2017.sciencesconf.org/">2017 Galaxy Community Conference (GCC2017)</a> [<a href="/events/2017-gcc">hub page</a>]</td>
  <td><img style="float:right;" src="/images/icons/EU.png"/>Montpellier, Fance</td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Galaxy community</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 2nd 2017</span></td>
  <td><a href="http://www.smbe2017.org/symposia/">Computation and reproducibility in molecular evolution</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://www.smbe2017.org/">SMBE 2017, Austin, Texas, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Anton Nekrutenko, Sergei Kosakovsky Pond</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 20th 2017</span></td>
  <td><a href="https://www.open-bio.org/wiki/Codefest_2017">OpenBio Codefest 2017</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://brmlab.cz/">Brmlab, Prague, Czechia</a></td>
  <td>Brad Chapman, Matúš Kalaš, Heather Wiencko</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 21st 2017</span></td>
  <td><a href="/events/2017-ismb-bosc">Galaxy @ ISMB/ECCB 2017 &amp; BOSC 2017 </a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.iscb.org/ismbeccb2017">Prague, Czechia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Presenters</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 25th 2017</span></td>
  <td><a href="/news/2017-03-g-on-ramp/">G-OnRamp Beta Testers Workshop</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">September 1st 2017</span></td>
  <td><a href="http://www.earlham.ac.uk/genome-10k-and-genome-science">The Galaxy Bioinformatics Platform: learning to use Galaxy for RNA-seq analysis</a> [<a href="/events/2017-genome-10k">hub page</a>]</td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.earlham.ac.uk/">Genome 10K and Genome Science, Earlham Institute, Norwich, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a></td>
</tr>
<tr>
  <td><span class="text-nowrap">October 23rd 2017</span></td>
  <td><a href="http://sciencegateways.org/gateways2017/">Gateways 2017</a></td>
  <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://www.umich.edu/">University of Michigan, Ann Arbor, Michigan, United States</a></td>
  <td>Enis Afgan</td>
</tr>
</tbody></table>


See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

# New Publications

**110 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in April.

Some highlights from the papers added last month:

 	
* [Genome Build Information Is An Essential Part Of Genomic Track Files](https://doi.org/10.1101/120923)
   Chakravarthi Kanduri, Diana Domanska, Eivind Hovig, Geir K. Sandve. *bioRxiv* (2017), doi:10.1101/120923

* [Antigen Receptor Galaxy: A User-Friendly, Web-Based Tool for Analysis and Visualization of T and B Cell Receptor Repertoire Data](https://doi.org/10.4049/jimmunol.1601921)
   Hanna IJspeert, Pauline A. van Schouwenburg, David van Zessen, Ingrid Pico-Knijnenburg, Andrew P. Stubbs, Mirjam van der Burg. *The Journal of Immunology* (17 April 2017), 1601921, doi:10.4049/jimmunol.1601921

* [Characterization of NIST Human Mitochondrial DNA SRM-2392 and SRM-2392-I Standard Reference Materials by Next Generation Sequencing](https://doi.org/10.1016/j.fsigen.2017.04.005)
  Sarah Riman, Kevin M. Kiesler, Lisa A. Borsuk, Peter M. Vallone. *Forensic Science International: Genetics* (April 2017), doi:10.1016/j.fsigen.2017.04.005

* [ReGaTE, Registration of Galaxy Tools in Elixir](https://doi.org/10.1093/gigascience/gix022)
  Olivia Doppelt-Azeroual, Fabien Mareuil, Eric Deveaud, et al. *GigaScience* (10 April 2017), doi:10.1093/gigascience/gix022

* [Transcriptomic Analysis of C. elegans RNA Sequencing Data Through the Tuxedo Suite on the Galaxy Project](https://doi.org/10.3791/55473)
  Francis R. G. Amrit, Arjumand Ghazi. *Journal of Visualized Experiments*, No. 122. (8 April 2017), doi:10.3791/55473

* [wft4galaxy: A Workflow Tester for Galaxy](https://doi.org/10.1101/132001 )
  Marco Enrico Piras, Luca Pireddu, Gianluigi Zanetti. *bioRxiv* (2017), doi:10.1101/132001 

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 72 | [methods](http://www.citeulike.org/group/16008/tag/methods) | | 31 | [usepublic](http://www.citeulike.org/group/16008/tag/usepublic) | | 17 | [workbench](http://www.citeulike.org/group/16008/tag/workbench) | | 13 | [usemain](http://www.citeulike.org/group/16008/tag/usemain) | 
| 12 | [refpublic](http://www.citeulike.org/group/16008/tag/refpublic) | | 10 | [uselocal](http://www.citeulike.org/group/16008/tag/uselocal) | | 8 | [reproducibility](http://www.citeulike.org/group/16008/tag/reproducibility) | | 3 | [cloud](http://www.citeulike.org/group/16008/tag/cloud) | 
| 3 | [tools](http://www.citeulike.org/group/16008/tag/tools) | | 3 | [unknown](http://www.citeulike.org/group/16008/tag/unknown) | | 3 | [isgalaxy](http://www.citeulike.org/group/16008/tag/isgalaxy) | | 2 | [shared](http://www.citeulike.org/group/16008/tag/shared) | 
| 2 | [other](http://www.citeulike.org/group/16008/tag/other) | | 1 | [howto](http://www.citeulike.org/group/16008/tag/howto) | | | | | | | 
  
----

# Who's Hiring

<div class='right'><br /><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Bioinformatician (Research Officer)](http://www.scientificomputing.uct.ac.za/news/opportunity-bioinformatics-academic-position), Scientific Computing Research Unit (SCRU), University of Cape Town, South Africa.  Bioinformatics and/or Cheminformatics research and development in the area of Cancer Glycobiology. The closing data for applications is 12 May 2017.
* [Automated deployment of Galaxy environments using Ansible](http://artbio.fr/?page_id=78), Institut de Biologie Paris Seine, France
* [BioInformaticien](https://talent.limagrain.com/Pages/Offre/detailoffre.aspx?idOffre=1407&idOrigine=2673&LCID=1036), HM.Clause, Business Unit du Groupe Limagrain,  Maine et Loire, La Bohalle, France
* [Scientific Research Programmer](http://goo.gl/cMZVCn), [Sethuraman Lab](http://arunsethuraman.weebly.com/), [California State University San Marcos](https://www.csusm.edu/). Develop model-based population genomics pipelines.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />


----

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them in April

## New Galaxy Services

### United Kingdom: CLIMB

<div class='right solid'><a href='http://www.climb.ac.uk/'><img src="/src/images/logos/mrc-climb-logo.png" alt="MRC CLIMB (Cloud infrastructure for Microbial Bioinformatics" width="300" /></a></div>

* *Links:*
  * **[CLIMB Home](http://www.climb.ac.uk/)**
  * [Request a CLIMB account](https://bryn.climb.ac.uk/user/register/)
  * [Login](https://bryn.climb.ac.uk/)
* *Eligibility:*
  *  To register for a CLIMB group account, you must:
    * Be a UK academic or public sector worker working on microbial genomics
    * Run your own research group (i.e. be a principal investigator)
    * Have a .ac.uk (or .gov.uk or .nhs.uk email account)
    * Agree to our service terms and conditions
* *Comments:*
  * The CLIMB project (Cloud Infrastructure for Microbial Bioinformatics) develops and deploys a world leading cyber-infrastructure for microbial bioinformatics; providing free cloud-based compute, storage, and analysis tools for academic microbiologists in the UK.
  * The project is supported by three world class Medical Research Fellows, a comprehensive training program and two newly refurbished Bioinformatics facilities in both Warwick and Swansea.
  * The Galaxy component of CLIMB is based on GVL. CloudBridge, and CloudMan infrastructure.
* *User Support:*
  * [CLIMB Community](https://discourse.climb.ac.uk/)
  * [Training](http://www.climb.ac.uk/training/)
* *Quotas:* 
* *Sponsor(s):*
  * The CLIMB project is a collaboration between Warwick, Birmingham, Cardiff and Swansea Universities.
  * CLIMB is funded by the UK’s [Medical Research Council](https://www.mrc.ac.uk/).


## New Public Galaxy Servers

### Antigen Receptor Galaxy (ARGalaxy)
	
<div class='right solid'><a href='https://bioinf-galaxian.erasmusmc.nl/argalaxy/'><img src="/src/use/argalaxy/argalaxy-logo.png" alt="Antigen Receptor Galaxy" width="300" /></a></div>

* *Link:*
  * [Antigen Receptor Galaxy (ARGalaxy)](https://bioinf-galaxian.erasmusmc.nl/argalaxy/)
* *Domain/Purpose:*
  * Automate the analysis and visual reporting of T- and B-cell receptor repertoire sequencing data.
* *Comments:*
  * Consists of four parts: the demultiplex tool, the IMGT concatenate tool, the immune repertoire pipeline and the SHM&CSR pipeline. Together they allow the analysis of all different aspects of the immune repertoire. All pipelines can be run independently or combined, dependent on the available data and the question of interest
    * The demultiplex tool allows data trimming and demultiplexing of samples
    * The concatenate tool allows the merging of multiple IMGT/HighV-QUEST output files into a single file.
    * The immune repertoire pipeline allows analysis of both B- and T-cell receptor rearrangements and was developed to visualize V(D)J gene usage, CDR3 characteristics (length and amino acid usage), junction characteristics and calculate the diversity of the immune repertoire.
    * The SHM&CSR pipeline allows analysis of somatic hypermutations (SHM) and/or class switch recombination (CSR) in B-cell receptor rearrangements. It analyzes the frequency and patterns of SHM, antigen selection (including BASELINe), clonality (Change-O) and CSR.
* *User Support:*
  * Contact [David van Zessen](mailto:d.vanzessen@erasmusmc.nl)
* *Quotas:*
  * Account creation required.  Anyone can create an account. 
* *Sponsor(s):*
  * Departments of [Immunology](https://www.erasmusmc.nl/immunologie/?lang=en) and [Bioinformatics](https://www.erasmusmc.nl/bioinformatica/?lang=en) at [Erasmus MC](https://www.erasmusmc.nl/).	

<!--

# Galaxy Community Directories

<table class="table">
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/teach/index.md'><img src="/src/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/logs/index.md'><img src="/src/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/deployments/index.md'><img src="/src/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your <a href='/src/teach/resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/teach/trainers/index.md'>experience</a> <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/logs/index.md#add-a-log-page'>Share your experience</a> <em><a href='/src/community/logs/index.md#add-a-log-page'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/deployments/index.md#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/src/community/deployments/index.md#add-your-galaxy-deployment'>now</a></em> </strong> </td>
  </tr>
</table>

<br />

No new resources were added to community directories in January or February.
-->

----

# Releases

## galaxy-lib 17.5.8

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).




## Earlier Releases

Other packages that have been released in the prior 4 months.

###  17.01 Galaxy Release

<div class='right'><a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.01 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.01_announce.html)** in February.


#### Highlights

Some highlights:

* **Conda auto initialization is enabled by default**
* **New interface for user preferences**
* **Support for compressed FASTQ formats**

#### Release Notes

For full details on all of the enhancements and fixes in this release, please see the [full release notes](https://docs.galaxyproject.org/en/master/releases/17.01_announce.html).


#### Galaxy Docker Image 17.01

And, thanks to [Björn Grüning](/src/people/bjoern-gruening/index.md), there is also now [a Docker image for Galaxy 17.01](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.01) as well.

### CloudLaunch ##

Technically, the [all-new Galaxy CloudLaunch service](https://beta.launch.usegalaxy.org/) has been in public beta since February but keep in mind that it will replace the [current CloudLaunch service](https://launch.usegalaxy.org/launch) eventually so give it a try and let us know how it performs for you.

### CloudBridge 0.2.0

[CloudBridge](https://github.com/gvlproject/cloudbridge) aims to provide a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud. It is currently under development and is in an Alpha state.  [Release 0.2.0](https://github.com/gvlproject/cloudbridge/releases/tag/v0.2.0) includes several fixes and enhancements.


### Planemo 0.39 - 0.40

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/src/images/galaxy-logos/planemo-logo.png" alt="" width="200" /></a></div>

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

### Starforge 0.3

<div class='right'>
<a href='https://github.com/galaxyproject/starforge/blob/master/README.rst'><img src="/src/images/galaxy-logos/starforge_logo_big.png" alt="Starforge" width="180" /></a>
</div>

[Starforge](https://pypi.python.org/pypi/starforge) is a collection of scripts that supports the building of components for Galaxy.  Specifically, with Starforge you can:

* Build [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) dependencies
* Build [Python Wheels](http://pythonwheels.com/) (e.g. for the [Galaxy Wheels Server](http://wheels.galaxyproject.org/))
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.

Documentation can be found at [starforge.readthedocs.org](http://starforge.readthedocs.org/). 

### Pulsar 0.7.4

<div class='right'><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/src/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" /></a></div>

A [Pulsar](https://pypi.python.org/pypi/pulsar-app/) update was released in February.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

[This release](https://github.com/galaxyproject/pulsar/blob/master/HISTORY.rst) contains Conda and flake8 updates.

### sequence_utils

Galaxy's [sequence utilities](https://github.com/galaxyproject/sequence_utils) are a set of Python modules for reading, analyzing, and converting sequence formats.


### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

[Tool Shed contributions from April](/src/toolshed/contributions/2017-04/index.md).


# Other News

<div class="right"><a href="http://gigasciencejournal.com/blog/guardians-of-the-galaxy-workflow"><img src='/src/images/news-graphics/gigascience-guardians.png' alt="Guardians of the Galaxy Workflow" width="150" /></a></div>

* [Guardians of the Galaxy Workflow](http://gigasciencejournal.com/blog/guardians-of-the-galaxy-workflow/)
* A nice writeup from [Scott Edmunds](http://gigasciencejournal.com/blog/author/scottedmunds/) about the [*GigaScience* Galaxy Series](https://academic.oup.com/gigascience/pages/galaxy_series_data_intensive_reproducible_research) (now on it's 12th paper) and GCC2017.  *[GigaScience](https://academic.oup.com/gigascience)* is a returning GCC sponsor this year and submissions relating to talks and posters at GCC2017 will be eligible for a continuing 15% discount. 
* From [Björn Grüning](/src/people/bjoern-gruening/index.md)
  * Are you interested in #RNA #Bioinformatics? Try our new community maintained [RNA Workbench](http://bgruening.github.io/galaxy-rna-workbench/)
* From [Peter Briggs](https://twitter.com/ironic_cog):
  * [Galactic Engineer: Securing Galaxy with HTTPS running with Nginx using Let’s Encrypt](http://galacticengineer.blogspot.co.uk/2017/04/securing-galaxy-with-https-running-with.html)
* The [Galaxy-P Project has a brand new website](http://galaxyp.org/).
* A new [comprehensive ChIP-seq tutorial is available](/src/tutorials/chip/index.md) using DeepTools, MACS2, and MEME.
* From Nathan Bott
  * [Galaxy: A Workflow Management System for Modern Life Sciences Research](http://emergingtechblog.emc.com/galaxy-workflow-management-system-modern-life-sciences-research/) - [EMC Emerging Tech Blog](http://emergingtechblog.emc.com/) 
