---
title: August 2016 Galaxy News
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 24em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/galaxy-updates/'><img src="/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the August 2016 *[Galactic News](/galaxy-updates/)*, a summary of what is going on in the Galaxy community.

If you have anything to include in the next News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org).


# Events

# GCC2016 presentations and training are online

<div class='right'>
<a href='https://web.archive.org/web/http://gcc2016.iu.edu/'><img src="/images/logos/GCC2016LogoTallBig.png" alt="2016 Galaxy Community Conference" width="120" /></a>
</div>

Slides for [accepted](http://gcc16.sched.org/overview/type/E.1+Conference+-+Talks) and [lightning](https://gcc16.sched.org/overview/type/E.5+Conference+-+Lightning) talks are online, as are [the posters](http://gcc16.sched.org/overview/type/E.2+Conference+-+Posters).  They have also been posted to the new [F1000Research Galaxy Channel](http://f1000research.com/channels/galaxy). Video of training and talks will also be posted online in the coming month.

Thanks to everyone for making GCC2016 a success, and looking forward to seeing you at [GCC2017](https://gcc2017.sciencesconf.org/)!

## Swiss-German Galaxy Days

There will be a Swiss-German Galaxy Tour this fall featuring a range of events over 2 days in October.  Events will held in Freiburg (Germany).

Registration is free, but space is limited on both days. We recommend you register soon to secure your spot for one or both days of the SG2016Tour: **[Register Now](https://docs.google.com/forms/d/1mRO4uNXQd8jL1I9JNiQ4xd0fnrZKlQPQSORgJCpDeqg)**

## Galaxy Admin Training, November 7-11, Salt Lake City, Utah

<div class='left'><a href='/events/admin-training2016/'><img src="/images/logos/AdminTraining2016-500.png" alt="Galaxy Project Admin Training" width="140" /></a></div>

The first ever [Galaxy Project Admin Training workshop](/events/admin-training2016/) will be held November 7-11 in Salt Lake City Utah.  The workshop features a 2 day basics session followed by a 3 day advanced session. There is a separate registration for each session, and participants may sign up for one or both.  We are working with the Utah High Performance Computing Center at the University of Utah for cluster access during the advanced session.

The introductory session will be at the Salt Lake City Library, and the advanced session will be on the University of Utah (the "U") campus.  Workshop housing is on the U as well.

The curriculum will be published and registration will open shortly.  Watch this space.  This workshop is the week before [Supercomputing'16](http://sc16.supercomputing.org/) meets in Salt Lake.

## Other upcoming events

There are [a plentitude of Galaxy related events coming up](/events/). Here's what's coming up over the next fe months:

<div class='center'>
<a href='http://icbo.cgrb.oregonstate.edu/node/41'><img src="/images/logos/ICBO_joint.png" alt="Integrating Ontological and Genomic Analysis with Galaxy Workshop @ ICBO" height="70" /></a> &nbsp;&nbsp;
<a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Aug16.html'><img src="/images/logos/GlasgowPolyomics.jpg" alt="Introduction to Galaxy Workshop" height="80" /></a>
&nbsp;&nbsp;&nbsp;
<a href='/community/galaxy-admins/meetups/2016-08-18/'><img src="/images/galaxy-logos/GalaxyAdmins.png" alt="GalaxyAdmins" height="60" /></a>
<a href='https://registration.genomecenter.ucdavis.edu/events/Aug_2016_Bioinformatics_Galaxy_Workshop/'><img src="/images/logos/UCDavisGenomeCenter_Bioinf.png" alt="Using Galaxy for Analysis of RNA-Seq, Exome-Seq, and Variants" height="75" /></a>
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> August 1-4 </th>
    <td> <em><a href='http://icbo.cgrb.oregonstate.edu/node/41'>Integrating Ontological and Genomic Analysis with Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> <a href='http://icbo.cgrb.oregonstate.edu/'>ICBO 2016</a>, Corvallis, Oregon, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 9-10 </th>
    <td> <em>Introduction to Galaxy Workflows</em> </td>
    <td> <img src='/images/icons/GL.png' alt='Around the World' align='right' /> <a href='https://sites.google.com/a/illinois.edu/workflows-workshop/home'>Workflows Workshop</a>, <em><a href='https://sites.google.com/a/illinois.edu/workflows-workshop/home/call-for-host-sites'>Distributed</a></em> </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 12 </th>
    <td> <em><a href='http://www.polyomics.gla.ac.uk/course-galaxy-workshop_Aug16.html'>Introduction to Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> University of Glasgow, Glasgow, United Kingdom </td>
    <td> <a href='http://www.polyomics.gla.ac.uk/biog-davidmeltzer.html'>David Meltzer</a> </td>
  </tr>
  <tr>
    <th> August 17 </th>
    <td> <em><a href='https://genomicsvirtuallab.wordpress.com/2016/07/22/galaxy-workshop-at-uq/'>Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> St Lucia campus, University of Queensland, Australia </td>
    <td> <a href="mailto:i.makunin@uq.edu.au">Igor Makunin</a> </td>
  </tr>
  <tr>
    <th> August 18 </th>
    <td> <a href='/community/galaxy-admins/meetups/2016-08-18/'>GalaxyAdmins Web Meetup</a> </td>
    <td> <img src='/images/icons/GL.png' alt='Around the World' align='right' />  <em><a href='/community/galaxy-admins/meetups/2016-08-18/'>Online</a></em> </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a>, <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> August 29 - September 2 </th>
    <td> <em><a href='https://registration.genomecenter.ucdavis.edu/events/Aug_2016_Bioinformatics_Galaxy_Workshop/'>Using Galaxy for Analysis of RNA-Seq, Exome-Seq, and Variants</a></em> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Davis, California, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:training.bioinformatics@ucdavis.edu">UC Davis Bioinformatics Training Program</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> September 3-7 </th>
    <td> <em><a href='http://www.eccb2016.org/programme/tutorials/t2/'>Tutorial: Scientific workflows under Galaxy. Use case: multiple genome comparison</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://www.eccb2016.org/'>European Conference on Computational Biology (ECCB 2016)</a>, The Hague, Netherlands </td>
    <td> <a href='http://www.ac.uma.es/~ots/'>Oswaldo Trelles Salazar</a> </td>
  </tr>
  <tr>
    <th> September 12-16 </th>
    <td> <em><a href='http://sites.unica.it/metabolomicaclinica/events/scientific-school-2016/'>The Workflow4Metabolomics (W4M) online infrastructure for omics analysis with Galaxy</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='http://sites.unica.it/metabolomicaclinica/events/scientific-school-2016/'>Metabolomics Sardinian Scientific School: How to bridge metabolomics and genomics</a>, Pula, Italy </td>
    <td> E Thévenot </td>
  </tr>
  <tr>
    <th> September 12-16 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/fr/evenements/summer_school_metagenomics'>Summer School 2016 in Metagenomics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Institut Pasteur, Paris, France </td>
    <td> <a href='http://www.france-bioinformatique.fr/fr/evenements/summer_school_metagenomics'>Contacts</a> </td>
  </tr>
  <tr>
    <th> September 19-23 </th>
    <td> <em><a href='https://drive.google.com/file/d/0Bw3CPLmTKRT8V1p0QlU5QnYyS28/view'>High Throughput Sequencing (HTS) Data Analyses for Scientists</a></em> <div class='indent red'>This workshop is full</div></td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Freiburg, Germany </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:galaxy@informatik.uni-freiburg.de">Contact</a> </td>
  </tr>
  <tr>
    <th> September 26-28 </th>
    <td> <em><a href='http://biosb.nl/education/course-portfolio/rna-seq/'>NGS Course: RNA-seq data analysis</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Leiden University Medical Center, The Netherlands </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> Peter-Bram ‘t Hoen, Jan Oosting, Celia van Gelder </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> October 10-13 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=83'>Metagenomic: amplicon analysis</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> INRA Auzeville, Auzeville, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> Sarah Maman </td>
  </tr>
  <tr>
    <th> October 18-20 </th>
    <td> <em><a href='http://www.france-bioinformatique.fr/en/evenements/galaxy4bioinformatics'>Galaxy4Bioinformatics</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Lyon, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='http://www.france-bioinformatique.fr/en/node/17'>Contacts</a> </td>
  </tr>
  <tr>
    <th> October 20 </th>
    <td> <em><a href='/events/switzerland2016/'>(third Swiss) Galaxy Workshop</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='/events/sg2016/'>2016 Swiss-German Galaxy Tour</a> in Freiburg, Germany </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <th> October 21 </th>
    <td> <em><a href='/events/germany2016/'>(second German) Galaxy Developers Day</a></em> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> <a href='/events/sg2016/'>2016 Swiss-German Galaxy Tour</a> in Freiburg, Germany </td>
    <td> <a href='/people/hansrudolf-hotz/'>Hans-Rudolf Hotz</a> and <a href='/people/bjoern-gruening/'>Bjoern Gruening</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> November 7-8 </th>
    <td> <strong><a href='/events/admin-training2016/'>Galaxy Admin Training: Introduction</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Salt Lake City, Utah, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/galaxy-team/'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <th> November 9-11 </th>
    <td> <strong><a href='/events/admin-training2016/'>Galaxy Admin Training: Advanced</a></strong> </td>
    <td> <img src='/images/icons/NA.png' alt='North America' align='right' /> Salt Lake City, Utah, United States </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='/galaxy-team/'>Galaxy Team</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" text-align: center; background-color: #eef;"> <span style="font-size: larger;"> <strong>2017</strong> </span> </td>
  </tr>
  <tr>
    <th> February 3-9 </th>
    <td> <strong><a href='http://www.embl-abr.org.au/GAMe2017'>Galaxy Australasia Meeting (GAMe 2017)</a></strong> </td>
    <td> <img src='/images/icons/AU.png' alt='Australia' align='right' /> University of Melbourne, Victoria, Australia </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href='https://www.embl-abr.org.au/organisers/'>Organisers</a> </td>
  </tr>
  <tr>
    <td colspan=4 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> June 26-30 </th>
    <td> <strong><a href='https://gcc2017.sciencesconf.org/'>2017 Galaxy Community Conference (GCC2017)</a></strong> </td>
    <td> <img src='/images/icons/EU.png' alt='Europe' align='right' /> Montpellier, France </td>
    <td> <a href='https://training.galaxyproject.org/'><img src='/images/galaxy-logos/GTN16.png' alt='Training offered by GTN Member' align='right' /></a> <a href="mailto:gcc2017_organization@groupes.france-bioinformatique.fr">Organisers</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='https://training.galaxyproject.org/'><img src="/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='https://training.galaxyproject.org/'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

<div class='right'>
<br /><a href='http://f1000research.com/channels/galaxy'><img src="/images/news-graphics/F1000GalaxyChannelThumb.png" alt="F1000Research Galaxy Channel" width="120" /></a>
</div>

# F1000Research Galaxy Channel

We are pleased to announce the new [F1000Research Galaxy Channel](http://f1000research.com/channels/galaxy).  This channel brings together in one place presentations and posters about the Galaxy framework, including presentations from Galaxy events such as the just finished GCC2016.  If you have presented Galaxy related material at a conference in either [talk](http://f1000research.com/channels/galaxy?&selectedDomain=SLIDES) or [poster](http://f1000research.com/channels/galaxy?&selectedDomain=POSTERS) format, please consider [submitting your presentation to the Galaxy channel](http://f1000research.com/channels/galaxy/for-authors/publish-your-research).

# New Papers

**55 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in July.

Some highlights from last month:

* **[SparkGalaxy: Workflow-based Big Data Processing](http://www.cs.uoregon.edu/Reports/DRP-201603-Riazi.pdf)**
    Sara Riazi, directed research project

* **[A GPU-based high performance computing infrastructure for specialized NGS analyses](https://doi.org/10.7287/peerj.preprints.2175v1)**
    Andrea Manconi *et al., PeerJ Preprints* (28 June 2016), doi:10.7287/peerj.preprints.2175v1

* **[A resource provisioning framework for bioinformatics applications in multi-cloud environments](https://doi.org/10.1016/j.future.2016.06.008)**
    Senturk *et al., Future Generation Computer Systems* (July 2016), doi:10.1016/j.future.2016.06.008

* **[VisualGraphX: Interactive Graph Visualization within Galaxy](https://doi.org/10.1093/bioinformatics/btw414)**
    Richard A. Schäfer and Björn Voß, *Bioinformatics* (13 July 2016), btw414, doi:10.1093/bioinformatics/btw414

## Tagged Publications

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculother'>Other</a>                      </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>     </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>  </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 35 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><br /><img src="/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [PhD Student Bioinformatics](http://www.nature.com/naturejobs/science/jobs/584723-phd-student-bioinformatics), German Cancer Research Center (DKFZ), Munich Germany
* [Ingénieur en développement de composants Galaxy pour la protéomique](http://www.sfbi.fr/content/ing%C3%A9nieur-en-d%C3%A9veloppement-de-composants-galaxy-pour-la-prot%C3%A9omique), MAIAGE (Jouy-en-Josas), France. Closes August 28
* [Post­doctoral Research Position](https://drive.google.com/file/d/0B5_H88SQ1C_7Z3dJY3FzU0hQMUU/view?pref=2&pli=1), Multi­omic Bioinformatics, University of Minnesota
* [eResearch Analyst - Bioinformatics](http://www.seek.com.au/job/31413839), University of Queensland  <div class='red'>Act fast.</span>
* [PhD student or postdoctoral researcher](http://www.uni-freiburg.de/verwaltung/stellenboerse/?erf=y&stellenid=9837&layout=v4), University of Freiburg, Freiburg, Germany. <div class='red'>Act fast.</span>
* [Postdoctoral Researcher: Forest Genomics Database and Software Developer](http://compgenomics.lab.uconn.edu/join/), University of Connecticut.  "... data sharing among partner databases through Galaxy modules." <div class='red'>Act fast.</span>  
* [Software developer and Post-docs](http://gehlenborg.com/open-positions/), Gehlenborg Lab, Harvard Medical School, Boston, Massachusetts, United States

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

----


# Public Galaxy Server News

There are [over 80 publicly accessible Galaxy servers](/use/) and [four semi-public Galaxy services](/use/).  Here's what happened with them in July.

## Galaxy-P

* [Four years of Galaxy-P papers](http://www.ncbi.nlm.nih.gov/pubmed/?term=26684897+26435507+26058579+25658277+25301683+25290099+25149441+26903422+26704769+26954258+26824330+27155584+27225952)!
* [Tim Griffin and colleagues are developing a multiomic platform to help cancer researchers identify meaningful mutations](http://cbs.umn.edu/blogs/cbs-connect/penchant-patterns)

# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

<br />

<div class='right'>
<img src="/images/logos/MapleLeaf.png" alt="Public Health Agency of Canada - National Microbiology Laboratory" height="75" />
</div>

New Galaxy Deployment Catalog and [Community Log Board](/news/) entries were added:

* The Public Health Agency of Canada - National Microbiology Laboratory Galaxy server helps identify, prevent and control infectious diseases.

* [Moving data between Galaxy instances](https://genomicsvirtuallab.wordpress.com/2016/07/13/moving-data-between-galaxy-instances/)

----

# Releases

## Earlier Releases

### Galaxy v16.04

<div class='right'><a href='http://getgalaxy.org'><img src="/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>

The [Galaxy Committers team](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) is pleased to announce the **[April 2016 (v16.04) release of Galaxy](https://docs.galaxyproject.org/en/master/releases/16.04_announce.html)**.

*Please note that Python 2.6 is no longer supported as of this release. See the [16.04 release announcement](https://wiki.galaxyproject.org/News/2016-04_GalaxyRelease) for details.*

<span style="font-size: larger;"> **Highlights** </span>

**Tool Profile Versions**

Tools may now [declare which version](http://planemo.readthedocs.io/en/latest/galaxy_changelog.html#tool-profile-version-pr-1688) of Galaxy they require. Tools requiring 16.04 or newer will have new default behaviors (such as using exit code for error detection) that should simplify tool development. See [PR #1688](https://github.com/galaxyproject/galaxy/pull/1688).

**Embedded Pulsar Job Runner**

Galaxy can now start a Pulsar application embedded within the Galaxy process itself. This allows using Pulsar’s job staging and isolation without requiring a RESTful web service or a message queue. This is enabling usegalaxy.org to run jobs to on the new [JetStream cloud](http://jetstream-cloud.org/). See [PR #2057](https://github.com/galaxyproject/galaxy/pull/2057).

**New Chemical Datatypes**

Galaxy now detects and supports many molecular datatypes. See [PR 1941](https://github.com/galaxyproject/galaxy/pull/1941). Thanks to [Björn Grüning (@bgruening)](https://github.com/bgruening).


<table>
  <tr>
    <td style=" border: none;"> <span style="font-size: larger;"> Github </span> <br /><br /><strong>New</strong><br /><code>% git clone -b master https://github.com/galaxyproject/galaxy.git</code><br /><br />U<strong>pdate to latest stable release</strong><br /><code>% git checkout master && git pull --ff-only origin master</code><br /><br /><strong>Update to exact version</strong><br /><code>% git checkout v16.04</code> </td>
    <td style=" vertical-align: top; border: none; width: 10%;"> </td>
    <td style=" vertical-align: top; border: none;"> <span style="font-size: larger;"> BitBucket </span> <br /><br /><strong>Upgrade</strong><br /><code>% hg pull</code><br /><code>% hg update latest_16.04</code> </td>
  </tr>
  <tr>
    <td colspan=3 style=" border: none;"> See the <a href='/admin/get-galaxy/'>Get Galaxy</a> page for additional details regarding the source code locations. </td>
  </tr>
</table>



#### Galaxy Docker Image 16.04

And, thanks to [Björn Grüning](https://github.com/bgruening), there is also now [a Docker image for Galaxy 16.04](https://github.com/bgruening/docker-galaxy-stable/releases/tag/16.04) as well.  

### blend4php 0.1 alpha

<div class='right'><a href='https://github.com/galaxyproject/blend4php'><img src="/images/galaxy-logos/blend4phpLogo.png" alt="" width="220" /></a></div>

**We are pleased to announce the alpha release of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html).** It follows the lead of [BioBlend](https://bioblend.readthedocs.io/en/latest/) which provides a Python package for interacting with Galaxy and CloudMan--hence the use of 'blend' in the name of this package. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases. Integration of Tripal and Galaxy will allow the community research databases to provide next-generation analytical tools to their users using Galaxy. However, this library was created independently of Tripal to support integration of any PHP application with Galaxy.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/docs-v0.1a/html/index.html) page for full information.


<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/images/galaxy-logos/planemo-logo.png" alt="" width="200" /></a></div>

### Planemo 0.27.0

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  Planemo 0.27.0 was released in June. Some highlights:

* Use ephemeris to handle syncing shed tools for workflow actions. [1c6cfbb](https://github.com/galaxyproject/planemo/commit/1c6cfbb)
* More planemo testing enhancements for testing artifacts that aren’t Galaxy tools. [Pull Request 491](https://github.com/galaxyproject/planemo/pull/491)
* Implement `docker_galaxy` engine type. [eb039c0](https://github.com/galaxyproject/planemo/commit/eb039c0), [Issue 15](https://github.com/galaxyproject/planemo/issues/15)
* Enhance profiles to be Dockerized Galaxy-aware. [Pull Request 488](https://github.com/galaxyproject/planemo/pull/488)
* Add linter for DOI type citation - thanks to [@mvdbeek](https://github.com/mvdbeek). [Pull Request 484](https://github.com/galaxyproject/planemo/pull/484)

See the [release history](http://planemo.readthedocs.org/en/latest/history.html#history).


### galaxy-lib 16.7.8 - 16.7.9

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

Highlights of June releases include:

* Updates for recent changes to Galaxy and cwltool.
* Updates to include Galaxy library for verifying test outputs and the latest dev changes to Galaxy.

### CloudMan 16.04

<div class='right'><a href='/cloudman/'><img src="/images/galaxy-logos/cloudman-logo.jpg" alt="CloudMan" width="300" /></a></div>

**We just released an update to Galaxy CloudMan on AWS.** CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration or imposed quotas. Once running, you have complete control over Galaxy, including the ability to install new tools.

This is a minor update release with the following changes:

* Galaxy 16.04 update
* Availability on Amazon's Ireland region
* A couple of bug fixes

See the [CHANGELOG](https://github.com/galaxyproject/cloudman/blob/master/CHANGELOG.md) for a more complete set of changes.

<div class='right'><br /><a href='https://pypi.python.org/pypi/pulsar-app/'><img src="/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="150" /></a></div>

### Pulsar 0.7.0

[Pulsar 0.7](https://pypi.python.org/pypi/pulsar-app/) was released in April.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.


### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

[Tool Shed contributions for June and July](/toolshed/contributions/2016-07/).

# Other News

* [Galaxy Communication server](https://github.com/galaxyproject/galaxy/pull/2515): Realtime communication server and its user-interface
* [Supporting Community Driven Automated QA](https://github.com/galaxyproject/galaxy/issues/2642)
* [Public Health England's SNP calling pipeline](https://github.com/phe-bioinformatics/PHEnix)
* [New tutorial](http://bit.ly/TAGC_GXY_PDF) on integrating phenotype data from ZFIN and MGI.
* [how to use GenomeSpace tools in a Galaxy workflow](https://www.youtube.com/watch?v=5QPtWS_ab0I). Video from the Genomics Virtual Lab folks.
* New interactive Galaxy tours are available:
    * [MultiQC](https://github.com/galaxyproject/galaxy-tours/tree/master/multiqc) and [RAD-Seq](https://github.com/galaxyproject/galaxy-tours/tree/master/rad-seq)
