---
title: Galaxy taking part @ Applied Bioinformatics & Public Health Microbiology 2023
date: '2023-05-06'
tease: "From the 3rd to the 5th of May, Engy Nasr from the Freiburg Galaxy team travelled to Hinxton, England to join and present her work at the Applied Bioinformatics & Public Health Microbiology 2023 conference"
location:
  name: Hinxton, UK
tags: [conference, talk, bioinformatics showcase, pathogen detection]
authors: engynasr
supporters:
- microbial genomics
- EOSC-Life
- ELIXIR
components: true
subsites: [all-eu]
---

From the 3rd to the 5th of May, Engy Nasr (I) from the Freiburg Galaxy team travelled to Hinxton, England to join and present my work about the 'Foodborne Pathogen Detection using Direct Nanopore' at the [**Applied Bioinformatics & Public Health Microbiology 2023 conference**](https://coursesandconferences.wellcomeconnectingscience.org/event/applied-bioinformatics-public-health-microbiology-20230503/).

<twitter tweet="1653738329647595521"></twitter>
<twitter tweet="1654050208056049665"></twitter>

The 9th Applied Bioinformatics and Public Health Microbiology conference, which took place in the **Wellcome Genome Campus**, provided a multidisciplinary forum to demonstrate how advances in microbial and viral genomics, bioinformatics, data science, and sequencing technology are being used to meet the needs of public health.

This year’s meeting focused on how these advances are transforming public health across the globe, including lessons learned from the SARS-Cov2 pandemic. The conference included sessions exploring public health genomics, genomic epidemiology, metagenomics for surveillance, and new bioinformatics approaches – focusing on pipelines and data management. A large proportion of the programme was chosen from submitted abstracts.

The conference aim is to share best practices for implementing genomics technology in routine, frontline diagnostics and surveillance, with a particular focus on the implications of using data-driven genomics approaches in public health. The conference also provided an opportunity for bioinformaticians working in this area to exhibit the innovative tools that they have developed.

The conference attracts international participants working in bioinformatics or microbiology research or public health, including reference laboratories or hospital settings. This conference had strong representation from clinical microbiologists and clinical bioinformaticians, to ensure appropriate migration of these technologies to diagnostic and surveillance applications.

On the first day; there were two talks sessions followed by a poster session. 111 posters were presented on the first and the second day. The first talks session was about new approches in bioinformatics and the second was about genomic epidemiology in the field.

On the second day; we had three talks sessions; two of them were about the public health and the third was about metagenomics for survellance. At the end of the second talks session, I have given a [short talk](https://docs.google.com/presentation/d/1zlk3NsbmnzUx4HVXr7wifXHcF93u7DtKONfVluk4Y20/edit?usp=sharing) about my project **Foodborne Pathogen Dectection using direct Nanopore** and how is it importand to implement it within Galaxy and I have also invited everyone to my session in the Bioinformatics showcase, taking place on the same day, for more details and more practical elaboration on the Galaxy platform. Before the third talks session, we had the Bioinformatics showcase where I presented my work and showed the Pathogen Detection workflow me, Bérénice Batut and Paul Zierep have created in Galaxy along with its [**training material**](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html). There were 16 abstracts presented in the bioinformatics showcase, and we won the award as the **best abstract** presented.

<twitter tweet="1654400374583746561"></twitter>

From the Bioinformatics showcase and after attending the poster sessions over the two days, I have got a chance to exchange some ideas, e.g. databases that I can use to have more test data for the pathogen detection workflow, other systems (e.g. [CZ ID](https://czid.org/)) to compare our workflow with also other tools to add to Galaxy and add to the workflow (e.g. [PGGB](https://github.com/pangenome/pggb), [Strainberry](https://github.com/rvicedomini/strainberry) and [cgMLST](https://github.com/aldertzomer/cgmlst)). Attendees visted my showcase had a chance to learn more about Galaxy and know how can they continue learing and using our platform. They have also taken pictures of our workflow and learned where is it openly avaiable śo that they can directly use it to update their pipelines or use it directly within Galaxy.

On the third and last day; we had a panel discussion regarding preparing for the next pandemic then we had a closing session where Chikwe Ihekweazu from the [**WHO Hub for Pandemic and Epidemic Intelligence**](https://pandemichub.who.int/) gave the final talk annoncing the launching of the [**International Paqthogen Surveillance Network (IPSN)**](file:///home/nasr/Downloads/9789240046979-eng-1.pdf) on the 20th of May 2023

<twitter tweet="1654485708378128388"></twitter>

Overall, I have really enjoyed attending the conference. I met a lot of scientist working on similar projects where we had a chance to exchange our knowledge. I would like to thank the Microbial Genomics for choosing my abstract to win the award and the **Wellcome Genome Campus** of having me attenging the conference. Wish to see all the people I met in 2 years in the 10th Applied Bioinformatics and Public Health Microbiology conference.
