---
autotoc: true
title: October 2013 Galaxy Update
---


<div class='right'></div>

<div class='left'><a href='/src/GalaxyUpdates/index.md'><img src="/src/Images/Logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the October 2013 *[Galaxy Update](/src/GalaxyUpdates/index.md)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/src/GalaxyUpdates/index.md)* complement the *[Galaxy Development News Briefs](/src/DevNewsBriefs/index.md)* which accompany new Galaxy releases and focus on Galaxy code updates.

<br />


# New Public Servers

*A record six new servers* joined [the list of over 40 publicly accessible Galaxy servers](/src/PublicGalaxyServers/index.md) in September.

<div class='left'><br /><div class='solid'><a href='http://galaxy.csdb.cn:8000/'><img src="/src/PublicGalaxyServers/CNIC.DarwinTreeHome.png" alt="CNIC.DarwinTree" width="200" /></a></div></div>

## CNIC.DarwinTree

[CNIC.DarwinTree](http://galaxy.csdb.cn:8000/) is a molecular data analysis and application environment for DarwinTree, GSQCS, iDNAbar, NANNO, IPIP, and many other tools for phylogenetic analysis.

[CNIC.DarwinTree](http://galaxy.csdb.cn:8000/) is supported by the [Chinese Academy of Sciences Computer Network Information Center (中国科学院计算机网络信息中心)](http://www.cnic.cn/), and has [mailto:support AT cnic DOT cn](Email support) as well.

## Fast UniFrac

<div class='right solid'><a href='http://unifrac.colorado.edu'><img src="/src/PublicGalaxyServers/FastUniFracHomePage.png" alt="Fast UniFrac" width="200" /></a></div>

[Fast UniFrac](http://unifrac.colorado.edu) provides a suite of tools for the comparison of microbial communities using phylogenetic information. See "[Fast UniFrac: facilitating high-throughput phylogenetic analyses of microbial communities including analysis of pyrosequencing and PhyloChip data](http://www.nature.com/ismej/journal/v4/n1/full/ismej200997a.html)" by M Hamady, C Lozupone and R Knight, *[The ISME Journal](http://www.nature.com/ismej/index.html)* (2010) 4, 17–27; doi:10.1038/ismej.2009.97

No login is required to use [Fast UniFrac](http://unifrac.colorado.edu) and the site has [mailto:MicrobiomeHelp AT colorado DOT edu](email support).  [Fast UniFrac](http://unifrac.colorado.edu) is sponsored by the [Knight Lab at the University of Colorado at Boulder](https://knightlab.colorado.edu/).

## kmer-SVM

<div class='left solid'><a href='http://kmersvm.beerlab.org/'><img src="/src/PublicGalaxyServers/kmer-SVM.png" alt="kmer-SVM" height="133" /></a></div>

[kmer-SVM](http://kmersvm.beerlab.org/) is "a tool suite designed to aid in analysis of next-generation sequencing (NGS) data. Our suite uses a support vector machine (SVM) with kmer sequence features to identify predictive combinations of short transcription factor binding sites which determine the tissue specificity of the original NGS assay. Information gained from kmer-SVM can be used as an additional source of confidence in genomic experiments by recovering known binding sites, and can also reveal novel sequence features and possible cooperative mechanisms to be tested experimentally."

The kmer-SVN server is describeded in "[kmer-SVM: a web server for identifying predictive regulatory sequence features in genomic data sets](http://nar.oxfordjournals.org/content/41/W1/W544.long)." Christopher Fletez-Brant; Dongwon Lee; [Andrew S. McCallion](http://www.hopkinsmedicine.org/geneticmedicine/people/faculty/mccallion.html); [Michael A. Beer](http://www.beerlab.org/) *[Nucleic Acids Research](http://nar.oxfordjournals.org/)* 2013; doi: 10.1093/nar/gkt519<br />

A [tutorial on using the web server](http://kmersvm.beerlab.org/tutorial/) and a [Galaxy Tool Shed repository](http://toolshed.g2.bx.psu.edu/view/cafletezbrant/kmersvm) are also available. [mailto:kmersvm DOT team AT gmail DOT com](Email support) is provided, and the project is a collaboration between Christopher Fletez-Brant and Dongown Lee respectively of the [McCallion Lab](http://www.hopkinsmedicine.org/geneticmedicine/people/faculty/mccallion.html) of the [McKusick-Nathans Institute of Genetic Medicine](https://igm.jhmi.edu/) at the Johns Hopkins University School of Medicine and the [Beer Lab](http://www.beerlab.org/) of the [Johns Hopkins University Department of Biomedical Engineering](http://www.bme.jhu.edu/).

## NGS-QC Generator

<div class='right solid'><a href='http://galaxy.igbmc.fr/galaxy/'><img src="/src/PublicGalaxyServers/NGS-QC_Generator.png" alt="Gronemeyer Lab Galaxy and NGS-QC Generator" width="200" /></a></div>

[NGS-QC Generator](http://galaxy.igbmc.fr/galaxy/) evaluates the quality of ChIP-seq and enrichment-related NGS data using [NGS-QC Generator](http://archive.igbmc.fr/recherche/Prog_FGC/Eq_HGron/Bioinfotools/NGS/website/index.php), which also includes a [database with preprocessed profiles](http://archive.igbmc.fr/recherche/Prog_FGC/Eq_HGron/Bioinfotools/NGS/website/database.php) and a [tutorial on how to analyze sequencing profiles yourself](http://archive.igbmc.fr/recherche/Prog_FGC/Eq_HGron/Bioinfotools/NGS/website/tutorial.php). 

See *[A quality control system for profiles obtained by ChIP sequencing](http://nar.oxfordjournals.org/content/early/2013/09/14/nar.gkt829)* by Mendoza-Parra, *et al.*, in *Nucl. Acids Res.* (2013)

The server is hosted by the [Gronemeyer lab](http://igbmc.fr/research/department/2/team/21/) and [email support](/src/PublicGalaxyServers/index.md#ngs-qc-generator) is available.  You must have an account to use the server; anyone can create an account. "Due to storage space constraints, uploaded datasets into the Galaxy instance may not be available for more than 24hours, thus we strongly suggest users to download their processed files as early as possible."

## USP-FCFRP

<div class='right solid'><a href='http://fisbio.fcfrp.usp.br:8080/'><img src="/src/PublicGalaxyServers/USP-FCFRP.png" alt="Laboratory of Biological Physics at USP-FCFRP" width="200" /></a></div>

The [Laboratory of Biological Physics at USP-FCFRP](http://fisbio.fcfrp.usp.br:8080/) Galaxy server has these tools:
 **ProtPred-GROMACS**: *Ab initio* protein structure prediction framework which uses evolutionary algorithms to optimize objective functions such as potential energy, number of hydrogen bonds and solvent accessible surface area. 
    2PG: Faster and simpler algorithm in which one objective function is optimized<br />
    3PG: A more robust approach in which multiple objective functions are optimized at the same time
 **Protein Validation**: Check the integrity of a PDB file<br />
 **Protein Desolvation**: Remove a solvation layer around your protein.

The [Laboratory of Biological Physics](http://fisbio.fcfrp.usp.br/), part of the [Faculty of Pharmaceutical Sciences of Ribeirão Preto (FCFRP)](http://www.fcfrp.usp.br/) at the [University of São Paulo (USP)](http://www.usp.br/).
"You can use tools developed at our laboratory and by our collaborators for free. You just need to sign up to the server using your academic e-mail."

## ValidatorMAX

<div class='right solid'><a href='http://msvalidator.uchicago.edu/galaxy/'><img src="/src/Images/Logos/UChicagoLogo.jpg" alt="ValidatorMAX at The University of Chicago" width="200" /></a></div>

[This server](http://msvalidator.uchicago.edu/galaxy/) implements Validator<sup>MAX</sup>: confident identification of stable isotope-labeled peptide pairs and associated abundance ratios. The server is sponsored by the [Department of Molecular Genetics and Cell Biology](http://mgcb.bsd.uchicago.edu/), [Ludwig Center for Metastasis Research](http://cancer.uchicago.edu/research/uchicago-advantage/metastasis.shtml), [Department of Pediatrics](http://pediatrics.uchicago.edu/), [Center for Research Informatics](http://cri.uchicago.edu/), [Computation Institute](http://www.ci.uchicago.edu/), [The University of Chicago](http://www.uchicago.edu/)

<br />

# New Papers

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=6 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=6 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 21 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseCloud</a> </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
</table>


44 new papers were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in September. In addition to papers featuring the [kmer-SVM](/src/GalaxyUpdates/2013_10/index.md#kmer-svm) and [NGS-QC Generator](/src/GalaxyUpdates/2013_10/index.md#ngs-qc-generator) (see [above](/src/GalaxyUpdates/2013_10/index.md#new-public-servers)) these papers may be particularly interesting to the Galaxy community:

* "[Genomics in the clouds](http://dx.doi.org/10.1038/nmeth.2654)" by Vivien Marx, *[Nature Methods](http://www.nature.com/nmeth/index.html)* 10, 941–945 (2013)

* "[Galaxy tools and workflows for sequence analysis with applications in molecular plant pathology](http://dx.doi.org/10.7717/peerj.167)." by Cock PJ, Grüning BA, Paszkiewicz K, Pritchard L. *[PeerJ](https://peerj.com/)* 1:e167

* "[TALENoffer: genome-wide TALEN off-target prediction](http://dx.doi.org/10.1093/bioinformatics/btt501)" by Jan Grau, Jens Boch, Stefan Posch, *Bioinformatics* (30 August 2013), doi:10.1093/bioinformatics/btt501

And the Galaxy Project now has it's own [CITATION file](https://bitbucket.org/galaxy/galaxy-central/src/tip/CITATION). [CITATION files](http://software-carpentry.org/blog/2013/09/introducing-citation-files.html) "tell readers how best to cite that software."  Also see the [Citing Galaxy](/src/CitingGalaxy/index.md) wiki page.

<br />

# Who's Hiring

<div class='right'><a href='/src/GalaxyIsHiring/index.md'><img src="/src/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.
* [Statistical Genomics Postdoc](/src/News/StatisticalGenomicsPostDocPennState/index.md) opening in the Makova lab at Penn State
* [Computational biology opening](/src/News/ComputationalBiologyParis/index.md) at University Pierre-et-Marie-Curie, Paris
* [M2 Développement et intégration d'outils pour la bioanalyse dans Galaxy](http://www.sfbi.fr/content/m2-d%C3%A9veloppement-et-int%C3%A9gration-doutils-pour-la-bioanalyse-dans-galaxy), Limagrain à Chappes (Puy-de-Dôme, Auvergne)
* [Part time position at GenePeeks](http://bit.ly/18ExEfk) 
* [Stage M2 Développement et intégration d'outils pour la bioanalyse dans Galaxy chez Vilmorin & Cie.](http://bit.ly/19AYF4e)
* [PhD or postdoc position available](http://bit.ly/15Iz3nL) at [Laboratory of Computational Biology](http://med.kuleuven.be/lcb/), University of Leuven 
* **The [Galaxy Project is hiring software engineers and post-docs](/src/GalaxyIsHiring/index.md)**.
* [Sr Bioinformatics Specialist, Tufts University](http://bit.ly/1bOLLBd), Boston MA.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

# Events

<div class='right'><a href='http://bit.ly/gxycal'><img src="/src/Images/Icons/CalendarIcon.gif" /></a></div> There is a lot going on in the next three months, *with half of it happening in the Southern Hemisphere.*  Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<div class='left'>
<br />
## Australia!

</div>
<div class='center'><a href='http://www.rcpa.edu.au/Continuing/CalendarOfEvents/September2013.htm'><img src="/src/Images/Logos/RCPALogo.jpg" alt="The Genomic Bioinformatics Workshop" height="60" /></a>
<a href='http://amataconference.com.au/2013/conference-program/pre-conference-workshops/'><img src="/src/Images/Logos/AMATA2013LogoName.png" alt="AMATA early career researcher workshop" height=80" /></a>
<a href='http://conference.eresearch.edu.au/eres2013/workshops-2013/'><img src="/src/Images/Logos/eResearchAustralAsiaSmall.png" alt="eResearch Australasia 2013" height="60" /></a>
<a href='http://www.qfab.org/events/upcoming/'><img src="/src/Images/Logos/QFABLogo.png" alt="QFAB Workshops" height="80" /></a>
</div>
<br />
In the next 3 months there are no less than 13 talks and workshops at at least 5 different venues on at least 7 distinct dates.  Those 13 known events include 8 workshops.  If you don't find some Galaxy training this spring then you aren't trying.  

<br />

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> September&nbsp;28 - October&nbsp;1 </th>
    <td> <em>Galaxy Workshop</em> </td>
    <td> <a href='http://www.rcpa.edu.au/Continuing/CalendarOfEvents/September2013.htm'>The Genomic Bioinformatics Workshop</a>, Sydney, Australia </td>
    <td> <a href='/src/RossLazarus/index.md'>Ross Lazarus</a>, <a href='/src/Dan/index.md'>Dan Blankenberg</a> </td>
  </tr>
  <tr>
    <th> October 13-16 </th>
    <td> <a href='http://amataconference.com.au/2013/conference-program/pre-conference-workshops/'>AMATA early career researcher workshop</a> </td>
    <td> <a href='http://amataconference.com.au/2013/'>AMATA 2013</a>, Gold Coast, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a>, Annette McGrath </td>
  </tr>
  <tr>
    <th rowspan=6> October 20-25 </th>
    <td> <em><a href='http://eresearchau.files.wordpress.com/2013/08/eresau2013_submission_5.pdf'>Genomics Virtual Laboratory Workshop</a></em> </td>
    <td rowspan=6> <strong><a href='http://conference.eresearch.edu.au/eres2013/workshops-2013/'>eResearch Australasia 2013</a></strong>, Brisbane, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a>, Ron Horst, Andrew Lonie </td>
  </tr>
  <tr>
    <td> <em><a href='http://eresearchau.files.wordpress.com/2013/08/eresau2013_submission_106.pdf'>Image Analysis and Processing in the Clouds using Scalable eResearch Workflow-Based System</a></em> </td>
    <td> <a href='http://cloudimaging.blogspot.com.au/p/about-team.html'>Project Team</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://eresearchau.files.wordpress.com/2013/08/eresau2013_submission_32.pdf'>MILXCloud: A faster, smarter way to process Medical imaging data in the cloud</a></em> </td>
    <td> <a href="mailto:Neil DOT Burdett AT csiro DOT au">Neil Burdett</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://eresearchau.files.wordpress.com/2013/08/eresau2013_submission_10.pdf'>Lessons from developing the Genomics Virtual Lab</a></em> </td>
    <td> <a href="mailto:r DOT horst AT uq DOT edu DOT au">Ron Horst</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://eresearchau.files.wordpress.com/2013/08/eresau2013_submission_69.pdf'>Early Outcomes of the Characterisation Virtual Laboratory</a></em> </td>
    <td> <a href="mailto:wojtek DOT goscinski AT monash DOT edu">Wojtek Goscinski</a>, <a href="mailto:anitha DOT kannan AT monash DOT edu">Anitha Kannan</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://eresearchau.files.wordpress.com/2013/08/eresau2013_submission_16-2.pdf'>The Human Communication Science Virtual Lab</a></em> </td>
    <td> <a href="mailto:p DOT sefton AT uws DOT edu DOT au">Peter Sefton</a> </td>
  </tr>
  <tr>
    <th> November 4 </th>
    <td> <em><a href='http://www.qfab.org/event/rna-seq-analysis-using-galaxy-workshop/'>RNA-Seq Analysis using Galaxy workshop</a></em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 6 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-an-introduction-to-variant-detection-using-galaxy/'>Workshop: An Introduction to Variant Detection using Galaxy</a></em> </td>
    <td> Children’s Medical Research Institute, Westmead, New South Wales, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 7 </th>
    <td> <em><a href='http://www.qfab.org/event/rna-seq-analysis-using-galaxy-4/'>RNA-Seq Analysis using Galaxy workshop</a></em> </td>
    <td> Children’s Medical Research Institute, Westmead, New South Wales, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> November 22 </th>
    <td> <em><a href='http://www.qfab.org/event/variant-detection-using-galaxy-workshop/'>Variant Detection using Galaxy workshop</a></em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> December 9 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-an-introduction-to-de-novo-genome-assembly-using-galaxy/'>Workshop : An Introduction to De Novo Genome Assembly using Galaxy</a></em> </td>
    <td> The University of Queensland, St Lucia, Queensland, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
</table>



<br />

## All Those Other Continents

And don't worry, you won't have to travel to Australia during its spring just to learn about Galaxy.  There are also upcoming events in North America, Europe, and Africa.

<div class='center'> 
<a href='http://individualizingmedicineconference.mayo.edu/schedule/'><img src="/src/Images/Logos/MayoClinicSmall.png" alt="Individualizing Medicine Conference" height="60" /></a> &nbsp;
<a href='http://www.beyond-the-genome.com/program/'><img src="/src/Images/Logos/BeyondTheGenome2013.png" alt="Beyond the Genome" height="60" /></a> &nbsp;
<a href='http://bit.ly/153avr8'><img src="/src/Images/Logos/OrioneNGSCourse.png" alt="Analisi dati Next Generation Sequencing in Galaxy: exome, RNA-Seq, metagenomica" height="65" /></a> &nbsp;
<a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=43'><img src="/src/Images/Logos/GennoToulLogo.png" alt="Galaxy Training Days" height="60" /></a>
<a href='/src/Events/SAWorkshopTour2013/index.md'><img src="/src/Images/Logos/SAGalaxyWorkshopTour2013Wide.png" alt="2013 South Africa Galaxy Workshop Tour" height="65" /></a>
</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> September 30 - October 2 </th>
    <td> <em>Analysis of Genomic Sequence Data With Galaxy</em>, part of the <em><a href='http://individualizingmedicineconference.mayo.edu/schedule/'>Cancer Care</a></em> session </td>
    <td> <a href='http://individualizingmedicineconference.mayo.edu/'>Individualizing Medicine Conference</a>, <a href='http://mayo.eduy/'>Mayo Clinic</a>, Rochester, Minnesota, United States </td>
    <td> <a href='/src/JamesTaylor/index.md'>James Taylor</a> </td>
  </tr>
  <tr>
    <th rowspan=2> October 1-3 </th>
    <td> <em><a href='http://www.beyond-the-genome.com/program/'>Galaxy as a platform for High-throughput Genomics</a></em> </td>
    <td rowspan=2> <strong><a href='http://www.beyond-the-genome.com/'>Beyond the Genome 2013</a></strong>, San Francisco, California, United States </td>
    <td> <a href='/src/JeremyGoecks/index.md'>Jeremy Goecks</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.beyond-the-genome.com/program/'>Development of a genomic region database and analysis tool for the Galaxy platform</a></em> </td>
    <td> <a href="mailto:Matloob DOT khushi AT sydney DOT edu DOT au">Matloob Khushi</a> </td>
  </tr>
  <tr>
    <th rowspan=2> October 7-8 </th>
    <td> <em><a href='https://www.gtcbio.com/conference/ngseurope-agenda#Day 1 -BIO'>Next-Generation Sequencing Data Interpretation: Enhancing Reproducibility and Accessibility</a></em></td>
    <td rowspan=2> <a href='http://www.gtcbio.com/conference/ngseurope-overview'>NGS & Bioinformatics Summit Europe</a>, Berlin, Germany </td>
    <td> <a href='/src/anton/index.md'>Anton Nektutenko</a> </td>
  </tr>
  <tr>
    <td> <em><a href='https://www.gtcbio.com/conference/ngseurope-agenda#Day 1 -BIO'>Using Galaxy to Provide a NGS Analysis Platform</a> </em> </td>
    <td> <a href='/src/HansrudolfHotz/index.md'>Hans-Rudolf Hotz</a> </td>
  </tr>
  <tr>
    <th> October 8-11 </th>
    <td> <em><a href='http://bit.ly/153avr8'>Analisi dati Next Generation Sequencing in Galaxy: exome, RNA-Seq, metagenomica</a></em> </td>
    <td> <a href='http://www.bioinformatica.crs4.it/'>CRS4, Loc. Pixinamanna, Pula CA, Italy</a> </td>
    <td> <a href="mailto:ngs-courses@crs4.it">Paolo Uva</a> </td>
  </tr>
  <tr>
    <th> October 9-11 </th>
    <td> <em><a href='http://bioinfo.genotoul.fr/index.php?id=34&tx_seminars_pi1%5BshowUid%5D=43'>Galaxy Training Days</a></em> </td>
    <td> <a href='http://bioinfo.genotoul.fr/'>GenoToul bioinformatics facility</a>, INRA, Toulouse Auzeville, France </td>
    <td> <a href="mailto:sarah DOT maman AT toulouse AT inra AT fr">Sarah Maman</a> </td>
  </tr>
  <tr>
    <th> October 14-18 </th>
    <td> <em><a href='/src/Events/UPretoria2013/index.md'>NGS Data Analysis and Galaxy Workshop</a></em> <div class='indent'>University of Pretoria</div> </td>
    <td rowspan=2> <a href='/src/Events/SAWorkshopTour2013/index.md'>2013 South Africa Galaxy Workshop Tour</a><div class='indent red'>Application deadline is 17 September</div> </td>
    <td rowspan=2> <a href='/src/DaveClements/index.md'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> October 21-25 </th>
    <td> <em><a href='/src/Events/UCapeTown2013/index.md'>NGS Data Analysis and Galaxy Workshop</a></em> <div class='indent'>University of Cape Town</div> </td>
  </tr>
  <tr>
    <th> October 22 </th>
    <td> <em><a href='http://bioteam.net/slipstream/galaxy-edition/galaxy-training/'>Analyzing NGS Data with Galaxy</a></em> </td>
    <td> <a href='http://www.dana-farber.org/'>Dana-Farber Cancer Institute</a>, Boston, Massachusetts, United States </td>
    <td> <a href='/src/anton/index.md'>Anton Nekrutenko</a>, <a href='/src/JenniferJackson/index.md'>Jennifer Jackson</a>, <a href='http://bioteam.net/company-leadership/'>Anushka Brownley</a> </td>
  </tr>
  <tr>
    <th rowspan=5> October 22-26 </th>
    <td> <em><a href='http://www.ashg.org/2013meeting/pages/workshops.shtml#intro'>Introduction to Integrative Analysis with GenomeSpace</a></em> </td>
    <td rowspan=5> <strong><a href='http://www.ashg.org/2013meeting/'>ASHG 2013</a></strong>, Boston, Massachusetts, United States </td>
    <td> <a href='http://www.broadinstitute.org/science/programs/cancer/icbp/integrated-team-0'>Michael Reich</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.ashg.org/2013meeting/pages/workshops.shtml#high'>High Throughput Data Analysis and Visualization with Galaxy</a></em> <div class='indent red'>This workshop is sold out.</div> </td>
    <td> <a href='/src/anton/index.md'>Anton Nekrutenko</a>, <a href='/src/JenniferJackson/index.md'>Jennifer Jackson</a> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.ashg.org/2013meeting/abstracts/fulltext/f130122946.htm'>Poster 1633T: Globus Genomics: Enabling high-throughput analysis and management of NGS data for neurodevelopmental disorders</a></em> </td>
    <td rowspan=3> <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a>, Dinanath Sulakhe, Alex Rodriguez and Paul Dave<br />And see the Globus Genomics Booth </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.ashg.org/2013meeting/abstracts/fulltext/f130121257.htm'>Poster 1530W: Implementing a High Performance, Reusable Consensus Calling Pipeline for Next Generation Sequencing using Globus Genomics</a></em> </td>
  </tr>
  <tr>
    <td> <em><a href='http://www.ashg.org/2013meeting/abstracts/fulltext/f130122827.htm'>Poster 1510T: Consensus Genotyper for Exome Sequencing: Improving the Quality of Exome Variant Genotypes</a></em> </td>
  </tr>
  <tr>
    <th> October 31 - November 1 </th>
    <td> <em><a href='http://www.nbic.nl/education/course-overview/pages3/1/coursedetails/advanced-ngs-course-rna-seq-data-analysis-3th-edition/'>Advanced NGS Course: RNA-Seq Data Analysis</a></em> </td>
    <td> Leiden, The Netherlands </td>
    <td> <a href='http://www.nbic.nl/about-nbic/nbic-faculty/details/celia-van-gelder/'>Celia van Gelder</a>, <a href='http://www.nbic.nl/about-nbic/nbic-faculty/details/hoen-t-peter-bram-dr/'>Hoen, 't Peter-Bram</a> </td>
  </tr>
  <tr>
    <th> November 2-6 </th>
    <td> <em><a href='https://apha.confex.com/apha/141am/webprogramadapt/Paper287924.html'>A zebrafish cloud computational resource for environmental health science research</a></em> </td>
    <td> <a href='http://www.apha.org/meetings/AnnualMeeting/'>141st American Public Health Association (APHA) Annual Meeting</a>, Boston, Massachusetts, United States </td>
    <td> <a href='https://apha.confex.com/apha/141am/webprogramadapt/Person216879.html'>Peter Tonellato</a> </td>
  </tr>
  <tr>
    <th> November 6-12 </th>
    <td> <em><a href='http://meetings.cshl.edu/courses/2013/c-ecg13.shtml'>Computational and Comparative Genomics Course</a></em><div class='indent red'>Application Deadline: July 15, 2013</div> </td>
    <td> <a href='http://cshl.edu'>Cold Spring Harbor Laboratory</a>, New York, United States </td>
    <td> <a href='/src/JamesTaylor/index.md'>James Taylor</a> </td>
  </tr>
  <tr>
    <th> January 11-15 </th>
    <td> <a href='http://www.intlpag.org/'>Plant and Animal Genome XXII (PAG 2014)</a> </td>
    <td> San Diego, California, United States </td>
    <td> <a href='/src/DaveClements/index.md'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> January 16-17 </th>
    <td> <a href='http://gmod.org/wiki/Jan_2014_GMOD_Meeting'>2014 GMOD Meeting</a> </td>
    <td> San Diego, California, United States </td>
    <td> <a href='/src/DaveClements/index.md'>Dave Clements</a> </td>
  </tr>
</table>


## Online Materials from Past Events

<div class='right'>
<a href='http://bit.ly/gcc2013vimeo'><img src="/src/Images/Logos/GCC2013Logo200.png" alt="2013 Galaxy Community Conference (GCC2013)" width="120px" /></a><br /><br /><div class='right'>
<a href='http://bit.ly/16rAUkf'><img src="/src/Images/Logos/UCDavisGenomeCenterLogo.jpg" alt="Introduction to Galaxy Boot Camp @ UC Davis Bioinformatics Core" width="80" /></a>
</div></div>

First, all videos from GCC2013 are [now also available on Vimeo](http://bit.ly/gcc2013vimeo).  They are still available on the Galaxy web site as well, but having them on Vimeo allows you to do all the things an online video provider supports, such as automatically embedding and linking videos from popular web sites.

Two other resources (at least) became available in September:

* [Informatics on High Throughput Sequencing Data Workshop](http://bit.ly/VyBEcS) from Bioinformatics.ca now has [slides & video online](http://bit.ly/VyBEcS).

* Course Materials for the [2013 UC Davis Bioinformatics Short Course](http://bit.ly/16rAUkf): This course featured 5 great days of lecture and hands-on exercises.  *And,* an Amazon Web Services AMI that includes all tools and data used in the course, is also available. 

See the [list of other tutorials](/src/Learn/index.md#other-tutorials) on the [Learn hub page](/src/Learn/index.md) for more.
<br />

# Galaxy Distributions

The most recent [Galaxy distribution was August 12](/src/DevNewsBriefs/2013_08_12/index.md).

A new version of [CloudMan](/src/CloudMan/index.md) was [released in July](/src/News/CloudManRelease/index.md).

<br />

# Tool Shed Contributions

<div class='right'><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/Images/Logos/ToolShed.jpg" alt="Galaxy Tool Shed" width=150 /></a></div>

There were many...
* [naive_variant_caller](http://bit.ly/157Yf3G): process aligned reads, produce VCF file containing per position variant calls 
* [barrnap](http://bit.ly/15kPjMj): ribosomal RNA finder for Bacterial genome sequences  
* [prokka](http://bit.ly/17XlJej) (and [a second wrapper](http://bit.ly/1d6tK3R)): rapid annotation of prokaryotic genomes
* [coverage_report](http://bit.ly/1e3yfiI):  Generate Detailed Coverage Report from BAM file  
* [mugsy](http://bit.ly/19IFR5c): multiple whole genome aligner 
* [edena](http://bit.ly/18Vnzez): a de novo short reads assembler
* [edge_pro](http://bit.ly/15TzWXS): efficient gene expression level estimation in prokaryotic genomes from RNA-seq
* [glimmer](http://bit.ly/1d6ta67): find genes in microbial DNA, especially bacteria, archaea, and viruses 
* [get_fasta_from_taxon](http://bit.ly/15Xmxgm): Get FASTA from NCBI taxonomy ID 
* [sniploid2](http://bit.ly/181OqEW): compare SNPs detected from a polyploid to SNPs derived from its parental genomes 
* [ssake](http://bit.ly/15kU6gR): de novo assembly of millions of very short DNA sequences  
* [sspace](http://bit.ly/15AqpbP): scaffolding pre-assembled contigs using paired-read data   
* [annovar](http://bit.ly/1gTsYXO): Functional annotation of genetic variants from high-throughput sequencing data 
* [merge_fna_qual](http://bit.ly/16tT2Dg): Merges 454 Fasta and Quality files into a Fastq file. 
* [ncbi_sra_toolkit](http://bit.ly/1fsby9r): NCBI Sequence Read Archive toolkit utilities 
* [sql_tools and sparql_tools](http://bit.ly/16tTfgD): In-memory SQL and SPARQL runners and related tools
* compute_motifs_frequency, ctd_batch, compute_motif_frequencies_for_all_motifs, indels_3way, t_test_two_samples, xy_plot, divide_pg_snp, draw_stacked_barplots, split_paired_reads, mutate_snp_codon, hgv_hilbertvis, hgv_fundo, categorize_elements_satisfying_criteria, microsatellite_birthdeath, multispecies_orthologous_microsats, fasta_nucleotide_changer, fasta_clipping_histogram, fasta_formatter, fastq_quality_boxplot, fastq_quality_converter fastq_quality_filter fastq_to_fasta, and the fastx_toolkit all from [devteam](http://http://bit.ly/15wwkeo).

