---
title: January 2018 Galaxy News
---
[<img class="float-right" src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 />](/src/galaxy-updates/index.md)

Welcome to the January 2018 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# Events

There are [a plenitude of Galaxy related events coming up](/src/events/index.md) in the next few months:

## GCCBOSC 2018 Training Nominations Extended to January 12

**[Nominate a training topic now](http://bit.ly/gccbosc2018-tr-nom)**.

[<img class="float-left" src="/src/images/logos/gcc-bosc-2018-logo-300.png" width="160" alt="GCCBOSC 2018: Nominate Training Topics" />](http://bit.ly/gccbosc2018-tr-nom)

The joint [2018 Galaxy Community and Bioinformatics Open Source Conferences](https://gccbosc2018.sched.com/) start with training and **the training topics are determined by you**.  **Topic nominations have been extended January 12**. Any topics of interest to these communities can be nominated. 

[GCCBOSC2018](https://gccbosc2018.sched.com/) will be held 25-30 June in Portland, Oregon, United States. It will feature two days of training: the second of which is multi-track and will feature content for both the BOSC and Galaxy communities.  

Nominated topics can cover a wide range. For example:

- How to use open source software using just a web browser (e.g., Intro to Using Galaxy) 
- How to use software packages (The Newbie's Guide to BioRuby)
- Advanced applications of software (Genome Assembly with Galaxy, or Proteomics with BioPython)
- Software installation and configuration (Tool Wrapping for Galaxy, or Using CloudLaunch)
- Open source community (Building a Curation Community using Apollo, or Training using the Galaxy Training Network)

*These are only examples.* If you are looking for ideas, see topics nominated for [GCC2017](https://docs.google.com/document/d/1jyCndVqfXx-HsyFJ4hjaK075vDbPM1kE58qO2lJTOG8/edit?usp=sharing), [GCC2016](http://bit.ly/gcc2016noms), [GCC2015](http://bit.ly/gcc2015vote), and the [Galaxy Events page](/src/events/index.md), which lists what training the community is offering outside of GCCs.  Training is not limited to Galaxy-related topics.  *Don't be afraid to think inside the BOSC!*

Nominated topics have been, and will continue to be [published](/src/events/gccbosc2018/training/index.md) as they come in. Topics will be compiled into a uniform list by the GCCBOSC2018 organizers, and then voted on by the community later this month.

## Galaxy (and GMOD) at Plant and Animal Genome XXVI

[<img class="float-right" src="/src/events/2018-pag/galaxy-at-pag-2018.png" alt="Galaxy at PAG XXVI" width="180" />](/src/events/2018-pag/index.md)

The [26th Plant and Animal Genome Conference](http://intlpag.org/) will be held January 13-17, in San Diego, California.  PAG is the largest ag-genomics meeting in the world and [Galaxy will be there](/src/events/2018-pag/index.md) to help researchers get the most from their data.  There will be (at least):

* a [Galaxy Workshop with 4 talks](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Session/4796),
* a Galaxy Community Update talk during the [GMOD session](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Session/4806)
* 10 other Galaxy-related talks, and 
* 12 Galaxy-related posters.
* 4 pre-conference [GMOD](#gmod-pag) and [Galaxy](#intro-to-galaxy-workshop-at-csusm) events

See the [Galaxy @ PAG page](/src/events/2018-pag/index.md) for details.

### GMOD @ PAG

<img class="float-right" src="/src/images/logos/gmod-in-san-diego-300.png" alt="GMOD at PAG XXVI" width="160" />

[GMOD](https://gmod.org/) (of which Galaxy is a part) has a bunch of things going on at PAG that are of interest to the Galaxy Community:

* GMOD will have a [project-wide hackathon on January 11-12](http://gmod.org/wiki/2018_PAG_Hackathon). 
* [Meg Staton](http://gst.tennessee.edu/people/meg-staton/) will [speak about](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Paper/28295) the Tripal-Galaxy bridge 
* There will be a [GMOD workshop](https://pag.confex.com/pag/xxvi/meetingapp.cgi/Session/4806) on the last day of PAG.
* The [Tripal Project](https://tripal.info) will have a user meeting on January 11, and a hackathon on January 12. [Register here](https://docs.google.com/document/d/1klQwUqaT_KxPKIfGYy_3rtGnOMVhJxEoD9dZ4PGc4yg/edit).

If you are interested in supporting better integration of Galaxy with other GMOD components then please give the two hackathons a look.

### Intro to Galaxy Workshop at CSUSM

[<img class="float-right" src="/src/images/logos/csusm-logo.gif" alt="Introduction to Galaxy Workshop, CSU San Marcos, January 12" width="160" />](/src/events/2018-01-csusm-workshop/index.md)

If you are at [Cal State San Marcos](https://csusm.edu/), or anywhere nearby, there will be an [Introduction to Galaxy Workshop](/src/events/2018-01-csusm-workshop/index.md) in San Marcos, California on Friday January 12.  The workshop is free, but space is limited, so you are encouraged to [register now](http://bit.ly/gxy-csusm-2018-reg).

## ELIXIR Galaxy Community Kickoff and Meeting

<div class="float-left">
[<img src="/src/images/logos/ElixirNoTextLogo.png" alt="ELIXIR Galaxy Community Kickoff and Meeting" width="160" />](https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference) <br />
[<img src="/src/images/logos/embl-australia-logo.png" alt="EMBL-Australia" width="120" />](http://www.emblaustralia.org/)
</div>

The ELIXIR Galaxy community will have a [kickoff meeting](https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference) 14-16th March 2018, in Freiburg/Germany. We will combine this with a Galaxy User Conference and the **official launch of the usegalaxy.eu server.**

This event will also be coordinated with the [EMBL Australia](http://www.emblaustralia.org/) to also **launch usegalaxy.org.au at the same time.**

Wednesday (the 14th) will be dedicated to the ELIXIR Galaxy communtiy, to discuss and plan our roadmap in 2018 and 2019. Thursday and Friday (15th, 16th) will be a conference where Galaxy users talk about their research and use-cases. However, a few spots we will reserved for official talks from ELIXIR.

*[Register now](https://docs.google.com/forms/d/e/1FAIpQLSfqKqyPwW-s6m64ObleUSv4B8RbGxCRIddBY1cpt3Za7af6XQ/viewform) as space is limited.*


## 2018 Big Genomics Data Skills Training course

[<img class="float-right" src="/src/images/logos/jax-logo-no-text-200.png" alt="2018 Big Genomics Data Skills Training course" width="160" />](https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors)

The [Jackson Lab (JAX)](https://www.jax.org/) is pleased to open the application period for the 2018 Big Genomics Data Skills Training course, funded by the NIH BD2K initiative: [Big Genomic Skills Training info and online application](https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors). The program provides training and hands-on experience working with genomic scale data and will enable participants to integrate genomic data analysis into their courses or launch new courses. Topics covered include RNAseq, Whole Exome Sequencing and variant detection, ChIPseq and network analysis. Participants will gain experience using Galaxy, R and Python.
 
As a grant funded activity housing is included; however you will need to cover your own travel.  If you teach at an HBCU or Minority-serving institution (by Dept. of Education criteria) JAX may be able to provide a modest travel award.
 
*The course is for faculty who primarily teach undergraduate students* and will be held at JAX Genomic Medicine in Farmington CT, May 21-25.

# All Upcoming events

<table class="table table-striped">
<thead>
<tr>
  <th>Date</th>
  <th>Topic/Event</th>
  <th>Venue/Location</th>
  <th>Contact</th>
</tr>
</thead>
<tbody>
<tr>
  <td><span class="text-nowrap">January 8th 2018</span></td>
  <td><a href="https://www.elixir-europe.org/events/european-galaxy-administrator-workshop">European Galaxy Administrator Workshop</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Oslo, Norway</td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Instructors</td>
</tr>
<tr>
  <td><span class="text-nowrap">January 12th 2018</span></td>
  <td><a href="/events/2018-01-csusm-workshop">Introduction to Galaxy Workshop</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://csusm.edu/">Cal State University San Marcos, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Arun Sethuraman, Dave Clements</td>
</tr>
<tr>
  <td><span class="text-nowrap">January 13th 2018</span></td>
  <td><a href="/events/2018-pag">Plant &amp; Animal Genome XXVI (PAG 2018)</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="http://www.intlpag.org/">San Diego, California</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>See web page</td>
</tr>
<tr>
  <td><span class="text-nowrap">January 18th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/bioconda-workshop/">Packaging your bioinformatics tool with Bioconda and Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Saskia Hiltemann, Helena Rasche</td>
</tr>
<tr>
  <td><span class="text-nowrap">January 22nd 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/developing-bioinformatics-training-materials/">Developing bioinformatics training materials</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Saskia Hiltemann, Helena Rasche</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 1st 2018</span></td>
  <td><a href="/events/2018-02-lorne-proteomics">Galaxy-P: an accessible resource for multi-omics analysis</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="http://www.australasianproteomics.org/lorne-proteomics-symposium-2018/">Lorne Proteomics Symposium, Lorne, Victoria, Australia</a></td>
  <td>Maria Doyle</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 12th 2018</span></td>
  <td><a href="http://www.triconference.com/Converged-IT-and-the-Cloud/#Day1">Supporting Highly Scalable Scientific Data Analysis with Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="http://www.triconference.com/">Converged IT &amp; the Cloud, San Francisco, California, United States</a></td>
  <td>James Taylor</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/RNA_Sequencing_2018.pdf">RNA Sequencing and Differential Expression</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 26th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/460-galaxy5">5th Galaxy high-throughput sequencing (HTS) data analysis workshop</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>University of Freiburg, Freiburg Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Freiburg Galaxy Team</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 7th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/469-introduction-to-rna-seq-data-analysis-with-galaxy">Introduction to RNA-Seq data analysis with Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Am Botanischen Garten, Kiel, Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Andrea Bagnacani, Markus Wolfien</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 14th 2018</span></td>
  <td><a href="https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference">Galaxy community kickoff meeting and Galaxy User Conference</a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.openstreetmap.org/?mlat=48.00189&amp;mlon=7.84628#map=17/48.00189/7.84628">Freiburg, Germany</a></td>
  <td>ELIXIR Galaxy Working Group</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/rna-seq-dge-analysis-galaxy-gvl/#more-1720">RNA-Seq Differential Gene Expression Analysis using Galaxy and the GVL</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/intro-galaxy-gvl/">Introduction to Galaxy and the GVL</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/Alignment_2018.pdf">Alignment, Visualisation and Variant Calling</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/variant-calling-galaxy-gvl/#more-1012">Variant Calling with Galaxy &amp; the GVL for beginners</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors">Big Genomics Data Skills Training course</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.jax.org/about-us/locations/farmington">JAX Genomic Medicine, Farmington, Connecticut, United States</a></td>
  <td>Charlie Wray</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 25th 2018</span></td>
  <td><a href="https://gccbosc2018.sched.com">GCC2018 &amp; BOSC 2018</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="/events/gccbosc2018/venue/">Reed College, Portland, Oregon, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
</tbody>
</table>

See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

----

# New Galactic Blog Entries: Galaksio

<div class="float-right" >
[<img src="/src/images/galaxy-logos/galactic-blog-400.png" alt="Galactic Blog" width="200" />](/src/blog/index.md) <br /><br />
[<img src="/src/images/logos/galaksio_logo.png" alt="Galaksio in the Galactic Blog" width="200" />](/src/blog/index.md)
</div>

We've got two new *[Galactic Blog](/src/blog/index.md)* entries, both related to [Galaksio](https://galaksio.readthedocs.io/en/latest/):

* [Galaksio: a Galaxy user interface focused on running prepared workflows](/src/blog/2017-12-galaksio/index.md), by Tomas Klingström
* [Bioinformatic training with the B3Africa project](https://usegalaxy-eu.github.io/galaxy-freiburg/2017/12/10/b3africa.html), by Tomas Klingström

The *Galactic Blog* is a place for longer-form posts that are relevant to Galaxy. (See [Galaxy News](/src/news/index.md) for shorter-form items.) 

# Publications

**155 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in December (and over 1500 publications were added in 2017).


## Highlighted Publications

The *Galactic* and *Stellar* publications added in December were:

* [TOSCA-based orchestration of complex clusters at the IaaS level](http://iopscience.iop.org/article/10.1088/1742-6596/898/8/082036), M Caballer, G Donvito, G Moltó, R Rocha and M Velten. *Journal of Physics: Conference Series*, Volume 898, Track 6: Infrastructures

* [Molecular property diagnostic suite (MPDS): Development of disease-specific open source web portals for drug discovery](http://www.tandfonline.com/doi/abs/10.1080/1062936X.2017.1402819?journalCode=gsar20), S. Nagamani, A. S. Gaur, K. Tanneeru, G. Muneeswaran, S. S. Madugula, MPDS Consortium, D. Druzhilovskiy, V. V. Poroikov & G. N. Sastry. *SAR and QSAR in Environmental Research*, Special Issue: 9th International Symposium on Computational Methods in Toxicology and Pharmacology Integrating Internet Resources (CMTPI-2017) - Part 2. Guest Editors: A.K. Saxena and M. Saxena

* [The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkx1152/4653531), Laurel Cooper, Austin Meier, Marie-Angélique Laporte, Justin L. Elser, Chris Mungall,
Brandon T. Sinn, Dario Cavaliere, Seth Carbon, Nathan A. Dunn, Barry Smith, Botong Qu, Justin Preece, Eugene Zhang, Sinisa Todorovic, Georgios Gkoutos, John H. Doonan, Dennis W. Stevenson, Elizabeth Arnaud and Pankaj Jaiswal. *Nucleic Acids Research*, 2017 1 doi: 10.1093/nar/gkx1152

* [FROGS: Find, Rapidly, OTUs with Galaxy Solution](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btx791/4708232?redirectedFrom=fulltext), Frédéric Escudié, Lucas Auer, Maria Bernard, Mahendra Mariadassou, Laurent Cauquil, Katia Vidal, Sarah Maman, Guillermina Hernandez-Raquet, Sylvie Combes, Géraldine Pascal. *Bioinformatics*, btx791, https://doi.org/10.1093/bioinformatics/btx791

* [Transcriptomic Data Analysis: RNA-Seq Analysis Using Galaxy](https://doi.org/10.1142/9789813229150_0005), Chia-Lang Hsu and Chantal Hoi Yin Cheung (2018), in A Practical Guide to Cancer Systems Biology: pp. 49-62.
https://doi.org/10.1142/9789813229150_0005

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 96 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Methods) | | 32 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UsePublic) | | 18 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Workbench) | | 14 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseLocal) | 
| 13 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseMain) | | 11 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+RefPublic) | | 7 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Tools) | | 6 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Reproducibility) | 
| 5 | [+Unknown](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Unknown) | | 4 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Cloud) | | 3 | [+Other](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Other) | | 2 | [+Visualization](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Visualization) | 
| 2 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Shared) | | 2 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+IsGalaxy) | | 2 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+HowTo) | | | | |


----
# Who's Hiring

<div class='right'><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Responsable Opérationnel de la Plateforme de Bioinformatique ARTbio](https://www.sfbi.fr/content/responsable-op%C3%A9rationnel-de-la-plateforme-de-bioinformatique-artbio), ARTbio, Institut de Biologie Paris Seine, France
* [Classification des micropolluants dans l'environnement : amélioration de l'algorithme online TyPol](https://www.sfbi.fr/content/classification-des-micropolluants-dans-lenvironnement-am%C3%A9lioration-de-lalgorithme-online-typ), INRA-LBE, Narbonne, France
* The [Blankenberg](/src/people/dan/index.md) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/src/news/2017-09-hiring-at-cc/index.md).
* [Galaxy Project is hiring software engineers and postdocs at Johns Hopkins](/src/news/2017-08-hiring-at-hopkins/index.md), Baltimore, Maryland, United States

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them last month.

## EuPathDB Galaxy Data Analysis Service

[<img class="float-right" src="/src/use/eu-path-db/eupathdb-logo.png" alt="EuPathDB Galaxy Data Analysis Service" width="240" />](http://eupathdb.org/eupathdb/app/galaxy-orientation)

The [EuPathDB Galaxy Data Analysis Service](http://eupathdb.org/eupathdb/app/galaxy-orientation) is a free Galaxy server for large-scale data analysis focused on eukaryotic pathogens.

* All [EuPathDB](http://eupathdb.org/) genomes are pre-loaded.
* [Pre-configured workflows](https://eupathdb.globusgenomics.org/workflow/list_published) are available.
* Powered by [Globus Genomics](https://www.globus.org/genomics)
* [EuPathDB tagged publications](https://www.zotero.org/groups/1732893/galaxy/items/tag/%3EEuPathDB) in [Galaxy Publication library](/src/publication-library/index.md).

User support is available via a [web form](http://eupathdb.org/eupathdb/contact.do). The service is supported by [EuPathDB](http://eupathdb.org/), University of Georgia, University of Pennsylvania, and University of Liverpool.

## Langille Lab PICRUSt server

[<img class="float-right" src="/src/use/langille-lab/picrust-workflow-450.jpg" width="280" alt="Langille Lab PICRUSt server" />](http://galaxy.morganlangille.com/)

The [Langille Lab PICRUSt server](http://galaxy.morganlangille.com/) hosts PICRUSt, a bioinformatics software package designed to predict metagenome functional content from marker gene (e.g., 16S rRNA) surveys and full genomes. [PICRUSt Documentation](http://picrust.github.io/picrust/index.html) and the [PICRUSt users list](https://groups.google.com/group/picrust-users/subscribe?note=1&hl=en&noredirect=true&pli=1) are available.

Anonymous access and account creation are both supported. This server is supported by the [Langille Lab](http://morganlangille.com/index.html) at [Dalhousie University](https://www.dal.ca/).

See:

* [Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences](https://www.nature.com/articles/nbt.2676), Morgan G I Langille et al. *Nature Biotechnology* 31, 814–821 (2013)
doi:10.1038/nbt.2676
* [Langille tagged publications](https://www.zotero.org/groups/1732893/galaxy/items/tag/%3ELangille) in [Galaxy Publication library](/src/publication-library/index.md).


## Public Servers in Publications

We tag papers that use, mention, implement or extend public Galaxy Servers.  Here are the counts for December's publications.

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 17 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Huttenhower) | | 8 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Galaxy-P) | | 4 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RepeatExplorer) | | 4 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Workflow4Metabolomics) | 
| 3 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Cistrome) | | 1 | [>Orione](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Orione) | | 1 | [>Pasteur](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Pasteur) | | 1 | [>EuPathDB](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>EuPathDB) | 
| 1 | [>APOSTL](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>APOSTL) | | 1 | [>OSDD MPDS](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>OSDD MPDS) | | 1 | [>RiboGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RiboGalaxy) | | 1 | [>RNA-Rocket](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RNA-Rocket) | 
| 1 | [>Nebula](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Nebula) | | 1 | [>PIA](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PIA) | | 1 | [>CBIB](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>CBIB) | | 1 | [>Langille](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Langille) | 

# Commercial Support: Intero Life Sciences and Galaxy Enterprise

[<img class="float-right" src="/images/logos/galaxy-enterprise-logo-200.png" alt="Galaxy Enterprise from Intero Life Sciences" />](https://www.galaxyinformatics.com/)

We are pleased to announce a new option for [commercial Galaxy support](/src/support/commercial/index.md). [Intero Life Sciences](https://www.interolifesciences.com/) offers *[Galaxy Enterprise](https://www.galaxyinformatics.com/)*,  a set of support services for Galaxy, including:

* Subscription/Contract Based Enterprise-Level Support
* Turn-key Customized Workflows
* Professional Services & Consulting
* Production Environment Optimization
* Data Integration Outsourcing Services
* Enterprise Cloud Environment with SLA
  * Fully managed and hosted on cloud Galaxy Enterprise with various levels of SLAs.
  * Dedicated servers with dedicated secure access (VPN) including data storage and management.
* Galaxy Enterprise Certification Training

----

# Tools

## ToolShed Contributions

<div class='right'>[<img src="/src/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in December](/src/toolshed/contributions/2017-12/index.md).

# Releases

## ephemeris 0.8.0

Ephemeris is a small Python library and set of scripts for managing the bootstrapping of Galaxy plugins - tools, index data, and workflows.

* Free software: Academic Free License version 3.0
* Documentation: https://ephemeris.readthedocs.org.
* Code: https://github.com/galaxyproject/ephemeris

## blend4php 0.1 beta

[<img class='float-right' src="/src/images/galaxy-logos/blend4phpLogo.png" alt="blend4php" width="220" />](https://github.com/galaxyproject/blend4php)

The beta version of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html), was released in December. It provides a PHP package for interacting with Galaxy and CloudMan. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/) page for full information.

## Earlier Releases

### 17.09 Galaxy Release

<div class="right">
<a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a><br /><br />
</div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.09 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html)** at the end of October.

Highlights include

* **Singularity** Tool execution using the HPC-friendly container technology Singularity is now supported. 
* **Download entire collection** Downloading whole collections is now possible from the history interface. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Switch tool versions in workflows** You can now select exactly what version of tool you want to use when building workflows. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Security patches**
* Several features were **deprecated**

See the [17.09 release announcement](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html) for details.

### Galaxy Docker Image 17.09

The [Galaxy Docker project has seen a matching release](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.09), for Galaxy 17.05. Major features include
* much improved documentation about using Galaxy Docker and an external cluster ([@rhpvorderman](https://github.com/rhpvorderman))
* CVMFS support - mounting in 4TB of pre-build reference data ([@chambm](https://github.com/chambm))
* Singularity support and tests (compose only)
* more work on K8s support and testing ([@jmchilton](https://github.com/jmchilton))
* using .env files to configure the compose setup for SLURM, Condor, K8s, SLURM-Singularity, Condor-Docker

And

* The [Galaxy Docker Project](https://github.com/bgruening/docker-galaxy-stable) has reached more than 31k downloads on [Dockerhub](https://hub.docker.com/r/bgruening/galaxy-stable/) - not counting quay.io and all flavors 

### galaxy-lib 17.9.10

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

This revision:

* Added docs for using mulled-build with your own quay.io account (thanks to @jerowe).
* Catch errors in Conda search if nothing is found (preventing planemo-monitor from functioning properly) (thanks to @bgruening).
* Make multi-requirement container building via mulled more stable (thanks to @bgruening).

### Planemo 0.47.0

[<img class="float-right" src="/src/images/galaxy-logos/planemo-logo.png" alt="Planemo" width="200" />](https://pypi.python.org/pypi/planemo/)

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).


Other packages that have been released in the prior 4 months.

### sequence_utils 1.1.2

Galaxy's [sequence utilities](https://github.com/galaxyproject/sequence_utils) are a set of Python modules for reading, analyzing, and converting sequence formats.  See the [release notes](https://github.com/galaxyproject/sequence_utils/blob/master/HISTORY.rst) for what's new this month.

### StarForge 0.3.1-5

[StarForge](https://github.com/galaxyproject/starforge) help build Galaxy things in virtualization:

* Build Galaxy Tool Shed dependencies
* Build Python Wheels (e.g. for the Galaxy Wheels Server)
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.  StarForge has had several updates this fall.  Fixes and new features include:

* Support xz/lzma tarballs for wheel builds Pull Request 166
* Native support for auditwheel and delocate. (#160)
* Do not build sdists with the wheel subcommand by default. (#155)
* Fix a bug where the wrong working directory was set when building wheels with multiple sources. (#154)
* Fix a bug with sudo and brew install on macOS. (#151).
* Short circuit platform caching on OS X (#150).

### BioBlend 0.10.0

[BioBlend](http://bioblend.readthedocs.org/) is a Python library for interacting with CloudMan and Galaxy‘s API.  BioBlend makes it possible to script and automate the process of cloud infrastructure provisioning and scaling via CloudMan, and running of analyses via Galaxy.

See the [release notes](https://github.com/galaxyproject/bioblend/releases) for what's new in release 0.10.0.


### Pulsar 0.8.0

[<img class="float-right" src="/src/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" />](https://pypi.python.org/pypi/pulsar-app/)

A [Pulsar](https://pypi.python.org/pypi/pulsar-app/) update was released in September.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

[This release](https://github.com/galaxyproject/pulsar/blob/master/HISTORY.rst) adds several features and 4 bug fixes.


## And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

# Other News

* Would you like to learn some Python or JavaScript and improve Galaxy at the same time? Check out our collection of very friendly issues tagged [help wanted](https://github.com/galaxyproject/galaxy/issues?q=label%3A%22help+wanted%22+is%3Aissue+is%3Aopen). Please feel free to ask, comment, and request guidance. 
* From [Björn Grüning](/src/people/bjoern-gruening/index.md)
  *  `shed-tools update` will now update all Galaxy tools in your Galaxy instance. What a wonderful [contribution](https://github.com/galaxyproject/ephemeris/pull/64) from [Ruben Vorderman](https://github.com/rhpvorderman)!
  * Our community maintained [CVFMS](http://cernvm.cern.ch/) based reference data storage has now a European mirror. Thanks to [JRC / EU Science Hub](https://ec.europa.eu/jrc/en), the [European Commission](http://ec.europa.eu/), [Nate Coraor](/src/people/nate/index.md) and [Guy Van den Eede](https://twitter.com/guyvandeneede). This is now also available in the [Galaxy Docker project](https://github.com/bgruening/docker-galaxy-stable)! Enjoy!
* From [John Chilton](/src/people/john-chilton/index.md)
  * WIP: [API to build hierarchical data in Galaxy](https://github.com/galaxyproject/galaxy/pull/5220) (libraries & collections) from zip files, YAML descriptions, BagIt archives, etc...   
  * [Allow Galaxy tools to detect out of memory errors and job runners to resubmit to other resources](https://github.com/galaxyproject/galaxy/pull/5196) based on this and other tool detected errors. 
  * Galaxy [no longer depends on having samtools on its PATH](https://github.com/galaxyproject/galaxy/pull/5037). Another huge deployability step forward from [Marius van den Beek](https://github.com/mvdbeek)!

