---
---
* [Wiki Root](/toolshed/)
* [All monthly summaries](/toolshed/contributions/)

# Tools

* *From [tiagoantao](https://toolshed.g2.bx.psu.edu/view/tiagoantao):*
    * [packages_stacks_1_35](https://toolshed.g2.bx.psu.edu/view/tiagoantao/packages_stacks_1_35):
    * [clone_filter](https://toolshed.g2.bx.psu.edu/view/tiagoantao/clone_filter): Stacks clone_filter (not available on the standard tool)

* *From [bornea](https://toolshed.g2.bx.psu.edu/view/bornea):*
    * [dotplot_runner](https://toolshed.g2.bx.psu.edu/view/bornea/dotplot_runner): This tool runs the dotplot program found at http://prohitstools.mshri.on.ca/ Runs the biclustering version of the ProHits DotPlot tool.

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
    * [gemini_region](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_region): Wrapper for the gemini tool suite: GEMINI region.
    * [gemini_de_novo](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_de_novo): Wrapper for the gemini tool suite: GEMINI de_novo
    * [gemini_amend](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_amend): Wrapper for the gemini tool suite: GEMINI amend
    * [gemini_qc](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_qc): Wrapper for the gemini tool suite: GEMINI qc
    * [gemini_fusions](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_fusions): Wrapper for the gemini tool suite: GEMINI fusions
    * [gemini_gene_wise](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_gene_wise): Wrapper for the gemini tool suite: GEMINI gene_wise
    * [gemini_mendel_errors](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_mendel_errors): Wrapper for the gemini tool suite: GEMINI mendel_errors
    * [gemini_dump](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_dump): Wrapper for the gemini tool suite: GEMINI dump
    * [suite_gemini](https://toolshed.g2.bx.psu.edu/view/iuc/suite_gemini): GEMINI: a flexible framework for exploring genome variation The intent of GEMINI (GEnome MINIing) is to provide a simple, flexible, and powerful framework for exploring genetic variation for personal and medical genetics. GEMINI is unique in that it integrates genetic variation (from VCF files) with a wealth of genome annotations into a unified database framework. Using this integrated database as the analysis framework, we aim to leverage the expressive power of SQL for data analysis, while attempting to overcome the fundamental challenges associated with using databases for very large (e.g. 1,000,000 variants times 1,000 samples yields one billion genotypes) datasets. In addition, by defining sample relationships with a PED file, GEMINI allows one to explore and test for variants that meet specific inheritance models (e.g., recessive, dominant, etc.).
    * [gemini_load](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_load): Wrapper for the gemini tool suite: GEMINI load
    * [gemini_query](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_query): Wrapper for the gemini tool suite: GEMINI query
    * [gemini_annotate](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_annotate): Wrapper for the gemini tool suite: GEMINI annotate
    * [gemini_roh](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_roh): Wrapper for the gemini tool suite: GEMINI roh
    * [gemini_lof_sieve](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_lof_sieve): Wrapper for the gemini tool suite: GEMINI lof_sieve
    * [gemini_db_info](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_db_info): Wrapper for the gemini tool suite: GEMINI db_info
    * [gemini_stats](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_stats): Wrapper for the gemini tool suite: GEMINI stats
    * [gemini_comp_hets](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_comp_hets): Wrapper for the gemini tool suite: GEMINI comp_hets
    * [gemini_actionable_mutations](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_actionable_mutations): Wrapper for the gemini tool suite: GEMINI actionable_mutations
    * [gemini_set_somatic](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_set_somatic): Wrapper for the gemini tool suite: GEMINI set_somatic
    * [gemini_pathways](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_pathways): Wrapper for the gemini tool suite: GEMINI pathways
    * [gemini_windower](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_windower): Wrapper for the gemini tool suite: GEMINI windower
    * [gemini_recessive_and_dominant](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_recessive_and_dominant): Wrapper for the gemini tool suite: GEMINI autosomal recessive/dominant
    * [gemini_interactions](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_interactions):  Wrapper for the gemini tool suite: GEMINI interactions
    * [gemini_burden](https://toolshed.g2.bx.psu.edu/view/iuc/gemini_burden): Wrapper for the gemini tool suite: GEMINI burden

* *From [lecorguille](https://toolshed.g2.bx.psu.edu/view/lecorguille):*
    * [xcms_summary](https://toolshed.g2.bx.psu.edu/view/lecorguille/xcms_summary): [W4M][LC-MS] XCMS R Package - Preprocessing - HTML Summary for XCMS and CAMERA analysis Part of the W4M project: http://workflow4metabolomics.org

XCMS: http://www.bioconductor.org/packages/release/bioc/html/xcms.html

This tool create a HTML summary of XCMS and CAMERA analysis

* *From [mkh](https://toolshed.g2.bx.psu.edu/view/mkh):*
    * [ips5](https://toolshed.g2.bx.psu.edu/view/mkh/ips5): InterProScan 5

* *From [fgiacomoni](https://toolshed.g2.bx.psu.edu/view/fgiacomoni):*
    * [hmdb_ms_search](https://toolshed.g2.bx.psu.edu/view/fgiacomoni/hmdb_ms_search): [W4M][LC-MS] HMDB database MS Search Package - Annotation - Returns annotation results (adducts and metabolites) from The Human Metabolome Database. Part of the W4M project: http://workflow4metabolomics.org / HMDB: http://www.hmdb.ca/. The wrapper script use the HMDB 'MS search' resource to annotate a list of m/z. The process returns outputs files (CSV and HTML formats) with links through metabocards.

* *From [mvdbeek](https://toolshed.g2.bx.psu.edu/view/mvdbeek):*
    * [r_goseq_1_22_0](https://toolshed.g2.bx.psu.edu/view/mvdbeek/r_goseq_1_22_0): goseq does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data Gene Ontology analyser. Does selection-unbiased testing for category enrichment amongst differentially expressed (DE) genes for RNA-seq data.

* [printenv](https://toolshed.g2.bx.psu.edu/view/mvdbeek/printenv):  print the galaxy environemnt executes printenv and dumps output. useful to check passing of environemntal variables to galaxy.

* [get_reference_fasta](https://toolshed.g2.bx.psu.edu/view/mvdbeek/get_reference_fasta): Obtain reference genome sequence. Places the reference genome sequence in the current history. Useful for sharing purposes or tools that work directly on fasta files.

* *From [tgac](https://toolshed.g2.bx.psu.edu/view/tgac):*
    * [miranda](https://toolshed.g2.bx.psu.edu/view/tgac/miranda):  Finds potential target sites for miRNAs in genomic sequences miRanda is an algorithm for the detection of potential microRNA target sites in genomic sequences.

* *From [computationaltranscriptomics](https://toolshed.g2.bx.psu.edu/view/computationaltranscriptomics):*
    * [vgx_converter](https://toolshed.g2.bx.psu.edu/view/computationaltranscriptomics/vgx_converter): Converter for VisualGraphX This tool converts the simple interaction format (.sif) to the JavaScript Object Notation (.json) that can be visualized using VisualGraphX. Furthermore, a attributes list can be provided to enhance the information of the nodes in the graph.

* *From [sarahinraauzeville](https://toolshed.g2.bx.psu.edu/view/sarahinraauzeville):*
    * [rnaseq_tophat2](https://toolshed.g2.bx.psu.edu/view/sarahinraauzeville/rnaseq_tophat2): NAseq - tophat 2 wrapper Sigenae - Sarah Maman - INRA Auzeville

* *From [galaxyp](https://toolshed.g2.bx.psu.edu/view/galaxyp):*
    * [msgfplus](https://toolshed.g2.bx.psu.edu/view/galaxyp/msgfplus): MSGF+ Identifies peptides in tandem mass spectra using the MSGF+ search engine. http://proteomics.ucsd.edu/software-tools/ms-gf/

#### Dependency Definitions

* *From [hyungrolee](https://toolshed.g2.bx.psu.edu/view/hyungrolee):*
    * [package_mgescan_3_0_0](https://toolshed.g2.bx.psu.edu/view/hyungrolee/package_mgescan_3_0_0): tool dependency definition that downloads and compiles version 3.0.0 of mgescan. MGEScan - software packages to identify long terminal repeats (LTR) and non-LTR retroelements in eukaryotic genomic sequence. MGEScan is now avaiable on a Galaxy workflow system for identifying long terminal repeats (LTR) and non-LTR retroelements in eukaryotic genomic sequences. With a Galaxy scientific workflow system, MGEScan becomes easier to manage input and output data through its rich and flexible web interface. UCSC Table Browser, ENA Browser or local storage is used to obtain input genome sequences including a traditional file upload. HMMER 3.1b1 is applied to gain speed boosts compared to a previous version HMMER 2+. In addition Generic Feature Format Version 3 is used for visualization of genome sequence data via  a web-based genome browser e.g. UCSC Genome Browser or Ensembl Genome Browser.
