---
title: Galaxy Proteomics
autotoc: false
gitter: usegalaxy-eu/Lobby
components: true
---

<Insert name="/bare/eu/usegalaxy/notices" />

<h1>

Welcome to **Galaxy Proteomics**

</h1>

<div class="left" style="width: 50%; margin-right: 0">
<div class="img-sizer" style="max-width: 350px">

[![GalaxyP logo](/images/galaxy-logos/galaxyp_banner.png)](http://galaxyp.org)

</div>

*This European Galaxy branches off from the main [usegalaxy.eu](https://usegalaxy.eu) server and focuses on one specific domain: Proteomics.*

**Galaxy Proteomics** is a multiple-'omics' data-analysis platform with particular emphasis on mass spectrometry-based proteomics. It is hosted by GalaxyEU and maintained by the [GalaxyP team](http://galaxyp.org/people/) from the University of Minnesota, USA.

**GalaxyP Resources:**  
[About Us](http://galaxyp.org/about/)  
[Publications](http://galaxyp.org/publications/)  
[Conference presentations](http://galaxyp.org/conference-presentations/)  
[Workshops](http://galaxyp.org/workshops/)  

</div>
<div class="right" style="width: calc(50% - 12px); max-width: 400px">
  <twitter user="usegalaxyp" :height="420"></twitter>
</div>
<div class="clearfix">
</div>

--------------------------------------------------------------------

# Table of Contents

--------------------------------------------------------------------

# Tutorials

Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take [a guided tour](https://proteomics.usegalaxy.eu/tours/core.galaxy_ui) through Galaxy's user interface.

**Want to learn more about proteomics? Check out the following lesson tutorials from the [Galaxy Training Network](https://galaxyproject.github.io/training-material/topics/proteomics/):**

| Lesson | Hands-on | Input dataset | Workflows | Galaxy tour |
| ------ | -------- | ------------- | --------- | ----------- |
Detection and quantitation of N-termini degradomics via N-TAILS | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/ntails/tutorial.html)   |  | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/ntails/workflows/) | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/ntails/tours/)
Label-free versus Labelled - How to Choose Your Quantitation Method | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/labelfree-vs-labelled/tutorial.html) |  |  | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/labelfree-vs-labelled/tours/)
Mass spectrometry imaging 1: Loading and exploring MSI data | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1560645)| [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/workflows/)
Metaproteomics tutorial | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.839701) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/metaproteomics/workflows/)
Peptide and Protein ID using OpenMS tools | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/protein-id-oms/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/546301) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-oms/workflows/) | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-oms/tours/)
Peptide and Protein ID using SearchGUI and PeptideShaker | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/protein-id-sg-ps/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/546301) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-sg-ps/workflows/) | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-sg-ps/tours/)
Peptide and Protein Quantification via Stable Isotope Labelling (SIL) | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/protein-quant-sil/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/1051552) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-quant-sil/workflows/) | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-quant-sil/tours/)
Protein FASTA Database Handling | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/database-handling/tutorial.html) |  | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/database-handling/workflows/) | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/database-handling/tours/)
Proteogenomics 1: Database Creation | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1489208) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/proteogenomics-dbcreation/workflows/)
Proteogenomics 2: Database Search | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1489208) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/proteogenomics-dbsearch/workflows/)
Proteogenomics 3: Novel peptide analysis | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1489208) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/workflows/)
Secretome Prediction | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/secretome-prediction/tutorial.html) | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/519260) | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/secretome-prediction/workflows/) | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/secretome-prediction/tours/)

--------------------------------------------------------------------

# Gateways

Other specialized Galaxy servers curated by the Galaxy-P team containing tools, workflows, and hands-on/video tutorials.

<div class="tooltable">

| Server | Tutorial(s) | Video |
| ------ | ----------- | ----- |
[Proteogenomics Gateway](https://z.umn.edu/proteogenomicsgateway) | [Proteogenomics 1: Database creation](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html)<br/>[Proteogenomics 2: Database search](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html)<br/>[Proteogenomics 3: Novel peptide analysis](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html) | [![proteogenomics thumbnail](https://img.youtube.com/vi/Xwks5qKApj8/0.jpg)](https://www.youtube.com/watch?v=Xwks5qKApj8)
[Metaproteomics Gateway](https://z.umn.edu/metaproteomicsgateway) | [Metaproteomics tutorial](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html) | [![metaproteomics thumbnail](https://img.youtube.com/vi/Vywt_-D5Kek/0.jpg)](https://www.youtube.com/watch?v=Vywt_-D5Kek)

</div>

--------------------------------------------------------------------

# Research Projects

Featured Galaxy-P tools supported with manuscripts, visualization plugins, docker containers, and hands-on tutorials.

| Tool | Description | Paper | Tutorial | Docker |
| ---- | ----------- | ----- | -------- | ------ |
CRAVAT-P | Stands for Cancer Related Analysis of Variants Toolkit for Proteogenomics. Submits, checks for, and retrieves data for cancer annotation | [Sajulga et. al 2018](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00404) | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.8b00404/suppl_file/pr8b00404_si_001.pdf) | [![docker-icon](https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_docker-128.png)](https://z.umn.edu/gpcravatdocker)
QuanTP | Correlation between protein and transcript | [Kumar et. al 2019](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00727) | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.8b00404/suppl_file/pr8b00404_si_001.pdf) | [![docker-icon](https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_docker-128.png)](https://z.umn.edu/quantpdocker)
MetaQuantome | Quantitative analysis of the function and taxonomy of microbiomes and their interaction |  | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproteomics.github.io/metaquantome_mcp_analysis/galaxy_tutorial/tutorial.html) |

--------------------------------------------------------------------

# News and Events

<!-- TODO: carousel content -->

<iframe title="Recent Galaxy Europe news"
 class="resize-y" src="/bare/eu/latest/news/" scrolling="no"
 style="width: 50%; border: none; vertical-align: top">
</iframe>
<iframe title="Recent Galaxy Europe events"
 class="resize-y" src="/bare/eu/latest/events/" scrolling="no"
 style="width: 50%; border: none; vertical-align: top">
</iframe>

--------------------------------------------------------------------

# Contributors

- [Bjoern Gruening](https://github.com/bgruening)
- [Helena Rasche](https://github.com/hexylena)
- [Ray Sajulga](https://github.com/jraysajulga)
- [Subina Mehta](https://github.com/bebatut)

--------------------------------------------------------------------

<Insert name="/eu/common/data-policy" />

<Insert name="/bare/eu/usegalaxy/jobs" />

<footer>
<Insert name="/eu/site-footer" />
</footer>
