---
title: June 2014 Galaxy Update
---
<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the June 2014 *[Galaxy Update](/galaxy-updates/)*, a monthly summary of what is going on in the Galaxy community. *[Galaxy Updates](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Galaxy-User Being Retired June 6

<div class='center'><a href='https://biostar.usegalaxy.org/'><img src="/images/logos/GalaxyBiostar.png" alt="Galaxy Biostar"  /></a><br />Join the conversation! Learn how [here](/support/biostar/).</div>
<br />

***Note: Galaxy's support forum has moved to [help.galaxyproject.org](https://help.galaxyproject.org/).***
<br />

The *[Galaxy Biostar](https://biostar.usegalaxy.org)* online forum was launched April 23 as a replacement for the Galaxy-User [mailing list](/mailing-lists/).  

During the past 5 weeks, Galaxy Biostar has been wildly successful with over 125 active threads, *more than 5 times the number of active threads on Galaxy-user in the 5 weeks before the switch.*

Galaxy-User has remained open during the transition, but now it's time to retire it.  All new posting to Galaxy-User will be stopped on Friday, June 6, **some 101 months, and 8,100 postings** after it was launched.  All those postings will remain available both in Galaxy Biostar (where they have been imported), and in the [online list archives](https://lists.galaxyproject.org/archives/list/galaxy-user@lists.galaxyproject.org/).

Thanks for making Galaxy Biostar, and Galaxy-User before it, such a great resource.


# Events

## GCC2014: June 30 - July 2, Baltimore

<div class='right'><a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoTall200.png" alt="GCC2014: June 30 - July 2" width="150" /></a></div>

**The [2014 Galaxy Community Conference (GCC2014)](/events/gcc2014/) will start one month from today, June 30, and run through July 2,** at the [Homewood Campus](http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/) of [Johns Hopkins University](http://jhu.edu), in [Baltimore, Maryland](http://visitors.baltimorecity.gov/), United States.

### Registration Closes June 6

[Registration closes the month](/events/gcc2014/register/). *Registering now saves 100% of late registration fees.  When you register you can also reserve [lodging at Charles Commons](/events/gcc2014/logistics/#lodging), a very affordable housing option in the same building as the conference.*

*[Training Day](/events/gcc2014/training-day/)* is an opportunity to learn about all things Galaxy including using Galaxy, deploying and managing Galaxy, extending Galaxy, and Galaxy internals.  There are 5 parallel tracks, each with 3 sessions, with each of those sessions two and half hours long.  *That's 15 sessions and over 37 hours of workshop material.*

### Abstracts and Program

The [program](/events/gcc2014/program/) has been published and all [titles and abstracts for accepted talks](/events/gcc2014/abstracts/talks/), are now online.  Accepted poster abstracts will posted within a week.

The conference is still accepting [late abstract submissions](/events/gcc2014/abstracts/).  These will be considered as cancellations occur and space frees up (*and we have always had a few cancellations*).

### Sponsorships and Exhibitors

<div class='right'>
<a href='http://www.penguincomputing.com/'><img src="/images/logos/PenguinComputing.png" alt="Penguin Computing" width="170" /></a>
</div>

We are happy to have [Penguin Computing](http://www.penguincomputing.com/) as a GCC2014 Silver Sponsor. Penguin, like all [Platinum, Gold, and Silver Sponsors](/events/gcc2014/sponsor-exhibit/) will have a table at GCC2104.

There are still [Silver and Bronze sponsorships](/events/gcc2014/sponsor-exhibit/) available for the [GCC2014](/events/gcc2014/) and Giga sponsorships for the [Hackathon](/events/gcc2014/hackathon/). Please contact the [Organizers](/events/gcc2014/organizers/) if your organization would like to help sponsor these events.

In 2014 we are also adding [non-sponsor exhibit spaces](/events/gcc2014/sponsor-exhibit/#exhibit) in addition to the sponsor exhibits.  This will significantly increase the size of the exhibit floor. Please contact the [Organizers](/events/gcc2014/organizers/) if your organization would like to have an exhibit space at GCC2014.

### Galaxy Hackathon at GCC2014

<div class='left'><a href='/events/gcc2014/hackathon/'><img src="/images/logos/GCC2014HackLogoSquare.png" alt="GCC2014 Hackathon" width="150" /></a></div>

The very first [Galaxy Project Hackathon](/events/gcc2014/hackathon/) will take place at Johns Hopkins immediately preceding [GCC2014](/events/gcc2014/) from starting June 28th.

Do you have a feature you've always wanted to implement?  Just want to hack on Galaxy (or CloudMan!) with other folks?  The [Galaxy Hackathon](/events/gcc2014/hackathon/) will be a great opportunity to meet and work closely with other community and [Galaxy Team](/galaxy-team/) members over the course of three days, culminating in some really great improvements and new features to show off at the [Galaxy Community Conference](/events/gcc2014/) afterward.

Participation in the hackathon itself is completely free, but there's limited space so if you're interested and would like to participate please go ahead and book both your lodging and hackathon seat at [EventBrite](http://bit.ly/gcc2014hackreg).  **[Register now](http://bit.ly/gcc2014hackreg).  As of this writing there are only 11 spots left.**

To help organize ideas and people into more concrete projects, we've also set up a [hackathon-specific Trello board](https://trello.com/b/3YgU637b/gcc2014-hackathon) that we'd love for everyone to go ahead and [start using it](https://trello.com/b/3YgU637b/gcc2014-hackathon).  The board is public and open to commentary and voting, but to create new cards you’ll need to be added as a member so please note the instructions on the board for that.

<div class='right'><a href='http://aws.amazon.com'><img src="/images/logos/AWSLogo1280.jpg" alt="Amazon Web Services" width="150" /></a></div>

Finally, we are happy to have [Amazon Web Services](http://aws.amazon.com) on board as the Cloud Infrastructure sponsor for the GCC2014 Hackathon!  

## Other Events

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div>

There are [at least 13 other Galaxy related events](/events/) in the next two months in Thailand, Canada, France, the United States, Italy, the Netherlands, Australia, and Brazil. Also see the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<br />

<div class='center'>
<a href='http://abims.sb-roscoff.fr/formation'><img src="/images/logos/ABiMSLogo.png" alt="ABiMS" height="50" /></a>
<a href='http://bioinformatics.ca/workshops/2014/informatics-high-throughput-sequencing-data-2014'><img src="/images/logos/CanadianBioinfWorkshopsLog.png" alt="Informatics on High Throughput Sequencing Data Workshop" height=80 /></a>&nbsp;
<a href='/events/asms2014/'><img src="/images/logos/ASMSLogo.png" alt="62nd ASMS Conference on Mass Spectrometry and Allied Topics" height="80" /></a>
<a href='http://training.bioinformatics.ucdavis.edu/2014/02/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-june-16-20-2014/'><img src="/images/logos/UCDavisGenomeCenterLogo.jpg" alt="Using Galaxy for Analysis of High Throughput Sequence Data @ UC Davis Bioinformatics Core" height="80" /></a> &nbsp;
<a href='http://www.bioinformatica.crs4.it/next_generation_sequencing_course_with_galaxy_june_2014'><img src="/images/logos/CRS4Logo.jpg" alt="Analisi dati Next Generation Sequencing in Galaxy" width="130" /></a> &nbsp;
<a href='/events/gcc2014/'><img src="/images/logos/GCC2014LogoWide200.png" alt="GCC2014 registration ends June 6" height="75" /></a>&nbsp;
</div>

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> June 2-3 </th>
    <td> <em><a href='http://www.oaspa-asia-conference.com/program/'>Open (and Big) Data – the next challenge</a></em> </td>
    <td> <a href='http://www.oaspa-asia-conference.com/'>1st Asian Conference on Open Access Scholarly Publishing</a>, Bangkok, Thailand </td>
    <td> Scott Edmunds </td>
  </tr>
  <tr>
    <th> June 9-10 </th>
    <td> <em><a href='http://bioinformatics.ca/workshops/2014/informatics-high-throughput-sequencing-data-2014'>Informatics on High Throughput Sequencing Data Workshop</a></em> </td>
    <td> Toronto, Canada </td>
    <td> <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> June 10 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module17-10juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td rowspan=2> <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2014</a>, <a href='http://migale.jouy.inra.fr/'>INRA Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin AT jouy DOT inra DOT fr">Véronique Martin</a>, <a href='http://genome.jouy.inra.fr/~schbath/'>Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 11 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module8bis-11juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> June 11-12 </th>
    <td> <em><a href='http://abims.sb-roscoff.fr/sites/abims.sb-roscoff.fr/files/files/formation_2014/module_formation_abims-galaxy_rnaseq.pdf'>Galaxy RNAseq de novo/avec référence & cleaning</a></em> </td>
    <td> <a href='http://abims.sb-roscoff.fr/formation'>Formation en Bioinformatique Plateforme ABiMS</a>, <a href='http://www.sb-roscoff.fr/'>Station Biologique de Roscoff</a>, France </td>
    <td> </td>
  </tr>
  <tr>
    <th> June 15-19 </th>
    <td> <strong><a href='/events/asms2014/'>62nd ASMS Conference on Mass Spectrometry and Allied Topics</a></strong><div class='indent'>Including at least 5 workshop/talk/posters</div> </td>
    <td> Baltimore, Maryland, United States </td>
    <td> See <a href='/events/asms2014/'>presenter list</a> </td>
  </tr>
  <tr>
    <th> June 16 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module17-16juin.pdf'>Initiation à l’utilisation de Galaxy</a></em> </td>
    <td rowspan=2> <a href='http://migale.jouy.inra.fr/?q=formations'>Cycle "Bioinformatique par la pratique" 2014</a>, <a href='http://migale.jouy.inra.fr/'>INRA Jouy-en-Josas</a>, France </td>
    <td rowspan=2> <a href="mailto:veronique.martin AT jouy DOT inra DOT fr">Véronique Martin</a>, <a href='http://genome.jouy.inra.fr/~schbath/'>Sophie Schbath</a> </td>
  </tr>
  <tr>
    <th> June 17 </th>
    <td> <em><a href='http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2014/module8bis-17juin.pdf'>Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></em> </td>
  </tr>
  <tr>
    <th> June 16-20 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2014/02/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-june-16-20-2014/'>Using Galaxy for Analysis of High Throughput Sequence Data</a></em><div class='indent'>Workshop is full; can join the waiting list.</div> </td>
    <td> <a href='http://training.bioinformatics.ucdavis.edu/'>UC Davis</a>, California, United States </td>
    <td> <a href="mailto:training DOT ucdbio AT gmail DOT com">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th> June 18-20 </th>
    <td> <em><a href='http://www.bioinformatica.crs4.it/next_generation_sequencing_course_with_galaxy_june_2014'>Analisi dati Next Generation Sequencing in Galaxy</a></em> </td>
    <td> <a href='http://goo.gl/maps/zvWnF'>Presidio Ospedaliero San Giovanni di Dio</a>, Cagliari, Italy </td>
    <td> <a href="mailto:segreteria-ngs@crs4.it">CRS4 Segreteria</a> </td>
  </tr>
  <tr>
    <th> June 23-34 </th>
    <td> <em><a href='http://abims.sb-roscoff.fr/sites/abims.sb-roscoff.fr/files/files/formation_2014/module_formation_galaxy_initiation_phylogenie.pdf'>Galaxy : initiation à la phylogénie</a></em> </td>
    <td> <a href='http://abims.sb-roscoff.fr/formation'>Formation en Bioinformatique Plateforme ABiMS</a>, <a href='http://www.sb-roscoff.fr/'>Station Biologique de Roscoff</a>, France </td>
    <td> <a href="mailto:christophe.caron@sb-roscoff.fr">Christophe Caron</a> </td>
  </tr>
  <tr>
    <th> June 28-30 </th>
    <td> <a href='/events/gcc2014/hackathon/'>Galaxy Hackathon</a> </td>
    <td rowspan=2> <a href='http://webapps.jhu.edu/jhuniverse/information_about_hopkins/campuses/homewood_campus/'>Homewood Campus</a> of <a href='http://jhu.edu'>Johns Hopkins University</a>, <a href='http://visitors.baltimorecity.gov/'>Baltimore, Maryland</a>, United States </td>
    <td> <a href='/events/gcc2014/organizers/'>Organizers</a> </td>
  </tr>
  <tr>
    <th> June 30 -<br />July 2 </th>
    <td> <strong><a href='/events/gcc2014/'>2014 Galaxy Community Conference (GCC2014)</a></strong> </td>
    <td> <a href='/events/gcc2014/organizers/'>Organizers</a> </td>
  </tr>
  <tr>
    <th> July 7-9 </th>
    <td> <em><a href='http://biosb.nl/education/course-portfolio/rna-seq/'>NBIC/BioSB RNA- seq data analysis course</a></em> </td>
    <td> Leiden, the Netherlands </td>
    <td> <a href="mailto:education AT biosb DOT nl">NBIC/BioSB</a> </td>
  </tr>
  <tr>
    <th> July 10 </th>
    <td> <em><a href='http://www.genetics.org.au/annual-meeting-2014/workshops/'>An Introduction to Galaxy with the Genomics Virtual Lab</a></em> </td>
    <td> <a href='http://www.genetics.org.au/annual-meeting-2014/'>Post-GSA 2014 Workshop</a>, Sydney, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> July 28 - August 1 </th>
    <td> <em><a href='/events/sao-paulo2014/'>Workshop: DNA Sequence Bioinformatics Analysis with the Galaxy Platform</a></em> </td>
    <td> <a href='http://usp.br/'>University of São Paulo (USP)</a>, São Paulo, Brazil </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
</table>


# New Papers

*49 papers* were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in May. Some papers that may be particularly interesting to the Galaxy community:

* [Implementation of Cloud based Next Generation Sequencing data analysis in a clinical laboratory](https://doi.org/10.1186/1756-0500-7-314), by Getiria Onsongo, Jesse Erdmann, Michael D Spears, [John Chilton](/people/john-chilton/), Kenneth B Beckman, Adam Hauge, Sophia Yohe, Matthew Schomaker, Matthew Bower, Kevin A T Silverstein and Bharat Thyagarajan, *BMC Research Notes*, Vol. 7, No. 1. (2014), 314, doi:10.1186/1756-0500-7-314

* [deepTools: a flexible platform for exploring deep-sequencing data](https://doi.org/10.1093/nar/gku365), by Fidel Ramírez, Friederike Dündar, Sarah Diehl, Björn A. Grüning, Thomas Manke, *Nucleic Acids Research* (05 May 2014), gku365, doi:10.1093/nar/gku365

* [Galaxy + Hadoop: Toward a Collaborative and Scalable Image Processing Toolbox in Cloud](https://doi.org/10.1007/978-3-319-06859-6_30), by Shiping Chen, Tomasz Bednarz, Piotr Szul, Dadong Wang, Yulia Arzhaeva, Neil Burdett, Alex Khassapov, John Zic, Surya Nepal, Tim Gurevey, John Taylor, In "Service-Oriented Computing – ICSOC 2013 Workshops", Vol. 8377 (2014), pp. 339-351, doi:10.1007/978-3-319-06859-6_30

The new papers were tagged in many different areas:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/cloud'>Cloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/project'>Project</a> </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/tools'>Tools</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/howto'>HowTo</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/refpublic'>RefPublic</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/visualization'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/isgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/reproducibility'>Reproducibility</a> </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/uselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 14 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/workbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 21 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/methods'>Methods</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/shared'>Shared</a> </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://www.citeulike.org/group/16008/tag/usemain'>UseMain</a> </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Two postdoc positions in integrative genomics](/news/post-docs-in-oslo/) available in Oslo, Norway
* [Statistical Genomics Postdoc](/news/statistical-genomics-post-doc-penn-state/) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

# New Public Servers

One new [public Galaxy servers](/use/) was added to the published list in May:

## deepTools

<div class='right solid'><a href='http://deeptools.ie-freiburg.mpg.de/'><img src="/use/archive/deeptools/deeptools.png" alt="deepTools Galaxy Server" width="200" /></a></div>

* *Links:*
    * [deepTools](http://deeptools.ie-freiburg.mpg.de/) server
    * [deepTools home page at GitHub](http://deeptools.github.io/)
    * Fidel Ramírez, Friederike Dündar, Sarah Diehl, Björn A. Grüning, and Thomas Manke. [deepTools: a flexible platform for exploring deep-sequencing data](http://nar.oxfordjournals.org/cgi/content/abstract/gku365) *Nucl. Acids Res.* first published online May 5, 2014 doi:10.1093/nar/gku365
    * *[deepTools: a flexible platform for exploring deep-sequencing data](/events/gcc2014/abstracts/talks/#deeptools-a-flexible-platform-for-exploring-deep-sequencing-data)* presentation by [Sarah Diehl](http://www.ie-freiburg.mpg.de/1892622/employee_page?c=1896591&employee_id=26993) at [GCC2014](/events/gcc2014/)
    * deepTools deployment description
* *Domain/Purpose:*
    deepTools is a suite of user-friendly tools for the visualization, quality control and normalization of data from high-throughput DNA sequencing experiments.
* *Comments:*
    deepTools offers multiple methods for highly-customizable data visualization that immensely aid hypothesis generation and data interpretation. It also offers all the tools needed to create coverage files in standard bedGraph and bigWig file formats allowing various normalization procedures and comparisons between two files (for example, treatment and control).
* *User Support:*
    * [Email](mailto:deeptools@googlegroups.com)
    * [deepTools wiki](https://github.com/fidelram/deepTools/wiki)
* *Quotas:*
    20 GB for unregistered users, 30 GB for registered users
* *Sponsor(s):*
    [Bioinformatics](http://www.ie-freiburg.mpg.de/bioinformaticsfac) and [Deep-Sequencing](http://www.ie-freiburg.mpg.de/deepfac) Units at the [Max Planck Institute for Immunobiology and Epigenetics](http://www3.ie-freiburg.mpg.de/).

<br />

# Galaxy Distributions

The most recent [Galaxy Distribution was released on April 14, 2014](/archive/dev-news-briefs/2014-04-14/).

[BioBlend 0.4.3](https://pypi.python.org/pypi/bioblend/0.4.3) was released on April 11, 2014.  

The most recent version of CloudMan was [released in January 2014](/news/cloudman-release201401/).


# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a></div>
<div class='right'><img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style="text-align: center;"> &nbsp;&nbsp; <a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a> </em> &nbsp;&nbsp;<br /> </td>
  </tr>
</table>

<br /><br />

One new deployment description was added in May:

* deepTools

The *[Community Log Board](/news/)* and  *Deployment Catalog* *[Galaxy community hubs](/community/)* were launched last your.  If you have a Galaxy deployment, or [experience](/news/) you want to share then please publish them.


<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

## Galaxy Project ToolShed Repos

In no particular order:

### Tools

* *From [bgruening](http://toolshed.g2.bx.psu.edu/view/bgruening)*
    * [crispr_recognition_tool](http://toolshed.g2.bx.psu.edu/view/bgruening/crispr_recognition_tool): automatic detection of clustered regularly interspaced palindromic repeats
    * [minced](http://toolshed.g2.bx.psu.edu/view/bgruening/minced): MinCED is a program to find Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) in full genomes or environmental datasets such as metagenomes, in which sequence size can be anywhere from 100 to 800 bp.

* *From [pieterlukasse](http://toolshed.g2.bx.psu.edu/view/pieterlukasse)*
    * [primo_multiomics](http://toolshed.g2.bx.psu.edu/view/pieterlukasse/primo_multiomics): Multi-omics module of Plant Research International's Mass Spectrometry (PRIMS) toolsuite

* *From [rnateam](http://toolshed.g2.bx.psu.edu/view/rnateam/)*
    * [blockclust](http://toolshed.g2.bx.psu.edu/view/rnateam/blockclust): Non-coding RNA clustering from deep sequencing read profiles

* *From [drosofff](http://toolshed.g2.bx.psu.edu/view/drosofff)*
    * [ged_bowtie](http://toolshed.g2.bx.psu.edu/view/drosofff/ged_bowtie): Bowtie wrapper for small RNA sequencing analysis

* *From [george-weingart](http://toolshed.g2.bx.psu.edu/view/george-weingart/) *
    * [metaphlan](http://toolshed.g2.bx.psu.edu/view/george-weingart/metaphlan): MetaPhlAn (Metagenomic Phylogenetic Analysis) is a computational tool for profiling the composition of microbial communities from metagenomic shotgun sequencing data
    * [lefse](http://toolshed.g2.bx.psu.edu/view/george-weingart/lefse): LDA Effect Size (LEfSe) ([Segata et. al 2010](http://genomebiology.com/content/12/6/R60)) is an algorithm for high-dimensional biomarker discovery and explanation that identifies genomic features (genes, pathways, or taxa) characterizing the differences between two or more biological conditions.
    * [micropita](http://toolshed.g2.bx.psu.edu/view/george-weingart/micropita): microPITA is a computational tool enabling sample selection in tiered studies.
    * [maaslin](http://toolshed.g2.bx.psu.edu/view/george-weingart/maaslin): MaAsLin is a multivariate statistical framework that finds associations between clinical metadata and microbial community abundance or function

* *From [devteam](http://toolshed.g2.bx.psu.edu/view/devteam)*
    * [fasta_filter_by_length](http://toolshed.g2.bx.psu.edu/view/devteam/fasta_filter_by_length): Outputs sequences between Minimal length and Maximum length.
    * [rmap](http://toolshed.g2.bx.psu.edu/view/devteam/rmap): RMAP for Solexa Short Reads Alignment
    * [convert_solid_color2nuc](http://toolshed.g2.bx.psu.edu/view/devteam/convert_solid_color2nuc): Convert Color Space to Nucleotides
    * [megablast_xml_parser](http://toolshed.g2.bx.psu.edu/view/devteam/megablast_xml_parser): processes the XML output of any NCBI blast tool
    * [fastqsolexa_to_fasta_qual](http://toolshed.g2.bx.psu.edu/view/devteam/fastqsolexa_to_fasta_qual): extracts sequences and quality scores from FASTQ data (Solexa variant), producing a FASTA dataset and a QUAL dataset.
    * [fasta_concatenate_by_species](http://toolshed.g2.bx.psu.edu/view/devteam/fasta_concatenate_by_species): attempts to parse FASTA headers to determine the species for each sequence in a multiple FASTA alignment. It then linearly concatenates the sequences for each species in the file, creating one sequence per determined species.
    * [blat_mapping](http://toolshed.g2.bx.psu.edu/view/devteam/blat_mapping): takes BLAT pslx output and returns a wig-like file showing the number of reads (coverage) mapped at each chromosome location.
    * [annotation_profiler](http://toolshed.g2.bx.psu.edu/view/devteam/annotation_profiler): Profile Annotations for a set of genomic intervals
    * [megablast_wrapper](http://toolshed.g2.bx.psu.edu/view/devteam/megablast_wrapper): Megablast compare short reads against htgs, nt, and wgs databases
    * [blat_coverage_report](http://toolshed.g2.bx.psu.edu/view/devteam/blat_coverage_report): Calculate the percentage of reads supporting each nucleotide at each location
    * [mapping_to_ucsc](http://toolshed.g2.bx.psu.edu/view/devteam/mapping_to_ucsc): Format mapping data as UCSC custom track
    * [tabular_to_fasta](http://toolshed.g2.bx.psu.edu/view/devteam/tabular_to_fasta): Converts tab delimited data into FASTA formatted sequences.
    * [fasta_compute_length](http://toolshed.g2.bx.psu.edu/view/devteam/fasta_compute_length): counts the length of each fasta sequence in the file. The output file has two columns per line (separated by tab): fasta titles and lengths of the sequences.
    * [short_reads_trim_seq](http://toolshed.g2.bx.psu.edu/view/devteam/short_reads_trim_seq): Select high quality segments
    * [fasta_to_tabular](http://toolshed.g2.bx.psu.edu/view/devteam/fasta_to_tabular): converts FASTA formatted sequences to TAB-delimited format.
    * [rmapq](http://toolshed.g2.bx.psu.edu/view/devteam/rmapq): runs rmapq, searching against a genome build with sequence qualities.
    * [kernel_principal_component_analysis](http://toolshed.g2.bx.psu.edu/view/devteam/kernel_principal_component_analysis): uses functions from 'kernlab' library from R statistical package to perform Kernel Principal Component Analysis (kPCA)
    * [short_reads_figure_high_quality_length](http://toolshed.g2.bx.psu.edu/view/devteam/short_reads_figure_high_quality_length): Histogram of high quality score reads
    * [principal_component_analysis](http://toolshed.g2.bx.psu.edu/view/devteam/principal_component_analysis): performs Principal Component Analysis on the given numeric input data using functions from R statistical package
    * [canonical_correlation_analysis](http://toolshed.g2.bx.psu.edu/view/devteam/canonical_correlation_analysis): uses functions from 'yacca' library from R statistical package to perform Canonical Correlation Analysis (CCA)
    * [kernel_canonical_correlation_analysis](http://toolshed.g2.bx.psu.edu/view/devteam/kernel_canonical_correlation_analysis): uses functions from 'kernlab' library from R statistical package to perform Kernel Canonical Correlation Analysis (kCCA)
    * [short_reads_figure_score](http://toolshed.g2.bx.psu.edu/view/devteam/short_reads_figure_score): Build base quality distribution

* *From [mb2013](http://toolshed.g2.bx.psu.edu/view/mb2013)*
    * [nepenthes_3dpca](http://toolshed.g2.bx.psu.edu/view/mb2013/nepenthes_3dpca): Tools for Principal Component Analysis

* *From [jade](http://toolshed.g2.bx.psu.edu/view/jade/)*
    * [abyss](http://toolshed.g2.bx.psu.edu/view/jade/abyss): abyss de novo assembler

* *From [fernando](http://toolshed.g2.bx.psu.edu/view/fernando/)*
    * [protein_funcional_analysis_similarities](http://toolshed.g2.bx.psu.edu/view/fernando/protein_funcional_analysis_similarities): Provides common characteristics among similar proteins.

* *From [jankanis](http://toolshed.g2.bx.psu.edu/view/jankanis)*
    * [package_python3_4](http://toolshed.g2.bx.psu.edu/view/jankanis/package_python3_4): The Python language version 3.4.1

* *From [plus](http://toolshed.g2.bx.psu.edu/view/plus/)*
    * [archer](http://toolshed.g2.bx.psu.edu/view/plus/archer): Gene Mutation Identification

* *From [qfab](http://toolshed.g2.bx.psu.edu/view/qfab)*
    * [usearch_dereplication](http://toolshed.g2.bx.psu.edu/view/qfab/usearch_dereplication): Removal of duplicate sequences - GVL
    * [usearch_cluster_otus](http://toolshed.g2.bx.psu.edu/view/qfab/usearch_cluster_otus): OTU clustering using the UPARSE-OTU algorithm - GVL
    * [usearch_uchime](http://toolshed.g2.bx.psu.edu/view/qfab/usearch_uchime): Detecting chimeric sequences - GVL
    * [usearch_map_reads_to_otus](http://toolshed.g2.bx.psu.edu/view/qfab/usearch_map_reads_to_otus): Maps read sequences to OTUs - GVL
    * [rdp_multiclassifier](http://toolshed.g2.bx.psu.edu/view/qfab/rdp_multiclassifier): Rapid assignment of rRNA sequences into the new bacterial taxonomy - GVL

* *From [plus91-technologies](http://toolshed.g2.bx.psu.edu/view/plus91-technologies)*
    * [softsearch_tool](http://toolshed.g2.bx.psu.edu/view/plus91-technologies/softsearch_tool): Sensitive Structural Variant detection(SV)

### Datatypes

* *From [qfab](http://toolshed.g2.bx.psu.edu/view/qfab)*
    * [metagenomics_datatypes](http://toolshed.g2.bx.psu.edu/view/qfab/metagenomics_datatypes): Galaxy datatypes required by the Metagenomics Workflow - GVL

### Workflows

* *From [rnateam](http://toolshed.g2.bx.psu.edu/view/rnateam)*
    * [blockclust_workflow](http://toolshed.g2.bx.psu.edu/view/rnateam/blockclust_workflow): a workflow for [BlockClust](http://toolshed.g2.bx.psu.edu/view/rnateam/blockclust).

### Packages

* *From [iuc](http://toolshed.g2.bx.psu.edu/view/iuc)*
    * [package_minced_0_1_5](http://toolshed.g2.bx.psu.edu/view/iuc/package_minced_0_1_5): tool dependency definition that downloads version 0.1.5 of [minced](http://toolshed.g2.bx.psu.edu/view/bgruening/minced), a CRISPR finder.
    * [package_mummer_3_23](http://toolshed.g2.bx.psu.edu/view/iuc/package_mummer_3_23): tool dependency definition for MUMmer, a system for rapidly aligning entire genomes

* *From [rnateam](http://toolshed.g2.bx.psu.edu/view/rnateam)*
    * [package_eden_1_1](http://toolshed.g2.bx.psu.edu/view/rnateam/package_eden_1_1): tool dependency definition that downloads and compiles version 1.1.x of the EDeN package.

* *From [devteam](http://toolshed.g2.bx.psu.edu/view/devteam)*
    * [package_blast_plus_2_2_26](http://toolshed.g2.bx.psu.edu/view/devteam/package_blast_plus_2_2_26): tool dependency definition that downloads and installs version 2.2.26+ of the NCBI BLAST+ package.
    * [package_rmap_2_05](http://toolshed.g2.bx.psu.edu/view/devteam/package_rmap_2_05): Tool dependency definition that downloads and compiles version 2.05 of the rmap package
    * [package_cran_kernlab_0_1_4](http://toolshed.g2.bx.psu.edu/view/devteam/package_cran_kernlab_0_1_4): tool dependency definition that downloads and installs version 0.1-4 of the kernlab R library.
    * [package_fontconfig_2_11_1](http://toolshed.g2.bx.psu.edu/view/devteam/package_fontconfig_2_11_1): tool dependency definition that downloads and compiles version 2.11.1 of the fontconfig package
    * [package_cran_yacca_1_0](http://toolshed.g2.bx.psu.edu/view/devteam/package_cran_yacca_1_0): tool dependency definition that downloads and installs version 1.0 of the yacca R library.

* *From [crs4](http://toolshed.g2.bx.psu.edu/view/crs4/)*
    * [package_pydoop_0_11](http://toolshed.g2.bx.psu.edu/view/crs4/package_pydoop_0_11): tool dependency definition that downloads and compiles version 0.11.1 of the pydoop package

**Tool Updates:**

* *From [peterjc](http://toolshed.g2.bx.psu.edu/view/peterjc)*
    * [mira_assembler](http://toolshed.g2.bx.psu.edu/view/peterjc/mira_assembler): updated to v0.0.10

* *From [stemcellcommons](http://toolshed.g2.bx.psu.edu/view/stemcellcommons)*
    * [macs2](http://toolshed.g2.bx.psu.edu/view/stemcellcommons/macs2): updated to include option to call broad peaks

# Other News

* [The Galaxy Docker Image Project](http://bit.ly/1iETpRg)

## usegalaxy.org offline May 31, 2014

<div class='right'><a href='http://usegalaxy.org'><img src="/images/logos/UseGalaxyOrgLogoShadow200.png" alt="usegalaxy.org will be down 2014/05/31"  /></a></div>

**Access to all [TACC](https://www.tacc.utexas.edu/)-based resources and services, including [usegalaxy.org](http://usegalaxy.org) and the [Galaxy Project Test Server](https://test.galaxyproject.org/), will be unavailable from 9 a.m. to 5 p.m., central US time, on Saturday, May ~~24~~ 31, 2014.** TACC staff will be performing an upgrade to the networking infrastructure. During this time, jobs will continue to run.

During this time you are encouraged to use [any of the 60+ public Galaxy servers](/use/).
