---
autotoc: true
title: The March 2015 Galaxy News
---


<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 30em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/src/GalaxyUpdates/index.md'><img src="/src/Images/Logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the March 2015 *[Galaxy Update](/src/GalaxyUpdates/index.md)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/src/GalaxyUpdates/index.md)* complement the *[Galaxy Development News Briefs](/src/DevNewsBriefs/index.md)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# New Papers

82 new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in February. Some highlights:

* [Multi-omic data analysis using Galaxy](http://dx.doi.org/10.1038/nbt.3134)
    Jorrit Boeke, [John M Chilton](/src/JohnChilton/index.md), Ira R Cooke, Peter L Horvatovich, Pratik D Jagtap, Lukas Käll, Janne Lehtiö, Pieter Lukasse, Perry D Moerland & Timothy J Griffin, *Nature Biotechnology*, Vol. 33, No. 2. (6 February 2015)

* [Online Resources for Genomic Analysis Using High-Throughput Sequencing](http://dx.doi.org/10.1101/pdb.top083667)
    [Daniel Blankenberg](/src/Dan/index.md), [James Taylor](/src/JamesTaylor/index.md), [Anton Nekrutenko](/src/anton/index.md),  *Cold Spring Harbor Protocols* (05 February 2015), doi:10.1101/pdb.top083667

* [Selected Approaches and Frameworks to Carry out Genomic Data Analysis on the Cloud](http://dx.doi.org/10.12694/scpe.v16i1.1058)
    Philip Church, Andrzej Goscinski, *Scalable Computing: Practice and Experience*, Vol. 16, No. 1. (07 February 2015), doi:10.12694/scpe.v16i1.1058

* [Data Interlocking: Coupling Analytics to the Data](http://dx.doi.org/10.1109/ucc.2014.113)
    Yousef Kowsar, Harriet Dashnow, Andrew Lonie, in Utility and Cloud Computing (UCC), 2014 IEEE/ACM 7th International Conference on (December 2014), pp. 696-701, doi:10.1109/ucc.2014.113

* [A Quick Guide for Building a Successful Bioinformatics Community](http://dx.doi.org/10.1371/journal.pcbi.1003972)
    Aidan Budd, Manuel Corpas, Michelle D. Brazas, Jonathan C. Fuller, [Jeremy Goecks](/src/JeremyGoecks/index.md), Nicola J. Mulder, Magali Michaut, B. F. Francis Ouellette, Aleksandra Pawlik, Niklas Blomberg, *PLoS Computational Biology*, Vol. 11, No. 2. (5 February 2015), e1003972, doi:10.1371/journal.pcbi.1003972

* [Draft sequencing and assembly of the genome of the world’s largest fish, the whale shark: Rhincodon typus Smith 1828](https://peerj.com/preprints/837/)
    Timothy D Read​, Robert A Petit III, Sandeep J Joseph, Md T Alam, Ryan Weil, Maida Ahmad, Ravila Bhimani, Jocelyn S Vuong, Chad P Haase, Harry Webb, Alistair D. M. Dove, *PeerJ !PrePrints* 3:e1036 doi:10.7287/peerj.preprints.837v1

* [Reproducibility and reusability of genome assembly evaluation](http://bit.ly/1BEnKPO)
    Sabba Ifzal, MS Thesis, University of Oslo

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 20 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 42 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


----

<div class='right'><br /><a href='http://bit.ly/gxycast1'><img src="/src/Images/ScreencastFrame/GalaxyCast1.png" alt="Watch the video" width="150" /></a></div>

# UseGalaxy.org Update

A refresh of [UseGalaxy.org](https://usegalaxy.org) has started.  It includes
* New versions of Bowtie2, bwa-mem, Picard, BAMTools, FastQC, ....
* New look and feel for tool forms
* New Scratchbook features
* New Charts functionality

**All of these are described in the new Galaxy Screencast [GXYCast1: 2015](http://bit.ly/gxycast1).** Look for more updates coming soon.

----

# Events

## GCC2015: 6-8 July, Norwich UK

<div class='right'><a href='http://gcc2015.tsl.ac.uk/'><img src="/src/Images/Logos/GCC2015LogoWide600.png" alt="Sponsor GCC2015" width="170" /></a></div>

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is the Galaxy community's annual gathering of users, developers, and administrators.  [Previous GCC's](/src/Events/GCC2014/index.md) have drawn over 200 participants, and we expect that to happen again in 2015.  GCC2015 is being hosted by [The Sainsbury Lab](http://tsl.ac.uk/) in Norwich, UK, immediately before [BOSC](http://www.open-bio.org/wiki/BOSC_2015) and [ISMB/ECCB](http://www.iscb.org/ismbeccb2015) in Dublin.

There are a lot of events going on at GCC2015, including:

* [Code Hackathon](/src/GalaxyUpdates/2015_03/index.md#code-hackathon), 4-5 July
* [Data Wrangling Hackathon](/src/GalaxyUpdates/2015_03/index.md#data-wrangling-hackathon), 4-5 July (*new*)
* [Training SunDay](/src/GalaxyUpdates/2015_03/index.md#training-sunday), 5 July (*new*)
* [Training Day](/src/GalaxyUpdates/2015_03/index.md#training-monday), 6 July
* [GCC Meeting](/src/GalaxyUpdates/2015_03/index.md#abstract-submission-is-now-open), 7-8 July

### Code Hackathon

An [intense two-day hands-on collaboration](http://gcc2015.tsl.ac.uk/Hackathon) to *develop working code that is useful to the Galaxy community*. If you know how to code, and want to contribute to one of the most successful open source projects in the life sciences, then please consider attending. See the [Code Hackathon](http://gcc2015.tsl.ac.uk/Hackathon) home page for more.

### Data Wrangling Hackathon

An [intense two-day hands-on collaboration](http://gcc2015.tsl.ac.uk/Data-Hackathon) to *develop cutting edge analysis pipelines that are useful to the Galaxy community*. If you know data analysis, we would love to have you here to help us beat back those seemingly unsurmountable analysis challenges. See the [Data Wrangling Hackathon home page](http://gcc2015.tsl.ac.uk/Data-Hackathon) for more.

### Training SunDay

Something new for GCC2015 is *[Training SunDay](http://gcc2015.tsl.ac.uk/training-day/)*, an additional day of training offered the day before its older sibling [Training Day](http://gcc2015.tsl.ac.uk/training-day/), and featuring a single track with the most in-demand topics. You can attend both Training Days, or just one.  Training !SunDay features these three topics:

* [Introduction to Galaxy](http://gcc2015.tsl.ac.uk/training-day/#Introduction_to_Galaxy)
* [RNA-Seq Analysis with Galaxy](http://gcc2015.tsl.ac.uk/training-day/#RNA-Seq_Analysis_with_Galaxy)
* [Visualisation of NGS Data](http://gcc2015.tsl.ac.uk/training-day/#Visualisation_of_NGS_Data)

These three topics are also offered on Monday as well.  You can register for one or both Training Days.

<div class='right'><br /><a href='http://gcc2015.tsl.ac.uk/training-day/'><img src="/src/Images/Logos/GCC2015_TrainingDay128.png" alt="Training Day Schedules set" width="100" /></a>
</div>

### Training (Mon)Day

The schedule for [Training Day, Monday, 6 July is available](http://gcc2015.tsl.ac.uk/training-day/). Training Day featuring five parallel tracks, each with three, two and a half hour workshops. There are topics on [using Galaxy](http://gcc2015.tsl.ac.uk/training-day/#Using_Galaxy), [interacting with it programmatically](http://gcc2015.tsl.ac.uk/training-day/#Using_Galaxy_Programmatically), and [deploying, administering, and extending Galaxy](http://gcc2015.tsl.ac.uk/training-day/#Deploying_Administering_and_Wrapping_Tools_for_Galaxy). No matter what you do with Galaxy, there are workshops for you.

### Early Registration opens in March

*Early* [registration](http://gcc2015.tsl.ac.uk/abstracts/registration/) (save *heaps)* will open in March. Early registration is very affordable and starts at less than £40 per day for students and postdocs. If you work in data-intensive life science research, then it is hard find a meeting more relevant than GCC2015.  We look forward to seeing you there.

### Abstract Submission is Now Open

[Abstract submission for Oral and Poster Presentations](http://gcc2015.tsl.ac.uk/Abstracts) at the [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is now open.

[Abstract submission](http://gcc2015.tsl.ac.uk/Abstracts) for oral presentations closes 10 April, while poster submission closes 1 May. Poster authors will be notified of acceptance status within two weeks of submission, while oral presentation authors will be notified no later than 24 April.  *Please consider presenting your work. If you are dealing with big biological data, then this meeting wants to hear about it.*


<div class='left'><a href='http://www.gigasciencejournal.com/'><img src="/src/Images/Logos/GigaScienceLogo250.png" alt="GigaScience Journal" height="50" /></a></div>

We are pleased to announce that again in 2015, any oral or poster presentation will also be eligible for consideration for publication in the *[GigaScience](http://www.gigasciencejournal.com/)* "[Galaxy: Data Intensive and Reproducible Research](http://www.gigasciencejournal.com/series/Galaxy)" series (first announced for GCC2013).  *[GigaScience](http://www.gigasciencejournal.com/)* is pleased to offer a 15% discount on article publishing charges for accepted papers from GCC2015.

 
<div class='right'><br /><a href='http://bioteam.net/'><img src="/src/Images/Logos/BioTeamLogo154.gif" alt="BioTeam" width="120" /></a></div>

### GCC2015 Sponsorships

We are pleased to announce that [BioTeam](http://bioteam.net) will again be a [Gold Sponsor](http://gcc2015.tsl.ac.uk/sponsors/) for this annual event.  This is the third year in a row that !BioTeam has been a GCC Gold Sponsor.

#### Call for Sponsors

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is now accepting [Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/).  Your organisation can play a prominent part in the Galaxy community by sponsoring GCC2015.  Sponsorship is an excellent way to raise your organization’s visibility.

Several sponsorship levels are available, including two levels of premier sponsorships that include presentations.  Premium sponsorships are limited, however, so you are encouraged to act soon.

Please let the [organisers](http://gcc2015.tsl.ac.uk/) ([mailto:gcc2015-org AT lists DOT galaxyproject DOT org](gcc2015-org AT lists DOT galaxyproject DOT org)) know if you are interested in helping make this event a success.

----

<div class='right'><br /><a href='http://training.bioinformatics.ucdavis.edu/2015/01/12/rna-seq-and-chip-seq-analysis-with-galaxy/'><img src="/src/Images/Logos/UCDavisGenomeCenter.png" alt="RNA-Seq and ChIP-Seq Analysis with Galaxy" height="50" /></a></div>

## RNA-Seq and ChIP-Seq Analysis with Galaxy, UC Davis, March 23-26

A hands-on, 4 day workshop on *[RNA-Seq and ChIP-Seq Analysis with Galaxy](http://training.bioinformatics.ucdavis.edu/2015/01/12/rna-seq-and-chip-seq-analysis-with-galaxy/)* will be held at the  [UC Davis Bioinformatics Core](http://bioinformatics.ucdavis.edu/) in Davis, California, United States, March 23-26.  There are just a few slots left.  

## February GalaxyAdmins Meetup

<div class='right'><a href='/src/Community/GalaxyAdmins/Meetups/2015_02_19/index.md'><img src="/src/Images/Logos/GalaxyAdmins.png" alt="GalaxyAdmins meetup February 19" width="150" /></a></div>

[Video and slides](/src/Community/GalaxyAdmins/Meetups/2015_02_19/index.md) from the first [GalaxyAdmins](/src/Community/GalaxyAdmins/index.md) [meetup of 2015](/src/Community/GalaxyAdmins/Meetups/2015_02_19/index.md) are now available online. 

[Hans-Rudolf Hotz](/src/HansrudolfHotz/index.md) also lead a short discussion on *GalaxyAdmins direction*.

[GalaxyAdmins](/src/Community/GalaxyAdmins/index.md) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  Our next meetup will be in April.

### Fall 2014 Questionnaire Summaries

A summary of both the user and admin/developer [2014 Galaxy Community Questionnaires](/src/News/2014Questionnaire/index.md) was presented by [Dave Clements](/src/DaveClements/index.md).  A [summary of the admin/devel questionnaire](/src/Community/GalaxyAdmins/Surveys/2014/index.md) is now available online.

### Gravity

[Nate Coraor](/src/nate/index.md) introduced [Gravity](/src/Community/GalaxyAdmins/Meetups/2015_02_19/index.md#galaxy-project-update-gravity), a process manager (supervisor) for Galaxy servers.  

## Other Events

<div class='center'> 
<a href='http://www.irb.hr/eng/Research/Divisions-and-Centers/Centre-for-Informatics-and-Computing/News/FREE-Workshop-Bioinformatics-Methods-in-Genomics'><img src="/src/Images/Logos/IRBLogo300.jpg" alt="Ruđer Bošković Institute (RBI)" height="70" /></a>
<a href='http://www.qfab.org/events/upcoming/'><img src="/src/Images/Logos/QFABLogo.png" alt="QFAB Workshops" height="80" /></a>
<a href='PLACEHOLDER_ATTACHMENT_URL/src/Documents/Flyers/DéveloppementEtIntégrationDApplicationsSousGalaxy.pdf'><img src="/src/Images/Logos/IFBSmallTransLogo.png" alt="Développement et intégration d’applications sous Galaxy" height="66" /></a>
<a href='http://training.bioinformatics.ucdavis.edu/2015/01/12/rna-seq-and-chip-seq-analysis-with-galaxy/'><img src="/src/Images/Logos/UCDavisGenomeCenter.png" alt="RNA-Seq and ChIP-Seq Analysis with Galaxy" height="50" /></a> &nbsp;
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src="/src/Images/Icons/CalendarIcon.gif" /></a></div>

There are [upcoming events](/src/Events/index.md) in 7 countries on 4 continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> March 2 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-rna-seq-analysis-using-galaxy/'>RNA-Seq analysis using Galaxy</a><em> </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md#qfab-bioinformatics'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> March 2-3 </th>
    <td> </em><a href='http://www.irb.hr/eng/Research/Divisions-and-Centers/Centre-for-Informatics-and-Computing/News/FREE-Workshop-Bioinformatics-Methods-in-Genomics'>Workshop on Bioinformatics Methods in Genomics</a><em> </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md#ruđer-bošković-institute-rbi'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.irb.hr/eng'>RBI</a>, Zagreb, Croatia </td>
    <td> <a href='/src/EnisAfgan/index.md'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> March 3-5 </th>
    <td> </em><a href='PLACEHOLDER_ATTACHMENT_URL/src/Documents/Flyers/DéveloppementEtIntégrationDApplicationsSousGalaxy.pdf'>Développement et intégration d’applications sous Galaxy</a><em> </td>
    <td> <div class='right'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></div> La Chapelle Sur Erdre, France </td>
    <td> <a href="mailto:frederique.malipier@nantes.inra.fr">Frederique Malipier</a> </td>
  </tr>
  <tr>
    <th> March 3-5 </th>
    <td> </em><a href='http://bmc.biomed.lu.lv/en/startpage/news/invitation-to-attend-courses/'>COST Basic training on NGS data analysis Course</a><em> </td>
    <td> University of Latvia, Riga </td>
    <td> <a href="mailto:gec@biomed.lu.lv">Latvian Genome Centre</a> </td>
  </tr>
  <tr>
    <th> March 23-26 </th>
    <td> </em><a href='http://training.bioinformatics.ucdavis.edu/2015/01/12/rna-seq-and-chip-seq-analysis-with-galaxy/'>RNA-Seq and ChIP-Seq Analysis with Galaxy</a><em> </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnUCDavis'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://bioinformatics.ucdavis.edu/'>UC Davis Bioinformatics Core</a>, Davis, California, United States </td>
    <td> <a href="mailto:bioinformatics.core@ucdavis.edu">UC Davis Bioinformatics</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> April 14-15 </th>
    <td> </em><a href='http://www.globusworld.org/'>GlobusWorld 2015</a><em> </td>
    <td> Chicago, Illinois, United States </td>
    <td> <a href="mailto:outreach AT globus.org">Globus Outreach</a> </td>
  </tr>
  <tr>
    <th> April 20-21 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-2/'>Workshop: Extended RNA-Seq analysis</a><em> </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md#qfab-bioinformatics'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th rowspan=3> April 21 </th>
    <td> </em><a href='http://www.bio-itworldexpo.com/Next-Gen-Sequencing-Informatics/'>Large-Scale NGS Analysis Using Globus Genomics: Challenges and User Success Stories</a><em> </td>
    <td rowspan=4> <a href='http://www.bio-itworldexpo.com/'>Bio-IT World 2015</a>, Boston, Massachusetts, United States </td>
    <td> <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a> </td>
  </tr>
  <tr>
    <td> </em><a href='http://www.bio-itworldexpo.com/Bio-It_Expo_Content.aspx?id=144286'>Workshop: Large Scale NGS Analysis Using Globus Genomics</a><em> </td>
    <td> Paul Davé, <a href='http://www.mcs.anl.gov/person/ravi-madduri'>Ravi Madduri</a>, Alex Rodriguez, Dinanath Sulakhe </td>
  </tr>
  <tr>
    <td> </em><a href='http://www.bio-itworldexpo.com/Bioinformatics/'>Extending Galaxy with External Microbiome Databases</a><em> </td>
    <td> Bob Brown </td>
  </tr>
  <tr>
    <th> April 23 </th>
    <td> </em><a href='http://www.bio-itworldexpo.com/Data-Visualization/'>Visualization Tools for the Refinery Platform</a><em> </td>
    <td> <a href='http://gehlenborg.com/'>Nils Gehlenborg</a> </td>
  </tr>
  <tr>
    <th> April 27 - May 3 </th>
    <td> </em><a href='http://bioinformatics.ca/workshops/2015/high-throughput-biology-sequence-networks-2015'>High-throughput Biology: From Sequence to Networks 2015</a><em> </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md#bioinformaticsca'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> New York City, New York, United States </td>
    <td> <a href='http://bioinformatics.ca/person/cbw-experts/francis-ouellette'>Francis Ouellette</a> </td>
  </tr>
  <tr>
    <th> April 28 </th>
    <td> <a href='/src/Events/Tokyo2015/index.md'>Galaxy Workshop Tokyo 2015</a> </td>
    <td> <a href='http://www.rcast.u-tokyo.ac.jp/index_en.html'>RCAST, The University of Tokyo</a>, Japan </td>
    <td> <a href='http://wiki.pitagora-galaxy.org/wiki/index.php/Galaxy_Workshop_Tokyo_2015#Contact'>Ryota Yamanaka</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> </strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a><strong> </td>
    <td> <div class='right'><a href='/src/Teach/Trainers/index.md'><img src="/src/Images/Icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div>   <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/src/Teach/Trainers/index.md'><img src="/src/Images/Icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/src/Teach/Trainers/index.md'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# Who's Hiring

<div class='right'><a href='/src/GalaxyIsHiring/index.md'><img src="/src/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Developer / bioinformatician](/src/News/UiODeveloperBioinformatician/index.md) at the University Center for Information Technology (USIT) at the University of Oslo
* [Computational Metabolomics Professor](https://psu.jobs/job/55363), Penn State University, Pennsylvania, United States
* [Software Developer for the Refinery Team](https://jobs.brassring.com/tgwebhost/jobdetails.aspx?jobId=1118216&PartnerId=25240&SiteId=5341&type=mail&JobReqLang=1&recordstart=1&JobSiteId=5341&JobSiteInfo=1118216_5341&gqid=0), Boston, Massachusetts, United States
* [Bioinformatics scientist/engineer](/src/News/MDCBerlinBuchOpening/index.md) for RNA bioinformatics and next-generation sequencing, [MDC Berlin-Buch](https://www.mdc-berlin.de/)  
* Searching for [bioinformaticians, post-docs, PhD students and software engineers](/src/News/BioinformaticianFreiburg2014/index.md) in Freiburg, Germany at [Max Planck Institute of Immunobiology and Epigenetics](http://www.ie-freiburg.mpg.de/jobs), and the [Bioinformatics Group at the University of Freiburg](http://www.bioinf.uni-freiburg.de//?en)
* **The [Galaxy Project is hiring software engineers and post-docs](/src/GalaxyIsHiring/index.md)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

----

# New Public Servers

One new [public Galaxy servers](/src/PublicGalaxyServers/index.md) was added in February:

## Whale Shark

<div class='right solid'><a href='http://whaleshark.georgiaaquarium.org/'><img src="/src/PublicGalaxyServers/WhaleSharkLanding300.png" alt="Whale Shark Galaxy @ The Georgia Aquarium" width=300 /></a></div>

* *Link:*
  * [Whale Shark Galaxy](http://whaleshark.georgiaaquarium.org/)
* *Domain/Purpose:*
  * "Dedicated to the analysis of whale shark *(Rhincodon typus)* genome."
* *Comments:*
  * See:
    [Draft sequencing and assembly of the genome of the world’s largest fish, the whale shark: Rhincodon typus Smith 1828](https://peerj.com/preprints/837/), Read TD, Petit III RA, Joseph SJ, Alam MT, RW, Ahmad M, Bhimani R, Vuong JS, Haase CP, Webb H, Dove ADM. (2015). *[PeerJ PrePrints](https://peerj.com/preprints/)* 3:e1036 http://dx.doi.org/10.7287/peerj.preprints.837v1
* *User Support:*
* *Quotas:* 
  * Max file size to upload: 2 GB
  * For registered users, you have a disk quota of 50GB and a job walltime of 24 hours.
  * For anonymous users, you have a disk quota of 50MB and a job walltime of 30 minutes.
* *Sponsor(s):*
  * "The whale shark genome project was made possible by financial and other support of [Georgia Aquarium](http://www.georgiaaquarium.org/) and its guests, and by a completion grant from [The Coca Cola Company](http://us.coca-cola.com/home/)."

# Galaxy Community Hubs

<div class='solid'>
<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/Teach/index.md'><img src="/src/Images/Logos/GTNLogo600.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/Community/Logs/index.md'><img src="/src/Images/Logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/Community/Deployments/index.md'><img src="/src/Images/Logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong>Share your <a href='/src/Teach/Resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/Teach/Trainers/index.md#add-a-trainer'>experience</a> </em>now<em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/src/Community/Logs/index.md#add-a-log-page'>Share your experience</a> </em><a href='/src/Community/Logs/index.md#add-a-log-page'>now</a><em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/src/Community/Deployments/index.md#add-your-galaxy-deployment'>Describe your instance</a> </em><a href='/src/Community/Deployments/index.md#add-your-galaxy-deployment'>now</a><em> <strong> </td>
  </tr>
</table>

</div>
<br />

Two new [Community Log Board](/src/Community/Logs/index.md) entires were added in February:

* [FTP upload to Galaxy using ProFTPd and PBKDF2](/src/Community/Log/2015/ProFTPdAndPBKDF2/index.md)
    ProFTPd implemented using PBKFD2 protocol for encryption, by [Peter Briggs](http://www.ls.manchester.ac.uk/people/profile/?alias=briggsp)
* [Instalando um servidor do Galaxy Project](/src/Community/Log/2015/InstalandoServidorGalaxy/index.md)
    Portuguese Galaxy Documentation, by [Léo Biscassi](https://biscassi.wordpress.com/quem-sou-eu/)

----

# Galaxy GitHub

Galaxy has moved its source from Bitbucket to [GitHub](https://github.com/galaxyproject/galaxy). The development of Galaxy takes place there from now on.

However, **the Bitbucket repositories are still the official source for deployment** until we polish the process. 

More information available in [Develop/SourceCode](/src/Develop/SourceCode/index.md).

# Releases

## Galaxy 2015/01/13 Distribution

The most recent Galaxy release was the [2015/01/13 distribution](/src/DevNewsBriefs/2015_01_13/index.md).  This addressed several *[critical security vulnerabilities](/src/DevNewsBriefs/2015_01_13/index.md#security)*, added [IPython integration](/src/DevNewsBriefs/2015_01_13/index.md#ipython_integration), and included a [Tool Form Upgrade for Beta Testing](/src/DevNewsBriefs/2015_01_13/index.md#tool_form_upgrade_28for_beta_testing29).

## Planemo 0.5.0

There were several releases of [Planemo](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst#050-2015-02-22) in February. Among many other new features, it is now install-able from [PyPI](https://pypi.python.org/pypi/planemo) and contains a new command to assist in [bootstrapping new tools](http://planemo.readthedocs.org/en/latest/writing.html).

## BioBlend

[BioBlend v0.5.2 was released in October](/src/News/BioBlendV0.5.2/index.md).  [BioBlend](https://github.com/galaxyproject/bioblend) is a python library for interacting with CloudMan and the [Galaxy API](/src/Learn/API/index.md).  In January the [BioBlend source](https://github.com/galaxyproject/bioblend) was moved to the [galaxyproject Github account](https://github.com/galaxyproject).

## CloudMan and blend4j

New versions [CloudMan](/src/GalaxyUpdates/2014_09/index.md#august-2014-cloudman-release), and [blend4j](/src/GalaxyUpdates/2014_09/index.md#blend4j-011-release) were released in August.

----

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/Images/Logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

## Galaxy Project ToolShed Repos

There was no shortage of new ToolShed repositories in February, but there was a shortage of time to describe them.  Look for a double batch of new tools next month.

----

# Other News

* From [Pierre Lindenbaum](https://twitter.com/yokofakun)
    [Automatic generation of tool.xml for Galaxy using xslt](http://tinyurl.com/mflwofl) 
* [The KLAST high-perfomance sequence similarity search tool is now available for the Galaxy platform.](http://www.korilog.com/news/klast-for-galaxy-platform/196)
* [VBI-led Team Introduces Infectious Disease-focused Resource for Analyzing RNA-seq Data](http://bit.ly/1uKqHLB) on [GenomeWeb](http://genomeweb.com)

