---
title: May 2018 Galaxy News
---
[<img class="float-left" src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 />](/src/galaxy-updates/index.md)

Welcome to the May 2018 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# GCCBOSC 2018

[<img class="float-right" src="/src/images/logos/gcc-bosc-2018-logo-boxed-300.png" width="160" alt="GCCBOSC 2018" />](https://gccbosc2018.sched.com/)

[GCCBOSC 2018](https://gccbosc2018.sched.com/) will be held 25-30 June in Portland, Oregon, United States. This brings the 2018 Galaxy Community Conference and the Bioinformatics Open Source Conference together into a unified week-long event. If you work in open source life science or data-intensive biomedical research, then there is no better place than GCCBOSC 2018 to present your work and to learn from others.

[GCCBOSC](https://gccbosc2018.sched.com/) starts with [two days of training](https://gccbosc2018.sched.com/overview/type/C.x+Training+-+All) with a wide range of topics nominated and selected by our communities.  Training is followed by a two day meeting, with joint and parallel tracks, featuring oral presentations, posters, demos, lightning talks, birds-of-a-feather and [invited keynotes](/src/events/gccbosc2018/keynotes/index.md).  The week finishes with *[CollaborationFest](/src/events/gccbosc2018/collaboration/index.md) Core* and *Encore*, two or four days of collaborative work on code, documentation, training and challenging data analysis problems.

## GCC2018 schedule posted

The schedule of accepted long talks for GCC2018 is [now available online](https://gccbosc2018.sched.com/).  [Sixteen topics](https://gccbosc2018.sched.com/overview/type/B.g+Conference+-+GCC) have been scheduled spanning the full Galactic spectrum from [deployment details](http://sched.co/EYCM) to [novel applications of Galaxy](http://sched.co/EYNG) and everything in between.

BOSC talks and GCC lightning talks will be added in the next week or so.

## Early registration ends May 11

Did you catch that?  **Early registration ends May 11!**  *So c'mon, register already:*

* [Register](https://gccbosc2018.eventbrite.com)
* [Book conference housing](https://gccbosc2018housing.eventbrite.com)
* [Childcare](/src/events/gccbosc2018/childcare/index.md).  **Space in childcare is very limited.**

If you register after May 11, you will have less money to fund reagents, storage, compute timepost-docs, field work, 3D printers, [VR systems](/src/events/gccbosc2018/icebreaker/index.md)...

## BOSC deadline for late-breaking lightning talks, demos and posters is May 17

* Submit your abstract [here](https://www.open-bio.org/wiki/BOSC_Abstract_Submission).

## GCCBOSC 2018 Sponsors

We are happy to have [sponsors](/src/events/gccbosc2018/sponsors/index.html) contributing to the success of GCCBOSC 2018. This month we are highlighting **[Advanced HPC](https://www.advancedhpc.com/industries/life-sciences/)** and **[GigaScience](https://academic.oup.com/gigascience)**.

### GigaScience and Oxford University Press

[<img class="float-left" src="/src/events/gccbosc2018/sponsors/gigascience-oup.png" alt="GigaScience, focusing on ‘big data’ research from the life and biomedical sciences." width="200" />](https://academic.oup.com/gigascience)

**[GigaScience](https://academic.oup.com/gigascience)** is an online open access, open data, open peer-review journal published by **[Oxford University Press](https://academic.oup.com/)** and **[BGI](https://www.bgi.com/us/)**. The journal offers ‘big data’ research from the life and biomedical sciences, and on top of 'Omics research includes the growing range of work that uses difficult-to-access large-scale data, such as imaging, neuroscience, ecology, systems biology, and other new types of shareable data. *GigaScience* is unique in the publishing industry as it publishes all research objects (data, software tools, source code, workflows, containers and other elements related to the work underpinning the findings in the article). Promoting Open Science, all published software needs to be under an OSI-license, all supporting data must be available and open, and all peer review is carried out transparently. Presenting workflows via our [GigaGalaxy.net](http://gigagalaxy.net/) server, novel work presented at the meeting utilising Galaxy is eligible to a 15% APC if it is submitted to our [Galaxy series](https://academic.oup.com/gigascience/pages/galaxy_series_data_intensive_reproducible_research).


### Advanced HPC

[<img class="float-right" src="/src/events/gccbosc2018/sponsors/advanced-hpc-logo.png" alt-"Advanced HPC" width="250" />](https://www.advancedhpc.com/industries/life-sciences/)

Life sciences is an ideal domain to take full advantage of all the things that HPC technology can do. Universities, research organizations, and biotechnology companies are embracing predictive analytics, machine learning and artificial intelligence to make major gains in fighting cancer, personalizing medicine and regenerating organs. The technology is paying huge dividends. For example, research institutions that are using Illumina’s<sup>®</sup> DNA sequence and array-based technologies for genotyping and epigenetic applications are leveraging our compute and parallel solutions to get better and faster results.

**[Advanced HPC's](https://www.advancedhpc.com/industries/life-sciences/)** experts offer many advantages, including extensive consultation and support. As HPC software specialists, our engineers can customize complex solutions for you more quickly and efficiently than our competitors.

Advanced HPC is pleased to be the first ever **[GCCBOSC Icebreaker](/src/events/gccbosc2018/icebreaker/index.md) Sponsor**.


<div class="float-right">
<br />
[<img src="/src/events/gccbosc2018/sponsors/summary-thumb.png" alt="Download the one page summary." />](https://depot.galaxyproject.org/hub/attachments/documents/gccbosc2018_sponsorship_summary_20170906.pdf) [<img src="/src/events/gccbosc2018/sponsors/prospectus-thumb.png" alt="Download the full prospectus." />](https://depot.galaxyproject.org/hub/attachments/documents/gccbosc2018_sponsorship_prospectus_20170906.pdf)  </div>

### Call for sponsors!

[Sponsors](/src/events/gccbosc2018/sponsors/index.html) are a key part of [GCCBOSC 2018](https://gccbosc2018.sched.com/).  Is your organization interested in playing a prominent role in the first joint gathering of the Galaxy and BOSC communities? Then [become a GCCBOSC 2018 sponsor](/src/events/gccbosc2018/sponsors/index.md#interested) and raise your organization's visibility in these active and engaged communities.

<br />

## ELIXIR Workshop for Galaxy training material and skills improvement

**The [ELIXIR Workshop for Galaxy training material and skills improvement](http://www.earlham.ac.uk/elixir-workshop-galaxy-training-material-and-skills-improvement) will be held 21-23 May at the [Earlham Institute](http://www.earlham.ac.uk/) in Norwich, UK.**

[<img class="float-right" src="/src/images/logos/ElixirNoTextLogo.png" width="140" />](http://www.earlham.ac.uk/elixir-workshop-galaxy-training-material-and-skills-improvement)

Education and training is an integral part of the Galaxy community. The Galaxy Training Network (GTN) has been working for several years with [GOBLET](http://www.mygoblet.org/) and the [ELIXIR Training Platform](https://www.elixir-europe.org/platforms/training) to create material and deliver training for scientists, developers and system administrators. A selection of 72 tutorials, developed by more than 60 contributors, is [already available](http://training.galaxyproject.org). 
In this workshop, we intend to improve the participants’ understanding of learning principles, training techniques and best practices for materials preparation. **We will then work together to expand the existing collection of Galaxy training materials by covering more topics relevant to the ELIXIR use cases, add missing annotation and metadata and make the materials easily accessible.**

The workshop will start with a 1-day "Train the Trainer" course, similar to the Carpentries' [Instructor Training](https://carpentries.github.io/instructor-training/) and the [ELIXIR-EXCELERATE Train the Trainer](https://github.com/TrainTheTrainer/EXCELERATE-TtT). Here Galaxy trainers will develop an insight into different learning styles, understand what makes a good trainer, and learn new approaches to training from experienced trainers. This course will help them to better contribute and shape the materials during the hackathon.

**[Register now](https://www.eiseverywhere.com/galaxy).**

## Galaxy Africa Report

<div class="float-left">
[<img src="/src/events/2018-04-galaxy-africa/galaxy-africa-logo.svg" alt="Galaxy Africa" height="100" />](http://galaxyafrica.sanbi.ac.za/) &nbsp;&nbsp;&nbsp;
<img src="/src/events/2018-04-galaxy-africa/2018-galaxy-africa-participants.jpg" alt="44 participants, 13 speakers, 7 countries, 12 institutions, 3 days of amazing learning and discussion!" height="120" />
</div>

The beginning of April saw a flocking of a large number of individuals interested in all things Galaxy to Cape Town! This exciting destination was home for the first ever [Galaxy Africa](http://galaxyafrica.sanbi.ac.za/) event that aimed to inform, educate, and bring together bioinformaticians, biologists, and computer scientists from around the African continent. Organized by [SANBI](http://www.sanbi.ac.za/) and held at the [University of the Western Cape](https://www.uwc.ac.za/), under the close eye of [Peter van Heusden](https://www.researchgate.net/profile/Peter_Van_Heusden), nearly 50 people participated in a packed schedule where everything from an introduction to Galaxy, training, analysis, experiences, and system management were covered. There were a number of BoF sessions as well to have open discussions and start future conversations. The week ended with a Data Carpentry R workshop event.

Even though the event ended, it planted a seed for a new Galaxy community that spans the African continent: emails were matched with faces, mailing lists were formed, meals shared, and lessons passed on. Thank you Galaxy Africa!

# Upcoming events

These and [other Galaxy related events](/src/events/index.md)  coming up in the next few months:

<table class="table table-striped">
<thead>
<tr>
  <th>Date</th>
    <th>Topic/Event</th>
    <th>Venue/Location</th>
    <th>Contact</th>
</tr>
</thead><tbody>
<tr>
  <td><span class="text-nowrap">May 8th 2018</span></td>
  <td><a href="https://faes.org/events/biotech-45-bioinformatic-analysis-next-generation-sequencing-data-4-day-hands-workshop-2">Bioinformatics Analysis of Next Generation Sequencing Data</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://events.r20.constantcontact.com/register/eventReg?llr=k4uamblab&amp;oeidk=a07ef2kmozmbd90f112">NIH, Bethesda, Maryland, United States</a></td>
  <td>registration@faes.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2018</span></td>
  <td><a href="https://sciencegateways.org/engage/bootcamp">Science Gateways Bootcamp</a>
    <div class="indent">*Call for Participation extended to Wednesday, May 16, 2018*</div></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.tacc.utexas.edu/">TACC, Austin, Texas, United States</a></td>
  <td>help@sciencegateways.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2018</span></td>
  <td><a href="https://wwz.ifremer.fr/Recherche-Technologie/Departements-scientifiques/Departement-Infrastructures-de-Recherche-et-Systemes-d-Information/Activites/Bioinformatique/Formation-metabarcoding-2018">Formation metabarcoding 2018</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.sb-roscoff.fr/">Station Biologique de Roscoff, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Intervenants</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2018</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2018/module17.pdf">Initiation à Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2018, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>formation.migale @ inra.fr</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 15th 2018</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2018/module8bis.pdf">Analyse primaire de données issues de séquenceurs nouvelle génération sous Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2018, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>formation.migale @ inra.fr</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 15th 2018</span></td>
  <td><a href="https://qfab.org/workshops/rna-seq-analysis-using-galaxy-1-2-may-2018">Variant detection using Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="http://www.qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 16th 2018</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2018/module18bis.pdf">Traitement bioinformatique des données RNA-seq sous Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2018, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>formation.migale @ inra.fr</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="http://www.earlham.ac.uk/elixir-workshop-galaxy-training-material-and-skills-improvement"><strong>ELIXIR Workshop for Galaxy training material and skills improvement</strong></a></td>
  <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.earlham.ac.uk/">Earlham Institute, Norwich, United Kingdom</a></td><td>training@earlham.ac.uk</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="http://www.hicomb.org/">HiCOMB 2018: 17th IEEE International Workshop on High Performance Computational Biology</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/>Vancouver, British Columbia, Canada</td>
  <td>James Taylor</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors">Big Genomics Data Skills Training course</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.jax.org/about-us/locations/farmington">JAX Genomic Medicine, Farmington, Connecticut, United States</a></td>
  <td>Charlie Wray</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2018</span></td>
  <td><a href="http://cnrsformation.cnrs.fr/stage-18239-Analyse-avancee-de-sequences.html?axe=65">Bioinformatique - Analyse avancée de séquences</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.labri.fr/">Bordeaux, France</a></td>
  <td>Intervenants</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 29th 2018</span></td>
  <td><a href="https://qfab.org/workshops/genome-assembly-using-galaxy-29-30-may-2018">Genome assembly using Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="http://qfab.org/">QFAB, St Lucia, Queensland, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>training@qfab.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 6th 2018</span></td>
  <td><a href="https://www.ebi.ac.uk/training/events/2018/phenomenal-gateway-portal-metabolomics-data-analysis-cloud">PhenoMeNal Gateway: Portal to metabolomics data analysis in the cloud</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="https://www.google.com/maps?q=Wellcome+Genome+Campus+Wellcome+Genome+Campus,+Hinxton,+Cambridge,+CB10+1SD,+United+Kingdom">Wellcome Genome Campus, Hinxton, United Kingdom</a></td>
  <td>Organisers</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 12th 2018</span></td>
  <td><a href="http://gonramp.wustl.edu/?page_id=724">G-OnRamp Workshop</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 13th 2018</span></td>
  <td><a href="http://imbio.de/ib2018">Integrative Bioinformatics</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Rothamsted Research, Harpenden, United Kingdom</td>
  <td>Björn Grüning</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 18th 2018</span></td>
  <td><a href="https://hackathon.colorado.edu/">Computational Genomics</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://hackathon.colorado.edu/">Rocky Mountain Genomics HackCon 2018, University of Colorado, Boulder, Colorado, United States</a></td>
  <td>James Taylor</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 19th 2018</span></td>
  <td><a href="http://migale.jouy.inra.fr/sites/all/downloads/Migale/Formations/2018/module23.pdf">Traitement bioinformatique et analyse différentielle de données d'expression RNA-seq sous Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://migale.jouy.inra.fr/?q=formations">Cycle &quot;Bioinformatique par la pratique&quot; 2018, INRA, Jouy-en-Josas, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>formation.migale @ inra.fr</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 25th 2018</span></td>
  <td><a href="https://www.genopole.fr/spip.php?page=rubrique_event&amp;id_rubrique=1108&amp;event=1108">Genopole Summer School</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="https://www.genopole.fr/spip.php?page=rubrique_event&amp;id_rubrique=1111&amp;event=1108">Campus des Berges de Seine, Seine-Port, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Roxane Brachet </td>
</tr>
<tr>
  <td><span class="text-nowrap">June 25th 2018</span></td>
  <td><a href="https://gccbosc2018.sched.com">**GCCBOSC 2018**</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="/events/gccbosc2018/venue/">Reed College, Portland, Oregon, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 27th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/555-a-primer-for-rna-seq-processing-interpreting-and-visualization">A primer for RNA-Seq processing, interpreting and visualization</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Leibniz Institute on Aging - Fritz Lipmann Institute (FLI), Jena, Germany</td>
  <td>Steve Hoffmann, Olaf Wolkenhauer, Wolfgang Hess</td>
</tr>
<tr>
  <td><span class="text-nowrap">July 16th 2018</span></td>
  <td><a href="http://gonramp.wustl.edu/?page_id=724">G-OnRamp Workshop</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
</tr>
<tr>
  <td><span class="text-nowrap">November 22nd 2018</span></td>
  <td><a href="/events/2018-europe-dev">**European Galaxy Developer Days**</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Freiburg, Germany</td>
  <td>Hans-Rudolf Hotz, Björn Grüning</td>
</tr>
</tbody></table>

----

# Publications

**132 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in April.

## Highlighted Publications

April added three *[Galactic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/%2BGalactic)* and *[Stellar](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/%2BStellar)* publications:

* [IRProfiler – a software toolbox for high throughput immune receptor profiling](https://doi.org/10.1186/s12859-018-2144-z), Christos Maramis, Athanasios Gkoufas, Anna Vardi, Evangelia Stalika, Kostas Stamatopoulos, Anastasia Hatzidimitriou, Nicos Maglaveras and Ioanna Chouvarda. *BMC Bioinformatics* (2018) 19:144, doi:10.1186/s12859-018-2144-z
* [SECIMTools: a suite of metabolomics data analysis tools](https://doi.org/10.1186/s12859-018-2134-1), 
Alexander S. Kirpich, Miguel Ibarra, Oleksandr Moskalenko, Justin M. Fear, Joseph Gerken, Xinlei Mi, Ali Ashrafi, Alison M. Morse and Lauren M. McIntyre. *BMC Bioinformatics* (2018) 19:151, doi:10.1186/s12859-018-2134-1
* [Notos - a galaxy tool to analyze CpN observed expected ratios for inferring DNA methylation types](https://doi.org/10.1186/s12859-018-2115-4), Ingo Bulla, Benoît Aliaga, Virginia Lacal, Jan Bulla, Christoph Grunau and Cristian Chaparro, *BMC Bioinformatics* (2018) 19:105, doi:10.1186/s12859-018-2115-4


## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 79 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Methods) | | 29 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UsePublic) | | 17 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Workbench) | | 16 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseMain) | 
| 11 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Reproducibility) | | 9 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+RefPublic) | | 7 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseLocal) | | 5 | [+Other](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Other) | 
| 5 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+IsGalaxy) | | 5 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Tools) | | 4 | [+Cloud](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Cloud) | | 2 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Shared) | 
| 1 | [+Unknown](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Unknown) | | | | | | | | | | |


----
# Who's Hiring

<div class='right'><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.
* [Software Developer](https://jobs.quadram.ac.uk/Details.asp?vacancyID=13373) and [Bioinformaticians](https://jobs.quadram.ac.uk/Details.asp?vacancyID=13372), [Quadram Institute](http://www.quadram.ac.uk/), Norwich , United Kingdom
* [Ingénieur en Bioinformatique NGS](https://www.sfbi.fr/content/ing%C3%A9nieur-en-bioinformatique-ngs-0), [Institut de Biologie Intégrative de la Cellule](https://www.i2bc.paris-saclay.fr/), Orsay, France
* [Scientist (f/m)](https://bms.candibase.de/atool/helmholtz/frontend.php?view=93d61bae347a0d201ff20cde45629683&lang=en), NGS bioinformatics core facility, [Helmholtz Zentrum München](https://www.helmholtz-muenchen.de/en), Germany.
* [Software developer (f/m) position at the NGS bioinformatics core facility](https://bms.candibase.de/atool/helmholtz/frontend.php?view=cb4196b9c4c29479037d5b8035422380&lang=en), [Helmholtz Zentrum München](https://www.helmholtz-muenchen.de/en), Germany.
* [Ingenieur Bioinformatique Evolutive](https://www.sfbi.fr/content/ingenieur-bioinformatique-evolutive-1) [LIRMM](http://www.lirmm.fr/), Montpellier, France
* [Post-Doc / IR analyse de données miRNA](https://www.sfbi.fr/content/post-doc-ir-analyse-de-donn%C3%A9es-mirna), Unité de Nutrition Humaine, Clermont-Fd, France
* [Freiburg Galaxy Team has open positions](https://usegalaxy-eu.github.io/posts/2018/01/18/openpositions/), Freiburg, Germany
* [Development of innovative algorithms to process real-time mass spectrometry data for the clinical analysis of exhaled breath](https://www.sfbi.fr/content/d%C3%A9veloppement-doutils-bioinformatiques-et-statistiques-pour-lanalyse-de-lair-expir%C3%A9-en-m%C3%A9dec), CEA, Saclay, France
* The [Blankenberg](/src/people/dan/index.md) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/src/news/2017-09-hiring-at-cc/index.md).
* [Galaxy Project is hiring software engineers and postdocs at Johns Hopkins](/src/news/2017-08-hiring-at-hopkins/index.md), Baltimore, Maryland, United States

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

# Public Galaxy Server News

There are [over 100 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Another public server was added to the directory in April:

## IRProfiler

[<img class="float-right" src="/src/use/irprofiler/irprofiler.gif" alt="IRProfiler" width="320" />](http://irprofiler.med.auth.gr:8080/)

[Immune Repertoire Profiler (IRProfiler)](http://irprofiler.med.auth.gr:8080/) is a novel software pipeline that delivers a number of core receptor repertoire quantification and comparison functionalities on high-throughput TR and BcR sequencing data. IRProfiler can be used anonymously or through an account.

This server is made available as supplementary material for the article "[IRProfiler - A Software Toolbox for High Throughput Immune Receptor Profiling](https://link.springer.com/article/10.1186%2Fs12859-018-2144-z)", authored by C. Maramis et al. in *BMC Bioinformatics*.  A [Docker image of the IRProfiler Galaxy toolbox](https://hub.docker.com/r/chmaramis/irprofiler/) and [wrapped IRProfiler tools](https://toolshed.g2.bx.psu.edu/repository?repository_id=8660186ea1d6b696&changeset_revision=acaa8e8a0b88) are also available.

IRProfiler is supported by the [Lab of Computing, Medical Informatics & Biomedical-Imaging Technologies](http://www1.med.auth.gr/depts/lomi/en/index_en.htm), [Department of Medicine](https://www.auth.gr/en/med), Aristotle University of Thessaloniki, in Thessaloniki, Greece; and the [Institute of Applied Biosiences](http://hbio.gr/member/institute-applied-biosciences-inab-centre-research-and-technology-certh), [Centre for Research & Technology Hellas](https://www.certh.gr/root.en.aspx), in Thermi, Greece

## Public Servers in Publications

We tag papers that use, mention, implement or extend public Galaxy Servers.  Here are the counts for March's publications.

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 13 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Huttenhower) | | 8 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RepeatExplorer) | | 4 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Workflow4Metabolomics) | | 2 | [>RiboGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RiboGalaxy) | 
| 2 | [>PRABI](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PRABI) | | 1 | [>NCGAS](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>NCGAS) | | 1 | [>Langille](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Langille) | | 1 | [>EpiToolKit](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>EpiToolKit) | 
| 1 | [>Pasteur](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Pasteur) | | 1 | [>NGS-QC](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>NGS-QC) | | 1 | [>Globus Genomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Globus Genomics) | | 1 | [>PlantGenie](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PlantGenie) | 
| 1 | [>ARGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>ARGalaxy) | | 1 | [>Osiris](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Osiris) | | 1 | [>IRProfiler](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>IRProfiler) | | 1 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Cistrome) | 
| 1 | [>Jetstream](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Jetstream) | | 1 | [>Alveo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Alveo) | | 1 | [>RNA Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RNA Workbench) | | | | |

----

# Tools

## ToolShed Contributions

<div class='right'>[<img src="/src/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in April](/src/toolshed/contributions/2018-04/index.md).

# Releases

## BioBlend 0.11.0

[BioBlend](http://bioblend.readthedocs.org/) is a Python library for interacting with CloudMan and Galaxy‘s API.  BioBlend makes it possible to script and automate the process of cloud infrastructure provisioning and scaling via CloudMan, and running of analyses via Galaxy.

See the [release notes](https://github.com/galaxyproject/bioblend/releases) for what's new in release 0.11.0.

## Starforge 0.3.6

[<img class="float-right" src="/src/images/galaxy-logos/starforge_logo_big.png" alt="Starforge" width="180" />](https://github.com/galaxyproject/starforge/blob/master/README.rst)

[Starforge](https://pypi.python.org/pypi/starforge) is a collection of scripts that supports the building of components for Galaxy.  Specifically, with Starforge you can:

* Build [Galaxy Tool Shed](http://toolshed.g2.bx.psu.edu/) dependencies
* Build [Python Wheels](http://pythonwheels.com/) (e.g. for the [Galaxy Wheels Server](http://wheels.galaxyproject.org/))
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems. Documentation can be found at [starforge.readthedocs.org](http://starforge.readthedocs.org/). 

The release features Python 3-compatible `setup.py` wrapping and a fix for pip 10.


## Earlier Releases

Other packages released in the prior 4 months.

### 18.01 Galaxy Release

<div class="right">
<a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a><br /><br />
</div>

The [18.01 release of Galaxy  is out](https://docs.galaxyproject.org/en/release_18.01/releases/18.01_announce.html ). Thanks to the [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) and you, our community!

**Highlights:**

*Performance and User Experience Improvements*
We made Galaxy more lively and responsive. Homepage, published workflows, published/saved histories, and data libraries should all load much faster now. Importing data from FTP will also take less of your time. 

*Web Server and Configuration*
The default web server used by Galaxy has changed from Paste to uWSGI and the default configuration file for Galaxy is now config/galaxy.yml instead of config/galaxy.ini. To minimize the impact of this change on existing Galaxy instances, if a Galaxy has a galaxy.ini file configured, it will continue to use Paste by default unless additional steps are taken by the administrator

*Dataset Collection Usability*
This release has significantly improved the usability of Galaxy dataset collections. Dozens of improvements to collections have been made, some of the key highlights include:

* Data library folders can now be sent to histories as a dataset collection.
* Failed dataset collection elements can now be fixed using job re-running 
* Collections now appear with state and progress bars in the history panel and contained datasets are hidden by default
* We added intuitive workflow post job actions for dataset collections
* The web interface now supports collections with arbitrary nesting and size
* More robust nametag discovery and propagation when using collections

*Client Architecture*
The architecture for the client code that powers the Galaxy user interface has been significantly overhauled. The code base has been converted to ES6, Yarn now powers the build and dependency management of the code, Prettier is now used to ensure consistent code formatting, and the VueJS framework has been integrated.

*New BAM datatypes*
Previously Galaxy only supported coordinate sorted BAM files by default (the bam datatype). In addition, this release of Galaxy now supports three new types of BAM:

* qname_sorted.bam, that ensures that the file is queryname sorted (e.g. SO:queryname);
* qname_input_sorted.bam, that can be used to describe the output of aligners which generally keep mate pairs adjacent
* unsorted.bam, that makes no assumptions about the sort order of the file.

*Experimental Job Caching*
Galaxy can now be configured to allow users the option of skipping duplicated jobs if one with identical parameters has been previously executed and simply reuse the previously generated outputs. 


See the [full release notes](https://docs.galaxyproject.org/en/release_18.01/releases/18.01_announce.html) for more.

Thanks for using Galaxy!

### Galaxy Docker Image 18.01

The [Galaxy Docker project has a matching release](https://github.com/bgruening/docker-galaxy-stable/releases/tag/18.01), for Galaxy 18.01. The release features the 18.01 enhancement and removes `nodejs-legacy` and `npm` from the Dockerfile and installs the latest version from `ansible-extras`.

### blend4j 0.2.0

blend4j is a partial reimplementation of the Python library [bioblend](https://github.com/galaxyproject/bioblend) for the JVM. bioblend for Python is a library for scripting interactions with Galaxy.

* Update to galaxy-bootstrap 0.7.0.
* Added support for accessing Galaxy Tool Data tables. Thanks to [Dan Fornika](https://github.com/apetkau/blend4j/pull/9).
* blend4j now requires Java 1.8+ to run.

### galaxy-lib 18.5.3 - 18.5.7

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

### Planemo 0.48.0

[<img class="float-right" src="/src/images/galaxy-logos/planemo-logo.png" alt="Planemo" width="200" />](https://pypi.python.org/pypi/planemo/)

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

### Pulsar 0.8.1-3

[<img class="float-right" src="/src/images/galaxy-logos/pulsar_transparent.png" alt="Pulsar" width="170" />](https://pypi.python.org/pypi/pulsar-app/)

[Pulsar](https://pypi.python.org/pypi/pulsar-app/) updates were [released in February](https://github.com/galaxyproject/pulsar/blob/master/HISTORY.rst).  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.

## And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).




