---
title: "AmrPlusPlus"
url: "http://megares.meglab.org/amrplusplus/latest/html/"
scope: "tool-publishing"
platforms:
  - platform_group: "container"
    platform_url: "http://megares.meglab.org/amrplusplus/latest/html/"
    platform_text: "AmrPlusPlus Docker"
    platform_location: US
summary: "A Galaxy-based metagenomics pipeline that is intuitive and easy to use."
image: "/src/use/amrplusplus/amrplusplus-flowchart.png"
comments:
  - "AmrPlusPlus takes advantage of current and new tools to help identify and characterize resistance genes within metagenomic sequence data."
  - "The pipeline can be used under a local instance of Galaxy and installed via Galaxy's Main Tool Shed. It is also available as a Galaxy-based Docker Image, using base images developed by Björn Grüning at the University of Freiburg."
user_support:
  - "[Documentation](http://megares.meglab.org/amrplusplus/latest/html/)"
  - "[Email support](http://megares.meglab.org/amrplusplus/latest/html/contact.html)"
quotas:
  - "None"
citations:
  - "[MEGARes: an antimicrobial resistance database for high throughput sequencing](https://doi.org/10.1093/nar/gkw1009), Steven M. Lakin, Chris Dean, Noelle R. Noyes, Adam Dettenwanger, Anne Spencer Ross, Enrique Doster, Pablo Rovira, Zaid Abdo. Kenneth L. Jones, Jaime Ruiz, Keith E. Belk, Paul S. Morley, Christina Boucher. *Nucleic Acids Research*, Volume 45, Issue D1, 4 January 2017, Pages D574–D580, doi:10.1093/nar/gkw1009"
pub_libraries:
  - "AmrPlusPlus"
sponsors:
  - "[Colorado State University](https://www.colostate.edu/)"
---
