---
title: New Pre-Print "Selection analysis identifies unusual clustered mutational changes
  in Omicron lineage BA.1 that likely impact Spike function"
date: '2022-01-16'
tease: 'Among the 30 non-synonymous nucleotide substitutions in the Omicron S-gene are 13 that
  have only rarely been seen in other SARS-CoV-2 sequences.'
hide_tease: true
doi: 10.1101/2022.01.14.476382
tags: [paper, citations, covid-19, virology]
supporters:
- crc1425
- crc992
- denbi
- elixir
subsites: [eu, pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
main_subsite: eu
---

![Graphical Abstract](/assets/media/2022-01-16-covid-preprint.jpg)

Among the 30 non-synonymous nucleotide substitutions in the Omicron S-gene are 13 that have only rarely been seen in other SARS-CoV-2 sequences. These mutations cluster within three functionally important regions of the S-gene at sites that will likely impact (i) interactions between subunits of the Spike trimer and the predisposition of subunits to shift from down to up configurations, (ii) interactions of Spike with ACE2 receptors, and (iii) the priming of Spike for membrane fusion. We show here that, based on both the rarity of these 13 mutations in intrapatient sequencing reads and patterns of selection at the codon sites where the mutations occur in SARS-CoV-2 and related sarbecoviruses, prior to the emergence of Omicron the mutations would have been predicted to decrease the fitness of any genomes within which they occurred. We further propose that the mutations in each of the three clusters therefore cooperatively interact to both mitigate their individual fitness costs, and adaptively alter the function of Spike. Given the evident epidemic growth advantages of Omicron over all previously known SARS-CoV-2 lineages, it is crucial to determine both how such complex and highly adaptive mutation constellations were assembled within the Omicron S-gene, and why, despite unprecedented global genomic surveillance efforts, the early stages of this assembly process went completely undetected.

--------------------

Read more about our COVID-19 efforts in our [summary document](https://docs.google.com/document/d/1Ajfy3RG-BOGTSdhQUMZ0hnGtePZplv-lK0rYzx8L-vo/preview#) and on the [project page](/projects/covid19/).

