---
title: February 2018 Galaxy News
---
[<img class="float-right" src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 />](/src/galaxy-updates/index.md)

Welcome to the February 2018 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.

----

# Events

There are [a plenitude of Galaxy related events coming up](/src/events/index.md) in the next few months, including *large regional meetings in Europe and Africa.*

## GCCBOSC 2018 Training Topic Voting Closes Jan 31

[<img class="float-right" src="/src/images/logos/gcc-bosc-2018-logo-300.png" width="160" alt="GCCBOSC 2018: Vote for the Training Topics you want" />](/src/events/gccbosc2018/training/index.md)

**Which is tomorrow.  [Vote now](http://bit.ly/gccbosc2018-vote)**

[Your vote](http://bit.ly/gccbosc2018-vote) will determine the topics that are offered, which topics should be offered more than once,  and which ones should not be scheduled at the same time.  Your vote matters. 

[GCCBOSC2018](https://gccbosc2018.sched.com/) will be held 25-30 June in Portland, Oregon, United States. It will feature two days of training: the second of which is multi-track and will feature content for both the BOSC and Galaxy communities.  Workshops will be hands-on and participants will be strongly encouraged to bring a laptop and follow along.  If you work in data-intensive biomedical research, there is no better place than GCCBOSC 2018 to present your work and to learn from others.

*Thanks to everyone who nominated topics and helped write topic descriptions!*


*GCCBOSC2018 registration will open in February.*  Watch this space.

## GCCBOSC 2018 Sponsors

We are happy to have announced several [GCCBOSC 2018 sponsors](/src/events/gccbosc2018/sponsors/index.html) in January. This month we are highlighting *PeerJ*:

### PeerJ

[<img class="float-right" src="/src/events/gccbosc2018/sponsors/peerj-logo-400.png" alt-"PeerJ, the award-winning biological, medical and environmental sciences journal" width="120" />](https://peerj.com/)

***[PeerJ](https://peerj.com/)*** is proud to sponsor GCCBOSC 2018 and support the unified event for bioinformatics developers and practitioners. PeerJ's mission is to make Open Access affordable, fast, and easier for researchers and institutions, and central to this mission is our open source ethos. We publish two peer-reviewed journals, *[PeerJ](https://peerj.com/)* and *[PeerJ Computer Science](https://peerj.com/computer-science/)* and we also host a free *[PeerJ Preprints](https://peerj.com/preprints/)* server. *PeerJ* requires code to be open and accessible, hence readers can be assured that they can access the tools appropriately. We also provide [open source tools](https://github.com/PeerJ) helpful to both academics and general software development professionals.

GCCBOSC attendees may be interested to hear that, in addition to research articles, *PeerJ* welcomes articles describing bioinformatics software tools. These articles provide descriptions of new software tools (or significant new functionality in existing software) of interest to the bioinformatics and/or computational biology communities. You can browse the [PeerJ Bioinformatics Software Tools Collection](http://bit.ly/PeerJBioinfo). Submit your bioinformatics manuscript [here](https://peerj.com/manuscripts/start/).

*PeerJ* is also excited to offer APC discounts to all GCCBOSC attendees - we'll be in touch with more details soon!

<div class="float-right">
<br />
[<img src="/src/events/gccbosc2018/sponsors/summary-thumb.png" alt="Download the one page summary." />](https://depot.galaxyproject.org/hub/attachments/documents/gccbosc2018_sponsorship_summary_20170906.pdf) [<img src="/src/events/gccbosc2018/sponsors/prospectus-thumb.png" alt="Download the full prospectus." />](https://depot.galaxyproject.org/hub/attachments/documents/gccbosc2018_sponsorship_prospectus_20170906.pdf)  </div>

### Call for sponsors!

[Sponsors](/src/events/gccbosc2018/sponsors/index.html) are a key part of [GCCBOSC 2018](https://gccbosc2018.sched.com/).  Is your organization interested in playing a prominent role in the first joint gathering of the Galaxy and BOSC communities? Then [become a GCCBOSC 2018 sponsor](/src/events/gccbosc2018/sponsors/index.md#interested) and raise your organization's visibility in these active and engaged communities.


## ELIXIR Galaxy Community Kickoff and Meeting, March, Freiburg

<div class="float-left">
[<img src="/src/images/logos/ElixirNoTextLogo.png" alt="ELIXIR Galaxy Community Kickoff and Meeting" width="160" />](https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference) <br />
[<img src="/src/images/logos/embl-australia-logo.png" alt="EMBL-Australia" width="120" />](http://www.emblaustralia.org/)
</div>

The ELIXIR Galaxy community will have a [kickoff meeting](https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference) 14-16th March 2018, in Freiburg/Germany. We will combine this with a Galaxy User Conference and the **official launch of the usegalaxy.eu server.**

This event will also be coordinated with the [EMBL Australia](http://www.emblaustralia.org/) to also **launch usegalaxy.org.au at the same time.**

Wednesday (the 14th) will be dedicated to the ELIXIR Galaxy communtiy, to discuss and plan our roadmap in 2018 and 2019. Thursday and Friday (15th, 16th) will be a conference where Galaxy users talk about their research and use-cases. However, a few spots we will reserved for official talks from ELIXIR.

*[Register now](https://docs.google.com/forms/d/e/1FAIpQLSfqKqyPwW-s6m64ObleUSv4B8RbGxCRIddBY1cpt3Za7af6XQ/viewform) as space is limited.*

## Galaxy Africa, April, Cape Town

[<img class="float-right" src="/src/events/2018-04-galaxy-africa/galaxy-africa-logo.svg" alt="Galaxy Africa" width="170" />](http://galaxyafrica.sanbi.ac.za/)

[Galaxy Africa](http://galaxyafrica.sanbi.ac.za/) will be held 3-5 April in Cape Town, South Africa at [SANBI](http://www.sanbi.ac.za/) on the [Universtiy of the Western Cape](https://www.uwc.ac.za/Pages/default.aspx) campus.

This is an opportunity to learn from leading bioinformaticists, systems administrators and engineers about Galaxy and accessible, reproducible analysis of biological data.

* **Analyse**:  Learn about using Galaxy to analyse your genomic and RNA data.
* **Manage**:  Meet fellow admins of research computing infrastructure and hear about local and cloud hosting of Galaxy servers.
* **Extend**:  Hear how bioinformaticts are extending Galaxy through custom tools and workflows.

Our programme will start with keynote speakers introducing biology, data science and Galaxy. In the afternoon there will be tutorials on biological data analysis using Galaxy.  The Wednesday
morning session will include presentations on genomic and metagenomic data analysis workflows and uses of Galaxy. In the afternoon we will continue our tutorial series. The final day of the meeting will focus on the technology underlying Galaxy and biological data analysis with a focus on containerisation, software packaging for easier use and cloud deployment. Tutorials on this day will focus on similar topics.

**[Register now](https://www.quicket.co.za/events/39729-galaxy-africa-2018/)**.

## Galaxy Administrators Course Report

<div class="float-right">
<img src="/src/events/2018-01-admin-oslo/oslo-admin-class.jpg" height="120" /> &nbsp;
[<img  src="/src/events/2018-01-admin-oslo/admins-oslo-2018-logo.png" alt="Galaxy Administrators Course" height="120" />](https://github.com/galaxyproject/dagobah-training)
</div>

A very successful and inspiring Galaxy admin workshop was held in Oslo in January, Thanks to 33 highly motivated participants and Nicola Soranzo, Marius Van Den Beek, Abdulrahman Azab, [Björn Grüning](/src/people/bjoern-gruening/index.md) and [Enis Afgan](/src/people/enis-afgan/index.md) as instructors!  Thanks also to [ELIXIR](https://www.elixir-europe.org/), [de.NBI](https://www.denbi.de/), and [ELIXIR Norway](https://www.elixir-europe.org/about-us/who-we-are/nodes/norway) for sponsoring this event.

Plenty of materials for Galaxy administration are now [up to date and available online](https://github.com/galaxyproject/dagobah-training).  The workshop included a sneak [preview of what's coming in the 18.01 release](https://galaxyproject.github.io/dagobah-training/2018-oslo/whatsnew/18.01.html#1) from [Nate Coraor](/src/people/nate/index.md).


# Select Upcoming events

<table class="table table-striped">
<thead>
<tr>
  <th>Date</th>
  <th>Topic/Event</th>
  <th>Venue/Location</th>
  <th>Contact</th>
</tr>
</thead>
<tbody>
<tr>
  <td><span class="text-nowrap">February 1st 2018</span></td>
  <td><a href="/events/2018-02-lorne-proteomics">Galaxy-P: an accessible resource for multi-omics analysis</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="http://www.australasianproteomics.org/lorne-proteomics-symposium-2018/">Lorne Proteomics Symposium, Lorne, Victoria, Australia</a></td>
  <td>Maria Doyle</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 6th 2018</span></td>
  <td><a href="https://faes.org/events/biotech-45-bioinformatic-analysis-next-generation-sequencing-data-4-day-hands-workshop-1">Bioinformatic Analysis of Next Generation Sequencing Data</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://events.r20.constantcontact.com/register/eventReg?llr=k4uamblab&amp;oeidk=a07ee6s3vec288242fe">National Institutes of Health (NIH), Bethesda, Maryland, United States</a></td>
  <td>training@faes.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 7th 2018</span></td>
  <td><a href="http://www.cpsa-europe.com/2018/program.shtml">An Introduction to Integrative Bioinformatics and Novel Tools in Multi-Omics</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.cpsa-europe.com/2018/index.shtml">CPSA Europe, Cambridge, United Kingdom</a></td>
  <td>Stephan Kang</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 12th 2018</span></td>
  <td><a href="http://www.triconference.com/Converged-IT-and-the-Cloud/#Day1">Supporting Highly Scalable Scientific Data Analysis with Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="http://www.triconference.com/">Converged IT &amp; the Cloud, San Francisco, California, United States</a></td>
  <td>James Taylor</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 17th 2018</span></td>
  <td><a href="http://opensource101.com/raleigh/talks/open-source-breakthroughs-life-sciences-precision-medicine/">Open Source Breakthroughs in Life Sciences and Precision Medicine</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="http://opensource101.com/raleigh/">Open Source 101 - Raleigh, North Carolina, United States</a></td>
  <td>Stephan Kang</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/RNA_Sequencing_2018.pdf">RNA Sequencing and Differential Expression</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">February 26th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/460-galaxy5">5th Galaxy high-throughput sequencing (HTS) data analysis workshop</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>University of Freiburg, Freiburg Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Freiburg Galaxy Team</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 7th 2018</span></td>
  <td><a href="http://www.denbi.de/22-training-cat/training-courses/469-introduction-to-rna-seq-data-analysis-with-galaxy">Introduction to RNA-Seq data analysis with Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/>Am Botanischen Garten, Kiel, Germany</td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Andrea Bagnacani, Markus Wolfien</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 14th 2018</span></td>
  <td><a href="https://www.elixir-europe.org/events/galaxy-community-kickoff-meeting-and-galaxy-user-conference">Galaxy community kickoff meeting and Galaxy User Conference</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="https://www.openstreetmap.org/?mlat=48.00189&amp;mlon=7.84628#map=17/48.00189/7.84628">Freiburg, Germany</a></td>
  <td>ELIXIR Galaxy Working Group</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/rna-seq-dge-analysis-galaxy-gvl/#more-1720">RNA-Seq Differential Gene Expression Analysis using Galaxy and the GVL</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 20th 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/intro-galaxy-gvl/">Introduction to Galaxy and the GVL</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.melbournebioinformatics.org.au/training-events/variant-calling-galaxy-gvl/#more-1012">Variant Calling with Galaxy &amp; the GVL for beginners</a></td>
  <td><img style="float:right" src="/images/icons/AU.png"/><a href="https://www.melbournebioinformatics.org.au/training-and-events/">Melbourne Bioinformatics, Carlton, Victoria, Australia</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Christina Hall</td>
</tr>
<tr>
  <td><span class="text-nowrap">March 21st 2018</span></td>
  <td><a href="https://www.abdn.ac.uk/genomics/documents/201718_Workshops/Alignment_2018.pdf">Alignment, Visualisation and Variant Calling</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://www.abdn.ac.uk/genomics/bioinformatics/training/">University of Aberdeen, Scotland, United Kingdom</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Sophie Shaw</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 3rd 2018</span></td>
  <td><a href="http://galaxyafrica.sanbi.ac.za/">Galaxy Africa</a></td>
  <td><img style="float:right" src="/images/icons/AF.png"/><a href="http://www.sanbi.ac.za/">SANBI, University of the Western Cape, Cape Town, South Africa</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>galaxyafrica@sanbi.ac.za</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 10th 2018</span></td>
  <td><a href="http://formationcontinue.agrocampus-ouest.fr/infoglueDeliverLive/toutes-sessions/programme?idModule=733&amp;nomModule=Galaxy--Traitement-de-donnees-de-sequences-par-Galaxy-module-7/8">Galaxy : Traitement de données de séquences par Galaxy</a></td>
  <td><img style="float:right" src="/images/icons/EU.png"/><a href="http://formationcontinue.agrocampus-ouest.fr/infoglueDeliverLive/zoom?contentId=12699">AGROCAMPUS OUEST, Campus de Rennes, France</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Yvan Le Bras, Sandrine LAGARRIGUE, formco@agrocampus-ouest.fr</td>
</tr>
<tr>
  <td><span class="text-nowrap">April 22nd 2018</span></td>
  <td><a href="http://conf.abrf.org/sw2-galaxy-platform-accessible-core-laboratory-solution-proteogenomic-analysis-and-informatics">The Galaxy Platform as an Accessible, Core Laboratory Solution for Proteogenomic Analysis and Informatics</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="http://conf.abrf.org/">ABRF 2018, Myrtle Beach, South Carolina, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Pratik Jagtap</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 14th 2018</span></td>
  <td><a href="https://sciencegateways.org/engage/bootcamp">Science Gateways Bootcamp</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.tacc.utexas.edu/">TACC, Austin, Texas, United States</a></td>
  <td>help@sciencegateways.org</td>
</tr>
<tr>
  <td><span class="text-nowrap">May 21st 2018</span></td>
  <td><a href="https://www.jax.org/education-and-learning/education-calendar/2018/may/big-genomic-data-skills-training-for-professors">Big Genomics Data Skills Training course</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="https://www.jax.org/about-us/locations/farmington">JAX Genomic Medicine, Farmington, Connecticut, United States</a></td>
  <td>Charlie Wray</td>
</tr>
<tr>
  <td><span class="text-nowrap">June 25th 2018</span></td>
  <td><a href="https://gccbosc2018.sched.com">GCC2018 &amp; BOSC 2018</a></td>
  <td><img style="float:right" src="/images/icons/NA.png"/><a href="/events/gccbosc2018/venue/">Reed College, Portland, Oregon, United States</a></td>
  <td><a href="/teach/gtn/"><img style="float:right" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Organizers</td>
</tr>
</tbody>
</table>

----

# New Galactic Blog Entry: Galaxy R Markdown Tools

<div class="float-right" >
[<img src="/src/images/galaxy-logos/galactic-blog-400.png" alt="Galactic Blog" width="200" />](/src/blog/index.md)
</div>

January saw one new *[Galactic Blog](/src/blog/index.md)* entry:

* [Galaxy R Markdown Tools](https://github.com/statonlab/galaxy-r-markdown-tools), by Ming Chen


# Galaxy Cloud News

January was a big month for Galaxy in the cloud:

* [Galaxy Cloud embraces the GVL](/src/news/2018-01-22-gvl430/index.md)
* [All-new CloudLaunch release](/src/news/2018-01-05-cloudlaunch/index.md)

# Publications

**111 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy Publication Library](https://www.zotero.org/groups/galaxy) in January.

## Highlighted Publications

The *Galactic* and *Stellar* publications added in January were:

* [META-pipe cloud setup and execution](https://f1000research.com/articles/6-2060/v2), Agafonov A, Mattila K, Tuan CD *et al.* *F1000Research* 2018, 6(ELIXIR):2060 (doi: 10.12688/f1000research.13204.2)
* [Precise temporal regulation of alternative splicing during neural development](https://www.biorxiv.org/content/early/2018/01/14/247601), Sebastien M Weyn-Vanhentenryck, Huijuan Feng, Dmytro Ustianenko, Rachel Duffie, Qinghong Yan, Martin Jacko, Jose C Martinez, Marianne Goodwin, Xuegong Zhang, Ulrich Hengst, Stavros Lomvardas, Maurice S Swanson, Chaolin Zhang. *bioRxiv* 247601; doi: 10.1101/247601
* [A-GAME: improving the assembly of pooled functional metagenomics sequence data](https://doi.org/10.1186/s12864-017-4369-z), Matteo Chiara, Antonio Placido, Ernesto Picardi, Luigi Ruggiero Ceci, David Stephen Horner and Graziano Pesole, *BMC Genomics* 2018 19:44 doi: 10.1186/s12864-017-4369-z


## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 67 | [+Methods](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Methods) | | 30 | [+Workbench](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Workbench) | | 22 | [+UsePublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UsePublic) | | 10 | [+UseMain](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseMain) | 
| 8 | [+UseLocal](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+UseLocal) | | 4 | [+Reproducibility](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Reproducibility) | | 4 | [+IsGalaxy](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+IsGalaxy) | | 3 | [+RefPublic](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+RefPublic) | 
| 3 | [+Tools](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Tools) | | 3 | [+Shared](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Shared) | | 2 | [+Other](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Other) | | 1 | [+Visualization](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Visualization) | 
| 1 | [+HowTo](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+HowTo) | | 1 | [+Unknown](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/+Unknown) | | | | | | | |

----
# Who's Hiring

<div class='right'><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* The [Freiburg Galaxy Team](https://usegalaxy-eu.github.io/galaxy-freiburg/people/) is [looking for 2 Postdoctoral researchers](https://usegalaxy-eu.github.io/galaxy-freiburg/2018/01/18/openpositions.html)
* The [Blankenberg](/src/people/dan/index.md) Lab in the [Genomic Medicine Institute](http://www.lerner.ccf.org/gmi/) at the [Cleveland Clinic Lerner Research Institute](https://www.lerner.ccf.org/) is [hiring postdocs](/src/news/2017-09-hiring-at-cc/index.md).
* [Galaxy Project is hiring software engineers and postdocs at Johns Hopkins](/src/news/2017-08-hiring-at-hopkins/index.md), Baltimore, Maryland, United States

Have a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them last month, including two new public servers:

[<img class="float-right" src="/src/use/a-game/agame-logo.png" alt="A-Game" width="150" />](http://beaconlab.it/agame)

## A-Game

[A-Game](http://beaconlab.it/agame) is *A Galaxy suite for tArgeted MEtagenomics*, a web service incorporating state of the art tools and workflows for the analysis of eDNA sequence data. A [site manual](http://159.149.160.68/galaxy/static/manual/) and [example workflows](http://159.149.160.68/galaxy/workflow/list_published) are available.

A-Game is supported by [BEACON](https://bioscienzebio.unimi.it/gruppo.php?id=26) at the [Dipartimento di Bioscienze](https://bioscienzebio.unimi.it/), [University of Milan](http://www.unimi.it/ENG/)


## Splicescope

[<img class="float-left" src="/src/use/splicescope/zhang-lab.png" alt="Splicescope" width="240" />](http://galaxy.splicebase.net/)

[Splicescope](http://galaxy.splicebase.net/) is a tool-publishing server for predicting neuronal maturation using splicing profiles. Splicescope takes either an exon inclusion ratio matrix file or junction bed files from multiple samples as input and outputs a zip file including prediction results, PCA analysis results and a PCA plot based on reference samples and an html file to summarize the prediction.

Splicescope can be accessed anonymously, or with an account, and anyone can create an account.  It is supported by the [Zhang Laboratory](https://zhanglab.c2b2.columbia.edu/index.php/Main_Page), Department of Systems Biology, the Department of Biochemistry and Molecular Biophysics, and the Motor Neuron Center at Columbia University Medical Center.

## New video from the Galaxy-P team

* [Community-driven data analysis training for Metaproteomics](https://www.youtube.com/watch?v=Vywt_-D5Kek)

## Public Servers in Publications

We tag papers that use, mention, implement or extend public Galaxy Servers.  Here are the counts for January's publications.

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 12 | [>Huttenhower](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Huttenhower) | | 3 | [>Workflow4Metabolomics](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Workflow4Metabolomics) | | 1 | [>GVL-Unspecified](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GVL-Unspecified) | | 1 | [>Galaxy-P](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Galaxy-P) | 
| 1 | [>Osiris](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Osiris) | | 1 | [>PreSTIGE](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PreSTIGE) | | 1 | [>PIA](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>PIA) | | 1 | [>Splicescope](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Splicescope) | 
| 1 | [>A-Game](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>A-Game) | | 1 | [>Cistrome](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Cistrome) | | 1 | [>Orione](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>Orione) | | 1 | [>GVL-QLD](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>GVL-QLD) | 
| 1 | [>RepeatExplorer](https://www.zotero.org/groups/1732893/galaxy/items/order/dateAdded/sort/desc/tag/>RepeatExplorer) | | | | | | | | | | |


----

# Tools

## ToolShed Contributions

<div class='right'>[<img src="/src/images/galaxy-logos/galaxy-toolshed-300.png" alt="Galaxy ToolShed" width="150"  />](http://toolshed.g2.bx.psu.edu/)</div>

[Tool Shed contributions in January](/src/toolshed/contributions/2018-01/index.md).

# Releases

## Earlier Releases

### 17.09 Galaxy Release

<div class="right">
<a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a><br /><br />
</div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.09 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html)** at the end of October.

Highlights include

* **Singularity** Tool execution using the HPC-friendly container technology Singularity is now supported. 
* **Download entire collection** Downloading whole collections is now possible from the history interface. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Switch tool versions in workflows** You can now select exactly what version of tool you want to use when building workflows. (Thanks to [@mvdbeek](https://github.com/mvdbeek).)
* **Security patches**
* Several features were **deprecated**

See the [17.09 release announcement](https://docs.galaxyproject.org/en/master/releases/17.09_announce.html) for details.

### Galaxy Docker Image 17.09

The [Galaxy Docker project has seen a matching release](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.09), for Galaxy 17.05. Major features include
* much improved documentation about using Galaxy Docker and an external cluster ([@rhpvorderman](https://github.com/rhpvorderman))
* CVMFS support - mounting in 4TB of pre-build reference data ([@chambm](https://github.com/chambm))
* Singularity support and tests (compose only)
* more work on K8s support and testing ([@jmchilton](https://github.com/jmchilton))
* using .env files to configure the compose setup for SLURM, Condor, K8s, SLURM-Singularity, Condor-Docker

And

* The [Galaxy Docker Project](https://github.com/bgruening/docker-galaxy-stable) has reached more than 31k downloads on [Dockerhub](https://hub.docker.com/r/bgruening/galaxy-stable/) - not counting quay.io and all flavors 

### ephemeris 0.8.0

Ephemeris is a small Python library and set of scripts for managing the bootstrapping of Galaxy plugins - tools, index data, and workflows.

* Free software: Academic Free License version 3.0
* Documentation: https://ephemeris.readthedocs.org.
* Code: https://github.com/galaxyproject/ephemeris

### blend4php 0.1 beta

[<img class='float-right' src="/src/images/galaxy-logos/blend4phpLogo.png" alt="blend4php" width="220" />](https://github.com/galaxyproject/blend4php)

The beta version of the [blend4php package](https://github.com/galaxyproject/blend4php), a PHP wrapper for the [Galaxy API](https://docs.galaxyproject.org/en/master/api_doc.html), was released in December. It provides a PHP package for interacting with Galaxy and CloudMan. blend4php currently offers a partial implementation of the Galaxy API and includes support for datasets, data types, folder contents, folders, genomes, group roles, groups, group users, histories, history contents, jobs, libraries, library contents, requests, roles, search, tools, toolshed repositories, users, visualizations and workflows.

The motivation for development of this library is for integration with [Tripal](http://tripal.info/), an open-source toolkit for creation of online genomic, genetic and biological databases.

Please see the [API documentation](http://galaxyproject.github.io/blend4php/) page for full information.

### galaxy-lib 17.9.10

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

This revision:

* Added docs for using mulled-build with your own quay.io account (thanks to @jerowe).
* Catch errors in Conda search if nothing is found (preventing planemo-monitor from functioning properly) (thanks to @bgruening).
* Make multi-requirement container building via mulled more stable (thanks to @bgruening).

### Planemo 0.47.0

[<img class="float-right" src="/src/images/galaxy-logos/planemo-logo.png" alt="Planemo" width="200" />](https://pypi.python.org/pypi/planemo/)

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).


Other packages that have been released in the prior 4 months.

### sequence_utils 1.1.2

Galaxy's [sequence utilities](https://github.com/galaxyproject/sequence_utils) are a set of Python modules for reading, analyzing, and converting sequence formats.  See the [release notes](https://github.com/galaxyproject/sequence_utils/blob/master/HISTORY.rst) for what's new this month.

### StarForge 0.3.1-5

[StarForge](https://github.com/galaxyproject/starforge) help build Galaxy things in virtualization:

* Build Galaxy Tool Shed dependencies
* Build Python Wheels (e.g. for the Galaxy Wheels Server)
* Rebuild Debian or Ubuntu source packages (for modifications)

These things will be built in Docker. Additionally, wheels can be built in QEMU/KVM virtualized systems.  StarForge has had several updates this fall.  Fixes and new features include:

* Support xz/lzma tarballs for wheel builds Pull Request 166
* Native support for auditwheel and delocate. (#160)
* Do not build sdists with the wheel subcommand by default. (#155)
* Fix a bug where the wrong working directory was set when building wheels with multiple sources. (#154)
* Fix a bug with sudo and brew install on macOS. (#151).
* Short circuit platform caching on OS X (#150).

## And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

# Other News

* The semi-annual update of the project statistics page is done.  [See what's new](/src/galaxy-project/statistics/index.md).
* From [John Chilton](/src/people/john-chilton/index.md):
  * Galaxy can now [by-pass re-running jobs with duplicate parameters/inputs](https://github.com/galaxyproject/galaxy/pull/4690) thanks to the brilliant [Marius Van Den Beek](https://twitter.com/mariusvdbeek/) 
