---
title: June 2017 Galaxy News
---
<div class='right'><a href='/src/galaxy-updates/index.md'><img src="/src/images/galaxy-logos/GalaxyNews.png" alt="Galaxy News" width=150 /></a></div>

Welcome to the June 2017 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community.  If you have anything to add to next month's newsletter, then please send it to outreach@galaxyproject.org.


# GCC2017 is almost here!

<div class="right"><a href="https://gcc2017.sciencesconf.org/"><img src="/src/images/logos/gcc2017-logo-300.png" alt='2017 Galaxy Community Conference (GCC2017)' width="220" /></a></div>

GCC2017 will be in Montpellier, France, 26-30 June and will feature two days of presentations, discussions, poster sessions, lightning talks, computer demos, keynotes, and birds-of-a-feather meetups, **all about data-intensive biology and the tools and infrastructure that support it.** GCC2017 also includes data and coding hacks, and **[two days of training](https://gcc2017.sched.com/overview/type/Training)** covering 16 different topics.
 
GCC2017 will be held at [Le Corum Conference Centre](http://www.montpellier-events.com/en/The-Corum/Presentation) in the heart of [Montpellier](https://gcc2017.sciencesconf.org/page/location), just 10km from the Mediterranean. This event will gather several hundred researchers addressing diverse questions and facing common challenges in data intensive life science research. GCC participants work across the tree of life, come from around the world, and work at universities, research organizations, industry, medical schools and research hospitals.

## Regular Registration Rates Extended to June 16

If you missed the May 31 regular registration deadline then don't worry because it has been extended to June 16.  **[Regular registration](https://gcc2017.sciencesconf.org/page/registration)** starts at 75€ / day for post-docs and students.  You can also [book low cost conference housing](https://gcc2017.sciencesconf.org/page/lodging) when you register.

## Keynotes

### Eytan Domany

**Personalized Analysis of  Cancer Data: From Genes to Pathways (and Back)**

[Dr. Eytan Domany](http://www.weizmann.ac.il/complex/compphys/) will cover his work in human genomics and computational biology principally in the cancer domain. Eytan Domany is a Professor of Computational Systems Biology in the Department of Physics and Complex Systems at the [Weizmann Institute of Science](http://www.weizmann.ac.il/) in Rehovot, Israel. The main focus of his group is to mine data from large-scale experiments in biology. The work includes development of mathematical methods, their implementation in algorithms (which are incorporated in user-friendly computational tools), which are then applied to study biological data. Eytan was awarded the [Sergio Lombroso Award in Cancer Research](http://www.weizmann.ac.il/pages/prof-eytan-domany) in 2016 for his research efforts.

 
### Anton Nekrutenko & James Taylor

**The Galaxy Project: from the Planck epoch to Montpellier**

[Dr. Anton Nekrutenko](/src/people/anton/index.md) and [Dr. James Taylor](/src/people/james-taylor/index.md) will present a keynote reviewing the history of the Galaxy project, and how it went from a humble Perl script to having its own international meeting in the South of France in the summer of 2017.

If you are reading this, you probably already know who they are.  If not, see their [bios here](https://gcc2017.sciencesconf.org/).

## And the rest of the schedule ...

... will be published this week on the [conference schedule web site](https://gcc2017.sched.com/).

## Submit a Lightning Talk!

<div class="left"><a href="https://gcc2017.sciencesconf.org/page/submit"><img src="/src/galaxy-updates/2017-06/lightning.png" alt='Stike out and strut your work with a GCC2017 lightning talk!' width="60" /></a></div>

Lightning talks are an excellent chance to share your work, preliminary or otherwise, with the community.  Lightning talks are 7 minutes long and are the quickest way to make a lasting impression on, and get feedback from all participants.  Submit your lightning talk proposal [by June 23](https://gcc2017.sciencesconf.org/page/submit). 


### Talk, Poster and Demo Submission is Closed, but ...

[Abstract submission](https://gcc2017.sciencesconf.org/page/submit) for regualr talks, posters and demos closed in May.  However you can still submit late oral prentations abstracts, posters and demos.  Late oral presentation abstracts are considered as cancellations occur, and posters and demos will be reviewed until the space is full.

<div class="right"><a href="https://gcc2017.sciencesconf.org/page/sponsors"><img src="/src/events/2017-gcc/sponsor-cloud.png" alt='2017 Galaxy Community Conference Sponsors' width="180" /></a></div>

## GCC2017 Sponsors

GCC2017 has [11 sponsors](https://gcc2017.sciencesconf.org/page/sponsors)!  Sponsors are potential partners in your research and sponsorships greatly lower the cost of conference registration.  Have a look at the [GCC2017 sponsor list](https://gcc2017.sciencesconf.org/page/sponsors) and plan on learning what they have to offer (and thanking them) while you are in Monpellier.

## Hack the Universe: Capture the Flag at GCC2017!

<div class="center"><a href="https://ctf.galaxians.org/"><img src="https://depot.galaxyproject.org/hub/attachments/events/2017-gcc/hack-the-universe.gif" alt='Capture the flag at GCC2017!' /></a>

<br />
(Quite possibly the first animated GIF used in a Galaxy Newsletter.)
</div>


## GCC2017 Call for BoFs!

<div class="left"><a href="https://docs.google.com/forms/d/e/1FAIpQLSdRCs97OWt47qmMxWZAtSV3TTLJ24IhW0m2lHGPLiqJU0Xkzw/viewform"><img src="/src/events/2017-gcc/gcc2017-bofs.png" alt='2017 Galaxy Community Conference BoFs' width="170" /></a></div>

There is no better place than a Galaxy Community Conference to meet and learn from others doing data-intensive biology.  [GCC2017](https://gcc2017.sciencesconf.org/) continues this tradition by again including *[Birds of a Feather (BoF)](http://en.wikipedia.org/wiki/Birds_of_a_feather_%28computing%29)* meetups.  Birds of a Feather meetups are informal gatherings where participants group together based on common interests.

Interested?
* [Start a BoF](/src/news/2017-05-call-for-bofs/index.md)
* [See already proposed BoFs](https://gcc2017.sched.com/overview/type/Conference/BoF)

# Galaxy Tutorial @ ISMB/ECCB 2017

<div class="right"><a href="https://www.iscb.org/ismbeccb2017-program/tutorials#p6"><img src="/src/images/logos/ismb-2017.png" alt="Galaxy Tutorial at ISMB/ECCB 2017: Making Galaxy Work for You" width="170" /></div>

Know someone who needs to learn more about deploying Galaxy in their organization? There will be a half day [Making Galaxy Work for You](https://www.iscb.org/ismbeccb2017-program/tutorials#p6) tutorial at [ISMB/ECCB 2017](https://www.iscb.org/ismbeccb2017) in Prague, Czechia. The tutorial will provide a practical, hands-on guide to adapting the Galaxy platform to the specific needs of individuals attending the ISMB.

Participants will learn to

- Create Galaxy compatible tool and workflow definitions that are publicly accessible and that make it easy for any instance administrator to add your work to their server. 
- Deploy Galaxy and scale it up to target production-ready resources such as a Postgres database, NGINX webserver, and distributed resource managers such SLURM or PBS.

Interested? Sign up when you [register](https://www.iscb.org/ismbeccb2017-registration#tutorials) for ISMB/ECCB (and register before the tutorial fills up).

We also expect a lot of other Galaxy-related content at ISMB/ECCB 2017 and [BOSC 2017](https://www.open-bio.org/wiki/BOSC_2017). We'll add the presentations to the [Galaxy @ ISMB/ECCB 2017 & BOSC 2017](/src/events/2017-ismb-bosc/index.md) event page as the schedules are posted.  (And, [please let us know](mailto:outreach@galaxyproject.org) if you have anything on the program.)

See you in Prague!



# All events

There are [a plenitude of Galaxy related events coming up](/src/events/index.md) in the next few months:

<table class="table table-striped">
<thead>
  <tr>
    <th>Date</th>
    <th>Topic/Event</th>
    <th>Venue/Location</th>
    <th>Contact</th>
  </tr></thead><tbody>
  <tr>
    <td><span class="text-nowrap">June 20th 2017</span></td>
    <td><a href="/news/2017-03-g-on-ramp/">G-OnRamp Beta Testers Workshop</a></td>
    <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">June 26th 2017</span></td>
    <td><a href="https://gcc2017.sciencesconf.org/">2017 Galaxy Community Conference (GCC2017)</a> [<a href="/events/2017-gcc">hub page</a>]</td>
    <td><img style="float:right;" src="/images/icons/EU.png"/>Montpellier, Fance</td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Galaxy community</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">June 29th 2017</span></td>
    <td><a href="http://www.denbi.de/22-training-cat/training-courses/353-bimsb-galaxy-course-using-galaxy-for-rna-seq-analysis">Using Galaxy for RNA-seq analysis</a></td>
    <td><img style="float:right;" src="/images/icons/EU.png"/>MDC, Berlin, Germany</td>
    <td>Dilmurat Yusuf, Bora Uyar</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">July 2nd 2017</span></td>
    <td><a href="http://www.smbe2017.org/symposia/">Computation and reproducibility in molecular evolution</a></td>
    <td><img style="float:right;" src="/images/icons/NA.png"/><a href="http://www.smbe2017.org/">SMBE 2017, Austin, Texas, United States</a></td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Anton Nekrutenko, Sergei Kosakovsky Pond</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">July 6th 2017</span></td>
    <td><a href="https://tild.sciencesconf.org/">Stratégies D'Analyse Données Omiques - Ateliers Introductifs Galaxy et Cytopscape</a></td>
    <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.univ-tours.fr/">Université François-Rabelais, Tours, France</a></td>
    <td>Franziska Liesecke</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">July 20th 2017</span></td>
    <td><a href="https://www.open-bio.org/wiki/Codefest_2017">OpenBio Codefest 2017</a></td>
    <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://brmlab.cz/">Brmlab, Prague, Czechia</a></td>
    <td>Brad Chapman, Matúš Kalaš, Heather Wiencko</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">July 21st 2017</span></td>
    <td><a href="/events/2017-ismb-bosc">Galaxy @ ISMB/ECCB 2017 &amp; BOSC 2017 </a></td>
    <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.iscb.org/ismbeccb2017">Prague, Czechia</a></td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Presenters</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">July 25th 2017</span></td>
    <td><a href="/news/2017-03-g-on-ramp/">G-OnRamp Beta Testers Workshop</a></td>
    <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://wustl.edu/">Washington University in St. Louis, Missouri, United States</a></td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Jeremy Goecks, Sarah Elgin</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">September 1st 2017</span></td>
    <td><a href="http://www.earlham.ac.uk/genome-10k-and-genome-science">The Galaxy Bioinformatics Platform: learning to use Galaxy for RNA-seq analysis</a> [<a href="/events/2017-genome-10k">hub page</a>]</td>
    <td><img style="float:right;" src="/images/icons/EU.png"/><a href="http://www.earlham.ac.uk/">Genome 10K and Genome Science, Earlham Institute, Norwich, United Kingdom</a></td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>Nicola Soranzo, Graham Etherington</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">September 25th 2017</span></td>
    <td><a href="http://compgen.mdc-berlin.de/">Computational Genomics and RNA Biology</a></td>
    <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.mdc-berlin.de/bimsb">Berlin Institute for Medical Systems Biology, Max Delbrück Center, Berlin, Germany</a></td>
    <td><a href="/teach/gtn/"><img style="float:right;" alt="Training offered by GTN Member" src="/images/galaxy-logos/GTN16.png" title="Training offered by GTN Member"/></a>compgen@mdc-berlin.de</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">October 23rd 2017</span></td>
    <td><a href="http://sciencegateways.org/gateways2017/">Gateways 2017</a></td>
    <td><img style="float:right;" src="/images/icons/NA.png"/><a href="https://www.umich.edu/">University of Michigan, Ann Arbor, Michigan, United States</a></td>
    <td>Enis Afgan</td>
  </tr>
  <tr>
    <td><span class="text-nowrap">October 23rd 2017</span></td>
    <td><a href="http://cnrsformation.cnrs.fr/stage-17012-Analyse-avancee-de-sequences.html?axe=65">Analyse avancée de séquences</a></td>
    <td><img style="float:right;" src="/images/icons/EU.png"/><a href="https://www.cbib.u-bordeaux.fr/">Carreire de l'Université Bordeaux Segalen, Bordeaux, France</a></td>
    <td>Aurélien Barré, Raluca Uricaru, Benjamin Dartigues</td>
  </tr>
  </tbody></table>

See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

## Training Material Re-use Hackathon Reports

<div class="right"><a href="https://github.com/galaxyproject/training-material"><img src="/src/images/news-graphics/gtn-materials-overview-2017-05.png" alt="GTN materials overview" width="220" /></div>

The ELIXIR/GOBLET/GTN hackathon for Galaxy training material re-use was held last month in Cambridge, UK.  A lot was accomplished:

* [Event report at de.NBI (written by Maria Doyle](http://www.denbi.de/events/358-review-of-elixir-goblet-gtn-hackathon-supported-by-de-nbi)
* [Simon Gladman's UK trip report](https://www.melbournebioinformatics.org.au/projects-blog/uk-tour-2017/)


----

# New Publications

**85 new publications** referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in May.

Some highlights from the papers added last month:

* [GSuite HyperBrowser: integrative analysis of dataset collections across the genome and epigenome](https://doi.org/10.1093/gigascience/gix032)
   Boris Simovski, Daniel Vodák, Sveinung Gundersen, Diana Domanska Abdulrahman Azab, Lars Holden, Marit Holden, Ivar Grytten, Knut Rand, Finn Drabløs Morten Johansen, Antonio Mora, Christin Lund-Andersen, Bastian Fromm Ragnhild Eskeland, Odd Stokke Gabrielsen, Egil Ferkingstad, Sigve Nakken Mads Bengtsen, Alexander Johan Nederbragt, Hildur Sif Thorarensen Johannes Andreas Akse, Ingrid Glad, Eivind Hovig, Geir Kjetil Sandve. *GigaScience* (27 April 2017), doi:10.1093/gigascience/gix032

* [Coherent Application Delivery on Hybrid Distributed Computing Infrastructures of Virtual Machines and Docker Containers](https://doi.org/10.1109/pdp.2017.29)
   German Molto, Miguel Caballer, Alfonso Perez, Carlos Alfonso, Ignacio Blanquer. In 2017 25th Euromicro International Conference on Parallel, Distributed and Network-based Processing (PDP) (2017), pp. 486-490, doi:10.1109/pdp.2017.29

* [ChimeRScope: a novel alignment-free algorithm for fusion transcript prediction using paired-end RNA-Seq data](https://doi.org/10.1093/nar/gkx315)
   You Li, Tayla B. Heavican, Neetha N. Vellichirammal, Javeed Iqbal, Chittibabu Guda
*Nucleic Acids Research* (02 May 2017), doi:10.1093/nar/gkx315

* [Jupyter and Galaxy: Easing entry barriers into complex data analyses for biomedical researchers](https://doi.org/10.1371/journal.pcbi.1005425)
   [Björn A. Grüning](/src/people/bjoern-gruening/index.md), [E. Rasche](/src/people/helena-rasche/index.md), Boris Rebolledo-Jaramillo, [Carl Eberhard](/src/people/carl-eberhard/index.md), Torsten Houwaart, [John Chilton](/src/people/john-chilton/index.md), [Nate Coraor](/src/people/nate/index.md), Rolf Backofen, [James Taylor](/src/people/james-taylor/index.md), [Anton Nekrutenko](/src/people/anton/index.md). *PLOS Computational Biology*, Vol. 13, No. 5. (25 May 2017), e1005425, doi:10.1371/journal.pcbi.1005425

* [MAGenTA: a Galaxy implemented tool for complete Tn-Seq analysis and data visualization](https://doi.org/10.1093/bioinformatics/btx320)
   Katherine M. McCoy, Margaret L. Antonio, Tim van Opijnen. *Bioinformatics* (11 May 2017), doi:10.1093/bioinformatics/btx320 

## Publication Topics

| # | Tag | | # | Tag | | # | Tag | | # | Tag | 
| ---: | --- | --- | ---: | --- | --- | ---: | --- | --- | ---: | --- |
| 49 | [methods](http://www.citeulike.org/group/16008/tag/methods) | | 21 | [usepublic](http://www.citeulike.org/group/16008/tag/usepublic) | | 16 | [usemain](http://www.citeulike.org/group/16008/tag/usemain) | | 15 | [workbench](http://www.citeulike.org/group/16008/tag/workbench) | 
| 10 | [refpublic](http://www.citeulike.org/group/16008/tag/refpublic) | | 5 | [tools](http://www.citeulike.org/group/16008/tag/tools) | | 5 | [isgalaxy](http://www.citeulike.org/group/16008/tag/isgalaxy) | | 4 | [cloud](http://www.citeulike.org/group/16008/tag/cloud) | 
| 3 | [reproducibility](http://www.citeulike.org/group/16008/tag/reproducibility) | | 2 | [uselocal](http://www.citeulike.org/group/16008/tag/uselocal) | | 1 | [shared](http://www.citeulike.org/group/16008/tag/shared) | | 1 | [project](http://www.citeulike.org/group/16008/tag/project) | 
| 1 | [unknown](http://www.citeulike.org/group/16008/tag/unknown) | | 1 | [other](http://www.citeulike.org/group/16008/tag/other) | | | | | | | 

----

# Who's Hiring

<div class='right'><br /><img src="/src/images/GalaxyIsExpandingCloud.png" alt="Please Help! Yes you!" width="220" /></div>

The Galaxy is expanding! Please help it grow.

* [Bioinformatics Research Associate](https://academicpositions.harvard.edu/postings/7556), Huttenhower Lab, Harvard School of Public Health, Boston, Massachusetts, United States
* [Adviser for Data Infrastructure](https://www.academictransfer.com/employer/SURFsara/vacancy/40163/lang/en/), SURFsara, Amsterdam, The Netherlands
* [Automated deployment of Galaxy environments using Ansible](http://artbio.fr/?page_id=78), Institut de Biologie Paris Seine, France
* [Scientific Research Programmer](http://goo.gl/cMZVCn), [Sethuraman Lab](http://arunsethuraman.weebly.com/), [California State University San Marcos](https://www.csusm.edu/). Develop model-based population genomics pipelines.

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />


----

# Public Galaxy Server News

There are [over 90 publicly accessible Galaxy servers](/src/use/index.md) and [six semi-public Galaxy services](/src/use/index.md).  Here's what happened with them in May

## ImmPort Galaxy Update

<div class='right solid'><a href='http://immportgalaxy.org/'><img src="/src/use/immport-galaxy/immport-graphs.jpg" alt="ImmPort Galaxy new visualizations" width="200" /></a></div>

A new release of [ImmPort Galaxy](http://immportgalaxy.org/) is out:

* R is all up to date logicle trans, 
* Added a transformation algorithm to the Automated gating
* Transformation and Converson of FCS files to text tool,
* And a couple of tools to generate 1D Density plots.

More cool visualizations to look at your flow cytometry data!


## Usegalaxy.org

* The STAR RNA-seq ultra fast aligner [is now on Usegalaxy.org](https://usegalaxy.org/?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Frgrnastar%2Frna_star%2F2.5.2b-0) and has 64GB of memory available for each job. 
* [TopHat is gone from usegalaxy.org because there are now superior alternatives](https://biostar.usegalaxy.org/p/23367/).


## New Public Galaxy Servers

### GVL MEL

<div class='right solid'><a href='http://galaxy-mel.genome.edu.au/'><img src="/src/images/logos/gvl-300.png" alt="Genomics Virtual Lab Melbourne" height="100" /></a></div>

[Galaxy Melbourne](http://galaxy-mel.genome.edu.au/) is a general purpose Galaxy based on the [Genomics Virtual Lab platform](https://genome.edu.au/).  Support options include [GVL Help](https://www.gvl.org.au/), tutorials at [GVL Learn](https://www.gvl.org.au/learn) and [Galaxy Tut](http://galaxy-tut.genome.edu.au/).  Registration is required but anyone can register. The default quota of 100GB which can be increased on request.  GVL MEL is supported by the [Genomics Virtual Lab](https://www.gvl.org.au/about), [Melbourne Bioinformatics](https://www.melbournebioinformatics.org.au), and [Nectar](https://www.nectar.org.au).

### ChimeRScope

<div class='right solid'><a href='https://galaxy.unmc.edu/'><img src="/src/use/chimerscope/chimerscope.png" alt="ChimeRScope Galaxy" width="200" /></a></div>

[ChimeRScope Galaxy](https://galaxy.unmc.edu/) is a tool publishing server for fusion gene discovery from transcriptome-sequencing (RNA-seq) datasets. ChimeRScope is a java application  that uses a novel alignment-free approach for discovering fusion genes from transcriptome-sequencing (RNA-seq) datasets. Prediction results from simulated datasets and real datasets show that our method achieves the best prediction accuracy, comparing to all other popular methods.

See You Li, Tayla B. Heavican, Neetha N. Vellichirammal, Javeed Iqbal, Chittibabu Guda; [ChimeRScope: a novel alignment-free algorithm for fusion transcript prediction using paired-end RNA-Seq data](https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx315). *Nucleic Acids Research* 2017 gkx315. doi: 10.1093/nar/gkx315

There is a [ChimeRScope Manual](https://github.com/ChimeRScope/ChimeRScope/wiki) including a [FAQ](https://github.com/ChimeRScope/ChimeRScope/wiki/FAQs). The server supports anonymous access and creation of user logins. ChimeRScope Galaxy is supported by the [Guda Lab](http://unmc.edu/gudalab/) at the [University of Nebraska Medical Center](http://www.unmc.edu/).


<!--

# Galaxy Community Directories

<table class="table">
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/teach/index.md'><img src="/src/images/galaxy-logos/GTNLogo300.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/logs/index.md'><img src="/src/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/deployments/index.md'><img src="/src/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your <a href='/src/teach/resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/teach/trainers/index.md'>experience</a> <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/logs/index.md#add-a-log-page'>Share your experience</a> <em><a href='/src/community/logs/index.md#add-a-log-page'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/deployments/index.md#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/src/community/deployments/index.md#add-your-galaxy-deployment'>now</a></em> </strong> </td>
  </tr>
</table>

<br />

No new resources were added to community directories in January or February.
-->

----

# Tools

## Qiime and mothur

<div class='center'>
<a href='http://qiime.org/'><img src="/src/images/logos/qiime.png" alt="Qiime: Quantitative Insights Into Microbial Ecology" height="100" /></a> &nbsp;&nbsp;&nbsp;
<a href="https://www.mothur.org/"><img src="/src/images/logos/mothur_300.png" alt="mothur" height="100" /></a><br />
</div>

Galaxy has reached many different communities far beyond classical NGS. To support this the [IUC](/src/iuc/index.md) is organising regularly contribution fests dedicated to special topics, with the aim to increase tool coverage and develop re-usable workflows. One of the first contribution fests focused on metagenomics, a field that is gaining more traction in the Galaxy community. Here we highlight two pull requests that were recently merged and added [Qiime (Quantitative Insights Into Microbial Ecology)](https://github.com/galaxyproject/tools-iuc/pull/431) and [mothur](https://github.com/galaxyproject/tools-iuc/pull/1287) to Galaxy. These tools are the most popular pipelines used by the microbial ecology community to perform microbiome analyses from raw DNA sequencing data.

Based on previous work by James Johnson, ~150 tools were developed and adapted to the most recent IUC standards for Galaxy tools, including integration with [BioConda](https://bioconda.github.io/) and [BioContainers](https://biocontainers.pro/). This effort was led by [Saskia Hiltemann](https://github.com/shiltemann) and [Bérénice Batut](http://bebatut.fr/) and several reviewers and contributors, who created more than 300 commits and touched more than 600 files (including test-data). Thanks Saskia and Bérénice for this effort!

The tools can be installed via the Galaxy Tool Shed:

* [Qiime](https://toolshed.g2.bx.psu.edu/view/iuc/suite_qiime)
* [mothur](https://toolshed.g2.bx.psu.edu/view/iuc/suite_mothur)

Both tool suites and many more metagenomics tools are integrated into the [metagenomic Galaxy flavor](https://github.com/shiltemann/galaxy-metagenomics).

[Björn Grüning](/src/people/bjoern-gruening)


## ToolShed Contributions

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

[Tool Shed contributions from May](/src/toolshed/contributions/2017-05/index.md).


# Releases

## May 2017 Galaxy Release (v 17.05)

<div class="right">
<a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a><br /><br />
<a href="https://vimeo.com/216884518"><img src="/src/images/news-graphics/dataset-tagging-17-05.png" alt="Tag your datasets to keep track of where computed datasets came from" width="200" /></a><br /># Tags are propogated<br />in histories<br /><br />
<a href="https://depot.galaxyproject.org/hub/attachments/documents/videos/dataset-drag-17-05.gif"><img src="/src/images/news-graphics/dataset-drag-17-05.png" alt='Drag and drop with Galaxy Histories and Tools' width="200" /></a><br />Drag and drop in Galaxy 17.05
</div>

The [Galaxy Committers](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst) published the **[17.05 release of Galaxy](https://docs.galaxyproject.org/en/master/releases/17.05_announce.html)** in May.

**Tag your data with propagating hashtags**

* Large Galaxy histories used to be messy. Hashtags make it easy to track dataset (and collection) relationships.
* These two movies (both under a minute) explain how to use this [with datasets](https://vimeo.com/216884518) and [with collections](https://vimeo.com/216895965).
* Learn more about Galaxy histories in our updated [tutorial](https://galaxyproject.org/tutorials/histories/).

**Drag & Drop datasets into tool inputs**

* Interface now allows dragging datasets from history panel into the content selectors of the tool form.
* Implemented in [Pull Request 3871](https://github.com/galaxyproject/galaxy/pull/3871).

**Upload directly to a collection**

* You can now bypass the history manipulation and upload your data straight into a collection for convenience.
*  Learn more about collections and how to use them in a new [tutorial](https://galaxyproject.org/tutorials/collections/).

We extend special thanks to the 64 [New Contributors](https://docs.galaxyproject.org/en/master/releases/17.05_announce.html#new-contributors) to Galaxy in the past year.

Additionaly there are [New Configuration Options](https://docs.galaxyproject.org/en/master/releases/17.05_announce.html#new-configuration-options)  and [New Datatypes](https://docs.galaxyproject.org/en/master/releases/17.05_announce.html#new-datatypes) sections.


### Galaxy Docker Image 17.05

The [Galaxy Docker project has seen a new release](https://github.com/bgruening/docker-galaxy-stable/releases/tag/17.05), following Galaxy 17.05. Major features are an additional Docker compose setup with SLURM and HT-Condor deployments, BioContainers integration and much more automatic testing.

And

* The [Galaxy Docker Project](https://github.com/bgruening/docker-galaxy-stable) has reached more than 23k downloads on [Dockerhub](https://hub.docker.com/r/bgruening/galaxy-stable/) - not counting quay.io and all flavors 
* From [John Chilton](/src/people/john-chilton/index.md): 
  * Awesome new mode for docker-galaxy-stable by  [Björn Grüning](/src/people/bjoern-gruening/index.md): [container scheduling with Condor](https://github.com/bgruening/docker-galaxy-stable/pull/323)


## Parsec 1.0.0 - 1.0.2

[Parsec](https://github.com/galaxy-iuc/parsec) is a newly released set of command-line utilities to assist in working with Galaxy servers.  It uses automatically generated wrappers for BioBlend functions. Manage histories, launch workflows, and more, all from the command line. The README includes several examples.


## Planemo 0.41.0

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src="/src/images/galaxy-logos/planemo-logo.png" alt="" width="200" /></a></div>

[Planemo](https://pypi.python.org/pypi/planemo) is a set of command-line utilities to assist in building tools for the Galaxy project.  These releases included [numerous fixes and enhancements](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).

See [GitHub for details](https://github.com/galaxyproject/planemo/blob/master/HISTORY.rst).


## BioBlend 0.9.0

[BioBlend](http://bioblend.readthedocs.org/) is a Python library for interacting with CloudMan and Galaxy‘s API.  BioBlend makes it possible to script and automate the process of cloud infrastructure provisioning and scaling via CloudMan, and running of analyses via Galaxy.

See the [release notes](https://github.com/galaxyproject/bioblend/releases) for what's new in release 0.9.0.


## galaxy-lib 17.5.9-11

galaxy-lib is a subset of the Galaxy core code base designed to be used as a library. This subset has minimal dependencies and should be Python 3 compatible.  It's available from [GitHub](https://github.com/galaxyproject/galaxy-lib) and [PyPi](https://pypi.python.org/pypi/galaxy-lib).

## Galaxy CloudMan 17.05 on AWS

A new release of [Galaxy CloudMan](/src/cloudman/index.md) is available on the Amazon Web Services cloud infrastructure. This release includes Galaxy 17.05, an updated [tool list](https://github.com/galaxyproject/galaxy-cloudman-playbook/blob/master/files/usegalaxy_main_tool_list.yaml), and Slurm configuration changes to improve job performance. To get started, vist https://beta.launch.usegalaxy.org/ or take a look at the [Getting Started](/src/cloudman/index.md#getting-started) guide.


## Earlier Releases

Other packages that have been released in the prior 4 months.

### CloudLaunch ##

Technically, the [all-new Galaxy CloudLaunch service](https://beta.launch.usegalaxy.org/) has been in public beta since February but keep in mind that it will replace the [current CloudLaunch service](https://launch.usegalaxy.org/launch) eventually so give it a try and let us know how it performs for you.

### CloudBridge 0.2.0

[CloudBridge](https://github.com/gvlproject/cloudbridge) aims to provide a simple layer of abstraction over different cloud providers, reducing or eliminating the need to write conditional code for each cloud. It is currently under development and is in an Alpha state.  [Release 0.2.0](https://github.com/gvlproject/cloudbridge/releases/tag/v0.2.0) includes several fixes and enhancements.

### And the rest ...

Other Galaxy packages that haven't had a release in the past four months can be [found on GitHub](https://github.com/galaxyproject).

----


# Other News

* From  [Paul Gordon](https://www.blogger.com/profile/02637194860456629067)
  * [The ungoogleable: Authenticating a Galaxy Portal with Active Directory ](http://achri.blogspot.ca/2017/05/the-final-frontier-authenticating.html)
* From  [Björn Grüning](/src/people/bjoern-gruening/index.md)
  * There is a new [HiCExplorer Galaxy Docker version](https://github.com/maxplanck-ie/docker-galaxy-hicexplorer), based on Galaxy 17.05, made possible by [Fidel Ramírez](https://twitter.com/fidel_ramirez) and friends.
* From [Mo Heydarian](/src/people/mo-heydarian/index.md)
 * Creating Galaxy workflows [is easier than you think](https://depot.galaxyproject.org/hub/attachments/documents/videos/2017-05-workflow-example.gif). If you like reproducibility, you'll like workflows!
* New video on [Creating collections by uploading from SRA](https://vimeo.com/217216264).
* From [Nate Coraor](/src/people/nate/index.md)
  * Heads up: Galaxy roles published on galaxy.ansible.com have changed namespaces from `galaxyprojectdotorg` to just `galaxyproject`


