---
title: The August 2015 Galactic News!
---
<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/src/galaxy-updates/index.md'><img src="/src/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the August 2015 *[Galactic News](/src/galaxy-updates/index.md)*, a summary of what is going on in the Galaxy community. It's been, and will be, a busy few months.  If you have anything to include in the September News, please send it to [Galaxy Outreach](mailto:outreach AT galaxyproject DOT org). 


# Events

## GCC2015 Report

<div class='right'><a href='http://gcc2015.tsl.ac.uk/'><img src="/src/images/logos/GCC2015LogoWide600.png" alt="GCC2015" width="170" /></a>
</div>

The **[2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/)** was held 4-8 July 2015, in Norwich, UK. 
GCC2015 was hosted by [The Sainsbury Lab](http://tsl.ac.uk/) in Norwich, UK.  By all measures, the meeting was a tremendous success.

GCC2015 registrations reached an all-time high, and GC2015 featured more events than any previous GCC:
* A record **230 researchers** signed up for one or more GCC2015 events:
  * 41 registered for the 2nd Annual [GCC Coding Hackathon](http://gcc2015.tsl.ac.uk/Hackathon/), held at TGAC, 4-5 July, 
  * 17 registered for the first ever [GCC Data Wrangling Hackathon](http://gcc2015.tsl.ac.uk/data-hackathon/), held simultaneously at TGAC,
  * 72 registered for the first ever (and sold out) [Training Sunday](http://gcc2015.tsl.ac.uk/Training-Day/) on 5 July.
  * 178 registered for [Training Day](http://gcc2015.tsl.ac.uk/Training-Day/) on 6 July,
  * 191 registered for the [GCC Meeting](http://gcc2015.tsl.ac.uk/Conference/), on 7-8 July,
* [15 topics were offered in 18 sessions by 32 instructors over two days of training](http://gcc2015.tsl.ac.uk/Training-Day/).
* The GCC2015 Meeting featured
  * [28 accepted and keynote talks](http://gcc2015.tsl.ac.uk/abstracts) (slides for most are online, videos are coming)
  * [34 posters](http://gcc2015.tsl.ac.uk/Posters/) (most are online)
  * [14 lightning talks](http://gcc2015.tsl.ac.uk/Lightning/) (slides for most are online, videos are coming)
  * [10 Birds-of-a-feather meetups](/src/events/gcc2015/bofs/index.md) (writeups for most are available)
* There was a record amount of Twitter traffic at GCC2015.  Peter Cock has created several Storify pages that captured the stream:
  * [GCC2015 Hackathon & Training Days](https://storify.com/pjacock/galaxy-community-conference-gcc2015-hackathon)
  * [GCC2015 Galaxy Community Conference - Day One](https://storify.com/pjacock/gcc2015-galaxy-community-conference)
  * [GCC2015 Galaxy Community Conference - Day Two](https://storify.com/pjacock/gcc2015-galaxy-community-conference-day-twp)

The [GCC2015 Organising Committee](http://gcc2015.tsl.ac.uk/organisers/#Organising_Committee) would like to thank the [Scientific Committee](http://gcc2015.tsl.ac.uk/organisers/#Scientific_Committee), [Coding Hackathon Organisers](http://gcc2015.tsl.ac.uk/organisers/#Coding_Hackathon_Committee), [Data Wrangling Hackathon Organisers](http://gcc2015.tsl.ac.uk/organisers/#Data_Wrangling_Hackathon_Committee), [Conference Sponsors](http://gcc2015.tsl.ac.uk/sponsors/), [BoF Organisers](/src/events/gcc2015/bofs/index.md), [Training Instructors](http://gcc2015.tsl.ac.uk/training-day) *and absolutely everyone else who contributed to the success of GCC2015.*

Which brings us to ...

## GCC2016: June 25-29, 2016, Bloomington, Indiana, United States

<div class='right'><a href='/src/events/gcc2016/index.md'><img src="/src/events/gcc2016/GCC2016LogoFull_big.png" alt="2016 Galaxy Community Conference" width="300" /></a></div>

**We are pleased to announce that the [2016 Galaxy Community Conference (GCC2016)](/src/events/gcc2016/index.md) will be held June 25-29 at Indiana University in Bloomington, Indiana, United States.**

GCC2016 is the 7th annual gathering of the Galaxy community. The conference will include keynotes, accepted talks, poster sessions, birds-of-a-feather meetups, exhibitors, and plenty of networking opportunities. There will also be three days of pre-conference activities, including hackathons and training.  Mark your calendars now.  Registration will open in the fall.

And, please help get the word out by posting the announcement at your organization.

If you work in data-intensive biomedical research, there is no better place than GCC2016 to present your work and to learn from others.

## August GalaxyAdmins Meetup

<div class='left'><a href='/src/community/galaxy-admins/meetups/2015-08-20/index.md'><img src="/src/images/logos/GalaxyAdmins.png" alt="GalaxyAdmins meetup August 20" height="75" /></a></div>
<div class='right'>
<a href='http://www.irida.ca/'><img src="/src/images/logos/IRIDA.png" alt="IRIDA" width="200" /></a>
</div>

Please join us for the [next GalaxyAdmins meetup on August 20](/src/community/galaxy-admins/meetups/2015-08-20/index.md) when [Aaron Petkau](https://github.com/apetkau) will cover 

 *Genomic data management at [Canada's National Microbiology Laboratory](https://www.nml-lnm.gc.ca/index-eng.htm) with [IRIDA](http://www.irida.ca/) and Galaxy*

[GalaxyAdmins](/src/community/galaxy-admins/index.md) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  


<div class='left'><br /><a href='http://wiki.pitagora-galaxy.org/wiki/index.php/Meetup_2015-07'><img src="/src/images/logos/PitagoraBoXLogo.png" alt="meetup on 23 July 2015" width="65" /></a>
</div>

## July 2015 Pitagora-Galaxy Meetup

The [Pitagora-Galaxy](http://www.pitagora-galaxy.org/) community in Japan had a [meetup on 23 July 2015](http://wiki.pitagora-galaxy.org/wiki/index.php/Meetup_2015-07) at the National Institute of Informatics in Tokyo.  The discussion is [summarized here](http://wiki.pitagora-galaxy.org/wiki/index.php/Meetup_2015-07) (in Japanese).

<div class='right'>
<br />
<div class='right'><a href='/src/events/ismb2015/index.md#ismbeccb-2015'><img src="/src/images/logos/ISMB-ECCB_2015_Logo.jpg" alt="ISMB ECCB 2015" width="100" /></a></div>
<a href='/src/events/ismb2015/index.md#bosc-2015'><img src="/src/images/logos/BOSC_logo.png" alt="BOSC 2015" width="100" /></a>
</div>

## Galaxy at ISMB/ECCB and BOSC 2015

[BOSC 2015](http://www.open-bio.org/wiki/BOSC_2015) and [ISMB/ECCB 2015](http://www.iscb.org/ismbeccb2015) followed immediately after GCC2015 this year, and Galaxy again a significant presence at both.

[Slides, posters and videos for most presentations](/src/events/ismb2015/index.md) are now available on the [Galaxy @ ISMB/ECCB and BOSC 2015 page](/src/events/ismb2015/index.md).

## Other Events

<div class='center'> 
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src="/src/images/icons/CalendarIcon.gif" /></a></div>

There are [upcoming events](/src/events/index.md) in four countries on three continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.


<div class='center'> 
<a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-3/'><img src="/src/images/logos/QFABLogo.png" alt="Workshop: Extended RNA-Seq analysis" height="80" /></a>
<a href='http://biosb.nl/education/course-portfolio/rna-seq/'><img src="/src/images/logos/BioSBResearchSchool.png" alt="RNA-seq data analysis NGS Course" height="80" /></a>
<a href='http://training.bioinformatics.ucdavis.edu/2015/01/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-september-14-18-2015/'><img src="/src/images/logos/UCDavisGenomeCenter.png" alt="Using Galaxy for Analysis of High Throughput Sequence Data" height="50" /></a> &nbsp;
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> August 6-7 </th>
    <td> <em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-3/'>Workshop: Extended RNA-Seq analysis</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/AU.png" alt="" /></div> <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <div class='right'><a href='/src/teach/trainers/index.md#qfab-bioinformatics'><img src="/src/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> September 7-9 </th>
    <td> <em><a href='http://biosb.nl/education/course-portfolio/rna-seq/'>RNA-seq data analysis NGS Course</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/EU.png" alt="" /></div> Leiden University Medical Center, The Netherlands </td>
    <td> <div class='right'> <a href='/src/teach/trainers/index.md#lumc-erasmusmc-dtl-learning-programme'><img src="/src/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href="mailto:education@biosb.nl">BioSB Education</a> </td>
  </tr>
  <tr>
    <th> September 14-18 </th>
    <td> <em><a href='http://training.bioinformatics.ucdavis.edu/2015/01/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-september-14-18-2015/'>Using Galaxy for Analysis of High Throughput Sequence Data</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/NA.png" alt="" /></div> <a href='http://bioinformatics.ucdavis.edu/'>UC Davis Bioinformatics Core</a>, Davis, California, United States </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnUCDavis'><img src="/src/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href="mailto:bioinformatics.core@ucdavis.edu">UC Davis Bioinformatics</a> </td>
  </tr>
  <tr>
    <th> September 15 </th>
    <td> <em><a href='https://hubzero.org/hubbub/speakers#yvan'>Towards accessible, reproducible, and transparent research in the life sciences: an innovative open source VRE approach in Western France</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/NA.png" alt="" /></div> <a href='https://hubzero.org/hubbub/2015'>HUBbub 2015</a>, Indianapolis, Indiana, United States </td>
    <td> <a href='https://www.e-biogenouest.org/members/1003'>Yvan Le Bras</a> </td>
  </tr>
  <tr>
    <th> September 21-25 </th>
    <td> <em><a href='http://workflow4metabolomics.org/training/W4Mcourse2015'>Traitement des données métabolomiques sous Galaxy</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/EU.png" alt="" /></div> Station Biologique de Roscoff, France </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnGUGGO'><img src="/src/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a></div> <a href="mailto:w4mcourse2015.organisation@sb-roscoff.fr">W4M Course Organisers</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> October<br />12-14 </th>
    <td> <em><a href='https://cnrsformation.cnrs.fr/stage.php?stage=15274&axe=77'>Analyse avancée de séquences</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/EU.png" alt="" /></div> <a href='http://www.cbib.u-bordeaux2.fr/content/bienvenue-au-centre-de-bioinformatique-de-bordeaux'>Bordeaux Bioinformatics Center</a>, University Bordeaux Segalen, France </td>
    <td> <a href="mailto:cfe.contact@cnrs.fr">Email</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> November 17-18 </th>
    <td> <em><a href='http://www.fc3bio.fr/Analyse-des-donnees-RNA-seq-sous-l-environnement-GALAXY_a39.html'>Analyse de données RNA-seq sous l’environnement Galaxy</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/EU.png" alt="" /></div> <a href='http://www.prabi.fr'>PRABI</a>, Lyon, France </td>
    <td> <div class='right'><a href='/src/teach/trainers/index.md#galaxy--prabi-project'><img src="/src/images/icons/GTN32.png" alt="Training offered by GTN Member" width="16" /></a> </div> <a href="mailto:contact@fc3bio.fr">Email</a> </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> December<br />7-11 </th>
    <td> <em><a href='http://bis15.amsi.org.au/galaxy/'>Galaxy and the Genomics Virtual Lab</a></em> </td>
    <td> <div class='right'><img src="/src/images/icons/AU.png" alt="" /></div> <a href='http://bis15.amsi.org.au/'>BioInfoSummer 2015</a>, University of Sydney, NSW, Australia </td>
    <td> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/src/teach/trainers/index.md'><img src="/src/images/icons/GTN32.png" alt="Designates a training event offered by GTN Member" width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/src/teach/trainers/index.md'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


----

# New Papers

143 new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in June and July. Highlights include:

* [AGEseq: Analysis of Genome Editing by Sequencing](http://bit.ly/1JeHagu), Xue & Tsai

* [Metaproteomic analysis using the Galaxy framework](http://bit.ly/1INZ2PS), Jagtap *et al.*

* [Darwintree: A Molecular Data Analysis and Application Environment for Phylogenetic Study](http://bit.ly/1LuDbhz), Meng *et al.*

* [BioMaS: a modular pipeline for Bioinformatic analysis of Metagenomic AmpliconS](http://bit.ly/1LOk2G2), Fosso *et al.*

* [xHeinz: an algorithm for mining cross-species network modules under a flexible conservation model](http://bit.ly/1g8gTp2), El-Kebir *et al.*

* [The Plant Genome Integrative Explorer Resource: PlantGenIE.org](http://bit.ly/1IguksS), Sundell *et al,*

* [Genome-wide analysis of signatures of selection in populations of African honey bees (Apis mellifera) using new web-based tools](http://bit.ly/1LE6QDp), Fuller *et al* 

* [Enabling Data-Intensive Biomedical Studies](http://bit.ly/leophd), Simone Leo

* [Enabling cloud bursting for life sciences within Galaxy](http://bit.ly/1TRbuRl), [Afgan](/src/people/enis-afgan/index.md) *et al.*

* [Building and Provisioning Bioinformatics Environments on Public and Private Clouds](http://bit.ly/1gOtVYQ), [Afgan](/src/people/enis-afgan/index.md) *et al.*

* [LiSIs: An Online Scientific Workflow System for Virtual Screening](https://doi.org/10.2174/1386207318666150305123341), Kannas, *et al.*

* [RNA-Rocket: an RNA-Seq analysis resource for infectious disease research](https://doi.org/10.1093/bioinformatics/btv002), Warren *et al.*

* [Genome-wide analysis of signatures of selection in populations of African honey bees (Apis mellifera) using new web-based tools](https://doi.org/10.1186/s12864-015-1712-0), Fuller, *et al.*

* [Context influences on TALE–DNA binding revealed by quantitative profiling](https://doi.org/10.1038/ncomms8440), Rogers, *et al.*

* [Compact graphical representation of phylogenetic data and metadata with GraPhlAn](https://doi.org/10.7717/peerj.1029), Asnicar *et al.*

* [ARC Control Tower A flexible generic distributed job management framework](http://bit.ly/1ROmd1q), Nilsen *et al.* 

The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 11 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 11 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 6 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 44 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 57 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 16 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>



----

# NGS 101 Tutorial Videos are Now Online

<div class='right'><a href='https://vimeo.com/album/3456159'><img src="/src/images/ngs101/NGS101PlaySlide.png" alt="Galaxy NGS 101 Videos" width="180" /></a>
</div>

The new [Galaxy NGS 101 Tutorial](/src/learn/galaxy-ngs101/index.md) was published this spring and introduces a wide range of topics related to analyzing next generation sequencing data with Galaxy.  We are happy to announce that [videos to accompany this tutorial are now online as well](https://vimeo.com/album/3456159).  The 20 videos cover:

1. [Uploading data from your computer](https://vimeo.com/album/3456159/video/120901536)
1. [Uploading data from FTP](https://vimeo.com/album/3456159/video/120972739)
1. [Uploading data from the Web](https://vimeo.com/album/3456159/video/120973708)
1. [Uploading data from EBI SRA](https://vimeo.com/album/3456159/video/121187220)
1. [Uploading data from NCBI SRA](https://vimeo.com/album/3456159/video/121190377)
1. [QC'ing with FastQC](https://vimeo.com/album/3456159/video/123453134)
1. [A quick into to read mapping](https://vimeo.com/album/3456159/video/123102338)
1. [Mapping to YOUR reference](https://vimeo.com/album/3456159/video/123108417)
1. [Tweaking BAM](https://vimeo.com/album/3456159/video/123113197)
1. [Non-diploid variant calling with NVC](https://vimeo.com/album/3456159/video/123442619)
1. [Non-diploid varinat calling with FreeBayes](https://vimeo.com/album/3456159/video/123449704)
1. [Looking at multiple datasets in IGV](https://vimeo.com/album/3456159/video/123414437)
1. [Calling variants with FreeBayes](https://vimeo.com/album/3456159/video/123436423)
1. [RNA-Seq: Mapping with TopHat](https://vimeo.com/album/3456159/video/128265983)
1. [RNA-Seq: Assembling and quantifying transcripts with CuffLinks](https://vimeo.com/album/3456159/video/128268401)
1. [RNA-Seq: Playing with CuffDiff output and Cummerbund to find differentially expressed transcripts](https://vimeo.com/album/3456159/video/128265982)

There are also 4 new videos that provide an overview of different sequencing platforms:
* [Illumina](https://vimeo.com/album/3456159/video/121178846)
* [Ion Torrent](https://vimeo.com/album/3456159/video/121289100)
* [PacBio](https://vimeo.com/album/3456159/video/121267426)
* [454](https://vimeo.com/album/3456159/video/121286060)

The videos join over 60 other videos on the [Galaxy Project Vimeo Channel](https://vimeo.com/galaxyproject).

# Genomic Data Science with Galaxy Launched on Coursera

<div class='left'>
<a href='https://www.coursera.org/course/gengalaxy'><img src="/src/images/logos/CourseraGBDS.png" alt="Genomic Data Science with Galaxy" width="150" /></a>
</div>

A [Genomic Data Science with Galaxy](https://www.coursera.org/course/gengalaxy) course was launched on the online teaching platform [Coursera](https://www.coursera.org/) in July.  The course is part of the [Genomic Data Science Specialization](https://www.coursera.org/specialization/genomics/41).  A new session starts every month from now through at least the end of 2015.  Coursera courses can be taken for free, or you can pay the course fee and receive a certificate upon completion.

The [video lectures for the Galaxy material is on Vimeo](http://vimeo.com/album/3456144). 

----

# Who's Hiring

<div class='right'><a href='/src/galaxy-is-hiring/index.md'><img src="/src/galaxy-is-hiring/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Assistant Computational Scientist](http://bit.ly/1OLwPbW) at [VIB](http://www.vib.be/), Gent, Belgium 

 * 
* **The [Galaxy Project is hiring software engineers and post-docs](/src/galaxy-is-hiring/index.md)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/news/index.md) and include it in next month's [update](/src/galaxy-updates/index.md).

<br />

----

<div class='right solid blue'> 
<a href='http://ab-openlab.csir.res.in/frog/'><img src="/src/use/ab-openlab/frog.png" alt="FingeRprinting Ontology of Genomic variations (FROG)" width="160" /></a><br />
<a href='http://aspendb.uga.edu/ageseq'><img src="/src/use/ageseq-aspendb/aspen-db-300.png" alt="AGEseq Galaxy @ AspenDB" width="160" /></a><br />
<a href='http://galaxy.cloud.ba.infn.it:8080/'><img src="/src/use/biomas/biomas-logo.jpg" alt="BioMaS Galaxy" width="160" /></a><br />
<a href='http://thebrain.bwh.harvard.edu/sifted.html'><img src="/src/use/archive/sifted/sifted-landing-page.png" alt="SIFTED" width="160" /></a><br />
<a href='http://mississippi.fr/'><img src="/src/use/mississippi/mississippi-logo.png" alt="MISSIPPI" width="160" /></a><br />
<a href='/src/use/index.md'><img src="/src/public-galaxy-servers/70PlusSlide.png" alt="70+ Public Galaxy Servers" width="160" /></a>
</div>

# New Public Galaxy Servers

July was a banner month, with the addition of [5 new publicly accessible Galaxy servers](/src/use/index.md), bringing the list of public servers to *over 70*.

1. **[AB-OpenLab FROG](http://ab-openlab.csir.res.in/frog/)**, from  [CSIR- GENESIS: Genomics and Informatics Solutions for Integrating Biology](http://crdd.osdd.net/genesis/genome.html)
  * **F**inge**R**printing **O**ntology of **G**enomic variations: *FROG* fingerprints capture genomic variations at different levels.  See [the FROG server description](/src/use/ab-openlab/index.md) for more.

1. **[AGEseq Galaxy @ AspenDB](http://aspendb.uga.edu/ageseq)**, from the [Tsai Lab](http://aspendb.uga.edu/), [Warnell School of Forestry and Natural Resources](http://warnell.uga.edu/) and [Department of Genetics](http://www.genetics.uga.edu/), University of Georgia.
  * AGEseq (Analysis of Genome Editing by Sequencing) compares amplicon sequences with expected target sequences and finds insertion/deletion sites in the amplicon sequences.  See [the AGEseq server description](/src/use/ageseq-aspendb/index.md) for more.

1. **[BioMaS Galaxy](http://galaxy.cloud.ba.infn.it:8080/)** from the [Istituto Nazionale di Fisica Nucleare](http://www.infn.it/)
  * Taxonomic studies of environmental microbial communities.  See [the server description](/src/use/biomas/index.md) for more.

1. **[MISSISSIPPI Server](http://mississippi.fr/)** from The ARTbio bioinformatics facility of the  [Institut de Biologie Paris Seine](http://www.ibps.upmc.fr/en) based at the [University Pierre & Marie Curie](http://www.upmc.fr/en/index.html).
  * RNA and small RNA sequencing dataset analysis as well as for epigenetics or metagenomics studies.  See the [server descriptions](/src/use/mississippi/index.md) for more.

1. **[SIFTED](http://thebrain.bwh.harvard.edu/sifted.html)** from [Bulyk Lab](http://thebrain.bwh.harvard.edu/index.html), [Division of Genetics](http://genetics.bwh.harvard.edu/genetics/index.html) in the [Department of Medicine at Brigham & Women's Hospital](http://www.brighamandwomens.org/medicine/) and [Harvard Medical School](http://hms.harvard.edu/hms/home.asp)
  * Specificity Inference For TAL-Effector Design (SIFTED) is a computational model for predicting the DNA-binding specificity of any Transcription activator-like effector (TALE) proteins. 
<br />

<div class='left'><br /><a href='https://github.com/galaxyproject/galaxy/pull/314'><img src="/src/images/CommunityCommitters1.png" alt="Community Committers" width="160" /></a>
</div>


# Community Committers!

Galaxy now has a formal and open policy for managing the project source code - including for how to add and remove committers. Anyone can issue pull requests and join in the conversation, but committers are trusted to decide how these pull requests are integrated and can participate in formal voting. Read more in ([Pull Request 314](https://github.com/galaxyproject/galaxy/pull/314)). Based on their frequent contributions both in terms of code written and contribution to discussions - the Galaxy Project added three new contributors as part of this process. **[Nicola Soranzo](https://github.com/nsoranzo), [Björn Grüning](https://github.com/bgruening), and [E. Rasche](https://github.com/erasche)** are now committers.


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/teach/index.md'><img src="/src/images/logos/GTNLogo600.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/blog/index.md'><img src="/src/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/src/community/deployment/index.md'><img src="/src/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your <a href='/src/teach/resources/index.md#add-a-training-resource'>training resources</a> and <a href='/src/teach/trainers/index.md#add-a-trainer'>experience</a> <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/blog/index.md'>Share your experience</a> <em><a href='/src/blog/index.md#add-a-log-page'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/src/community/deployment/index.md'>Describe your instance</a> <em><a href='/src/community/deployment/index.md'>now</a></em> </strong> </td>
  </tr>
</table>

<br />

Three new **[Community Log Board](/src/blog/index.md)** entries were added in July:

* **[Moving from MySQL to PostgreSQL](/src/blog/2015-07-mysql-2-postgresql/index.md)**
    by [Hans-Rudolf Hotz](/src/people/hansrudolf-hotz/index.md)

* **[BioJS2Galaxy – A step by step guide](http://www.benjamenwhite.com/2015/07/biojs2galaxy-a-step-by-step-guide/)**
    by [Benedikt Rauscher](https://twitter.com/bengrauscher) and [Benjamen White](http://www.benjamenwhite.com/2015)

* **[Exposing Galaxy reports via nginx in a production instance](http://galacticengineer.blogspot.co.uk/2015/06/exposing-galaxy-reports-via-nginx-in.html)**
    by [Peter Briggs](http://www.ls.manchester.ac.uk/people/profile/?alias=briggsp)

And the new [Galaxy NGS 101](https://wiki.galaxyproject.org/Learn/GalaxyNGS101) set of exercises, and the [Galaxy Coursera material](https://www.coursera.org/course/gengalaxy) were added to the **[Teaching Resources Directory](/src/teach/resources/index.md)**:

* **[Galaxy NGS 101](/src/teach/resource/galaxy-ngs101/index.md)**
    by [Anton Nekrutenko](/src/people/anton/index.md)

* **[Genomic Data Science with Galaxy on Coursera](/src/teach/resource/genomic-data-science-galaxy-coursera/index.md)**
    by [James Taylor](/src/people/james-taylor/index.md)

----

# Updated Galaxy Logos

<div class='left'><a href='/src/images/galaxy-logos/index.md'><img src="/src/images/galaxy-logos/galaxy_project_logo_blue.png" alt="Updated Galaxy Project Logo" width="180" /></a></div>

<div class='right'><a href='http://puradesign.cz/'><img src="/src/images/logos/puradesignlogo.png" alt="Pura Design" /></a></div>

You may have noticed in the past two months that an [updated Galaxy logo](/src/images/galaxy-logos/index.md) has been slowly appearing on project web sites and presentations.  The new logo was created by [Petr Kadlec](http://puradesign.cz/en) of [Pura Design](http://puradesign.cz/en), and it slowly making its way into all Galaxy Project logos.  As this happens, they will be added to the [Images/GalaxyLogos page](/src/images/galaxy-logos/index.md).

----

# Releases

## BioBlend 0.6.0 and 0.6.1

[BioBlend versions 0.6.0 and 0.6.1](https://github.com/galaxyproject/bioblend/blob) were released in June and July. BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/src/develop/api/index.md)([CloudMan](/src/cloudman/index.md) offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG):

* BioBlend.objects: Rename `ObjDatasetClient` abstract class to `ObjDatasetContainerClient`.
* BioBlend.objects: Add `ABCMeta` metaclass and `list()` method to `ObjClient`.
* BioBlend.objects: Add `io_details` and `link_details parameters` to `ObjToolClient.get()` method.
* Open port 8800 when launching cloud instances for use by NodeJS proxy for Galaxy IPython Interactive Environments.
* When launching cloud instances, propagate error messages back to the caller. The return types for methods `create_cm_security_group`, `create_key_pair` in `CloudManLauncher` class have changed as a result of this.
* Add support for Python >= 3.3.
* Add `get_library_permissions()` method to `LibraryClient`.
* Add `update_group(), get_group_users(), get_group_roles(), add_group_user(), add_group_role(), delete_group_user()` and `delete_group_role()` methods to `GroupsClient`.
* Add `full_details parameter` to `JobsClient.show_job()` (thanks to Rossano Atzeni).
* BioBlend.objects: add `ObjJobClient` and `Job` wrapper (thanks to Rossano Atzeni).
* BioBlend.objects: add check to verify that all tools in a workflow are installed on the Galaxy instance (thanks to Gianmauro Cuccuru).
* Remove several deprecated parameters: see commits [19e168f](https://github.com/galaxyproject/bioblend/commit/19e168f5342f4c791d37694d7039a85f2669df71) and [442ae98](https://github.com/galaxyproject/bioblend/commit/442ae98037be7455d57be15542553dc848d99431).
* Verify SSL certificates by default.
* Add documentation about the Tool Sed and properly link all the docs on ReadTheDocs.
* Solidify automated testing by using [tox](https://tox.readthedocs.org/) and [flake8](https://gitlab.com/pycqa/flake8).

## Planemo 0.13.0 through 0.13.2

<div class='right'><a href='https://pypi.python.org/pypi/planemo/'><img src="/src/images/logos/PlanemoLogo.png" alt="" width="200" /></a></div>

Planemo saw [one major and several minor](https://pypi.python.org/pypi/planemo/0.13.2#history) releases in the past two months.  The most recent is [v0.13.2](https://pypi.python.org/pypi/planemo/0.13.2).  From the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2):

* Fix for `shed_init` producing non-standard type hints. [Issue 243](https://github.com/galaxyproject/planemo/issues/243), [f0610d7](https://github.com/galaxyproject/planemo/commit/f0610d7)
* Fix tool linting for parameters that define an `argument` but not a `name`. [Issue 245](https://github.com/galaxyproject/planemo/issues/245), [aad1eed](https://github.com/galaxyproject/planemo/commit/aad1eed)
* Many doc updates including a tutorial for developing tools in a test-driven fashion and instructions for using the planemo appliance through Kitematic (with Kitematic screenshots from Eric Rasche).
* If planemo cannot find a Galaxy root, it will now automatically fetch one (specifying `--galaxy_install` will still force a fetch). [Pull Request 235](https://github.com/galaxyproject/planemo/pull/235)
* [Documentation](http://planemo.readthedocs.org/en/latest/appliance.html) has been updated to reflect new and vastly improved Docker and Vagrant virtual appliances are now available, as well as a new VirtualBox OVA variant.
* Update linting for new tool XML features (including `detect_errors` and output collections). [Issue 233](https://github.com/galaxyproject/planemo/issues/233), [334f2d4](https://github.com/galaxyproject/planemo/commit/334f2d4)
* Fix `shed_test` help text. [Issue 223](https://github.com/galaxyproject/planemo/issues/223)
* Fix code typo (thanks to Nicola Soranzo). [Pull Request 230](https://github.com/galaxyproject/planemo/pull/230)
* Improvements to algorithm used to guess if an XML file is a tool XML file. [Issue 231](https://github.com/galaxyproject/planemo/issues/231)
* Fix configuration file handling bug. [Issue 240](https://github.com/galaxyproject/planemo/issues/240)

Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.


## Others

<div class='right'><a href='http://getgalaxy.org'><img src="/src/images/logos/GalaxyNewLogo_GalaxyProject_Trans.png" alt="GalaxyProject" width="200" /></a></div>
 
 **May 2015 Galaxy Release (v 15.05)**
    [Release Notes v 15.05](http://galaxy.readthedocs.org/en/master/releases/15.05_announce.html)


 **Pulsar**
   [Pulsar 0.5.0](https://pypi.python.org/pypi/pulsar-app/0.5.0) was released in May.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.
 
 **[CloudMan](/src/cloudman/index.md) **
   The [most recent edition of CloudMan](/src/galaxy-updates/2014-09/index.md#august-2014-cloudman-release) was released in August.

 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and BioCloudCentral.


----

# Other News

* [Michael Crusoe](http://impactstory.org/MichaelRCrusoe) created a [checklist for Galaxy DataSource docs](/src/admin/internals/data-sources/index.md#adding-a-new-datasource) 
* [Michael Crusoe](http://impactstory.org/MichaelRCrusoe) is also working on supporting the [Open Science Framework](https://osf.io/) as a [data source in Galaxy](https://osf.io/en2ak/).
* The [main Galaxy Project repository on GitHub](https://github.com/galaxyproject/galaxy) has over 100 forks! Thank you very much for all the contributions!
* [Samuel Lampa's](http://bionics.it/) created a blog post on [Workflow tool makers: Allow defining data flow, not just task dependencies](http://bionics.it/posts/workflows-dataflow-not-task-deps)
* The semi-annual update of the [Galaxy Project Statistic page](/src/galaxy-project/statistics/index.md) is done.

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/src/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in June and July](/src/toolshed/contributions/2015-07/index.md).
