---
autotoc: true
title: The September 2015 Galactic News, !
---


<div class='right'>
<table>
  <tr>
    <td style=" border: none; width: 28em;">  </td>
  </tr>
</table>

</div>

<div class='left'><a href='/GalaxyUpdates'><img src='/Images/GalaxyLogos/GalaxyNews.png' alt='Galaxy News' width=150 /></a></div>

Welcome to the September 2015 *[Galactic News](/src/GalaxyUpdates/index.md)*, a summary of what is going on in the Galaxy community. It's been, and will be, a busy few months.  If you have anything to include in the September News, please send it to [outreach AT galaxyproject DOT org](Galaxy Outreach). 

# New Papers

**97** new papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in August. Highlights include:

* [FROG - Fingerprinting Genomic Variation Ontology](http://dx.doi.org/10.1371/journal.pone.0134693)
    E. Abinaya, Pankaj Narang, Anshu Bhardwaj, *PLoS ONE*, Vol. 10, No. 8. (5 August 2015), e0134693, doi:10.1371/journal.pone.0134693

* [EpiToolKit—a web-based workbench for vaccine design](http://dx.doi.org/10.1093/bioinformatics/btv116)
    Benjamin Schubert, Hans-Philipp Brachvogel, Christopher Jürges, Oliver Kohlbacher, *Bioinformatics*, Vol. 31, No. 13. (01 July 2015), pp. 2211-2213, doi:10.1093/bioinformatics/btv116

* [Experimental validation of methods for differential gene expression analysis and sample pooling in RNA-seq](http://dx.doi.org/10.1186/s12864-015-1767-y)
    Anto P. Rajkumar, Per Qvist, [Ross Lazarus](/RossLazarus), Francesco Lescai, Jia Ju, Mette Nyegaard, Ole Mors, Anders D. BÃ¸rglum, Qibin Li, Jane H. Christensen, *BMC Genomics*, Vol. 16, No. 1. (25 July 2015), 548, doi:10.1186/s12864-015-1767-y

* [Galaxy Integrated Omics: Web-based standards-compliant workflows for proteomics informed by transcriptomics](http://dx.doi.org/10.1074/mcp.o115.048777)
    Jun Fan, Shyamasree Saha, Gary Barker, Kate J. Heesom, Fawaz Ghali, Andrew R. Jones, David A. Matthews, Conrad Bessant, *Molecular & Cellular Proteomics* (12 August 2015), mcp.O115.048777, doi:10.1074/mcp.o115.048777

* [From Peer-Reviewed to Peer-Reproduced in Scholarly Publishing: The Complementary Roles of Data Models and Workflows in Bioinformatics](http://dx.doi.org/10.1371/journal.pone.0127612)
    Alejandra González-Beltrán, Peter Li, Jun Zhao, Maria S. Avila-Garcia, Marco Roos, Mark Thompson, Eelke van der Horst, Rajaram Kaliyaperumal, Ruibang Luo, Tin-Lap Lee, Tak-wah Lam, Scott C. Edmunds, Susanna-Assunta Sansone, Philippe Rocca-Serra, *PLoS ONE*, Vol. 10, No. 7. (8 July 2015), e0127612, doi:10.1371/journal.pone.0127612

* [Experiences with workflows for automating data-intensive bioinformatics](http://dx.doi.org/10.1186/s13062-015-0071-8)
    Ola Spjuth, Erik Bongcam-Rudloff, Guillermo Carrasco C. Hernández, Lukas Forer, Mario Giovacchini, Roman Valls V. Guimera, Aleksi Kallio, Eija Korpelainen, Maciej M. Kańduła, Milko Krachunov, David P. Kreil, Ognyan Kulev, Paweł P. Łabaj, Samuel Lampa, Luca Pireddu, Sebastian Schönherr, Alexey Siretskiy, Dimitar Vassilev, *Biology Direct*, Vol. 10, No. 1. (19 August 2015), 43, doi:10.1186/s13062-015-0071-8

* [Challenges of Large-Scale Biomedical Workflows on the Cloud -- A Case Study on the Need for Reproducibility of Results](http://dx.doi.org/10.1109/cbms.2015.28)
    Sehrish Kanwal, Andrew Lonie, Richard O. Sinnott, Charlotte Anderson, In Computer-Based Medical Systems (CBMS), 2015 IEEE 28th International Symposium on (June 2015), pp. 220-225, doi:10.1109/cbms.2015.28


The new papers were tagged with:

<table>
  <tr>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th style=" width: 19%;"> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 18 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 22 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 57 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 12 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>



<div class='right'><a href='/News/First2500Papers'><img src='/First2500Grid.png' alt='First 2500 papers by tag and year' width="150" /></a></div>
 
# Galaxy's First 2500 Publications

And in case you missed it, we [reached 2500 publications](/src/News/First2500Papers/index.md) in the [Galaxy CiteULike Library](http://bit.ly/gxyCiteULike).  [This writeup](/src/News/First2500Papers/index.md) analyzes the papers by [CiteULike tag](/src/CiteULike/index.md#citeulike_tags) and journal, across the past 11 years, and identifies some trends.  Of those 2500 papers, less than 2% have a [Galaxy Team member](/src/GalaxyTeam/index.md) as a coauthor.  

---

# IUC Tools & Collections Hackathon

<div class='right'><a href='https://github.com/galaxyproject/tools-iuc/issues/239'><img src='/Images/Icons/DatasetCollectionIconInForm.png' alt='' width="150" /></a>
<div class='center'>**[Make this happen!](https://github.com/galaxyproject/tools-iuc/issues/239)**</div>
</div>

*We are planning a [remote hackathon](https://github.com/galaxyproject/tools-iuc/issues/239) on September 17<sup>th</sup> and 18<sup>th</sup> for [Galaxy tool developers to hack on dataset collections](https://github.com/galaxyproject/tools-iuc/issues/239).* 

Dataset collections enable !MapReduce style workflows in Galaxy and have come a long way over the last year+. Several groups are using dataset collections and newer tools to express workflows of various degrees of complexity that were not possible in Galaxy before.

The remote nature of this gives people who don't have the opportunity to come to GCC hackathons (which have always been productive and a lot of fun) a chance to participate in a Galaxy hackathon. Having a well defined topic will allow us to accomplish a lot and let people who don't have particular tasks in mind find something to work on very quickly.

We are [collecting ideas to work on](https://github.com/galaxyproject/tools-iuc/issues/239), but we expect to attract the most participation by simply getting tool developers interested in getting help adding collection support to their existing tools and workflows to show up and participate.

If you are interested in participating in the hackathon but not interested in actual tool development - we will assemble a list of smaller, manageable Python and !JavaScript tasks to work on and certainly documentation is a chronically lacking for collections so we could use help there and no actual coding would be required.

This is the first remote hackathon we have organized and we encourage ideas or advice about how to organize it please let us know. A core group of will be available on IRC all day and we will have 4 google hangouts across those days to organize, answer questions, and report progress. 

Thanks, <br />
Galaxy IUC


# Galaxy Project Google Calendars

If you want to know what events are happening in the Galaxy Community, the goto place has always been the [Galaxy Events page](/src/Events/index.md).  This lists any upcoming events we know about that feature Galaxy content.  This events list provides dates, locations, and contact information for each event.

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif' alt='Galaxy Calendar' /></a></div>

There has also been the complementary *[Galaxy Events Google Calendar](http://bit.ly/gxycal)* which listed events of interest to the community, whether or not they featured significant Galaxy content.  **To make this calendar more useful, we have split it in to two:**

 **[Galaxy Events Google Calendar](http://bit.ly/gxycal)** <a href='http://bit.ly/gxycalrss'><img src='/Images/Icons/RSSIcon16x16.gif' alt='Galaxy Calendar RSS Feed' /></a>
   This calendar now lists *only* events that feature Galaxy content.  Pretty much, if it's on the [Events page](/src/Events/index.md), it's also in this calendar.<br />

 **[Galaxy Other Events Google Calendar](http://bit.ly/gxyothercal)**  <a href='http://bit.ly/gxyothercalrss'><img src='/Images/Icons/RSSIcon16x16.gif' alt='Galaxy Calendar RSS Feed' /></a> 
   This lists additional events and deadlines that are relevant to the Galaxy Community, but that are not known to feature significant Galaxy content. 

This split allows us to present Galaxy events in a much sparser calendar, that is more useful for setting reminders and publishing Galaxy events. And we will continue to link to past Galaxy events (and presentations, videos, examples, ...) from the  *[Events Archive](/src/Events/index.md#past-events)*.

# August GalaxyAdmins Slides and Video

<div class='right'><a href='/Community/GalaxyAdmins/Meetups/2015_08_20'><img src='/Images/Logos/GalaxyAdmins.png' alt='GalaxyAdmins meetup August 20' width="175" /></a><br /><br />
<a href='http://www.irida.ca/'><img src='/Images/Logos/IRIDA.png' alt='IRIDA' width="175" /></a>
</div>

The [next GalaxyAdmins meetup on August 20](/src/Community/GalaxyAdmins/Meetups/2015_08_20/index.md) was attended by 27 *!GalaxyAdmins*.  [Aaron Petkau](https://github.com/apetkau) covered *Genomic data management at [Canada's National Microbiology Laboratory](https://www.nml-lnm.gc.ca/index-eng.htm) with [IRIDA](http://www.irida.ca/) and Galaxy* and [Enis Afgan](/src/EnisAfgan/index.md) presented *Adding transparency and automation into the Galaxy tool installation process*.

In case you missed it, [slides and video are now available online](/src/Community/GalaxyAdmins/Meetups/2015_08_20/index.md).

[GalaxyAdmins](/src/Community/GalaxyAdmins/index.md) is a special interest group for Galaxy community members who are responsible for Galaxy installations.  Our next meeting will be on [October 15](/src/Community/GalaxyAdmins/Meetups/2015_10_15/index.md) where [David Kovalic](https://www.linkedin.com/in/davidkovalic) of [Analome](http://analome.com) will present.  Hope to see you there.


# Upcoming Events

<div class='center'> 
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif' /></a></div>

There are [upcoming events](/src/Events/index.md) in five countries on three continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.


<div class='center'> 
<a href='http://escience2015.mnm-team.org/?page_id=463'><img src='/Images/Logos/eScience2015.png' alt='Scaling Up Bioinformatics Workflows with Dynamic Job Expansion: A Case Study Using Galaxy and Makeflow @ IEEE eScience 2015' height="65" /></a> &nbsp;
<a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-3/'><img src='/Images/Logos/QFABLogo.png' alt='Workshop: Extended RNA-Seq analysis' height="80" /></a>
<a href='http://biosb.nl/education/course-portfolio/rna-seq/'><img src='/Images/Logos/BioSBResearchSchool.png' alt='RNA-seq data analysis NGS Course' height="80" /></a>
<a href='http://training.bioinformatics.ucdavis.edu/2015/01/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-september-14-18-2015/'><img src='/Images/Logos/UCDavisGenomeCenter.png' alt='Using Galaxy for Analysis of High Throughput Sequence Data' height="50" /></a> &nbsp;
</div>

<table>
  <tr class="th" >
    <th style=" width: 7em;"> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> September 2 </th>
    <td> </em><a href='http://ccl.cse.nd.edu/research/papers/scaling-escience-2015.pdf'>Scaling Up Bioinformatics Workflows with Dynamic Job Expansion: A Case Study Using Galaxy and Makeflow</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) <a href='http://escience2015.mnm-team.org/'>IEEE eScience 2015</a>, Munich, Germany </td>
    <td> <a href='http://www3.nd.edu/~nhazekam/'>Nicholas Hazekamp</a> </td>
  </tr>
  <tr>
    <th> September 2-3 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-3/'>Workshop: Extended RNA-Seq analysis</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Australia) <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> <div class='right'><a href='/Teach/Trainers#qfab-bioinformatics'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> September 7-9 </th>
    <td> </em><a href='http://biosb.nl/education/course-portfolio/rna-seq/'>RNA-seq data analysis NGS Course</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) Leiden University Medical Center, The Netherlands </td>
    <td> <div class='right'> <a href='/Teach/Trainers#lumc-erasmusmc-dtl-learning-programme'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:education@biosb.nl">BioSB Education</a> </td>
  </tr>
  <tr>
    <th> September 14-18 </th>
    <td> </em><a href='http://training.bioinformatics.ucdavis.edu/2015/01/13/using-galaxy-for-analysis-of-high-throughput-sequence-data-september-14-18-2015/'>Using Galaxy for Analysis of High Throughput Sequence Data</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) <a href='http://bioinformatics.ucdavis.edu/'>UC Davis Bioinformatics Core</a>, Davis, California, United States </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnUCDavis'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:bioinformatics.core@ucdavis.edu">UC Davis Bioinformatics</a> </td>
  </tr>
  <tr>
    <th> September 15 </th>
    <td> </em><a href='https://hubzero.org/hubbub/speakers#yvan'>Towards accessible, reproducible, and transparent research in the life sciences: an innovative open source VRE approach in Western France</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) <a href='https://hubzero.org/hubbub/2015'>HUBbub 2015</a>, Indianapolis, Indiana, United States </td>
    <td> <a href='https://www.e-biogenouest.org/members/1003'>Yvan Le Bras</a> </td>
  </tr>
  <tr>
    <th> September 17-18 </th>
    <td> <a href='/News/ToolsCollectionsHack'>Remote Hackathon for Tools and Dataset Collections</a> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/World) </em>Online<em> </td>
    <td> <a href="mailto:galaxy-iuc@lists.galaxyproject.org">IUC</a> </td>
  </tr>
  <tr>
    <th> September 21-23 </th>
    <td> <a href='https://www.regonline.com/builder/site/Default.aspx?EventID=1692764'>JHU-DaSH: Data Science Hackathon</a> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) <a href='https://www.regonline.com/builder/site/tab2.aspx?EventID=1692764'>Baltimore</a>, Maryland, United States </td>
    <td> <a href="mailto:jhuDaSH@jhu.edu">Email</a> </td>
  </tr>
  <tr>
    <th> September 21-25 </th>
    <td> </em><a href='http://workflow4metabolomics.org/training/W4Mcourse2015'>Traitement des données métabolomiques sous Galaxy</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) Station Biologique de Roscoff, France </td>
    <td> <div class='right'><a href='http://bit.ly/gxytrnGUGGO'><img src='/Images/GalaxyLogos/GTN16.png' alt='Training offered by GTN Member' /></a></div> <a href="mailto:w4mcourse2015.organisation@sb-roscoff.fr">W4M Course Organisers</a> </td>
  </tr>
  <tr>
    <th> September 28 </th>
    <td> </em><a href='http://www.emgs-us.org/AM2015/agendamon.asp'>Mutational Analysis with Random DNA Identifiers (MARDI), a High-Fidelity NGS Approach That Simultaneously Identifies Gene Marker Mutations from Heterogeneous Mutant Cell Populations</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) <a href='http://www.emgs-us.org/AM2015/index.asp'>Environmental Mutagenesis and Genomics Society (EMGS)</a>, New Orleans, Loisiana, United States </td>
    <td> Javier Revollo </td>
  </tr>
  <tr>
    <th> </th>
    <td colspan=3 style=" background-color: #eef;"> </td>
  </tr>
  <tr>
    <th> October<br />12-14 </th>
    <td> </em><a href='https://cnrsformation.cnrs.fr/stage.php?stage=15274&axe=77'>Analyse avancée de séquences</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Europe) <a href='http://www.cbib.u-bordeaux2.fr/content/bienvenue-au-centre-de-bioinformatique-de-bordeaux'>Bordeaux Bioinformatics Center</a>, University Bordeaux Segalen, France </td>
    <td> <a href="mailto:cfe.contact@cnrs.fr">Email</a> </td>
  </tr>
  <tr>
    <th> october 15 </th>
    <td> </em><a href='/Community/GalaxyAdmins/Meetups/2015_10_15'>October Galaxy Admins Meetup</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/World) </em>Online<em> </td>
    <td> <a href='/HansrudolfHotz'>Hans-Rudolf Hotz</a>, <a href='/DaveClements'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> October 19 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-variant-detection-using-galaxy-2/'>Workshop: Variant detection using Galaxy</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Australia) <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> October 28-29 </th>
    <td> </em><a href='http://www.qfab.org/event/workshop-extended-rna-seq-analysis-4/'>Workshop: Extended RNA-Seq analysis</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/Australia) <a href='http://www.qfab.org/'>QFAB</a>, University of Queensland, St Lucia, Australia </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='http://www.qfab.org/mark-crowe/'>Mark Crowe</a> </td>
  </tr>
  <tr>
    <th> October 28 - November 3 </th>
    <td> </em><a href='http://meetings.cshl.edu/courses.aspx?course=c-ecg&year=15'>Computational & Comparative Genomics Course</a><em> </td>
    <td> PLACEHOLDER_INCLUDE(/Events/Badges/NorthAmerica) Cold Spring Harbor Laboratory, New York, United States </td>
    <td> PLACEHOLDER_INCLUDE(/Teach/GTN/Badge16) <a href='/JamesTaylor'>James Taylor</a> </td>
  </tr>
</table>


<table>
  <tr>
    <td style=" border: none;"> <a href='/Teach/Trainers'><img src='/Images/Icons/GTN32.png' alt='Designates a training event offered by GTN Member' width="24" /></a> </td>
    <td style=" border: none;"> <span style="font-size: smaller;"> <a href='/Teach/Trainers'>Designates a training event offered by GTN member(s)</a> </span> </td>
  </tr>
</table>


---

# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Systems Administrator IV](http://bit.ly/1NTuOfw), [Fred Hutchinson Cancer Institute](http://fredhutch.org), Seattle, Washington, United States 
* [Assistant Computational Scientist](http://bit.ly/1OLwPbW) at [VIB](http://www.vib.be/), Gent, Belgium 
* **The [Galaxy Project is hiring software engineers and post-docs](/src/GalaxyIsHiring/index.md)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/src/News/index.md) and include it in next month's [update](/src/GalaxyUpdates/index.md).

<br />

---

# New Public Galaxy Servers

[2 new publicly accessible Galaxy servers](/src/PublicGalaxyServers/index.md) were added in August.

## Workflow4Metabolomics

<div class='right'>
<div class='solid'><a href='http://galaxy.workflow4metabolomics.org/'><img src='/PublicGalaxyServers/Workflow4metabolomics.png' alt='Workflow4metabolomics Galaxy server' width=300 /></a></div>
<br />
<div class='right solid'>[{{attachment:PublicGalaxyServers/Workflow4MetabolomicsSponsors.png|Workflow4Metabolomics Sponsors|width="200"}}|&do=get,target="sponsorgraph"](ATTACHMENT_URLPublicGalaxyServers/Workflow4MetabolomicsSponsors.png)
<br /><div class='center'>*Sponsors*</div> 
</div></div>

* *Links:*
  * [workflow4metabolomics Galaxy server](http://galaxy.workflow4metabolomics.org/)
  * [Request an account](http://abims.sb-roscoff.fr/account)
  * [Workflow4metabolomics.org](http://workflow4metabolomics.org/)
* *Domain/Purpose:*
  * A collaborative portal dedicated to metabolomics data processing, analysis and annotation.
* *Comments:*
  * [LC-MS](http://workflow4metabolomics.org/the-lc-ms-workflow), [GC-MS](http://workflow4metabolomics.org/the-gc-ms-workflow) and [NMR](http://workflow4metabolomics.org/the-nmr-workflow) workflows are provided. 
  * See: Giacomoni F., Le Corguillé G., Monsoor M., Landi M., Pericard P., Pétéra M., Duperier C., Tremblay-Franco M., Martin J.-F., Jacob D., Goulitquer S., Thévenot E.A. and Caron C. (2014). [Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics](http://dx.doi.org/10.1093/bioinformatics/btu813). *Bioinformatics*, [doi: 10.1093/bioinformatics/btu813](http://dx.doi.org/10.1093/bioinformatics/btu813)
* *User Support:*
  * [HowTo Tutorials](http://workflow4metabolomics.org/howto) 
  * [support AT workflow4metabolomics DOT org](Help Desk)
* *Quotas:* 
  * Account required. Anyone can [request an account](http://abims.sb-roscoff.fr/account).
* *Sponsor(s):*
  * See [sponsor graph|&do=get,target="sponsorgraph"](ATTACHMENT_URLPublicGalaxyServers/Workflow4MetabolomicsSponsors.png)

## EpiToolKit

<div class='right solid'><a href='http://www.epitoolkit.de/'><img src='/PublicGalaxyServers/EpiToolKit_moleculepeptide.png' alt='EpiToolKit Galaxy Server' height="200" /></a></div>

* *Link:*
  * [EpiToolKit Galaxy](http://www.epitoolkit.de/)
* *Domain/Purpose:*
  * Computational immunology for the development of novel epitope-based vaccines.
* *Comments:*
  * [EpiToolKit](http://www.epitoolkit.de/) "provides a collection of methods from computational immunology for the development of novel epitope-based vaccines including HLA ligand or potential T-Cell epitope prediction, an epitope selection framework for vaccine design, and a method to design optimal string-of-beads vaccines. Additionally, [EpiToolKit](http://www.epitoolkit.de/) provides several other tools ranging from HLA typing based on NGS data, to prediction of polymorphic peptides."
  * See [EpiToolKit—a web-based workbench for vaccine design](http://bioinformatics.oxfordjournals.org/content/31/13/2211.full) by Benjamin Schubert, Hans-Philipp Brachvogel, Christopher Jürges and Oliver Kohlbacher, *Bioinformatics* (2015) 31 (13): 2211-2213. doi: 10.1093/bioinformatics/btv116
* *User Support:*
  * There is [extensive documentation](http://epitoolkit-wiki.informatik.uni-tuebingen.de/doku.php) inlcuding step-by-step instructions. 
* *Quotas:* 
  * Anyone can create an account. Alternatively, a guest account is provided with login test_user@informatik.uni-tuebingen.de and password workflowTest. Please note that the guest account shares uploaded data with all other guest users.
  * Data of unregistered users and delete data will be permanently deleted after 7 days.
* *Sponsor(s):*
  * [ABI EpiToolKit](http://www.epitoolkit.de/) is a service of the [Applied Bioinformatics Group at University of Tuebingen](http://www.kohlbacherlab.org/). 
<br />


---

# Releases

## Security Advisories

Two security vulnerabilities were recently discovered in Galaxy, and a third in the Galaxy Tool Shed. Because of this, the Galaxy Committers have released a number of fixes which have been rolled in to affected versions of Galaxy dating back to the 14.10 release.

* **[Galaxy XSS Reflection Vulnerability](https://lists.galaxyproject.org/pipermail/galaxy-announce/2015-August/000148.html)**
* **[Galaxy LDAP Authentication Vulnerability](https://lists.galaxyproject.org/pipermail/galaxy-announce/2015-August/000147.html)**
* **[Tool Shed Unauthorized Upload Vulnerability](https://lists.galaxyproject.org/pipermail/galaxy-announce/2015-August/000149.html)**


## Other Releases
<div class='right'><a href='http://getgalaxy.org'><img src='/Images/Logos/GalaxyNewLogo_GalaxyProject_Trans.png' alt='GalaxyProject' width="200" /></a></div>
 
 **May 2015 Galaxy Release (v 15.05)**
    [Release Notes v 15.05](http://galaxy.readthedocs.org/en/master/releases/15.05_announce.html)

 **!BioBlend 0.6.0 and 0.6.1**
    [BioBlend version 0.6.1](https://github.com/galaxyproject/bioblend/) was released in July. !BioBlend is a python library for interacting with CloudMan and the [Galaxy API](/src/Learn/API/index.md)(CloudMan offers an easy way to get a personal and completely functional instance of Galaxy in the cloud in just a few minutes, without any manual configuration.)  From the [release CHANGELOG](http://bit.ly/bbCHANGELOG):

<div class='right'><br /><a href='https://pypi.python.org/pypi/planemo/'><img src='/Images/Logos/PlanemoLogo.png' alt='' width="150" /></a></div>

 **Planemo 0.13.2**
   Planemo is a set of command-line utilities to assist in building tools for the Galaxy project.  [Planemo 0.13.2](https://pypi.python.org/pypi/planemo/0.13.2#history) is the most recent release. See the [release history](http://planemo.readthedocs.org/en/latest/history.html#id2).

 **Pulsar**
   [Pulsar 0.5.0](https://pypi.python.org/pypi/pulsar-app/0.5.0) was released in May.  Pulsar is a Python server application that allows a Galaxy server to run jobs on remote systems (including Windows) without requiring a shared mounted file systems. Unlike traditional Galaxy job runners - input files, scripts, and config files may be transferred to the remote system, the job is executed, and the results are transferred back to the Galaxy server - eliminating the need for a shared file system.
 
 **CloudMan **
   The [most recent edition of CloudMan](/src/GalaxyUpdates/2014_09/index.md#august-2014-cloudman-release) was released in August.

 **blend4j v0.1.2**
   [blend4j v0.1.2](https://github.com/jmchilton/blend4j) was released in December 2014.  blend4j is a JVM partial reimplemenation of the Python library bioblend for interacting with Galaxy, CloudMan, and !BioCloudCentral.


---


# Galaxy Community Hubs

<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Teach'><img src='/Images/Logos/GTNLogo600.png' alt='Galaxy Training Network' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Community/Logs'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/Community/Deployments'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong>Share your <a href='/Teach/Resources#add-a-training-resource'>training resources</a> and <a href='/Teach/Trainers#add-a-trainer'>experience</a> </em>now<em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/Community/Logs#add-a-log-page'>Share your experience</a> </em><a href='/Community/Logs#add-a-log-page'>now</a><em> <strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> </strong><a href='/Community/Deployments#add-your-galaxy-deployment'>Describe your instance</a> </em><a href='/Community/Deployments#add-your-galaxy-deployment'>now</a><em> <strong> </td>
  </tr>
</table>

<br />

One new new **[Galaxy Training Network member](/src/Teach/Trainers/index.md)** was added in August:

* **[University of Bradford, School of Medical Sciences, Krzysztof Poterlowicz](/src/Teach/Trainers/index.md#university-of-bradford-school-of-medical-sciences-krzysztof-poterlowicz)**
    Krzysztof offers training in Assembly, RNAseq, ChIPseq, Hi-C, DNase-Seq, and Pathway enrichment.

---
<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

See [list of tools contributed in August](/src/ToolShed/Contributions/2015_08/index.md).


# Other News

* [The Galaxy Reports application got a new homepage](https://twitter.com/galaxyproject/status/635893207520407552/photo/1) by [Daniel Bouchard](https://twitter.com/D_J_Bouchard)
