---
title: January 2015 Galaxy Update
---
<div class='left'><a href='/galaxy-updates/'><img src="/images/logos/GalaxyUpdate200.png" alt="Galaxy Updates" width=150 /></a></div>

Welcome to the January *[Galaxy Update](/galaxy-updates/)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/galaxy-updates/)* complement the *[Galaxy Development News Briefs](/docs/)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# New Papers

63 papers referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in December, *bringing the total to over 2000 papers*. Some of those new papers are:

* [Workflow4Metabolomics: A collaborative research infrastructure for computational metabolomics](https://doi.org/10.1093/bioinformatics/btu813)
    *Bioinformatics* (19 December 2014), btu813, doi:10.1093/bioinformatics/btu813, by Franck Giacomoni, Gildas Le Corguillé, Misharl Monsoor, *et al.*

* [MetaNET - a web-accessible interactive platform for biological metabolic network analysis](https://doi.org/10.1186/s12918-014-0130-2)
    *BMC Systems Biology*, Vol. 8, No. 1. (5 December 2014), doi:10.1186/s12918-014-0130-2, by Pankaj Narang, Shawez Khan, Anmol J. Hemrom, Andrew M. Lynn

* [ImmunoGlobulin galaxy (IGGalaxy) for simple determination and quantitation of immunoglobulin heavy chain rearrangements from NGS](https://doi.org/10.1186/s12865-014-0059-7)
    *BMC Immunology*, Vol. 15, No. 1. (13 December 2014), 59, doi:10.1186/s12865-014-0059-7, by Michael J. Moorhouse, David van Zessen, Hanna IJspeert, *et al.*<br />
    Also see the [Immunoglobulin Galaxy VM](http://bit.ly/IGGXY_VM)

* [VectorBase: an updated bioinformatics resource for invertebrate vectors and other organisms related with human diseases](https://doi.org/10.1093/nar/gku1117)
    *Nucleic Acids Research* (15 December 2014), doi:10.1093/nar/gku1117, by Gloria I. Giraldo-Calderón, Scott J. Emrich, Robert M. MacCallum, *et al.*

The new papers covered these topics:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none;"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 2 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 9 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 5 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 22 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 21 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>



# Events

## GCC2015: 6-8 July, Norwich UK

### Training Day Topic Nominations Close 6 January

<br />
<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://bit.ly/gcc2015tdnom'>Nominate a topic now!</a> &nbsp;&nbsp; </th>
  </tr>
</table>


<br />
[GCC2015 Training Day](http://gcc2015.tsl.ac.uk/training-day/) topics are [nominated by you](http://bit.ly/gcc2015tdnom), the Galaxy Community.  Please take a few minutes to nominate a topic.  Any topic of interest to the Galaxy Community can be nominated and you are encouraged to nominate more than one topic. If you are looking for ideas, see what topics were nominated in [2013](http://bit.ly/1i2j1gN) and [2014](http://bit.ly/1s6NtMN), and the [Events](/events/) and the [Events Archives](/events/archive/).

Nominated topics will be published on the [Training Day](http://gcc2015.tsl.ac.uk/training-day/) page as they come in.  **Nominations close 6 January**.  Topics will be compiled into a uniform list by the [GCC2015 Organising Committee](http://gcc2015.tsl.ac.uk/organisers), and topics will be voted on by the Galaxy Community 12-23 January.

Topics will then be selected and scheduled based on topic interest, and the organisers' ability to confirm instructors for each session.  Some very popular sessions may be scheduled more than once. The final schedule will be posted before registration opens.

[GCC2015 Organising Committee](http://gcc2015.tsl.ac.uk/organisers)

<div class='right'><a href='http://gcc2015.tsl.ac.uk/'><img src="/images/logos/GCC2015LogoWide600.png" alt="2015 Galaxy Community Conference (GCC2015)" width="250" /></a></div>

### Call for Sponsors

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is now accepting [Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/).  Your organisation can play a prominent part in the Galaxy community by sponsoring GCC2015.  Sponsorship is an excellent way to raise your organization’s visibility.

Several sponsorship levels are available, including two levels of premier sponsorships that include presentations.  Premium sponsorships are limited, however, so you are encouraged to act soon.

Please let the GCC2015 Organising Committee ([gcc2015-org AT lists DOT galaxyproject DOT org](mailto:gcc2015-org AT lists DOT galaxyproject DOT org)) know if you are interested in helping make this event a success.


## Other Events

<div class='center'>
<a href='https://pag.confex.com/pag/xxiii/webprogram/Session2641.html'><img src="/images/logos/PAGLogo114.png" alt="Galaxy @ Plant and Animal Genome (PAG 2015)" height="80" /></a>
<a href='/events/sdsu2014/'><img src="/images/logos/SDSULogo.jpg" alt="Galaxy Workshops @ San Diego State University" height="80" /></a> &nbsp;
<a href='http://www.vibconferences.be/event/revolutionizing-next-generation-sequencing-tools-and-technologies'><img src="/images/logos/VIBLogoBig.jpg" alt="Revolutionizing Next-Generation Sequencing: Tools and Technologies" height="75" /></a> &nbsp;
<a href='http://www.supagro.fr/web/pages/?page=1964&id_page=2666'><img src="/images/logos/SupAgroMontpelierLogo.jpg" alt="Analyse bioinformatique de séquences sous Galaxy" height="60" /></a>
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src="/images/icons/CalendarIcon.gif" /></a></div>

There are [upcoming events](/events/) on two continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> January 10-14 </th>
    <td> <em>Galaxy for SNP and Variant Data Analysis</em> </td>
    <td> <a href='http://www.intlpag.org/'>Plant and Animal Genome XXIII (PAG2014)</a>, San Diego, California, United States </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> January 15 </th>
    <td> <em><a href='/events/sdsu2014/'>Galaxy Workshops</a></em> </td>
    <td> <a href='http://sdsu.edu/'>San Diego State University</a>, San Diego, California, United States </td>
    <td> <a href='/people/dave-clements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> January 15-16 </th>
    <td> <em>Accessible and Reproducible Genomics at Scale with Galaxy</em> </td>
    <td> <a href='http://www.vibconferences.be/event/revolutionizing-next-generation-sequencing-tools-and-technologies'>Revolutionizing Next-Generation Sequencing: Tools and Technologies</a>, Leuven, Belgium </td>
    <td> <a href='/people/james-taylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <th> January 19-20 </th>
    <td> <em>NGS pipelines with Galaxy</em> </td>
    <td> <a href='http://www.uppnex.se/events/eInfraMPS2015'>e-Infrastructures for Massively Parallel Sequencing</a>, <a href='http://www.scilifelab.se/'>SciLifeLab</a>, Uppsala, Sweden </td>
    <td> <a href='http://www.crs4.it/crs4/peopledetails/people/148/Luca_Pireddu'>Luca Pireddu</a> </td>
  </tr>
  <tr>
    <th> February 9-13 </th>
    <td> <em><a href='http://www.supagro.fr/web/pages/?page=1964&id_page=2666'>Analyse bioinformatique de séquences sous Galaxy</a></em> </td>
    <td> Montpellier, France </td>
    <td> <a href="mailto:jean-francois.dufayard@cirad.fr">J.F. Dufayard</a> </td>
  </tr>
  <tr>
    <th rowspan=3> February 16-18 </th>
    <td> <em>Accessible and Reproducible Large-Scale Analysis with Galaxy</em> </td>
    <td rowspan=1> <a href='http://www.triconference.com/Sequencing-Analysis/'>Genome and Transcriptome Analysis</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='/people/james-taylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <td> <em>Large-Scale NGS data Analysis on Amazon Web Services Using Globus Genomic</em> </td>
    <td rowspan=2> <a href='http://www.triconference.com/Integration-Analysis-Visualization/'>Genomics & Sequencing Data Integration, Analysis and Visualization</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='http://www.ci.anl.gov/profile/250'>Ravi Madduri</a> </td>
  </tr>
  <tr>
    <td> <em>iReport: An Integrative “omics” Reporting and Visualisation Platform</em> </td>
    <td> Andrew Stubbs </td>
  </tr>
  <tr>
    <th> May 25-29 </th>
    <td> <a href='http://www.mipro.hr/MIPRO2015.DCVIS/ELink.aspx'>MIPRO</a> </td>
    <td> Opatija, Croatia </td>
    <td> <a href='/people/enis-afgan/'>Enis Afgan</a> </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> <strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a></strong> </td>
    <td> <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>


# GalaxyAdmins is back!

<div class='right'><a href='/community/galaxy-admins/'><img src="/images/logos/GalaxyAdmins.png" alt="GalaxyAdmins is back!" width="150" /></a></div>

*We just don't know exactly when, yet ...*

Except for a [meetup at GCC2014](/events/gcc2014/bofs/galaxy-admins/), the [GalaxyAdmins group](/community/galaxy-admins/) has been hiatus for a while.  Well, thanks to the efforts of [Hans-Rudolf Hotz](/people/hansrudolf-hotz/) the GalaxyAdmins group *is back.*   Starting this month, we will resume our bi-monthly conference calls.  

*Galaxy-Admins* is a discussion group for Galaxy community members who are responsible for large Galaxy installations.

In an effort to better serve the global community we will rotate the time of the call in different months to be convenient for different parts of the world.  To help us pick the best set of times for the calls, **please take a few minutes and fill out this [Doodle poll](http://doodle.com/dq3bexyuihs59h3c).**  It covers every hour in a 72 hour period.  *Please don't forget to select the timezone before you start filling it out.*

The likely presentation for the January meetup will be a summary of the Galaxy Community Survey last fall, presented by [Dave Clements](/people/dave-clements/).  The survey covered both Galaxy usage and deployment.  The survey results will be published online before the call.

<table>
  <tr>
    <th> &nbsp;&nbsp; <a href='http://doodle.com/dq3bexyuihs59h3c'>Take the Doodle poll</a> &nbsp;&nbsp; </th>
  </tr>
</table>



# Who's Hiring

<div class='right'><a href='/galaxy-is-hiring/'><img src="/images/news-graphics/GalaxyIsHiringWordCloud2.png" alt="Please Help! Yes you!" width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Systems Administrator / Information Technologist](/news/mc-master-opening/), [McArthur Lab](http://mcarthurbioinformatics.ca/) in the [McMaster University Department of Biochemistry & Biomedical Sciences](http://fhs.mcmaster.ca/biochem/)
* [Senior Development Engineer - Bioinformatics](http://bit.ly/1yGvefn), and [Bioinformatician II](http://bit.ly/1vBhgxB), University of Massachusetts Medical School
* Searching for [bioinformaticians, post-docs, PhD students and software engineers](/news/bioinformatician-freiburg2014/) in Freiburg, Germany at [Max Planck Institute of Immunobiology and Epigenetics](http://www.ie-freiburg.mpg.de/jobs), and the [Bioinformatics Group at the University of Freiburg](http://www.bioinf.uni-freiburg.de//?en)
* [Statistical Genomics Postdoc](/news/statistical-genomics-post-doc-penn-state/) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/galaxy-is-hiring/)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/news/) and include it in next month's [update](/galaxy-updates/).

<br />

# New Public Servers

2 new [public Galaxy servers](/use/) were added in December

## MetaNET

<div class='right solid'><a href='http://metanet.osdd.net/'><img src="/public-galaxy-servers/archive/metanet-logo-200.png" alt="MetaNET"  /></a></div>
* *Links:*
  * [MetaNET server](http://metanet.osdd.net/)
  * [MetaNET - a web-accessible interactive platform for biological metabolic network analysis](https://doi.org/10.1186/s12918-014-0130-2), by Pankaj Narang, Shawez Khan, Anmol J. Hemrom, Andrew M. Lynn, *BMC Systems Biology,* Vol. 8, No. 1. (5 December 2014), doi:10.1186/s12918-014-0130-2
* *Domain/Purpose:*
  * [MetaNET](http://metanet.osdd.net/) is a web-accessible interactive platform for biological metabolic network analysis.
* *Comments:*
  * From the [User Manual](http://metanet.osdd.net/static/user_manual.pdf):
      MetaNET is designed to provide a user-friendly rich interface for the analysis of genome-scale metabolic networks under various genetic and environmental conditions. The framework is built with a set of tools for data management including data upload/download, file format conversion, file operations and data extraction capabilities from SBML files, optimizing network using flux balance analysis, flux variability analysis, perturbation analysis via single or pairwise genes/reactions/catalysts knock-out. The tools can also be interconnected through workflows to perform simulations of higher order.
* *User Support:*
  * [User Manual](http://metanet.osdd.net/static/user_manual.pdf)
  * [Metanet User Group](https://groups.google.com/forum/#!forum/metanet-user)
  * [MetaNET Team](http://metanet.osdd.net/static/contact.html)
* *Sponsor(s):*
  * [School of Computational and Integrative Sciences](http://www.jnu.ac.in/SCIS/), [Jawaharlal Nehru University](http://www.jnu.ac.in/)

## VectorBase Galaxy

<div class='right solid'><a href='http://galaxy.vectorbase.org:8080/'><img src="/images/logos/VectorBaseLogoWithText.png" alt="VectorBase Galaxy" width=300 /></a></div>

* *Link:*
    * [Vectorbase Galaxy](http://galaxy.vectorbase.org:8080/)
* *Domain/Purpose:*
    * A full Galaxy server that includes reference information and workflows focusing on invertebrate vectors of human pathogens.
* *Comments:*
    * from [VectorBase: an updated bioinformatics resource for invertebrate vectors and other organisms related with human diseases](http://nar.oxfordjournals.org/content/early/2014/12/15/nar.gku1117.full), Gloria I. Giraldo-Calderón *et al.*, *Nucleic Acids Ressearch* (2014) doi: 10.1093/nar/gku1117:
      "[VectorBase](http://vectorbase.org/) has also made available the latest relevant canonical data in this Galaxy instance. Examples of workflow analyses include alignment of Next Generation Sequence (NGS) data sets, calculation of expression values, or predicting single nucleotide polymorphisms (SNPs). Registered users can store their raw data, intermediate files and final analysis results for download or direct sharing via the BRC, which greatly improves the ability to collaborate with colleagues and VectorBase developers."
* *User Support:*
    * [VectorBase Help Hub Page](https://www.vectorbase.org/navigation/help)
    * [VectorBase Help Desk](https://www.vectorbase.org/contact)
* *Sponsor(s):*
    * The [University of Notre Dame](http://www.nd.edu/), [EMBL-EBI](http://www.ebi.ac.uk/), and [Imperial College London](http://www.imperial.ac.uk/)

# Galaxy Community Hubs

<div class='solid'>
<table>
  <tr>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='https://training.galaxyproject.org/'><img src="/images/logos/GTNLogo600.png" alt="Galaxy Training Network" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <a href='/news/'><img src="/images/logos/LogBoardWText200.png" alt="Galaxy Community Log Board" height="100" /></a> </td>
    <td style=" text-align: center; border: none; width: 20%;"> <img src="/images/logos/GalaxyDeploymentCatalog200.png" alt="Galaxy Deployment Catalog" height="100" /> </td>
  </tr>
  <tr>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong>Share your training resources and experience <em>now</em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong><a href='/news/'>Share your experience</a> <em><a href='/news/'>now</a></em> </strong> </td>
    <td style=" text-align: center; vertical-align: top; border: none;"> <strong> </strong> </td>
  </tr>
</table>

</div>

## New Galaxy Training Resources

Several new Training Resources were added in December:

* UC Davis RNA-Seq and ChIP-Seq Analysis with Galaxy Workshop from the [UC Davis Bioinformatics Core](http://bioinformatics.ucdavis.edu/)
* Running your own Galaxy instance (in the cloud from the [Center for Health Bioinformatics](http://compbio.sph.harvard.edu/chb/) at the [Harvard School of Public Health](http://www.hsph.harvard.edu/)
* Introduction to Galaxy from the [Center for Health Bioinformatics](http://compbio.sph.harvard.edu/chb/) at the [Harvard School of Public Health](http://www.hsph.harvard.edu/)

# New Releases

[BioBlend v0.5.2 was released in October](/news/bio-blend-v0-5-2/).  [BioBlend](https://github.com/afgane/bioblend) is a python library for interacting with CloudMan and the [Galaxy API](/develop/api/).  

New versions of [Galaxy](/galaxy-updates/2014-09/#august-11-2014-galaxy-distribution), [CloudMan](/galaxy-updates/2014-09/#august-2014-cloudman-release), and [blend4j](/galaxy-updates/2014-09/#blend4j-011-release) were all released in August.

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src="/images/logos/ToolShed.jpg" alt="Galaxy ToolShed" width=150 /></a></div>

# ToolShed Contributions

## Galaxy Project ToolShed Repos

Here are new contributions for the past month.

In no particular order:

### Tools

* *From [rnateam](https://toolshed.g2.bx.psu.edu/view/rnateam):*
    * [rnabob](https://toolshed.g2.bx.psu.edu/view/rnateam/rnabob): Fast pattern searching for RNA structural motifs RNABOB is an implementation of D. Gautheret's RNAMOT, but with a different underlying algorithm using a nondeterministic finite state machine with node rewriting rules.

* *From [schang](https://toolshed.g2.bx.psu.edu/view/schang):*
    * [frp_tool](https://toolshed.g2.bx.psu.edu/view/schang/frp_tool): Scripts to create a fragment recruitment plot Python scripts using matlibplot to create scatter plots of metagenomic reads aligned against a reference genome.

* *From [subazini](https://toolshed.g2.bx.psu.edu/view/subazini):*
    * [ngsaligners](https://toolshed.g2.bx.psu.edu/view/subazini/ngsaligners): NGS aligners Aligners for NGS sequence analysis

* *From [mrvollger](https://toolshed.g2.bx.psu.edu/view/mrvollger):*
    * [trtr](https://toolshed.g2.bx.psu.edu/view/mrvollger/trtr): Version 1.0. TRTR Trim Reads of Tandem Repeats. Recommended before calling SNPs. This tool removes tandem repeats from ends of unaligned sequencing reads (leaving one copy). This prevents reads that don't span the repeated region from overlapping, leading to innaccurate SNPs calls.

     The maximum repeat length is adjustable (use 1 to trim only homopolymers).

     The "aggressive" option should not be touched in general. Setting to 0 will prevent the program from trimming to exactly 1 copy of the repeat, instead leaving between 1 and 2 copies.

     This could also be a useful first step before assembly. More testing needs to be done.

* *From [wolma](https://toolshed.g2.bx.psu.edu/view/wolma):*
    * [mimodd_fileinfo](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_fileinfo): use MiModD to explore metadata in various NGS file formats install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_bamsort](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_bamsort): use MiModD to sort a BAM file by coordinates (or names) of the mapped reads install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_deletion_prediction](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_deletion_prediction): use MiModD to predict deletions in one or more samples of aligned paired-end reads install this tool from the suite_mimodd_0_1_3 repository
    * [mimodd_convert](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_convert): use MiModD to convert between NGS reads sequence formats install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_reheader](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_reheader): use MiModD to reheader a BAM file install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_vcf_filter](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_vcf_filter): use MiModD to extract lines from a vcf variant file based on sample- and field-specific filters install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_snpeff_genomes](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_snpeff_genomes): use MiModD to list installed SnpEff genomes install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_extract_variants](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_extract_variants): use MiModD to extract variant sites from BCF input generated with mimodd_variant_calling and report them in VCF install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_ngs_run_annotation](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_ngs_run_annotation): use MiModD to generate a SAM format header from an NGS run description install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_variant_calling](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_variant_calling): use MiModD to call variants from an aligned reads BAM file install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_coverage_stats](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_coverage_stats): use MiModD to obtain a coverage report for a bcf file generated with mimodd_variant_calling install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_cloudmap_prepare](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_cloudmap_prepare): use MiModD to generate CloudMap-compatible output from a vcf file install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_read_alignment](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_read_alignment): use MiModD to align NGS reads to a reference genome install this tool from the suite_mimodd_0_1_5 repository
    * [mimodd_annotate_variants](https://toolshed.g2.bx.psu.edu/view/wolma/mimodd_annotate_variants): use MiModD to annotate a vcf variant file with information about the affected genes install this tool from the suite_mimodd_0_1_5 repository

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [cuffquant](https://toolshed.g2.bx.psu.edu/view/devteam/cuffquant): Cuffquant is part of Cufflinks. Cuffquant allows precalculation of gene expression levels. Output can be used in cuffdiff and cuffnorm.
    * [bwa](https://toolshed.g2.bx.psu.edu/view/devteam/bwa): Uploaded Wrapper for bwa mem. aln, sampe, and samse Al collection of Galaxy bwa wrapper based on version 0.7.10 (039ea206392ada2542bc41ff2581c53fa2fe2bf2).
    * [cuffnorm](https://toolshed.g2.bx.psu.edu/view/devteam/cuffnorm): Cuffnorm is part of Cufflinks. Cuffnorm is similar to cuffdiff, but does not perform differential expression testing. It provides normalized gene expression tables for use in downstream tools (R/matlab/...).

     Please cite: Trapnell C, Williams BA, Pertea G, Mortazavi AM, Kwan G, van Baren MJ, Salzberg SL, Wold B, Pachter L. Transcript assembly and abundance estimation from RNA-Seq reveals thousands of new transcripts and switching among isoforms. Nature Biotechnology doi:10.1038/nbt.1621
* [cummerbund](https://toolshed.g2.bx.psu.edu/view/devteam/cummerbund): Initial commit with version 1.0.0 of the cummeRbund wrapper. Wrapper for the Bioconductor cummeRbund library Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data. In addition, provides numerous plotting functions for commonly used visualizations.
* [cummerbund_to_tabular](https://toolshed.g2.bx.psu.edu/view/devteam/cummerbund_to_tabular): Initial commit with version 1.0.0 of the tool. Regenerate the tabular files generated by cuffdiff from a cummeRbund SQLite database. This tool extracts one or more of the original tabular data files from a cummeRbund SQLite database.

* *From [pjbriggs](https://toolshed.g2.bx.psu.edu/view/pjbriggs):*
    * [pal_finder](https://toolshed.g2.bx.psu.edu/view/pjbriggs/pal_finder): Find microsatellite repeat elements sequencing reads and design PCR primers to amplif y them  Runs the pal_finder Perl script and PRIMER3 to find microsatellite repeat elements sequencing reads and design PCR primers to amplify them
    * [trimmomatic](https://toolshed.g2.bx.psu.edu/view/pjbriggs/trimmomatic): A flexible read trimming tool for Illumina NGS data Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data.

* *From [eric-rasche](https://toolshed.g2.bx.psu.edu/view/eric-rasche):*
    * [gsaf_downloader](https://toolshed.g2.bx.psu.edu/view/eric-rasche/gsaf_downloader): Download data from GSAF Easy download utility for fastq.gz files provided by GSAF (Genomic Sequencing and Analysis Facility)

* *From [nikhil-joshi](https://toolshed.g2.bx.psu.edu/view/nikhil-joshi):*
    * [ceas](https://toolshed.g2.bx.psu.edu/view/nikhil-joshi/ceas): CEAS - Cis-regulatory Element Annotation System A tool designed to characterize genome-wide protein-DNA interaction patterns from ChIP-chip and ChIP-Seq of both sharp and broad binding factors.

* *From [shian_su](https://toolshed.g2.bx.psu.edu/view/shians):*
    * [voom_rnaseq](https://toolshed.g2.bx.psu.edu/view/shians/voom_rnaseq): Perform RNA-Seq analysis using limma voom pipeline

### Tool Suites

* *From [wolma](https://toolshed.g2.bx.psu.edu/view/wolma):*
    * [suite_mimodd_0_1_5](https://toolshed.g2.bx.psu.edu/view/wolma/suite_mimodd_0_1_5): This metapackage should be used to install the MiModD suite of tools for the analysis of genome-wide sequencing data from model organisms along with its Galaxy tool wrappers.

### Packages / Tool Dependency Definitions

* *From [wolma](https://toolshed.g2.bx.psu.edu/view/wolma):*
    * [package_mimodd_0_1_5](https://toolshed.g2.bx.psu.edu/view/wolma/package_mimodd_0_1_5): dependency package for the MiModD suite of tools
    * [package_python3_zlib_dependent_1_0](https://toolshed.g2.bx.psu.edu/view/wolma/package_python3_zlib_dependent_1_0): a lean build of Python3.4.1 including the zlib module - based on package_python_3_4 by jankanis Several modules of the Python standard library depend on external libraries being installed. Of these modules, this package forces only the installation of the zlib module, so the zlib library version 1.2.8 is its only requirement.
    * [package_zlib_1_2_8](https://toolshed.g2.bx.psu.edu/view/wolma/package_zlib_1_2_8): zlib library dependency definition

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
    * [package_cummerbund_2_8_2](https://toolshed.g2.bx.psu.edu/view/iuc/package_cummerbund_2_8_2): Contains a tool dependency definition that downloads and installs version 2.8.2 of the cummeRbund R library. Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data. In addition, provides numerous plotting functions for commonly used visualizations.
    * [package_r_3_1_2](https://toolshed.g2.bx.psu.edu/view/iuc/package_r_3_1_2):  Contains a tool dependency definition that downloads and compiles version 3.0.3 of the the R package.
* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [package_bowtie_2_2_4](https://toolshed.g2.bx.psu.edu/view/devteam/package_bowtie_2_2_4): tool dependency definition that downloads and compiles version 2.2.4 of the Bowtie package Contains a tool dependency definition that downloads and compiles version 2.2.4 of the Bowtie package
    * [package_cufflinks_2_2_1](https://toolshed.g2.bx.psu.edu/view/devteam/package_cufflinks_2_2_1): tool dependency definition that downloads and compiles version 2.2.1 of the cufflinks RNA-Seq suite. This repository is intended to be [defined as a complex repository dependency within a separate repository.](/toolshed/defining-repository-dependencies/#complex-repository-dependencies-tool-dependency-definitions-that-contain-repository-dependency-definitions)
    * [package_picard_1_126_0](https://toolshed.g2.bx.psu.edu/view/devteam/package_picard_1_126_0): tool dependency definition that downloads and compiles version 1.126.0 of the Picard package. This repository is intended to be [defined as a complex repository dependency within a separate repository.](/toolshed/defining-repository-dependencies/#complex-repository-dependencies-tool-dependency-definitions-that-contain-repository-dependency-definitions)
    * [package_freebayes_0_9_18_0059bdf](https://toolshed.g2.bx.psu.edu/view/devteam/package_freebayes_0_9_18_0059bdf): tool dependency definition that downloads and compiles version 0.9.18 of FreeBayes. Program: freebayes (Bayesian haplotype-based polymorphism discovery and genotyping.)  Version: 0.9.18 (0059bdf)

* *From [mrvollger](https://toolshed.g2.bx.psu.edu/view/mrvollger):*
    * [package_trtr_0_1](https://toolshed.g2.bx.psu.edu/view/mrvollger/package_trtr_0_1): Trim Reads of Tandem Repeat in a fastq file. This tool removes tandem repeats from ends of unaligned sequencing reads (leaving one copy). This prevents reads that don't span the repeated region from overlapping and leading to inaccurate SNPs calls.

### Select Updates

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/devteam):*
    * [ncbi_blast_plus](https://toolshed.g2.bx.psu.edu/view/devteam/ncbi_blast_plus): Uploaded v0.1.01 - Requires blastdbd datatype (blast_datatypes v0.0.19). Support for makeprofiledb to create protein domain databases and use them in RPS-BLAST and RPS-TBLASTN. Tools now support GI and SeqID filters, and embed the citations.

# Other News

* [Bioinformatics WikiBook Collaborative Workshop](http://bit.ly/15PpUxA) 24-26 February at TGAC.  And we encourage contributions [about Galaxy](http://bit.ly/1zS0PuW) to the [NGS WikiBook](http://bit.ly/NGSWikiBook).
* [PDACS: a portal for data analysis services for cosmological simulations](http://bit.ly/1s0gMLe), Chard *et al.*  *PDACS is a Galaxy implementation for cosmology.* And PDACS slides are [here](http://bit.ly/1pYSlSN).  
* Just scheduled: Intro to variant calling for pathologists and laboratory managers, Sydney, 22 June. [Ross Lazarus](/people/fubar/) & Andrew Lonie
* [ASH: A Galaxy VM](http://bit.ly/ASH_VM) from Erasmus Medical Centre for Automated Selection of Hotspots in cancer.
