---
autotoc: true
---
<div class="title">December 2014 Galaxy Update</div>


<div class='left'><a href='/GalaxyUpdates/'><img src='/Images/Logos/GalaxyUpdate200.png' alt='Galaxy Updates' width=150 /></a></div>

Welcome to the December 2014 *[Galaxy Update](/GalaxyUpdates)*, a summary of what is going on in the Galaxy community. *[Galaxy Updates](/GalaxyUpdates)* complement the *[Galaxy Development News Briefs](/DevNewsBriefs)* which accompany new Galaxy releases and focus on Galaxy code updates.  

# Events

## Galaxy Day: 3 December, Paris

The [French Working Group GALAXY-IFB (Institut Français de Bioinformatique)](http://www.ifb-galaxy.org/index.html) is organizing a second [Galaxy Day](http://www.ifb-galaxy.org/GALAXYDAYS2014.html) around the Galaxy portal.  The event will be at Institut Curie in Paris over two days.  This year, we want to involve two communities: *biologists* (also known as Galaxy 'users') and *bioinformaticians* (Galaxy 'developers').  The goal is to present user experience around the portal, from a single user to a wider community:

* Dec 3 (09:00-17:00)
  * Galaxy's user experiences, and discussion on how the platform is (or is not) useful for building analysis.
  * Technology talks (new environment, Galaxy in production, ...)

Interested? Please contact [ifb DOT galaxy AT sb DASH roscoff DOT fr](ifb DOT galaxy AT sb DASH roscoff DOT fr) for more information.

The [French IFB Galaxy Working Group](http://www.ifb-galaxy.org/index.html):

[URGI](http://urgi.versailles.inra.fr/), [GenoToul](http://bioinfo.genotoul.fr/), [MIGALE](http://migale.jouy.inra.fr/), [PFEM](http://metabohub.fr/), [SouthGreen](http://southgreen.cirad.fr/), [Institut Curie](http://curie.fr/), [ABiMS](http://abims.sb-roscoff.fr/)

<div class='center'>
<a href='http://urgi.versailles.inra.fr/'><img src='/Images/Logos/URGILogo.png' alt='URGI' width="70" /></a> &nbsp;
<a href='http://bioinfo.genotoul.fr/'><img src='/Images/Logos/GenoToulLogo.png' alt='GenoToul' width="100" /></a> &nbsp;
<a href='http://migale.jouy.inra.fr/'><img src='/Images/Logos/MIGALELogo.png' alt='MIGALE' width="120" /></a> &nbsp;
<a href='http://metabohub.fr/'><img src='/Images/Logos/MetaboHubLogo.jpg' alt='PFEM' width="120" /></a> &nbsp;
<a href='http://southgreen.cirad.fr/'><img src='/Images/Logos/SouthGreenLogo.png' alt='SouthGreen' width="120" /></a>
<a href='http://curie.fr/'><img src='/Images/Logos/CurieLogo.gif/' alt='Institut Curie' width="80" /></a> &nbsp;
<a href='http://abims.sb-roscoff.fr/'><img src='/Images/Logos/ABiMSLogo.png' alt='ABiMS' width="70" /></a>
</div>

## Intro to Galaxy Workshop, Dec 12, Virginia State U

<div class='left'><a href='/Events/VSU2014/'><img src='/Images/Logos/VSULogo.jpg' alt='Virginia State University' width="125" /></a></div>

If you are anywhere close to Richmond, Virginia and you want to attend an [Introduction to Galaxy Workshop](/Events/VSU2014), then you are in luck. Thanks to sponsorship from the [College of Natural and Health Sciences](http://www.cnhs.vsu.edu/index.php) at [Virginia State University](http://vsu.edu/), the workshop is free and open to all researchers and students.  The [workshop](/Events/VSU2014) is split into morning and afternoon sessions.  No knowledge of programming or command line interfaces is required.

However, space is limited, so you are encouraged to register ***[right now](/Events/VSU2014)***.

## GCC2015: 6-8 July, Norwich UK

<div class='right'><a href='http://gcc2015.tsl.ac.uk/'><img src='/Images/Logos/GCC2015LogoWide600.png' alt='2015 Galaxy Community Conference (GCC2015)' width="250" /></a></div>

<span style="font-size: larger;"> Save these dates: 6-8 July 2015 </span>

### Call for Sponsors

The [2015 Galaxy Community Conference (GCC2015)](http://gcc2015.tsl.ac.uk/) is now accepting [Sponsorships](http://gcc2015.tsl.ac.uk/sponsors/).  Your organisation can play a prominent part in the Galaxy community by sponsoring GCC2015.  Sponsorship is an excellent way to raise your organization’s visibility.

Several sponsorship levels are available, including two levels of premier sponsorships that include presentations.  Premium sponsorships are limited, however, so you are encouraged to act soon.

Please let the GCC2015 Organising Committee ([gcc2015-org AT lists DOT galaxyproject DOT org](gcc2015-org AT lists DOT galaxyproject DOT org)) know if you are interested in helping make this event a success.

### GCC2015 Training Day Topic Nominations ...

... will open shortly.  The topics offered at the GCC2015 Training Day will be determined by you, the Galaxy Community.  Topic nominations will open shortly (watch those Galaxy channels), and nominate topics will be voted on by the community early next year.

Please start giving some thought to what topics you would like to see covered at the GCC2015 Training Day.

## Other Events

<div class='center'> 
<a href='http://www.ifb-galaxy.org/GALAXYDAYS2014.html'><img src='/Images/Logos/GalaxyIFB.png' alt='Galaxy Day' width="140" /></a>&nbsp;
<a href='/Events/VSU2014/'><img src='/Images/Logos/VSULogo.jpg' alt='Intro to Galaxy Workshop at Virginia State University' height="70" /></a>
<a href='https://pag.confex.com/pag/xxiii/webprogram/Session2641.html'><img src='/Images/Logos/PAGLogo114.png' alt='Galaxy @ Plant and Animal Genome (PAG 2015)' height="80" /></a>
<a href='http://www.supagro.fr/web/pages/?page=1964&id_page=2666'><img src='/Images/Logos/SupAgroMontpelierLogo.jpg' alt='Analyse bioinformatique de séquences sous Galaxy' height="60" /></a>
</div>

<div class='right'><a href='http://bit.ly/gxycal'><img src='/Images/Icons/CalendarIcon.gif/' /></a></div>

There are [upcoming events](/Events) on two continents.  See the [Galaxy Events Google Calendar](http://bit.ly/gxycal) for details on other events of interest to the community.

<table>
  <tr class="th" >
    <th> Date </th>
    <th> Topic/Event </th>
    <th> Venue/Location </th>
    <th> Contact </th>
  </tr>
  <tr>
    <th> December 3 </th>
    <td> <em><a href='http://www.ifb-galaxy.org/GALAXYDAYS2014.html'>Galaxy Day</a></em> </td>
    <td> Institut Curie, Paris, France </td>
    <td> <a href="mailto:ifb DOT galaxy AT sb DASH roscoff DOT fr">IFB Galaxy</a> </td>
  </tr>
  <tr>
    <th> December 5-8 </th>
    <td> <em><a href='http://manuals.bioinformatics.ucr.edu/workshops'>Next Generation Data Analysis Workshop</a></em> </td>
    <td> UC Riverside, Riverside, California, United States </td>
    <td> <a href="mailto:rkaundal AT ucr DOT edu">Rakesh Kaundal</a> </td>
  </tr>
  <tr>
    <th> December 9-11 </th>
    <td> <em><a href='http://www.pf-bird.univ-nantes.fr/85361781/0/fiche___pagelibre/&RH=1357829686766'>Microarray data analysis on Galaxy</a></em> </td>
    <td> <a href='http://www.pf-bird.univ-nantes.fr/'>BIRD IFB core facility</a> Nantes University/INSERM, Nantes, France </td>
    <td> <a href="mailto:raluca DOT teusan AT inserm DOT fr">Raluca Teusan</a>, Audrey Bihouée, Edouard Hirchaud</td>
  </tr>
  <tr>
    <th> December 12 </th>
    <td> <em><a href='/Events/VSU2014/'>Introduction to Galaxy Workshop</a></em> </td>
    <td> <a href='http://vsu.edu/'>Virginia State University</a>, Petersburg, Virginia, United States </td>
    <td> <a href='http://sest.vsu.edu/biology/Faculty/harris.html'>Glenn Harris</a>, <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> December 16-19 </th>
    <td> <em><a href='https://training.bioinformatics.ucdavis.edu/2014/10/10/rna-seq-and-chip-seq-analysis-with-galaxy-december-16-19-2014/'>RNA-Seq and ChIP-Seq Analysis with Galaxy</a></em> </td>
    <td> <a href='http://training.bioinformatics.ucdavis.edu/'>UC Davis</a>, California, United States </td>
    <td> <a href="mailto:bioinformatics DOT core AT ucdavis DOT edu">UC Davis Bioinformatics Training</a> </td>
  </tr>
  <tr>
    <th colspan=4> 2015 </th>
  </tr>
  <tr>
    <th> January 10-14 </th>
    <td> <em>Galaxy for SNP and Variant Data Analysis</em> </td>
    <td> <a href='http://www.intlpag.org/'>Plant and Animal Genome XXIII (PAG2014)</a>, San Diego, California, United States </td>
    <td> <a href='/DaveClements/'>Dave Clements</a> </td>
  </tr>
  <tr>
    <th> February 9-13 </th>
    <td> <em><a href='http://www.supagro.fr/web/pages/?page=1964&id_page=2666'>Analyse bioinformatique de séquences sous Galaxy</a></em> </td>
    <td> Montpellier, France </td>
    <td> <a href="mailto:jean-francois.dufayard@cirad.fr">J.F. Dufayard</a> </td>
  </tr>
  <tr>
    <th rowspan=3> February 16-18 </th>
    <td> <em>Accessible and Reproducible Large-Scale Analysis with Galaxy</em> </td>
    <td rowspan=1> <a href='http://www.triconference.com/Sequencing-Analysis/'>Genome and Transcriptome Analysis</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='/JamesTaylor/'>James Taylor</a> </td>
  </tr>
  <tr>
    <td> <em>Large-Scale NGS data Analysis on Amazon Web Services Using Globus Genomic</em> </td>
    <td rowspan=2> <a href='http://www.triconference.com/Integration-Analysis-Visualization/'>Genomics & Sequencing Data Integration, Analysis and Visualization</a>, part of Molecular Medicine Tri-Conference, San Francisco, California, United States </td>
    <td> <a href='http://www.ci.anl.gov/profile/250'>Ravi Madduri</a> </td>
  </tr>
  <tr>
    <td> <em>iReport: An Integrative “omics” Reporting and Visualisation Platform</em> </td>
    <td> Andrew Stubbs </td>
  </tr>
  <tr>
    <th> July 6-8 </th>
    <td> </strong><a href='http://gcc2015.tsl.ac.uk/'>2015 Galaxy Community Conference (GCC2015)</a><strong> </td>
    <td> <a href='http://www.tsl.ac.uk/'>The Sainsbury Lab</a>, <a href='http://www.visitnorwich.co.uk/'>Norwich</a>, United Kingdom </td>
    <td> <a href="mailto:outreach@galaxyproject.org">Galaxy Outreach</a> </td>
  </tr>
</table>



# New Papers

**96 papers** (a new record) referencing, using, extending, and implementing Galaxy were added to the [Galaxy CiteULike Group](http://www.citeulike.org/group/16008/) in November.  That's a new record.  Some of those papers are:

* "[Galaxy Cluster to Cloud - Genomics at Scale](http://dx.doi.org/10.1109/GCE.2014.13)" by [Enis Afgan](/EnisAfgan), *et al.* in [Proceedings of the 9th Gateway Computing Environments Workshop (2014)](http://dl.acm.org/citation.cfm?id=2690887&picked=prox&cfid=603148869&cftoken=86409151), pp. 47-50, doi:10.1109/gce.2014.13

* "[Automatically exposing OpenLifeData via SADI semantic Web Services](http://dx.doi.org/10.1186/2041-1480-5-46)" by Alejandro Gonzalez, *et al.* *Journal of Biomedical Semantics*, Vol. 5, No. 1. (2014), 46, doi:10.1186/2041-1480-5-46

* "[Using phylogenetically-informed annotation (PIA) to search for light-interacting genes in transcriptomes from non-model organisms](http://dx.doi.org/10.1186/s12859-014-0350-x)" by Daniel I. Speiser, *et al.* *BMC Bioinformatics*, Vol. 15, No. 1. (19 November 2014), 350, doi:10.1186/s12859-014-0350-x

* "[A case study for cloud based high throughput analysis of NGS data using the Globus Genomics system](http://dx.doi.org/10.1016/j.csbj.2014.11.001)" by Krithika Bhuvaneshwar, *et al.* *Computational and Structural Biotechnology Journal* (November 2014), doi:10.1016/j.csbj.2014.11.001

* "[RNA-Seq Analysis of Differential Gene Expression in Electroporated Chick Embryonic Spinal Cord](http://bit.ly/112U5OV)" by Felipe M. Vieceli, C. Y. Irene Yan, *Journal of Visualized Experiments,* No. 93. (1 November 2014), doi:10.3791/51951

* "[A Hadoop-Galaxy Adapter for User-friendly and Scalable Data-intensive Bioinformatics in Galaxy](http://dx.doi.org/10.1145/2649387.2649429)" by Luca Pireddu, *et al.* in Proceedings of the 5th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (2014), pp. 184-191, doi:10.1145/2649387.2649429

The new papers covered these topics:

<table>
  <tr>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=5 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
    <td rowspan=4 style=" border: none"> &nbsp;&nbsp; </td>
    <th> # </th>
    <th> Tag </th>
  </tr>
  <tr>
    <td style=" text-align: right;"> 4 </td>
    <td> <a href='http://bit.ly/gxyculcloud'>Cloud</a>       </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculproject'>Project</a>                  </td>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://bit.ly/gxycultools'>Tools</a>       </td>
    <td style=" text-align: right;"> 13 </td>
    <td> <a href='http://bit.ly/gxyculusepublic'>UsePublic</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculhowto'>HowTo</a>       </td>
    <td style=" text-align: right;"> 10 </td>
    <td> <a href='http://bit.ly/gxyculrefpublic'>RefPublic</a>              </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculusecloud'>UseCloud</a> </td>
    <td style=" text-align: right;"> - </td>
    <td> <a href='http://bit.ly/gxyculviz'>Visualization</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 7 </td>
    <td> <a href='http://bit.ly/gxyculisgalaxy'>IsGalaxy</a> </td>
    <td style=" text-align: right;"> 1 </td>
    <td> <a href='http://bit.ly/gxyculrepro'>Reproducibility</a>            </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculuselocal'>UseLocal</a> </td>
    <td style=" text-align: right;"> 32 </td>
    <td> <a href='http://bit.ly/gxyculworkbench'>Workbench</a> </td>
  </tr>
  <tr>
    <td style=" text-align: right;"> 34 </td>
    <td> <a href='http://bit.ly/gxyculmethods'>Methods</a>   </td>
    <td style=" text-align: right;"> 3 </td>
    <td> <a href='http://bit.ly/gxyculshared'>Shared</a>                    </td>
    <td style=" text-align: right;"> 8 </td>
    <td> <a href='http://bit.ly/gxyculusemain'>UseMain</a>   </td>
  </tr>
</table>


# Who's Hiring

<div class='right'><a href='/GalaxyIsHiring/'><img src='/GalaxyIsHiring/GalaxyIsHiringWordCloud2.png' alt='Please Help! Yes you!' width="200" /></a></div>

The Galaxy is expanding! Please help it grow.

* [Senior Development Engineer - Bioinformatics](http://it.ly/1yGvefn), and [Bioinformatician II](http://bit.ly/1vBhgxB), University of Massachusetts Medical School
* Searching for [bioinformaticians, post-docs, PhD students and software engineers](/News/BioinformaticianFreiburg2014) in Freiburg, Germany at [Max Planck Institute of Immunobiology and Epigenetics](http://www.ie-freiburg.mpg.de/jobs), and the [Bioinformatics Group at the University of Freiburg](http://www.bioinf.uni-freiburg.de//?en)
* [Research Specialist](http://bit.ly/13pqd0T), Michigan State University, United States
* [Computational Science Developer I](http://bit.ly/1lLkAMR), Cold Spring Harbor Laboratory (CSHL), New York, United States 
* [Statistical Genomics Postdoc](/News/StatisticalGenomicsPostDocPennState) opening in the Makova lab at Penn State
* **The [Galaxy Project is hiring software engineers and post-docs](/GalaxyIsHiring)**

Got a Galaxy-related opening? Send it to outreach@galaxyproject.org and we'll put it in the [Galaxy News feed](/News) and include it in next month's [update](/GalaxyUpdates).

<br />
# Mailing Lists Moved

The [Galaxy mailing lists](/MailingLists) have been moved from the lists.bx.psu.edu domain to the lists.galaxyproject.org domain.  This transition should be largely transparent, but there are a few things to be aware of:
* List sender addresses and headers will change to reflect the updated domain: from bx.psu.edu to galaxyproject.org.  
* Existing email filters you have set up may require adjustments.
* Posts from lists.galaxyproject.org could be categorized as spam until you train your filtering method.

The prior bx.psu.edu list posting email addresses will continue to accept email, which will be forwarded to the new list addresses.

# New Galaxy-Training Mailing List

The [Galaxy-Training mailing list](https://lists.galaxyproject.org/listinfo/galaxy-training) is for anything related to training with Galaxy and training about Galaxy.  Galaxy-Training is also the official mailing of the [Galaxy Training Network](/Teach/GTN), a [world-wide](/Teach/Trainers) group of training organizations that give Galaxy-based training.

If you are at all interested in Galaxy Training then you are encouraged to [join the list](https://lists.galaxyproject.org/listinfo/galaxy-training). Messages to this list are [publicly archived](http://galaxy-training-mailing-list-archive.35427.n7.nabble.com/) and will be included in the [Galaxy search engines](http://galaxyproject.org/search/web/) as well.

<div class='right'><a href='/Teach/'><img src='/Images/Logos/GTNLogo600.png' alt='Galaxy Training Network' width="200" /></a></div>

*And please welcome the latest member of the Galaxy Training Network:*

* [HSPH Bioinformatics Core (HBC)](http://bit.ly/gxytrnhbc), Boston, Massachussetts, United States

This brings total membership to [22 training organizations on 5 continents](http://bit.ly/gxytrnmap).

# New Public Servers

Two new [public Galaxy servers](/PublicGalaxyServers) were announced in November:

## Pitagora-Galaxy

<div class='right'><div class='solid'><a href='http://try.pitagora-galaxy.org/galaxy/'><img src='/PublicGalaxyServers/Pitagora.png' alt='Pitagora-Galaxy' width="300" /></a></div>

* *Links:*
  * [Pitagora-Galaxy Server](http://try.pitagora-galaxy.org/galaxy/)
  * [Pitagora-Galaxy Project](http://www.pitagora-galaxy.org/about_en)
* *Domain/Purpose:*
  * The public, general purpose Galaxy servers of the [Pitagora-Galaxy Project](http://www.pitagora-galaxy.org/about_en). This server is intended for testing and sharing. Heavy analysis should be performed using the project's identical virtual machine (VM) or Amazon Machine Image (AMI).
* *Comments:*
  * "We are running a website for sharing users' know-how, and distributing a virtual environment where we configured Galaxy with selected workflows and tools. Now, you can perform our analysis workflows on the following three environments.
    * Access to the public web site for testing.
    * Download the virtual machine to your own PC or server.
    * Launch AMI (Amazon machine image) on AWS cloud.
    Since Pitagora-Galaxy enables us to run the same workflows on any infrastructure and rebuild the environments in any time, we can quickly use Galaxy, and at the same time, ensure the reproducibility of the analyses. In addition, we plan to add a connector for Garuda Desktop, a desktop application platform, for data analyses that cannot be covered only with Galaxy tools.
* *User Support:*
  * Email: [yamanaka AT genome DOT rcast.u-tokyo.ac.jp](Ryota Yamanaka)
* *Quotas:* 
  * Public Server:
    * See the [instructions](http://try.pitagora-galaxy.org/galaxy/) at Pitagora-Galaxy Server.
  * VM and AMI:
    * None.
* *Sponsor(s):*
  * [Genome Science Division, RCAST, The University of Tokyo](http://www.genome.rcast.u-tokyo.ac.jp/en/)
  * [Laboratory for Disease Systems Modeling, IMS, RIKEN](http://www.riken.jp/en/research/labs/ims/dis_sys_model/)
  * [Database Center for Life Science, ROIS](http://dbcls.rois.ac.jp/en/)

## PIA

<div class='right'><div class='solid'><a href='http://galaxy-dev.cnsi.ucsb.edu/pia/'><img src='/PublicGalaxyServers/PiaImage.jpg' alt='PIA' /></a></div>

* *Links:*
  * [PIA Galaxy](http://galaxy-dev.cnsi.ucsb.edu/pia/)
  * "[Using phylogenetically-informed annotation (PIA) to search for light-interacting genes in transcriptomes from non-model organisms](http://www.biomedcentral.com/1471-2105/15/350/abstract)," by Speiser, *et al.* in *[BMC Bioinformatics](http://www.biomedcentral.com/bmcbioinformatics/)* 2014, 15:350  doi:10.1186/s12859-014-0350-x
  * [PIA code in Bitbucket](https://bitbucket.org/osiris_phylogenetics/pia/)
* *Domain/Purpose:*
  * Pylogenetics and gene interaction
* *Comments:*
  * "PIA (Phylogenetically Informed Annotation) is a set of tools for the Galaxy Bioinformatics Platform. In general, PIA uses BLAST, an alignment program, and RAxML's read placement algorithm to put unknown sequences into pre-calculated phylogenetic trees. We provide 102 genes called LIT (Light Interaction Toolkit) - vision genes like phototransduction genets - for use in PIA."
* *User Support:*
  * [PIA Manual](https://bytebucket.org/osiris_phylogenetics/pia/raw/05d213dcb31e4cda9ccc68201ee7f8854161b2c9/docs/PIAwebManual_090914.pdf)
  * [Getting Started with PIA](https://www.youtube.com/watch?v=m23KxhxhTAE) screencast.
* *Quotas:* 
* *Sponsor(s):*
  * [Oakley Lab](https://labs.eemb.ucsb.edu/oakley/todd/) at [UC Santa Barbara](http://www.ucsb.edu/)

# Galaxy Community Hubs

<div class='left'>&nbsp;&nbsp; <a href='/Community/Logs/'><img src='/Images/Logos/LogBoardWText200.png' alt='Galaxy Community Log Board' height="100" /></a></div>
<div class='right'><a href='/Community/Deployments/'><img src='/Images/Logos/GalaxyDeploymentCatalog200.png' alt='Galaxy Deployment Catalog' height="100" /></a> &nbsp;&nbsp;</div>
<br />

<table>
  <tr>
    <td style=" class="blue"  text-align: center;"> &nbsp;&nbsp; <a href='/Community/Logs/#add-a-log-page'>Share your experience</a> <em><a href='/Community/Logs/#add-a-log-page'>now</a> </em> &nbsp;&nbsp;<br /><a href='/Community/Deployments/#add-your-galaxy-deployment'>Describe your instance</a> <em><a href='/Community/Deployments/#add-your-galaxy-deployment'>now</a></em> </td>
  </tr>
</table>

<br /><br />

**There were no new [Log Board](/Community/Logs) or [Deployment Catalog](/Community/Deployments) entries in November!**  Eek!  Please don't let this happen again!  

The *[Community Log Board](/Community/Logs)* and  *[Deployment Catalog](/Community/Deployments)* *[Galaxy community hubs](/Community)* were launched last your.  If you have a [Galaxy deployment](/Community/Deployments), or [experience](/Community/Logs) you want to share then please publish them **this month.**


# New Releases

[BioBlend v0.5.2 was released in October](/News/BioBlendV0.5.2).  [BioBlend](https://github.com/afgane/bioblend) is a python library for interacting with CloudMan and the [Galaxy API](/Learn/API).  

New versions of [Galaxy](/GalaxyUpdates/2014_09/#august-11-2014-galaxy-distribution), [CloudMan](/GalaxyUpdates/2014_09/#august-2014-cloudman-release), and [blend4j](/GalaxyUpdates/2014_09/#blend4j-011-release) were all released in August.

Look for a new Galaxy distribution soon.

<div class='right'><br /><a href='http://toolshed.g2.bx.psu.edu/'><img src='/Images/Logos/ToolShed.jpg' alt='Galaxy ToolShed' width=150 /></a></div>

# ToolShed Contributions

## Galaxy Project ToolShed Repos

Here are new contributions for the past *two* months.

In no particular order: 

### Tools

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/peterjc):*
  * [mira_datatypes](https://toolshed.g2.bx.psu.edu/view/peterjc/mira_datatypes):  Defines 'mira' datatype for the MIRA Assembly Format. Note that Galaxy already has a 'maf' datatype for the Multiple (sequence) Alignment Format (MAF). This is specifically for the MIRA Assembly Format (also called MAF). Previously only on the Test Tool Shed.
  * [clc_assembly_cell](https://toolshed.g2.bx.psu.edu/view/peterjc/clc_assembly_cell): Galaxy wrapper for the CLC Assembly Cell suite from CLCBio. This is a wrapper for the commercial "CLC Assembly Cell" suite from CLCBio which includes a de novo assembler and read mapper: http://www.clcbio.com/products/clc-assembly-cell/  Uploaded v0.0.2, previously only on the Test Tool Shed. 
  * [seq_composition](https://toolshed.g2.bx.psu.edu/view/peterjc/seq_composition): Uploaded v0.0.1 (with embedded citation). Sequence composition. Counts the letters in given sequence files, returning a table listing them with percentages. Suitable for use on assemblies or gene/protein sets. Probably not suitable for raw NGS reads.
  * [mira4_assembler](https://toolshed.g2.bx.psu.edu/view/peterjc/mira4_assembler): MIRA 4.0 assembler Wrapper for core functionality of assembly tool MIRA 4.0.  Accepts data from Solexa/Illumina, Roche 454, Ion Torrent, !PacBio and Sanger capillary sequencing.  The key MIRA output files are captured, but the other files are deleted when the job finishes.  Uploaded v0.0.4, previously only on the Test Tool Shed. 
  * [samtools_depad](https://toolshed.g2.bx.psu.edu/view/peterjc/samtools_depad): Runs "samtools depad" to remap a SAM/BAM file using a padded reference (with gap characters) giving a new BAM file using an unpadded (ungapped) reference. Uploaded v0.0.1, previously only on the Test Tool Shed. 
  * [coverage_stats](https://toolshed.g2.bx.psu.edu/view/peterjc/coverage_stats): This tool runs the commands ``samtools idxstats`` and ``samtools depth`` from the SAMtools toolkit, and parses their output to produce a consise summary of the coverage information for each reference sequence.

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [bedtools](https://toolshed.g2.bx.psu.edu/view/iuc/bedtools): bedtools: a powerful toolset for genome arithmetic Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetic: that is, set theory on the genome. For example, bedtools allows one to intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, VCF. While each individual tool is designed to do a relatively simple task (e.g., intersect two interval files), quite sophisticated analyses can be conducted by combining multiple bedtools operations. 
    * Repository-Maintainer: Björn Grüning
    * Repository-Development: https://github.com/galaxy-iuc/tool_shed/

* *From [acrylamide](https://toolshed.g2.bx.psu.edu/view/acrylamide):*
  * [misc_tool_workflow_linkers](https://toolshed.g2.bx.psu.edu/view/acrylamide/misc_tool_workflow_linkers): Contains my Tool Factory tools So far includes only one tool for dealing with seqprep output's gzip'd nature.

* *From [big-tiandm](https://toolshed.g2.bx.psu.edu/view/big-tiandm):*
  * [boost_graph](https://toolshed.g2.bx.psu.edu/view/big-tiandm/boost_graph): perl model  Boost-Graph This is a perl model named Boost-Graph

* *From [pjbriggs](https://toolshed.g2.bx.psu.edu/view/pjbriggs):*
  * [weeder2](https://toolshed.g2.bx.psu.edu/view/pjbriggs/weeder2): Motif discovery in sequences from coregulated genes of a single species. Weeder2 is a program for finding novel motifs (transcription factor binding sites) conserved in a set of regulatory regions of related genes

### Tool Suites

* *From [arkarachai-fungtammasan](https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan):*
  * [microsat_ngs_profiling_suite_trfm](https://toolshed.g2.bx.psu.edu/view/arkarachai-fungtammasan/microsat_ngs_profiling_suite_trfm): all dependency for microsattelite_ngs package and microsattelite_ngs itself

### Packages / Tool Dependency Definitions

* *From [agordon](https://toolshed.g2.bx.psu.edu/view/agordon):*
  * [package_datamash_1_0_6](https://toolshed.g2.bx.psu.edu/view/agordon/package_datamash_1_0_6): GNU Datamash is a grouping and summarizing tool on tabular data files GNU Datamash is a command-line program which performs basic numeric, textual and statistical operations on input textual data files.  it is designed to be portable and reliable, and aid researchers to easily automate analysis pipelines, without writing code or even short scripts. Home page: http://www.gnu.org/software/datamash

* *From [iuc](https://toolshed.g2.bx.psu.edu/view/iuc):*
  * [package_gnuplot_4_6](https://toolshed.g2.bx.psu.edu/view/iuc/package_gnuplot_4_6): Contains a tool dependency definition that downloads and compiles version 4.6 of gnuplot.  Gnuplot is a portable command-line driven graphing utility for Linux, OS/2, MS Windows, OSX, VMS, and many other platforms. It was originally created to allow scientists and students to visualize mathematical functions and data interactively, but has grown to support many non-interactive uses such as web scripting. It is also used as a plotting engine by third-party applications like Octave. Uploaded package as tested on the Test Tool Shed. http://www.gnuplot.info/
    * Repository-Maintainer: Björn Grüning
    * Repository-Development: https://github.com/galaxy-iuc/tool_shed/

* *From [iyad](https://toolshed.g2.bx.psu.edu/view/iyad):*
  * [package_blast_2_2_26](https://toolshed.g2.bx.psu.edu/view/iyad/package_blast_2_2_26): adapted tool_dependencies.xml from Blast Plus 2.2.26 repository. Legacy NCBI Blast tools v2.2.26 Based on the NCBI Blast Plus package repository, this package will build and install the legacy NCBI Blast tools v2.2.26 for various operating systems and architectures.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [package_fastqc_0_11_2](https://toolshed.g2.bx.psu.edu/view/devteam/package_fastqc_0_11_2): Uploaded from GH fastqc v 0.11.2 fastqc v 0.11.2


### Select Updates

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/devteam):*
  * [blast_datatypes](https://toolshed.g2.bx.psu.edu/view/devteam/blast_datatypes):  v0.0.19, adds blastdbp and pssm-asn1 datatypes.

* *From [devteam](https://toolshed.g2.bx.psu.edu/view/tyty):*
  * [structurefold](https://toolshed.g2.bx.psu.edu/view/tyty/structurefold): updated scripts, removed *.pyc and .DS_Store. Uploaded wrapper that correctly handles structure prediction without constraints.

* *From [geert-vandeweyer](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer):*
  * [coverage_report](https://toolshed.g2.bx.psu.edu/view/geert-vandeweyer/coverage_report): new version 0.0.3 (fix on headless R); changed tool.xml to request R 3.0.3; Correction to png calls to use cairo instead of x11. thanks to Eric Enns for pointing  this out.

* *From [peterjc](https://toolshed.g2.bx.psu.edu/view/peterjc):*
  * [effectivet3](https://toolshed.g2.bx.psu.edu/view/peterjc/effectivet3): Uploaded v0.0.13, embed citation, relax test for floating point differences
  * [clinod](https://toolshed.g2.bx.psu.edu/view/peterjc/clinod): Uploaded v0.0.7, uses $GALAXY_SLOTS and embeds citation in tool XML.
  * [predictnls](https://toolshed.g2.bx.psu.edu/view/peterjc/predictnls): Uploaded v0.0.7 with embedded citations
  * [blast_rbh](https://toolshed.g2.bx.psu.edu/view/peterjc/blast_rbh): Uploaded v0.1.5, NCBI BLAST+ 2.2.30 etc
  * [tmhmm_and_signalp](https://toolshed.g2.bx.psu.edu/view/peterjc/tmhmm_and_signalp): Uploaded v0.2.6, embedded citations and uses $GALAXY_SLOTS

* *From [crs4](https://toolshed.g2.bx.psu.edu/view/crs4):*
  * [mosaik2](https://toolshed.g2.bx.psu.edu/view/crs4/mosaik2): Update Orione citation. Upgrade Mosaik dependency to v. 2.2.28 (2.2.30 is buggy, see https://github.com/wanpinglee/MOSAIK/issues/11 ). Use 2.1.78 neural networks. Add package_zlib_1_2_8 and package_samtools_0_1_19 dependencies. Add <citations>.

# Other News

* Doing data-intensive biology in Poland? There's [a server for you](https://galaxy.plgrid.pl)!  [Become a user](http://bit.ly/1u0UDAC). 
* From [Yvan Le Bras](https://twitter.com/Yvan2935): You want to test [Stacks](http://creskolab.uoregon.edu/stacks/) on Galaxy? Find the links towards our Genocloud VM on [GUGGO training page](http://tinyurl.com/ku3etrv) (with a nod to [Julian Catchen](https://twitter.com/jcatchen))
* [Claus from the University of Oslo](https://github.com/Tarostar) developed an [app for checking Galaxy on your phone](https://github.com/Tarostar/QMLGalaxyPortal). [You can help test it](https://biostar.usegalaxy.org/p/9624/).
* From [CRS4 Galaxy](https://twitter.com/CRS4Galaxy): [Seven Mycoplasma hyosynoviae strains assembled using Orione CRS4 Galaxy](http://goo.gl/XMMci1) 
* From [Enis Afgan](https://twitter.com/EnisAfgan): Just wrapped up my first talk at Supercomputing 14: [Deciphering Big Data Stacks: An Overview of Big Data Tools](http://bit.ly/BDStacks).
* From [Robert Davidson](https://twitter.com/bobbledavidson): My conference presentation on metabolomics and Galaxy Project at [#MMW2014](https://twitter.com/hashtag/MMW2014?src=hash) citable via [FigShare](http://dx.doi.org/10.6084/m9.figshare.1243500)
* From [Ron Horst](https://twitter.com/Captnupoz): Been away for a month, come back to a new GVL with Galaxy, IPython, Rstudio. See the [GVL Dashboard](http://130.56.250.190) and GET your own from http://genome.edu.au
* From [Edinburgh Genomics](https://twitter.com/edgenome): Workshop announcement (Feb 2015): "[Introduction to Python for Biologists](http://bit.ly/1uKSsBu)"
* [Discussion on Command to compare repositories in different Tool Sheds](https://github.com/galaxyproject/planemo/issues/27) started by Peter Cock.
* December 31, 2014, is the last day for submissions to [GigaScience](http://www.gigasciencejournal.com/) that will not be charged an article processing charge.
